| Tool                                                   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Topics                                                     | Functionality                         | Type                                | License      | How To   |
|:-------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------|:--------------------------------------|:------------------------------------|:-------------|:---------|
| [FAIRifier](https://bio.tools/FAIRifier)               | A general-purpose FAIRifier on the basis of the OpenRefine data cleaning and wrangling tool and the RDF plugin. This FAIRifier enables a post-hoc FAIRification workflow: load an existing dataset, perform data wrangling tasks, add FAIR attributes to the data, generate a linked data version of the data and, finally, push the result to an online FAIR data infrastructure to make it accessible and discoverable. Literal values in a dataset can be replaced by identifiers either manually or by embedded, customizable script expressions. The interoperability of the dataset can be improved by connecting these identifiers into a meaningful semantic graph-structure of ontological classes and properties using the integrated RDF model editor. A provenance trail automatically keeps track of each modification and additionally enables “undo” operations and repetition of operations on similar datasets. A FAIR data export function opens up a metadata editor to provide information about the dataset itself. | <ul><li>Data submission, annotation and curation</li></ul> | <ul></ul>                             | <ul><li>Web application</li></ul>   |              |          |
| [GeoBoost2](https://bio.tools/geoboost2)               | A natural languageprocessing pipeline for GenBank metadata enrichment for virus phylogeography.<br>GeoBoost2 (part of the ZoDo project) is a system for extracting the location of infected hosts (LOIH) for a given set of GenBank metadata accessions. This involves API implementations that utilize NCBI API's for accessing and parsing information and delivering it to the researcher interested in investigating a set of sequences. You can also use this tool mine Geographic Locations in scientific articles (available as abstracts on PubMed or PMC-OA) or text files.                                                                                                                                                                                                                                                                                                                                                                                                                                                     | <ul><li>Biotechnology</li></ul>                            | <ul><li>Data retrieval</li></ul>      | <ul><li>Web application</li></ul>   | Apache-2.0   |          |
| [MartView](https://bio.tools/martview)                 | Tool for data retrieval and data mining that integrates data from Ensembl. Through the web interface it allows you to apply a series of filters to create custom datasets which can be converted to several useful output formats.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | <ul><li>Gene transcripts</li></ul>                         | <ul><li>Data retrieval</li></ul>      | <ul><li>Web application</li></ul>   |              |          |
| [DISQOVER](https://bio.tools/disqover)                 | DISQOVER is a data integration platform for public, licensed and internal data. The Data Ingestion Engine enables transforming data into Linked Data which can be searched, navigated and analysed via the user interface and the API. The publicly accessible DISQOVER platform contains 140+ public data sources in life sciences and related domains.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | <ul><li>Biology</li></ul>                                  | <ul><li>Filtering</li></ul>           | <ul><li>Database portal</li></ul>   | Unlicense    |          |
| [Sigflow](https://bio.tools/sigflow)                   | An automated and comprehensive pipeline for cancer genome mutational signature analysis.<br>Sigflow: Streamline Analysis Workflows for Mutational Signatures.<br>Sigflow provides useful mutational signature analysis workflows. It can auto-extract mutational signatures, fit mutation data to all/specified COSMIC reference signatures (SBS/DBS/INDEL) and run bootstrapping analysis for studying signature stability.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | <ul><li>Oncology</li></ul>                                 | <ul><li>Variant calling</li></ul>     | <ul></ul>                           | AFL-3.0      |          |
| [PGA](https://bio.tools/pga)                           | This package provides functions for construction of customized protein databases based on RNA-Seq data with/without genome guided, database searching, post-processing and report generation. This kind of customized protein database includes both the reference database (such as Refseq or ENSEMBL) and the novel peptide sequences form RNA-Seq data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | <ul><li>Data submission, annotation and curation</li></ul> | <ul><li>RNA-Seq analysis</li></ul>    | <ul><li>Library</li></ul>           | GPL-2.0      |          |
| [CLOBNET](https://bio.tools/CLOBNET)                   | Open Source Infrastructure for Health Care Data Integration and Machine Learning Analyses <br> PURPOSE:We have created a cloud-based machine learning system (CLOBNET) that is an open-source, lean infrastructure for electronic health record (EHR) data integration and is capable of extract, transform, and load (ETL) processing. CLOBNET enables comprehensive analysis and visualization of structured EHR data. We demonstrate the utility of CLOBNET by predicting primary therapy outcomes of patients with high-grade serous ovarian cancer (HGSOC) on the basis of EHR data. MATERIALS AND METHODS:CLOBNET is built using open-source software to make data preprocessing, analysis, and model training user friendly. The source code of CLOBNET is available in GitHub.                                                                                                                                                                                                                                                   | <ul><li>Machine learning</li></ul>                         | <ul></ul>                             | <ul></ul>                           | BSD-2-Clause |          |
| [Chemotion](https://bio.tools/chemotion)               | Chemotion is a repository for chemistry research data that provides solutions for current challenges to store research data in a feasible manner, allowing the conservation of domain specific information in a machine readable format. A main advantage of the repository Chemotion is the comprehensive functionality, which offers options to collect, prepare and reuse data using discipline specific methods and data processing tools. For selected analytical data, automated procedures are implemented to facilitate the curation of the data. Chemotion provides functions to facilitate the publishing process of data and the citation of the deposited data.                                                                                                                                                                                                                                                                                                                                                              | <ul><li>Workflows</li></ul>                                | <ul><li>Anonymisation</li></ul>       | <ul><li>Database portal</li></ul>   |              |          |
| [Query Tabular](https://bio.tools/Query_Tabular)       | Query Tabular is a Galaxy-based tool which manipulates tabular files.  Query Tabular automatically creates a SQLite database directly from a tabular file within a Galaxy workflow.  The SQLite database can be saved to the Galaxy history, and further process to generate tabular outputs containing desired information and formatting.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | <ul><li>Workflows</li></ul>                                | <ul><li>Query and retrieval</li></ul> | <ul><li>Command-line tool</li></ul> | CC-BY-4.0    |          |
| [PDBUSQLExtractor](https://bio.tools/PDBUSQLExtractor) | Scalable Extraction of Big Macromolecular Data in Azure Data Lake Environment.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | <ul><li>Nucleic acid structure analysis</li></ul>          | <ul><li>Parsing</li></ul>             | <ul><li>Library</li></ul>           | GPL-3.0      |          |