| Tool                                               | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Topics                                                     | Functionality                                  | Type                                | License      | How To   |
|:---------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------|:-----------------------------------------------|:------------------------------------|:-------------|:---------|
| [MySeq](https://bio.tools/MySeq)                   | Privacy-protecting browser-based personal Genome analysis for genomics education and exploration.<br>MySeq is a web-application for privacy-protecting interactive analysis of personal genomes (distributed as compressed-and-indexed VCF files) inspired by GENOtation (previously the Interpretome) and DNA.LAND Compass. MySeq is intended for use as a genomics educational platform.<br>Analyzing the PTC Tasting Phenotype with MySeq.<br>This page is an example analysis of the "bitter tasting" trait using the MySeq application in an embedded context. Here MySeq is used to both query a whole genome VCF for NA12878 (from Genome in a Bottle) by genomic coordinates and predict the bitter tasting phenotype directly. All of the queries demonstrated here are performed "live" in the browser, that is these are not pre-generated results. Try MySeq as a "standalone" application.<br>MySeq is a single-page web application for privacy-protecting personal genome analysis. | <ul><li>Informatics</li></ul>                              | <ul><li>Anonymisation</li></ul>                | <ul><li>Web application</li></ul>   |              |          |
| [Query Tabular](https://bio.tools/Query_Tabular)   | Query Tabular is a Galaxy-based tool which manipulates tabular files.  Query Tabular automatically creates a SQLite database directly from a tabular file within a Galaxy workflow.  The SQLite database can be saved to the Galaxy history, and further process to generate tabular outputs containing desired information and formatting.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | <ul><li>Bioinformatics</li></ul>                           | <ul><li>Parsing</li></ul>                      | <ul><li>Command-line tool</li></ul> | CC-BY-4.0    |          |
| [SNAPPy](https://bio.tools/SNAPPy)                 | A snakemake pipeline for scalable HIV-1 subtyping by phylogenetic pairing <br> SNAPPy is a Snakemake pipeline for HIV-1 subtyping by phylogenetic pairing <br> This is the repository for SNAPPy, a Snakemake pipeline for HIV-1 subtyping by phylogenetic pairing. SNAPPy allows high-throughput HIV-1 subtyping locally while being resource efficient and scalable. This pipeline was constructed using Snakemake , and it uses MAFFT and for multiple sequence alignment, BLAST for similarirty querys, IQ-TREE for phylogenetic inference, and several Biopython modules for data parsing an analysis. For in-depth information on how the tool works please visit the documentation page. SNAPPy was design for Linux based operative systems <br> Welcome to snappy’s documentation! — SNAPPy-HIV1-Subtyping 1.0.0 documentation <br> Free document hosting provided by Read the Docs.                                                                                                      | <ul><li>Phylogenetics</li></ul>                            | <ul><li>Parsing</li></ul>                      | <ul></ul>                           | MIT          |          |
| [pmparser](https://bio.tools/pmparser)             | Pmparser enables one to easily create and maintain a relational database of data from PubMed/MEDLINE. pmparser can download the publicly available XML files, parse them, incorporate PubMed’s regular updates, and combine the data with the NIH Open Citation Collection. PMDB, our implementation of the database, is available to download on Zenodo. For a detailed description of pmparser and PMDB, check out the preprint.<br>The tar file contains a compressed dump of PMDB, which was created in PostgreSQL 12 using the pmparser R package.                                                                                                                                                                                                                                                                                                                                                                                                                                            | <ul><li>Medicine</li></ul>                                 | <ul><li>Parsing</li></ul>                      | <ul></ul>                           | GPL-2.0      |          |
| [GenEx_ontology](https://bio.tools/genex_ontology) | Enabling semantic queries across federated bioinformatics databases.<br>Ontology Documentation generated by WIDOCO.<br>Gene expression (GenEx) semantic model.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | <ul><li>Gene expression</li></ul>                          | <ul><li>ID mapping</li></ul>                   | <ul></ul>                           |              |          |
| [FamFetch](https://bio.tools/famfetch)             | FamFetch is the general interface used to query the families databases developed using the HOBACGEN model.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | <ul><li>Phylogenetics</li></ul>                            | <ul><li>Query</li></ul>                        | <ul><li>Command-line tool</li></ul> |              |          |
| [biomaRt](https://bio.tools/biomart)               | Provides an interface  to a growing collection of databases implementing the BioMart software suite, that is a database portal formed by dozens of biological databases spanning genomics, protemics, model organisms, cancer data, ontology information and more. The package enables retrieval of large amounts of data in a uniform way without the need to know the underlying database schemas or write complex SQL queries.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | <ul><li>Database management</li></ul>                      | <ul><li>Structural similarity search</li></ul> | <ul><li>Command-line tool</li></ul> | Artistic-2.0 |          |
| [REDCap](https://bio.tools/redcap)                 | Secure web application for building and managing online surveys and databases.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | <ul><li>Data submission, annotation and curation</li></ul> | <ul><li>Loading</li></ul>                      | <ul><li>Web application</li></ul>   | Other        |          |
| [cellbaseR](https://bio.tools/cellbaser)           | This package makes use of the exhaustive RESTful Web service API that has been implemented for the Cellabase database. It enable researchers to query and obtain a wealth of biological information from a single database saving a lot of time. Another benefit is that researchers can easily make queries about different biological topics and link all this information together as all information is integrated.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            | <ul><li>Database management</li></ul>                      | <ul><li>Annotation</li></ul>                   | <ul><li>Library</li></ul>           | Apache-2.0   |          |
| [eTBLAST](https://bio.tools/etblast)               | Textual similarity search engine. This server can parse and summarize the results of an abstract similarity search to find appropriate journals for publication, authors with expertise in a given field, and documents similar to a submitted query.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | <ul><li>Natural language processing</li></ul>              | <ul><li>Structural similarity search</li></ul> | <ul><li>Web application</li></ul>   |              |          |