
<head>

    <link rel="stylesheet" href="https://cdn.datatables.net/1.10.21/css/jquery.dataTables.min.css">
    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.21/css/jquery.dataTables.min.css" />
    <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
    <script src="https://cdn.datatables.net/1.10.21/js/jquery.dataTables.min.js"></script>

<style>

#title {
  font-size: 2em;
  font-family: sans-serif;
  text-align: center;
}

#subtitle {
  font-size: 1em;
  font-family: sans-serif;
  text-align: center;
}

.parameters {
  font-size: 1em;
  font-family: sans-serif;
  width: 80%;
  margin-left: auto;
  margin-right: auto;
}

.styled-table{
  width: 95%;
  margin-left: auto;
  margin-right: auto;
  font-size: 0.9em;
  font-family: sans-serif;
  white-space: pre-line;
}

.dataTable {
    border-collapse: collapse;
    font-size: 0.9em;
    font-family: sans-serif;
    width: 80%;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.15);
}


.dataTable thead tr {
    background-color: #156094;
    color: #ffffff;
    text-align: left;
    padding-top: 5%;
}

table.dataTable td {
    min-width: 50px;
    box-sizing: border-box;
}

table.dataTable thead tr th input{
    min-width: 50px;
    max-width: 200px;
    width:100%;
    box-sizing: border-box;
}

.dataTable tbody tr {
    border-bottom: thin solid #dddddd;
}

.dataTable tbody tr:nth-of-type(even) {
    background-color: #f3f3f3;
}

.dataTables_filter{
  margin-top: 1%;
  margin-bottom: 1%;
}

.dataTables_length{
  margin-top: 1%;
  margin-bottom: 1%;
}

.short{
  max-height: 150px;
  overflow: hidden;
}

.link{
  max-width: 200px;
  font-size: 0.9em;
  padding-left: 2%;
  padding-top: 2%;
  word-wrap: break-word;
}

.citations{
    max-width: 50px;
    text-align: center;
}

.name {
    min-width: 120px;
}

.desc {
  text-align: left;
  padding-left: 1.2%;
  padding-top: 1.2%;
  padding-right:1.2%;
  padding-bottom: 1.2%;
  font-size: 0.9em;
  min-width: 200px;
}

.type {
    min-width: 180px;
    vertical-align: text-top;
}

.topic {
    min-width: 200px;
    vertical-align: text-top;
}

.operation {
    min-width: 200px;
    vertical-align: text-top;
}

.formats {
    min-width: 200px;
    vertical-align: text-top;
}

</style>

</style>

      
</head>
<body>

    ​​​<h1 id=title>Tools discovery results</h1>
    <h2 id=subtitle>identifier_minting_mapping</h2>
        <div class=parameters>
        <h3> Search parameters: </h3>
        <ul>
            <li><span style="font-weight: bold">Name</span>: identifier_minting_mapping</li>
            <li><span style="font-weight: bold">Keywords</span>: Identifier minting, Identifier resolution, id generation, identifier generation, Globally Unique Identifier, Universally Unique Identifier, identifier mapping, id generation, id mapping, database cross reference</li>
        </ul> 
    </div>
    <div class=parameters>
        <h3> Results: </h3>
    
        <div class="styled-table">
        <table border="0" class="dataframe" id="my-table">
  <thead>
    <tr style="text-align: right;">
      <th style="min-width: 50px;">Tool</th>
      <th style="min-width: 50px;">Score</th>
      <th style="min-width: 50px;">Description</th>
      <th style="min-width: 50px;">Topics</th>
      <th style="min-width: 50px;">Functionality</th>
      <th style="min-width: 50px;">Inputs</th>
      <th style="min-width: 50px;">Outputs</th>
      <th style="min-width: 50px;">Type</th>
      <th style="min-width: 50px;">Citations</th>
      <th style="min-width: 50px;">URL</th>
      <th style="min-width: 50px;">License</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/organism.dplyr" target="_blank">Organism.dplyr</a></div></td>
      <td><div class='short click_expand'>0.800</div></td>
      <td><div class='desc short click_expand'>This package provides an alternative interface to Bioconductor 'annotation' resources, in particular the gene identifier mapping functionality of the 'org' packages (e.g., org.Hs.eg.db) and the genome coordinate functionality of the 'TxDb' packages</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/Organism.dplyr/" target="_blank">http://bioconductor.org/packages/Organism.dplyr/</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bridgedbr" target="_blank">BridgeDbR</a></div></td>
      <td><div class='short click_expand'>0.800</div></td>
      <td><div class='desc short click_expand'>Use BridgeDb functions and load identifier mapping databases in R.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data identity and mapping</li><li>Rare diseases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>ID mapping</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>compound identifier : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>compound identifier : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bridgedb/BridgeDb" target="_blank">https://github.com/bridgedb/BridgeDb</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/idmappinganalysis" target="_blank">IdMappingAnalysis</a></div></td>
      <td><div class='short click_expand'>0.800</div></td>
      <td><div class='desc short click_expand'>Identifier mapping performance analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>plot : image format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bridgedb_api" target="_blank">BridgeDb API</a></div></td>
      <td><div class='short click_expand'>0.800</div></td>
      <td><div class='desc short click_expand'>BridgeDb is a framework for finding and mapping equivalent database identifiers. It has many facets: it is both a framework, live services, and are identifier mapping files for genes, proteins, and metabolites.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data identity and mapping</li><li>Rare diseases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>62</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/idmappingretrieval" target="_blank">IdMappingRetrieval</a></div></td>
      <td><div class='short click_expand'>0.800</div></td>
      <td><div class='desc short click_expand'>Data retrieval for identifier mapping performance analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Databio" target="_blank">Datab.io</a></div></td>
      <td><div class='short click_expand'>0.800</div></td>
      <td><div class='desc short click_expand'>Automated Gene Data Integration with Databio | Automated data transformation for bioinformatics | Automated data integration tools | A simple Gene ID mapping service for Bioinformatics. Upload a gene-oriented data set, we'll automatically detect the identifiers in use, and offer to convert them to match other providers in just a few clicks | A web portal that does automated identifier translation for bioinformatics</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li><li>Protein expression</li><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Data retrieval</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/joiningdata/databio/" target="_blank">https://github.com/joiningdata/databio/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bridgedb" target="_blank">BridgeDb</a></div></td>
      <td><div class='short click_expand'>0.800</div></td>
      <td><div class='desc short click_expand'>BridgeDb is a framework to map identifiers between various databases.  It includes a Java library that provides an API to work with identifier-identifier mapping databases and resources.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li><li>Rare diseases</li><li>Proteomics</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>61</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/upimapi" target="_blank">UPIMAPI</a></div></td>
      <td><div class='short click_expand'>0.800</div></td>
      <td><div class='desc short click_expand'>UniProt Id Mapping through API</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>ID mapping</li><li>Gene functional annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein sequence record : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/iquasere/UPIMAPI/issues" target="_blank">https://github.com/iquasere/UPIMAPI/issues</a></div></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ah-db" target="_blank">AH-DB</a></div></td>
      <td><div class='short click_expand'>0.800</div></td>
      <td><div class='desc short click_expand'>Database of protein structure pairs (apo-holo structure pairs) before and after binding. AH-DB is designed to provide an easy and unified way to prepare apo-holo structure pair data for studies. Data is generated by identifying/mapping molecules in Protein Data Bank (PDB) entries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Protein structure analysis</li><li>Proteins</li><li>Structure prediction</li><li>Experimental design and studies</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structure assignment</li><li>Mapping</li><li>ID mapping</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/seqxref" target="_blank">seqxref</a></div></td>
      <td><div class='short click_expand'>0.800</div></td>
      <td><div class='desc short click_expand'>Retrieve all database cross-references for a sequence entry.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
  </tbody>
</table>
        </div>
    </div>
    

</body>

<script>
    $('#my-table').dataTable( {
      "order": [],
      } );

    var userSelection = document.getElementsByClassName('click_expand');

    for(var i = 0; i < userSelection.length; i++) {
      (function(index) {
        userSelection[index].addEventListener("click", function() {
          console.log("Clicked index: ");
          $(this).closest("tr").find('div').toggleClass("short");

        })
      })(i);
    }

    
    $('#my-table thead th').each(function() {
      var title = $('#my-table thead th').eq($(this).index()).text();
      $(this).html(title+'</br><input type="text" placeholder="Search"' + title + '/>');
      $(this).css('text-align', 'left');
      });

    var r = $('#my-table thead th');
      r.find('input').each(function(){
      $(this).css('margin', 4);
      $(this).css('padding', 4);
    });

    // DataTable
    var table = $('#my-table').DataTable();

    // Apply the search
    table.columns().eq(0).each(function(colIdx) {
        $('input', table.column(colIdx).header()).on('keyup change', function() {
          table
                .column(colIdx)
                .search(this.value)
                .draw();
        });

        $('input', table.column(colIdx).header()).on('click', function(e) {
            e.stopPropagation();
        });
    });

    
</script>
