
<head>

    <link rel="stylesheet" href="https://cdn.datatables.net/1.10.21/css/jquery.dataTables.min.css">
    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.21/css/jquery.dataTables.min.css" />
    <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
    <script src="https://cdn.datatables.net/1.10.21/js/jquery.dataTables.min.js"></script>

<style>

#title {
  font-size: 2em;
  font-family: sans-serif;
  text-align: center;
}

#subtitle {
  font-size: 1em;
  font-family: sans-serif;
  text-align: center;
}

.parameters {
  font-size: 1em;
  font-family: sans-serif;
  width: 80%;
  margin-left: auto;
  margin-right: auto;
}

.styled-table{
  width: 95%;
  margin-left: auto;
  margin-right: auto;
  font-size: 0.9em;
  font-family: sans-serif;
  white-space: pre-line;
}

.dataTable {
    border-collapse: collapse;
    font-size: 0.9em;
    font-family: sans-serif;
    width: 80%;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.15);
}


.dataTable thead tr {
    background-color: #156094;
    color: #ffffff;
    text-align: left;
    padding-top: 5%;
}

table.dataTable td {
    min-width: 50px;
    box-sizing: border-box;
}

table.dataTable thead tr th input{
    min-width: 50px;
    max-width: 200px;
    width:100%;
    box-sizing: border-box;
}

.dataTable tbody tr {
    border-bottom: thin solid #dddddd;
}

.dataTable tbody tr:nth-of-type(even) {
    background-color: #f3f3f3;
}

.dataTables_filter{
  margin-top: 1%;
  margin-bottom: 1%;
}

.dataTables_length{
  margin-top: 1%;
  margin-bottom: 1%;
}

.short{
  max-height: 150px;
  overflow: hidden;
}

.link{
  max-width: 200px;
  font-size: 0.9em;
  padding-left: 2%;
  padding-top: 2%;
  word-wrap: break-word;
}

.citations{
    max-width: 50px;
    text-align: center;
}

.name {
    min-width: 120px;
}

.desc {
  text-align: left;
  padding-left: 1.2%;
  padding-top: 1.2%;
  padding-right:1.2%;
  padding-bottom: 1.2%;
  font-size: 0.9em;
  min-width: 200px;
}

.type {
    min-width: 180px;
    vertical-align: text-top;
}

.topic {
    min-width: 200px;
    vertical-align: text-top;
}

.operation {
    min-width: 200px;
    vertical-align: text-top;
}

.formats {
    min-width: 200px;
    vertical-align: text-top;
}

</style>

</style>

      
</head>
<body>

    
        <div class="styled-table">
        <table border="0" class="dataframe" id="my-table">
  <thead>
    <tr style="text-align: right;">
      <th style="min-width: 50px;">Tool</th>
      <th style="min-width: 50px;">Score</th>
      <th style="min-width: 50px;">Description</th>
      <th style="min-width: 50px;">Topics</th>
      <th style="min-width: 50px;">Functionality</th>
      <th style="min-width: 50px;">Inputs</th>
      <th style="min-width: 50px;">Outputs</th>
      <th style="min-width: 50px;">Type</th>
      <th style="min-width: 50px;">Citations</th>
      <th style="min-width: 50px;">URL</th>
      <th style="min-width: 50px;">License</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MySeq" target="_blank">MySeq</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>Privacy-protecting browser-based personal Genome analysis for genomics education and exploration.<br>MySeq is a web-application for privacy-protecting interactive analysis of personal genomes (distributed as compressed-and-indexed VCF files) inspired by GENOtation (previously the Interpretome) and DNA.LAND Compass. MySeq is intended for use as a genomics educational platform.<br>Analyzing the PTC Tasting Phenotype with MySeq.<br>This page is an example analysis of the "bitter tasting" trait using the MySeq application in an embedded context. Here MySeq is used to both query a whole genome VCF for NA12878 (from Genome in a Bottle) by genomic coordinates and predict the bitter tasting phenotype directly. All of the queries demonstrated here are performed "live" in the browser, that is these are not pre-generated results. Try MySeq as a "standalone" application.<br>MySeq is a single-page web application for privacy-protecting personal genome analysis</div></td>
      <td><div class='topic short click_expand'><ul><li>Exome sequencing</li><li>Pharmacogenomics</li><li>Genotype and phenotype</li><li>Whole genome sequencing</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Query and retrieval</li><li>Genotyping</li><li>Parsing</li><li>Anonymisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/mlinderm/myseq" target="_blank">https://github.com/mlinderm/myseq</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Query_Tabular" target="_blank">Query Tabular</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>Query Tabular is a Galaxy-based tool which manipulates tabular files.  Query Tabular automatically creates a SQLite database directly from a tabular file within a Galaxy workflow.  The SQLite database can be saved to the Galaxy history, and further process to generate tabular outputs containing desired information and formatting.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Filtering</li><li>Query and retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>text : tsv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>text : tsv</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://toolshed.g2.bx.psu.edu/view/iuc/query_tabular/0c95a3f1654f" target="_blank">https://toolshed.g2.bx.psu.edu/view/iuc/query_tabular/0c95a3f1654f</a></div></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SNAPPy" target="_blank">SNAPPy</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>A snakemake pipeline for scalable HIV-1 subtyping by phylogenetic pairing | SNAPPy is a Snakemake pipeline for HIV-1 subtyping by phylogenetic pairing | This is the repository for SNAPPy, a Snakemake pipeline for HIV-1 subtyping by phylogenetic pairing. SNAPPy allows high-throughput HIV-1 subtyping locally while being resource efficient and scalable. This pipeline was constructed using Snakemake , and it uses MAFFT and for multiple sequence alignment, BLAST for similarirty querys, IQ-TREE for phylogenetic inference, and several Biopython modules for data parsing an analysis. For in-depth information on how the tool works please visit the documentation page. SNAPPy was design for Linux based operative systems | Welcome to snappy’s documentation! — SNAPPy-HIV1-Subtyping 1.0.0 documentation | Free document hosting provided by Read the Docs</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Public health and epidemiology</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Sequence trimming</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pmparser" target="_blank">pmparser</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>Pmparser enables one to easily create and maintain a relational database of data from PubMed/MEDLINE. pmparser can download the publicly available XML files, parse them, incorporate PubMed’s regular updates, and combine the data with the NIH Open Citation Collection. PMDB, our implementation of the database, is available to download on Zenodo. For a detailed description of pmparser and PMDB, check out the preprint.<br>The tar file contains a compressed dump of PMDB, which was created in PostgreSQL 12 using the pmparser R package.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Biological databases</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://doi.org/10.5281/zenodo.4008109" target="_blank">https://doi.org/10.5281/zenodo.4008109</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genex_ontology" target="_blank">GenEx_ontology</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>Enabling semantic queries across federated bioinformatics databases.<br>Ontology Documentation generated by WIDOCO.<br>Gene expression (GenEx) semantic model.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Database management</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/famfetch" target="_blank">FamFetch</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>FamFetch is the general interface used to query the families databases developed using the HOBACGEN model.</div></td>
      <td><div class='topic short click_expand'><ul><li>Databases</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>121</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/famfetch-2-0-search-tree-patterns-databases-phylogenetic-trees.html" target="_blank">http://www.mybiosoftware.com/famfetch-2-0-search-tree-patterns-databases-phylogenetic-trees.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biomart" target="_blank">biomaRt</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>Provides an interface  to a growing collection of databases implementing the BioMart software suite, that is a database portal formed by dozens of biological databases spanning genomics, protemics, model organisms, cancer data, ontology information and more. The package enables retrieval of large amounts of data in a uniform way without the need to know the underlying database schemas or write complex SQL queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Molecular interactions, pathways and networks</li><li>Data acquisition</li><li>Model organisms</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1,704</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/redcap" target="_blank">REDCap</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>Secure web application for building and managing online surveys and databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Database management</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>13,528</div></td>
      <td></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cellbaser" target="_blank">cellbaseR</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>This package makes use of the exhaustive RESTful Web service API that has been implemented for the Cellabase database. It enable researchers to query and obtain a wealth of biological information from a single database saving a lot of time. Another benefit is that researchers can easily make queries about different biological topics and link all this information together as all information is integrated.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/melsiddieg/cellbaseR" target="_blank">https://github.com/melsiddieg/cellbaseR</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/etblast" target="_blank">eTBLAST</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>Textual similarity search engine. This server can parse and summarize the results of an abstract similarity search to find appropriate journals for publication, authors with expertise in a given field, and documents similar to a submitted query.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Query and retrieval</li><li>Database search</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>130</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/X2H" target="_blank">X2H</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>Massive XML-based biological data management platform.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/lyotvincent/X2H/issues" target="_blank">https://github.com/lyotvincent/X2H/issues</a></div></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pubcrawler" target="_blank">PubCrawler</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>It goes to the library. You go to the pub; receive email alerts for current contents of PubMed and GenBank; e.g. use accession number of htg record as query to receive sequence updates (as the version number changes).</div></td>
      <td><div class='topic short click_expand'><ul><li>Medicine</li><li>Sequence composition, complexity and repeats</li><li>Literature and language</li><li>Nucleic acids</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Query and retrieval</li><li>Data retrieval</li><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>37</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/festival" target="_blank">FESTIval</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>FESTIval is a versatile framework for conducting experimental evaluations of spatial indices. FESTIval also provides default spatial datasets for evaluating spatial indices. Algorithms to generate query windows for range queries and points for point queries are further provided.</div></td>
      <td><div class='topic short click_expand'><ul><li>Laboratory techniques</li><li>Literature and language</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Quantification</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Workbench</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/LINCS_Data_Portal_2.0" target="_blank">LINCS Data Portal 2.0</a></div></td>
      <td><div class='short click_expand'>1.000</div></td>
      <td><div class='desc short click_expand'>Next generation access point for perturbation-response signatures.<br>The Library of Integrated Network-Based Cellular Signatures (LINCS) is an NIH Common Fund program with the goal of generating a large-scale and comprehensive catalogue of perturbation-response signatures by utilizing a diverse collection of perturbations across many model systems and assay types. The LINCS Data Portal (LDP) has been the primary access point for the compendium of LINCS data and has been widely utilized. Here, we report the first major update of LDP (http: lincsportal.ccs.miami.edu signatures) with substantial changes in the data architecture and APIs, a completely redesigned user interface, and enhanced curated metadata annotations to support more advanced, intuitive and deeper querying, exploration and analysis capabilities.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data architecture, analysis and design</li><li>Drug discovery</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CIViCpy" target="_blank">CIViCpy</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A Python software development and analysis toolkit for the CIViC knowledgebase | Clinical Interpretations of Variants in Cancer | CIViCpy — civicpy 1.0.0-rc1 documentation | A python interface for the CIViC db application | Sorry, something went wrong. Reload? | Sorry, we cannot display this file | CIViCpy is an analysis toolkit and SDK for extracting and analyzing knowledge from the CIViC knowledgebase | You have reached the code repository for CIViCpy, a python client and analysis toolkit for the Clinical Interpretations of Variants in Cancer knowledgebase (CIViC) | Sorry, this file is invalid so it cannot be displayed | CIViCpy lets you pull data from CIViC using the civic module and interact with records as dynamic objects. With the aid of caching, it is easy to explore relationships between CIViC records (e.g. assertions, genes, variants) without worrying about querying CIViC more than once for the same data</div></td>
      <td><div class='topic short click_expand'><ul><li>Allergy, clinical immunology and immunotherapeutics</li><li>Preclinical and clinical studies</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Parsing</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="http://git.io/civicpy" target="_blank">http://git.io/civicpy</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fasttargetpred" target="_blank">FastTargetPred</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>FastTargetPred is a program enabling the fast prediction of putative protein targets for input chemical databases. It allows to predict putative protein target(s) for a query compound or for a collection of compounds.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein targeting and localisation</li><li>Compound libraries and screening</li><li>Molecular biology</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Subcellular localisation prediction</li><li>Virtual screening</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dichromatch" target="_blank">DichroMatch</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>DichroMatch webserver identifies proteins with similar spectral characteristics as a way to detect possible structurally and functionally related proteins and homologues. Users input a query spectrum which is searched against the protein circular dichromism data bank or a user provided data set.</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>Protein secondary structure</li><li>Proteins</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein secondary structure comparison</li><li>Protein secondary structure assignment</li><li>Query and retrieval</li><li>Protein secondary structure analysis</li><li>Protein structure comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rescript" target="_blank">RESCRIPt</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Reproducible sequence taxonomy reference database management for the masses.<br>REference Sequence annotation and CuRatIon Pipeline.<br>RESCRIPt is a python 3 package to support a variety of operations for managing and curating reference sequence databases, DNA/RNA sequence data, and taxonomic data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microbial ecology</li><li>Taxonomy</li><li>Sequencing</li><li>Metagenomics</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Deposition</li><li>Sequence trimming</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hmdb" target="_blank">HMDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Freely available electronic database containing detailed information about small molecule metabolites found in the human body. The database supports extensive text, sequence, chemical structure and relational query searches.</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>Metabolomics</li><li>Surgery</li><li>Model organisms</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Database search</li><li>Structure database search</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3,130</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/boag" target="_blank">BoaG</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>BoaG is a language and an infrastructure for deeper analysis of the public genomics databases. The BoaG database is a domain-specific language and shared data science infrastructure for genomics, along with a CD-HIT clustering of all these protein sequences at different sequence similarity levels.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenomics</li><li>Biological databases</li><li>Proteins</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Query and retrieval</li><li>Parsing</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://boa.cs.iastate.edu/boag/?q=boa/job/public/31" target="_blank">http://boa.cs.iastate.edu/boag/?q=boa/job/public/31</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/targetmine" target="_blank">TargetMine</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Integrated data warehouse for retrieval of target genes and proteins for experimental characterisation and drug discovery. Provides integration of custom annotations and in-house experimental data, as well as it facilitates the analysis of query gene/protein sets for enriched biological themes and associations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Drug discovery</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>94</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/targetmine-20121115-data-warehouse-candidate-gene-prioritisation-target-discovery.html" target="_blank">http://www.mybiosoftware.com/targetmine-20121115-data-warehouse-candidate-gene-prioritisation-target-discovery.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/torque" target="_blank">Torque</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Web server for querying of protein-protein interaction networks or pathways across species. Networks of up to 25 proteins may be input.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li><li>Protein interaction experiment</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network comparison</li><li>Pathway or network prediction</li><li>Query and retrieval</li><li>Pathway or network visualisation</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>28</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/miRPathDB" target="_blank">miRPathDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A novel release of the miRNA Pathway Dictionary Database.<br>Since the initial release of miRPathDB, tremendous progress has been made in the field of microRNA (miRNA) research. New miRNA reference databases have emerged, a vast amount of new miRNA candidates has been discovered and the number of experimentally validated target genes has increased considerably. Hence, the demand for a major upgrade of miRPathDB, including extended analysis functionality and intuitive visualizations of query results has emerged. Here, we present the novel release 2.0 of the miRNA Pathway Dictionary Database (miRPathDB) that is freely accessible at https: mpd.bioinf.uni-sb.de. miRPathDB 2.0 comes with a ten-fold increase of pre-processed data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Functional, regulatory and non-coding RNA</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Query and retrieval</li><li>miRNA target prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>19</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rupee" target="_blank">RUPEE</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Protein structure search to support the development of protein structure prediction methods.<br><br>RUPEE: A fast and accurate purely geometric protein structure search.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structural motifs and surfaces</li><li>Sequence analysis</li><li>Structure prediction</li><li>Database management</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structure prediction</li><li>Fold recognition</li><li>Structure alignment</li><li>Parsing</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/rayoub/rupee" target="_blank">https://github.com/rayoub/rupee</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/chemdraw_ultra" target="_blank">ChemDraw Ultra</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Used for drawing accurate, chemically-aware structures for use in database queries, preparation of publication-quality graphics, and entry for modeling and other programs that require an electronic description of molecules and reactions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Chemical structures</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/chemdraw-ultra-chemdraw-activexplugin-viewer-12-0-draw-view-chemical-structures.html" target="_blank">http://www.mybiosoftware.com/chemdraw-ultra-chemdraw-activexplugin-viewer-12-0-draw-view-chemical-structures.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biomoby" target="_blank">BioMoby</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>BioMOBY is an international research project involving biological data hosts, biological data service providers, and coders whose aim is to explore various methodologies for biological data representation, distribution, and discovery.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Ontology and terminology</li><li>Sequence assembly</li><li>Database management</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Service discovery</li><li>Service composition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>299</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/plethy" target="_blank">plethy</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>This package provides the infrastructure and tools to import, query and perform basic analysis of whole body plethysmography and metabolism data. Currently support is limited to data derived from Buxco respirometry instruments as exported by their FinePointe software.</div></td>
      <td><div class='topic short click_expand'><ul><li>Human biology</li><li>Endocrinology and metabolism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Analysis</li><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/FastFeatGen" target="_blank">FastFeatGen</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Faster parallel feature extraction from genome sequences and efficient prediction of DNA N6-methyladenine sites.<br>Faster parallel feature extraction from genome sequence.<br>This is a tool for faster feature extraction from genome sequences and making efficient prediction. To build efficient prediction model, user can go through the following instructions step by step. If user only wants to predict query sequences from our built model, then just go to step Make prediction for query sequences. Currently, FastFeatGen supports text file as shown in dataset.txt file in datasets directory; however, fasta file can also be used with a simple preprocessing</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Machine learning</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PASTMUS" target="_blank">PASTMUS</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Mapping functional elements at single amino acid resolution in human cells.<br>Identification of functional elements for a protein of interest is important for achieving a mechanistic understanding. However, it remains cumbersome to assess each and every amino acid of a given protein in relevance to its functional significance. Here, we report a strategy, PArsing fragmented DNA Sequences from CRISPR Tiling MUtagenesis Screening (PASTMUS), which provides a streamlined workflow and a bioinformatics pipeline to identify critical amino acids of proteins in their native biological contexts. Using this approach, we map six proteins-three bacterial toxin receptors and three cancer drug targets, and acquire their corresponding functional maps at amino acid resolution</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Sequencing</li><li>Workflows</li><li>Mapping</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein identification</li><li>Protein structure validation</li><li>Mapping</li><li>Editing</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://bitbucket.org/WeiLab/pastmus" target="_blank">https://bitbucket.org/WeiLab/pastmus</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/TogoStanza" target="_blank">TogoStanza</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Generic Web framework for rendering an information block.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Taxonomy</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/togostanza/togostanza-gem" target="_blank">https://github.com/togostanza/togostanza-gem</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/alignparse" target="_blank">alignparse</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A Python package for parsing complex features from high-throughput long-read sequencing.<br>Align sequences and then parse features.<br>alignparse documentation — alignparse 0.1.2 documentation.<br>alignparse is a Python package written by the Bloom lab. It is designed to align long sequencing reads (such as those from PacBio circular consensus sequencing) to targets, filter these alignments based on user-provided specifications, and parse out user-defined sequence features. For each read that passes the filters, information about the features (e.g. accuracy, sequence, mutations) is retained for further analyses.<br>alignparse is a Python package written by the Bloom lab. It is designed to align long sequencing reads (such as those from PacBio circular consensus sequencing) to targets, filter these alignments based on user-provided specifications, and parse out user-defined sequence features</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Sequence assembly</li><li>Sequencing</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Read mapping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/jbloomlab/alignparse" target="_blank">https://github.com/jbloomlab/alignparse</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/casnp" target="_blank">CaSNP</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Database for storing and interrogating quantitative copy number alterations (CNA) data from SNP arrays on 34 different cancer types in 104 studies. With a user input of region or gene of interest, it will return the CNA information summarizing the frequencies of gain/loss and averaged copy number for each study. It also displays the heatmap showing copy numbers estimated at each SNP marker around the query region across all studies for a more comprehensive visualization.</div></td>
      <td><div class='topic short click_expand'><ul><li>Experimental design and studies</li><li>DNA polymorphism</li><li>Genetic variation</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Copy number estimation</li><li>Query and retrieval</li><li>SNP detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>22</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ipda" target="_blank">iPDA</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>IPDA (Integrated Protein Disorder Analyzer) predicts disordered regions of a query protein sequence.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein disordered structure</li><li>Sequence composition, complexity and repeats</li><li>Proteins</li><li>Protein sites, features and motifs</li><li>Reproductive health</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein secondary structure prediction</li><li>Protein feature detection</li><li>Protein secondary structure prediction (coils)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>120</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/weget" target="_blank">WeGet</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>WeGET is a computational tool to find mammalian genes that strongly co-express with a human query gene set of interest.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/weget-weighted-gene-coexpression-tool.html" target="_blank">http://www.mybiosoftware.com/weget-weighted-gene-coexpression-tool.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bgt" target="_blank">BGT</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Compact file format for efficiently storing and querying whole-genome genotypes of tens to hundreds of thousands of samples.</div></td>
      <td><div class='topic short click_expand'><ul><li>Whole genome sequencing</li><li>Sample collections</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>22</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/bgt-genotype-query-across-many-samples.html" target="_blank">http://www.mybiosoftware.com/bgt-genotype-query-across-many-samples.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nrps-pks" target="_blank">NRPS-PKS</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>NRPS-PKS is an tool comprising four integrated databases for the analysis of large multi-enzymatic multi-domain megasynthases.  The user can submit a query sequence to search for domains or view the properties of the products.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Database search</li><li>Sequence database search (by property)</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>216</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dasmiweb" target="_blank">DASMIweb</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Server that allows integration, analysis and quantitative assessment of distributed sources of protein and domain interactions. Users can query numerous sources simultaneously, which can then be configured and can support incorporation of user data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li><li>Protein interaction experiment</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Protein interaction network analysis</li><li>Protein interaction analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bionumbers" target="_blank">BioNumbers</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Database of key numbers in molecular and cell biology--the quantitative properties of biological systems of interest to computational, systems and molecular cell biologists. Contents of the database range from cell sizes to metabolite concentrations, from reaction rates to generation times, from genome sizes to the number of mitochondria in a cell. All numbers are taken directly from a literature source and that reference is provided with the number.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Systems biology</li><li>Zoology</li><li>Database management</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Text mining</li><li>Molecular replacement</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>433</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/isobase" target="_blank">IsoBase</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>IsoBase is a database identifying functionally related proteins across five major eukaryotic model organisms: yeast, fly, worm, mouse, and human. Clusters were computed using the IsoRankN algorithm, which uses spectral methods, to combine sequence and PPI data to produce clusters of functionally related proteins. IsoBase may be browsed for functionally related proteins across two or more species and may also be queried by accession numbers, species-specific identifiers, gene name or keyword.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Query and retrieval</li><li>Functional clustering</li><li>Pathway or network comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>72</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/morphdb" target="_blank">MorphDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Prioritizing Genes for Specialized Metabolism Pathways and Gene Ontology Categories in Plants.<br>1. MORPH bulk run command line interface.<br>Get MORPH candidate gene predictions for your gene(s) of interest.<br>2. Install morph-bulk using Singularity (recommended).<br>Get MORPH candidate gene predictions for your pathway or GO term of interest for your favourite organism(s). Make a MORPH network for your process of interest.<br>3. Install morph-bulk the classical way.<br>Perform complex SPARQL queries on MORPH prediction data.<br>rdf Generate RDF graph (for MorphDB).</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Plant biology</li><li>Gene expression</li><li>Ontology and terminology</li><li>Endocrinology and metabolism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic pathway prediction</li><li>Query and retrieval</li><li>Gene prediction</li><li>Expression correlation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/arzwa/morph-bulk" target="_blank">https://github.com/arzwa/morph-bulk</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CDEK" target="_blank">Clinical Drug Experience Knowledgebase (CDEK)</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Clinical Drug Experience Knowledgebase | Washington University in St. Louis | The Clinical Drug Experience Knowledgebase (CDEK) is a platform intended to explore every active pharmaceutical ingredient (API) with evidence of clinical testing | The Clinical Drug Experience Knowledgebase (CDEK) is a platform intended to explore every active pharmaceutical ingredient (API) with evidence of clinical testing | YOU AGREE THAT THE INFORMATION PROVIDED ON THIS WEBSITE IS PROVIDED “AS IS”, WITHOUT ANY WARRANTY OF ANY KIND, EXPRESSED OR IMPLIED, INCLUDING WITHOUT LIMITATION WARRANTIES OF MERCHANTABILITY OR FITNESS FOR ANY PARTICULAR PURPOSE, OR NON-INFRINGEMENT OF ANY THIRD-PARTY PATENT, COPYRIGHT, OR ANY OTHER THIRD-PARTY RIGHT | Build your own query by selecting various table columns and a filtration predicate; then press the to launch the query</div></td>
      <td><div class='topic short click_expand'><ul><li>Drug development</li><li>Drug discovery</li><li>Preclinical and clinical studies</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="http://cdek.wustl.edu/search/" target="_blank">http://cdek.wustl.edu/search/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gast" target="_blank">GAST</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The GAST algorithm assigns taxonomy to SSU sequences by comparing them to a set of reference sequences with known taxonomy. In the simplest terms, taxonomy is assigned to each query sequence based on the consensus of the nearest references sequences. The alignment distance between the query and references is also reported as a measure of how similar the query and reference sequences really are.</div></td>
      <td><div class='topic short click_expand'><ul><li>References</li><li>Taxonomy</li><li>Sequences</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Global alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>675</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/gast-global-alignment-sequence-taxonomy.html" target="_blank">http://www.mybiosoftware.com/gast-global-alignment-sequence-taxonomy.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/starbase" target="_blank">starBase</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Facilitates the comprehensive exploration of miRNA-target interaction maps from CLIP-Seq and Degradome-Seq data. Two web servers are provided to discover novel miRNA target sites from CLIP-Seq and Degradome-Seq data. Our web implementation supports diverse query types and exploration of common targets, gene ontologies and pathways.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Functional, regulatory and non-coding RNA</li><li>Mapping</li><li>Zoology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Mapping</li><li>microRNA detection</li><li>Transcriptional regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>520</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phylopat" target="_blank">PhyloPat</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Webserver which can be used to query the orthologies of all complete genomes within the EnsMart database using phylogenetic patterns. This enables the determination of sets of genes that occur only in certain evolutionary branches or even single species. The output can be saved in HTML, Excel or plain text format for further analysis. Output is linked to the FatiGO web interface creating easy access to functional information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Gene expression</li><li>Phylogeny</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree analysis</li><li>Query and retrieval</li><li>Phylogenetic tree analysis (gene family prediction)</li><li>Phylogenetic species tree construction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tripal_elasticsearch" target="_blank">Tripal Elasticsearch</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The module allows you to easily manage the indexing and display of Elasticsearch on your Tripal website. It also easily enables Cross-Site Querying, allowing you to connect to other Tripal sites and provide additional search results to your users.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Service composition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Plug-in</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/polysearch" target="_blank">PolySearch</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PolySearch allows users to conduct comprehensive and associative queries, such as given X, find all Y&apos;s, where X or Y can be diseases, tissues, cell compartments, gene/protein names, SNPs, mutations, drugs and metabolites. PolySearch also identifies, highlights and ranks abstracts, paragraphs or sentences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li><li>Genetic variation</li><li>Molecular interactions, pathways and networks</li><li>Pathology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Text mining</li><li>Protein modelling (mutation)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>181</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/datasets2tools" target="_blank">Datasets2Tools</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Repository and search engine for bioinformatics datasets, tools and canned analyses.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data acquisition</li><li>Bioinformatics</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ispectron" target="_blank">iSPECTRON</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A simulation interface for linear and nonlinear spectra with ab-initio quantum chemistry software.<br>We introduce iSPECTRON, a program that parses data from common quantum chemistry software (NWChem, OpenMolcas, Gaussian, Cobramm, etc.), produces the input files for the simulation of linear and nonlinear spectroscopy of molecules with the Spectron code, and analyzes the spectra with a broad range of tools. Vibronic spectra are expressed in term of the electronic eigenstates, obtained from quantum chemistry computations, and vibrational bath effects are incorporated in the framework of the displaced harmonic oscillator model, where all required quantities are computed at the Franck-Condon point. The program capabilities are illustrated by simulating linear absorption, transient absorption and two dimensional electronic spectra of the pyrene molecule</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>Molecular biology</li><li>Chemistry</li><li>Molecular dynamics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Spectral analysis</li><li>Parsing</li><li>Molecular dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cramer" target="_blank">CRAMER</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The Cranfield Genome Browser (CRAMER) is a lightweight, highly customizable web-based genome browser supporting multiple visualization instances. CRAMER is a customisable, JavaScript and Jade based genome browser for interactive exploration of genomic data. Data is visualized in the browser, meaning CRAMER can be installed on any website and show data from a wide range of online, ftp links or local sources. CRAMER works with a variety of formats, such as XML, JSON, BED, VCF, GFF, GFF3, BAM or delimited text files, and can be customised to parse and display any data source as required.</div></td>
      <td><div class='topic short click_expand'><ul><li>Imaging</li><li>Gene expression</li><li>Model organisms</li><li>DNA polymorphism</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Genome visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ctcrbase" target="_blank">ctcRbase</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>CtcRbase is agene expression database of circulating tumor cells and microemboli Circulating tumor cells/microemboli (CTCs/CTM) are those cells that depart from cancerous lesions and shed into the bloodstream. Enumeration and analysis of CTCs/CTM may enable non-invasive monitoring of advanced cancers, as well as early detection of tumor metastases. ctcRbase (Gene expression database of circulating tumor cells/ microemboli) is a database to collect expression data of CTCs and CTM in all kinds of cancer types, including long non-coding RNA (lncRNA) and messenger RNA (mRNA) derived from RNA-seq data analyses. ctcRbase provides the expression level of primary tumors, CTCs/CTM and white blood cells (WBC). Moreover, the expression level of tumor metastasis was also provided. An easy-to-use interface was constructed to query and browse gene expression in CTCs/CTM.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Oncology</li><li>Functional, regulatory and non-coding RNA</li><li>Gene transcripts</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Query and retrieval</li><li>Expression analysis</li><li>Sequence trimming</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ratt" target="_blank">RATT</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Transfer annotation from a reference (annotated) genome to an unannotated query genome. It is able to transfer any entry present on a reference sequence, such as the systematic id or an annotator’s notes.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>Assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>186</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/ratt-r18-rapid-annotation-transfer-tool.html" target="_blank">http://www.mybiosoftware.com/ratt-r18-rapid-annotation-transfer-tool.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lemon-pdb" target="_blank">Lemon</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Lemon is a tool for mining features used in downstream structural biology software dealing with 3D Macromolecules. It is designed to be fast and flexible, allowing users to quickly query the 3D features of a given collection of 3D structures.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Data mining</li><li>Drug discovery</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Virtual screening</li><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pathpred" target="_blank">PathPred</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Enzyme catalyzed metabolic pathway prediction server. Beginning with a query compound and chemical logic of enzymatic reactions (chemical structure transformation patterns of small molecules), biodegradation pathways of environmental compounds and biosynthesis pathways of secondary metabolites are predicted.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Compound libraries and screening</li><li>Molecular interactions, pathways and networks</li><li>Endocrinology and metabolism</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Query and retrieval</li><li>Pathway or network prediction</li><li>Protein structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>135</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gmql" target="_blank">GMQL</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Next-generation query language for querying NGS data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>55</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/gmql-1-0-1-genometric-query-language.html" target="_blank">http://www.mybiosoftware.com/gmql-1-0-1-genometric-query-language.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/parameciumdb" target="_blank">ParameciumDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Online community database for Paramecium species. Contains annotation of genome sequences and features, genome-wide data sets, advanced capabilities to query, retrieve, visualize and compare data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular genetics</li><li>Ontology and terminology</li><li>Model organisms</li><li>Sequence analysis</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Query and retrieval</li><li>DNA mapping</li><li>Genome annotation</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>expression data : tsv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>expression data : tsv</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>71</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/epivia" target="_blank">EpiVIA</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>EpiVIA is joint profiling of chromatin accessibility and CAR-T integration site analysis at population and single-cell levels. EpiVIA parses each read and identifies the chimeric fragments, which are further classified into different categories based on how the read pair is aligned to the combined reference genome.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li><li>Cell biology</li><li>Immunoproteins and antigens</li><li>Gene transcripts</li><li>Chromosome conformation capture</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Read mapping</li><li>Peak calling</li><li>Parsing</li><li>Sequence trimming</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iDASH" target="_blank">iDASH</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Using Ethereum blockchain to store and query pharmacogenomics data via smart contracts.<br>Solution to iDASH'19 challenge track 1.</div></td>
      <td><div class='topic short click_expand'><ul><li>Preclinical and clinical studies</li><li>Sequencing</li><li>Pharmacogenomics</li><li>Personalised medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Anonymisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ucsc-genome-browser" target="_blank">UCSC Genome Browser database</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The UCSC Genome Browser database.<br>Sequence and Annotation Downloads.<br>This page contains links to sequence and annotation data downloads for the genome assemblies featured in the UCSC Genome Browser. Table downloads are also available via the Genome Browser FTP server . For quick access to the most recent assembly of each genome, see the current genomes directory. This directory may be useful to individuals with automated scripts that must always reference the most recent assembly. To query and download data in JSON format, use our JSON API .<br>To view the current descriptions and formats of the tables in the annotation database, use the "describe table schema" button in the Table Browser.<br>All tables in the Genome Browser are freely usable for any purpose except as indicated in the README.txt files in the download directories.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Model organisms</li><li>Sequence assembly</li><li>Transcription factors and regulatory sites</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Query and retrieval</li><li>Genome annotation</li><li>Genome assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MG-RAST_API_explorer" target="_blank">MG-RAST API explorer</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>An on-ramp for RESTful query composition.<br>an on-ramp for learning to use the MG-RAST API.<br>MG-RAST (the Metagenomics RAST) server is an automated analysis platform for metagenomes providing quantitative insights into microbial populations based on sequence data. The server provides upload, quality control, automated annotation and comparative analysis for shotgun and amplicon metagenomic samples as well as metatranscriptomes.<br>MG-RAST API Explorer -- an on-ramp for learning to use the MG-RAST API.<br>An object store for scientific data.<br>MG-RAST provides a RESTful API at https: api.mg-rast.org.<br>workflow and resource management system for bioinformatics data analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Taxonomy</li><li>Sequencing</li><li>Workflows</li><li>Metagenomics</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Sequence trimming</li><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/MG-RAST" target="_blank">https://github.com/MG-RAST</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/endometdb" target="_blank">EndometDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A relational database to identify differentially expressed genes in the endometrium and endometriosis lesions.<br>EndometDB web frontend works as the input/output layer for the EndometDB, providing user's query for the backend and plotting it after backend has finished processing it.<br><br>An extensive collection of human samples and data from patients, that aids in the research for better diagnosis or cure for endometriosis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Reproductive health</li><li>Pathology</li><li>Endocrinology and metabolism</li><li>Transcriptomics</li><li>Biobank</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Expression analysis</li><li>Visualisation</li><li>Essential dynamics</li><li>Phasing</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/micawo/EndometDB" target="_blank">https://github.com/micawo/EndometDB</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/neartree" target="_blank">NearTree</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>NearTree is a software tool that constructs neartrees and provides a number of different query tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>X-ray diffraction</li><li>Applied mathematics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/OntoKin" target="_blank">OntoKin</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>An Ontology for Chemical Kinetic Reaction Mechanisms.<br>A knowledge-graph built with the integration of semantic technologies and software agents for enhancing the experience of chemists in querying chemical kinetic reaction mechanisms. This User Interface (UI) demonstrates the query capability of OntoKin</div></td>
      <td><div class='topic short click_expand'><ul><li>Chemistry</li><li>Enzymes</li><li>Literature and language</li><li>Ontology and terminology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phasing</li><li>Query and retrieval</li><li>Named-entity and concept recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td><div class="link short click_expand"><a href="http://theworldavatar.com/ontokin" target="_blank">http://theworldavatar.com/ontokin</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pdm-utils" target="_blank">pdm_utils</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A SEA-PHAGES MySQL phage database management toolkit.<br>MySQL phage genomics database management utilities.<br>The pdm_utils repository provides the source code for pdm_utils Python package, developed in the SEA-PHAGES program to create, update, and maintain MySQL phage genomics databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Genomics</li><li>Bioinformatics</li><li>Protein interaction experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/SEA-PHAGES/pdm_utils/" target="_blank">https://github.com/SEA-PHAGES/pdm_utils/</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biodataome" target="_blank">BioDataome</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A database of uniformly preprocessed and disease annotated omics data. We followed the same preprocessing pipeline for each biological mart to produce ready for downstream analysis and meta-analysis datasets and automatically annotate them with Disease-Ontology terms. We also designate datasets that share common samples. It includes ~2000 datasets, ~200000 samples spanning ~300 diseases. All datasets are publicly available for querying, browsing and downloading via BioDataome web application.</div></td>
      <td><div class='topic short click_expand'><ul><li>Omics</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iedb" target="_blank">The Immune Epitope Database (IEDB)</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The Immune Epitope Database provides a catalog of experimentally characterized B and T cell epitopes, as well as data on Major Histocompatibility Complex (MHC) binding and MHC ligand elution experiments. The database represents the molecular structures recognized by adaptive immune receptors and the experimental contexts in which these molecules were determined to be immune epitopes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Immunoproteins, genes and antigens</li><li>Database management</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Epitope mapping</li><li>Peptide immunogenicity prediction</li><li>Query and retrieval</li><li>Deposition</li><li>Mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>484</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/multiplierz" target="_blank">multiplierz</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Scriptable framework for access to manufacturers' formats via mzAPI.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data visualisation</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Ion counting</li><li>Protein identification</li><li>Filtering</li><li>Peptide database search</li><li>Peptide identification</li><li>Formatting</li><li>iTRAQ</li><li>Data handling</li><li>Mass spectra calibration</li><li>Query and retrieval</li><li>Spectral analysis</li><li>Data retrieval</li><li>Peak detection</li><li>Validation of peptide-spectrum matches</li><li>Label-free quantification</li><li>Protein quantification</li><li>Deisotoping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>query script : sqlite format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : xls</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>114</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>LGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Myo-REG" target="_blank">Myo-REG</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A Portal for Signaling Interactions in Muscle Regeneration.<br>myogenesis and regeneration portal.<br>Myo-REG offers easy access to signaling information in the context of muscle regeneration.<br>Myogenesis & REGeneration Portal.<br>Myo-REG offers easy access to signaling information in the context of muscle regeneration. The information is organized in two layers: the first concerns cell-cell interactions, while the second layer describes the most relevant signaling pathways underlying the crosstalk between the cell types participating in muscle regeneration. A user-friendly interface allows users to explore, via an interactive model, cell and pathway crosstalk</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Molecular interactions, pathways and networks</li><li>Model organisms</li><li>Database management</li><li>Drug metabolism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Editing</li><li>Network simulation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/strms" target="_blank">STRMS</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Structural RNA motif search that takes as input the secondary structure of the query and a target sequence database, and reports all occurrences of the query in the target. The search is based on subtree homeomorphism for ordered, rooted trees (which we use to represent the RNA structures) .</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Motif search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/strms-0-5-2-structural-rna-motif-search.html" target="_blank">http://www.mybiosoftware.com/strms-0-5-2-structural-rna-motif-search.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nadbinder" target="_blank">NADbinder</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>NADbinder server discriminate the NAD interacting residues and non-interacting residues from a given sequence. The NADbinder server uses the SVM based method by using the Position Specific Scoring Matrix (PSSM) generated from the query sequence(s).</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>44</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/nadbinder-prediction-of-nad-interacting-residues-in-proteins.html" target="_blank">http://www.mybiosoftware.com/nadbinder-prediction-of-nad-interacting-residues-in-proteins.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Tn-seq" target="_blank">Tn-seq</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Scripts for analyzing transposon sequencing data used by the Whiteley lab | Custom scripts for analyzing (parsing, mapping, and tallying) Tn-seq reads and determining differentially abundant transposon insertion mutants | The scripts contained herein can be used to automatically analyze high-throughput (Illumina) sequencing reads derived from transposon-genome junctions</div></td>
      <td><div class='topic short click_expand'><ul><li>Laboratory animal science</li><li>Sequencing</li><li>Mobile genetic elements</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence trimming</li><li>Virulence prediction</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/spang" target="_blank">SPANG</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A command-line SPARQL client supporting query generation for distributed RDF databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Data acquisition</li><li>Laboratory information management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Cafe_Variome" target="_blank">Cafe Variome</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>General‐purpose software for making genotype–phenotype data discoverable in restricted or open access contexts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Rare diseases</li><li>Genotype and phenotype</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>18</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bifrost" target="_blank">Bifrost</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Bifrost is a command-line tool for sequencing that features a broad range of functions, such as indexing, editing, and querying the graph, and includes a graph coloring method that maps each k-mer of the graph to the genomes it occurs in.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Sequence assembly</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Genome indexing</li><li>De-novo assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-2-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pazar" target="_blank">PAZAR</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Open access and open source database of transcription factor and regulatory sequence annotation with associated web interface and programming tools designed to submit and query data related to the regulation of gene expression.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li><li>Gene transcripts</li><li>Gene regulation</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Transcriptional regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>69</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pyrice" target="_blank">PyRice</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PyRice is an API to access some Rice public databases at the same time with consistent output. PyRice design is modular and implements a smart query system which fits the computing resources to optimize the query speed.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biological databases</li><li>Plant biology</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Information extraction</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nl4dv" target="_blank">NL4DV</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A Toolkit for Generating Analytic Specifications for Data Visualization from Natural Language Queries.<br>NL4DV is a Python toolkit that takes a natural language (NL) query about a given dataset as input and outputs a structured JSON object containing Data attributes, Analytic tasks, and Visualizations (Vega-Lite specifications).</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Natural language processing</li><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Editing</li><li>Named-entity and concept recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://nl4dv.github.io/nl4dv/showcase.html" target="_blank">https://nl4dv.github.io/nl4dv/showcase.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GPress" target="_blank">GPress</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A framework for querying General Feature Format (GFF) files and feature expression files in a compressed form”.<br>This is the GPress, a framework for querying GTF, GFF3 and expression files in a compressed form.<br>Here, we will use the test files to provide an example of how to use GPress. We will use GTF file for illustration since the GFF3 file is nearly the same except slight format variations. We will store the compressed GTF file and compressed expression file in a new folder gtf1</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Gene transcripts</li><li>Plant biology</li><li>Exome sequencing</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li><li>Data retrieval</li><li>Database search</li><li>Sorting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/imex" target="_blank">IMEx</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>This consortium is an international collaboration between a group of major public interaction data providers.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Suite</li></ul></div></td>
      <td><div class='citations short click_expand'>288</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/osirix" target="_blank">OsiriX</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>OsiriX is an image processing software dedicated to DICOM images (“.dcm” / “.DCM” extension) produced by imaging equipment (MRI, CT, PET, PET-CT, SPECT-CT, Ultrasounds, …). It is fully compliant with the DICOM standard for image comunication and image file formats. OsiriX is able to receive images transferred by DICOM communication protocol from any PACS or imaging modality (C-STORE SCP/SCU, and Query/Retrieve : C-MOVE SCU/SCP, C-FIND SCU/SCP, C-GET SCU/SCP, WADO) .</div></td>
      <td><div class='topic short click_expand'><ul><li>MRI</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>1,303</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/osirix-3-9-2-dicom-viewer.html" target="_blank">http://www.mybiosoftware.com/osirix-3-9-2-dicom-viewer.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dictybase_update_2011" target="_blank">dictyBase update 2011</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Database for the biology and genomics of the social amoeba Dictyostelium discoideum. It provides well integrated, high quality data and tools for Dictyostelium discoideum and related species. It houses the complete genome sequence, ESTs, and the entire body of literature relevant to Dictyostelium. This information is curated to provide accurate gene models and functional annotations, with the goal of fully annotating the genome to provide a reference genome in the Amoebozoa clade.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Gene transcripts</li><li>Model organisms</li><li>Database management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li><li>Gene functional annotation</li><li>Text annotation</li><li>Query and retrieval</li><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>112</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/miamexpress" target="_blank">MIAMExpress</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Compliant microarray data submission tool.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li><li>Microarray experiment</li><li>Model organisms</li><li>Database management</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Deposition</li><li>Query and retrieval</li><li>Gene expression clustering</li><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>661</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GenomicsDB" target="_blank">GenomicsDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Advancing clinical cohort selection with genomics analysis on a distributed platform.<br>Highly performant data storage in C++ for importing, querying and transforming variant data with Java/Spark. Used in gatk4.<br>Sparse Array Storage for Genomics.<br>GenomicsDB, originally from Intel Health and Lifesciences, is built on top of a fork of htslib and a tile-based array storage system for importing, querying and transforming variant data.<br>Open source project providing a collaboration to optimizing sparse array storage for genomics.<br>Using high-level APIs provided in C++, Java*, and Spark*, users can both write and read variant records to and from GenomicsDB shared-nothing instances in parallel using multiple processes in a Single Process Multiple Data (SPMD) manner.</div></td>
      <td><div class='topic short click_expand'><ul><li>Personalised medicine</li><li>Complementary medicine</li><li>Exome sequencing</li><li>Systems medicine</li><li>Biobank</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Query and retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/GenomicsDB/GenomicsDB" target="_blank">https://github.com/GenomicsDB/GenomicsDB</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rdgidb" target="_blank">rDGIdb</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Wrapper for the Drug Gene Interaction Database. For simplicity, its query function and output resembles the user interface and results format provided on the DGIdb website.</div></td>
      <td><div class='topic short click_expand'><ul><li>Compound libraries and screening</li><li>Drug discovery</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biosystems" target="_blank">BioSystems</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Database that centralizes and cross-links existing biological systems databases, increasing their utility and target audience by integrating their pathways and systems into NCBI resources. This integration allows users of NCBI's Entrez databases to quickly categorize proteins, genes and small molecules by metabolic pathway, disease state or other type.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Pathology</li><li>Endocrinology and metabolism</li><li>Database management</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Relation extraction</li><li>ID mapping</li><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>447</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dockNmine" target="_blank">dockNmine</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Web Portal to Assemble and Analyse Virtual and Experimental Interaction Data.<br>The dockNmine portal aims at gathering public and private data into a unique service. Automated queries on protein targets and ligand definitions are performed to Uniprot, PubChem and ChemBL to enhance the results of pre-computed docking experiments. When available, public data are automatically added to the docking results to produce state-of-the-art protein-ligand binding analysis such as ROC curves or enrichment analysis. Users can also upload their own private data to analyze them automatically</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein interactions</li><li>Molecular modelling</li><li>Sequence assembly</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein-protein docking</li><li>Protein-ligand docking</li><li>Quantification</li><li>Sequence assembly</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/encodexplorer" target="_blank">ENCODExplorer</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>This package allows user to quickly access ENCODE project files metadata and give access to helper functions to query the ENCODE rest api, download ENCODE datasets and save the database in SQLite format.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gotohscan" target="_blank">GotohScan</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The GotohScan program is a search tool that finds shorter sequences (usually genes) in large database sequences (chromosomes, genomes, ..)by computing all semi-global alignments.  Thus, the query sequence is never truncated or split into subsequences, but always mapped to the database over its complete length. The alignment is computed via the Gotoh-alignment algorithm using affine gap costs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Chromosomes</li><li>DNA</li><li>Sequences</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene finding</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/gotohscan-2-0-gene-finding-semi-global-alignment.html" target="_blank">http://www.mybiosoftware.com/gotohscan-2-0-gene-finding-semi-global-alignment.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/foldminer" target="_blank">FoldMiner and LOCK 2</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>FoldMiner aligns a user-supplied or identified query structure to a database of single domain targets to discover structural neighbours and characteristic motifs.  Query structures can also be aligned to one or more user-specified structures using the LOCK 2 algorithm.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structural motifs and surfaces</li><li>Protein folds and structural domains</li><li>Structure analysis</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Structure alignment</li><li>Protein structural motif recognition</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>37</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mkdom" target="_blank">mkdom</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Program we use routinely to build each new release of ProDom, Protein domain families automatically generated from protein databases. It relies on the assumption that the shortest amino acid sequence corresponds to a single domain, and may be used as a query to screen the database with the psi-blast program, in order to cluster homologous domains.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Database management</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Local alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>57</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mkdomxdom-2-build-prodom-database.html" target="_blank">http://www.mybiosoftware.com/mkdomxdom-2-build-prodom-database.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/arrayprospector" target="_blank">ArrayProspector</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Set of predicted functional associations between genes that have been inferred from microarray expression data from the Standford Microarray Database. Users can search for genes linked to query or for links between two genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Gene and protein families</li><li>Animals</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression analysis</li><li>Query and retrieval</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>25</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nbrp_resources" target="_blank">NBRP resources</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>System for collecting, preserving and providing bioresources for use as experimental materials for life science research.</div></td>
      <td><div class='topic short click_expand'><ul><li>Plant biology</li><li>Ontology and terminology</li><li>Mapping</li><li>Zoology</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Deposition</li><li>Database search</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>28</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gem-trend" target="_blank">GEM-TREND</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>GEM-TREND, a web tool for searching gene expression data, allows users to search data from GEO using gene-expression signatures or gene expression ratio data as a query and retrieve gene expression data by comparing gene-expression pattern between the query and GEO gene expression data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Systems biology</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>31</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/gem-trend-gene-expression-data-mining-toward-relevant-network-discovery.html" target="_blank">http://www.mybiosoftware.com/gem-trend-gene-expression-data-mining-toward-relevant-network-discovery.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/amped" target="_blank">Antimicrobial Peptide Database (AMPed)</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Annotated collection of normalized data describing antimicrobial peptides, their sources, their 3D structures, and their target organisms. It enables users to perform quick queries and allows the entire peptide community to contribute to the repository.</div></td>
      <td><div class='topic short click_expand'><ul><li>Drug discovery</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Analysis</li><li>Comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>experiment report : xml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence set : xml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li><li>Suite</li><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://martinlab.homeip.net" target="_blank">http://martinlab.homeip.net</a></div></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/virus_genotyping_tools" target="_blank">Virus Genotyping Tools</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>This web server facilitates high-throughput virus genotyping by aligning query sequences to a pre-defined set of reference strains and then computing phylogenetic analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Genotype and phenotype</li><li>Sequencing</li><li>Sequence analysis</li><li>Virology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Sequence alignment analysis</li><li>DNA substitution modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>125</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bioextract" target="_blank">BioExtract</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The BioExtract Server is a platform for the creation of bioinformatic workflows to aid the analysis of genomic data. A series of recorded tasks can be saved and shared. Integrated resources include NCBI databases, EMBL-Bank, UniProt and UniRef, as well as a large number of computational tools such as EMBOSS and KEGG.</div></td>
      <td><div class='topic short click_expand'><ul><li>Computational biology</li><li>Bioinformatics</li><li>Workflows</li><li>Database management</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Query and retrieval</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>26</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sledgehmmer" target="_blank">SledgeHMMER</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Tool for searching the Pfam database using a parallelized version of the program hmmpfam. The user can perform queries with one or more sequences at a time and then receive the results by e-mail.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Proteins</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Database search</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>22</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/yogy" target="_blank">YOGY</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Eukaryotic Orthology (YOGY) is a resource for retrieving orthologous proteins from nine eukaryotic organisms. Using a gene or protein identifier as a query, this database provides comprehensive, combined information on orthologs in other species using data from five independent resources: KOGs, Inparanoid, Homologene, OrthoMCL, and a table of curated orthologs between budding yeast and fission yeast. Associated Gene Ontology (GO) terms of orthologs can also be retrieved.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Gene and protein families</li><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Gene prediction</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>41</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ccdb" target="_blank">CCDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The Cervical Cancer gene DataBase is a manually curated catalog of experimentally validated genes that are involved in the different stages of cervical carcinogenesis. Each record contains details related to gene. It provides information on microRNA altered in cervical cancer as well as search facility for querying, several browse options and an online tool for sequence similarity search.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Gene transcripts</li><li>Oncology</li><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Sequence similarity search</li><li>Query and retrieval</li><li>Gene-set enrichment analysis</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>53</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/predus" target="_blank">PredUs</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PredUs is a web server for the prediction of protein-protein interfaces. The method predicts protein-protein interfaces of a given monomeric query protein using its close and remote structural neighbors.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structural motifs and surfaces</li><li>Structure prediction</li><li>Proteins</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein-protein interaction prediction</li><li>Query and retrieval</li><li>Protein secondary structure prediction</li><li>Protein structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>186</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genenames.org" target="_blank">genenames.org</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>This tool displays all approved nomenclature within Symbol Reports that contain data curated by HUGO Gene Nomenclature Committee (HGNC) editors and links to related genomic, phenotypic and proteomic information. New features include a new Quick Gene Search, a new List Search, an integrated HGNC BioMart and a new Statistics and Downloads facility.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Genotype and phenotype</li><li>Gene and protein families</li><li>Human biology</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene prediction</li><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>460</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iCite" target="_blank">iCite</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A public access, broad coverage resource.<br>NIH Office of Portfolio Analysis.<br>iCite is a tool to access a dashboard of bibliometrics for papers associated with a portfolio. Users type in a PubMed query or upload the PubMed IDs of articles of interest. iCite has three modules: Influence, Translation, and Open Citations.<br>iCite: Influence delivers metrics of scientific influence such as the Relative Citation Ratio (RCR), which is field- and time-adjusted, and benchmarked to NIH publications as the baseline. Read about how RCR is calculated at PLOS Biology.<br>Bulk data can be accessed at the iCite database snapshot repository</div></td>
      <td><div class='topic short click_expand'><ul><li>Preclinical and clinical studies</li><li>Workflows</li><li>Machine learning</li><li>Medicine</li><li>Zoology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>Proprietary</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iot-stream" target="_blank">IoT-Stream</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A Lightweight Ontology for Internet of Things Data Streams and Its Use with Data Analytics and Event Detection Services.<br>A Lightweight Ontology for IoT Data Streams.<br>T. Elsaleh, M. Bermudez-Edo, S. Enshaeifar, S. T. Acton, R. Rezvani and P. Barnaghi, "IoT-Stream: A Lightweight Ontology for Internet of Things Data Streams," 2019 Global IoT Summit (GIoTS), Aarhus, Denmark, 2019, pp. 1-6.<br>||| CORRECT NAME OF TOOL COULD ALSO BE 'IoT' (bio.tools/IoT)</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Model organisms</li><li>Workflows</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Parsing</li><li>Ontology visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td><div class="link short click_expand"><a href="http://purl.org/iot/ontology/iot-stream" target="_blank">http://purl.org/iot/ontology/iot-stream</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/museqbox" target="_blank">MuSeqBox</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Multi-query Sequence Blast Output Examination is a program designed for multi-query sequence BLAST output examination. It examines the BLAST output, extracts the informative parameters of BLAST hits, and saves them in tabular form in either text or HTML format. The hit tables are optionally further analyzed with the program to produce subsets of BLAST hits according to user-specified criteria.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>Sequences</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>27</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/museqbox-multi-query-sequence-blast-output-examination.html" target="_blank">http://www.mybiosoftware.com/museqbox-multi-query-sequence-blast-output-examination.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pda" target="_blank">PDA</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Primer Design Assistant (PDA) is a PCR primer design tool that will accept batch submissions of query sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Nucleic acid structure analysis</li><li>PCR experiment</li><li>Probes and primers</li><li>Nucleic acid sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Primer removal</li><li>Query and retrieval</li><li>Primer and probe design</li><li>PCR primer design</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>20</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/zifit" target="_blank">ZiFiT</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>ZiFiT (Zinc Finger Targeter) assists in the design of zinc finger proteins that can bind to specific DNA sequences. ZiFiT has been updated to integrate additional tools and resources including scoring schemes for modular assembly, an interface with the Zinc Finger Database of engineered ZFPs, and direct query of NCBI BLAST servers for identification of potential off-target sites within a host genome.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>DNA binding sites</li><li>DNA</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li><li>Query and retrieval</li><li>Protein-nucleic acid binding prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>417</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/blastfrost" target="_blank">BlastFrost</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>BlastFrost is a highly efficient method for querying 100,000s of genome assemblies. BlastFrost builds on the recently developed Bifrost, which generates a dynamic data structure for compacted and colored de Bruijn graphs from bacterial genomes. BlastFrost queries a Bifrost data structure for sequences of interest, and extracts local subgraphs, thereby enabling the efficient identification of the presence or absence of individual genes or single nucleotide sequence variants.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>DNA polymorphism</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Genome assembly</li><li>De-novo assembly</li><li>Query and retrieval</li><li>Genotyping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/super" target="_blank">Super</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Super is a webserver for rapidly screening the PDB to identify all oligopeptide fragments that superpose with a query under a pre-specified threshold of root mean-square deviation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structural motifs and surfaces</li><li>Mathematics</li><li>Proteins</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Phylogenetic sub/super tree detection</li><li>Protein structural motif recognition</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pcrpi-db" target="_blank">PCRPi-DB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Public repository that archives computationally annotated hot spots in protein complexes for which the 3D structure is known. Hot spots have been annotated using a new and highly accurate computational method developed in the lab. Besides browsing and querying the contents of the database, extensive documentation and links to relevant on-line resources and contents are available to users.</div></td>
      <td><div class='topic short click_expand'><ul><li>Drug discovery</li><li>Proteins</li><li>Protein folding, stability and design</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Residue interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/atav" target="_blank">ATAV</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>ATAV is a comprehensive platform for population-scale genomic analyses.<br>ATAV stores variant and per site coverage data for all samples in a centralized database, which is efficiently queried by ATAV to support diagnostic analyses for trios and singletons, as well as rare-variant collapsing analyses for finding disease associations in complex diseases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Gene expression</li><li>Mapping</li><li>Exome sequencing</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Variant calling</li><li>Query and retrieval</li><li>Variant filtering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/nickzren/atav" target="_blank">https://github.com/nickzren/atav</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pids" target="_blank">PIDS</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A User-Friendly Plant DNA Fingerprint Database Management System.<br>The high variability and somatic stability of DNA fingerprints can be used to identify individuals, which is of great value in plant breeding. DNA fingerprint databases are essential and important tools for plant molecular research because they provide powerful technical and information support for crop breeding, variety quality control, variety right protection, and molecular marker-assisted breeding. Building a DNA fingerprint database involves the production of large amounts of heterogeneous data for which storage, analysis, and retrieval are time and resource consuming. To process the large amounts of data generated by laboratories and conduct quality control, a database management system is urgently needed to track samples and analyze data<br>||| HOMEPAGE MISSING!</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Plant biology</li><li>Database management</li><li>DNA polymorphism</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Data retrieval</li><li>DNA profiling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aCLImatise" target="_blank">aCLImatise</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>ACLImatise is a Python library and command-line utility for parsing the help output of a command-line tool and then outputting a description of the tool in a more structured format, for example a Common Workflow Language tool definition.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>text data : cwl</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/aCLImatise/CliHelpParser" target="_blank">https://github.com/aCLImatise/CliHelpParser</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/askomics" target="_blank">Askomics</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>AskOmics is a visual SPARQL query interface supporting both intuitive data integration and querying while shielding the user from most of the technical difficulties underlying RDF and SPARQL</div></td>
      <td><div class='topic short click_expand'><ul><li>Data integration and warehousing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>text data : dsv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : rdf format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tdr_targets" target="_blank">TDR Targets</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Tropical Disease Research (TDR) Database: Designed and developed to facilitate the rapid identification and prioritization of molecular targets for drug development, focusing on pathogens responsible for neglected human diseases. The database integrates pathogen specific genomic information with functional data for genes collected from various sources, including literature curation. Information can be browsed and queried.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pathology</li><li>Model organisms</li><li>Drug discovery</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein subcellular localisation prediction</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>86</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/emage" target="_blank">EMAGE</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Freely available online database of in situ gene expression patterns in the developing mouse embryo. Gene expression domains from raw images are extracted and integrated spatially into a set of standard 3D virtual mouse embryos at different stages, allowing data interrogation by spatial methods. An anatomy ontology is also used to describe sites of expression, data can be queried using text-based methods. Data coverage has been increased by sourcing from a greater selection of journals.</div></td>
      <td><div class='topic short click_expand'><ul><li>Tomography</li><li>Anatomy</li><li>Imaging</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression comparison</li><li>Query and retrieval</li><li>Gene expression clustering</li><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>192</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Mabellini" target="_blank">Mabellini</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A genome-wide database for understanding the structural proteome and evaluating prospective antimicrobial targets of the emerging pathogen Mycobacterium abscessus.<br>An on-line source for Mycobacterium abscessus modeled structural proteome.<br>MabeLLINI (Mycobacterium ABscessus modELLing INItiative) is a database available to anyone that wishes to know in detail the structural proteome of Mycobacterium abscessus. It contains thousands of models, generated with an unique domain-based methodology and thoroughly annotated with information from UniProt, CATH, SCOP and many more resources. These models are readily available for download, either individually or in bundles. To access them, simply use one of the many available queries or explore the data using our hierarchical interactive sunbursts</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Proteomics</li><li>Surgery</li><li>Sequence analysis</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Molecular docking</li><li>Protein modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gopubmed" target="_blank">GoPubMed</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Tool that allows users to explore the results of PubMed queries in the context of Gene Ontology (GO) terms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Ontology visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>351</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ccsbase" target="_blank">CCSbase</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>CCSbase is an all-in-one web interface for querying the CCS database and accessing the predictive model to support unknown compound identifications.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biological databases</li><li>Proteomics experiment</li><li>Physics</li><li>Metabolomics</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Natural product identification</li><li>Query and retrieval</li><li>Nucleic acid structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/repo" target="_blank">repo</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A data manager meant to avoid manual storage/retrieval of data to/from the file system. It builds one (or more) centralized repository where R objects are stored with rich annotations, including corresponding code chunks, and easily searched and retrieved.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Database management</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pintail" target="_blank">Pintail</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Program for detecting and analysing 16S rRNA chimeras and other anomalies. Pintail is a tool for identifying anomalies, including chimeras, within 16S rDNA sequences. The program works by comparing evolutionary distances between a query and subject sequence over the length of the 16S rRNA gene (small subunit rRNA), by employing a sampling window of specified size, progressing a fixed number of bases at a time along the length of the gene.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/pintail-1-1-analyse-16s-rrna-chimera-anomalies.html" target="_blank">http://www.mybiosoftware.com/pintail-1-1-analyse-16s-rrna-chimera-anomalies.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pdb-tools-web" target="_blank">PDB-Tools Web</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A user-friendly interface for the manipulation of PDB files.<br>A swiss army knife for manipulating and editing PDB files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Database management</li><li>Structural biology</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Editing</li><li>Data retrieval</li><li>Parsing</li><li>Protein structure validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://pypistats.org/packages/pdb-tools" target="_blank">https://pypistats.org/packages/pdb-tools</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/searchgtr" target="_blank">SEARCHGTr</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Tool for the analysis of glycosyltransferases (GTrs) that allows you to compare a query sequence with the sequences of characterized GTrs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence analysis</li><li>Query and retrieval</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>31</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hit" target="_blank">HIT</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A comprehensive and fully curated database for Herb Ingredients’ Targets. Molecular target information involves those proteins being directly/indirectly activated/inhibited, protein binders and enzymes whose substrates or products are those compounds. Those up/down regulated genes are included under the treatment of individual ingredients. The experimental condition, observed bioactivity and various references are provided. The database can be queried via keyword or similarity search.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein targeting and localisation</li><li>Proteins</li><li>Drug development</li><li>Drug discovery</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li><li>Protein subcellular localisation prediction</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>179</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MedicMine" target="_blank">MedicMine</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>MedicMine integrates many types of data for Medicago truncatula. You can run flexible queries, export results and analyse lists of genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/keanu" target="_blank">Keanu</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A Novel Visualization Tool to Explore Biodiversity in Metagenomes.<br>A tool for viewing the contents of metagenomic samples.<br>format_input.py: takes a file that contains a single BLAST query and taxon ID per line and formats it correctly as input for Keanu.<br>The following command is used to create the taxonomy.dat and merged_deleted.dat databases necessary for running Keanu. There are no optional parameters. The input files - names.dmb, nodes.dmp, delnodes.dmp, and merged.dmp - come from the taxdmp file located at the NCBI Taxonomy FTP site: ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biodiversity</li><li>Metagenomics</li><li>Taxonomy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Sequence assembly</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/RatMine" target="_blank">RatMine</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>RatMine integrates many types of data for Rattus Norvegicus, Homo Sapiens, Mus Musculus and other organisms. You can run flexible queries, export results and analyse lists of data. The Rat Genome Database (RGD) was established in 1999 and is the premier site for genetic, genomic, phenotype, and disease data generated from rat research. In addition, it provides easy access to corresponding human and mouse data for cross-species comparisons. RGD's comprehensive data and innovative software tools make it a valuable resource for researchers worldwide.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metabolic" target="_blank">metabolic</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A scalable high-throughput metabolic and biogeochemical functional trait profiler based on microbial genomes | A scalable high-throughput metabolic and biogeochemical functional trait profiler | This software enables the prediction of metabolic and biogeochemical functional trait profiles to any given genome datasets. These genome datasets can either be metagenome-assembled genomes (MAGs), single-cell amplified genomes (SAGs) or pure culture sequenced genomes. It can also calculate the genome coverage. The information is parsed and diagrams for elemental/biogeochemical cycling pathways (currently Nitrogen, Carbon, Sulfur and "other") are produced</div></td>
      <td><div class='topic short click_expand'><ul><li>Endocrinology and metabolism</li><li>Microbial ecology</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Metabolic labeling</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biojs-io-clustal" target="_blank">biojs-io-clustal</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Parses clustal files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment : clustalw format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metag" target="_blank">MetaG</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>MetaGenomic analysis of short and long reads.<br>This directory contains the query files used to test MetaG.<br>Due to file size limitations, the databases can be found here:.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomic sequencing</li><li>Taxonomy</li><li>Functional, regulatory and non-coding RNA</li><li>Metagenomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Virulence prediction</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/canevolve" target="_blank">canEvolve</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Query functionalities are designed to fulfill most frequent analysis needs of cancer researchers with a view to generate novel hypotheses. It stores gene, microRNA (miRNA) and protein expression profiles, copy number alterations for multiple cancer types, and protein-protein interaction information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Protein expression</li><li>Molecular interactions, pathways and networks</li><li>Microarray experiment</li><li>RNA</li><li>DNA polymorphism</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/canevolve-integrative-cancer-genomics-analysis-expression-copy-number-mirnas-network.html" target="_blank">http://www.mybiosoftware.com/canevolve-integrative-cancer-genomics-analysis-expression-copy-number-mirnas-network.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mitominer" target="_blank">MitoMiner</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>MitoMiner integrates mitochondrial proteomics data for a range of organisms. You can run flexible queries, export results and analyse lists of data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Proteomics</li><li>Geriatric medicine</li><li>Pathology</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Query and retrieval</li><li>Gene-set enrichment analysis</li><li>Database comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>77</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bioproject" target="_blank">BioProject</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Collection of biological data related to a single initiative, originating from a single organization or from a consortium. A record provides users a single place to find links to the diverse data types generated for that project.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sample collections</li><li>Mapping</li><li>Model organisms</li><li>Database management</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>155</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pecan_dia" target="_blank">PECAN (PEptide-Centric Analysis)</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PECAN (PEptide-Centric Analysis) is a tool for peptide detection directly from DIA data without the need of a spectral library. PECAN takes a list of peptide sequences and query each peptide against the DIA data (in centroid mzML format) and reports the best evidence of detection for each peptide. The PECAN report is designed for Percolator to separate correct from incorrect matches with false discovery rate (FDR) control.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Peptide identification</li><li>Peak detection</li><li>Query and retrieval</li><li>Target-Decoy</li><li>Spectrum calculation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence set : tsv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : sqlite format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>60</div></td>
      <td></td>
      <td><div class='short click_expand'>LGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GenFam" target="_blank">GenFam</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A web application and database for gene family-based classification and functional enrichment analysis.<br>Genome-scale studies using high-throughput sequencing (HTS) technologies generate substantial lists of differentially expressed genes under different experimental conditions. These gene lists need to be further mined to narrow down biologically relevant genes and associated functions in order to guide downstream functional genetic analyses. A popular approach is to determine statistically overrepresented genes in a user-defined list through enrichment analysis tools, which rely on functional annotations of genes based on Gene Ontology (GO) terms. Here, we propose a new computational approach, GenFam, which allows annotation, classification, and enrichment of genes based on their gene family, thus simplifying identification of candidate gene families and associated genes that may be relevant to the query.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Plant biology</li><li>Gene expression</li><li>Gene and protein families</li><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene functional annotation</li><li>Query and retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://reneshbedre.github.io/blog/genfam.html" target="_blank">https://reneshbedre.github.io/blog/genfam.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/AQUA-MER" target="_blank">AQUA-MER</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A database and toolkit for researchers working on environmental mercury geochemistry.<br>AQUA-MER is useful for researchers who want to investigate mercury speciation in defined environments i.e., to which chemical elements and groups it will bind and in what proportions, given an input of concentrations of chemicals.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Database management</li><li>Computational chemistry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Molecular dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ensemblqdb" target="_blank">EnsemblQDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Query-based biclustering techniques allow interrogating a gene expression compendium with a given gene or gene list. They do so by searching for genes in the compendium that have a profile close to the average expression profile of the genes in this query-list. As it can often not be guaranteed that the genes in a long query-list will all be mutually coexpressed, it is advisable to use each gene separately as a query.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Processing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/ensemblqdb-ensembl-query-based-biclustering.html" target="_blank">http://www.mybiosoftware.com/ensemblqdb-ensembl-query-based-biclustering.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/admixpipe" target="_blank">ADMIXPIPE</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A method for parsing and filtering VCF files for admixture analysis. The method is able to filters SNPs to allow the delineation of population structure in ADMIXTURE and parses the output for summarization and graphical representation via CLUMPAK.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>DNA polymorphism</li><li>Ecology</li><li>Population genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Filtering</li><li>Linkage disequilibrium calculation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Workflow</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hgd" target="_blank">The Hymenoptera Genome Database (HGD)</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The Hymenoptera Genome Database is a comprehensive model organism database for insect species of the order Hymenoptera. HGD contains data from 9 different species across 200 million years in the phylogeny of Hymenoptera. Curated data at HGD includes predicted and annotated gene sets supported with evidence tracks such as ESTs/cDNAs, small RNA sequences and GC composition domains.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogeny</li><li>Gene transcripts</li><li>Model organisms</li><li>Database management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Query and retrieval</li><li>Database search</li><li>Scaffolding</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>121</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/komenti" target="_blank">Komenti</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A semantic text mining framework.<br>Komenti is a tool for semantic query, annotation, and analysis of text using ontologies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Natural language processing</li><li>Workflows</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Query and retrieval</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/t1dbase" target="_blank">T1DBase</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Database for type 1 diabetes (T1D). It integrates genetic, genomic and expression data relevant to T1D research across mouse, rat and human and presents this to the user as a set of web pages and tools. New data sets include curated summary data from four genome-wide association studies relevant to T1D, HaemAtlas-a data set and tool to query gene expression levels in haematopoietic cells and a manually curated table of human T1D susceptibility loci.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Gene expression</li><li>Pathology</li><li>GWAS study</li><li>Experimental design and studies</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression QTL analysis</li><li>Gene expression profiling</li><li>Query and retrieval</li><li>Gene expression analysis</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>63</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/parsefiles-gofiles" target="_blank">ParseFiles-GoFiles</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Function goal is to parse TAB delimited Gene Ontology [GO] file and return GO structure</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Bioinformatics</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/MayaGalili/ParseFiles-GoFiles" target="_blank">https://github.com/MayaGalili/ParseFiles-GoFiles</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pombase" target="_blank">PomBase</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Comprehensive online database for fission yeast. PomBase provides comprehensive manual literature curation, annotation of genomic sequence and features,  genome-wide data sets, and supports sophisticated user-defined queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Biology</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://twitter.com/PomBase" target="_blank">https://twitter.com/PomBase</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mimo" target="_blank">MIMO</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>MIMO addresses the first problem by allowing the introduction of gaps and mismatches between query and template pathways and permits -when necessary- supervised queries incorporating a priori biological information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Pathways</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mimo-1-0-molecular-interaction-maps-overlap.html" target="_blank">http://www.mybiosoftware.com/mimo-1-0-molecular-interaction-maps-overlap.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cargo" target="_blank">CARGO</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Portal that uses widgets to allow users to mine literature using iHOP, retrieve disease information from OMIM, visualize 3D SNPs, query protein interactions, and view summarized gene annotation information for cancer related genes in human.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Molecular interactions, pathways and networks</li><li>Humans</li><li>Pathology</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/idhs-dasts" target="_blank">iDHS-DASTS</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Identifying DNase I hypersensitive sites based on LASSO and stacking learning.<br>Enter the query DNA sequences in FASTA format (Example):.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mathematics</li><li>Sequencing</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Feature extraction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pocketannotate" target="_blank">PocketAnnotate</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PocketAnnotate performs binding site level annotation of query proteins.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Protein binding sites</li><li>Nucleic acid sites, features and motifs</li><li>Transcription factors and regulatory sites</li><li>DNA binding sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein binding site prediction</li><li>Query and retrieval</li><li>Transcription factor binding site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Anabel" target="_blank">Anabel</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A software designed for the analysis of binding curves with which to evaluate the interactions of biomolecules.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Literature and language</li><li>Database management</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Query and retrieval</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/SKscience/Anabel" target="_blank">https://github.com/SKscience/Anabel</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PyGMQL" target="_blank">PyGMQL</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Scalable data extraction and analysis for heterogeneous genomic datasets.<br>Python Library for data analysis based on GMQL.<br>The doucumentation can be found at the following link: http://pygmql.readthedocs.io.<br>API for calling interactively the GMQL Engine from Python</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Database management</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Collapsing methods</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iccare" target="_blank">Iccare</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Compares all available EST and mRNA sequences for a query organism against the set of transcripts for a reference organism. The results are presented graphically and relative to the location of genes on the chromosomes of the reference organism.</div></td>
      <td><div class='topic short click_expand'><ul><li>Animals</li><li>Gene transcripts</li><li>Mapping</li><li>Model organisms</li><li>Plants</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence tagged site (STS) mapping</li><li>Sequence tag mapping</li><li>Query and retrieval</li><li>Mapping</li><li>PCR primer design</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>33</div></td>
      <td><div class="link short click_expand"><a href="http://genoweb.toulouse.inra.fr/iccare/Blast/index.php?model=UNDEF" target="_blank">http://genoweb.toulouse.inra.fr/iccare/Blast/index.php?model=UNDEF</a></div></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cellhts2" target="_blank">cellHTS2</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>This package analyzes high-throughput assays performed in microtitre plate formats. The functionality includes data import and management, normalisation, quality assessment, replicate summarisation and statistical scoring. A webpage that provides a detailed graphical overview of the data and analysis results is produced. In our work, we have applied the package to RNAi screens on fly and human cells, and for screens of yeast libraries.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNAi experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li><li>Standardisation and normalisation</li><li>Loading</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>experimental measurement : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>38</div></td>
      <td><div class="link short click_expand"><a href="http://www.dkfz.de/signaling" target="_blank">http://www.dkfz.de/signaling</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/database_manager" target="_blank">Database Manager</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Sequence database manager, including random database generation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Proteomics</li><li>Sequence databases</li><li>Sequence analysis</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Random sequence generation</li><li>Sequence mutation and randomisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database metadata : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database metadata : fasta</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pacsy" target="_blank">PACSY</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Relational database management system that integrates information from the Protein Database Bank (PDB), the Biological Magnetic Resonance Data Bank (BMRB), and the Structural Classification of Proteins (SCOP) database.</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>Database management</li><li>Proteins</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Structure database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>20</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/pacsy-pacsy-maker-b032814-pacsy-analyzer-database-management-system-for-protein-structure-and-chemical-shift-analysis.html" target="_blank">http://www.mybiosoftware.com/pacsy-pacsy-maker-b032814-pacsy-analyzer-database-management-system-for-protein-structure-and-chemical-shift-analysis.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/seqminer2" target="_blank">seqminer2</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>An efficient tool to query and retrieve genotypes for statistical genetics analyses from biobank scale sequence dataset.<br>Seqminer is a highly efficient R-package for retrieving sequence variants from biobank scale datasets of millions of individuals and billions of genetic variants. It supports all variant types, including multi-allelic variants and imputation dosages. It takes VCF/BCF/BGEN/PLINK format as input file, indexes, queries them based upon variant-based index and loads them as R data types such as list or matrix.<br>cfh2 <- seqminer::createSingleChromosomeVCFIndex(vcf.ref.file, vcf.idx.file).<br>vcf.idx.file <- "input.vcf.scIdx".</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Genotype and phenotype</li><li>Biobank</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Imputation</li><li>Data retrieval</li><li>Database search</li><li>Genotyping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/yang-lina/seqminer/blob/master/seqminer2%20benchmark%20script.txt" target="_blank">https://github.com/yang-lina/seqminer/blob/master/seqminer2%20benchmark%20script.txt</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/flextyper" target="_blank">FlexTyper</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>FlexTyper is a tool which enables the creation of an FM-index of raw, next generation sequencing data to enable rapid and flexible sequence queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Genotype and phenotype</li><li>Whole genome sequencing</li><li>DNA polymorphism</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>k-mer counting</li><li>Query and retrieval</li><li>Genome indexing</li><li>Genotyping</li><li>Read mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://wassermanlab.github.io/OpenFlexTyper/" target="_blank">https://wassermanlab.github.io/OpenFlexTyper/</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sprite" target="_blank">SPRITE</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>SPRITE is a protein structural motif database searching program. Users can upload a PDB file of a query protein structure, and SPRITE compares the input coordinates to a database of 3D side chain arrangements of sites and motifs using a graph theoretical approach.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein folds and structural domains</li><li>Proteins</li><li>Protein structural motifs and surfaces</li><li>Structure prediction</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structural motif recognition</li><li>Protein feature detection</li><li>Query and retrieval</li><li>Protein modelling (side chains)</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gfakluge" target="_blank">gfakluge</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A C++ library and command line utilities for the Graphical Fragment Assembly formats | A C++ library and utilities for manipulating the Graphical Fragment Assembly format | GFAKluge is a C++ parser/writer and a set of command line utilities for manipulating GFA files. It parses GFA to a set of data structures that represent the encoded graph. You can use these components and their fields/members to build up your own graph representation. You can also convert between GFA 0.1 <-> 1.0 <-> 2.0 to glue programs that use different GFA versions together</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence trimming</li><li>Parsing</li><li>De-novo assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biojs-io-fasta" target="_blank">biojs-io-fasta</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Parses FASTA files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/progmap" target="_blank">ProGMap</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Facilitates the annotation of newly sequenced proteins by helping users assess the coherence of protein groups found in various databases. It is based on a non-redundant dataset of over 6.6 million protein sequences mapped to 240,000 protein groups. Queries may be made using sequence identifiers, gene symbols, protein functions or amino acid sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Sequencing</li><li>Gene and protein families</li><li>Mapping</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Mapping</li><li>Query and retrieval</li><li>Sequence tagged site (STS) mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ppai" target="_blank">PPAI</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PPAI is a web server for predicting protein-aptamer interactions.<br>The interactions between proteins and aptamers are prevalent in organisms and play an important role in various life activities. Thanks to the rapid accumulation of protein-aptamer interaction data, it is very important and feasible to construct an accurate computational model to predict protein-aptamer interactions, which is benefit for understanding mechanisms of protein-aptamer interactions and improving aptamer-based therapies.<br>PPAI is a novel web-server to predict aptamers and protein-aptamer interactions with sequence features of proteins/aptamers and machine learning framework which integrated Adaboost and Random Forest. PPAI not only provides more accurate prediction functions, but also provides the protein aptamer information to the user for query.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Functional, regulatory and non-coding RNA</li><li>Machine learning</li><li>Protein interactions</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence feature detection</li><li>Protein feature detection</li><li>Protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/texmed" target="_blank">TeXMed</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Query PubMed and store references in BibTeX format.</div></td>
      <td><div class='topic short click_expand'><ul><li>References</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/texmed-2-0-7-bibtex-interface-ncbi-pubmed.html" target="_blank">http://www.mybiosoftware.com/texmed-2-0-7-bibtex-interface-ncbi-pubmed.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/grasp" target="_blank">GRASP</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>GRASP is a gene annotation tool for metagenomic studies. GRASP assembles the fragmented short-peptides, which are called from the NGS reads, and aligns the assembled contigs to the query reference protein. GRASP achieves much higher sensitivity than BLASTP for gene annotation purpose.</div></td>
      <td><div class='topic short click_expand'><ul><li>Studies</li><li>NGS</li><li>Peptides</li><li>Assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/grasp-0-0-4-guided-reference-based-assembly-of-short-peptides.html" target="_blank">http://www.mybiosoftware.com/grasp-0-0-4-guided-reference-based-assembly-of-short-peptides.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/icgc_data_portal" target="_blank">ICGC Data Portal</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The International Cancer Genome Consortium (ICGC) is a collaborative effort to characterize genomic abnormalities in 50 different cancer types. Currently has data from 24 cancer projects. Open access data (eg. Somatic mutations, copy number alterations, structural rearrangements, gene expression, miRNAs). Easy to search and analyze data, construct queries across multiple data sets.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Genetic variation</li><li>Oncology</li><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Generation</li><li>Query and retrieval</li><li>Deposition</li><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>283</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/marq" target="_blank">MARQ</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Web based application that allows users to compare a query set of genes (query signature) against a gene signature database built from GEO datasets for different organisms and platforms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Pathology</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression analysis</li><li>Query and retrieval</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>20</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/TCW" target="_blank">TCW</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Analysis of single and comparative transcriptomes | Download TCW package: www.agcol.arizona.edu/software/tcw or from this site | Transcriptome Computational Workbench | AGCoL : TCW Home : Doc Index : singleTCW Guide : DE Guide : multiTCW Guide : Tour | Description: For single-transcriptome (singleTCW) - similarity search against annotation databases, GO annotation, ORF finding and differential analysis. For multi-transcriptome (multiTCW) - compute similar pairs, provide statistics for pairs, compute clusters, provide statistics for clusters. Both singleTCW and multiTCW provide graphical interfaces for extensive query and display of the results | TCW provides analysis and query for sequences (transcriptome, proteome), and comparison of species sequences. TCW uses Java, mySQL, optionally R, and various External packages. It has been tested on Linux and Mac</div></td>
      <td><div class='topic short click_expand'><ul><li>Plant biology</li><li>Transcriptomics</li><li>RNA-Seq</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>de Novo sequencing</li><li>Coding region prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.agcol.arizona.edu/software/tcw" target="_blank">http://www.agcol.arizona.edu/software/tcw</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/netmatch" target="_blank">NetMatch</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>NetMatch: a Cytoscape plugin for searching biological networks. Searches networks for subcomponents matching a given query. A bioinformatics software platform for the visualization and analysis of biological networks.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network analysis</li><li>Network simulation</li><li>Weighted correlation network analysis</li><li>Query and retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>66</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/imaaagine" target="_blank">IMAAAGINE</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The IMAAAGINE server is a protein structural motif database searching program. Users can define an amino acid arrangment of 3 to 8 residues which will be converted into a search pattern. The search using the query is carried out against the PDB. 3D arrangements of amino acids in PDB structures that conform to the query will be returned to the user.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Protein structural motifs and surfaces</li><li>Small molecules</li><li>Structure prediction</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structural motif recognition</li><li>Residue distance calculation</li><li>Query and retrieval</li><li>Protein modelling (side chains)</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/planttfdb" target="_blank">PlantTFDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Plant Transcription Factor Database (PlantTFDB) is database that attempts to identify a comprehensive set of plant genes coding for transcription factors. A web interface to the database allows users to browse by species, search using a sequence identifier, or query using BLAST.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Plant biology</li><li>Gene expression</li><li>Gene and protein families</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network prediction</li><li>Transcription factor binding site prediction</li><li>Gene regulatory network analysis</li><li>Query and retrieval</li><li>Transcriptional regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>448</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/glassesviewer" target="_blank">GlassesViewer</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Open-source software for viewing and analyzing data from the Tobii Pro Glasses 2 eye tracker.<br>This matlab utility can parse recording files from the Tobii Glasses 2 as stored on the SD card (no need for intervening Tobii software). Once parsed, it will display this data in a MATLAB GUI. Optional eye videos are also supported. In this GUI, sync events can be shown, and data can be optionally annotated (e.g. as slow and fast phase intervals) by hand or by a classifier algorithm, or such annotations can be loaded from file.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Sequencing</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phasing</li><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cyby2" target="_blank">CyBy2</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>CyBy2 is a library for writing chemical and biological data management systems. It is a strongly typed, purely functional framework for chemical data management. This library provides a set of utility functions and data types for writing purely functional client/server applications for handling and querying chemical and biological data. It is split into several modules and comes with an example implementation of a HTTP-server and web client for a fully functional chemical data management system.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li><li>Biological databases</li><li>Medicinal chemistry</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Splitting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ares-gt" target="_blank">ARES-GT</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>ARES-GT is a python script that identify CRISPR targets in query sequences (in FASTA format) and can also identify CRISPR targets matching multiple genes. Several online CRISPR tools exists with very good performance, however only a set of availables genomes can be selected (in some cases only a few, though in some tools more than 1500 can be selected) but none of them can identify sequences to be able to target multiple genes. This work usually needs manual (an slow) evaluation of alignments and offtargets.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li><li>Gene expression</li><li>Genetic engineering</li><li>RNA</li><li>Sequence assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Editing</li><li>SNP detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/megante" target="_blank">MEGANTE</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Perform genomic sequence annotations by uploading a sequence and selecting the species to query. Automatically runs several analysis and integrates the results to select the appropriate consensus exon-intron structures and to predict open reading frames (ORFs) at each locus. Functional annotation and a functional domain search, are also performed for the predicted ORFs. The resultant annotation information is visualized with GBrowse and the results can be downloaded in Microsoft Excel format.</div></td>
      <td><div class='topic short click_expand'><ul><li>Agricultural science</li><li>Genomics</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mallard" target="_blank">Mallard</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Program for screening 16S rRNA clone libraries, and similar multiple sequence alignments for chimeras and other anomalies.The program is based on the Pintail algorithm which works by comparing evolutionary distances between a query and subject sequence over the length of the 16S rRNA gene (small subunit rRNA), by employing a sampling window of specified size, progressing a fixed number of bases at a time along the length of the gene.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Nucleic acid sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mallard-1-02-screen-16s-rrna-clone-libraries-chimeras.html" target="_blank">http://www.mybiosoftware.com/mallard-1-02-screen-16s-rrna-clone-libraries-chimeras.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tfgd" target="_blank">Tomato Functional Genomics Database (TFGD)</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Tomato Functional Genomics Database provides a comprehensive resource to store, query, mine, analyze, visualize and integrate large-scale tomato functional genomics data sets. Computational pipelines have been developed to process microarray, metabolite and sRNA data sets archived in the database. It is also designed to enable users to easily retrieve biologically important information through a set of efficient query interfaces and analysis tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Gene transcripts</li><li>Gene expression</li><li>Functional, regulatory and non-coding RNA</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression analysis</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>73</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/residb" target="_blank">resiDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>ResiDB is a user-friendly sequence similarity-dependent database manager for bacteria, fungi, viruses, protozoa, invertebrate, plants, archaea, environmental and whole genome shotgun sequence data.<br>Create a new database Access existing databases Load sample results. An automated database manager for sequence data.<br>Your rapid sequence similiarity-dependent database manager!.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogeny</li><li>Plant biology</li><li>Model organisms</li><li>Whole genome sequencing</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Editing</li><li>Data retrieval</li><li>Antimicrobial resistance prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PDBUSQLExtractor" target="_blank">PDBUSQLExtractor</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Scalable Extraction of Big Macromolecular Data in Azure Data Lake Environment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Nucleic acid structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Parsing</li><li>Nucleic acid folding analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/wft4galaxy" target="_blank">wft4galaxy</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Testing module for Galaxy workflows.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data quality management</li><li>Bioinformatics</li><li>Laboratory information management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/e-pix" target="_blank">E-PIX</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>E-PIX® is a record linkage tool for managing multi-million patients in research projects at a large university hospital in Germany. The Record Linkage and ID Management solution E-PIX (Enterprise Identifier Cross Referencing) applies the propabilistic Fellegi-Sunter-algorithm and the Levenshtein distance to avoid duplicate participant entries. The independent software module facilitates participant management and multisite-aggregation of medical research data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biomaterials</li><li>Public health and epidemiology</li><li>Mapping</li><li>Database management</li><li>Data security</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Information extraction</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/mosaic-hgw/E-PIX/tree/master/docker/standard" target="_blank">https://github.com/mosaic-hgw/E-PIX/tree/master/docker/standard</a></div></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/geoboost2" target="_blank">GeoBoost2</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A natural languageprocessing pipeline for GenBank metadata enrichment for virus phylogeography.<br>GeoBoost2 (part of the ZoDo project) is a system for extracting the location of infected hosts (LOIH) for a given set of GenBank metadata accessions. This involves API implementations that utilize NCBI API's for accessing and parsing information and delivering it to the researcher interested in investigating a set of sequences. You can also use this tool mine Geographic Locations in scientific articles (available as abstracts on PubMed or PMC-OA) or text files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Infectious disease</li><li>Public health and epidemiology</li><li>Workflows</li><li>Biotechnology</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Information extraction</li><li>Data retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ZooPhy/geoboost2" target="_blank">https://github.com/ZooPhy/geoboost2</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/zodiac-ms" target="_blank">ZODIAC</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Zodiac is a database-independent molecular formula annotation using Gibbs sampling reveals unknown small molecules.<br>1 Abstract The confident high-throughput identification of small molecules remains one of the most challenging tasks in mass spectrometry-based metabolomics. SIRIUS has become a powerful tool for the interpretation of tandem mass spectra and shows outstanding performance for identifying the molecular formula of a query compound, being the first step of structure identification. Nevertheless, the identification of both molecular formulas for large compounds above 500 Daltons and novel molecular formulas remains highly challenging. Here, we present ZODIAC, a network-based algorithm for the de novo estimation of molecular formulas.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li><li>Molecular biology</li><li>Plant biology</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Natural product identification</li><li>Query and retrieval</li><li>Statistical calculation</li><li>Spectral library search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/craft-genome" target="_blank">CRAFT</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>CRAFT is a tool for Compact Representations of large-scale Alignment-Free genomic/metagenomic sequences daTabase and fast sequence comparisons between sequencing data/long genomic sequences and the compact database.CRAFT parses the input genomic/metagenomic sequences, counts the co-occurrences of adjacent k-mer pairs, learns the embeddings, searches against archived repositories by the dissimilarity between the input and the archived sequences in the embedding space, and finally reports the best matches in various ways.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomic sequencing</li><li>Sequence assembly</li><li>Metagenomics</li><li>Model organisms</li><li>Whole genome sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li><li>k-mer counting</li><li>Parsing</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/togotable" target="_blank">TogoTable</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>TogoTable TogoTable is a web tool to add user-specified annotations from several biological databases to a user-uploaded table that contains identifiers of a biological database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Systems biology</li><li>Sequence analysis</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Query and retrieval</li><li>Data retrieval</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pcfamily" target="_blank">PCFamily</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The PCFamily server identifies template-based homologous protein complexes or families, and infers functional modules for query proteins. Useful for binding model visualizations and annotating query proteins.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence composition, complexity and repeats</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Gene and protein families</li><li>Molecular modelling</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein identification</li><li>Query and retrieval</li><li>Protein fold recognition</li><li>Protein modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/xman" target="_blank">xMAN</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Simple and efficient method for the mapping of millions of query oligonucleotide fragments to the subject genome of any given length.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/xman-20060717-map-oligonucleotides-subject-genome.html" target="_blank">http://www.mybiosoftware.com/xman-20060717-map-oligonucleotides-subject-genome.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gretel" target="_blank">Gretel</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>On the complexity of haplotyping a microbial community.<br>An algorithm for recovering haplotypes from metagenomes. Sister to Hansel.<br>Gretel is a Python package providing a command line tool for the recovery of haplotypes from metagenomic data sets. Gretel parses an alignment of reads into a Hansel matrix and uses the evidence of SNP pairs observed to appear on the same reads to probabilistically reconstruct the most likely haplotypes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microbial ecology</li><li>Metagenomics</li><li>Sequence assembly</li><li>DNA polymorphism</li><li>Metatranscriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li><li>Parsing</li><li>Haplotype mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PEATmoss" target="_blank">PEATmoss</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A unified gene expression atlas for the model plant Physcomitrella patens.<br>PEATmoss is an interactive tool to visualize gene expression data. These help pages explain several ways to query the expression data and to visualize the results, enabling the comparison of the expression profiles of several genes simultaneously, as well as to visualize co-expression information. One section describes how to download the expression data in a format that is easy to export to spreadsheet software and to generate figures that are ready for presentations or publications.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Imaging</li><li>Plant biology</li><li>Microarray experiment</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Expression data visualisation</li><li>Query and retrieval</li><li>Expression profile comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bio-cigar" target="_blank">Bio-Cigar</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Parse CIGAR strings and translate coordinates to/from reference/query.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/martview" target="_blank">MartView</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Tool for data retrieval and data mining that integrates data from Ensembl. Through the web interface it allows you to apply a series of filters to create custom datasets which can be converted to several useful output formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Gene transcripts</li><li>Data integration and warehousing</li><li>Pathology</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Query and retrieval</li><li>Data retrieval</li><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>321</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hpastainr" target="_blank">HPAStainR</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A Bioconductor and Shiny app to query protein expression patterns in the Human Protein Atlas.<br>HPAStainR package to be submitted to Bioconductor.<br>HPAStain.R is an R package/Shiny app used to query the Human Protein Atlas for staining data. The purpose of this tool is to test if a list of proteins/genes is associated with a certain cell type in a HPA tested tissue. E.g. you have a list of protein coding genes from a differential expression single cell analysis and want to see if these proteins are associated with a known cell type. Instead of querying HPA multiple times you can load your list in HPAStainR which will return a ranked table of the cell types with the most protein staining.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Genotype and phenotype</li><li>Cell biology</li><li>Protein expression</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Expression analysis</li><li>Deisotoping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/tnieuwe/HPAStainR" target="_blank">https://github.com/tnieuwe/HPAStainR</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/netmatchstar" target="_blank">NetMatchStar</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Find all the occurrences of a query graph in a network and check for its significance as a motif with respect to seven different random models.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Plug-in</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/fabiorinnone/NetMatchStar/tree/v3.1" target="_blank">https://github.com/fabiorinnone/NetMatchStar/tree/v3.1</a></div></td>
      <td><div class='short click_expand'>CC-BY-NC-SA-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tritrypdb" target="_blank">Kinetoplastid genome resources (TritryPDB)</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Integrated database providing access to genome-scale datasets for kinetoplastid parasites. Integrates datasets from Leishmania braziliensis, L. infantum, L. major, L. tarentolae, Trypanosoma brucei and T. cruzi. Users may examine individual genes or chromosomal spans in their genomic context, including syntenic alignments with other kinetoplastid organisms. Data can be interrogated utilizing a search strategy system that enables a user to construct complex queries combining multiple data types.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Animals</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li><li>Generation</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>489</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gpsdb" target="_blank">GPSDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Collection of gene and protein names, organized by species that can be used to search for a given gene/protein name, retrieve all synonyms for this entity, and query Medline with a set of user-selected terms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Gene and protein families</li><li>Ontology and terminology</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein feature detection</li><li>Named-entity and concept recognition</li><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/prodmx" target="_blank">ProdMX</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Rapid query and analysis of protein functional domain based on compressed sparse matrices.<br>ProdMX : Protein Functional Domain based on Compressed Sparse Matrices.<br>ProdMX is a tool with user-friendly utilities developed to facilitate high-throughput analysis of protein functional domains and domain architectures. The ProdMX employs a compressed sparse matrix algorithm to reduce computational resources and time used to perform the matrix manipulation during functional domain analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Gene expression</li><li>Function analysis</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein function prediction</li><li>Protein architecture analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/somm4mc" target="_blank">SOMM4mC</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A second-order Markov model for DNA N4-methylcytosine site prediction in six species.<br>With Cluvio you can run SQL queries against your database and visualize the results as beautiful interactive dashboards that can easily be shared with your team. Cluvio supports all major SQL databases like Postgres, MySQL, Redshift, Athena, BigQuery, Snowflake, Presto, Microsoft SQL Server, Oracle, Google Cloud Platform and Exasol.<br>A web server for Identifying Family of Efflux Proteins in Transporters using Convolutional Neural Networks Model.<br>We proposed a new identification method nemed SOMM4mC based on a Markov model which makes use of the transition probability between adjacent nucleotides to identify 4mC site.<br>DNA N4-methylcytosine (4mC) modification occurs mainly in prokaryotic DNA.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenetics</li><li>Machine learning</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Network analysis</li><li>PTM site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="http://www.insect-genome.com/MM4mc/server.html" target="_blank">http://www.insect-genome.com/MM4mc/server.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/REPIC" target="_blank">REPIC</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A database for exploring N6-methyladenosine methylome.<br>REPIC (RNA Epitranscriptome Collection) is a database dedicated to provide a new resource to investigate potential functions and mechanisms of N6-adenosine methylation (m6A) modifications. Currently, The database includes about 700 samples of 50 public studies that were reprocessed by our refined pipeline. The database also contains ENCODE histone ChIP-seq and DNase-seq data to correlate m6A modifications. It provided multi-dimensional information (e.g. cell or tissue specificity) to query m6A modification relevance in cellular processes. In addition, a built-in modern genome browser presented a comprehensive atlas of m6A modifications which is helpful to visualize m6A modification sites between different samples and conditions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenetics</li><li>Sequencing</li><li>Gene transcripts</li><li>Workflows</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Query and retrieval</li><li>Peak calling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/atomium" target="_blank">atomium</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A Python library for parsing, manipulating and saving PDB, mmCIF and MMTF file formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li><li>Splitting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="http://github.com/samirelanduk/atomium" target="_blank">http://github.com/samirelanduk/atomium</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sigreannot-mart" target="_blank">SigReannot-mart</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Re-annotating oligo sets used for expression microarrays is essential to supply biologists with up-to-date annotations. The tool is a query environment populated with regularly updated annotations for different oligo sets. It permits easy extraction in different formats using filters. It is used to compare probe sets on different criteria, to choose the set for a given experiment to mix probe sets in order to create a new one.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li><li>Microarray experiment</li><li>Probes and primers</li><li>Zoology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Query and retrieval</li><li>Primer and probe design</li><li>Microarray probe design</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/calchas" target="_blank">Calchas</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A web based framework that takes advantage of domain specific ontologies, and Natural Language Processing, aiming to empower exploration of biomedical resources via semantic-based querying and search. The NLP engine analyzes the input free-text query and translates it into targeted queries with terms from the underlying ontology. Each query is passed to the semantically-annotated tools repository, and based on similarity matches, it ranks the available resources.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Natural language processing</li><li>Medical informatics</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Query and retrieval</li><li>Service discovery</li><li>Service composition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>text data : annotated text format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>tool identifier : plain text format (unformatted)</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>19</div></td>
      <td><div class="link short click_expand"><a href="http://calchas.ics.forth.gr" target="_blank">http://calchas.ics.forth.gr</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ppisearch" target="_blank">PPISearch</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PPISearch is a web server for identification of homologous protein-protein interactions (PPIs), inferring transferability of interacting domains and functions of a query protein pair. Homology is searched across multiple species.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Gene and protein families</li><li>Protein interactions</li><li>Protein interaction experiment</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein identification</li><li>Protein modelling</li><li>Protein-protein interaction prediction</li><li>Query and retrieval</li><li>Protein fold recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>27</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/prgdb" target="_blank">Plant Resistance Genes database (prgdb)</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Database provides a comprehensive overview of resistance genes (R-genes) in plants. PRGdb holds more than 16,000 known and putative R-genes belonging to 192 plant species challenged by 115 different pathogens and linked with useful biological information. Data can be examined using different query tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Zoology</li><li>Plant biology</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene prediction</li><li>Query and retrieval</li><li>Antimicrobial resistance prediction</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>94</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sapp" target="_blank">SAPP</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>SAPP is a Semantic Annotation Platform with Provenance and is designed on the basis of Semantic Web and is FAIR by design. The platform and corresponding modules allows you to annotate genomes of various qualities with the full chain of provenance. Resulting is a RDF genome data model which you can query and analyse using SPARQL. Various modules are available which allows you to annotate, visualise and export to various formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Omics</li><li>Genomics</li><li>Computational biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence conversion</li><li>Gene prediction</li><li>Query and retrieval</li><li>Sequence database search</li><li>Genome annotation</li><li>Coding region prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence : fasta</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>18</div></td>
      <td><div class="link short click_expand"><a href="http://sapp.gitlab.io" target="_blank">http://sapp.gitlab.io</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phplabware" target="_blank">PhpLabWare</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>It is a web-driven database management system. Data-entry takes place in forms, which can be altered using phplabware’s plugin mechanism. Search results can be output in reports, which are generated based on simple HTML templates. The (php-based) plugin mechanism allows for easy and fast adaptation of phplabware to any specific needs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Databases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/phplabware-0-61-web-driven-life-sciences-lab-database-management-system.html" target="_blank">http://www.mybiosoftware.com/phplabware-0-61-web-driven-life-sciences-lab-database-management-system.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/multiSLIDE" target="_blank">multiSLIDE</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A query-driven heatmap visualization tool for multi-omics data.<br>Multi-omics Systems-Level Interactive Data Exploration (multiSLIDE).<br>multiSLIDE is an open-source tool for query-driven visualization of quantitative single- or multi-omics data. Using pathways and networks as the basis for data linkage, multiSLIDE provides an interactive platform for querying the multi-omics data by genes, pathways, and intermolecular relationships</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Genotype and phenotype</li><li>Molecular interactions, pathways and networks</li><li>Gene transcripts</li><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Quantification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tie" target="_blank">TIE</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A pan-cancer database of miRNA isoforms.<br>The Tumor IsomiR Encyclopedia (TIE) is a dynamic tool that allows researchers to query and compare isomiR expression across more than 11,000 adult and pediatric tumor samples from The Cancer Genome Atlas (TCGA) and the Therapeutically Applicable Research to Generate Effective Treatment (TARGET) datasets.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li><li>Gene transcripts</li><li>Gene expression</li><li>Pathology</li><li>Exome sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>miRNA expression analysis</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/baget" target="_blank">BAGET</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>BAGET (Bacterial and Archaeal Gene Exploration Tool) is a web service designed to facilitate extraction of specific gene and protein sequences from completely determined prokaryotic genomes. Query results can be exported as a rich text format file for printing, archival or further analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Sequence analysis</li><li>Sequencing</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence analysis</li><li>Protein sequence analysis</li><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mousemine" target="_blank">MouseMine</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Data warehouse for accessing mouse data from Mouse Genome Informatics (MGI). Supports powerful query, reporting, and analysis capabilities, the ability to save and combine results from different queries, easy integration into larger workflows, and a comprehensive Web Services layer.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Genotype and phenotype</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>54</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biospytial" target="_blank">Biospytial</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Biospytial is a modular open source knowledge engine designed to import, organise, analyse and visualise big spatial ecological datasets using the power of graph theory. It handles species occurrences and their taxonomic classification for performing ecological analysis on biodiversity and species distributions. The engine uses a hybrid graph-relational approach to store and access information. The data are linked with relationships that are stored in a graph database, while tabular and geospatial (vector and raster) data are stored in a relational database management system (RDBMS). The graph data structure provides a scalable design that eases the problem of merging datasets from different sources. The linkage relationships use semantic structures (objects and predicates) to answer scientific questions represented as complex data structures stored in the graph database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ecology</li><li>Computational biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Network visualisation</li><li>Query and retrieval</li><li>Network analysis</li><li>Ecological modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>biodiversity data : tiff</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Suite</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/molgor/biospytial" target="_blank">https://github.com/molgor/biospytial</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rnn-virseeker" target="_blank">RNN-VirSeeker</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A deep learning method for identification of short viral sequences from metagenomes.<br>To utilize RNN-VirSeeker, Python packages "sklearn", "numpy" and "matplotlib" are needed to be previously installed.<br>To make a prediction, users' own query contigs should be edited into a ".csv" file, where every line contains a single query contig. Through test.py, RNN-VirSeeker will give a set of scores to each query contig, higher of which represents its classification result.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomics</li><li>Metagenomic sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CCLA" target="_blank">Cancer Cell Line Authentication (CCLA)</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>An accurate method and web server for cancer cell line authentication using gene expression profiles.<br>Abstract Cancer cell lines (CCLs) as important model systems play critical roles in cancer researches. The misidentification and contamination of CCLs are serious problems, leading to unreliable results and waste of resources. Current methods for CCL authentication are mainly based on the CCL-specific genetic polymorphisms, whereas no method is available for CCL authentication using gene expression profiles. Here, we developed a novel method and homonymic web server (CCLA, Cancer Cell Line Authentication, http: bioinfo.life.hust.edu.cn web CCLA ) to authenticate 1,291 human CCLs of 28 tissues using gene expression profiles. CCLA curated CCL-specific gene signatures and employed machine learning methods to measure overall similarities and distances between the query sample and each reference CCL</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Oncology</li><li>Genetic variation</li><li>Gene expression</li><li>Microarray experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Query and retrieval</li><li>Expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pred-gpcr" target="_blank">PRED-GPCR</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PRED-GPCR is a tool which queries user-supplied sequences against a database of HMMs corresponding to G-protein coupled receptor (GPCR) families in order to determine which GPCR family the query sequence most resembles.</div></td>
      <td><div class='topic short click_expand'><ul><li>Physiology</li><li>Sequencing</li><li>Gene and protein families</li><li>Sequence analysis</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Transmembrane protein analysis</li><li>Query and retrieval</li><li>Transmembrane protein prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>47</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/kobas" target="_blank">KOBAS</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>KEGG Orthology-Based Annotation System (KOBAS) annotates sequences with KEGG Orthology terms and identifies the frequently occurring (or significantly enriched) pathways among the queried sequences compared against a background distribution. 5 pathway databases are used (KEGG Pathway, PID, BioCyc, Reactome, Panther) and 5 human databases (OMIM, KEGG Disease, FunDO, GAD, NHGRI GWAS).</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Molecular interactions, pathways and networks</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1,863</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cyvcf2" target="_blank">cyvcf2</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Library and software package for fast parsing and querying of VCF and BCF files and illustrate its speed, simplicity and utility.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Genotype and phenotype</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/LIDC" target="_blank">LIDC</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Standardized representation of the LIDC annotations using DICOM.<br>Scripts for converting TCIA LIDC-IDRI collection derived data into standard DICOM representation from project-specific XML format.<br>This repository contains the script used to convert the TCIA LIDC-IDRI XML representation of nodule annotations and characterizations into the DICOM Segmentation object (for annotations) and DICOM Structured Reporting objects (for nodule characterizations).<br>pylidc is an Object-relational mapping (using SQLAlchemy) for the data provided in the LIDC dataset. This means that the data can be queried in SQL-like fashion, and that the data are also objects that add additional functionality via functions that act on instances of data obtained by querying for particular attributes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical imaging</li><li>Oncology</li><li>Tomography</li><li>Model organisms</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image annotation</li><li>Query and retrieval</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/QIICR/lidc2dicom" target="_blank">https://github.com/QIICR/lidc2dicom</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/4dxpress" target="_blank">4DXpress</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A platform to query and compare gene expression data during the development of the major model animals (zebrafish, drosophila, medaka, mouse). The high resolution expression data was acquired through whole mount in situ hybridsation-, antibody- or transgenic experiments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>28</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pevosoar" target="_blank">pevoSOAR</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PvSOAR takes a PDB ID or structure file as input, and searches for other proteins with surface regions that are similar to the query structure.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein folds and structural domains</li><li>Proteins</li><li>Protein structure analysis</li><li>Protein structural motifs and surfaces</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Molecular surface calculation</li><li>Query and retrieval</li><li>Database search</li><li>Structural similarity search</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>58</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ispider_central" target="_blank">ISPIDER Central</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Provides a range of services for integrative data analysis in mass spectrometry-based proteomics. It enables users to query multiple proteomic repositories and to add information to proteins retrieved from searches for integrative in silico experiments in proteomics.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data architecture, analysis and design</li><li>Proteomics experiment</li><li>Proteomics</li><li>Proteins</li><li>Data integration and warehousing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pogos" target="_blank">pogos</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Provide simple utilities for querying bhklab PharmacoDB, modeling API outputs, and integrating to cell and compound ontologies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Pharmacogenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/pogos/" target="_blank">http://bioconductor.org/packages/pogos/</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biomart_central" target="_blank">BioMart Central</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>This tool is a first of its kind, community-driven effort to provide unified access to dozens of biological databases spanning genomics, proteomics, model organisms, cancer data, ontology information and more. The system also simplifies cross-database searches that might otherwise require several complicated steps.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Zoology</li><li>Database management</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>115</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genecite" target="_blank">GeneCite</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Generalized query application that allows you to specify sophisticated sets of queries and generates a table of the number of bio-related records found for each query. The table can be presented as a Web page or in a standard spreadsheet format that will allow you to view the full output of only those queries that generate an interesting number of records.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/genecite-3-0-high-throughput-literature-pathway-mining.html" target="_blank">http://www.mybiosoftware.com/genecite-3-0-high-throughput-literature-pathway-mining.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/strainflair" target="_blank">StrainFLAIR</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Strain-level profiling of metagenomic samples using variation graphs.<br>StrainFLAIR (STRAIN-level proFiLing using vArIation gRaph) is a tool for strain identification and quantification that uses variation graph representation of genes sequences. StrainFLAIR is sub-divided into two main parts: first, an indexing step that stores clusters of reference genes into variation graphs, and then, a query step using mapping of metagenomic reads to infere strain-level abundances in the queried sample. The input for indexation is a collection of complete genomes, draft genomes or metagenome-assembled genomes from which genes will be predicted. The input for query is the so-created graph and any read file.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Sequence assembly</li><li>Metagenomics</li><li>Genotype and phenotype</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Sequence assembly</li><li>Query and retrieval</li><li>Expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ddbj" target="_blank">DDBJ</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Nucleotide sequence archive database and accompanying database tools for sequence submission, entry retrieval and annotation analysis. A new archive for quantitative genomics data, the DDBJ Omics aRchive (DOR). The DOR stores quantitative data both from the microarray and high-throughput new sequencing platforms. Other improvements include improved content of the patent sequence, released a new submission tool of the Sequence Read Archive (DRA) and the Read Annotation Pipeline.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Database management</li><li>Sequencing</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li><li>Query and retrieval</li><li>Deposition</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>129</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/RNAactDrug" target="_blank">RNAactDrug</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A comprehensive database of RNAs associated with drug sensitivity from multi-omics data.<br>RNAactDrug a comprehensive database of RNAs associated with Drug sensitivity from multi-omics data.<br>RNAactDrug is a comprehensive resource for querying associations between drug sensitivity and RNA molecules including mRNAs, lncRNAs and miRNAs at four molecular levels (expression, copy number variation, mutation and methylation) from integrated analysis of three large-scale pharmacogenomic databases (GDSC, CellMiner and CCLE). Currently, RNAactDrug contains more than 4924200 RNA molecules-drug sensitivity associations at multidimensional molecular levels covering more than 19770 mRNAs, 11119 lncRNAs, 438 miRNAs and 4155 drugs. RNAactDrug aims to help make use of the full breadth and depth of existing large-scale pharmacogenomic data by making this knowledge accessible to both basic researchers</div></td>
      <td><div class='topic short click_expand'><ul><li>Pharmacogenomics</li><li>Functional, regulatory and non-coding RNA</li><li>Gene transcripts</li><li>Transcription factors and regulatory sites</li><li>Molecular biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>miRNA expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/APPAGATO" target="_blank">APPAGATO</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>APproximate PArallel and stochastic GrAph querying TOol for biological networks.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Local alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td><div class="link short click_expand"><a href="https://bitbucket.org/federico_busato/appagato" target="_blank">https://bitbucket.org/federico_busato/appagato</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/causaltrail" target="_blank">CausalTrail</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Provides causal hypotheses testing using causal bayesian networks (CBN) and the do-calculus. Includes include multiple data import methods, a flexible query language for formulating hypotheses, as well as an intuitive graphical user interface.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Statistical calculation</li><li>Statistical inference</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>pathway or network : sif</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/archpred" target="_blank">ArchPred</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>ArchPRED predicts the structure of loop regions in protein structures based on a fragment-search based method. Given a query loop of unknown structure, ArchPRED identifies which loops of know structures are likely to share conformational similarity with the query loop.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Protein structure analysis</li><li>Protein structural motifs and surfaces</li><li>Structure prediction</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structure prediction</li><li>Query and retrieval</li><li>Protein modelling (loops)</li><li>Protein modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>82</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/myvariant.info" target="_blank">MyVariant.info</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Provides simple-to-use REST web services to query/retrieve variant annotation data, aggregated from many popular data resources.</div></td>
      <td><div class='topic short click_expand'><ul><li>Nucleic acid structure analysis</li><li>Data acquisition</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>70</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biodb-loader" target="_blank">BioDB-Loader</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>BioDB-Loader toolkit loads several bioinformatics databases into Lisp for facile processin. BioDB-Loader contains utilities for loading flatfiles from the Swiss-Prot, Prosite, Enzyme, EcoCyc, and MetaCyc databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Databases</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/biodb-loader-lisp-tools-bioinformatics.html" target="_blank">http://www.mybiosoftware.com/biodb-loader-lisp-tools-bioinformatics.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/plantpis" target="_blank">PlantPIs</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>It is a web querying system for a database collecting plant protease inhibitors data. Protease inhibitors in plants are naturally occurring proteins that inhibit the function of endogenous and exogenous proteases. PlantPIs contains information extracted principally from MEROPS database, filtered, annotated and compared with data stored in other protein and gene public databases, using both automated techniques and domain expert evaluations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>112</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sagat" target="_blank">SAGAT</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>R package enabling the integration of currently existing microarray data from repositories like NCBIâ€™s Gene Expression Omnibus (GEO) with microarray data querying conditions of interest. The goal of this integration is to better identify differentially expressed genes in the query conditions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/sagat-1-0-svd-augmented-gene-expression-analysis-tool.html" target="_blank">http://www.mybiosoftware.com/sagat-1-0-svd-augmented-gene-expression-analysis-tool.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gcview" target="_blank">GCView</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>GCView facilitates the analysis of the genomic context of a query protein. It uses protein homology to assign corresponding genomic context.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular modelling</li><li>Genomics</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structure assignment</li><li>Protein sequence analysis</li><li>Protein modelling</li><li>Query and retrieval</li><li>Structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>237</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rpf" target="_blank">RPF</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>RPF webserver uses RECALL, PRECISION, and F-MEASURE (RPF) for calculating global NMR structure quality scores, or how well a query 3D structure(s) fit to the experimental NOESY peak list and resonance assignment data.</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>Proteins</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structure assignment</li><li>Query and retrieval</li><li>Mapping</li><li>Protein property calculation</li><li>Protein model validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>31</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/haplor" target="_blank">haploR</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Query web based genome annotation tools HaploReg and RegulomeDB. It gathers information in a data frame which is suitable for downstream bioinformatic analyses. This will facilitate post-genome wide association studies streamline analysis for rapid discovery and interpretation of genetic associations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>GWAS study</li><li>DNA polymorphism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Text mining</li><li>Gene expression QTL analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td><div class="link short click_expand"><a href="https://cran.r-project.org/web/packages/haploR/index.html" target="_blank">https://cran.r-project.org/web/packages/haploR/index.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rgmql" target="_blank">RGMQL</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>This package brings the GenoMetric Query Language functionalities into the R environment. It is a high-level, declarative language to manage heterogeneous genomic datasets for biomedical purposes, using simple queries to process genomic regions and their metadata and properties. It adopts algorithms efficiently designed for big data using cloud-computing technologies allowing GMQL to run on modern infrastructures, in order to achieve scalability and high performance.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/RGMQL/" target="_blank">http://bioconductor.org/packages/RGMQL/</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pcddb" target="_blank">PCDDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The Protein Circular Dichroism Data Bank is a public repository for circular dichroism and synchrotron radiation CD spectral data and their associated experimental metadata. All entries undergo validation and curation procedures to ensure completeness, consistency and quality of the data included. A web-based interface enables users to browse and query sample types, sample conditions, experimental parameters and provides spectra in both graphical display format and as downloadable text files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Structural biology</li><li>Proteins</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein secondary structure assignment</li><li>Deposition</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>55</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gpcrpred" target="_blank">GPCRpred</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Tool that uses a support vector machine based method to make GPCR family and subfamily predictions for a user-supplied query sequence.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Transmembrane protein analysis</li><li>Gene prediction</li><li>Transmembrane protein prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>151</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metabase" target="_blank">MetaBase</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The database of biological databases. A community-curated database containing more than 2000 commonly used biological databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li><li>Systems biology</li><li>Model organisms</li><li>Database management</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Database search</li><li>Database comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>28</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ChemDataExtractor" target="_blank">ChemDataExtractor</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Comparative dataset of experimental and computational attributes of UV/vis absorption spectra.<br>ChemDataExtractor toolkit tailored towards UV/vis absorption spectroscopy peak data.<br>ChemDataExtractor toolkit tailored towards UV/vis absorption spectroscopy peak data. This work adapts the underlying table extraction mechanics of ChemDataExtractor version 1.3 , for higher precision extraction of maximum absorption peak wavelength data and associated extinction coefficients.</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>Cheminformatics</li><li>Database management</li><li>Computer science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Text mining</li><li>Protein extinction coefficient calculation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/edbeard/chemdataextractor-uvvis2018" target="_blank">https://github.com/edbeard/chemdataextractor-uvvis2018</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/neuprint" target="_blank">neuPrint</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Analysis Tools for EM Connectomics.<br>Query and View plugins used by the neuPrintExplorer site to generate reusable query forms and visualisations.<br>Implements a connectomics REST interface that leverages the neuprint data model. neuPrintHTTP can be run in a user authenticated mode or without any authentication. Note: that the authenticated mode (which requires more configuration and setup) is needed to use with neuPrintExplorer web application. The un-authenticated mode is the ideal way to access the neuPrint data programmatically.<br>Python client utilties for interacting with the neuPrint connectome analysis service.<br>exploring inter and intra cellular interactions.<br>Query plugins generate a form to take user input and query the neuPrint cypher api to fetch data from the server.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Machine learning</li><li>Neurobiology</li><li>Electron microscopy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/connectome-neuprint/neuPrintHTTP" target="_blank">https://github.com/connectome-neuprint/neuPrintHTTP</a></div></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/alfy" target="_blank">alfy</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Computer program for comparing one or more query DNA sequences to a set of subject sequences. For each query sequence the output consists of a list of intervals indicating the closest homologue among the set of subject sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>37</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/alfy-1-5-alignment-free-local-homology.html" target="_blank">http://www.mybiosoftware.com/alfy-1-5-alignment-free-local-homology.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/epivizFileServer" target="_blank">Epiviz File Server</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A library to query and transform genomic data from indexed files</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Oncology</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Genome indexing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/epiviz/epivizFileServer" target="_blank">https://github.com/epiviz/epivizFileServer</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ProCarbDB" target="_blank">ProCarbDB</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A database of carbohydrate-binding proteins.<br>An open access database ready to answer your sweetest queries.<br>Carbohydrate-binding proteins play crucial roles across all viruses and organisms</div></td>
      <td><div class='topic short click_expand'><ul><li>Carbohydrates</li><li>Proteins</li><li>Genetic variation</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Structure visualisation</li><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tpms" target="_blank">TPMS</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Tree Pattern Matching Suite. It can manage phylogenetic tree collections, root them, query some peculiar pattern, and detect horizontal transfer.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/tpms-tree-pattern-matching-suite.html" target="_blank">http://www.mybiosoftware.com/tpms-tree-pattern-matching-suite.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GTDB-Tk" target="_blank">GTDB-Tk</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A toolkit to classify genomes with the Genome Taxonomy Database.<br>GTDB-Tk: a toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.<br>GTDB-Tk is a software toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes based on the Genome Database Taxonomy GTDB. It is designed to work with recent advances that allow hundreds or thousands of metagenome-assembled genomes (MAGs) to be obtained directly from environmental samples. It can also be applied to isolate and single-cell genomes. The GTDB-Tk is open source and released under the GNU General Public License (Version 3).</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Proteins</li><li>Taxonomy</li><li>Metagenomics</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome alignment</li><li>Sequence assembly</li><li>Query and retrieval</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>190</div></td>
      <td><div class="link short click_expand"><a href="http://gtdb.ecogenomic.org" target="_blank">http://gtdb.ecogenomic.org</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/atlasrdf" target="_blank">AtlasRDF</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Query the Gene Expression Atlas RDF data at the European Bioinformatics Institute using genes, experimental factors (such as disease, cell type, compound treatments), pathways and proteins. Also contains a function to perform an enrichment of your gene list across Experimental Factor Ontology (EFO) using the Atlas background set.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular genetics</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>139</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/csnap" target="_blank">CSNAP</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>CSNAP is a computational approach for compound target identification based on network similarity graphs. Query and reference compounds are populated on the network connectivity map and a graph-based neighbor counting method is applied to rank the consensus targets among the neighborhood of each query ligand. The CSNAP approach can facilitate high-throughput target discovery and off-target prediction for any compound set identified from phenotype-based or cell-based chemical screens.</div></td>
      <td><div class='topic short click_expand'><ul><li>Targets</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>36</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/csnap-chemical-similarity-network-analysis-pull-down.html" target="_blank">http://www.mybiosoftware.com/csnap-chemical-similarity-network-analysis-pull-down.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pcfun" target="_blank">PCfun</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PCfun (Protein Complex Function). Towards a systematic characterization of protein complex function.<br>Fast and accurate tool for the functional annotation of protein complex queries built upon hybrid unsupervised and supervised machine learning on PubMed Central full-text word embeddings.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li><li>Proteomics</li><li>Protein expression</li><li>Gene expression</li><li>Machine learning</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Gene-set enrichment analysis</li><li>Quantification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mavequest" target="_blank">MaveQuest</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A web resource for planning experimental tests of human variant effects.<br>Explore potential assays and phenotypes for human gene variants.<br>MaveQuest is an online database for querying literature-curated functional assays, phenotypes and clinical interests of human genes for Multiplex Assays of Variant Effect (MAVE) studies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Preclinical and clinical studies</li><li>Personalised medicine</li><li>Pathology</li><li>Human biology</li><li>Rare diseases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant prioritisation</li><li>Variant calling</li><li>Query and retrieval</li><li>Variant classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/kvnkuang/mavequest-front-end/" target="_blank">https://github.com/kvnkuang/mavequest-front-end/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rbiopaxparser" target="_blank">rBiopaxParser</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Parses BioPAX files and represents them in R, at the moment BioPAX level 2 and level 3 are supported.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/frankkramer-lab/rBiopaxParser" target="_blank">https://github.com/frankkramer-lab/rBiopaxParser</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/posmed" target="_blank">PosMed</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>To assist in prioritizing candidate genes discovered in a linkage analysis, the tool executes a full-text search of documents given a query word input and ranks the positional cloning candidate genes based on direct and indirect inference of the hit documents. It currently supports prioritization of positional cloning candidate genes in human, mouse, rat and Arabidopsis thaliana.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Humans</li><li>Mapping</li><li>Phenomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression QTL analysis</li><li>Gene expression correlation analysis</li><li>Linkage analysis</li><li>Query and retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/blocks_www_server" target="_blank">Blocks WWW Server</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The Blocks WWW server provides tools to search DNA and protein queries against the Blocks database of multiple alignments, which represent conserved protein regions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Structure prediction</li><li>Proteins</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Multiple sequence alignment</li><li>Query and retrieval</li><li>Sequence alignment analysis (conservation)</li><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>503</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phage-commander" target="_blank">Phage Commander</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Phage Commander is a software tool for rapid annotation of bacteriophage genomes using multiple programs.<br>A graphical utility for querying various gene prediction tools for FASTA formatted DNA sequences.<br>Phage Commander provides utilities to query various gene-encoding prediction tools and results in various formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li><li>Proteomics experiment</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene prediction</li><li>Genome annotation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dbsnp-q" target="_blank">dbSNP-Q</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>DbSNP-Q is a web application for working with the dbSNP relational database. dbSNP is a source of information on single nucleotide polymorphisms (SNPs) and other genetic variation for many different organisms, including humans. dbSNP-Q can be used to query human data from dbSNP for a variety tasks such as genomic mapping, SNP/gene functional properties, allele frequency data and information on the underlying experiments. It also includes documentation and download tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>DNA polymorphism</li><li>Mapping</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>SNP detection</li><li>Query and retrieval</li><li>Genetic variation analysis</li><li>Mapping</li><li>Genotyping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>20</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/datelife" target="_blank">datelife</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Leveraging databases and analytical tools to reveal the dated Tree of Life.<br>R package containing datelife's core functionality.<br>Dashboard ⋅ phylotastic/datelife.<br>Get a phylogenetic tree with branch lengths proportional to geologic time (aka a chronogram) of any two or more lineages of interest to you: use this R package or go to www.datelife.org to make a query of chronograms available for your lineages in the Open Tree of Life’s tree store.<br>Welcome to the DateLife project.<br>An R package, datelife for doing the calculations.<br>Code coverage done right</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenomics</li><li>Literature and language</li><li>Taxonomy</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Phylogenetic tree reconstruction</li><li>Imputation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://codecov.io/gh/phylotastic/datelife" target="_blank">https://codecov.io/gh/phylotastic/datelife</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/updowndistance" target="_blank">UpdownDistance</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Evaluate closeness of tree matches. The method computes the Updown distance between two phylogenetic trees, the query tree and the data tree.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/updowndistance-tool-comparing-phylogenetic-trees.html" target="_blank">http://www.mybiosoftware.com/updowndistance-tool-comparing-phylogenetic-trees.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metalims" target="_blank">MetaLIMS</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A Simple Open-Source Laboratory Information Management System for Small Metagenomic Labs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomics</li><li>Data management</li><li>Laboratory information management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/recode" target="_blank">Recode</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Recode2 is a database of genes that utilize non-standard translation for gene expression purposes. Recoding events described in the database include programmed ribosomal frameshifting, translational bypassing (aka hopping) and mRNA specific codon redefinition. Recode-2 provides access to detailed information on genes known to utilize translational recoding and allows complex search queries, browsing of recoding data and enhanced visualization of annotated sequence elements.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein expression</li><li>Transcription factors and regulatory sites</li><li>Gene transcripts</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Database search</li><li>Frameshift detection</li><li>SECIS element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>35</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biocyc" target="_blank">BioCyc Knowledge Library</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>BioCyc is a collection of pathway/genome databases derived either from the literature (EcoCyc and MetaCyc) or computationally (ie. HumanCyc). EcoCyc is used to visualize gene layout, biochemical reactions, and pathways for the E. coli chromosome; MetaCyc contains the enzymes, reactions, and pathways for a variety of organisms (mostly micro-organisms)</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li><li>Endocrinology and metabolism</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Query and retrieval</li><li>Pathway or network analysis</li><li>Gene to pathway mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>541</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tap-php" target="_blank">TAP</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A static analysis model for PHP vulnerabilities based on token and deep learning technology.<br>With the widespread usage of Web applications, the security issues of source code are increasing. The exposed vulnerabilities seriously endanger the interests of service providers and customers. There are some models for solving this problem. However, most of them rely on complex graphs generated from source code or regex patterns based on expert experience. In this paper, TAP, which is based on token mechanism and deep learning technology, was proposed as an analysis model to discover the vulnerabilities of PHP: Hypertext Preprocessor (PHP) Web programs conveniently and easily. Based on the token mechanism of PHP language, a custom tokenizer was designed, and it unifies tokens, supports some features of PHP and optimizes the parsing.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>RNA immunoprecipitation</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sorting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/geo-platform-annotation-table-reader" target="_blank">GEO Platform Annotation Table Reader</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>GEO Platform Annotation Table Reader is MATLAB tool that reads and parses platform annotation table (*.annot) from GEO.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene functional annotation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/signature" target="_blank">Signature</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>This tool identifies signature genes or genes that are unique to a taxonomic clade and thus useful in phylogenetic analysis. Users input a set of query sequences and returned is a list of taxonomic clades that share signatures with the query.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogeny</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence motif recognition</li><li>Gene prediction</li><li>Transposon prediction</li><li>Query and retrieval</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/spell" target="_blank">SPELL</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Query-driven search engine for large gene expression microarray compendia. Given a small set of query genes, identifies which datasets are most informative for these genes, then within those datasets additional genes are identified with expression profiles most similar to the query set.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Microarray data analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>188</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/spell-serial-pattern-of-expression-levels-locator.html" target="_blank">http://www.mybiosoftware.com/spell-serial-pattern-of-expression-levels-locator.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/schnel" target="_blank">SCHNEL</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Scalable clustering of high dimensional single-cell data.<br>Scalable Clustering of Hierarchical Stochastic Neighbour Embedding hierarchies using Louvain community detection.<br>Clustering of high dimensional data using manifold learning and subsampling. Clustering is performed on a small representative subset of the data and translated back to the full dataset.<br>See the notebooks in the Python Parser folder for explanation on the HSNE datastructure, how to parse the HSNE hierarchy into a python object and a clustering example.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Cell biology</li><li>Transcriptomics</li><li>Cytometry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Dimensionality reduction</li><li>Standardisation and normalisation</li><li>Essential dynamics</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sparqlgraph" target="_blank">SPARQLGraph</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A web-based platform for the visual creation and execution of biological semantic queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Data visualisation</li><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>18</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/sparqlgraph-visual-query-builder-for-biological-rdf-databases.html" target="_blank">http://www.mybiosoftware.com/sparqlgraph-visual-query-builder-for-biological-rdf-databases.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gem_library" target="_blank">GEM library</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A set of very optimized tools for indexing/querying huge genomes/files. Provided so far: a very fast exact mapper, and an unconstrained split-mapper</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Read mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/protrack" target="_blank">ProTrack</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>An Interactive Multi-Omics Data Browser for Proteogenomic Studies.<br>The Clinical Proteomic Tumor Analysis Consortium (CPTAC) initiative has generated extensive multi-omics data resources of deep proteogenomic profiles for multiple cancer types. To enable the broader community of biological and medical researchers to intuitively query, explore, and download data and analysis results from various CPTAC projects, we built a prototype user-friendly web application called “ProTrack” with the CPTAC clear cell renal cell carcinoma (ccRCC) data set ( http: ccrcc.cptac-data-view.org ). Here we describe the salient features of this application which provides a dynamic, comprehensive, and granular visualization of the rich proteogenomic data. Statement of Significance The CPTAC initiative ( https: proteomics.cancer.gov ) has generated multi-omics data for multiple cancer types to understand the proteogenomic aberrations of these malignancies</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Epigenomics</li><li>Proteomics</li><li>Proteogenomics</li><li>Exome sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>DNA barcoding</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ncbi_resources" target="_blank">NCBI Resources</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The National Center for Biotechnology Information (NCBI) provides analysis and retrieval resources for the data in GenBank and other biological data made available through the NCBI web site.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Data mining</li><li>Data integration and warehousing</li><li>Rare diseases</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>PCR primer design</li><li>Service discovery</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,276</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CellFishing.jl" target="_blank">CellFishing.jl</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Ultrafast and scalable cell search method for single-cell RNA-sequencing.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-seq</li><li>Cell biology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Read binning</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/topsan" target="_blank">TOPSAN</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Platform for exploring and annotating structures determined by structural genomics efforts. The platform facilitates collaborative annotation and investigation via a user-friendly web-based interface pre-populated with automatically generated information. Semantic web technologies expand and enrich the database’s content through links to larger sets of related databases, and thus, enable data integration from disparate sources and data mining via conventional query languages.</div></td>
      <td><div class='topic short click_expand'><ul><li>Structural genomics</li><li>Structural biology</li><li>Proteins</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Enrichment analysis</li><li>Protein structure analysis</li><li>Protein structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/prodoc" target="_blank">PRODOC</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Repository of domain assignments from Pfam for proteins encoded in different complete genomes and a tool for querying this information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li><li>Proteins</li><li>Protein structure analysis</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structure assignment</li><li>Protein domain recognition</li><li>Protein structure analysis</li><li>Query and retrieval</li><li>Protein fold recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/apid" target="_blank">APID</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Agile Protein Interaction DataAnalyzer (APID) allows you to query protein-protein interactions using a common platform facilitating comparison across different datasets. Currently supported interaction databases are BIND, HPRD, DIP, IntAct, and MINT.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein folding, stability and design</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li><li>Protein interaction experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein-protein interaction prediction</li><li>Query and retrieval</li><li>Protein interaction analysis</li><li>Protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>174</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/samql" target="_blank">SamQL</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>A Structured Query Language and filtering tool for the SAM/BAM file format.<br>SQL-like query language for the SAM/BAM file format.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Software engineering</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Query and retrieval</li><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Workflow</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hms-ics" target="_blank">HMS-ICS</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>The Hyperlink Management System (HMS) automatically updates and maintains hyperlinks among major databases using various data IDs (e.g. HUGO Gene Symbols, IDs from PDB, UniProt). The ID Converter System (ICS) supports the conversion of data IDs using the corresponding tables in HMS.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Sequence analysis</li><li>Database management</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>21</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pgql" target="_blank">pGQL</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Software tool in particular for analyzing gene expression time courses. It allows its user to interactively define linear HMM queries on time course data using rectangular graphical widgets called probabilistic time boxes. The analysis is fully interactive and the graphical display shows the time courses along with the graphical query. Afterwards the results can be submitted to gPROF directly.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/pgql-analyzing-gene-expression-time-courses.html" target="_blank">http://www.mybiosoftware.com/pgql-analyzing-gene-expression-time-courses.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/msqt" target="_blank">MSQT</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>MSQT is a SNP Query Tool.It was developed to make polymorphism information readily accessible and to assist researchers in selecting distinct sequence changes and developing SNP detection assays.</div></td>
      <td><div class='topic short click_expand'><ul><li>Polymorphism</li><li>DNA</li><li>SNP</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>SNP detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>36</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/msqt-0-7-3-multiple-snp-query-tool.html" target="_blank">http://www.mybiosoftware.com/msqt-0-7-3-multiple-snp-query-tool.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ncbi-taxonomist" target="_blank">NCBI-taxonomist</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>Collecting and managing taxonomic data with NCBI-taxonomist.<br>ncbi-taxonomist handles and manages phylogenetic data from NCBI. Its primary use is to use taxonomic data to find and link related metadata from other Entrez databases, e.g. fetching data for metagenomic data-sets for specific or diverse group of org.<br>Toolbox to manage NCBI taxonomical data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Taxonomy</li><li>Workflows</li><li>Metagenomics</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Parsing</li><li>Taxonomic classification</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://gitlab.com/janpb/ncbi-taxonomist" target="_blank">https://gitlab.com/janpb/ncbi-taxonomist</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PIPE-T" target="_blank">PIPE-T</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>PIPE-T is a Galaxy Workflow for processing and analyzing miR expression profiles by RTqPCR.<br>It is a tool that offers several state-of-the-art options for parsing, filtering, normalizing, imputing and analyzing RT-qPCR expression data. Integration of PIPE-T into Galaxy allows experimentalists with strong bioinformatic background, as well as those without any programming or development expertise, to perform complex analysis in a simple to use, transparent, accessible, reproducible, and user-friendly environment</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Biomarkers</li><li>PCR experiment</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Imputation</li><li>Standardisation and normalisation</li><li>Essential dynamics</li><li>Quantification</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rdfscape" target="_blank">RDFScape</a></div></td>
      <td><div class='short click_expand'>0.667</div></td>
      <td><div class='desc short click_expand'>This is a project that brings Semantic Web “features” to the popular Systems Biology software Cytoscape. It allows to query, visualize and reason on ontologies represented in OWL or RDF within Cytoscape.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Ontology and terminology</li><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>31</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/rdfscape-0-4-2-semantic-web-meets-systems-biology.html" target="_blank">http://www.mybiosoftware.com/rdfscape-0-4-2-semantic-web-meets-systems-biology.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/toxicodb" target="_blank">ToxicoDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ToxicoDB is a database of curated toxicogenomics datasets that provides convenient data summar and visualization to mine these complex data.<br><br>A statistical analysis package for use exploring the relationship between toxicity and gene expression in specific cell types. The package includes a specialized object, the ToxicoSet, which allows unified storage of all data related to a specific toxico-genomic study. Methods accepting this object are inlcuded for summarize results and annotaiton, conduct various analyses as well as to visualize the results thereof.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Toxicology</li><li>Gene expression</li><li>Endocrinology and metabolism</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="http://github.com/bhklab/ToxicoGx" target="_blank">http://github.com/bhklab/ToxicoGx</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ctdquerier" target="_blank">CTDquerier</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Comparative Toxicogenomics Database data extraction, visualization and enrichment of environmental and toxicological studies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data acquisition</li><li>Toxicology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genomenet" target="_blank">GenomeNet</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Network of database and computational resources including KEGG (pathways, interactions, etc.) and DBGET/LinkDB (an integrated database retrieval system). It also hosts several web-based tools for sequence analysis (i.e. Blast, Motif, Clustal W).</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Molecular interactions, pathways and networks</li><li>Proteomics</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence feature detection</li><li>Query and retrieval</li><li>Database search</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>776</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ensembldb" target="_blank">ensembldb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The package provides functions to create and use transcript centric annotation databases/packages. The annotation for the databases are directly fetched from Ensembl using their Perl API. The functionality and data is similar to that of the TxDb packages from the GenomicFeatures package, but, in addition to retrieve all gene/transcript models and annotations from the database, the package provides also a filter framework allowing to retrieve annotations for specific entries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,277</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/jotsetung/ensembldb" target="_blank">https://github.com/jotsetung/ensembldb</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rre" target="_blank">RRE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Extract non-coding regions associated to annotated genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Databases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/rre-20040401-retrieval-non-coding-regulative-elements-annotated-genomic-databases.html" target="_blank">http://www.mybiosoftware.com/rre-20040401-retrieval-non-coding-regulative-elements-annotated-genomic-databases.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scdb" target="_blank">SCDb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated database of stomach cancer.<br>Stomach cancer (SC) is a type of cancer, which is derived from the stomach mucous membrane. As there are non-specific symptoms or no noticeable symptoms observed at the early stage, newly diagnosed SC cases usually reach an advanced stage and are thus difficult to cure. Therefore, in this study, we aimed to develop an integrated database of SC. METHODS:SC-related genes were identified through literature mining and by analyzing the publicly available microarray datasets. Using the RNA-seq, miRNA-seq and clinical data downloaded from The Cancer Genome Atlas (TCGA), the Kaplan-Meier (KM) survival curves for all the SC-related genes were generated and analyzed. The miRNAs (miRanda, miRTarget2, PicTar, PITA and TargetScan databases), SC-related miRNAs (HMDD and miR2Disease databases), single nucleotide polymorphisms (SNPs, dbSNP database), and SC-related SNPs (ClinVar database) were also retrieved from the indicated databases</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Oncology</li><li>Functional, regulatory and non-coding RNA</li><li>Microarray experiment</li><li>DNA polymorphism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Text mining</li><li>miRNA target prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MaizeMine" target="_blank">MaizeMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated data warehouse for MaizeGDB</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/machine_boss" target="_blank">Machine Boss</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Rapid Prototyping of Bioinformatic Automata.<br>Machine Boss allows you to manipulate state machines by concatenating, composing, intersecting, reverse complementing, Kleene-starring, and other such operations. Brief descriptions of these operations are included below. Any state machine resulting from such operations can be run through the usual inference algorithms too (Forward, Backward, Viterbi, EM, beam search, prefix search, and so on).</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Gene expression</li><li>Machine learning</li><li>Sequence sites, features and motifs</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Base-calling</li><li>Local alignment</li><li>Reverse complement</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cosinor" target="_blank">cosinor</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Time distributed data analysis by Cosinor.Online application.<br>Tools for estimating and predicting the cosinor model version 1.1 from CRAN.<br>cosinor is a set of simple functions that transforms longitudinal data to estimate the cosinor linear model as described in Tong (1976). Methods are given to summarize the mean, amplitude and acrophase, to predict the mean annual outcome value, and to test the coefficients.<br>cosinor: Tools for estimating and predicting the cosinor model.<br>Molcan, L. (2019). Time distributed data analysis by Cosinor. Online application. bioRxiv, 805960.<br>Preprint available at BioRxiv, doi: https://doi.org/10.1101/805960.<br>Cosinor Online, cosinor on the web, is a simple web-based application evaluating the presence of 24-h periodicity. Loading data is easy, just copy and paste columns from an Excel file to the web form</div></td>
      <td><div class='topic short click_expand'><ul><li>Physiology</li><li>Database management</li><li>Biology</li><li>Neurobiology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Scatter plot plotting</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://rdrr.io/cran/cosinor/" target="_blank">https://rdrr.io/cran/cosinor/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PhytoMine" target="_blank">PhytoMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An InterMine interface to data from Phytozome, the Plant Comparative Genomics portal of the Department of Energy's Joint Genome Institute, which hosts 81 assembled and annotation genomes, from 78 viridiplantae species.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ibws_ws" target="_blank">IBWS: IST Bioinformatics Web Services</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Bioinformatics group at the National Cancer Research Institute (IST) of Genoa compiled a suite of free web services IBWS.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li><li>Bioinformatics</li><li>Animals</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Service invocation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Suite</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/frogs_upload_tar" target="_blank">frogs_upload_tar</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Allow to upload a TAR archive as one dataset in Galaxy.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>937</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/upload_tar/FROGS_upload_tar/1.1.0" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/upload_tar/FROGS_upload_tar/1.1.0</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phever" target="_blank">PhEVER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database that provides accurate evolutionary and phylogenetic information to analyse the nature of virus-virus and virus-host lateral gene transfers. It is a database of homologous families both (i) between sequences from different viruses and (ii) between viral sequences and sequences from cellular organisms. It offers a clustering of proteins into homologous families containing at least one viral sequences, as well as alignments and phylogenies for each of these families.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Phylogeny</li><li>Gene and protein families</li><li>Model organisms</li><li>Virology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Query and retrieval</li><li>Data retrieval</li><li>Sequence clustering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Chipper" target="_blank">Chipper</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Open-source software for semi-automated segmentation and analysis of birdsong and other natural sounds.<br>Abstract <jats:list list-type="order"><jats:list-item> Audio recording devices have changed significantly over the last 50 years, making large datasets of recordings of natural sounds, such as birdsong, easier to obtain. This increase in digital recordings necessitates an increase in high-throughput methods of analysis for researchers. Specifically, there is a need in the community for open-source methods that are tailored to recordings of varying qualities and from multiple species collected in nature. We developed Chipper, a Python-based software to semi-automate both the segmentation of acoustic signals and the subsequent analysis of their frequencies and durations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Database management</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Standardisation and normalisation</li><li>Data retrieval</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mmsinc" target="_blank">MMsINC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-oriented database of commercially-available compounds for virtual screening and chemoinformatics applications. It contains over 4 million non-redundant chemical compounds in 3D formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>56</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/protag" target="_blank">ProTag</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Smart Tag and Add-In solution for Microsoft Office applications and allows querying the ProThesaurus Web Service from within an Office application while typing your documents.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/167" target="_blank">https://www.biocatalogue.org/services/167</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Py-EM" target="_blank">Py-EM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Software tools for automated transmission electron microscopy.</div></td>
      <td><div class='topic short click_expand'><ul><li>Tomography</li><li>Light microscopy</li><li>Electron microscopy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Microscope image visualisation</li><li>Parsing</li><li>Single particle analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>60</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/egmrg" target="_blank">EGMRG</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Encyclopedia of microbiota regulated genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>207</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sisyphus" target="_blank">SISYPHUS database of protein structure alignments</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Structural alignments for proteins with non-trivial relationships.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>structure alignment (protein) : xml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>88</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pewo" target="_blank">PEWO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PEWO: "Placement Evaluation WOrkflows" a collection of workflows to benchmark phylogenetic placement.<br>Benchmark existing placement software and compare placement accuracy on different reference trees.<br>PEWO compiles a set of workflows dedicated to the evaluation of phylogenetic placement algorithms and their software implementation. It focuses on reporting placement accuracy under different conditions and associated computational costs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Taxonomy</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Nucleic acid feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/taxontree" target="_blank">TaxOnTree</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>TaxOnTree is a phylogenetic program for associating Taxonomic information in a phylogenetic tree.<br>The output is a NEX format tree file configured to be opened in FigTree, that users can promptly color by any taxa or by the ancestrality shared by sequences with query.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Phylogenetics</li><li>Taxonomy</li><li>Proteins</li><li>Workflows</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree visualisation</li><li>Phylogenetic reconstruction</li><li>Phylogenetic tree annotation</li><li>Phylogenetic tree editing</li><li>Species tree construction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://sourceforge.net/projects/taxontree/" target="_blank">https://sourceforge.net/projects/taxontree/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dictybase" target="_blank">dictyBase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database for the biology and genomics of the social amoeba Dictyostelium discoideum.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>80</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bedtk" target="_blank">bedtk</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Bedtk is a set of simple tools to process BED files. It so far implements intersection, subtraction, sorting, merging and computing the breadth of coverage.</div></td>
      <td><div class='topic short click_expand'><ul></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Query and retrieval</li><li>Sorting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/argonaut" target="_blank">Argonaut</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Argonaut is a web-based utility which facilitates collaborative exploration of Condition/Control-type mass spectrometry experiments. Spreadsheets of quantitative values can be uploaded and organized in a way that best describes your specific experimental setup. Brief statistical analysis is conducted on uploaded data to calculate the significance of molecular perturbations for each condition. These data can then be queried on-the-fly to conduct commonly conducted analyses inside the data portal, including volcano plots, GO enrichment, linear correlations, and others.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Lipids</li><li>Metabolomics</li><li>Proteomics</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Quantification</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://www.coonlabdatadev.com/" target="_blank">https://www.coonlabdatadev.com/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/discovery_environment" target="_blank">Discovery Environment</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Numerous methods for expression and promoter analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>GWAS study</li><li>Molecular interactions, pathways and networks</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree analysis</li><li>Query and retrieval</li><li>Genetic mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gen2vcf" target="_blank">GEN2VCF</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A converter for human genome imputation output format to VCF format.<br>BACKGROUND:For a genome-wide association study in humans, genotype imputation is an essential analysis tool for improving association mapping power. When IMPUTE software is used for imputation analysis, an imputation output (GEN format) should be converted to variant call format (VCF) with imputed genotype dosage for association analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>GWAS study</li><li>DNA polymorphism</li><li>Genotype and phenotype</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Parsing</li><li>Imputation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cutevariant" target="_blank">Cutevariant</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Cutevariant is a cross-plateform application dedicated to maniupulate and filter variation from annotated VCF file.</div></td>
      <td><div class='topic short click_expand'><ul><li>Whole genome sequencing</li><li>Biological databases</li><li>Genomics</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Query and retrieval</li><li>SNP annotation</li><li>SNP detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metabosearch" target="_blank">metabosearch</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Performs mass-based metabolite search simultaneously against the four major metabolite databases: Human Metabolome DataBase (HMDB), Madison Metabolomics Consortium Database (MMCD), Metlin, and LipidMaps.</div></td>
      <td><div class='topic short click_expand'><ul><li>Databases</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/metabosearch-mass-based-metabolite-identification-using-multiple-databases.html" target="_blank">http://www.mybiosoftware.com/metabosearch-mass-based-metabolite-identification-using-multiple-databases.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/patentdata" target="_blank">EBI patent sequence database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Collection of non-redundant patent sequence databases, which cover the EMBL-Bank nucleotides patent class and the patent protein databases and contain value-added annotations from patent documents.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Gene and protein families</li><li>Biotechnology</li><li>Sequence analysis</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence database search (by property)</li><li>Sequence database search</li><li>Protein feature detection</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/qax" target="_blank">Qiime Artifact eXtractor (qax)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Qiime2 artifacts are structured compressed archives containing a dataset (e.g., FASTQ reads, representative sequences in FASTA format, a phylogenetic tree in Newick format, etc.) and an exhaustive set of metadata (including the command that generated it, information on the execution environment, citations on the used software, and all the metadata of the artifacts used to produce it).<br>While artifacts can improve the shareability and reproducibility of Qiime workflows, they are less easily integrated with general bioinformatics pipelines, and even accessing metadata in the artifacts requires the full Qiime2 installation (not to mention that every release of Qiime2 will produce incompatible artifacts). Qiime Artifact Extractor (qxa) allows to easily interface with Qiime2 artifacts from the command line, without needing the full Qiime2 environment installed.</div></td>
      <td><div class='topic short click_expand'><ul></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence : zip format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pyneal" target="_blank">Pyneal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Pyneal is an open source software package to support real-time functional magnetic resonance imaging (fMRI). It is entirely Python-based and depends exclusively on free, open source neuroimaging libraries.<br>This repository contains a set of tutorials to accompany the Pyneal real-time fMRI software package. For more information on Pyneal, please see Pyneal Docs.<br>In order to help with the initial setup, as well as test any analysis scripts and network communications later on, Pyneal includes a suite of simulation tools that mimic various inputs and outputs along the data flow path.<br>This respository contains representative data from GE, Siemens, and Philips scanning environments. For a detailed, step-by-step walk-through on setting up and running Pyneal across each of these environments, please see link above.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Medical imaging</li><li>Neurobiology</li><li>MRI</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Scaffolding</li><li>Network analysis</li><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/niasgbdb" target="_blank">NIASGBdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genetic resource database and a plant diseases database, linked by a web retrieval database. The genetic resources database has plant and microorganism search systems to provide information on research materials, including passport and evaluation data for genetic resources with the desired properties. A database of plant diseases in Japan has been developed based on the listing of common names of plant diseases compiled by the Phytopathological Society of Japan.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Genetic variation</li><li>Genetics</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li><li>Genetic mapping</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/AdditiveChem" target="_blank">AdditiveChem</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive bioinformatics knowledge-base for food additive chemicals.<br>A comprehensive food additive database.<br>Supported search options:Text query, Structure, Fragment, Similarity, MCS.<br>Food additives are considered to be the catalysts and headstones of the modern food industry, affecting every step of food production, processing, and storage.<br>Fields searchable in this way are:CAS number,JECFA number,FEMA number,CoE number,Name,InChI,InChI key,INS</div></td>
      <td><div class='topic short click_expand'><ul><li>Endocrinology and metabolism</li><li>Toxicology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/icrf_generator" target="_blank">iCRF Generator</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Generating interoperable electronic case report forms using online codebooks.<br>Semantic interoperability of clinical data requires the use of a common vocabulary, such as SNOMED-CT. Unfortunately, mapping data to such a terminology is time-consuming and requires expert knowledge of both the dataset and the terminology. A viable alternative can be the reuse of codebooks - published dataset definitions which, in some cases, have already been mapped to a terminology. We designed the iCRF Generator, a tool which makes it easy to generate interoperable electronic Case Report Forms for (currently) three major EDCs: OpenClinica 3, Castor and REDCap. The tool currently provides access to several codebooks stored online in Art-Decor, such as:.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Pathology</li><li>Workflows</li><li>Tomography</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Editing</li><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lailaps" target="_blank">LAILAPS Search Engine (transPLANT instance)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A search engine which provides an exploration of plant genomic data for forward genetic research and comprises millions of quarterly updated and indexed information from major life science databases as well as millions of evidence ranked gene annotations to associate these facts to plant genome loci.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>keyword : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database search results : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td><div class="link short click_expand"><a href="https://jib.tools/details.php?id=60" target="_blank">https://jib.tools/details.php?id=60</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/eurocarbdb" target="_blank">EuroCarbDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>EuroCarbDB is a repository of carbohydrate structures, experimental evidence, and carbohydrate-oriented research and analysis tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Carbohydrates</li><li>Experimental design and studies</li><li>NMR</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Structure analysis</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>83</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/VCF-Server" target="_blank">VCF-Server</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web-based visualization tool for high-throughput variant data mining and management.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Literature and language</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Variant filtering</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/biojiang/VCF-Server" target="_blank">https://github.com/biojiang/VCF-Server</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/osml" target="_blank">OSML</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Query-driven dynamic machine learning model for predicting protein-ligand binding sites.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein binding sites</li><li>Machine learning</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein binding site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>20</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/osml-predicting-protein-ligand-binding-sites.html" target="_blank">http://www.mybiosoftware.com/osml-predicting-protein-ligand-binding-sites.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rols" target="_blank">rols</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An interface to the Ontology Lookup Service (OLS) to access and query hundred of ontolgies directly from R.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>ontology term : html</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>ontology data : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td><div class="link short click_expand"><a href="http://lgatto.github.com/rols/" target="_blank">http://lgatto.github.com/rols/</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bio-tds" target="_blank">Bio-TDS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Bioscience query tool discovery system.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Data acquisition</li><li>Biomedical science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Literature search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aceparser" target="_blank">AceParser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>AceParser: Parse an Ace file (Phred/Phrap output) into a tab-delimited assembly alignment file. Options to include embedded annotation from multiple sources.</div></td>
      <td><div class='topic short click_expand'><ul><li>Assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/aceparser-1-33-parse-ace-file-phredphrap-output-tab-delimited-assembly-alignment-file.html" target="_blank">http://www.mybiosoftware.com/aceparser-1-33-parse-ace-file-phredphrap-output-tab-delimited-assembly-alignment-file.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rdf2json" target="_blank">RDF2JSON</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An effective biomedical data migration tool from resource description framework to JSON.<br>A desktop application for transform RDF files to JSON files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Software engineering</li><li>DNA polymorphism</li><li>Database management</li><li>Biotechnology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Mapping</li><li>Data retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/comspari" target="_blank">COMSPARI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Compares two datasets in netCDF or ASCII format.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : netcdf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>25</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/kaleidoscope" target="_blank">Kaleidoscope</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A New Bioinformatics Pipeline Web Application for In Silico Hypothesis Exploration of Omics Signatures.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Protein interactions</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Enrichment analysis</li><li>Expression profile clustering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/kalganem/kaleidoscope" target="_blank">https://github.com/kalganem/kaleidoscope</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/omicsdb" target="_blank">OmicsDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database for exploring functional networks of plant pathogens.<br>Integration of biological networks and omicsdata across species.<br>Accelerating in-silico gene discovery in plants, animals and microbes.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Molecular interactions, pathways and networks</li><li>Plant biology</li><li>Microarray experiment</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Enrichment analysis</li><li>Expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://omicsdb.org" target="_blank">http://omicsdb.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/autoclass" target="_blank">AutoClass</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>AutoClass is a deep learning tool for scRNA-Seq data cleaning and denoising. AutoClass integrates two deep neural network components, an autoencoder and a classifier, as to maximize both noise removal and signal retention.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>RNA-Seq</li><li>Data quality management</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Expression analysis</li><li>Imputation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/shournal" target="_blank">shournal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Bashing irreproducibility with shournal.<br>A (file-) journal for your shell.<br>Note: for a more formal introduction please read Bashing irreproducibility with shournal on bioRxiv.<br>Using the external software shournal-to-snakemake an observed shell-command-series can be directly transformed into rules for the Snakemake workflow engine, a tool to create reproducible and scalable data analyses.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/amypdb" target="_blank">AMYPdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An online database dedicated to amyloid precursor families and to their amino acid sequence signatures.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein properties</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Prediction and recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein sequence : fasta-like (text)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence set (protein) : fasta search results format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>36</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/chimeraslayer" target="_blank">ChimeraSlayer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A chimeric sequence detection utility, compatible with near-full length Sanger sequences and shorter 454-FLX sequences. It involves the different series of steps that operate to flag chimeric 16S rRNA sequences.To run it, you need NAST-formatted sequences generated by the included NAST-iEr utility.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>Sequences</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Chimeric sequence detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1,865</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/chimeraslayer-20101212-chimeric-sequence-detection-utility.html" target="_blank">http://www.mybiosoftware.com/chimeraslayer-20101212-chimeric-sequence-detection-utility.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/excape-db" target="_blank">ExCAPE-DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrated large scale dataset facilitating Big Data analysis in chemogenomics.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li><li>Genomics</li><li>Synthetic chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>36</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/structator" target="_blank">Structator</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A complete and robust open-source software solution for index-based search of RNA sequence-structure patterns.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>RNA</li><li>Nucleic acid structure analysis</li><li>RNA-Seq</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/structator-1-1-fast-index-based-search-for-rna-sequence-structure-patterns.html" target="_blank">http://www.mybiosoftware.com/structator-1-1-fast-index-based-search-for-rna-sequence-structure-patterns.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PBWT-Query" target="_blank">PBWT-Query</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Efficient haplotype matching between a query and a panel for genealogical search | Efficient haplotype matching between a query and panel using PBWT-Query | Ardalan Naseri, Erwin Holzhauser, Degui Zhi, Shaojie Zhang</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li><li>Biobank</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Genome indexing</li><li>Haplotype mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/kegggraph" target="_blank">KEGGgraph</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Interface between a KEGG pathway and a graph object, as well as a collection of tools to analyze, dissect and visualize these graphs. It parses the regularly updated KGML (KEGG XML) files into graph models maintaining all essential pathway attributes. The package offers other functionalities including parsing, graph operation, visualization, etc.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Pathway or network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>197</div></td>
      <td><div class="link short click_expand"><a href="http://www.nextbiomotif.com" target="_blank">http://www.nextbiomotif.com</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/trichdb" target="_blank">Trichomonas genome resources (TrichDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomic-scale datasets associated with the eukaryotic Trichomonas.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>158</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bigdataprocessor2" target="_blank">BigDataProcessor2</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BigDataProcessor2 (BDP2) is a Fiji plugin for interactive processing of TB-sized image data. BDP2 uses BigDataViewer for rendering and the ImgLib2 library for image processing.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioimaging</li><li>Light microscopy</li><li>Electron microscopy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Image analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li><li>Plug-in</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bigdataprocessor" target="_blank">https://github.com/bigdataprocessor</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nanite" target="_blank">nanite</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Using machine learning to assess the quality of atomic force microscopy-enabled nano-indentation data | Loading, fitting, and rating AFM force-indentation data | Documentation — nanite 1.2.4 documentation | The documentation, including the code reference and examples, is available at nanite.readthedocs.io | Loading, fitting, and rating AFM force-distance data | Free document hosting provided by Read the Docs</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Imaging</li><li>Probes and primers</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sorting</li><li>Quantification</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Chordomics" target="_blank">Chordomics</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A visualisation tool for linking function to phylogeny in microbiomes | Interactive visualisation and data annotation for meta-omics data | Chordomics is a tool to visualize and interpret linked data, such as from metagenomics or metaproteomics where both taxonomic and functional data is obtained | The demonstration version of Chordomics is available at https://kmcd.shinyapps.io/chordomics/. Here, a toy dataset is provided to allow you to see the explorative capabilities Chordomics provides before you need to install it!</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Microbial ecology</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/kevinmcdonnell6/chordomics" target="_blank">https://github.com/kevinmcdonnell6/chordomics</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/indigo" target="_blank">INDIGO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>INtegrated Data Warehouse of MIcrobial GenOmes with Examples from the Red Sea Extremophiles.</div></td>
      <td><div class='topic short click_expand'><ul><li>Nucleic acid structure analysis</li><li>Genomics</li><li>Microbiology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>53</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cct" target="_blank">CCT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CCT (Current Comparative Table) is a software package that you can install and set-up on your own system to help you to maintain and search databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Proteins</li><li>Computational biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ihop_ws" target="_blank">iHOP WS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Text-mining methods used to identify information on specific genes and proteins, their physical interactions and regulatory relationships, their relevance in pathologies, and their interactions with chemical compounds. iHOP parses thousands of PubMed documents every day and collects information specific to thousands of different biological molecules.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Literature and language</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Literature search</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>373</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/3107" target="_blank">https://www.biocatalogue.org/services/3107</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pangenome-blast" target="_blank">PLAST</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PLAST is a heuristical method to search for highest scoring local alignments between a DNA sequence query and a graphical pangenome. It takes as input a plain DNA sequence and a pangenome which may either be a set of (multiple) FASTA or FASTQ files or a sequence graph constructed by the tool Bifrost. It then outputs statistically meaningful (gapped) alignments in the style of the NCBI BLAST standard output format. Alignments are calculated based on a "seed-and-extend approach" while traversing the sequence graph. Biologically meaningful alignments are filtered by using an alignment statistic explicitly developed for sequence-to-graph alignments involving graphical pangenomes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Comparative genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Local alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence set (nucleic acid) : binary format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid sequence alignment : blast results</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://gitlab.ub.uni-bielefeld.de/gi/plast" target="_blank">https://gitlab.ub.uni-bielefeld.de/gi/plast</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MINERVA_Platform" target="_blank">MINERVA Platform</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MINERVA (Molecular Interaction NEtwoRk VisuAlization) platform is a standalone webserver for visualization, exploration and management of molecular networks encoded in SBGN-compliant format, including files produced using CellDesigner or SBGN editors. Visualization of uploaded networks generated by the platform is accessible via a web browser to all viewers with the weblink to the resource.<br>MINERVA is a webservice using the Java Server Faces 2 technology. The server side, including data parsing, integration, annotation and verification, is implemented in Java 8. The platform uses the Postgres SQL database for data storage and the Hibernate framework as a middle layer between web server and database. The user web-interface is generated using JSF and PrimeFaces. The displayed content is visualized by OpenLayers or Google Maps API, dedicated JavaScript and CSS.</div></td>
      <td><div class='topic short click_expand'><ul><li>Omics</li><li>Translational medicine</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Surface rendering</li><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence variations : vcf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>125</div></td>
      <td><div class="link short click_expand"><a href="https://git-r3lab.uni.lu/minerva/core" target="_blank">https://git-r3lab.uni.lu/minerva/core</a></div></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fasthmm" target="_blank">FastHMM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>FastHMM and FastBLAST are fast heuristics to replace HMM search, InterProScan, and all-versus-all BLAST. FastHMM uses PSI-BLAST to quickly select likely members of the family and then uses HMMer to confirm those hits. FastBLAST relies on alignments of proteins to known families from FastHMM and from rpsblast against COG. FastBLAST uses these alignments to avoid most of the work of all-versus-all BLAST. FastBLAST further reduces the work by clustering similar sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Databases</li><li>Sequence databases</li><li>Proteins</li><li>Sequences</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/fasthmm-fastblast-1-3-analyzing-large-protein-sequence-databases.html" target="_blank">http://www.mybiosoftware.com/fasthmm-fastblast-1-3-analyzing-large-protein-sequence-databases.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/piqmie" target="_blank">PIQMIe</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Proteomics Identifications & Quantitations Data Management & Integration Service (PIQMIe) is a web server for semi-quantitative proteomics data management and analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Database management</li><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Differential protein expression analysis</li><li>Protein quantification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : sqlite format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/arnikz/PIQMIe" target="_blank">https://github.com/arnikz/PIQMIe</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pronetview-ccrcc" target="_blank">ProNetView-ccRCC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web-based portal to interactively explore clear cell renal cell carcinoma proteogenomics networks.<br>an interactive WebGL visualizer of 3D Networks.<br>NCI supported Clinical Proteomic Tumor Analysis Consortium (CPTAC), is a comprehensive and coordinated effort to accelerate the understanding of the molecular basis of cancer through the application of large-scale proteome (proteins) and genome (DNA and RNA) analysis technologies to different cancer types. CPTAC is composed of expertise in proteomics, genomics, cancer biology, oncology, and clinical chemistry, while creating open community resources that are widely used by the cancer community.<br>This page provides visualization of the networks estimated based on proteogenomic data from CPTAC-CCRCC discovery project. For more information, visit the CPTAC-CCRCC data portal.<br>Co-expression network was constructed based on phospho-peptide data of 103 Clear Cell Renal Cell Carcinoma (ccRCC) samples.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Proteomics</li><li>Cell biology</li><li>Molecular interactions, pathways and networks</li><li>Proteogenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Network visualisation</li><li>Query and retrieval</li><li>Expression correlation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ST-SIM" target="_blank">ST-SIM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Subjective Evaluation of the Spectral Temporal SIMilarity (ST-SIM) Measure for Vibrotactile Quality Assessment.<br>Recent standardization efforts for Tactile Internet (TI) and haptic codecs have paved the route for delivering tactile experiences in synchrony with audio and visual interaction components. Since humans are the ultimate consumers of tactile interactions, it is utmost important to develop objective quality assessment measures that are in close agreements with human perception. In this work, we present the results of a large-scale subjective study of a recently proposed objective quality assessment approach for vibrotactile signals called ST-SIM (Spectral Temporal SIMilarity). ST-SIM encompasses two components: perceptual spectral and temporal similarity measures. Two subjective experiments were conducted to validate ST-SIM, and elicited subjective ratings are used to create a VibroTactile Quality Assessment (VTQA) database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Laboratory techniques</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Standardisation and normalisation</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_CloneSearch" target="_blank">IMGT CloneSearch</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/CloneSearch is the IMGT® tool allowing the search of a clone in IMGT/LocusView. Search is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>map : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>354</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/JointvetchMine" target="_blank">JointvetchMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Aeschynomene evenia genomic data from the Legume Information System</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>155</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/enano" target="_blank">ENANO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An encoder for nanopore FASTQ files.<br>ENANO is a FASTQ lossless compression algorithm especially designed for nanopore sequencing FASTQ files. We tested ENANO and current state-of-the-art compressors on several publicly available nanopore datasets. The results show that our algorithm consistently achieves the best compression performance on every nanopore dataset, while being computationally efficient in terms of speed and memory requirements when compared to existing alternatives.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Base-calling</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pride" target="_blank">PRIDE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Centralized, standards compliant, public data repository that provides protein and peptide identifications together with supporting evidence. This collection references experiments and assays.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteins</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rebaseextract" target="_blank">rebaseextract</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Process the REBASE database for use by restriction enzyme applications.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA binding sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biosounds" target="_blank">BioSounds</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BioSounds is a web application for ecoacoustics that archives and organises soundscape collections, hosts reference recording collections, generates navigable sound spectrograms, and allows annotation of animal vocalisations or other sounds.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Biodiversity</li><li>Database management</li><li>Workflows</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://soundefforts.uni-goettingen.de/biosounds/" target="_blank">https://soundefforts.uni-goettingen.de/biosounds/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fish_oracle" target="_blank">FISH Oracle</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Visualizes data from multiple array CGH or SNP array experiments in a genomic context. Its fast visualization engine and advanced web and database technology supports highly interactive use. It comes with a convenient data import mechanism, powerful search options for genomic elements (like gene names or karyobands), quick navigation and  zooming into interesting regions, and mechanisms to export the visualization into different high quality image formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li><li>DNA polymorphism</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/fish-oracle-visualize-data-multiple-array-cgh-snp-array-experiments.html" target="_blank">http://www.mybiosoftware.com/fish-oracle-visualize-data-multiple-array-cgh-snp-array-experiments.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dbsts" target="_blank">NCBI database of sequence tagged sites</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive database of sequence tagged sites (STSs) derived from STS-based maps and other experiments. STSs are defined by PCR primer pairs and are associated with additional information such as genomic position, genes, and sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>686</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/varadb" target="_blank">VARAdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive variation annotation database for human.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li><li>Molecular interactions, pathways and networks</li><li>DNA polymorphism</li><li>Transcription factors and regulatory sites</li><li>Chromosome conformation capture</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Variant prioritisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/what_if" target="_blank">WHAT IF</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PDB Related Datbases present a series of databases that run parallel to the PDB. DSSP holds the secondary structure of the proteins. PDBREPORT holds reports on the structure quality and lists errors. HSSP holds a multiple sequence alignment for all proteins. The PDBFINDER holds easy to parse summaries of the PDB file content. PDB_REDO holds re-refined, and often improved, copies of all structures solved by X-ray.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Protein structure analysis</li><li>Structure analysis</li><li>Protein secondary structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein secondary structure assignment</li><li>Protein secondary structure prediction</li><li>Protein secondary structure analysis</li><li>Protein secondary structure comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>432</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SCEptRe" target="_blank">SCEptRe</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>SCEptRe provides weekly updated, non-redundant, user customized benchmark datasets with information on the immune receptor features for receptor-specific epitope predictions. This tool extracts weekly updated 3D complexes of antibody-antigen, TCR-pMHC and MHC-ligand from the Immune Epitope Database (IEDB) and clusters them based on antigens, receptors and epitopes to generate benchmark datasets. Users can customize structural quality and clustering parameters (e.g. resolution, R free factors, antigen or epitope sequence identity) to generate these datasets based on their need.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Immunoproteins and antigens</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Peptide immunogenicity prediction</li><li>Epitope mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/baca" target="_blank">BACA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R-based graphical tool to concisely visualise and compare biological annotations queried from the DAVID web service.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Proteins</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence annotation</li><li>Sequence comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/monosaccharidedb" target="_blank">Monosaccharide database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Information on the residues used in glycan databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Carbohydrates</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>carbohydrate report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gemmapper" target="_blank">The GEM (GEnome Multi-tool) Library (GEM Mapper)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A set of very optimized tools for indexing/querying huge genomes/files. Provided so far: a very fast exact mapper, and an unconstrained split-mapper</div></td>
      <td><div class='topic short click_expand'><ul><li>Nucleic acids</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence : fastq-like format (text)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>alignment : sam</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>291</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/natalieq" target="_blank">NatalieQ</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web server for protein-protein interaction network querying.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein folds and structural domains</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein-protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/digger" target="_blank">DIGGER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Domain Interaction Graph Guided ExploreR (DIGGER) integrates protein-protein interactions and domain-domain interactions into a joint graph and maps interacting residues to exons. DIGGER allows the users to query exons or isoforms individually or as a set to visually explore their interactions.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA splicing</li><li>Proteomics</li><li>Gene transcripts</li><li>Protein interactions</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>RNA-Seq analysis</li><li>Protein-protein interaction analysis</li><li>Alternative splicing prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/louadi/RNA-Seq-DIGGER" target="_blank">https://github.com/louadi/RNA-Seq-DIGGER</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mirnameconverter" target="_blank">miRNAmeConverter</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Package containing an S4 class for translating mature miRNA names to different miRBase versions, sequence retrieval, checking names for validity and detecting miRBase version of a given set of names.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Functional, regulatory and non-coding RNA</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/corum" target="_blank">CORUM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database which provides a resource of manually annotated protein complexes from mammalian organisms. Annotation includes protein complex function, localization, subunit composition, literature references and more. All information is obtained from individual experiments published in scientific articles, data from high-throughput experiments is excluded.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein targeting and localisation</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein structure report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>493</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/diprodb" target="_blank">Dinucleotide property database (DiProDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database which is designed to collect and analyse thermodynamic, structural and other dinucleotide properties. The table presenting all the dinucleotide properties can be pruned and rearranged by different criteria. The database contains different export and analysis functions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Nucleic acid structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid structure report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>61</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gene3d_structural" target="_blank">Gene3D structural</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Structural and functional annotation of protein families incorporates several different sources including CATH, DRUGBANK, PFAM etc.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein family report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>59</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CDAP" target="_blank">CDAP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An Online Package for Evaluation of Complex Detection Methods | Complex Detection Analyzer Package (CADP) helps you to execute protein complex detection methods and compare their results in a quick and compact ways. It computes a wide range of evaluation criteria and visualize the comparison of methods in tables and multiplots. It assists you in filtering detected clusters by providing a protein name (STRING ID) as a query and by setting constraint on maximum and minimum number of proteins in detected clusters. The list of methods for detecting protein complexes is as follows: | You may look for the values of evaluation criteria for a specific threshold. This can also be done by CDAP. The PPI datasets are: | You can also upload the result of any other method for evaluating its results. The valid format for uploaded file is described here | The list of evaluation criteria is as follows:</div></td>
      <td><div class='topic short click_expand'><ul><li>Taxonomy</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enrichment analysis</li><li>Protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/uropa" target="_blank">UROPA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Intended for universal genomic range (e.g. peaks) annotation. Based on a configuration file, different target features can be prioritized with multiple integrated queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>ChIP-seq</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Peak calling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/starr" target="_blank">Starr</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This tool facilitates the analysis of ChIP-data, in particular the output of Affymetrix tiling arrays. It provides functions for data import, quality assessment, data visualization and exploration. Furthermore, it includes high-level analysis features like association of ChIP signals with annotated features, correlation analysis of ChIP signals and other genomic data (e.g. gene expression), peak-finding with the CMARRT algorithm and comparative display of multiple clusters of ChIP-profiles.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>ChIP-on-chip</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bykdb" target="_blank">BYKdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Bacterial protein tYrosine Kinase database (BYKdb). Bacterial tyrosine-kinases share no resemblance with their eukaryotic counterparts and they have been unified in a new protein family named BY-kinases. However, BY-kinase sequence annotations in primary databases remain incomplete. This prompted us to develop a specialized database of computer-annotated BY-kinase sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Sequence analysis</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein sequence analysis</li><li>Sequence annotation</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>19</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/servertell" target="_blank">servertell</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Display information about a public server.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/adopt_bbmri-eric" target="_blank">ADOPT BBMRI-ERIC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Pan-European Data Harmonization for Biobanks in ADOPT BBMRI-ERIC.<br>The Samply.MDR-based ETL tools from ADOPT BBMRI-ERIC.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Personalised medicine</li><li>Workflows</li><li>Ontology and terminology</li><li>Biobank</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li><li>Anonymisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lgicdb" target="_blank">Ligand-gated ion channel database (LGICdb)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Nucleic and proteic sequences of the subunits. Multiple sequence alignments can be generated, and some phylogenetic studies of the superfamilies are provided. Finally, the atomic coordinates of subunits, or portion of subunits, are provided when available.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structural motifs and surfaces</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein features : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>59</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/swisstargetprediction" target="_blank">SwissTargetPrediction</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>SwissTargetPrediction is a web server for target prediction of bioactive small molecules. This website allows you to predict the targets of a small molecule. Using a combination of 2D and 3D similarity measures, it compares the query molecule to a library of 280 000 compounds active on more than 2000 targets of 5 different organisms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular modelling</li><li>Model organisms</li><li>Drug discovery</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein binding site prediction</li><li>Protein subcellular localisation prediction</li><li>Molecular docking</li><li>Protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>325</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phenopred" target="_blank">PhenoPred</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based tool designed to detect novel gene-disease associations in humans. It is based on known gene-disease associations, protein-protein interaction data, protein functional annotation at a molecular level and protein sequence data. Machine learning principles are used to integrate heterogeneous data sources. It can be used to prioritize genes based on their likelihood to be associated with a given disease or to prioritize diseases for a given query gene.</div></td>
      <td><div class='topic short click_expand'><ul><li>Human biology</li><li>Machine learning</li><li>Genetics</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>127</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/phenopred-detect-novel-gene-disease-associations-in-humans.html" target="_blank">http://www.mybiosoftware.com/phenopred-detect-novel-gene-disease-associations-in-humans.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pepsweetener" target="_blank">PepSweetener</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Manual annotation of MS data for intact glycopeptides. It displays an interactive map of all theoretical glycopeptides that match the molecular masses of queried precursor ions. Ion matching can be performed in two modes: simple when the search space spans the combination of all tryptic peptides including N-glycosites in the human proteome, and advanced when the search space is custom-built by specifying peptides and glycans.</div></td>
      <td><div class='topic short click_expand'><ul><li>Structural genomics</li><li>Protein modifications</li><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein post-translation modification site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genedb" target="_blank">GeneDB database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sequence data and annotation/curation for the whole range of organisms sequenced by the PSU (Sanger Institute Pathogen Sequencing Units).</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>167</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cmnpd" target="_blank">CMNPD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CMNPD is the comprehensive marine natural products database based on manually curated data. CMNPD currently contains more than 31 000 chemical entities with various physicochemical and pharmacokinetic properties, standardized biological activity data, systematic taxonomy and geographical distribution of source organisms, and detailed literature citations. It is an integrated platform for structure dereplication (assessment of novelty) of (marine) natural products, discovery of lead compounds, data mining of structure-activity relationships and investigation of chemical ecology.</div></td>
      <td><div class='topic short click_expand'><ul><li>Plant biology</li><li>Data mining</li><li>Ecology</li><li>Drug discovery</li><li>Drug metabolism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Relation extraction</li><li>Virtual screening</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biostructurem" target="_blank">bioStructureM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A basic matlab toolkit for analysis protein structure</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein expression</li><li>Biology</li><li>Protein structure analysis</li><li>Biomedical science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/Yuan-Yu/bioStructureM" target="_blank">https://github.com/Yuan-Yu/bioStructureM</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/supertarget" target="_blank">SuperTarget</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web resource for analyzing drug-target interactions. Integrates drug-related info associated with medical indications, adverse drug effects, drug metabolism, pathways and Gene Ontology (GO) terms for target proteins.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Endocrinology and metabolism</li><li>Drug metabolism</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Pathway or network prediction</li><li>Query and retrieval</li><li>Protein subcellular localisation prediction</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>108</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cellinker" target="_blank">Cellinker</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Cellinker is a platform of ligand-receptor interactions for intercellular communication analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Expression data visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scalce" target="_blank">SCALCE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>SCALCE (ske?lz) is fast FASTQ compression utility that utilizes locally consistent parsing for better compression rate. It achieves around 2X more compression than gzip alone.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>93</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/4DGenome" target="_blank">4DGenome</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Curated database of chromatin interaction data covering five organisms: Homo sapiens, Mus musculus, Drosophila melanogaster, Plasmodium falciparum, and Saccharomyces cerevisiae.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li><li>Immunoprecipitation experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>56</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-NC-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/jmztab-m" target="_blank">jmztab-m</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Reference implementation for mzTab 2.0 for metabolomics</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Data submission, annotation and curation</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Format validation</li><li>Loading</li><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>ontology mapping : xml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : mztab</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/lifs-tools/jmzTab-m" target="_blank">https://github.com/lifs-tools/jmzTab-m</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/atcc" target="_blank">American Type Culture Collection (ATCC)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Global biological materials resource and standards organization whose mission focuses on the acquisition, authentication, production, preservation, development, and distribution of standard reference microorganisms, cell lines, and other materials.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>cell line report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Arteria" target="_blank">Arteria</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An automation system for a sequencing core facility.<br>StackStorm packs to automate sequencing center operations.<br>Event-driven sequencing center automation.<br>It forms the core of the Arteria automation system, which you can read about on our website or preprint. This pack integrates with a series of bioinformatic micro-services, which can be found at https://github.com/arteria-project.<br>This pack provides re-usable units for automating tasks at a sequencing core facility using the StackStorm event-driven automation platform.<br>Handling sequencing data from massive parallel sequencing can be a daunting task! And while the process of handling sequencing data will share many of its characteristics across centers, the current norm is one center one solution. This creates a situation where reuse is difficult to achieve and the wheel is invented over and over again. This is a situation that we hope can be remedied in the form of the Arteria project</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Whole genome sequencing</li><li>Literature and language</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Aggregation</li><li>Deposition</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/arteria-project/arteria-packs" target="_blank">https://github.com/arteria-project/arteria-packs</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/clima" target="_blank">CLIMA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Cell line integrated molecular authentication.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>cell line report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>92</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/peptide_match" target="_blank">Peptide Match</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Quickly retrieve all occurrences of a given query peptide from UniProt Knowledgebase (UniProtKB) with isoforms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Sequence composition, complexity and repeats</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence composition calculation</li><li>Protein sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>56</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/3735" target="_blank">https://www.biocatalogue.org/services/3735</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/myvariant" target="_blank">myvariant</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A wrapper for querying MyVariant.info services.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein variants</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Formatting</li><li>Variant pattern analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>matrix : vcf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database cross-mapping : matrix format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/adherer" target="_blank">AdhereR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Computation of adherence to medications from Electronic Healthcare Data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Taxonomy</li><li>Data visualisation</li><li>Drug metabolism</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>20</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phewas-me" target="_blank">PheWAS-ME</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PheWAS me is a web-app for interactive exploration of multimorbidity patterns in PheWAS.<br>Electronic health records (EHRs) linked with a DNA biobank provide unprecedented opportunities for biomedical research in precision medicine.<br>Should multiple-comparisons adjustment be done on P-Values of phewas results file?.<br>Supplied are snps that have been preloaded.</div></td>
      <td><div class='topic short click_expand'><ul><li>Infectious disease</li><li>Personalised medicine</li><li>DNA polymorphism</li><li>Phenomics</li><li>Biobank</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://prod.tbilab.org/phewas_me_info/" target="_blank">https://prod.tbilab.org/phewas_me_info/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pycogent" target="_blank">PyCogent</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Software library for genomic biology. It is a fully integrated and thoroughly tested framework for: controlling third-party applications; devising workflows; querying databases; conducting novel probabilistic analyses of biological sequence evolution; and generating publication quality graphics. It is distinguished by many unique built-in capabilities (such as true codon alignment) and the frequent addition of entirely new methods for the analysis of genomic data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>136</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/YeastMine" target="_blank">YeastMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Search and retrieve S. cerevisiae data, populated by SGD and powered by InterMine</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MAC-INMV-SSR" target="_blank">MAC-INMV-SSR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web application dedicated to genotyping members of Mycobacterium avium complex (MAC) including Mycobacterium avium subsp. paratuberculosis strains.<br>Genotyping of Mycobacterium avium subsp. paratuberculosis (Map) is an indispensable tool for surveillance of this significant veterinary pathogen. For Map, multi-locus variable number tandem repeat analysis (MLVA) targeting mycobacterial interspersed repetitive units (MIRUs) and other variable number variable-number tandem repeats (VNTRs) was established using 8 markers. In the recent past this standard, portable, reproducible and discriminatory typing method has been frequently applied alone or in combinations with multi-locus short-sequence-repeat (MLSSR) sequencing. With the widespread use of these genotyping methods, standardization between laboratories needs to be managed, and knowledge of existing profiles and newly defined genotypes should be indexed and shared.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA polymorphism</li><li>Genotype and phenotype</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Cerebro" target="_blank">Cerebro</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Software for interactive visualization of scRNA-seq data.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Molecular interactions, pathways and networks</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enrichment analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/romanhaa/cerebroApp" target="_blank">https://github.com/romanhaa/cerebroApp</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gs-lage" target="_blank">GS-LAGE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>GS-LAGE (Gene Specific Large-scale Analysis of Gene Expression) is a global analysis strategy for large-scale microarray datasets integrated from public databases such as NCBI GEO.</div></td>
      <td><div class='topic short click_expand'><ul><li>Databases</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Microarray data analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/gs-lage-gene-specific-analysis-of-microarray-global-data.html" target="_blank">http://www.mybiosoftware.com/gs-lage-gene-specific-analysis-of-microarray-global-data.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mygenomebrowser" target="_blank">myGenomeBrowser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based environment that provides biologists with a way to build, query and share their genome browsers.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>CECILL-B</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/echobase" target="_blank">EchoBASE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database that curates new experimental and bioinformatic information about the genes and gene products of the model bacterium Escherichia coli K-12 strain MG1655.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>60</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/allergome" target="_blank">Allergome</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A platform for allergen knowledge.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>89</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mirortho" target="_blank">miROrtho</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MiROrtho contains predictions of precursor miRNA genes covering several animal genomes combining orthology and a Support Vector Machine. We provide homology extended alignments of already known miRBase families and putative miRNA families exclusively predicted by our SVM and orthology pipeline.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogeny</li><li>Functional, regulatory and non-coding RNA</li><li>Evolutionary biology</li><li>Sequence analysis</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>56</div></td>
      <td><div class="link short click_expand"><a href="http://expasy.org" target="_blank">http://expasy.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Alyx" target="_blank">Alyx</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Data architecture and visualization for a large-scale neuroscience collaboration.<br>Database for experimental neuroscience laboratories.<br>Documentation: http://alyx.readthedocs.io.<br>The setup.py script sets up postgres (it creates the database and postgres user), it sets up the alyx/alyx/settings_secret.py file with the postgres database connection information, it creates the Python virtual environment with the dependencies (including django), and it creates all the required SQL tables. Note that the postgres username and password are distinct from Alyx (Django) users and password. There is only one postgres user that is only used locally for maintenance task or by Django</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Data architecture, analysis and design</li><li>Neurobiology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pdbfinder" target="_blank">PDBFINDER reformatted PDB files</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This database holds for each PDB file a structured, search-engine-friendly-formatted entry that holds the data-items most likely needed for people search for certain types of PDB entries. It is not useful to search in atomic coordinates: it is meant to ad searches in the administrative records of PDB files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>structure report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>363</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/wheat_genome" target="_blank">Wheat Genome</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Individual chromosome arms are being assembled and annotated. They are made publically available as they are produced using the genome viewer GBrowse2.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>34</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SPARQL" target="_blank">SPARQL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A hands-on introduction to querying evolutionary relationships across multiple data sources using SPARQL.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Bioinformatics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://mbgd.genome.ad.jp/sparql/" target="_blank">http://mbgd.genome.ad.jp/sparql/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bsgenome" target="_blank">BSgenome</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Infrastructure shared by all the Biostrings-based genome data packages.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence analysis</li><li>Sequence masking</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence record : sequence record format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biostrings" target="_blank">Biostrings</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Memory efficient string containers, string matching algorithms, and other utilities, for fast manipulation of large biological sequences or sets of sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence analysis</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment : sequence record format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,247</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pyomeroupload" target="_blank">PyOmeroUpload</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Python toolkit for uploading images and metadata to OMERO.<br>Code snipets for updating whole properties of whole ISA tree, for example permissions settings.<br>When building a distribution for release through BioConda, the Python setuptools are used.<br>Library of artifacts for integration with Seek4Science using Seek's REST API.<br>Example data, notebooks and scripts for demonstrating features of OMEROConnect and pyOmeroUpload.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biological databases</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Data retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/SynthSys/" target="_blank">https://github.com/SynthSys/</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ncbi_dbprobe" target="_blank">NCBI dbProbe</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Public registry of nucleic acid reagents designed for use in a wide variety of biomedical research applications, together with information on reagent distributors, probe effectiveness, and computed sequence similarities.</div></td>
      <td><div class='topic short click_expand'><ul><li>Laboratory information management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>oligonucleotide probe annotation : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>489</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/AFND" target="_blank">The Allele Frequency Net Database (AFND)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Allele Frequency Net Database (AFND) provides the scientific community with a freely available repository for the storage of frequency data (alleles, genes, haplotypes and genotypes) related to human leukocyte antigens (HLA), killer-cell immunoglobulin-like receptors (KIR), major histocompatibility complex Class I chain related genes (MIC) and a number of cytokine gene polymorphisms in worldwide populations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pharmacogenomics</li><li>Genotype and phenotype</li><li>Immunoproteins and antigens</li><li>Genetic variation</li><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>40</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/EvoFreq" target="_blank">EvoFreq</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Visualization of the Evolutionary Frequencies of Sequence and Model Data | Evolutionary frequency visualization tool of temporal data | Users have options ot install EvoFreq depending on if they want to use 'devtools'. Below we show two popular methods for installation. You will need a fully operational install of R (≥3.5.0) and, optionally, RStudio | Visualization Package for Evolutionary Dynamics from Sequence and Model Data. biorxiv pre-print here</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Oncology</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mmp" target="_blank">Marine Metagenomics Portal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The mission of Marine Metagenomics Portal (MMP) is to provide the marine scientific community with high-quality curated and freely accessible microbial genomics and metagenomics resources.</div></td>
      <td><div class='topic short click_expand'><ul><li>Marine biology</li><li>Microbial ecology</li><li>Taxonomy</li><li>Function analysis</li><li>Data mining</li><li>Data integration and warehousing</li><li>Metagenomics</li><li>Microbiology</li><li>Sequence analysis</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Analysis</li><li>Sequence annotation</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>22</div></td>
      <td><div class="link short click_expand"><a href="https://mmp.sfb.uit.no/" target="_blank">https://mmp.sfb.uit.no/</a></div></td>
      <td><div class='short click_expand'>CC-BY-NC-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mascp_gator" target="_blank">MASCP Gator</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Proteomics aggregation utility  that queries and aggregates information from a variety of online Arabidopsis proteomic resources.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Molecular biology</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mascp-gator-proteomics-aggregation-utility.html" target="_blank">http://www.mybiosoftware.com/mascp-gator-proteomics-aggregation-utility.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/circRNA" target="_blank">circRNA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Biogenesis mechanisms of circular RNA can be categorized through feature extraction of a machine learning model | Contextual_Regression_for_CircRNA | Contextual Regression for CircRNA</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>RNA</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>CpG island and isochore detection</li><li>Parsing</li><li>RNA binding site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/FHIR" target="_blank">FHIR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>FHIR is a standard for health care data exchange. It is a platform specification that defines a set of capabilities use across the healthcare process, in all jurisdictions, and in lots of different contexts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Infectious disease</li><li>Medical informatics</li><li>Allergy, clinical immunology and immunotherapeutics</li><li>Ontology and terminology</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Essential dynamics</li><li>Sequence trimming</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SEQdata-BEACON" target="_blank">SEQdata-BEACON</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>SEQdata-BEACON is a comprehensive database of sequencing performance and statistical tools for performance evaluation and yield simulation in BGISEQ-500.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Sequencing</li><li>Plant biology</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Regression analysis</li><li>Modelling and simulation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aging-atlas" target="_blank">Aging Atlas</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Aging Atlas is a curated biomedical database comprising a range of aging-related multi-omics datasets (i.e. transcriptomics, single-cell transcriptomics, epigenomics, proteomics, and pharmacogenomics), as well as bioinformatics tools to query and visualize these datasets.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Pharmacogenomics</li><li>Epigenomics</li><li>Proteomics</li><li>Geriatric medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Expression analysis</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mippie" target="_blank">MIPPIE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The mouse integrated protein-protein interaction reference.<br>» Mouse Integrated Protein-Protein Interaction rEference.<br>Welcome to MIPPIE, a web tool to generate reliable and meaningful mouse protein-protein interaction networks.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Proteomics</li><li>Protein interaction experiment</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Network visualisation</li><li>Protein interaction network prediction</li><li>Query and retrieval</li><li>Protein interaction prediction</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/glypniro" target="_blank">GlypNirO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An automated workflow for quantitative N- and O-linked glycoproteomic data analysis.<br>A automated script for processing and combining Byonic and PD standard output.<br>Pythons 3.4+ with the following packages installed.<br>The program can be run from the location of the script using python main.py. The following parameters can be used for operating the script within the commandline.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Proteomics</li><li>Workflows</li><li>Biomarkers</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Quantification</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pid" target="_blank">Pathway Interaction Database (PID)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Biomolecular interactions and cellular processes assembled into human signaling pathways.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>pathway or network report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>898</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GUcal" target="_blank">GUcal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Expanding the capillary electrophoresis based glucose unit database of the GUcal app.<br>GUcal 1.1c - Internal / bracketing standard approach with GlycoStore database</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/coolpuppy" target="_blank">coolpup.py</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Coolpup.py is a versatile Python tool to perform pile-up analysis on Hi-C data in .cool format. coolpup.py is particularly well suited performance-wise for analysing huge numbers of potential interactions, since it loads whole chromosomes into memory one by one (or in parallel to speed it up) to extract small submatrices quickly. Having to read everything into memory makes it relatively slow for small numbers of loops, but performance doesn't decrease until you reach a huge number of interactions.</div></td>
      <td><div class='topic short click_expand'><ul><li>ChIP-seq</li><li>RNA-Seq</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ImmPort" target="_blank">ImmPort</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Data repository with analysis tools that is mainly focused on cytometry and immunology.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>97</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pharmgkb" target="_blank">PharmGKB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database on the impact of human genetic variations on drug response. PharmGKB curates primary genotype and phenotype data, annotate gene variants and gene-drug-disease relationships via literature review, and summarize important PGx genes and drug pathways.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pharmacogenomics</li><li>Genotype and phenotype</li><li>Pharmacology</li><li>Human biology</li><li>Rare diseases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>878</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-NC-SA-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/europe_pmc" target="_blank">Europe PMC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Archive of life sciences journal literature.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rtracklayer" target="_blank">rtracklayer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Extensible framework for interacting with multiple genome browsers (currently UCSC built-in) and manipulating annotation tracks in various formats (currently GFF, BED, bedGraph, BED15, WIG, BigWig and 2bit built-in). The user may export/import tracks to/from the supported browsers, as well as query and modify the browser state, such as the current viewport.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular biology</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>annotation track : wig</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>annotation track : wig</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>189</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scgcn" target="_blank">scGCN</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Graph Convolutional Networks Algorithm for Knowledge Transfer in Single Cell Omics.<br>single cell graph convolutional network (scGCN).<br>This is a TensorFlow implementation of scGCN for leveraging differnt single cell datasets, which is described in our paper:.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Literature and language</li><li>Cell biology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genefarm" target="_blank">GeneFarm</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Structural and functionnal annotation of plant gene and protein families by a network of experts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/UniprotR" target="_blank">UniprotR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Retrieving and visualizing protein sequence and functional information from Universal Protein Resource (UniProt knowledgebase).<br>Retrieving Information of Proteins from Uniprot.<br>Connect to Uniprot <https://www.uniprot.org/> to retrieve information about proteins using their accession number such information could be name or taxonomy information.<br>UniprotR: Retrieving and visualizing protein sequence and functional information from Universal Protein Resource (UniProtknowledgebase).<br>M. Soudy, A. Anwar, E.A. Ahmed, et al., UniprotR: Retrieving and visualizing protein sequence and functional information from Universal Protein Resource (UniProt knowledgebase), Journal of Proteomics (2019), https://doi.org/10.1016/j.jprot.2019.103613</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Proteins</li><li>Taxonomy</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Data retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/Proteomicslab57357/UniprotR" target="_blank">https://github.com/Proteomicslab57357/UniprotR</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ModMine" target="_blank">ModMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A data warehouse for the modENCODE project</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PlanMine" target="_blank">PlanMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated web resource of data & tools to mine Planarian biology</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pgd" target="_blank">The Protein Geometry Database (PGD)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This tool enables biologists to easily and flexibly query information about the conformation alone, the backbone geometry alone, and the relationships between them. It is valuable for assessing the uniqueness of observed conformational or geometric features in protein structure as well as discovering novel features and principles of protein structure.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Protein structure analysis</li><li>Protein structural motifs and surfaces</li><li>Structure prediction</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structural motif recognition</li><li>Protein geometry calculation</li><li>Protein structure prediction</li><li>Protein modelling (backbone)</li><li>Protein geometry validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>30</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/Karplus-Lab-OSU/pgd" target="_blank">https://github.com/Karplus-Lab-OSU/pgd</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metagen" target="_blank">METAGEN</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Metagen performs meta-analysis using fixed- and random-effects logistic regression models, treating the multiple genotypes as independent variables</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>DNA polymorphism</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Statistical calculation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li><li>Command-line tool</li><li>Plug-in</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/imgt_3dstructure" target="_blank">IMGT 3Dstructure-DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/3Dstructure-DB is the IMGT® database for 3D structures of immunoglobulins (IG) or antibodies, T cell receptors (TR), major histocompatibility (MH) proteins, related proteins of the immune system (RPI) and fusion proteins for immune applications (FPIA).<br>Annotation is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li><li>Immunoinformatics</li><li>Model organisms</li><li>Immunogenetics</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein structural motif recognition</li><li>Structure visualisation</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>pdb id : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>453</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/virus-ckb" target="_blank">Virus-CKB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated bioinformatics platform and analysis resource for COVID-19 research.<br>A generic knowledgebase computation platform integrated with our established chemogenomics tools as well as our algorithms for data visualization and analyses.<br>Virus-CKB is a knowledgebase for COVID-19 and similar viral infection research that implemented with our established chemogenomics tools as well as our algorithms for data visualization and analyses. This tool predicts the BioActivities on 92 viral related targets for a query compound and provides a handy user interface for viewing, downloading and plotting the output results.</div></td>
      <td><div class='topic short click_expand'><ul><li>Respiratory medicine</li><li>Pharmacology</li><li>Preclinical and clinical studies</li><li>Pathology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Subcellular localisation prediction</li><li>Virtual screening</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://www.cbligand.org/g/virus-ckb/job/list" target="_blank">https://www.cbligand.org/g/virus-ckb/job/list</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MeLAD" target="_blank">MeLAD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated resource for metalloenzyme-ligand associations | The Metalloenzyme-Ligand Association Database (MeLAD)</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metal-binding site prediction</li><li>Enrichment analysis</li><li>Protein-ligand docking</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mygene" target="_blank">mygene</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package provides simple-to-use REST web services to query/retrieve gene annotation data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene symbol : matrix format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>ensembl gene id : matrix format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bridgedb" target="_blank">BridgeDb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BridgeDb is a framework to map identifiers between various databases.  It includes a Java library that provides an API to work with identifier-identifier mapping databases and resources.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Rare diseases</li><li>Genomics</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>61</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/araqtl" target="_blank">AraQTL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>AraQTL is the A.thaliana workbench and database for eQTL analysis.<br>It allows easy access to the data of all published Arabidopsis thaliana genetical genomics experiments.</div></td>
      <td><div class='topic short click_expand'><ul><li>GWAS study</li><li>Biomarkers</li><li>Genotype and phenotype</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression QTL analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/BioGRID_Open_Repository_of_CRISPR_Screens_ORCS" target="_blank">BioGRID Open Repository of CRISPR Screens (ORCS)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BioGRID ORCS is an open repository of CRISPR screens compiled through comprehensive curation efforts. Our current index searches more than 49 publications and 58,161 genes to return more than 895 CRISPR screens from 3 major model organism species and 629 cell lines.All screen data are freely provided through our search index and available via download in a wide variety of standardized formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein function prediction</li><li>Pathway or network prediction</li><li>Gene regulatory network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>249</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Phen2Gene" target="_blank">Phen2Gene</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Phen2Gene is a phenotype-driven gene prioritization tool, that takes HPO (Human Phenotype Ontology) IDs as inputs, searches and prioritizes candidate causal disease genes. It is distributed under the MIT License by Wang Genomics Lab. Additionally, we have provided a web server and an associated RESTful API service for running Phen2Gene. Finally, a mobile app for Phen2Gene and several other genetic diagnostic tools from our lab is being tested and will be available soon.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Gene transcripts</li><li>Ontology and terminology</li><li>Rare diseases</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant prioritisation</li><li>Data retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/WGLab/Phen2Gene" target="_blank">https://github.com/WGLab/Phen2Gene</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/prosplicer" target="_blank">ProSplicer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database which stores alternative splicing information generated from EST, mRNA and protein alignments with genomic sequence; text-based queries and graphical views of putative splice variants.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein variants</li><li>Proteins</li><li>Gene transcripts</li><li>RNA splicing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence merging</li><li>Splicing model analysis</li><li>EST assembly</li><li>Sequence tag mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>32</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/msgraph" target="_blank">MSGraph</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An MS-DOS program to visualize LC-MS data in 2D and 3D (converter from mzXML available).</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Chromatogram visualisation</li><li>Mass spectrum visualisation</li><li>Plotting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : mass spectrometry data format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>plot : eps</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rnaeditingatlas" target="_blank">RNAeditingAtlas</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An Atlas of RNA editing events in human tissues.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/neptune-med" target="_blank">Neptune-med</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An environment for the delivery of genomic medicine.<br>Genomic medicine holds great promise for improving healthcare, but integrating searchable and actionable genetic data into electronic health records remains a challenge. Here, we describe Neptune, a system for managing the interaction between a clinical laboratory and an electronic health record system. <h4>Methods< h4> We developed Neptune and applied it to two clinical sequencing projects that required report customization, variant reanalysis and EHR integration. <h4>Results< h4> Neptune enabled the analysis of data for generation of and delivery to EHR systems of over 15,000 clinical genomic reports. These projects demanded customizable clinical reports that contained a variety of genetic data types including SNVs, CNVs, pharmacogenomics and polygenic risk scores</div></td>
      <td><div class='topic short click_expand'><ul><li>Pharmacogenomics</li><li>Copy number variation</li><li>Workflows</li><li>Medicine</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Variant calling</li><li>Variant prioritisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ageguess" target="_blank">AgeGuess</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>AgeGuess is a simple online game in which you can post your photos, have other people guess your age, as well as guess the age of other users.</div></td>
      <td><div class='topic short click_expand'><ul><li>Geriatric medicine</li><li>Machine learning</li><li>Biomarkers</li><li>Literature and language</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="http://www.facebook.com/ageguess.org/" target="_blank">http://www.facebook.com/ageguess.org/</a></div></td>
      <td><div class='short click_expand'>CC-BY-NC-SA-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mfpaq" target="_blank">MFPaQ</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mascot file parsing and quantitation using ICAT and SILAC.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Validation of peptide-spectrum matches</li><li>Differential protein expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : mascot .dat file</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : xml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>76</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/airlab" target="_blank">AirLab</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Cloud-based platform to manage and share antibody-based single-cell research.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunoproteins, genes and antigens</li><li>Cell biology</li><li>Data acquisition</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/leproma" target="_blank">Mycobacterium leprae genome database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mycobacterium leprae genome database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gviz" target="_blank">Gviz</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomic data analyses requires integrated visualization of known genomic information and new experimental data. This package uses the biomaRt and the rtracklayer packages to perform live annotation queries to Ensembl and UCSC and translates this to e.g. gene/transcript structures in viewports of the grid graphics package. This results in genomic information plotted together with your data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pathview" target="_blank">pathview</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool set for pathway based data integration and visualization that maps and renders a wide variety of biological data on relevant pathway graphs. It downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, it integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Data visualisation</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network analysis</li><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>555</div></td>
      <td><div class="link short click_expand"><a href="http://pathview.uncc.edu/" target="_blank">http://pathview.uncc.edu/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/eccparacorp" target="_blank">ECCParaCorp</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>English-Chinese Cancer Parallel Corpus (ECParaCorp) is a database involving information of 6 cancers in three different themes: cancer prevention, cancer screening and cancer treatment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Machine learning</li><li>Medical informatics</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tbl2knowngene" target="_blank">Tbl2KnownGene</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A wrapper tool that parses the contents of a NCBI .tbl annotation table and produces a UCSC Known Genes annotation table.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Literature and language</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/tbl2knowngene-convert-ncbi-tbl-to-ucsc-knowngene-txt-data-file.html" target="_blank">http://www.mybiosoftware.com/tbl2knowngene-convert-ncbi-tbl-to-ucsc-knowngene-txt-data-file.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/maizegdb" target="_blank">Maize genetics and genomics database (MaizeGDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Biological information about the crop plant Zea mays ssp. mays. The following datatypes are accessible through this site: genetic, genomic, sequence, gene product, functional characterization, literature reference, and person/organization contact information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene transcript report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>35</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cyp11b1" target="_blank">CYP11B1</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CYP11B1 mutation browser.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/seqtk" target="_blank">seqtk</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A tool for processing sequences in the FASTA or FASTQ format. It parses both FASTA and FASTQ files which can also be optionally compressed by gzip.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Sequence file editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GMrepo" target="_blank">GMrepo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of curated and consistently annotated human gut metagenomes | GMrepo (data repository for Gut Microbiota) is a database of curated and consistently annotated human gut metagenomes. Its main purpose is to facilitate the reusability and accessibility of the rapidly growing human metagenomic data. This is achieved by consistently annotating the microbial contents of collected samples using state-of-art toolsets and by manual curation of the meta-data of the corresponding human hosts. GMrepo organizes the collected samples according to their associated phenotypes and includes all possible related meta-data such as age, sex, country, body-mass-index (BMI) and recent antibiotics usage. To make relevant information easier to access, GMrepo is equipped with a graphical query builder, enabling users to make customized, complex and biologically relevant queries</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomics</li><li>Genotype and phenotype</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/LIBR_MOCD" target="_blank">LIBR MOCD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>LIBR Methamphetamine and Opioid Cue Database (LIBR MOCD) | Repository for the LIBR Meth Opioid Cue Reactivity Database | This project contains two folders:</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical imaging</li><li>Database management</li><li>MRI</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bart" target="_blank">BART</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Transcription factor prediction tool with query gene sets or epigenomic profiles.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li><li>Sequence analysis</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Transcriptional regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/predictprotein_open" target="_blank">PredictProtein Open</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Prediction of various aspects of protein structure and function. A user may submit a query to the server without registration.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li><li>Proteins</li><li>Data mining</li><li>Machine learning</li><li>Ontology and terminology</li><li>Sequence analysis</li><li>Function analysis</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence composition calculation</li><li>Protein signal peptide detection</li><li>Protein subcellular localisation prediction</li><li>Protein hydropathy calculation</li><li>Protein-protein interaction prediction</li><li>Taxonomic classification</li><li>Sequence profile generation</li><li>Ontology visualisation</li><li>Heat map generation</li><li>Protein property calculation</li><li>Protein function prediction</li><li>Residue interaction prediction</li><li>Transmembrane protein visualisation</li><li>Transmembrane protein prediction</li><li>Sequence motif discovery</li><li>Protein feature detection</li><li>Protein modelling (mutation)</li><li>Sequence annotation</li><li>Data retrieval</li><li>Sequence alignment</li><li>Literature search</li><li>Variant classification</li><li>Protein sequence analysis</li><li>Protein secondary structure prediction</li><li>Protein structural motif recognition</li><li>Helical wheel drawing</li><li>Sequence visualisation</li><li>Protein binding site prediction</li><li>Text mining</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein sequence : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1,652</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/picard_reordersam" target="_blank">picard_reordersam</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Reorder SAM/BAM to match contig ordering in a particular reference file. Note that this is not the same as sorting as done by the SortSam tool, which sorts by either coordinate values or query name. The ordering in ReorderSam is based on exact name matching of contigs/chromosomes. Reads that are mapped to a contig that is not in the new reference file are not included in the output.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence trace : bam</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>934</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/picard_pasteur_wrapper/picard_ReorderSam/1.56.0" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/picard_pasteur_wrapper/picard_ReorderSam/1.56.0</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bloutminer" target="_blank">bloutminer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Aa script for mining data from BLAST reports. Allows you to specify e.g. an E-value cutoff, a length cutoff and a percent identity cutoff, and extract a list of the hits satisfying these cutoffs.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>Data acquisition</li><li>Sequences</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/bloutminer-0-9-5-script-mining-data-blast-reports.html" target="_blank">http://www.mybiosoftware.com/bloutminer-0-9-5-script-mining-data-blast-reports.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PADS_Arsenal" target="_blank">PADS Arsenal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of prokaryotic defense systems related genes.<br>Procaryotic Antiviral Defense System.<br>procaryotic database, defense system database, pangenome database.<br>PADS Arsenal A Database of Prokaryotic Defense Systems Related Genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biochemistry</li><li>DNA</li><li>Gene and protein families</li><li>Model organisms</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Data retrieval</li><li>Operon prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PathwayCommons_web_service_API" target="_blank">PathwayCommons web service API</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Pathway Commons integrates a number of pathway and molecular interaction databases supporting BioPAX and PSI-MI formats into one large BioPAX model, which can be queried using our web API (documented below). This API can be used by computational biologists to download custom subsets of pathway data for analysis, or can be used to incorporate powerful biological pathway and network information retrieval and query functionality into websites and software.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Computational biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>uri : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>pathway or network : rdf/xml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>661</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/catdb" target="_blank">Complete Arabidopsis transcriptome database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This database contains data on 105 Arabidopsis transcriptome projects (3262 hybridized samples) in public access out of a total of 172 projects (~6440 hybridized samples).</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>125</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/keggparser" target="_blank">KEGGParser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MATLAB based tool for KEGG pathway parsing, semiautomatic fixing, editing, visualization and analysis in MATLAB environment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>19</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/keggparser-20130225-parsing-editing-kegg-pathway-maps-matlab.html" target="_blank">http://www.mybiosoftware.com/keggparser-20130225-parsing-editing-kegg-pathway-maps-matlab.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/STing" target="_blank">STing</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Accurate and ultrafast genomic profiling with exact sequence matches.<br>Ultrafast sequence typing and gene detection from NGS raw reads.<br>See the full documentation at: https://sting.readthedocs.io/</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>Public health and epidemiology</li><li>Database management</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Multilocus sequence typing</li><li>k-mer counting</li><li>Genome indexing</li><li>Antimicrobial resistance prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ChlamDB" target="_blank">ChlamDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comparative genomics database of the phylum Chlamydiae and other members of the Planctomycetes-Verrucomicrobiae-Chlamydiae superphylum.<br>ChlamDB documentation — ChlamDB 2.0 (June 2019) documentation.<br>A comparative genomics database of the phylum Chlamydiae and other PVC (Planctomycetes, Verrucomicrobia, Lentisphaerae).<br>Problems and errors encountered on the ChlamDB website can be announced on the github page of the website</div></td>
      <td><div class='topic short click_expand'><ul><li>Comparative genomics</li><li>Database management</li><li>Model organisms</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Database search</li><li>Fold recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/metagenlab/chlamdb" target="_blank">https://github.com/metagenlab/chlamdb</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rtcgatoolbox" target="_blank">RTCGAToolbox</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Managing data from large scale projects such as The Cancer Genome Atlas (TCGA) for further analysis is an important and time consuming step for research projects. Firehose project makes TCGA pre-processed data publicly available via web services and data portals but it requires managing, downloading and preparing the data for following steps. We developed an open source and extensible R based data client for Firehose pre-processed data and demonstrated its use with sample case studies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>86</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/enzymine" target="_blank">EnzyMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>EnzyMine:Addition of chemical structural information in enzymatic reactions has proven to be significant for accurate enzyme function prediction. a comprehensive database for enzyme function annotation with enzymatic reaction chemical feature.<br>.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biochemistry</li><li>Enzymes</li><li>Molecular interactions, pathways and networks</li><li>Endocrinology and metabolism</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic pathway prediction</li><li>Query and retrieval</li><li>Virtual screening</li><li>Enzyme kinetics calculation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/T-psi-C" target="_blank">T-psi-C</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>User friendly database of tRNA sequences and structures.<br>Discover the best tRNA database available!.<br>There is simplified list of what you can do:.<br>Search by sequence, structure and other properties.<br>See available and generated models</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Functional, regulatory and non-coding RNA</li><li>Pathology</li><li>Database management</li><li>Molecular biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Data retrieval</li><li>Database search</li><li>tRNA gene prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mesophotic.org" target="_blank">mesophotic.org</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A repository for scientific information on mesophotic ecosystems.<br>Repository for information on mesophotic coral ecosystems.<br>Mesophotic.org: an information repository on mesophotic coral ecosystems.<br>Mesophotic.org is a curated repository for information on mesophotic coral ecosystems, with the aim to provide an independent, open-source platform for and by the ever-growing mesophotic research community.<br>Latest update: November 19, 2019.<br>You are currently looking at a beta release of the new website. Apologies for the inconvenience, but for now expect some (= lots of) bugs & mistakes!.<br>The following are instructions to download and manipulate the metadata from the "Mesophotic.org" repository in R (https: www.r-project.org ).<br>The following graphs summarize the metadata across the 689 scientific articles in this database that are validated and present original data from mesophotic depths.<br>Behind the science: Echinoderm diversity with new genera described in Rapa-Nui (Easter Island)</div></td>
      <td><div class='topic short click_expand'><ul><li>Biodiversity</li><li>Database management</li><li>Literature and language</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="http://mesophotic.org/metadata" target="_blank">http://mesophotic.org/metadata</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/FASTA_EBI" target="_blank">FASTA (EBI)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sequence similarity searching against protein databases. FASTX and FASTY translate a DNA query. Optimal searches with SSEARCH (local), GGSEARCH (global) and GLSEARCH (global query, local database).</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence comparison</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>419</div></td>
      <td><div class="link short click_expand"><a href="http://www.ebi.ac.uk/support/" target="_blank">http://www.ebi.ac.uk/support/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phytebyte" target="_blank">PhyteByte</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Identification of foods containing compounds with specific pharmacological properties.<br>Phytebyte is an extensible software framework used to train machine learning models to identify bioactive compounds found in food. The use case captured in 'run.py' is training models using known drug compounds that target a specific gene, and identifying plant compounds in FooDB that may behave similarly.</div></td>
      <td><div class='topic short click_expand'><ul><li>Public health and epidemiology</li><li>Compound libraries and screening</li><li>Pharmacology</li><li>Nutritional science</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein fragment weight comparison</li><li>Subcellular localisation prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/g-root" target="_blank">G-RooT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Global Root Traits (GRooT) Database.<br>Ready-to-use data by combining the expertise of root ecologist with data mobilization and curation.<br>Global Root Trait (GRooT) Database.<br>Our principal objective with GRooT is to contribute towards the inclusion of root traits in large-scale comparative studies and global models by offering standardized and curated data of key root traits.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Data submission, annotation and curation</li><li>Plant biology</li><li>Ecology</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Phylogenetic analysis</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/q1d" target="_blank">Q1D</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>High-throughput design of Peierls and charge density wave phases in Q1D organometallic materials.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Biological databases</li><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phasing</li><li>Query and retrieval</li><li>Visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bceconsensus" target="_blank">BCEconsensus</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A B-cell epitope consensus predictor.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Sequence analysis</li><li>Biophysics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Peptide immunogenicity prediction</li><li>Epitope mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/compmap" target="_blank">CompMap</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CompMap is an allele-specific expression read-counter based on competitive mapping. It compares reads mapped to two references and counts reads for allele-specific expression.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>RNA-Seq</li><li>Genotype and phenotype</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Read mapping</li><li>Parsing</li><li>Modelling and simulation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/clustomics" target="_blank">ClustOmics</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Consensus clustering applied to multi-omic disease subtyping.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Functional, regulatory and non-coding RNA</li><li>Workflows</li><li>Pathology</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Enrichment analysis</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_mAb-DB" target="_blank">IMGT mAb-DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/mAb-DB is the IMGT® database for monoclonal antibodies (mAb) or immunoglobulins (IG), fusion proteins for immune applications (FPIA) and composite proteins for clinical applications (CPCA).<br>IMGT/mAb-DB provides links to IMGT/2Dstructure-DB and IMGT/3Dstructure-DB.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunogenetics</li><li>Immunoinformatics</li><li>Allergy, clinical immunology and immunotherapeutics</li><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database search results : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>205</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MOAT" target="_blank">MOAT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Efficient detection of highly mutated regions with the Mutations Overburdening Annotations Tool.</div></td>
      <td><div class='topic short click_expand'><ul><li>Whole genome sequencing</li><li>Genetic variation</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Parsing</li><li>Read binning</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cornai" target="_blank">coRNAi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Analysis of combinatorial cell-based RNAi screens.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Molecular interactions, pathways and networks</li><li>RNAi experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li><li>Pathway or network prediction</li><li>Standardisation and normalisation</li><li>Validation</li><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>experimental measurement : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>report : image format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MIMIC-CXR" target="_blank">MIMIC-CXR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>De-identified publicly available database of chest radiographs with free-text reports.<br>If you use the MIMIC-CXR Database in your work, please cite the following publication:.<br>MIMIC-CXR: A large publicly available database of labeled chest radiographs. Johnson AEW, Pollard TJ, Berkowitz S, Greenbaum NR, Lungren MP, Deng C-Y, Mark RG, Horng S. arXiv.. Available from: https://arxiv.org</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical imaging</li><li>Database management</li><li>Tomography</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Image annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/multi-fingerprint_browser" target="_blank">Multi-fingerprint Browser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Multi-Fingerprint Browser for the ZINC Database enables one to rapidly identify close analogs among commercially available compounds in the ZINC database (> 13 million molecules).</div></td>
      <td><div class='topic short click_expand'><ul><li>Probes and primers</li><li>Compound libraries and screening</li><li>Drug discovery</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Query and retrieval</li><li>Data retrieval</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>28</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/entrez" target="_blank">Entrez</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>NCBI information retrieval system, including GenBank, MMDB (structures), genomes, population sets, OMIM, taxonomy and PubMed.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Literature and language</li><li>Data integration and warehousing</li><li>Mapping</li><li>Sequence analysis</li><li>Rare diseases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Query and retrieval</li><li>Service invocation</li><li>Mapping</li><li>Sequence analysis</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>327</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ELIMU-MDx" target="_blank">ELIMU-MDx</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Electronic Laboratory Information and Management Utensil for Molecular Diagnostics (ELIMU-MDx) is a web-based, open-source platform for storage, management and analysis of diagnostic qPCR data. A webapplication for managing, documenting and result analyzing of aPCR Data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Infectious disease</li><li>Personalised medicine</li><li>PCR experiment</li><li>Laboratory information management</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Quantification</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://elimu.ch" target="_blank">https://elimu.ch</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/drfast" target="_blank">drFAST</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Software designed to map di-base reads (SOLiD color space reads) to reference genome assemblies; in a fast and memory-efficient manner.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>Genomics</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>185</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/drfast-0-0-0-3-di-base-read-fast-alignment-search-tool.html" target="_blank">http://www.mybiosoftware.com/drfast-0-0-0-3-di-base-read-fast-alignment-search-tool.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/compluyeast2dpage" target="_blank">COMPLUYEAST-2DPAGE database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A two-dimensional polyacrilamide gel electrophoresis federated database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>experiment report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/quin" target="_blank">QuIN</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool for visualizing, annotating, and querying chromatin interactions derived from technologies such as ChIA-PET or HiC.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Zombi" target="_blank">Zombi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A phylogenetic simulator of trees, genomes and sequences that accounts for dead linages | Evolution simulator with extinct lineages | Zombi: A phylogenetic simulator of trees, genomes and sequences that accounts for dead lineages | Zombi is a flexible platform of genome evolution which can be of great interest to those who want to test different evolutionary hypotheses under simulations and need to use a fast and easy-to-use tool to generate species trees, gene trees or sequences. Zombi's output is especially simple and easy to read, understand and parse</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic species tree construction</li><li>Phylogenetic tree reconstruction</li><li>Phylogenetic tree comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genbankr" target="_blank">genbankr</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Reads Genbank files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gxa_expt" target="_blank">GXA Expt</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Semantically enriched database of meta-analysis based summary statistics over a curated subset of ArrayExpress Archive, servicing queries for condition-specific gene expression patterns as well as broader exploratory searches for biologically interesting genes/samples. This collection references experiments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene expression data : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>119</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/VariantValidator" target="_blank">VariantValidator</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>VariantValidator is a user-friendly software tool designed to validate the syntax and parameters of DNA variant descriptions according to the HGVS Sequence Variant Nomenclature.<br>VariantValidator ensures that users are guided through the intricacies of the HGVS nomenclature, e.g. if the user makes a mistake, VariantValidator automatically corrects the mistake if it can, or provides helpful guidance if it cannot. In addition, VariantValidator accurately inter-converts between transcript variant descriptions and genomic variant descriptions in HGVS and Variant Call Format (VCF)<br>VariantValidator interfaces with the hgvs package to parse, format, and manipulate biological sequence variants. See https://github.com/biocommons/hgvs/ for details of the hgvs package<br>VariantValidator is a highly functional platform enabling high-throughput and embeddable utilisation of functionality of https://variantvalidator.org/</div></td>
      <td><div class='topic short click_expand'><ul></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>33</div></td>
      <td></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PowerShell" target="_blank">PowerShell</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Using the Object-Oriented PowerShell for Simple Proteomics Data Analysis | Scripting languages such as Python and Bash are appreciated for solving simple, everyday tasks in bioinformatics</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li><li>Peptide identification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ibis_ncbi" target="_blank">IBIS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>NCBI Inferred Biomolecular Interaction Server. This server organizes, analyzes and predicts interaction partners and locations of binding sites in proteins. It provides annotations for different types of binding partners (protein, chemical, nucleic acid and peptides), and facilitates the mapping of a comprehensive biomolecular interaction network for a given protein query.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li><li>Protein binding sites</li><li>Nucleic acid sites, features and motifs</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network comparison</li><li>Pathway or network prediction</li><li>Protein binding site prediction</li><li>Protein interaction prediction</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>66</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lemarns" target="_blank">LeMaRns</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Length-based Multi-species analysis by numerical simulation in R.<br>Length-Based Multispecies Analysis by Numerical Simulation.<br>Set up, run and explore the outputs of the Length-based Multi-species model (LeMans; Hall et al. 2006 <doi:10.1139/f06-039>), focused on the marine environment.<br>LeMaRns is an R package to set up, run and explore the outputs of the Length-based Multi-species model LeMans, focused on the marine environment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Zoology</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Simulation analysis</li><li>Ecological modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/CefasRepRes/LeMaRns" target="_blank">https://github.com/CefasRepRes/LeMaRns</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/seqstrap" target="_blank">SeqStrap</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>“Extrapolating” partial or fragmentary nucleic acid sequences prior to assay design. It works by first loading a set of user specified sequences in FASTA format, iteratively comparing every sequence to every other sequence, and using the best matching sequence as a template for adding additional sequence at the 5′ and 3′ ends of the sequence.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>Sequences</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/seqstrap-1-0-extrapolating-partial-fragmentary-nucleic-acid-sequences-prior-assay-design.html" target="_blank">http://www.mybiosoftware.com/seqstrap-1-0-extrapolating-partial-fragmentary-nucleic-acid-sequences-prior-assay-design.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/expression_atlas" target="_blank">Expression Atlas</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Open science resource that gives users a powerful way to find information about gene and protein expression across species and biological conditions such as different tissues, cell types, developmental stages and diseases among others.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>318</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PhenX_Toolkit" target="_blank">PhenX Toolkit</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web catalog of high-priority measures for consideration and inclusion in biomedical and epidemiologic research.</div></td>
      <td><div class='topic short click_expand'><ul><li>GWAS study</li><li>Rare diseases</li><li>Genotype and phenotype</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Quantification</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-NC-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bioviz-connect" target="_blank">BioViz Connect</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BioViz Connect is a web application linking CyVerse science gateway resources to genomic visualization in the Integrated Genome Browser.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>RNA splicing</li><li>Gene transcripts</li><li>Plant biology</li><li>Probes and primers</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>RNA-Seq analysis</li><li>Genome visualisation</li><li>Data retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://bitbucket.org/lorainelab/bioviz-connect" target="_blank">https://bitbucket.org/lorainelab/bioviz-connect</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_StructuralQuery" target="_blank">IMGT StructuralQuery</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/StructuralQuery is the IMGT® tool to retrieve IMGT/3Dstructure-DB entries using amino acid structural criteria for variable (V), constant (C) or groove (G) domains. Annotation is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>keyword : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>structure : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>388</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ccnet" target="_blank">CCNet</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Criss-Cross Network (CCNet) is a method for obtaining full-image contextual information in a very effective and efficient way. Contextual information is vital in visual understanding problems, such as semantic segmentation and object detection.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Aggregation</li><li>Dimensionality reduction</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/DRfit" target="_blank">DRfit</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool for the analysis of discrete data from multi-well plate assays.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Drug discovery</li><li>Environmental science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/visant" target="_blank">VisANT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>VisANT is an integrative visual analysis tool for biological networks and pathways that contains modules for querying and integrating KEGG pathways with expression data. VisANT is java-based, and can be run as a java applet, as a java web application, or downloaded and run locally. New functions allow determination of over-representation and expression enrichment, integrated search and navigation of disease and drug hierarchies, drug/disease annotation, etc.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pharmacology</li><li>Molecular interactions, pathways and networks</li><li>Pathology</li><li>Endocrinology and metabolism</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network analysis</li><li>Weighted correlation network analysis</li><li>Gene expression correlation</li><li>Gene to pathway mapping</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>694</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ped-draw" target="_blank">ped_draw</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Pedigree drawing with ease.<br>A simple python script to draw (complex and multi generation!) pedigrees with graphviz.<br>ped_draw reads in a ped file (from a specified file or from /dev/stdin) and generates a dot file (to stdout). The subsequent dot output can be piped directly to dot/graphviz to generate png, svg, pdf, ps, or any other image option available in the graphviz documentation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Bioinformatics</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Map drawing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/mvelinder/ped_draw" target="_blank">https://github.com/mvelinder/ped_draw</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PhilDB" target="_blank">PhilDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PhilDB is capable of storing data at any frequency supported by Pandas. At this time only daily data has been extensively tested with some limited sub-daily usage.<br>Timeseries database project is for storing potentially changing timeseries data. For example, hydrological data, like streamflow data, where the timeseries may be revised as quality control processes improve the recorded dataset over time.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Data acquisition</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/DEPICTER" target="_blank">DEPICTER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Intrinsic disorder and disorder function prediction server.<br>Predictors of disordered regions and their molecular functions - web server from Biomine lab.<br>DEPICTER : DisorderEd PredictIon CenTER</div></td>
      <td><div class='topic short click_expand'><ul><li>Neurology</li><li>Protein disordered structure</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein function prediction</li><li>Protein disorder prediction</li><li>Parsing</li><li>Protein secondary structure prediction (coils)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/atlasgeneticsoncology" target="_blank">AGCOH</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Atlas of Genetics and Cytogenetics in Oncology and Haematology is a peer-reviewed internet journal/encyclopaedia/database focused on genes implicated in cancer, cytogenetics and clinical entities in cancer and cancer-prone hereditary diseases. The main goal of the Atlas is to provide review articles that describe complementary topics, namely, genes, genetic abnormalities, histopathology, clinical diagnoses and a large iconography.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>article : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>131</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-NC-ND-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/the_ithanet_portal" target="_blank">ITHANET</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The portal integrates information on news, events, clinical trials and thalassaemia-related organisations, research projects and other scientific networks, wiki-based content of protocols, clinical guidelines and educational articles. Most importantly, disease-specific databases are developed and maintained on the portal, such as databases of haemoglobin variations (IthaGenes), epidemiology (IthaMaps) and HbF inducers (IthaDrugs).</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular genetics</li><li>Public health and epidemiology</li><li>Computational biology</li><li>Genetics</li><li>Data visualisation</li><li>Genotype and phenotype</li><li>Genetic variation</li><li>Haematology</li><li>Biomarkers</li><li>Population genetics</li><li>Rare diseases</li><li>Database management</li><li>Molecular biology</li><li>Human genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>90</div></td>
      <td><div class="link short click_expand"><a href="https://www.ithanet.eu/home/faqs" target="_blank">https://www.ithanet.eu/home/faqs</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ribominer" target="_blank">RiboMiner</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A toolset for mining multi-dimensional features of the translatome with ribosome profiling data.<br>A python toolset for mining multi-dimensional features of the translatome with ribosome profiling data.<br>The RiboMiner is a python toolset for mining multi-dimensional features of the translatome with ribosome profiling data. This package has four function parts:.<br><br>RiboMiner could offer other useful functions that would help users analyze ribosome profiling data, such as statistics of read coverage, visualization of read density for a single transcript or some transcript sets.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Cell biology</li><li>Protein expression</li><li>Gene transcripts</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Sequence trimming</li><li>Expression analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://pypi.org/project/RiboMiner" target="_blank">https://pypi.org/project/RiboMiner</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/grape_database" target="_blank">Grape Database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database which provides several ways to access to the grape genomic data: as a blast search page, a genome browser and a direct link to download the data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>144</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/jmzreader" target="_blank">jmzReader</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Java parser library to process and visualize multiple text and XML-based mass spectrometry data formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : mzml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>26</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/PRIDE-Utilities/jmzReader" target="_blank">https://github.com/PRIDE-Utilities/jmzReader</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/DTCLab" target="_blank">DTCLab</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>How Precise Are Our Quantitative Structure-Activity Relationship Derived Predictions for New Query Chemicals? | QSPR Modeling of the Refractive Index for Diverse Polymers Using 2D Descriptors | QSAR Model Development Using DTC Lab. Software Tools | DTC lab - Software Tools - DTC lab is working in the field of molecular modelling mainly using different QSAR methodologies in various diverse area such as drug designing, toxicity, anti-oxidant studies etc | Drug Theoretics and Cheminformatics Laboratory | *Genetic Algorithm(MAE-based Fitness Function) | *Intelligent Consensus Predictor</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li><li>Medicinal chemistry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Quantification</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>47</div></td>
      <td><div class="link short click_expand"><a href="http://dtclab.webs.com/software-tools" target="_blank">http://dtclab.webs.com/software-tools</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/citexplore" target="_blank">CiteXplore literature database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Combine literature search with text mining tools for biology.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>bibliography : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>667</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Ordino" target="_blank">Ordino</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Visual cancer analysis tool for ranking and exploring genes, cell lines and tissue samples.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Oncology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Aggregation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/Caleydo/ordino" target="_blank">https://github.com/Caleydo/ordino</a></div></td>
      <td><div class='short click_expand'>MPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pickle" target="_blank">PICKLE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PICKLE (Protein InteraCtion KnowLedgebasE) is a meta-database for the direct protein-protein interactome of the human and the mouse proteomes, integrating publicly available source protein-protein interaction (PPI) databases via genetic information ontology. PICKLE integrates the primary PPI datasets by superimposing them on the UniProtKB/Swiss-Prot reviewed complete proteome of the human (RHCP) and the mouse (RMCP) ontology network without any a priori transformations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Protein interactions</li><li>Human biology</li><li>Systems medicine</li><li>Human genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Protein interaction network analysis</li><li>Query and retrieval</li><li>Pathway or network visualisation</li><li>Sorting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>33</div></td>
      <td><div class="link short click_expand"><a href="http://www.pickle.gr" target="_blank">http://www.pickle.gr</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dbmts" target="_blank">dbMTS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DbMTS is a comprehensive database of putative human microRNA target site (MTS) SNVs and their functional predictions. dbMTS collects all potential SNVs microRNA target seed regions in human 3’UTRs and provides their functional predictions and annotations to facilitate the steps of filtering and prioritizing SNVs from a huge list of all SNVs discovered in a whole exome sequencing (WES) study.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li><li>Functional, regulatory and non-coding RNA</li><li>Gene expression</li><li>Gene transcripts</li><li>Exome sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>miRNA target prediction</li><li>miRNA expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fhitings" target="_blank">FHiTINGS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>"FHiTINGS is designed for use in rapidly identifying, classifying, and parsing internal transcribed spacer (ITS) DNA sequences after a BLASTn search. This software is useful for fungal ecology studies using next generation sequencing (NGS)."</div></td>
      <td><div class='topic short click_expand'><ul><li>Comparative genomics</li><li>Metagenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>47</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iodata" target="_blank">IOData</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A python library for reading, writing, and converting computational chemistry file formats and generating input files.<br>IOData is a HORTON 3 module for input/output of quantum chemistry file formats. Documentation is here: https://iodata.readthedocs.io/en/latest/index.html.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li><li>Chemistry</li><li>Molecular dynamics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li><li>Molecular dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>LGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dssp" target="_blank">DSSP secondary structure database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Secondary structure assignments (and much more) for all protein entries in the Protein Data Bank (PDB).</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Protein secondary structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein features : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>347</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/seqtypedetect" target="_blank">seqtypedetect</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Detect sequence datatype on fasta file.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>937</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/oneclick_phylogeny/seqtypedetect/1.0" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/oneclick_phylogeny/seqtypedetect/1.0</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metabolights" target="_blank">MetaboLights</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database for Metabolomics experiments and derived information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>414</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/LAVA" target="_blank">LAVA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A streamlined visualization tool for longitudinal analysis of viral alleles.<br>LAVA: Longitudinal Analysis of Viral Alleles.<br>LAVA analyzes and visualizes minor allele variants in longitudinal sequence data. LAVA takes a reference fasta (normally representing the first sample in your longitudinal analysis), fastq files (for every sample in your analysis), and a metadata sheet (providing information on what day or passage each sample was collected). Output will be displayed as an interactive graph in your web browser. LAVA will only work on Mac and Linux machines</div></td>
      <td><div class='topic short click_expand'><ul><li>Virology</li><li>Genetic variation</li><li>Exome sequencing</li><li>Transcription factors and regulatory sites</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Variant calling</li><li>Sequence trimming</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PCAViz" target="_blank">PCAViz</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An Open-Source Python/JavaScript Toolkit for Visualizing Molecular Dynamics Simulations in the Web Browser.<br>PCAViz is an open-source Python/JavaScript toolkit for sharing and visualizing molecular dynamics trajectories via a web browser.<br>PCAViz is an open-source Python/JavaScript toolkit for sharing and visualizing MD trajectories via a web browser. To encourage use, an easy-to-install PCAViz-powered WordPress plugin enables 'plug-and-play' trajectory visualization.<br>Jacob Durrant software July 29, 2019September 30, 2019</div></td>
      <td><div class='topic short click_expand'><ul><li>X-ray diffraction</li><li>Molecular interactions, pathways and networks</li><li>Molecular dynamics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Trajectory visualization</li><li>Essential dynamics</li><li>Principal component visualisation</li><li>Molecular dynamics</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="http://git.durrantlab.com/jdurrant/pcaviz" target="_blank">http://git.durrantlab.com/jdurrant/pcaviz</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/halodom" target="_blank">HaloDom</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>HaloDom is a database of halophilic organisms across all life domains. It provides taxonomy info, halotolerance classification, salinity metadata, genomic/proteomic links and publication info.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biological databases</li><li>Computational biology</li><li>Comparative genomics</li><li>Metagenomics</li><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gonuts" target="_blank">GONUTS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Third-party documentation for new users of GO.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>19</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PAFScaff" target="_blank">PAFScaff</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PAFScaff (Pairwise mApping Format reference-based scaffold anchoring and super-scaffolding) is designed for mapping genome assembly scaffolds to a closely-related chromosome-level reference genome assembly. It uses (or runs) Minimap2 to perform an efficient (if rough) all- against-all mapping, then parses the output to assign assembly scaffolds to reference chromosomes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Mapping assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence assembly : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence assembly : fasta</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/slimsuite/pafscaff" target="_blank">https://github.com/slimsuite/pafscaff</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pir" target="_blank">PIR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrated protein informatics resource for genomic, proteomic and systems biology research.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence record : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>328</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/vesiclepedia" target="_blank">Vesiclepedia</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Manually curated compendium of molecular data (lipid, RNA and protein) identified in different classes of extracellular vesicles.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Functional, regulatory and non-coding RNA</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>54</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/BlobToolKit" target="_blank">BlobToolKit</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BlobToolKit Viewer is a genome-scale dataset visualistion tool developed as part of the blobtoolkit project to allow browser-based identification and filtering of target and non-target data in genome assemblies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Workflows</li><li>Phylogenomics</li><li>Model organisms</li><li>Sequence assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome assembly</li><li>Visualisation</li><li>Parsing</li><li>Deposition</li><li>Scaffolding</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/blobtoolkit" target="_blank">https://github.com/blobtoolkit</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gene_expression_atlas" target="_blank">Gene Expression Atlas</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Semantically enriched database of meta-analysis based summary statistics over a curated subset of ArrayExpress Archive, servicing queries for condition-specific gene expression patterns as well as broader exploratory searches for biologically interesting genes/samples.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Experimental design and studies</li><li>Functional, regulatory and non-coding RNA</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression analysis</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>119</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/VOLARE" target="_blank">VOLARE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Visual analysis of disease-associated microbiome-immune system interplay | Visualization of Linear Regression Elements | Visualization of linear regression elements comparing microbiome and immune system readouts</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Microbial ecology</li><li>Allergy, clinical immunology and immunotherapeutics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Regression analysis</li><li>Enrichment analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/btod_blast" target="_blank">BTOD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An algorithm for comparing primary biological sequence information, such as the amino-acid sequences of proteins or the nucleotides of DNA sequences. A BLAST search enables a researcher to compare a query sequence with a library or database of sequences, and identify library sequences that resemble the query sequence above a certain threshold.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Proteins</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Local alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment (pair) : blast xml results format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/disprot" target="_blank">Database of protein disorder (DisProt)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of protein disorder. It is a community resource annotating protein sequences for intrinsically disordered regions from the literature.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein disordered structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein structure report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>18</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-NC-1.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Mascot_Parser" target="_blank">Mascot Parser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mascot Parser provides an object-oriented Application Programmer Interface (API) to Mascot result files and configuration files, making it easy for programs written in C++, C#, Java, Perl or Python to access Mascot results.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database search results : mascot .dat file</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Proprietary</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/omicsdi" target="_blank">omicsDI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides dataset discovery across a heterogeneous, distributed group of Transcriptomics, Genomics, Proteomics and Metabolomics data resources.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Genomics</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/harp" target="_blank">HARP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of structural impacts of systematic missense mutations in drug targets of <i>Mycobacterium leprae</i>.<br>Last updated on September 22, 2020.<br>HARP - A Web Resource for Predicted Structural Impacts of Anti-microbial Resistance Mutations in Leprosy.<br>HARP (a database of Hansen's Disease Antimicrobial Resistance Profiles) is an integrated web-resource with predicted structural implications of systematic mis-sense mutations in known drug targets of Mycobacterium leprae.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein folding, stability and design</li><li>Nucleic acid structure analysis</li><li>Genetic variation</li><li>Molecular modelling</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant effect prediction</li><li>Nucleic acid folding analysis</li><li>Query and retrieval</li><li>Molecular docking</li><li>Antimicrobial resistance prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/affyio" target="_blank">affyio</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Routines for parsing Affymetrix data files based upon file format information. Primary focus is on accessing the CEL and CDF file formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Data management</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bmbolstad/affyio" target="_blank">https://github.com/bmbolstad/affyio</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/invacost" target="_blank">invacost</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Analysing global economic costs of invasive alien species with the invacost R package.<br>The invacost R Package: Global Costs of Biological Invasions.<br>Leroy B, Kramer A, Courchamp C, Vaissière AC & Diagne C 03/12/2020.<br>This approach is generalisable to any subset of the InvaCost database, regardless of data quantity. However, it does not account for the temporal dynamics of costs, and may thus underestimate the costs of invasions, especially for recent years for which we likely have incomplete data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ecology</li><li>Agricultural science</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Essential dynamics</li><li>Query and retrieval</li><li>Regression analysis</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PathwayCommons_SPARQL_endpoint" target="_blank">PathwayCommons SPARQL endpoint</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Pathway Commons aims to collect and disseminate biological pathway and interaction data. Data is collected from partner databases and is represented in the BioPAX standard. By representing data in BioPAX, Pathway Commons is able to provide a detailed representation of a variety of biological concepts including: Biochemical reactions; gene regulatory networks; and genetic interactions; transport and catalysis events; and physical interactions involving proteins, DNA, RNA and small molecules and complexes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Computational biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>query script : sparql</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>pathway or network : rdf format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>SPARQL endpoint</li></ul></div></td>
      <td><div class='citations short click_expand'>661</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/michelanglo" target="_blank">MichelaNglo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sculpting protein views on web pages without coding.<br>Michelaɴɢʟo is a web app to convert a PyMOL PSE file or PDB file to a easy to implement NGL.js view that can be implemented easily on any site.<br>Convert PyMOL files, upload PDB files or submit PDB codes and create a webpage to edit, share or implement standalone on your site.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Editing</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/thesgc/MichelaNGLo" target="_blank">https://github.com/thesgc/MichelaNGLo</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/drtext" target="_blank">drtext</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Get data resource entries complete text.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/legiolist" target="_blank">Legionella pneumophila genome database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Legionella pneumophila genome database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>165</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/unite_rdna" target="_blank">UNITE rDNA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>RDNA sequence database focused on ectomycorrhizal asco- and basidiomycetes.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,615</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/amoebadb" target="_blank">AmoebaDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>It is a new functional genomic databases serving the amoebozoa research communities. It contains the genomes of three Entamoeba species (E. dispar, E. invadens and E. histolityca) and microarray expression data for E. histolytica. Importantly it utilizes the graphical strategy builder that affords a database user the ability to ask complex multi-data-type questions with relative ease and versatility.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Phylogenetics</li><li>Gene transcripts</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>61</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cellosaurus" target="_blank">Cellosaurus</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A knowledge resource on cell lines. It attempts to describe all cell lines used in biomedical research, including immortalized cell lines, naturally immortal cell lines (ie stem cell lines), finite life cell lines when those are distributed and widely used, vertebrate cell lines with an emphasis on human, mouse and rat cell lines, and invertebrate (insect and tick) cell lines.</div></td>
      <td><div class='topic short click_expand'><ul><li>Anatomy</li><li>Cell biology</li><li>Gene expression</li><li>Systems biology</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tfextract" target="_blank">tfextract</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Process TRANSFAC transcription factor database for use by tfscan.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dcmqi" target="_blank">DCMQI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DCMQI (DICOM for Quantitative Imaging) is a free, open source library that can help with the conversion between imaging research formats and the standard DICOM representation for image analysis results.</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical imaging</li><li>Biomarkers</li><li>MRI</li><li>Tomography</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Deposition</li><li>Quantification</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>31</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/QIICR/dcmqi/issues" target="_blank">https://github.com/QIICR/dcmqi/issues</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/map4" target="_blank">MAP4</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>One Molecular Fingerprint to Rule them All.<br>Extended-Benchmark: compounds and query lists used for the peptide benchmark.<br>MAP4-Similarity-Search: source code for the similarity search app.<br>Supplementary Material for the manuscript: "One Molecular Fingerprint to Rule them All: Drugs, Biomolecules, and the Metabolome".<br>TMAPs of the ChEMBL, non-Lipinski ChEMBL, Human Metabolome, SwissProt up to 50 residues sequences, and Drugbank MAP4 Chemical Space.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li><li>Cheminformatics</li><li>Molecular biology</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Treemap visualisation</li><li>Virtual screening</li><li>Essential dynamics</li><li>Database search</li><li>Protein fragment weight comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/reymond-group/map4" target="_blank">https://github.com/reymond-group/map4</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ensembl_genomes" target="_blank">Ensembl Genomes</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A system for the annotation, analysis and display of vertebrate genomes and for bacteria, protists, fungi, plants and invertebrate metazoa.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>331</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/googlegenomics" target="_blank">GoogleGenomics</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An R package to interact with the homonimous API.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td><div class="link short click_expand"><a href="https://cloud.google.com/genomics/" target="_blank">https://cloud.google.com/genomics/</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/give" target="_blank">GIVE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Open source programming library that allows anyone without much programming experience to build custom genome browser websites or apps. With a few lines of codes, one can set up GIVE service on a local machine, manage custom data and build an interactive genome browser.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li><li>Data management</li><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ramp" target="_blank">RaMP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Comprehensive Relational Database of Metabolomics Pathways for Pathway Enrichment Analysis of Genes and Metabolites.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li><li>Data management</li><li>Metabolic pathways</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bioparser" target="_blank">BioParser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Perl program for parsing sequence similarity analysis reports.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Nucleic acid structure analysis</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence alignment comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/bioparser-1-2-3-parsing-sequence-similarity-analysis-reports.html" target="_blank">http://www.mybiosoftware.com/bioparser-1-2-3-parsing-sequence-similarity-analysis-reports.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/enzymeportal_protein_sequence_search" target="_blank">Enzyme Portal - Protein sequence search</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Advanced search of the EnzymePortal.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td><div class="link short click_expand"><a href="http://www.ebi.ac.uk/support/" target="_blank">http://www.ebi.ac.uk/support/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PlotTwist" target="_blank">PlotTwist</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PlotTwist (Plotting data from Time-lapse experiments With Indicators of conditions at Set Times) is an app for visualizing the data and statistics of continuous data, with a focus on time-dependent measurements. The philosophy of the approach is that plotting the raw data (instead of a summary) improves transparency and interpretation. To facilitate the visualization of individual measurements, the data can be shown as 'small mutliples' or in 'heatmap-style'. To further facilitate the comparison, the mean and 95% CI can be added. The user has full control over the visibility of the raw data and statistics by adjustment of the transparency (alpha). Several color blind friendly palettes are available to label the data. The comparison of multiple conditions is enabled by multiple file upload (csv or xls format), where each file represents a condition.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Neurobiology</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Read binning</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://goedhart.shinyapps.io/PlotTwist/" target="_blank">https://goedhart.shinyapps.io/PlotTwist/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/RAEdb" target="_blank">RAEdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of enhancers identified by high-throughput reporter assays.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li><li>Gene transcripts</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Enrichment analysis</li><li>Transcriptional regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/AlvisIR" target="_blank">AlvisIR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Text mining tool for extracting information about microbial biodiversity in food.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biodiversity</li><li>Natural language processing</li><li>Microbiology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Text mining</li><li>Read binning</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/facta_plus" target="_blank">FACTA plus</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Real-time text-mining system for finding and visualizing indirect associations between biomedical concepts from MEDLINE abstracts. The system can be used as a text search engine like PubMed, with additional features to help users to discover and visualize indirect associations between important biomedical concepts such as genes, diseases and chemical compounds.</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical informatics</li><li>Machine learning</li><li>Pathology</li><li>Natural language processing</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Text mining</li><li>Named-entity and concept recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>194</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iACetyP" target="_blank">iACetyP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Identifying Acetylation Protein by Fusing Its PseAAC and Functional Domain Annotation.<br>iAcetyP : Identify acetylation protein via Chou's 5-steps rule and general pseudo components : Read Me : Supporting Information : Citation :.<br>Enter the sequences of query proteins in FASTA format (Example), and select Submit button for identifying acetylation protein. The number of proteins is limited at 5 or less for each submission</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Protein modifications</li><li>Informatics</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Post-translation modification site prediction</li><li>Subcellular localisation prediction</li><li>PTM localisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/quickgo" target="_blank">QuickGO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A fast browser for Gene Ontology terms and annotations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Ontology visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/epivizr" target="_blank">epivizr</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package provides connections to the epiviz web app for interactive visualization of genomic data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>37</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ROBOKOP_KG" target="_blank">ROBOKOP KG</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ROBOKOP KG is an Integrated Knowledge Graphs from Federated Sources.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="http://robokop.renci.org" target="_blank">http://robokop.renci.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rpinbase" target="_blank">RPINBASE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An online toolbox to extract features for predicting RNA-protein interactions.<br>Explore and create dataset.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>RNA</li><li>RNA immunoprecipitation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>RNA-binding protein prediction</li><li>Feature extraction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Conservation_Archive" target="_blank">Conservation Archive</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database portal containing esources that can be used to infer baseline species conditions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Ecology</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ccpdb" target="_blank">ccPDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CcPDB (Compilation and Creation of datasets from PDB) is designed to provide service to scientific community working in the field of function or structure annoation of proteins. This database of datasets is complied from the literature and the Protein Data Bank (PDB), where all datasets were derived from PDB.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein secondary structure</li><li>Protein folding, stability and design</li><li>Proteins</li><li>Protein structure analysis</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein secondary structure assignment</li><li>Protein secondary structure prediction</li><li>Protein model validation</li><li>Protein secondary structure comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>22</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cancerppd" target="_blank">CancerPPD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A repository of experimentally verified anticancer peptides (ACPs) and anticancer proteins. Data were manually collected from published research articles, patents and from other databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Oncology</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>103</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/backtranseq-ebi" target="_blank">backtranseq (EBI)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Back-translate a protein sequence to a nucleotide sequence, using codon frequency.It reads a protein sequence and writes the nucleic acid sequence it is most likely to have come from.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>DNA translation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://www.ebi.ac.uk/support/" target="_blank">http://www.ebi.ac.uk/support/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/oRNAment" target="_blank">oRNAment</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of putative RNA binding protein target sites in the transcriptomes of model species.<br>Dashboard - Overview of the database.<br>Welcome to oRNAment - o RNA motifs enrichment in complete transcriptomes.<br>Here, we developed a computational tool to enable the global mapping of putative RBP target motifs across full transcriptomes, including messenger RNAs (mRNAs) and non-coding RNAs, in multiple species. The oRNAment database provides a user-friendly interface to parse the motif scan data. See tutorial for further detail</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Functional, regulatory and non-coding RNA</li><li>Protein interactions</li><li>Endocrinology and metabolism</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>In vitro selection</li><li>Filtering</li><li>RNA binding site prediction</li><li>Sequence motif recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/reactome" target="_blank">Reactome</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of human biological pathways and processes ranging from basic processes of metabolism to complex regulatory pathways. The data is curated by biologists and subsequently peer-reviewed for accuracy and consistency. Cross-references with UniProt, PubMed, Ensembl, Gene Ontology and LocusLink are also provided. It supersedes The Genome Knowledgebase project.</div></td>
      <td><div class='topic short click_expand'><ul><li>Endocrinology and metabolism</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Query and retrieval</li><li>Pathway or network analysis</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mfungd" target="_blank">MIPS mouse functional genome database (MfunGD)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mouse genome database including DNA and protein sequences, annotation, gene models and mapping etc.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/blastgui" target="_blank">BlastGUI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BlastGUI is A Python-Based Cross-Platform Local BLAST Visualization Software.<br>Users can build databases and perform sequence filtering and sequence alignment through a graphical user interface. The operation visualization, automatic sequence filtering, and cross-platform use can significantly facilitate the analysis of biological data and the mining of biological information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Database management</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Filtering</li><li>Sequence alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mrdp" target="_blank">mrdp</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A design pattern for networked, data-intensive science.<br>Hello Guest Register Now Sign In : Forgot Password?.<br>BPRC Arctic System Reanalysis (ASR).<br>ECMWF 20th Century Reanalysis (ERA-20C)</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Data security</li><li>Computer science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Service management</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://rda.ucar.edu" target="_blank">http://rda.ucar.edu</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/titta" target="_blank">Titta</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A toolbox for creating PsychToolbox and Psychopy experiments with Tobii eye trackers.<br>The current repository furthermore offers a C++ wrapper around the Tobii SDK, which is in turn used as a basis for the MATLAB MEX file TittaMex providing MATLAB Titta with connectivity to the Tobii eye trackers. This C++ wrapper can be consumed by your own C++ projects as well, or be wrapped for other programming languages (a wrapper to Python is also provided).<br>Titta is a toolbox for using eye trackers from Tobii Pro AB with Python, specifically offering integration with PsychoPy.</div></td>
      <td><div class='topic short click_expand'><ul><li>Laboratory techniques</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Scaffolding</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/marcus-nystrom/Titta" target="_blank">https://github.com/marcus-nystrom/Titta</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/grok" target="_blank">GROK</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This tool is “Swiss Army knife” library for processing genomic interval data. It operates on genomic regions, annotated chromosomal intervals that represent sequencing short reads, gene locations, ChIP-seq peaks or other genomic features. Applications of GROK include file format conversions, set operations, overlap queries, and filtering and transformation operations. Supported file formats include BAM/SAM, BED, BedGraph, CSV, FASTQ, GFF/GTF, VCF and Wiggle.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>ChIP-seq</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/grok-genomic-region-operation-toolkit.html" target="_blank">http://www.mybiosoftware.com/grok-genomic-region-operation-toolkit.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ncbi_bookshelf" target="_blank">NCBI Bookshelf</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Searchable collection of biomedical books linked to terms in PubMed abstracts; when viewing an abstract, click on Books link to see phrases in abstract hyperlinked to book sections.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biomedical science</li><li>Biotechnology</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Query and retrieval</li><li>Deposition</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ggrna" target="_blank">GGRNA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Google-like, ultrafast search engine for genes and transcripts. Users input gene names, IDs, gene descriptions, annotations or sequence and are returned relevant RefSeq transcripts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Gene expression</li><li>Probes and primers</li><li>Transcription factors and regulatory sites</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>PCR primer design</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/p2cs" target="_blank">P2CS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database for prokaryotic two-component systems (TCSs). It annotates and classifies TCS proteins from completely sequenced prokaryotic genomes and metagenomes. TCS complements can be browsed by metagenome, replicon or sequence cluster (and these genesets are available for download by users).</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Gene and protein families</li><li>Metagenomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Genome annotation</li><li>Query and retrieval</li><li>Protein fold recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>43</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/wattos" target="_blank">Wattos</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Collection of Java programs for structural biology and NMR spectroscopy. It’s programs analyze, annotate, parse, archive, and disseminate experimental NMR data deposited by authors world wide into the PDB and BMRB.</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>Structural biology</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Spectral analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>38</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/wattos-software-structural-biology-nmr-spectroscopy.html" target="_blank">http://www.mybiosoftware.com/wattos-software-structural-biology-nmr-spectroscopy.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ebi_metagenomics" target="_blank">MGnify</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web resource targeted at metagenomic researchers.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>68</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/emirit" target="_blank">emiRIT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>EmiRIT is a text-mining based database for microRNA information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Functional, regulatory and non-coding RNA</li><li>Pathology</li><li>Natural language processing</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Text mining</li><li>miRNA expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/comparative_genometrics" target="_blank">Comparative genometrics CG database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Devoted to the characterization of complete chromosome sequences by standardized genometric methods, this website displays for sequenced genomes, three different genometric analyses: the DNA walk and the GC and TA skews during the initial phase. Although primarly focused on prokaryotic chromosomes, the CG website posts genometric information on paradigm plasmids, phages, viruses, and organelles.</div></td>
      <td><div class='topic short click_expand'><ul><li>Comparative genomics</li><li>Nucleic acid structure analysis</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid structure report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>29</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/taxa" target="_blank">Taxa</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Define and manipulate taxonomic data.  Robust and flexible solution to storing and manipulating taxonomic data and any application-specific information associated with it. It provides parsers that can read common sources of taxonomic information from nearly any format while preserving associated data. It takes into account the hierarchical nature of taxa and can modify the taxonomy or associated data in such a way that both are kept in sync.</div></td>
      <td><div class='topic short click_expand'><ul><li>Taxonomy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Parsing</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td><div class="link short click_expand"><a href="https://cran.r-project.org/web/packages/taxa/index.html" target="_blank">https://cran.r-project.org/web/packages/taxa/index.html</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/spliceaid-f" target="_blank">SpliceAid-F</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of human splicing factors and their binding sites.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>69</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PGxMine" target="_blank">PGxMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This is the codebase for the PGxMine project to using text-mining to identify papers for curation into PharmGKB. It is a Python3 project that makes use of the Kindred relation classifier along with the PubRunner project to run tools across PubMed and accessible PubMed Central.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pharmacogenomics</li><li>Personalised medicine</li><li>Machine learning</li><li>Natural language processing</li><li>Drug metabolism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant filtering</li><li>Text mining</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/jakelever/pgxmine" target="_blank">https://github.com/jakelever/pgxmine</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IPD_project" target="_blank">IPD project</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A centralised resource for the study of polymorphism in genes of the immune system.<br>The Immuno Polymorphism Database.<br>IPD was developed in 2003 to provide a centralised system for the study of polymorphism in genes of the immune system. The IPD project was established by the HLA Informatics Group of the Anthony Nolan Research Institute in close collaboration with the European Bioinformatics Institute.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunoproteins and antigens</li><li>Genetic variation</li><li>Ontology and terminology</li><li>Database management</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Genetic variation analysis</li><li>Epitope mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/slinky" target="_blank">slinky</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R package for L1000 data set to streamline the process of identifying samples of interest and their corresponding control samples, and loading their associated expression data and metadata.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deisotoping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/annmap" target="_blank">annmap</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides annotation mappings for Affymetrix exon arrays and coordinate based queries to support deep sequencing data analysis. Database access is hidden behind the API which provides a set of functions such as genesInRange(), geneToExon(), exonDetails(), etc. Functions to plot gene architecture and BAM file data are also provided. Underlying data are from Ensembl.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li><li>Nucleic acids</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid features : gene annotation format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database search results : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>53</div></td>
      <td><div class="link short click_expand"><a href="http://annmap.cruk.manchester.ac.uk" target="_blank">http://annmap.cruk.manchester.ac.uk</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/HymenopteraMine" target="_blank">HymenopteraMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated data warehouse for the Hymenoptera Genome Database</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/xtandem_parser" target="_blank">XTandem Parser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Parsing and analysing X! Tandem MS/MS search results.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Mass spectrum visualisation</li><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : x!tandem xml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : dta</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>37</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gnpis" target="_blank">Genetic and Genomic Information System (GnpIS)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A multispecies integrative information system dedicated to plant and fungi pests.. It allows researchers to access genetic, phenotypic and genomic data. It is used by both large international projects and the French National Research Institute for Agriculture, Food and Environment</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Genotype and phenotype</li><li>Genomics</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Indexing</li><li>Data handling</li><li>Query and retrieval</li><li>Ontology visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>52</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aging_mouse_brain" target="_blank">aging mouse brain</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Single-cell transcriptomic profiling of the aging mouse brain | Methodios Ximerakis, Scott L. Lipnick, Brendan T. Innes, Sean K. Simmons, Xian Adiconis, Danielle Dionne, Brittany A. Mayweather, Lan Nguyen, Zachary Niziolek, Ceren Ozek, Vincent L. Butty, Ruth Isserlin, Sean M. Buchanan, Stuart S. Levine, Aviv Regev, Gary D. Bader, Joshua Z. Levin, and Lee L. Rubin</div></td>
      <td><div class='topic short click_expand'><ul><li>Geriatric medicine</li><li>Neurobiology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene-set enrichment analysis</li><li>Parsing</li><li>Transcriptome assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>46</div></td>
      <td><div class="link short click_expand"><a href="http://shiny.baderlab.org/AgingMouseBrain/" target="_blank">http://shiny.baderlab.org/AgingMouseBrain/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iRO-PsekGCC" target="_blank">iRO-PsekGCC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Identify DNA Replication Origins Based on Pseudo k-Tuple GC Composition.<br>: Server : ReadMe : Supporting Information : Citation : Contact Us :</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/idconvert" target="_blank">idconvert</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A converter between pepXML and mzIdentML formats for peptide identifications from tandem mass spectrometry. Developed by the ProteoWizard team.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Conversion</li><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : mzidentml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : mzidentml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1,070</div></td>
      <td><div class="link short click_expand"><a href="http://proteowizard.sourceforge.net/downloads.shtml" target="_blank">http://proteowizard.sourceforge.net/downloads.shtml</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fastscop" target="_blank">fastSCOP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>FastSCOP identifies structural domains and determines evolutionary superfamilies of a query protein structure using 3D-BLAST to scan SCOP and MAMMOTH to create a structural alignment for refining domain boundaries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Structural biology</li><li>Proteins</li><li>Protein structure analysis</li><li>Structure prediction</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein fold recognition</li><li>Structure alignment</li><li>Protein structure analysis</li><li>Protein secondary structure comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>18</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/drug-target_explorer" target="_blank">Drug-Target Explorer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Multifunctional platform for exploring chemical space as it relates to biological targets, and may be useful at several steps along the drug development pipeline including target discovery, structure–activity relationship, and lead compound identification studies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Drug discovery</li><li>Data management</li><li>Drug development</li><li>Cheminformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/twofeat" target="_blank">twofeat</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Find neighbouring pairs of features in sequence(s).</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cutgextract" target="_blank">cutgextract</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Extract codon usage tables from CUTG database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Staphopia" target="_blank">Staphopia</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Staphopia is a database focusing on the genomes of the bacterial pathogen Staphylococcus aureus. Life-threatening Methicillin-resistant S. aureus (MRSA) infections strike across our society, both in the community and hospital settings. In recent years whole genome shotgun sequencing of clinical isolates has become common. As of August 2017 there were over 44,000 publicly available S. aureus sequencing projects in EBI's ENA database. Staphopia aims to provide rapid analysis of these whole-genome datasets as well as useful S. aureus specific results, such as genotype, antibiotic resistance and virulence gene profiles.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Sample collections</li><li>Sequence assembly</li><li>DNA polymorphism</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Antimicrobial resistance prediction</li><li>Multilocus sequence typing</li><li>Genome indexing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/staphopia/staphopia-r" target="_blank">https://github.com/staphopia/staphopia-r</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/query_win" target="_blank">QUERY_WIN</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Retrieval program with a graphical user interface for ACNUC databases . It can query any ACNUC database, e.g., for nucleotide of protein sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Proteins</li><li>Nucleic acids</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>25</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/query_win-retrieval-program-graphical-user-interface-acnuc-databases.html" target="_blank">http://www.mybiosoftware.com/query_win-retrieval-program-graphical-user-interface-acnuc-databases.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pyir" target="_blank">PyIR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PyIR is a scalable wrapper for processing billions of immunoglobulin and T cell receptor sequences using IgBLAST.<br>PyIR is a minimally-dependent high-speed wrapper for the IgBLAST immunoglobulin and T-cell analyzer. This is achieved through chunking the input data set and running IgBLAST single-core in parallel to better utilize modern multi-core and hyperthreaded processors.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Immunogenetics</li><li>Immunoproteins and antigens</li><li>Workflows</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Parsing</li><li>Splitting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IndigoMine" target="_blank">IndigoMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>INDIGO enables the integration of annotations for the exploration and analysis of newly sequenced microbial genomes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/excemplify" target="_blank">Excemplify</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based application that was developed to support the exchange and long-time storage of experimental data, especially spreadsheets in general and immunoblot-related experiments in particular. It is able to parse these data from the initial experimental setup stage and to automatically generate the following spreadsheet stages in the experimental workflow.</div></td>
      <td><div class='topic short click_expand'><ul><li>Laboratory information management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Workbench</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/alfie" target="_blank">Alfie</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Alfie is an alignment-free, kingdom level taxonomic classifier for DNA barcode data. Alfie classifies sequences using a neural network which takes k-mer frequencies (default k = 4) as inputs and makes kingdom level classification predictions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Biodiversity</li><li>Metagenomics</li><li>Zoology</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>DNA barcoding</li><li>k-mer counting</li><li>Parsing</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://pypi.org/project/alfie/" target="_blank">https://pypi.org/project/alfie/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/asd" target="_blank">ASD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>AlloSteric Database (ASD) provides a central resource for the display, search and analysis of structure, function and related annotation for allosteric molecules. Proteins are annotated with a detailed description of allostery, biological process and related diseases, and modulators with binding affinity, physicochemical properties and therapeutic area.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Medicinal chemistry</li><li>Pathology</li><li>Structure analysis</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein property calculation</li><li>Protein function analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>105</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ROMOP" target="_blank">ROMOP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A light-weight R package for interfacing with OMOP-formatted electronic health record data.<br>R package to easily interface with OMOP-formatted EHR data.<br>ROMOP, please wait... This may take a minute or two.<br>Launching ROMOP, please wait... This may take a minute or two. ...<br>ROMOP is a flexible R package to interface with the Observational Health Data Sciences and Informatics (OHDSI) OMOP Common Data Model. Briefly, OMOP is a standardized relational database schema for Electronic Health Record (EHR) or Electronic Medical Record (EMR) data (i.e., patient data collected during clinical visits to a health system). The main benefit of a standardized schema is that it allows for interoperability between institutions, even if the underlying EHR vendors are disparate.<br>Benjamin S. Glicksberg 9/14/2018</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical informatics</li><li>Medicine</li><li>Ontology and terminology</li><li>Database management</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Database search</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://github.com/BenGlicksberg/ROMOP" target="_blank">http://github.com/BenGlicksberg/ROMOP</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ccap" target="_blank">Culture collection of algae and protozoa (CCAP)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Culture collection of algae and protozoa.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>cell line report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>20</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pdbe" target="_blank">PDBe</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Protein Data Bank in Europe is the European resource for the collection, organisation and dissemination of data on biological macromolecular structures. The database infrastructure is based on the remediated PDB archive data and a specially designed database for storing information on interactions between proteins and bound molecules.</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>X-ray diffraction</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Design</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>85</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/vcfview" target="_blank">vcfView</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An Extensible Data Visualization and Quality Assurance Platform for Integrated Somatic Variant Analysis.<br>An interactive Rshiny tool designed to support the evaluation of somatic mutation calls from cancer sequencing data. The tool takes as input a single variant call format (VCF) file and enables researchers to explore the impacts of analytical choices on the mutant allele frequency spectrum, on mutational signatures and on annotated somatic variants in genes of interest. It allows variants that have failed caller filters to be re-examined in order to improve sensitivity or guide strategies for optimal sample / sequencing preparation. It is extensible allowing other algorithms to take advantage of its VCF preprocessing capabilities.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Sequencing</li><li>Genetic variation</li><li>Quality affairs</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Variant calling</li><li>Variant filtering</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/VDJdb" target="_blank">VDJdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>VDJdb is a database extension, new analysis infrastructure and a T-cell receptor motif compendium.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunoproteins and antigens</li><li>Bioinformatics</li><li>Sequence sites, features and motifs</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence motif discovery</li><li>Query and retrieval</li><li>Epitope mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>34</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_DomainSuperimpose" target="_blank">IMGT DomainSuperimpose</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/DomainSuperimpose is the IMGT® tool to superimpose 3D structures of two domains from IMGT/3Dstructure-DB. Domain 3D structures are numbered according to the IMGT-ONTOLOGY concept of IMGT unique numbering.</div></td>
      <td><div class='topic short click_expand'><ul><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>keyword : html</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>205</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/paxtoolsr" target="_blank">paxtoolsr</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The package provides a set of R functions for interacting with BioPAX OWL files using Paxtools and the querying Pathway Commons (PC) molecular interaction database that are hosted by the Computational Biology Center at Memorial Sloan-Kettering Cancer Center (MSKCC). Pathway Commons databases include: BIND, BioGRID, CORUM, CTD, DIP, DrugBank, HPRD, HumanCyc, IntAct, KEGG, MirTarBase, Panther, PhosphoSitePlus, Reactome, RECON, TRANSFAC.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Formatting</li><li>Gene regulatory network analysis</li><li>Database search</li><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>pathway or network : biopax</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>pathway or network : sbgn-ml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/BioPAX/paxtoolsr" target="_blank">https://github.com/BioPAX/paxtoolsr</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pdbsum" target="_blank">PDBsum overview of macromolecular structures</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>At-a-glance overview of every macromolecular structure deposited in the Protein Data Bank (PDB), giving schematic diagrams of the molecules in each structure and of the interactions between them.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>structure report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>229</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sra" target="_blank">SRA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sequence Read Archive. Public repository for next-generation sequencing data. Stores raw sequencing data and alignment information from high-throughput sequencing platforms, including Roche 454 GS System®, Illumina Genome Analyzer®, Applied Biosystems SOLiD System®, Helicos Heliscope®, Complete Genomics®, and Pacific Biosciences SMRT®.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Database management</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Deposition</li><li>Data retrieval</li><li>Read pre-processing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,349</div></td>
      <td></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/um-bbd" target="_blank">UM-BBD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The University of Minnesota Biocatalysis/Biodegradation Database (UM-BBD) contains information on compounds, enzymes, reactions and microorganism entries. It includes a Biochemical Periodic Table (UM-BPT) and a rule-based Pathway Prediction System (UM-PPS) that predicts plausible pathways for microbial degradation of organic compounds. UM-BBD compound data are contributed to PubChem and ChemSpider, the public chemical databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Microbiology</li><li>Molecular interactions, pathways and networks</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Pathway or network prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>169</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/BEL_Commons" target="_blank">BEL Commons</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Environment for exploration and analysis of networks encoded in Biological Expression Language.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Environmental science</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enrichment analysis</li><li>Parsing</li><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bel-commons" target="_blank">https://github.com/bel-commons</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/echmet" target="_blank">ECHMET</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Suite containing tools in the field of chromatography.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Regression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Suite</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hops_tk" target="_blank">HOPS_tk</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>High-performance library for (non-)uniform sampling of convex-constrained models.<br>The Highly Optimized Polytope Sampling toolbox is an open-source C++17 library for efficient and scalable MCMC algorithms for sampling convex-constrained spaces possibly equipped with arbitrary target functions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sample collections</li><li>Bioinformatics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Virtual screening</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Suite</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pptasedb" target="_blank">Prokaryotic protein phosphatase database (PptaseDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Prokaryotic protein phosphatases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/echo-circadian" target="_blank">Extended Circadian Harmonic Oscillators (ECHO)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An Application for Detection and Analysis of Oscillators Identifies Metabolic Regulation on Genome-Wide Circadian Output | An application to find and visualize circadian rhythms in time course data using extended harmonic oscillators | Finding Rhythms Using Extended Circadian Harmonic Oscillators (ECHO) | Provides a function (echo_find()) designed to find rhythms from data using extended harmonic oscillators. For more information, see H. De los Santos et al. (2017) <doi:10.1145/3107411.3107420>  | This is the first step in the PAICE (Pipeline for Amplitude Integration of Circadian Exploration) Suite! This suite of tools provides high-throughput applications for circadian, ultradian, and infradian rhythms | ECHO (Extended Circadian Harmonic Oscillators) is an R-powered application designed to find and visualize circadian rhythms from your data using extended harmonic oscillators | Use and First-Time Set-Up Instructions</div></td>
      <td><div class='topic short click_expand'><ul><li>Endocrinology and metabolism</li><li>Biology</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://cran.r-project.org/web/packages/echo.find/index.html" target="_blank">https://cran.r-project.org/web/packages/echo.find/index.html</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sRNA" target="_blank">sRNA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>AGO-unbound cytosolic pool of mature miRNAs in plant cells reveals a novel regulatory step at AGO1 loading | Small RNAs were sequenced from FPLC-fractionated Arabidopsis leaf and flower crude extracts | Here we provide the scripts that reproduce the results and figures of the paper Dalmadi et al. 2019. These scripts are not meant to be reusable. Most of the figures and tables underwent post-processing in Excel, Powerpoint, or other GUI softwares, therefore the results will not exactly be the same as they appear in the paper | Download this directory and run the sRNA_analysis.sh script which downloads sRNA-seq reads from the SRA database and processes them. The results are placed in the 'sRNA-seq' directory | Linux operation system with zsh shell</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li><li>Plant biology</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>miRNA target prediction</li><li>Parsing</li><li>miRNA expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/glycosciences" target="_blank">Glycosciences</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Carbohydrate structure database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Carbohydrates</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>carbohydrate report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>199</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mousebook" target="_blank">Mousebook</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The database and web portal provide access to information about mutant mouse lines held as live or cryopreserved stocks at MRC Harwell. It portal integrates curated information from the MRC Harwell stock resource, and other Harwell databases, with information from external data resources to provide value-added information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Zoology</li><li>Genotype and phenotype</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Query and retrieval</li><li>Database search</li><li>Format detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biopax-parser" target="_blank">BioPAX-Parser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BioPAX Parser is graphical tool to perform pathway enrichment analysis using<br>BioPAX files. In addition BiP enable user to extract and save pathway information<br>in a tabular format.</div></td>
      <td><div class='topic short click_expand'><ul><li>Endocrinology and metabolism</li><li>Molecular interactions, pathways and networks</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway analysis</li><li>Enrichment analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_LocusView" target="_blank">IMGT LocusView</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/LocusView is the IMGT® tool providing a view of immunoglobulin (IG), T cell receptor (TR) and major histocompatibility (MH) loci.<br>Display is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence image : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>186</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/glassgo" target="_blank">GLASSgo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>GLobal Automated sRNA Search go, combines iterative BLAST searches, pairwise identity filtering, and structure based clustering in an automated prediction pipeline to find sRNA homologs from scratch. The web server provides predefined parameter sets for a non-expert usage as well as enables a manual setup of the query parameters. The returned GLASSgo result is in FASTA format, whereby the first entry represents the input sequence.</div></td>
      <td><div class='topic short click_expand'><ul><li>Taxonomy</li><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Transcriptional regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid sequence : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>taxonomy : json</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/greg" target="_blank">GREG</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>GREG-studying transcriptional regulation using integrative graph databases.<br>Scripts related to GREG (Gene Regulation Graph Database) and its papers. Use GREG at: http://www.moralab.science/GREG/.<br>(The Gene Regulation Graph Database).</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li><li>Molecular interactions, pathways and networks</li><li>Functional, regulatory and non-coding RNA</li><li>Gene regulation</li><li>DNA polymorphism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Gene regulatory network prediction</li><li>Gene regulatory network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/mora-lab/GREG" target="_blank">https://github.com/mora-lab/GREG</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pcxn_bioconductor" target="_blank">pcxn</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Discover the correlated pathways/gene sets of a single pathway/gene set or discover correlation relationships among multiple pathways/gene sets. Draw a heatmap or create a network of your query and extract members of each pathway/gene set found in the available collections.</div></td>
      <td><div class='topic short click_expand'><ul><li>Laboratory techniques</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Modelling and simulation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/pcxn/" target="_blank">http://bioconductor.org/packages/pcxn/</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/writefcs" target="_blank">writeFCS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>WriteFCS is a MATLAB script that creates FCS file specified by Data File Standard for Flow Cytometry</div></td>
      <td><div class='topic short click_expand'><ul><li>Biotherapeutics</li><li>Cytometry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PulmonDB" target="_blank">PulmonDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A curated lung disease gene expression database | An accessible R package to download PulmonDB data | PulmonDB is a transcriptomic database of IPF and COPD with curated annotation | PulmonDB is a gene expression database with Chronic Obstructive Pulmonary Diseases (COPD) and Idiopathic Pulmonary Disease (IPF) experiments. It has homogenized values using individual contrast and manual curated annotation that can be download or accessed by http://pulmondb.liigh.unam.mx</div></td>
      <td><div class='topic short click_expand'><ul><li>Data submission, annotation and curation</li><li>Transcriptomics</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Expression analysis</li><li>Transcriptome assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/AnaBVA/pulmondb" target="_blank">https://github.com/AnaBVA/pulmondb</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tam_2.0" target="_blank">TAM 2.0</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MicroRNA set analysis. Test the functional and disease annotations of miRNAs by overrepresentation analysis, compare the input de-regulated miRNAs with those de-regulated in other disease conditions via correlation analysis. Also miRNA set query and result visualization are also enabled.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Functional, regulatory and non-coding RNA</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>42</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/omamer" target="_blank">OMAmer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tree-driven and alignment-free protein assignment to subfamilies outperforms closest sequence approaches.<br>OMAmer is a novel alignment-free protein family assignment method, which limits over-specific subfamily assignments and is suited to phylogenomic databases with thousands of genomes. It is based on an innovative method using subfamily-informed k-mers for alignment-free mapping to ancestral protein subfamilies. Whilst able to reject non-homologous family-level assignments, it has provided better and quicker subfamily-level assignments than a method based on closest sequences (using DIAMOND).</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Phylogenomics</li><li>Gene and protein families</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>k-mer counting</li><li>Query and retrieval</li><li>Taxonomic classification</li><li>Genome indexing</li><li>Local alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ROBOKOP" target="_blank">ROBOKOP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An Abstraction Layer and User Interface for Knowledge Graphs to Support Question Answering | ROBOKOP is a tool for reasoning over structured biomedical knowledge databases as part of the NCATS translator and reasoner programs. The ROBOKOP system consists of a web based user interface, an API server, and several worker servers. The code is separated into three separate repositories</div></td>
      <td><div class='topic short click_expand'><ul><li>Natural language processing</li><li>Environmental science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="http://robokopkg.renci.org" target="_blank">http://robokopkg.renci.org</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/oligomap" target="_blank">Oligomap</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Program for fast identification of nearly-perfect matches of small RNAs in sequence databases. It allows to exhaustively identify all the perfect and 1-error (where an error is defined to be a mismatch, insertion or deletion) matches of large sets of small RNAs to target sequences. Optimal performance is achieved at about 500000 query sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>49</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/oligomap-1-01-identification-nearly-perfect-matches-small-rnas-sequence-databases.html" target="_blank">http://www.mybiosoftware.com/oligomap-1-01-identification-nearly-perfect-matches-small-rnas-sequence-databases.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/paxdb-parser" target="_blank">PaxDB Parser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Parser for PaxDB-file which contains the proteins level of genes in the cell</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/MayaGalili/ParseFiles-PaxDB" target="_blank">https://github.com/MayaGalili/ParseFiles-PaxDB</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/vhldb" target="_blank">VHLdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of interactors and mutations of the human von Hippel-Lindau tumor suppressor protein (pVHL).</div></td>
      <td><div class='topic short click_expand'><ul><li>Rare diseases</li><li>Protein structure analysis</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein function analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mutation identifier : sequence feature table format (text)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : tsv</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/g-links" target="_blank">G-Links</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A gateway server for querying and retrieving gene annotation data</div></td>
      <td><div class='topic short click_expand'><ul><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome annotation</li><li>Data retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/cory-ko/G-Links" target="_blank">https://github.com/cory-ko/G-Links</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ENdb" target="_blank">ENdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A manually curated database of experimentally supported enhancers for human and mouse. Enhancers are a class of cis-regulatory elements that can increase gene transcription by forming loops in intergenic regions, introns and exons</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA polymorphism</li><li>Transcription factors and regulatory sites</li><li>Gene transcripts</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Expression analysis</li><li>Promoter prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phonbank" target="_blank">PhonBank</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Markedness and implicational relationships in phonological development.<br>Phon is a software program that supports the building of textual and phonological data corpora. While it was originally created to support the study of child language development, Phon can be used ...<br>PhonBank is the child phonology component of the TalkBank system. TalkBank is a system for sharing and studying conversational interactions. PhonBank is supported by grant RO1-HD051698 from NIH-NICHHD to Brian MacWhinney and Yvan Rose. PHON is designed and built by Yvan Rose and Greg Hedlund. Currently available materials include:.<br>Phon is a software program that supports the building of textual and phonological data corpora.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Biological databases</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Query and retrieval</li><li>Regression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://www.phon.ca" target="_blank">https://www.phon.ca</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/varied" target="_blank">VariED</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The first integrated database of gene annotation and expression profiles for variants related to human diseases.<br>An integrated database of variants and gene expression profiles for genetic diseases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Gene expression</li><li>Pathology</li><li>Rare diseases</li><li>Genomics</li><li>Biobank</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant prioritisation</li><li>Variant calling</li><li>Expression analysis</li><li>Query and retrieval</li><li>Variant classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CATHER" target="_blank">CATHER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A novel threading algorithm with predicted contacts.<br>CATHER is a threading algorithm for template-based modeling. It uses an iterative algorithm to combine both sequential information and contact maps to improve the threading performance. For a submitted query protein sequence, the returned results include the predicted structure</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Structure prediction</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Residue contact prediction</li><li>Protein threading</li><li>Protein structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/meta-prism" target="_blank">Meta-Prism</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Ultra-fast and highly accurate microbial community structure search utilizing dual indexing and parallel computation.<br>Meta-Prism aims to conduct quick query in large-scale metagenomics database. The method is based on three computational techniques: dual-indexing approach for sample sub-grouping, refined scoring function that could scrutinize the minute differences among samples, and parallel computation based on GPU computing.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sample collections</li><li>Metagenomics</li><li>Microbial ecology</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sample comparison</li><li>Regression analysis</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sibils" target="_blank">SIBiLS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>RESTful customizable search engines in biomedical literature, enriched with automatically mapped biomedical concepts.<br>COVID-19 outbreak: explore our literature triage interface and APIs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Personalised medicine</li><li>Workflows</li><li>Literature and language</li><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Literature search</li><li>Data retrieval</li><li>Parsing</li><li>Relation extraction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/compendiumdb" target="_blank">compendiumdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Systematic retrieval and storage of functional genomics data via a MySQL database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li><li>Data acquisition</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mico" target="_blank">MICO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Meta-tool for automatically querying and collecting related information from multiple biology/bioinformatics enabled web servers with prediction capabilities.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Genetic variation</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mico-mutation-information-collector.html" target="_blank">http://www.mybiosoftware.com/mico-mutation-information-collector.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ccancer" target="_blank">CCancer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CCancer is an automatically collected database of gene lists reported in various studies. The current coverage is 3369 gene lists from 2644 papers. Enrichment analysis reports intersecting gene lists with an inputted gene list.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenetics</li><li>Experimental design and studies</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Gene-set enrichment analysis</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ebi_tools" target="_blank">EBI Tools</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>EBI Tools is a project that aims to provide programmatic access to the various databases and retrieval and analysis services that the European Bioinformatics Institute (EBI) provides through Simple Object Access Protocol (SOAP) and other related web service technologies. Example tools include those to compute sequence similarity searches, pairwise/multiple sequence alignment and protein functional analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Sequence analysis</li><li>Rare diseases</li><li>Function analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Multiple sequence alignment</li><li>Protein function analysis</li><li>Query and retrieval</li><li>Data retrieval</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>1,372</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/arrayexpress" target="_blank">ArrayExpress</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Public repository of functional genomics data supporting publications. It includes data generated by sequencing or array-based technologies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional genomics</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Gene expression clustering</li><li>Gene expression analysis</li><li>Simulated gene expression data generation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>260</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/avatar" target="_blank">AVAtar</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Avatar is a standalone software for analysis and visualisation of alteration data. Avatar features interactively customizable alteration plots, extensive data import capabilities and a multi-objective optimization algorithm to find alteration patterns, e.g. gene sets with mutually exclusive alterations or small gene sets with high sample coverage. License: EPL-2.0 (https://spdx.org/licenses/EPL-2.0.html)</div></td>
      <td><div class='topic short click_expand'><ul><li>Preclinical and clinical studies</li><li>Genetic variation</li><li>Gene expression</li><li>Personalised medicine</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/flyrnai" target="_blank">FlyRNAi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of Dorsophila RNA interference (RNAi) Screening Center.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Compound libraries and screening</li><li>RNAi experiment</li><li>Electron microscopy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Database search</li><li>PCR primer design</li><li>Transcriptional regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>35</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ensembl_fugu_genome_browser" target="_blank">Ensembl Fugu Genome Browser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Fugu Ensembl is a joint project between The IMCB and the EMBL - EBI to produce and maintain an automatic annotation of the Fugu Genome.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Human biology</li><li>Whole genome sequencing</li><li>Database management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Genome visualisation</li><li>Gene prediction</li><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,052</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gene" target="_blank">Gene</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A curated transcriptome dataset collection to investigate inborn errors of immunity.<br>Primary immunodeficiencies (PIDs) are a heterogeneous group of inherited disorders, frequently caused by loss-of-function and less commonly by gain-of-function mutations, which can result in susceptibility to a broad or a very narrow range of infections but also in inflammatory, allergic or malignant diseases. Owing to the wide range in clinical manifestations and variability in penetrance and expressivity, there is an urgent need to better understand the underlying molecular, cellular and immunological phenotypes in PID patients in order to improve clinical diagnosis and management. Here we have compiled a manually curated collection of public transcriptome datasets mainly obtained from human whole blood, peripheral blood mononuclear cells (PBMCs) or fibroblasts of patients with PIDs and of control subjects for subsequent meta-analysis, query and interpretation</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li><li>Genotype and phenotype</li><li>Allergy, clinical immunology and immunotherapeutics</li><li>Microarray experiment</li><li>Rare diseases</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Data retrieval</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GenIE-Sys" target="_blank">GenIE-Sys</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genome Integrative Explorer System.<br>Genome Integrative Explorer System (GenIE-Sys).<br>Website : Documentation : Demo : Users.<br>Welcome to GenIE-Sys’s documentation! — GenIE-Sys 0.1 documentation.<br>The Genome Integrative Explorer System(GenIE-Sys) based on a flat files Content Management System which is a dedicative in-house system to facilitate external groups in setting up their own web resource for searching, visualizing, editing, sharing and reproducing their genomic and transcriptomic data while using project raw data(gff3,fasta,fatsq) as an input.<br>This is the Development and the latest version of GenIE-Sys. Our main goal is to add admin interface where users can easily create database plus integrate different types of data, create new pages and menus, configure tools and changing website layout by using Themes.<br>Free document hosting provided by Read the Docs</div></td>
      <td><div class='topic short click_expand'><ul><li>Data governance</li><li>RNA-Seq</li><li>Plant biology</li><li>Database management</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Editing</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://github.com/plantgenie/geniesys.git" target="_blank">http://github.com/plantgenie/geniesys.git</a></div></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gwar" target="_blank">GWAR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Robust Analysis and Meta-Analysis of Genome-Wide Association Studies.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>DNA polymorphism</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tfbstools" target="_blank">TFBSTools</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Package for the analysis and manipulation of transcription factor binding sites. It includes a matrix conversion system between Position Frequency Matrix (PFM), Position Weight Matrix (PWM) and Information Content Matrix (ICM). It can also scan putative TFBS from sequence/alignment, query the JASPAR database and provide a wrapper of de novo motif discovery software.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Transcriptional regulatory element prediction</li><li>Transcription factor binding site prediction</li><li>Sequence motif recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid sequence record : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>plot : image format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>74</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ge11232002/TFBSTools" target="_blank">https://github.com/ge11232002/TFBSTools</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mygene.info" target="_blank">MyGene.Info</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Simple-to-use REST web services to query/retrieve gene annotation data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>70</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/3123" target="_blank">https://www.biocatalogue.org/services/3123</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/the_chickpea_portal" target="_blank">The Chickpea Portal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A collaborative chickpea genomics program comprising advanced pre-breeding tools for the discovery and validation of novel genes and traits leading to development of abiotic stress tolerant chickpea. This is a project for an abiotic stress tolerant chickpea, and contains: Chickpea GBrowse Viewer; Sequence Similarity Search Tools; Raw data download.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_GENE-DB" target="_blank">IMGT GENE-DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/GENE-DB is the IMGT® database for immunoglobulin (IG) and T cell receptor (TR) genes and alleles (international nomenclature).<br>Annotation is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunogenetics</li><li>Immunoinformatics</li><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene name : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence : fasta</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>467</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dodo" target="_blank">DODO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DODO is a graph database to facilitate access and interaction with disease and phenotype ontologies. The aim of DODO is to allow an easier way to interact and explore disease ontologies and their identifiers. The database is build on Neo4j and incorporates different ontologies with an accompagnying R package that allows easy access, exploration, and definition of disease concepts of interest. It can work as the intermediate player to facilitate access and exhaustive extraction of information from other life science databases without the need to harmonize these up front.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Imaging</li><li>Workflows</li><li>Pathology</li><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Parsing</li><li>Named-entity and concept recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Ontology</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/Elysheba/DODO" target="_blank">https://github.com/Elysheba/DODO</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PGConverter" target="_blank">PGConverter</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This library is primarily to offer a command-line tool to convert between the following formats for proteogenomics related data. It is also possible to convert regular non-proteogenomics mzIdentML or PRIDE XML files to mzTab. A secondary function of this library is to gather metadata and statistics about mzIdentML, mzTab, and PRIDE XML files for validation purposes. Primarily this tool is powered by the ms-data-core-api library to read/write files. Primarily this tool is powered by the ms-data-core-api library to read/write files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : mzidentml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : mztab</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/PRIDE-Toolsuite/PGConverter" target="_blank">https://github.com/PRIDE-Toolsuite/PGConverter</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/EK-DRD" target="_blank">EK-DRD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Comprehensive Database for Drug Repositioning Inspired by Experimental Knowledge | Drug Repositioning Drug repositioning, or the identification of new indications for approved therapeutic drugs, has gained substantial traction with both academics and pharmaceutical companies because it reduces the cost and duration of the drug development pipeline and the likelihood of unforeseen adverse events. To date there has not been a systematic effort to identify such opportunities, in part because of the lack of a comprehensive resource for an enormous amount of unsystematic drug repositioning information to support scientists who could benefit from this endeavor | Experimental Knowledge-Based Drug Repositioning Database (EK-DRD) provides a versatile resource for the display, search and analysis of structure, function, pathway, disease, network and related experimentally-validated drug repositioning annotation for 1861 FDA-approved and 102 withdrawn small molecule drugs</div></td>
      <td><div class='topic short click_expand'><ul><li>Drug discovery</li><li>Preclinical and clinical studies</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phasing</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/orthodisease" target="_blank">Orthodisease</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of disease gene orthologs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/diana-mirgen" target="_blank">DIANA-miRGen</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides accurate cell-line-specific miRNA gene transcription start sites, coupled with genome-wide maps of transcription factor binding sites in order to unveil the mechanisms of miRNA transcription regulation. The database schema and web interface facilitates user interaction, provides advanced queries and innate connection with other DIANA resources for miRNA target identification and pathway analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene structure</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>372</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hora" target="_blank">HorA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>HorA (Homology or Analogy) is a web server that identifies likely homologs for a given query protein structure. HorA combines sequence information with structure information from spatial similarity measures.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Protein structure analysis</li><li>Molecular modelling</li><li>Structure prediction</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structural motif recognition</li><li>Protein modelling</li><li>Sequence similarity search</li><li>Protein structure comparison</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/emap" target="_blank">Edinburgh Mouse Atlas EMAP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A digital atlas of mouse embryonic development consisting of a database of the anatomical structures that are found at different stages of mouse embryo development, and a set of 3D volumetric digital embryo models at different stages of mouse development. It creates interactive three-dimensional computer models of mouse embryos at successive stages of development with defined anatomical domains linked to a stage-by-stage ontology of anatomical names.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Anatomy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>fate map : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>74</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/subs2vec" target="_blank">subs2vec</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Word embeddings from subtitles in 55 languages.<br>Van Paridon & Thompson (2019) introduces pretrained embeddings and precomputed word/bigram/trigram frequencies in 55 languages. The files can be downloaded from the links in this table. Word vectors trained on subtitles are available, as well as vectors trained on Wikipedia, and a combination of subtitles and Wikipedia (for best predictive performance).<br>This repository contains the subs2vec module, a number of Python 3.7 scripts and command line tools to evaluate a set of word vectors on semantic similarity, semantic and syntactic analogy, and lexical norm prediction tasks. In addition, the subs2vec.py script will take an OpenSubtitles archive or Wikipedia and go through all the steps to train a fastText model and produce word vectors as used in the paper associated with this repository.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Imaging</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/data-Mining-pubmed" target="_blank">Data Mining of PubMed</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Visualisation and data mining of the public database of biomedical literature PubMed</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Natural language processing</li><li>Mathematics</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/reyesaldasoro/PubMed-Data-Mining-And-Visualisation" target="_blank">https://github.com/reyesaldasoro/PubMed-Data-Mining-And-Visualisation</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/panorama" target="_blank">Panorama</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web application for storing, sharing, analyzing, and reusing targeted assays created and refined with Skyline,1 an increasingly popular Windows client software tool for targeted proteomics experiments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Spectral library search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>118</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cygd" target="_blank">Comprehensive yeast genome database (CYGD)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Information on the molecular structure and functional network of the entirely sequenced, well-studied model eukaryote, the budding yeast Saccharomyces cerevisiae. In addition the data of various projects on related yeasts are used for comparative analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Comparative genomics</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>238</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CytoConverter" target="_blank">CytoConverter</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web-based tool to convert karyotypes to genomic coordinates | Sequence-Cytogenetic Bulk Conversion | Bulk Sequence - Cytogenetic Conversion Service | Use this page to get cytogenetic locations for features or sequence locations, or get sequence locations for cytogenetic locations | Organism: Drosophila melanogaster Homo sapiens Mus musculus Rattus norvegicus</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Sequencing</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://www.ncbi.nlm.nih.gov/genome/tools/cyto_convert/" target="_blank">https://www.ncbi.nlm.nih.gov/genome/tools/cyto_convert/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/plan2l" target="_blank">PLAN2L</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web based tool that integrates text mining and information extraction techniques to access useful information for analyzing genetic, cellular and molecular aspects of Arabidposis thaliana.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Natural language processing</li><li>Molecular interactions, pathways and networks</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Text mining</li><li>Named-entity and concept recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>26</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genbank" target="_blank">GenBank nucleotide sequence database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive database of public nucleotide sequences for more than 300 000 organisms named, from small and large-scale sequencing projects. Data exchange with the European Molecular Biology Laboratory Nucleotide Sequence Database in Europe and the DNA Data Bank of Japan ensures worldwide coverage.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence record : genbank-html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,562</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/crimson" target="_blank">CRIMSON</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Crimson (cardiovascular integrated modelling & simulation) is an advanced simulation environment for subject-specific hemodynamic analysis. Crimson is integrating best-in-class open source-solutions for parallel flow solvers, fluid-structure interactions, GUI-based boundary condition specification, data assimilation, medical image processing, mesh generation, transitional hemodynamics, and much more.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Medical imaging</li><li>Molecular modelling</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Editing</li><li>Modelling and simulation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/carthurs/CRIMSONFlowsolver" target="_blank">https://github.com/carthurs/CRIMSONFlowsolver</a></div></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Loxodonta_Localizer" target="_blank">Loxodonta Localizer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Software Tool for Inferring the Provenance of African Elephants and Their Ivory Using Mitochondrial DNA.<br>git clone https://github.com/kaizhao86/loxodontalocalizer.<br>Loxodonta Localizer maps the localities from which mitochondrial DNA sequences have been reported for African elephants, Loxodonta africana and L. cyclotis, using the dataset described in Ishida et al., 2013, updated with additional sequences.<br>#first, install Python 2.7, pip and git</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Literature and language</li><li>Probes and primers</li><li>Mapping</li><li>DNA polymorphism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence trimming</li><li>Haplotype mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/kaizhao86/loxodontalocalizer" target="_blank">https://github.com/kaizhao86/loxodontalocalizer</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/promex" target="_blank">ProMEX protein mass spectral library</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A mass spectral library consisting of tryptic peptide product ion spectra generated by liquid chromatography coupled to ion trap mass spectrometry (LC-ITMS). Developed using samples derived from Arabidopsis thaliana and Medicago truncatula. The database serves as a reference and can be used for protein identification in uncharacterized samples. Protein identification is linked to other molecular levels of biological organization such as metabolite, pathway and transcript data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein identification</li><li>Peptide identification</li><li>Spectral library search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : mzxml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : png</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>32</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rpx" target="_blank">rpx</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package implements an interface to proteomics data submitted to the ProteomeXchange consortium.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,780</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/lgatto/rpx" target="_blank">https://github.com/lgatto/rpx</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pmap-cutdb" target="_blank">PMAP-CutDB proteolytic event database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Curated proteolytic events and pathways from the literature.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data reference : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>44</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nibr_2d-page" target="_blank">NIBR 2D-PAGE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>World-wide gel-based proteomics database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>experiment report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>30</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cohorts" target="_blank">cohorts</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Package for standardized and reproducible processing, analysis, and integration of clinical proteomics data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Biomarkers</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Read binning</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ngiangre/cohorts/issues" target="_blank">https://github.com/ngiangre/cohorts/issues</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/DAStk_-_Differential_ATAC-seq_toolkit" target="_blank">DAStk - Differential ATAC-seq toolkit</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Differential ATAC-seq Toolkit (DAStk) is a set of command-line tools to aid analyzing differential ATAC-seq data. By leveraging changes in accessible chromatin, we can detect significant changes in transcription factor (TF) activity. This is a simple but powerful tool for cellular perturbation analysis. In fact, the applications of DAStk are not necessarily limited to ATAC-seq, but can extend to comparing any pair of bedGraphs containing regions of interest, e.g. transcription regulatory elements (TRE) from nascent transcription assays like PRO-seq, or others like ChIP-seq peaks. To aid downstream analysis, DAStk also includes a graph that combines all human proteins in Uniprot, with their respective annotations, and all human Reactome data on biological pathways, biochemical reactions, and complex formation. This allows the user to query for existing knowledge about what aspects of the significantly changing TFs are shared among two or more of them.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Differential binding analysis</li><li>Gene regulatory network prediction</li><li>Differential protein expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/blur" target="_blank">BLUR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BLUR (BLast Unexpected Ranking) is a tool designed to highlight, on the whole proteome level, protein divergences between species that result from divergence or loss of a domain and/or motif in a specific taxon. It is based on precomputed BLAST searches for a variety of model organism queries, allowing the study of all major life groups.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Genotype and phenotype</li><li>Protein interactions</li><li>Evolutionary biology</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Database search</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/clinvar" target="_blank">ClinVar</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A freely accessible, public archive of reports of relationships among medically important variants and phenotypes hosted by the NCBI. It integrates and cross-references data from multiple databases at NCBI. In addition to dbSNP and dbVar, it depends on MedGen to represent phenotype, Gene to represent genes, and on human RefSeqs to represent the location of sequence variation. Each record represents the submitter, the variation and the phenotype.</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical informatics</li><li>Genotype and phenotype</li><li>Genetic variation</li><li>Human biology</li><li>Rare diseases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Aggregation</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,222</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/multi_experiment_matrix" target="_blank">Multi Experiment Matrix</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web based tool for performing co-expression queries over large collections of gene expression experiments. It provides access to several hundreds of publicly available gene expression datasets of different tissues, diseases and conditions, arranged by the species and microarray platform types. The main user input is the name or ID of a query gene (various syononyms are accepted) and the choice of the collection of microarray experiments to be used in the co-expression query.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional genomics</li><li>Data integration and warehousing</li><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression clustering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>424</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/uniprot.ws" target="_blank">UniProt.ws</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A collection of functions for retrieving, processing and repackaging the UniProt web services.</div></td>
      <td><div class='topic short click_expand'><ul></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein sequence : uniprotkb format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,247</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/jaspextract" target="_blank">jaspextract</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Extract data from JASPAR.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mirmine" target="_blank">miRmine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MiRmine database is a collection of expression profiles from different publicly available miRNA-seq datasets, Panwar et al (2017) miRmine: A Database of Human miRNA Expression, prepared with this data package as RangedSummarizedExperiment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/miRmine/" target="_blank">http://bioconductor.org/packages/miRmine/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/colocr" target="_blank">colocr</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R package for conducting co-localization analysis on fluorescence microscopy images.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ropensci/colocr/issues" target="_blank">https://github.com/ropensci/colocr/issues</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pair" target="_blank">PAIR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Experimentally reported molecular interactions and predicted interactions in A. thaliana. It predicts interactions by a fine-tuned support vector machine model that integrates indirect evidences for interaction, such as gene co-expressions, domain interactions, shared GO annotations, co-localizations, phylogenetic profile similarities and homologous interactions in other organisms. It features a user-friendly query interface, providing rich annotation on the relationships between two proteins.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li><li>Model organisms</li><li>Protein interaction experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network comparison</li><li>Pathway or network prediction</li><li>Protein interaction network analysis</li><li>Protein interaction prediction</li><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>47</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metacoder" target="_blank">Metacoder</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Package for easily parsing, manipulating, and graphing publication-ready plots of hierarchical data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li><li>Taxonomy</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree generation</li><li>Phylogenetic tree visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>ecological data : biodiversity data format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>phylogenetic tree : image format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>113</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/JUDI" target="_blank">JUDI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Bioinformatics Pipeline using JUDI.<br>This repository contains the source code of JUDI, a workflow management system for developing complex bioinformatics software with many parameter settings. Bioinformatics pipeline: Just Do It!.<br>JUDI - Bioinformatics Pipeline: Just Do It.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Database management</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mass-up" target="_blank">Mass-Up</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Open-source mass spectrometry software for proteomics designed to support the preprocessing and analysis of MALDI-TOF mass spectrometry data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Biomarkers</li><li>Statistics and probability</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Clustering</li><li>Standardisation and normalisation</li><li>Validation of peptide-spectrum matches</li><li>Peak detection</li><li>Classification</li><li>Data handling</li><li>Heat map generation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide mass fingerprint : csv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide mass fingerprint : csv</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>53</div></td>
      <td><div class="link short click_expand"><a href="http://www.sing-group/mass-up/download" target="_blank">http://www.sing-group/mass-up/download</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/kssd" target="_blank">kssd</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sequences Dimensionality-Reduction by K-mer Substring Space Sampling Enables Effective Resemblance- and Containment-Analysis for Large-Scale omics-data | K-mer substring space decomposition | Kssd: K-mer substring space sampling/shuffling Decomposition | Kssd is a command-line tool for large-scale sequences sketching and resemblance- and containment-analysis. It sketches sequences by k-mer substring space sampling/shuffling, please see Methods part of our preprint (https://www.biorxiv.org/content/10.1101/729665v1) for how it works. It handles DNA sequences of both fasta or fastq format, whether gzipped or not. Kssd run on linux system, currently not support for MacOS and Windows OS</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenomics</li><li>Metagenomics</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>k-mer counting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/LaMMA" target="_blank">LaMMA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The open data geoportal of the Lamma Consortium.<br>In questa categoria sono immagazzinati i dati relativi a bollettini,...<br>In attuazione del decreto legge D. Lgs n. 33 del 14 marzo 2013, che sancisce...</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://geoportale.lamma.rete.toscana.it/MapStore/public/" target="_blank">http://geoportale.lamma.rete.toscana.it/MapStore/public/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bindingdb" target="_blank">BindingDB database of measured binding affinities</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of measured binding affinities, focusing chiefly on the interactions of protein considered to be drug-targets with small, drug-like molecules.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein-ligand interaction report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>989</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scaleqc" target="_blank">ScaleQC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Scalable Lossy to Lossless Solution for NGS Sequencing Data Compression.<br>Per-base quality values in NGS sequencing data take a significant portion of storage even after compression. Lossy compression technologies could further reduce the space used by quality values. However, in many applications lossless compression is still desired. Hence, sequencing data in multiple file formats have to be prepared for different applications. Results We developed a scalable lossy to lossless compression solution for quality values named ScaleQC. ScaleQC is able to provide bit-stream level scalability. More specifically, the losslessly compressed bit-stream by ScaleQC can be further truncated to lower data rates without re-encoding. Despite its scalability, ScaleQC still achieves same or better compression performance at both lossless and lossy data rates compared to the state-of-the-art lossless or lossy compressors</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Splitting</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/causalmgm" target="_blank">CausalMGM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CausalMGM is a data analysis tool to explore large, complex datasets. The method learns a graphical model of the data where the nodes are variables and edges display the dependencies among variables. The graphical model allows users to query their data to find the direct influences of a target variable of interest, or to find novel associations between pairs of variables.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Oncology</li><li>Genetics</li><li>Personalised medicine</li><li>Data mining</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Relation extraction</li><li>Regression analysis</li><li>Feature selection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MTBGT" target="_blank">MTBGT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MTB Genome to Test (MTBGT) tool transforms WGS-derived data into laboratory-validated results of the primary RDTs â€“ Xpert MTB/RIF, XpertMTB/RIF Ultra, GenoType MDRTBplus v2.0, and GenoscholarNTM+MDRTB II.</div></td>
      <td><div class='topic short click_expand'><ul><li>Whole genome sequencing</li><li>DNA polymorphism</li><li>Probes and primers</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/KamelaNg/MTBGT/issues" target="_blank">https://github.com/KamelaNg/MTBGT/issues</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/prints" target="_blank">PRINTS protein motif fingerprint database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A compendium of protein fingerprints, or group of conserved motifs used to characterise a protein family.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Proteins</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein signature : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>84</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/TabSINT" target="_blank">TabSINT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Open-source mobile software for distributed studies of hearing.<br>TabSINT is an open source platform for administering tablet based hearing-related exams, as well as general-purpose questionnaires. It is meant to be flexible, easy-to-use, and useful for administrators who manage studies of all sizes.<br>Open source platform for hearing assessments,and general purpose questionnaires</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Medical informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://gitlab.com/creare-com/tabsint" target="_blank">https://gitlab.com/creare-com/tabsint</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bio-formats" target="_blank">Bio-Formats</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Bio-Formats is a standalone Java library for reading and writing life sciences image file formats. It is capable of parsing both pixels and metadata for a large number of formats, as well as writing to several formats. See the table below for a complete list (click the headers to sort, and format names to see all information).</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/bio-formats-4-2-2-java-library-reading-writing-biology-image-file-formats.html" target="_blank">http://www.mybiosoftware.com/bio-formats-4-2-2-java-library-reading-writing-biology-image-file-formats.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/riddle" target="_blank">RIDDLE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A network-based method for characterizing gene sets. It asks if an input query set is significantly “close” to a known pathway or disease set in the human functional network.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene functional annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/riddle-network-assisted-gene-set-analysis.html" target="_blank">http://www.mybiosoftware.com/riddle-network-assisted-gene-set-analysis.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mirconnx" target="_blank">mirConnX</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web server for inferring, displaying and parsing mRNA and microRNA gene regulatory networks. It combines sequence information with gene expression data analysis to create a disease specific, genome wide regulatory network. Only supports human and mouse.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Functional, regulatory and non-coding RNA</li><li>Gene expression</li><li>Gene transcripts</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression correlation analysis</li><li>Pathway or network prediction</li><li>Gene expression profiling</li><li>Gene regulatory network analysis</li><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>91</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/geneva" target="_blank">GENEVA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mining transcriptomics and clinical data reveals ACE2 expression modulators and identifies cardiomyopathy as a risk factor for mortality in COVID-19 patients.<br>GENEVA allows you to identify RNA-sequencing datasets from the Gene Expression Omnibus (GEO) that contains conditions modulating a gene or a gene signature. See the Help page fore more detail.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Infectious disease</li><li>Cardiology</li><li>Epigenetics</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Expression analysis</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/NavchetanKaur/geneva-webtool" target="_blank">https://github.com/NavchetanKaur/geneva-webtool</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pdb-blast" target="_blank">PDB-BLAST</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Two step protocol which uses BLAST to identify putative protein structures similar to a query sequence.</div></td>
      <td><div class='topic short click_expand'><ul><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Structure database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/treeio" target="_blank">treeio</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Base classes and functions for parsing and exporting phylogenetic trees. This package supports parsing analysis findings from commonly used software packages, allows linking external data to phylogeny and merging tree data obtained from different sources. It also supports exporting phylogenetic tree with heterogeneous associated data to a single tree file.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>15</div></td>
      <td><div class="link short click_expand"><a href="https://guangchuangyu.github.io/software/treeio" target="_blank">https://guangchuangyu.github.io/software/treeio</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ribodipa" target="_blank">RiboDiPA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Novel tool for differential pattern analysis in Ribo-seq data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Protein expression</li><li>Gene transcripts</li><li>Gene expression</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Differential gene expression analysis</li><li>Parsing</li><li>Quantification</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/admixr" target="_blank">admixr</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides a convenient interface for performing reproducible population genetic analyses as implemented by command-line programs in the ADMIXTOOLS software suite.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>DNA polymorphism</li><li>Genotype and phenotype</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/flybase" target="_blank">FlyBase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database and web portal for genetic and genomic information on fruit fly Drosophila melanogaster and related fly species.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Experimental design and studies</li><li>Genomics</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Splitting</li><li>Data handling</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>290</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/homolens" target="_blank">HOMOLENS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of homologous genes from Ensembl organisms and Ensembl families, structured under ACNUC sequence database management system.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Phylogeny</li><li>Gene and protein families</li><li>Evolutionary biology</li><li>Genomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>95</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/homolens-r5-homologous-sequences-ensembl-animal-genomes.html" target="_blank">http://www.mybiosoftware.com/homolens-r5-homologous-sequences-ensembl-animal-genomes.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GRANVIL" target="_blank">GRANVIL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>GRANVIL (Gene- or Region-based ANalysis of Variants of Intermediate and Low frequency) - performs rare-variant analysis of binary or quantitative phenotypes. The method is based on accumulation of minor alleles of rare or uncommon markers discovered through dense genotyping or resequencing data. Association analyses are based on gene- or other pre-defined regions, determined by analyst.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Sequencing</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Parsing</li><li>Collapsing methods</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pymeshsim" target="_blank">pyMeSHSim</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrative python package for biomedical named entity recognition, normalization, and comparison of MeSH terms.</div></td>
      <td><div class='topic short click_expand'><ul><li>GWAS study</li><li>Genotype and phenotype</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Standardisation and normalisation</li><li>Parsing</li><li>Named-entity and concept recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CHOmine" target="_blank">CHOmine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated database for Cricetulus griseus and CHO cells</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pubmed_central" target="_blank">PubMed Central</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>U.S. National Institutes of Health (NIH) free digital archive of biomedical and life sciences journal literature.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Biomedical science</li><li>Probes and primers</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Data retrieval</li><li>Database search</li><li>Genotyping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>373</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/plasmodb" target="_blank">Plasmodium genome resources (PlasmoDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomic-scale dataset associated with the eukaryotic pathogens Plasmodium.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>661</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bionimbus" target="_blank">Bionimbus</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Cloud environment accessing, managing and analyzing large genomics and phenotypic datasets.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Data governance</li><li>Data integration and warehousing</li><li>Data architecture, analysis and design</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Workbench</li></ul></div></td>
      <td><div class='citations short click_expand'>52</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tofsims" target="_blank">tofsims</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This packages offers a pipeline for import, processing and analysis of ToF-SIMS 2D image data. Import of Iontof and Ulvac-Phi raw or preprocessed data is supported. For rawdata, mass calibration, peak picking and peak integration exist. General funcionality includes data binning, scaling, image subsetting and visualization. A range of multivariate tools common in the ToF-SIMS community are implemented (PCA, MCR, MAF, MNF).</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular biology</li><li>Imaging</li><li>Biophysics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Mass spectrum visualisation</li><li>Molecular surface analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MSDB" target="_blank">MSDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive, annotated database of microsatellites.<br>Multiple Loci VNTR Analysis databases and software.<br>Multi Locus VNTR Analysis (MLVA) databases and software.<br>MICAS 3.0, Microsatellite Analysis Server is the online repository of All Bacterial Microsatellites.<br>MICAS: Microsatellite Analysis Server 3.0.<br>An Online Repository of All Prokaryotic Microsatellites.<br>with a focus on Viruses of Microbes and Molecular Epidemiology of bacterial pathogens.<br>CRISPRCasdb: database for CRISPRs and associated Cas genes.<br>MICAS: Microsatellite Analysis Server.</div></td>
      <td><div class='topic short click_expand'><ul><li>Public health and epidemiology</li><li>Database management</li><li>Mapping</li><li>DNA polymorphism</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Multilocus sequence typing</li><li>Phylogenetic tree generation (from polymorphism data)</li><li>Repeat sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="http://www.mcr.org.in/micas/" target="_blank">http://www.mcr.org.in/micas/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mir-tv" target="_blank">miR-TV</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An interactive microRNA Target Viewer for microRNA and target gene expression interrogation for human cancer studies.<br>MicroRNAs (miRNAs) have been found in many organisms and play essential roles in gene expression regulation of many critical cellular processes, including human cancers.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Oncology</li><li>Gene transcripts</li><li>Functional, regulatory and non-coding RNA</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>miRNA target prediction</li><li>miRNA expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/consent" target="_blank">Consent</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Ligand-based virtual screening with consensus queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li><li>Compound libraries and screening</li><li>Protein-ligand interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Virtual ligand screening</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/world-2dpage" target="_blank">The world-2DPAGE database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A public repository, standards compliant, for proteomics image data published in the literature.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>experiment report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>30</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phenoSEED" target="_blank">phenoSEED</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PhenoSEED is a tool for phenotyping of seed samples on an individual seed basis. Originally designed for the BELT system, it includes functions for colour calibration and morphological, colour and patterning analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Data acquisition</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Image annotation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GBOL" target="_blank">GBOL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Empusa code generator and its application to GBOL.<br>GBOL; Genome Biology Ontology Language.<br>To enable interoperability of genome annotations, we have developed the Genome Biology Ontology Language (GBOL) and associated stack (GBOL stack).<br>A group is a collection of several projects.<br>If you organize your projects under a group, it works like a folder.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Cell biology</li><li>Genomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome annotation</li><li>Query and retrieval</li><li>Ontology visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Ontology</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="http://www.gitlab.com/GBOL" target="_blank">http://www.gitlab.com/GBOL</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/oi_gene" target="_blank">Osteogenesis imperfecta collagen mutation</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>All published accounts of variants resulting in osteogenesis imperfecta. Such variants occur in the COL1A1, COL1A2, CRTAP and LEPRE1 genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence variations : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>133</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/propsearch" target="_blank">PROPSEARCH</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PROPSEARCH was designed to detect functional and / or structural homologs, if the sequence identity is below about 25%. It uses the amino acid physico-chemical properties, content of bulky residues, content of small residues, average hydrophobicity, average charge a.s.o. and are used as query vector. Sequences in the database are transformed into vectors as well, and the euclidian distance between the query and database sequences is calculated.</div></td>
      <td><div class='topic short click_expand'><ul><li>Function analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein hydrophobic region calculation</li><li>Sequence composition calculation</li><li>Sequence alignment</li><li>Sequence motif recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>158</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/BRM" target="_blank">BRM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Systems biology web tool for microRNA and omics data integration.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Proteomics</li><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Parsing</li><li>miRNA target prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_GeneView" target="_blank">IMGT GeneView</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT® tool providing the view of a gene in IMGT/LocusView. Display is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunogenetics</li><li>Immunoinformatics</li><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>map : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>354</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/utrdb" target="_blank">UTRdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of Untranslated regions (UTRs).</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular biology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>232</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aftol" target="_blank">Assembling the Fungal Tree of Life (AFTOL)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Fungal structural and biochemical database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>organism report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>529</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/flyfactorsurvey" target="_blank">FlyFactorSurvey</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of DNA binding specificities for Drosophila transcription factors determined using the bacterial one-hybrid system. The database provides to recognition motifs and position weight matrices for TFs. Search tools and flat file downloads are provided to retrieve binding site information for individual TFs, groups of TFs or for all TFs with characterized binding specificities.</div></td>
      <td><div class='topic short click_expand'><ul><li>Nucleic acid sites, features and motifs</li><li>Transcription factors and regulatory sites</li><li>DNA binding sites</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence motif recognition</li><li>Transcription factor binding site prediction</li><li>Query and retrieval</li><li>Data retrieval</li><li>Transcriptional regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>132</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PerMemDB" target="_blank">PerMemDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database for eukaryotic peripheral membrane proteins.<br>The majority of all proteins in cells interact with membranes either permanently or temporarily. Peripheral membrane proteins form transient complexes with membrane proteins and or lipids, via non-covalent interactions and are of outmost importance, due to numerous cellular functions in which they participate. In an effort to collect data regarding this heterogeneous group of proteins we designed and constructed a database, called PerMemDB. PerMemDB is currently the most complete and comprehensive repository of data for eukaryotic peripheral membrane proteins deposited in UniProt or predicted with the use of MBPpred - a computational method that specializes in the detection of proteins that interact non-covalently with membrane lipids, via membrane binding domains. The first version of the database contains 231,770 peripheral membrane proteins from 1009 organisms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Lipids</li><li>Literature and language</li><li>Model organisms</li><li>Membrane and lipoproteins</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Protein interaction prediction</li><li>Fold recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dat2mgf" target="_blank">dat2mgf</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Converts Mascot results files back to MGF.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : mascot .dat file</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : mgf</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/smart" target="_blank">Simple modular architecture research tool (SMART)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>SMART protein domain database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein structure report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,014</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ensembl" target="_blank">Ensembl</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genome databases for vertebrates genomes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Rare diseases</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Genome visualisation</li><li>Deposition</li><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>4,510</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/i-lbr" target="_blank">I-LBR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Identification of ligand-binding residues using protein sequence profile alignment and query-specific support vector machine model.<br>Information embedded in ligand-binding residues (LBRs) of proteins is important for understanding protein functions. How to accurately identify the potential ligand-binding residues is still a challenging problem, especially only protein sequence is given. In this paper, we establish a new query-specific computational method, named I-LBR, for the identification of LBRs without directly using the information of protein 3D structure. I-LBR includes two modes, named as I-LBRGP and I-LBRLS, for the general-purpose and ligand-specific LBR identification</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Proteins</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence profile alignment</li><li>Protein identification</li><li>Expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/memtype-2l" target="_blank">MemType-2L</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>2-layer predictor for predicting membrane protein types, the first layer will predict whether the query sequence belongs to membrane proteins or not; and the second layer aims to predict exactly the membrane protein types when the output of the first layer is “membrane proteins”.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structure analysis</li><li>Proteins</li><li>Membrane and lipoproteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>401</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/memtype-2l-predicting-membrane-protein-types.html" target="_blank">http://www.mybiosoftware.com/memtype-2l-predicting-membrane-protein-types.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gene_set_builder" target="_blank">Gene Set Builder</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database-driven, web-based tool designed to help researchers compile, store, export, and share sets of genes. This application supports the 17 eukaryotic genomes found in version 32 of the Ensembl database, which includes species from yeast to human. User-created information such as sets and customized annotations are stored and can be shared to facilitate easy access.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Database management</li><li>Human biology</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene functional annotation</li><li>Gene prediction</li><li>Gene set testing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ncigraph" target="_blank">NCIgraph</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides various methods to load the pathways from the NCI Pathways Database in R graph objects and to re-format them.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/s2rsldb" target="_blank">S2RSLDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sigma-2 receptor selective Ligands Database is an online ligand database consisting of the Sigma-2 selective receptor ligand published since 1993. The S2RSLDB is based on structural 2D information as well as pharmacological and computed physicochemical properties and experimental procedures, retrieved from the literature.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li><li>Immunoproteins, genes and antigens</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cdd" target="_blank">Conserved domain database CDD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Resource for the annotation of protein sequences with the location of conserved domain footprints, and functional sites inferred from these footprints. It includes manually curated domain models that make use of protein 3D structure to refine domain models and provide insights into sequence/structure/function relationships.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structural motifs and surfaces</li><li>Sequence analysis</li><li>Gene and protein families</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment (protein) : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>4,146</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/knetminer" target="_blank">KnetMiner</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive approach for supporting evidence-based gene discovery and complex trait analysis across species.<br>KnetMiner is a web application to search and visualise genome-scale knowledge networks. It is used by scientists in academia and industry to accelerate gene-trait discovery research. KnetMiner offers features such as keyword, gene list and genome region search, query refinement, gene ranking, gene set enrichment analysis and interactive visualisation of knowledge networks and genomic information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Plant biology</li><li>Data mining</li><li>Literature and language</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Network visualisation</li><li>Data retrieval</li><li>Database search</li><li>Genetic mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/Rothamsted/knetminer" target="_blank">https://github.com/Rothamsted/knetminer</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biologicalnetworks" target="_blank">BiologicalNetworks</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Allows retrieval, construction, and visualization of biological networks and allows for the analysis of high-throughput expression data by mapping onto network graphs. It queries the PathSys system, a combined database of biological pathways, gene regulatory networks, and protein interaction maps.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network analysis</li><li>Pathway or network prediction</li><li>Gene regulatory network analysis</li><li>Mapping</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>113</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ramigo" target="_blank">RamiGO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R interface sending requests to AmiGO visualize, retrieving DAG GO trees, parsing GraphViz DOT format files and exporting GML files for Cytoscape. Also uses RCytoscape to interactively display AmiGO trees in Cytoscape.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Ontology visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>23</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tuberculist" target="_blank">TubercuList</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Knowledge base integrates genome details, protein information, drug and transcriptome data, mutant and operon annotation, bibliography, structural views and comparative genomics, in a structured manner required for the rational development of new diagnostic, therapeutic and prophylactic measures against tuberculosis. With the means of expert curation and continuous updates, we deliver a broad view of the Mycobacterium tuberculosis genome.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>277</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lncrnadb" target="_blank">lncRNAdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database containing a comprehensive list of long noncoding RNAs (lncRNAs) that have been shown to have, or to be associated with, biological functions in eukaryotes, as well as messenger RNAs that have regulatory roles. Each entry contains referenced information about the RNA, including sequences, structural information, genomic context, expression, subcellular localization, conservation, functional evidence and other relevant information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Functional, regulatory and non-coding RNA</li><li>Gene expression</li><li>RNA</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Transcriptional regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>424</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iedb-3d" target="_blank">IEDB-3D</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The 3D structural component of the Immune Epitope Database (IEDB). It catalogs B- and T-cell epitopes and Major Histocompatibility Complex (MHC) ligands for which 3D structures of complexes with antibodies, T-cell receptors or MHC molecules are available in the Protein Data Bank (PDB).</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Immunoproteins, genes and antigens</li><li>Immunology</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein property calculation</li><li>Structural similarity search</li><li>Epitope mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>41</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/disease_ontology" target="_blank">DO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Disease Ontology (DO) has been developed as a standardized ontology for human disease with the purpose of providing the biomedical community with consistent, reusable and sustainable descriptions of human disease terms, phenotype characteristics and related medical vocabulary disease concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical informatics</li><li>Molecular interactions, pathways and networks</li><li>Pathology</li><li>Ontology and terminology</li><li>Rare diseases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li><li>Ontology comparison</li><li>Query and retrieval</li><li>Ontology visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>471</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-1.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/teamtat" target="_blank">TeamTat</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Text Annotation Tool for Team Collaboration.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Literature and language</li><li>Medicine</li><li>Database management</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Data retrieval</li><li>Image annotation</li><li>Anonymisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ncbi-nlp/TeamTat" target="_blank">https://github.com/ncbi-nlp/TeamTat</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/colorgff" target="_blank">colorgff</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Parses a DOOR report file and writes the information in a gff3 out file.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : gff</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : gff3</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>934</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/urgi_tools/colorGff/1.0.0" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/urgi_tools/colorGff/1.0.0</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PSMD" target="_blank">PSMD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An extensive database for pan-species microsatellite investigation and marker development.<br>A Django app for Pan-Species Microsatellite Database.<br>Pan-Species Microsatellite Database.<br>Microsatellites (simple repeat sequences, SSRs or short tandem repeats, STRs), are tandem repeats of short DNA motifs with length of 1-6 bp which have extensively been exploited as powerful genetic markers for diversity applications, and are involved in gene regulation as well as implicated in numerous genetic diseases even in cancers. PSMD was developed as a web-based comprehensive database comprising 678,106,741 perfect microsatellites and 43,848,943 compound microsatellites from 18,408 organisms covered almost all species with available genomic data. PSMD enables scientists to easily browse, filter, compare, download SSRs with relevant information</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Database management</li><li>Probes and primers</li><li>DNA polymorphism</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>PCR primer design</li><li>Database search</li><li>Repeat sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="http://psmd.biosv.com" target="_blank">http://psmd.biosv.com</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/AsCRISPR" target="_blank">AsCRISPR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web server for allele-specific sgRNA design in precision medicine | Abstract Allele-specific targeting by CRISPR provides a point of entry for personalized gene therapy of dominantly inherited diseases, by selectively disrupting the mutant alleles or disease-causing single nucleotide polymorphisms (SNPs), ideally while leaving normal alleles intact. Despite unprecedented specificity and tremendous therapeutic utility of allele-specific targeting by CRISPR, few bioinformatic tools have been implemented for the allele-specific purpose. We thus developed AsCRISPR ( A llele- s pecific CRISPR ), a web tool to aid the design of guide sequences that can discriminate between alleles. It can process with query sequences harboring single-base or short insertion-deletion (indel) mutations, as well as heterozygous SNPs deposited in the dbSNP database. Multiple CRISPR nucleases and their engineered variants including newly-developed Cas12b and CasX are included for users’ choice</div></td>
      <td><div class='topic short click_expand'><ul><li>Geriatric medicine</li><li>DNA polymorphism</li><li>Personalised medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/branchpointer" target="_blank">Branchpointer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Predicts branchpoint probability for sites in intronic branchpoint windows. Queries can be supplied as intronic regions; or to evaluate the effects of mutations, SNPs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li><li>Machine learning</li><li>Human biology</li><li>Sequence analysis</li><li>DNA polymorphism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Splicing model analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Bio2BEL" target="_blank">Bio2BEL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integration of Structured Biological Data Sources using Biological Expression Language.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Cheminformatics</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Expression profile pathway mapping</li><li>Enrichment analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bio2bel/bio2bel" target="_blank">https://github.com/bio2bel/bio2bel</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/codon_usage_database" target="_blank">Codon Usage Database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Find GC content and frequency of codon usage for any organism that has a sequence in GenBank.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li><li>Model organisms</li><li>Sequence analysis</li><li>Experimental design and studies</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Codon usage table comparison</li><li>Codon usage fraction calculation</li><li>Codon usage table generation</li><li>Codon usage analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>765</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Aneuvis" target="_blank">Aneuvis</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based exploration of numerical chromosomal variation in single cells.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Whole genome sequencing</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Quantification</li><li>Copy number estimation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/dpique/aneuvis" target="_blank">https://github.com/dpique/aneuvis</a></div></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/3d-partner" target="_blank">3D-partner</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool to predict interacting partners and binding models of a query protein sequence through the analysis of structural complexes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence composition, complexity and repeats</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li><li>Structure prediction</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein sequence analysis</li><li>Protein structure prediction</li><li>Protein structure analysis</li><li>Protein binding site prediction</li><li>Protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>42</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/flymine" target="_blank">FlyMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated database for Drosophila and Anopheles genomics.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>384</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aaindexextract" target="_blank">aaindexextract</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Extract amino acid property data from AAINDEX.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Plastid_Genome_Annotator" target="_blank">Plastid Genome Annotator</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Software package for rapid, accurate, and flexible batch annotation of plastomes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Plant biology</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>137</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mitobank" target="_blank">MitoBank</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MitoBank is a simple perl script to obtain from GeneBank mitochondrial genomes, parse them, and store the different genes in separate files. It can be used to retrieve both coding genes and RNA genes. Coding genes might be obtained in their amino acid or nucleotide sequence.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>113</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mitobank-2-1-automatic-retrieval-parsing-mitochondrial-genomes.html" target="_blank">http://www.mybiosoftware.com/mitobank-2-1-automatic-retrieval-parsing-mitochondrial-genomes.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/omim" target="_blank">OMIM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Online Mendelian inheritance in Man (OMIM) - catalog of human genes and genetic disorders with descriptive text, references, and links to many NCBI resources including GenBank and PubMed.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Genotype and phenotype</li><li>Genetic variation</li><li>Rare diseases</li><li>Human genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Aggregation</li><li>Query and retrieval</li><li>Genetic mapping</li><li>Service invocation</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,595</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/squamatabase" target="_blank">SquamataBase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A natural history database and R package for comparative biology of snake feeding habits.<br>Squamatabase is a database of prey items recorded in diet samples from the world's snakes, compiled by me, during my time as a PhD student at the University of Michigan.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sample collections</li><li>Biodiversity</li><li>Biological databases</li><li>Taxonomy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Query and retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/DORMAN" target="_blank">DORMAN</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database Of Reconstructed MetAbolic Networks.<br>We apologize for the inconvenience, but it appears that an internal error has occurred in the Dorman system. If this is the first time you have encountered this message, please use the Back button on your browser and try your last action again. If you are receiving this message repeatedly, please send an e-mail to (aec51 [at] case [dot] edu) with a detailed description of what you were doing when this message occurred so that we may try and resolve this issue. Thank you!</div></td>
      <td><div class='topic short click_expand'><ul><li>Endocrinology and metabolism</li><li>Literature and language</li><li>Genotype and phenotype</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Query and retrieval</li><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/APHIRM_toolkit" target="_blank">APHIRM toolkit</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The APHIRM Toolkit is a set of OHS risk management procedures that target both physical and psychosocial hazards. A participative hazard identification and risk management toolkit for improving the risk management of musculoskeletal disorders.</div></td>
      <td><div class='topic short click_expand'><ul><li>Physics</li><li>Database management</li><li>Musculoskeletal medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pathway_tools" target="_blank">Pathway Tools</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Supports creation, editing, querying, visualization, and analysis of PGDBs. The software also allows users to publish a PGDB on the Web for access by the scientific community. A PGDB is a bioinformatics DB that integrates genomic data with detailed functional annotations of the genome, such as descriptions of metabolic and signalling pathways, and of the regulatory network.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Endocrinology and metabolism</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>340</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/pathway-tools-16-editing-querying-visualization-analysis-pgdbs.html" target="_blank">http://www.mybiosoftware.com/pathway-tools-16-editing-querying-visualization-analysis-pgdbs.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/decompTumor2Sig" target="_blank">decompTumor2Sig</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R package for identification of mutational signatures active in individual tumors. It decomposes an individual tumor genome into a given set of Alexandrov-type or Shiraishi-type signatures, thus quantifying the contribution of the corresponding mutational processes to the somatic mutations identified in the tumor.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Genetic variation</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Sequence motif recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/rmpiro/decompTumor2Sig/issues" target="_blank">https://github.com/rmpiro/decompTumor2Sig/issues</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/splicedetector" target="_blank">SpliceDetector</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Recognizes alternative splicing events in human and model organisms. Splice Detector employs transcript identifiers to proceed. It provides the frequency of active splice sites in pre-mRNA. This tool can be used to discover genomic coordinates of query transcript exons. It creates a graph showing the alternative splicing regions and gives an understanding of splice sites and alternative splicing events.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>RNA-seq</li><li>RNA splicing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Splice site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lucegene" target="_blank">LuceGene</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>LuceGene is an open-source document/object search and retrieval system specially tuned for bioinformatics text databases and documents. LuceGene is similar in concept to the widely used, commercially successful, bioinformatics program SRS (Sequence Retrieval System).</div></td>
      <td><div class='topic short click_expand'><ul><li>Databases</li><li>Bioinformatics</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/lucegene-1-4-search-system-genome-bioinformatic-databases.html" target="_blank">http://www.mybiosoftware.com/lucegene-1-4-search-system-genome-bioinformatic-databases.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/p53" target="_blank">p53</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>P53 mutation browser.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cogeme" target="_blank">COGEME</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Phytopathogenic fungi and oomycete EST database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence record : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>32</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/trend-db" target="_blank">TREND-DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>TREND-DB – A Transcriptome-wide Atlas of the Dynamic Landscape of Alternative Polyadenylation.<br>The diversity and dynamics of the transcriptome are important means for development and adaptation.<br>This work contains a number of TREND-seq datasets investigating transcriptome-wide APA, covering >170 RNAis conditions.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA splicing</li><li>Epigenetics</li><li>Functional, regulatory and non-coding RNA</li><li>Gene transcripts</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mmcif_dictionary_suite" target="_blank">MMCIF Dictionary Suite</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MMCIF Dictionary Suite is a collection of tools to parse, validate, manage data dictionaries used by the PDB. Applications are also included, which convert data dictionaries to XML schema, provide HTML browsing, produce mapping information for relational database loaders (Db Loader) and convert data files between related CIF dialects</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mmcif-dictionary-suite-2-200-parse-validate-manage-data-dictionaries-pdb.html" target="_blank">http://www.mybiosoftware.com/mmcif-dictionary-suite-2-200-parse-validate-manage-data-dictionaries-pdb.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/prophecy" target="_blank">PROPHECY</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Profiling of phenotypic characteristics in yeast.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>18</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/kipar" target="_blank">KiPar</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>KiPar is a standalone Java application for the retrieval of textual documents likely to contain information relevant for kinetic modelling of a given metabolic pathway.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Pathways</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/kipar-systematic-information-retrieval-regarding-parameters-for-kinetic-modelling-of-metabolic-pathways.html" target="_blank">http://www.mybiosoftware.com/kipar-systematic-information-retrieval-regarding-parameters-for-kinetic-modelling-of-metabolic-pathways.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/capipy" target="_blank">CapiPy</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CapiPy (Computer Assistance for Protein Immobilisation – Python) is a small collection of 4 main packages to retrieve useful information for the immobilisation of a query protein. Information from each module, specially the Blast and modeller is preferentially used by the others. In case no such information is given, the user will have to input the necessary information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Proteins</li><li>Biotechnology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Data retrieval</li><li>Protein modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/manor" target="_blank">MANOR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Importation, normalization, visualization, and quality control functions to correct identified sources of variability in array-CGH experiments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Molecular genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Sequencing quality control</li><li>Loading</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>67</div></td>
      <td><div class="link short click_expand"><a href="http://bioinfo.curie.fr/projects/manor/index.html" target="_blank">http://bioinfo.curie.fr/projects/manor/index.html</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/peryton" target="_blank">Peryton</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Peryton is a manual collection of experimentally supported microbe-disease associations.<br>Peryton constitutes a novel resource of experimentally supported microbe-disease associations, currently hosting more than 7,900 entries linking 43 diseases and 1,396 microorganisms. Peryton’s content is exclusively sustained by manual curation of biomedical articles. Importantly, diseases and microorganisms are provided in a systematic, standardized manner using reference resources to create database dictionaries. Information about the experimental design, study cohorts and the applied high- or low-throughput techniques are meticulously annotated and catered to users.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cardiology</li><li>Microbial ecology</li><li>Literature and language</li><li>Pathology</li><li>Gastroenterology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/qdb" target="_blank">QDB (Query Driven Biclustering)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Query-driven module discovery in microarray data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>raw microarray data : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>score : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>23</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/findbase" target="_blank">FINDbase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Records frequencies of causative genetic variations worldwide. Database records include the population and ethnic group or geographical region, the disorder name and the related gene, accompanied by links to any related external resources and the genetic variation together with its frequency in that population.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li><li>Population genetics</li><li>Pharmacogenomics</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Data retrieval</li><li>Genetic variation analysis</li><li>Genetic mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>19</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/reptar" target="_blank">RepTar</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>RepTar is a database of miRNA target predictions, based on the RepTar algorithm that is independent of evolutionary conservation considerations and is not limited to seed pairing sites.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li><li>Sequence sites, features and motifs</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein binding site prediction</li><li>Query and retrieval</li><li>Protein subcellular localisation prediction</li><li>microRNA detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>40</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/havoc" target="_blank">HaVoC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Bioinformatic pipeline for reference-based consensus assembly and lineage assignment for SARS-CoV-2 sequences.<br>HAVoC, a bioinformatic pipeline for reference-based consensus assembly and lineage assignment for SARS-CoV-2 sequences. HAVoC is a pipeline utilizing several bioinformatic tools to perform multiple necessary analyses for investigating genetic variance among SARS-CoV-2 samples.The pipeline is ideal for those who need a more accessible and fast tool to detect and monitor the spread of SARS-CoV-2 variants of concern during local outbreaks.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Sequence assembly</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li><li>Query and retrieval</li><li>Sequence alignment</li><li>Variant calling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://bitbucket.org/auto_cov_pipeline/havoc" target="_blank">https://bitbucket.org/auto_cov_pipeline/havoc</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/redidb" target="_blank">RNA editing database (REDIdb)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>RNA editing database in organelles.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular biology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/refseq" target="_blank">NCBI reference sequences</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive, integrated, non-redundant, well-annotated set of sequences, including genomic DNA, transcripts, and proteins. Provides a stable reference for genome annotation, gene identification and characterization, mutation and polymorphism analysis (especially RefSeqGene records), expression studies, and comparative analyses.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Comparative genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence record : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ThaleMine" target="_blank">ThaleMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ThaleMine enables you to analyze Arabidopsis thaliana genes, proteins, gene expression, protein-protein interactions, orthologs, and more.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>155</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/AAI-profiler" target="_blank">AAI-profiler</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>AAI-profiler [1] is a fast homology search tool that takes a query proteome (protein sequences in FASTA format) as input and plots the AAI (Average Amino-acid Identity) values of species in the Uniprot database. AAI-profiler is powered by SANS [2] and the processing time for a bacterial proteome is a few minutes.</div></td>
      <td><div class='topic short click_expand'><ul></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/KoBra" target="_blank">KoBra</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Rate constant method for prediction of the diffusion of sorbates inside nanoporous materials at different loadings.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mathematics</li><li>Chemistry</li><li>Physics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CIE" target="_blank">Causal Inference Enrichment (CIE)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A platform for directional gene set enrichment analysis and inference of active transcriptional regulators.<br>Causal Inference and Directional Enrichment Methods on Biological Networks.<br>Launching Causal Inference Engine...<br>CIE is a platform for inference of active transcriptional regulators of differential gene expression. CIE provides a user-friendly web-app and an R-package to run inference queries on casual regulatory netowrks. Several regulatory networks are provided in the platform, including a TF-gene intraction network constructed from ChIP-seq and tissue specific gene expression data, a netowrk based on STRING-DB, and curated TF-gene interaction networks such as TRRUST and TRED. The R-package provides functionality for custom networks as well.<br>Causal Inference Engine: Directional Enrichment Analysis on Biological Networks.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>ChIP-seq</li><li>Genotype and phenotype</li><li>Gene regulation</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene-set enrichment analysis</li><li>Pathway or network visualisation</li><li>Pathway or network prediction</li><li>Gene regulatory network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/umbibio/CIE-R-Package" target="_blank">https://github.com/umbibio/CIE-R-Package</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GenomeHubs" target="_blank">GenomeHubs</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Simple containerised setup of a custom Ensembl database and web server for any species.<br>Forked from molluscdb/molluscdb-plugin.<br>Example configurations to set up custom Ensembl sites using GenomeHubs.<br>GenomeHubs example configuration and demo script.<br>Generic Ensembl plugin for genomehubs sites.<br>GenomeHubs is designed to make it easy to set up and host a core set of bioinformatics tools to help research communities share and access genomic datasets for non-model organisms.<br>Sets up a GenomeHubs site hosting a mirror of the core database for the Glanville fritillary, Melitaea cinxia and imports the genome of the winter moth, Operophtera brumata, into a GenomeHubs site from FASTA and GFF.</div></td>
      <td><div class='topic short click_expand'><ul><li>Plant biology</li><li>Workflows</li><li>Model organisms</li><li>Sequence assembly</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li><li>Genome visualisation</li><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/GenomeHubs/" target="_blank">https://github.com/GenomeHubs/</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rgalaxy" target="_blank">RGalaxy</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Given an R function and its manual page, make the documented function available in Galaxy.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phosphoelm" target="_blank">PhosphoELM database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Experimentally verified phosphorylation sites in eukaryotic proteins.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein modifications</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence motif (protein) : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>393</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/entrez_protein_clusters" target="_blank">Entrez Protein Clusters</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A collection of related protein sequences (clusters) consists of proteins derived from the annotations of whole genomes, organelles and plasmids. It currently limited to Archaea, Bacteria, Plants, Fungi, Protozoans, and Viruses.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Sequence analysis</li><li>Phylogenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein family report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>203</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/AtFusionDB" target="_blank">AtFusionDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of fusion transcripts in Arabidopsis thaliana.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-seq</li><li>Gene transcripts</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Local alignment</li><li>Expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMMUNOPHARMACOLOGY" target="_blank">IMMUNOPHARMACOLOGY</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Extending immunopharmacology content and introducing the IUPHAR/MMV Guide to MALARIA PHARMACOLOGY.<br>IUPHAR Guide to IMMUNOPHARMACOLOGY.<br>The IUPHAR/BPS Guide to IMMUNOPHARMACOLOGY. Description.</div></td>
      <td><div class='topic short click_expand'><ul><li>Parasitology</li><li>Immunology</li><li>Metabolomics</li><li>Pharmacology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Peptide immunogenicity prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>35</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/haplotype_explorer" target="_blank">Haplotype Explorer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An infection cluster visualization tool for spatiotemporal dissection of the COVID-19 pandemic.<br>HaplotypeExplorer is an interactive haplotype network viewer for spatiotemporal dissection of multimodal epidemic spectra.</div></td>
      <td><div class='topic short click_expand'><ul><li>Infectious disease</li><li>Public health and epidemiology</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Network visualisation</li><li>Clustering</li><li>Parsing</li><li>Haplotype mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/isa4j" target="_blank">isa4j</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A scalable Java library for creating ISA-Tab metadata.<br>isa4j is a comprehensive and scalable Java Library for the programmatic generation of experimental metadata descriptions using the ISATab container format. We're assuming you're familiar with the ISA-Tab framework in the remainder of the manual; if you're not, please read up about it first.<br>This is an XML namespace defined in the XML Linking Language (XLink) specification.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Database management</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/IPK-BIT/isa4j" target="_blank">https://github.com/IPK-BIT/isa4j</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mapgaps" target="_blank">MAPGAPS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Uses a set of multiply-aligned profiles both as a query to detect and classify related sequences and as a template to multiply-align the sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Nucleic acids</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>39</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mapgaps-multiply-aligned-profiles-for-global-alignment-of-protein-sequences.html" target="_blank">http://www.mybiosoftware.com/mapgaps-multiply-aligned-profiles-for-global-alignment-of-protein-sequences.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ensembleqdb" target="_blank">Ensembl query-based biclustering</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An ensemble biclustering approach for querying gene expression compendia with experimental lists.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence cluster (nucleic acid) : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/panomicon" target="_blank">Panomicon</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Panomicon (formerly Toxygates) is a web application for user-friendly analysis of multi-omics data (presently focusing on mRNA and mRNA-miRNA interaction data). It was originally developed for the Open TG-GATEs dataset but is now being used for more general purposes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Proteomics</li><li>Functional, regulatory and non-coding RNA</li><li>Gene transcripts</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Network visualisation</li><li>Query and retrieval</li><li>Editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/Toxygates/Panomicon" target="_blank">https://github.com/Toxygates/Panomicon</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/drugbank" target="_blank">DrugBank</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Unique bioinformatics/cheminformatics resource that combines detailed drug (i.e. chemical) data with comprehensive drug target (i.e. protein) information. The database contains >4100 drug entries including >800 FDA approved small molecule and biotech drugs as well as >3200 experimental drugs. Additionally, >14,000 protein or drug target sequences are linked to these drug entries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pharmacology</li><li>Cheminformatics</li><li>Drug discovery</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Database search</li><li>Design</li><li>Molecular docking</li><li>Protein subcellular localisation prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,887</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rdavidwebservice" target="_blank">RDAVIDWebService</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool for retrieving data from DAVID database that offers the main functionalities of the website, including a connectivity to upload gene background lists, or the reports of the submitted information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data acquisition</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>98</div></td>
      <td><div class="link short click_expand"><a href="http://www.bdmg.com.ar" target="_blank">http://www.bdmg.com.ar</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genemania_plugin" target="_blank">GeneMania Cytoscape plugin</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The plugin brings fast gene function prediction capabilities to the desktop. It identifies the most related genes to a query gene set using a guilt-by-association approach. The plugin uses a large database of functional interaction networks from multiple organisms and each related gene is traceable to the source network used to make the prediction. Users may add their own interaction networks and expression profile data to complement or override the default data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Gene expression</li><li>Protein interactions</li><li>Model organisms</li><li>Function analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene prediction</li><li>Pathway or network prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>333</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pars" target="_blank">PARS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Server for the prediction of allosteric and regulatory sites on protein structures. It queries protein dynamics and structural conservation to identify pockets that may exert a regulatory effect upon binding of a small-molecule ligand.</div></td>
      <td><div class='topic short click_expand'><ul><li>Structural biology</li><li>Proteins</li><li>Molecular dynamics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>57</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/pars-protein-allosteric-and-regulatory-sites.html" target="_blank">http://www.mybiosoftware.com/pars-protein-allosteric-and-regulatory-sites.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/merav" target="_blank">MERAV</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based tool that can query a database comprising ～4300 microarrays, representing human gene expression in normal tissues, cancer cell lines and primary tumors.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Cell biology</li><li>Oncology</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>55</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/merav-metabolic-gene-rapid-visualizer.html" target="_blank">http://www.mybiosoftware.com/merav-metabolic-gene-rapid-visualizer.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/snpediar" target="_blank">SNPediaR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides some tools for downloading and parsing data from the SNPedia web site. The implemented functions allow users to import the wiki text available in SNPedia pages and to extract the most relevant information out of them. If some information in the downloaded pages is not automatically processed by the library functions, users can easily implement their own parsers to access it in an efficient way.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA polymorphism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/genometra/SNPediaR" target="_blank">https://github.com/genometra/SNPediaR</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/topsearch" target="_blank">TopSearch</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>TopSearch is a new powerful gateway to protein structures and their structural relationships. Given a query structure the search engine instantly returns a list of known protein structures sorted by structural similarity. TopSearch operates on three structural levels: chains, biological assemblies and quaternary structures, and asymmetric units.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/amigo_2" target="_blank">AmiGO 2</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The GO Consortium GO browser and search engine.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li><li>Query and retrieval</li><li>Sequence comparison</li><li>Ontology visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>944</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/berex" target="_blank">BEReX</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BEReX is a new biomedical knowledge integration, search and exploration tool. BEReX integrates eight popular databases (STRING, DrugBank, KEGG, PhamGKB, BioGRID, GO, HPRD and MSigDB) and delineates an integrated network by combining the information available from these databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Databases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/berex-1-0-iberex-biomedical-entity-relation-explorer.html" target="_blank">http://www.mybiosoftware.com/berex-1-0-iberex-biomedical-entity-relation-explorer.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SNPs" target="_blank">SNPs</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genome-wide analysis of polymorphisms identified domestication-associated polymorphism desert carrying important rice grain size/weight QTL | By using SNP-Seek, you abide by the data use license stated | Query for SNPs from the 3000 genome project | - Mansueto, et al. Rice SNP-seek database update: new SNPs, indels, and queries . Nucl. Acids Res. (2017) 45 (D1): D1075-D1081. doi: 10.1093/nar/gkw1135 This paper describes new features and datasets added to SNP-Seek in 2015-2017 as well as software and database updates | This site provides Genotype, Phenotype, and Variety Information for rice (Oryza sativa L.)</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>DNA polymorphism</li><li>Genotype and phenotype</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>SNP detection</li><li>SNP annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/disqover" target="_blank">DISQOVER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DISQOVER is a data integration platform for public, licensed and internal data. The Data Ingestion Engine enables transforming data into Linked Data which can be searched, navigated and analysed via the user interface and the API. The publicly accessible DISQOVER platform contains 140+ public data sources in life sciences and related domains.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biomedical science</li><li>Biology</li><li>Chemistry</li><li>Medicine</li><li>Omics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Box-Whisker plot plotting</li><li>Scatter plot plotting</li><li>Filtering</li><li>Visualisation</li><li>ID mapping</li><li>Aggregation</li><li>Formatting</li><li>Text annotation</li><li>Query and retrieval</li><li>Ontology visualisation</li><li>Pathway or network visualisation</li><li>Structure database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_DomainDisplay" target="_blank">IMGT DomainDisplay</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/DomainDisplay is the IMGT® tool for the display of the amino acid sequences of the variable (V), constant (C) and groove (G) domains.<br>Analysis is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunogenetics</li><li>Immunoinformatics</li><li>Protein sites, features and motifs</li><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein sequence : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>205</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cncrnadb" target="_blank">cncRNAdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CncRNAdb is a manually curated resource of experimentally supported RNAs with both protein-coding and noncoding function.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein expression</li><li>Functional, regulatory and non-coding RNA</li><li>Gene expression</li><li>Gene transcripts</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Expression analysis</li><li>trans-regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/irefindex" target="_blank">iRefIndex</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides an index of protein interactions available in a number of primary interaction databases including BIND, BioGRID, CORUM, DIP, HPRD, InnateDB, IntAct, MatrixDB, MINT, MPact, MPIDB and MPPI.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>353</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lsd" target="_blank">LSD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This database contains senescence associated genes (SAGs) from 21 species retrieved based on genetic, genomic, proteomic, physiological or other experimental evidence, and classified into different categories according to their functions in leaf senescence or morphological phenotypes. Users can browse or search the database to obtain information including literatures, mutants, phenotypes, expression profiles, miRNA interactions, orthologs in other plants and cross links to other databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Genotype and phenotype</li><li>Functional, regulatory and non-coding RNA</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>64</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/uniref" target="_blank">UniProt Reference Clusters (UniRef)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Clustered sets of sequences from UniProt Knowledgebase (including splice variants and isoforms) and selected UniParc records, in order to obtain complete coverage of sequence space at several resolutions while hiding redundant sequences (but not their descriptions) from view.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Gene structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence cluster (protein) : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>338</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rentrez" target="_blank">rentrez</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An R package for the NCBI eUtils API.<br>WIP on a manuscript for rentrez (at last!).<br>A manuscript for rentrez (at last!).<br>rentrez provides functions that work with the NCBI Eutils API to search, download data from, and otherwise interact with NCBI databases.<br>rentrez is on CRAN, so you can get the latest stable release with install.packages("rentrez"). This repository will sometimes be a little ahead of the CRAN version, if you want the latest (and possibly greatest) version you can install the current github version using Hadley Wickham's devtools</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Phylogeny</li><li>Literature and language</li><li>Metagenomics</li><li>Biotechnology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Database search</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://github.com/ropensci/rentrez" target="_blank">http://github.com/ropensci/rentrez</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/typhon" target="_blank">Typhon Beta</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An algorithm for building an index from a multiple alignment for use in seeding local alignments to a query sequence.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Nucleic acid structure analysis</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Indexing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/typhon-beta-multiple-alignment-indexing-tool.html" target="_blank">http://www.mybiosoftware.com/typhon-beta-multiple-alignment-indexing-tool.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/repeatmasker" target="_blank">RepeatMasker</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A program that screens DNA sequences for interspersed repeats and low complexity DNA sequences. The output of the program is a detailed annotation of the repeats that are present in the query sequence as well as a modified version of the query sequence in which all the annotated repeats have been masked (default: replaced by Ns).</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Sequence composition, complexity and repeats</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>OSL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/newt-omics" target="_blank">Newt-omics</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of data from the newt Notophthalmus viridescens. Newt-Omics aims to provide a comprehensive platform of expressed genes during tissue regeneration, including extensive annotations, expression data and experimentally verified peptide sequences with yet no homology to other publically available gene sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Gene transcripts</li><li>Transcriptomics</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Data handling</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>15</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Doc2Hpo" target="_blank">Doc2Hpo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web application for efficient and accurate Human Phenotype Ontology (HPO) concept curation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Genotype and phenotype</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Ontology visualisation</li><li>Named-entity and concept recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/stormliucong/doc2hpo" target="_blank">https://github.com/stormliucong/doc2hpo</a></div></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sgrna-psm" target="_blank">sgRNA-PSM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Predict sgRNAs On-Target Activity Based on Position-Specific Mismatch.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic engineering</li><li>Synthetic biology</li><li>RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Editing</li><li>Feature extraction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/savr" target="_blank">savR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Parse Illumina Sequence Analysis Viewer (SAV) files, access data, and generate QC plots.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence analysis</li><li>Sequencing quality control</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bcalder/savR" target="_blank">https://github.com/bcalder/savR</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/u-ANT" target="_blank">u-ANT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Semantic Microaggregation-based Anonymization Tool.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Data security</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Anonymisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/isa_api" target="_blank">ISA API</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An open platform for interoperable life science experimental metadata.<br>Built around the ‘Investigation’ (the project context), Study’ (a unit of research) and ‘Assay’ (analytical measurement) general-purpose Tabular format, the ISA tools helps you to provide rich description of the experimental metadata (i.e. sample characteristics, technology and measurement types, sample-to-data relationships) so that the resulting data and discoveries are reproducible and reusable.<br>The ISA API aims to provide you, the developer, with a set of tools to help you easily and quickly build your own ISA objects, validate, and convert between serializations of ISA-formatted datasets and other formats/schemas (e.g. SRA schemas). The ISA API is published on PyPI as the isatools package.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Biology</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Editing</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/epiTAD" target="_blank">epiTAD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web application for visualizing chromosome conformation capture data in the context of genetic epidemiology.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>DNA polymorphism</li><li>Public health and epidemiology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>SNP annotation</li><li>SNP detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/GerkeLab/epiTAD" target="_blank">https://github.com/GerkeLab/epiTAD</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tessa" target="_blank">Tessa</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Novel MATLAB Program for Automated Tessitura Analysis.</div></td>
      <td><div class='topic short click_expand'><ul></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fragger" target="_blank">Fragger</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Protein fragment picker that allows protein fragment databases to be created and queried. It can efficiently search a fragment database with a query fragment and a distance threshold. Matching fragments are ranked by distance to the query. It also incorporates a tool to compute the backbone RMSD of one versus many fragments in high throughput.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Indexing</li><li>Sequence comparison</li><li>Protein comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sc1cc" target="_blank">SC1CC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Computational cell cycle analysis of single cell RNA-Seq data.<br>SC1: scRNA-Seq Analysis Pipeline.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>RNA-Seq</li><li>Cell biology</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Expression analysis</li><li>Clustering</li><li>Essential dynamics</li><li>Phasing</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/RADAR-base" target="_blank">RADAR-base</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Open Source Mobile Health Platform for Collecting, Monitoring, and Analyzing Data Using Sensors, Wearables, and Mobile Devices | BACKGROUND:With a wide range of use cases in both research and clinical domains, collecting continuous mobile health (mHealth) streaming data from multiple sources in a secure, highly scalable, and extensible platform is of high interest to the open source mHealth community. The European Union Innovative Medicines Initiative Remote Assessment of Disease and Relapse-Central Nervous System (RADAR-CNS) program is an exemplary project with the requirements to support the collection of high-resolution data at scale; as such, the Remote Assessment of Disease and Relapse (RADAR)-base platform is designed to meet these needs and additionally facilitate a new generation of mHealth projects in this nascent field</div></td>
      <td><div class='topic short click_expand'><ul><li>Psychiatry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Deposition</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PertInInt" target="_blank">PertInInt</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PertInInt is a tool using an integrative approach uncovers genes with perturbed interactions in cancers. The tool enables rapid uncovering of significantly somatically mutated protein functional regions. The goal of the analytical PertInInt method is to rapidly uncover proteins with significant enrichments of somatic Perturbations In Interaction and other functional sites.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Genetic variation</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enrichment analysis</li><li>Parsing</li><li>Nucleic acids-binding site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gdbchembl" target="_blank">GDBChEMBL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ChEMBL-Likeness Score and Database GDBChEMBL.<br>GDBChEMBL is a library of 10 million chemical structures that was extracted from the exhaustively generated database GDB17. It was generated by scoring input SMILES structures by circular substructures shared with the reference database of known chemical compounds, ChEMBL v.24.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biochemistry</li><li>Cheminformatics</li><li>Computational chemistry</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Virtual screening</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td><div class="link short click_expand"><a href="https://www.ebi.ac.uk/chembl/" target="_blank">https://www.ebi.ac.uk/chembl/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phi-base" target="_blank">PHI-base</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PHI-base contains expertly curated molecular and biological information on genes proven to affect the outcome of pathogen–host interactions reported in peer reviewed research articles.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>104</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/t3db" target="_blank">T3DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Unique bioinformatics resource that compiles comprehensive information about common or ubiquitous toxins and their toxin-targets. Each record (ToxCard) contains over 80 data fields providing detailed information on chemical properties and descriptors, toxicity values, protein and gene sequences (for both targets and toxins), molecular and cellular interaction data, toxicological data, mechanistic information and references.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li><li>Molecular interactions, pathways and networks</li><li>Toxicology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein property calculation</li><li>Query and retrieval</li><li>Protein subcellular localisation prediction</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>80</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pirsf" target="_blank">PIRSF whole-protein classification database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Non-overlapping clustering of UniProtKB sequences into a hierarchical order to reflect their evolutionary relationships. The classification system is based on whole proteins rather than on the component domains therefore, it allows annotation of generic biochemical and specific biological functions, as well as classification of proteins without well-defined domains.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Gene and protein families</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein family report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SKIMMR" target="_blank">SKIMMR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Facilitating knowledge discovery in life sciences by machine-aided skim reading.<br>SKIMMR - a research prototype for machine-aided skim reading (includes back-end code for generating a graph-like knowledge base from texts and a standalone HTTP server-based UI)</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical imaging</li><li>Biology</li><li>Pathology</li><li>Ontology and terminology</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Parsing</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/DR-thermo" target="_blank">DR-thermo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Thermodynamically Consistent Estimation of Gibbs Free Energy from Data.<br>The DR-thermo algorithm can be used to curate experimental thermodynamic data from different sources to generate reliable data-banks such that all measurements are modified to be thermodynamically consistent and any missing reaction or formation Gibbs energies can be predicted either directly or using group contributions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Molecular interactions, pathways and networks</li><li>Physics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Quantification</li><li>Regression analysis</li><li>Imputation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/samansalike/DR-thermo/issues" target="_blank">https://github.com/samansalike/DR-thermo/issues</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gpcrs" target="_blank">GPCRs</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>GPCR family profiles, mutation data (in various formats), ligand binding data, sequence annotations and cross-references to other databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Membrane and lipoproteins</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein features : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>83</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bankit" target="_blank">BankIt</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based submission of one or a few sequences to GenBank.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Metagenomics</li><li>Mapping</li><li>Whole genome sequencing</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence tagged site (STS) mapping</li><li>Deposition</li><li>Data retrieval</li><li>Mapping</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>227</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GazeVisual-Lib" target="_blank">GazeVisual-Lib</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Development of Open-source Software and Gaze Data Repositories for Performance Evaluation of Eye Tracking Systems.<br>Repository for GazeVisual performance evaluation software tools.<br>The GazeVisual-Lib is a repository of software tools for practical evaluation of eye trackers and gaze data quality. All codes in this repository are released under GNU General Public License v3.0</div></td>
      <td><div class='topic short click_expand'><ul><li>Data quality management</li><li>Machine learning</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Quantification</li><li>Parsing</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/anuradhakar49/GazeVisual-Lib.git" target="_blank">https://github.com/anuradhakar49/GazeVisual-Lib.git</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pub2tools2020__metatropismdb" target="_blank">MetaTropismDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of organ-specific metastasis induced by human cancer cell lines in mouse models.<br>The organotropism is the propensity of metastatic cancer cells to colonize preferably certain distant organs, resulting in a non-random distribution of metastases. In order to shed light on this behaviour, several studies were performed by the injection of human cancer cell lines into immunocompromised mouse models.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Microarray experiment</li><li>Literature and language</li><li>Model organisms</li><li>Mouse clinic</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genetclassifier" target="_blank">geNetClassifier</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Comprehensive package to automatically train and validate a multi-class SVM classifier based on gene expression data. Provides transparent selection of gene markers, their coexpression networks, and an interface to query the classifier.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pathology</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td><div class="link short click_expand"><a href="http://www.cicancer.org" target="_blank">http://www.cicancer.org</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aresite" target="_blank">AREsite</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Online resource for the investigation of AU-rich elements (ARE) in human and mouse mRNA 3' UTR sequences. The new updated and enhanced version allows detailed investigation of AU, GU and U-rich elements (ARE, GRE, URE) in the transcriptome of Homo sapiens, Mus musculus, Danio rerio, Caenorhabditis elegans and Drosophila melanogaster. It contains information on genomic location, genic context, RNA secondary structure context and conservation of annotated motifs as well as related publications.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Map drawing</li><li>Query and retrieval</li><li>Structure visualisation</li><li>Sequence database search (by property)</li><li>Structure database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>map : xls</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>38</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/inparanoid" target="_blank">Inparanoid</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of orthologs in eukaryotes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Evolutionary biology</li><li>Function analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>533</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/compareoverlappingsmallref" target="_blank">compareoverlappingsmallref</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provide the queries that overlap with a reference, when the reference is small.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : gtf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : gff3</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>937</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/urgi_tools/CompareOverlappingSmallRef/1.0.0" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/urgi_tools/CompareOverlappingSmallRef/1.0.0</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/consensuspathdb" target="_blank">ConsensusPathDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Information on physical molecular interactions, biochemical pathways, and gene regulatory networks in human.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein interaction raw data : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>423</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/xanthusbase" target="_blank">Xanthusbase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Model organism database for the bacterium Myxococcus xanthus.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Genomics</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome annotation</li><li>Query and retrieval</li><li>Database comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/crazyquant" target="_blank">CrazyQuant</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CrazyQuant is a spot-finding tool. This JAVA-based applet allows the loading and quantitation of your cDNA microarray images and membranes.</div></td>
      <td><div class='topic short click_expand'><ul><li>cDNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/crazyquant-spot-finding-tool.html" target="_blank">http://www.mybiosoftware.com/crazyquant-spot-finding-tool.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metaimprint" target="_blank">MetaImprint</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An information repository of mammalian imprinted genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Genomics</li><li>Genetics</li><li>Nucleic acid sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>21</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_GeneFrequency" target="_blank">IMGT GeneFrequency</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/GeneFrequency is the IMGT® tool providing the frequency usage of immunoglobulin (IG) and T cell receptor (TR) variable (V), diversity (D) and joining (J) genes in Homo sapiens and Mus musculus rearranged sequences from IMGT/LIGM-DB.<br>Analysis is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunogenetics</li><li>Immunoinformatics</li><li>Immunology</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>keyword : gene expression report format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>287</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/a2a" target="_blank">A2A</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A platform for research in biomedical literature search and benchmarking.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Preclinical and clinical studies</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Literature search</li><li>Data retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GLycan_Array_Dashboard" target="_blank">GLycan Array Dashboard</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based tool to visualize, analyze, present and mine glycan microarray data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Literature and language</li><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sorting</li><li>Sequencing quality control</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/transsynw" target="_blank">TransSynW</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A single-cell RNA-sequencing based web application to guide cell conversion experiments.<br>Convert between cell subtypes by using single-cell data to identify an optimal, subtype-specific set of core transcription factors personalized for your own human or mouse cell study.<br>Starting cell population: Select or upload a tab-separated file with the single-cell gene expression data from the cell type you started your differentiation protocol from. Genes should be labeled according to the Gene Symbols nomenclature.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Cell biology</li><li>Regenerative medicine</li><li>Ontology and terminology</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li><li>Expression profile clustering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://git-r3lab.uni.lu/mariana.ribeiro/transsynw" target="_blank">https://git-r3lab.uni.lu/mariana.ribeiro/transsynw</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/profkin" target="_blank">ProfKin</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Comprehensive Web Server for Structure-based Kinase Selectivity Profiling.<br>Kinase profiling is an efficient strategy for kinase inhibitor discovery, polypharmacological drug discovery and drug repositioning. A variety of methods have been established for kinase profiling, of which the combined mode of experimental and computational approaches is attractive, particularly in the early drug discovery. With the increasing number of kinase-inhibitor complex structures, structure-based kinase profiling is of great interest. To fully exploit the potential of structure-based kinase profiling in drug discovery and drug repositioning, we developed a versatile web server, termed ProfKin, for structure-based kinase profiling for small molecules of interest, which is established based on an in-house comprehensive structural database (KinLigDB) of manually curated kinase-ligand complex structures and associated information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Pharmacology</li><li>Molecular modelling</li><li>Drug discovery</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Phylogenetic tree visualisation</li><li>Molecular docking</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/OakMine" target="_blank">OakMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Quercus robur annotation database</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/shortread" target="_blank">ShortRead</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package implements sampling, iteration, and input of FASTQ files. The package includes functions for filtering and trimming reads, and for generating a quality assessment report. Data are represented as DNAStringSet-derived objects, and easily manipulated for a diversity of purposes. The package also contains legacy support for early single-end, ungapped alignment formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence editing</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>265</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/ShortRead/" target="_blank">http://bioconductor.org/packages/ShortRead/</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fresco" target="_blank">FRESCO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>General open-source framework to compress large amounts of biological sequence data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Computer science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>52</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/fresco-a-framework-for-referential-sequence-compression.html" target="_blank">http://www.mybiosoftware.com/fresco-a-framework-for-referential-sequence-compression.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/svim-asm" target="_blank">SVIM-asm</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Structural variant detection from haploid and diploid genome assemblies.<br>SVIM-asm - Structural variant identification method (Assembly edition).<br>SVIM-asm (pronounced SWIM-assem) is a structural variant caller for haploid or diploid genome-genome alignments. It analyzes a given sorted BAM file (preferably from minimap2) and detects five different variant classes between the query assembly and the reference: deletions, insertions, tandem and interspersed duplications and inversions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>DNA polymorphism</li><li>Genotype and phenotype</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Variant calling</li><li>Genome assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/threadom_ex" target="_blank">ThreaDomEx</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Takes the amino acid sequence of the query protein as input. It first creates multiple threading alignments to recognize homologous and analogous template structures, from which a domain conservation score is then calculated for deducing the domain boundaries. Next, a boundary clustering method is used to optimize the domain model selections. For discontinuous domain structures, a symmetric alignment algorithm is applied to further integrate and refine the domain assignments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>Structure prediction</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein sequence analysis</li><li>Protein secondary structure prediction</li><li>Protein domain recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein sequence (raw) : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/LSD_3.0" target="_blank">LSD 3.0</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive resource for the leaf senescence research community.<br>Leaf Senescence DataBase.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Functional, regulatory and non-coding RNA</li><li>Plant biology</li><li>Gene expression</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/plant-3d" target="_blank">Plant 3D</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Plant 3D is a plant phenotyping toolkit for 3D point clouds.<br>Plant 3D (P3D) automatically extracts common phenotyping features of interest from high-resolution 3D scans of plant architectures. P3D is open-source and is bundled with a stand-alone Windows application. P3D is written in C++ using OpenGL, QT, TensorFlow, and the point cloud library (PCL). P3D can visualize and process data imported as a 3D point cloud (pcd or txt formats) or a mesh (obj format).</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Genotype and phenotype</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Genotyping</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nrprof" target="_blank">NRProF</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>It is a novel automated protein functional assignment method based on the neural response algorithm, which simulates the neuronal behavior of the visual cortex in the human brain. The main idea of this algorithm is to define a distance metric that corresponds to the similarity of the subsequences and reflects how the human brain can distinguish between different sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein function prediction</li><li>Sequences</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/nrprof-protein-function-prediction-based-neural-response-algorithm.html" target="_blank">http://www.mybiosoftware.com/nrprof-protein-function-prediction-based-neural-response-algorithm.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/miga-taxa" target="_blank">MiGA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Classification of an unknown query genomic sequence, complete or partial, against all taxonomically classified taxa with available genome sequences, as well as comparisons to other related genomes including uncultivated ones, based on the genome-aggregate Average Nucleotide and Amino Acid Identity (ANI/AAI) concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Taxonomy</li><li>Microbiology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>127</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bio-miga/miga-web" target="_blank">https://github.com/bio-miga/miga-web</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/theseed" target="_blank">The SEED genome comparison environment</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Environment for comparative genome analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Comparative genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>genome report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>6,386</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nextbio" target="_blank">NextBio gene-centric data</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The platform enables biologists to examine a gene, protein, compound or disease of interest in a global, biological, or clinical context.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Comparative genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>246</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/acre" target="_blank">Absolute concentration robustness explorer (ACRE)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Absolute concentration robustness explorer (ACRE) explores combinatorial biochemical networks for the Absolute concentration robustness property.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Synthetic biology</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Network analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hmd-arg" target="_blank">HMD-ARG</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Hierarchical multi-task deep learning for annotating antibiotic resistance genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Gene and protein families</li><li>Sequencing</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Antimicrobial resistance prediction</li><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pbsword" target="_blank">PBSword</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web server designed for efficient and accurate comparisons and searches of geometrically similar protein-protein binding sites from a large-scale database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein binding sites</li><li>Nucleic acid sites, features and motifs</li><li>Transcription factors and regulatory sites</li><li>Drug discovery</li><li>DNA binding sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein binding site prediction</li><li>Query and retrieval</li><li>Molecular docking</li><li>Peak calling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hgvbase" target="_blank">HGVbase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Attempt to summarize all known sequence variations in the human genome, to facilitate research into how genotypes affect common diseases, drug responses, and other complex phenotypes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Genetic variation</li><li>Pathology</li><li>Human biology</li><li>DNA polymorphism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>SNP detection</li><li>Query and retrieval</li><li>Genetic variation analysis</li><li>Genotyping</li><li>Haplotype mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>116</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rasp-rna" target="_blank">RASP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An atlas of transcriptome-wide RNA secondary structure probing data.<br><br>RASP (RNA Alatas of Structure Probing) is a transcriptome-scale RNA structure probing database. Currently, RASP contains 161 deduplicated transcriptome-wide RNA secondary structure probing datasets from 38 papers. RASP covers 18 species across animals, plants, bacteria, fungi, and also viruses, and categorizes 18 experimental methods including DMS-Seq, SHAPE-Seq, SHAPE-MaP, and icSHAPE, etc. RASP also provides a user-friendly interface to query, browse, and visualize RNA structure profiles, offering a shortcut to accessing RNA secondary structures grounded in experimental data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Nucleic acid structure analysis</li><li>Plant biology</li><li>RNA</li><li>Probes and primers</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Expression analysis</li><li>RNA secondary structure alignment</li><li>Dimethyl</li><li>Structure visualisation</li><li>RNA secondary structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/drfindid" target="_blank">drfindid</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Find public databases by identifier.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/repeatfs" target="_blank">RepeatFS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A file system providing reproducibility through provenance and automation.<br>RepeatFS is a Python, FUSE-based file system with the goal of promoting scientific informatics reproducibility by recording all file and IO operations during analysis. This provenance record can then be exported and used to replicate the analysis on other systems. During replication, RepeatFS will verify that all software versions, command line parameters, and other relevant attributes match, and will report out any deviation from the original record. While other provenance software often involves learning scripting languages or migrating your workflow to a confined platform, RepeatFS operates invisibily at the file system level, and is compatible with virtually all Linux command-line software.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>Transcription factors and regulatory sites</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biojs-io-newick" target="_blank">biojs-io-newick</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Parses newick and extended newick strings(nhx) into JSON.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Phylogenetic tree analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>70</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biopython" target="_blank">Biopython</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Biopython is a set of freely available tools for biological computation written in Python by an international team of developers.</div></td>
      <td><div class='topic short click_expand'><ul><li>Software engineering</li><li>Data management</li><li>Computational biology</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Aggregation</li><li>Service invocation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,569</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/indigenomes" target="_blank">IndiGenomes</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive resource of genetic variants from over 1000 Indian genomes.<br>IndiGenomes Resource of Population Genomes from India.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Genetic variation</li><li>Population genetics</li><li>Exome sequencing</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Variant calling</li><li>Query and retrieval</li><li>Variant classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MESSA" target="_blank">MESSA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Predicting Sequence Features, Function, and Structure of Proteins Using MESSA | MEta-Server for protein Sequence Analysis (MESSA) is a tool that facilitates widespread protein sequence analysis by gathering structural (local sequence properties and three-dimensional structure) and functional (annotations from SWISS-PROT, Gene Ontology terms, and enzyme classification) predictions for a query protein of interest. MESSA uses multiple well-established tools to offer consensus-based predictions on important aspects of protein sequence analysis. Being freely available for noncommercial users and with a user-friendly interface, MESSA serves as an umbrella platform that overcomes the absence of a comprehensive tool for predictive protein analysis. This article reveals how to access MESSA via the Web and shows how to input a protein sequence to analyze using the MESSA web server</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Gene and protein families</li><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Biobtree" target="_blank">Biobtree</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A tool to search and map bioinformatics identifiers and special keywords | A bioinformatics tool to search and map identifiers and special keywords | Biobtree is a bioinformatics tool which process large datasets effectively and provide uniform search and mapping functionalities with web interface and web services for genomic research</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Database management</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/LocustMine" target="_blank">LocustMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated Omics data warehouse for Locust, Locusta migratoria</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/blood-exposome-db" target="_blank">Blood Exposome Database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Generating the Blood Exposome Database Using a Comprehensive Text Mining and Database Fusion Approach | The exposome represents the sum of all exposures during the life-span of an organism (from chemicals to microbes, viruses, radiation and other sources)</div></td>
      <td><div class='topic short click_expand'><ul><li>Natural language processing</li><li>Toxicology</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Text mining</li><li>Chemical similarity enrichment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/barupal/exposome" target="_blank">https://github.com/barupal/exposome</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/euadb" target="_blank">EUAdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>EUAdb is database for COVID-19 test development that contains standardized information about Eemergency Use Authorizations-issued tests and is focused on RT-qPCR diagnostic tests, or high complexity molecular-based laboratory developed tests.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>PCR experiment</li><li>Sequencing</li><li>Probes and primers</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Ontology visualisation</li><li>Quantification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/targetfinder.org" target="_blank">Targetfinder.org</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Targetfinder.org provides a web based resource that allows users to find genes that have a similar expression to a query gene signature.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Gene expression</li><li>Gene regulation</li><li>Transcription factors and regulatory sites</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression analysis</li><li>Gene prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/topodb" target="_blank">TopoDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A novel multifunctional management system for laboratory animal colonies.<br>TopoDB requires Ruby version >= 2.5.2, Ruby on Rails version >= 5.2.0, and bundler version 2.0 . We recommend using rbenv to install and manage Ruby gems.<br>Log in as the admin user that was created as part of the sample data set: email: admin@topodb.topo, password: admin123 . Once logged in as the administrator it is strongly recommended that you create new users to handle day-to-day activities in the application and that you change the Administrator account password (failure to do so can put your data at risk). Both of these activities can be done either directly in the users database table, or through the included rails admin feature, available in the main navigation bar.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Medicine</li><li>Zoology</li><li>Database management</li><li>Laboratory animal science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/unicarb-db" target="_blank">UniCarb-DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of LC-MS structural data for oligosaccharides.</div></td>
      <td><div class='topic short click_expand'><ul><li>Carbohydrates</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>89</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/jaspar_api" target="_blank">JASPAR RESTful API</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Widely used open-access database of curated, non-redundant transcription factor binding profiles. Currently, data from JASPAR can be retrieved as flat files or by using programming language-specific interfaces. Here, we present a programming language-independent application programming interface (API) to access JASPAR data using the Representational State Transfer (REST) architecture.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data acquisition</li><li>DNA</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tripal-galaxy" target="_blank">Tripal Galaxy</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Tripal Galaxy module is designed to support integration of Tripal with Galaxy. It uses the blend4php library. In the past, community databases have often provided analytical tools that come prepared with site-specific data. Examples include BLAST, CAP3, and InterProScan servers, for example. By integrating Tripal with Galaxy, the Tripal-based community database can offer more complicated analytical tools that support larger data sets using Galaxy as a backend.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Database management</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cmd" target="_blank">Collagen Mutation Database (CMD)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of osteogenesis imperfecta and type III collagen variants.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence variations : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>547</div></td>
      <td><div class="link short click_expand"><a href="https://eds.gene.le.ac.uk/docs/LOVD_manual_2.0.pdf" target="_blank">https://eds.gene.le.ac.uk/docs/LOVD_manual_2.0.pdf</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dpvweb" target="_blank">DPVweb database of plant viruses</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Information about viruses, viroids and satellites of plants, fungi and protozoa, with some additional data on related animal viruses.</div></td>
      <td><div class='topic short click_expand'><ul><li>Virology</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>organism report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>46</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/vkcdb" target="_blank">VKCDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Voltage-gated K(+) Channel DataBase contains full-length or nearly full-length unique channel sequences from Bacteria, Archaea and Eukaryotes. Corresponding nucleotide sequences of the open reading frames corresponding to the amino acid sequences are now available and can be extracted in parallel with sets of protein sequences. Channels are categorized into subfamilies by phylogenetic analysis and by using hidden Markov model analyses.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Phylogeny</li><li>Proteins</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein sequence analysis</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CellTag" target="_blank">CellTag</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genetic barcode-based sample multiplexing for single-cell genomics.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Cell biology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Community profiling</li><li>Parsing</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li><li>Workflow</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/morris-lab/CellTagWorkflow" target="_blank">https://github.com/morris-lab/CellTagWorkflow</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Sci-Hub" target="_blank">Sci-Hub</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sci-Hub provides access to nearly all scholarly literature.<br>Source code and data analyses for the Sci-Hub Coverage Study.<br>Analyses for the Sci-Hub Coverage Study.<br>Manuscript for the Sci-Hub Coverage Study.<br>Manuscript source for the Sci-Hub Coverage Study.<br>Data for the Sci-Hub Stats Browser.<br>removing barriers in the way of science.<br>Din bedste informationskilde om sci hub.<br>This project investigates the coverage of scholarly literature in the Sci-Hub and LibGen scimag repositories. It's based on using DOIs to uniquely identify articles. For more information, see the corresponding study:.<br>This is the manuscript source for the study titled Sci-Hub provides access to nearly all scholarly literature.<br>Sci-Hub is a pirate website that provides access to full texts from the scholarly literature, including paywalled articles.<br>The first pirate website in the world to open mass and public access to tens of millions research papers</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Database management</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Deposition</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://greenelab.github.io/scihub/#/journals" target="_blank">https://greenelab.github.io/scihub/#/journals</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pubchem" target="_blank">PubChem</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The PubChem BioAssay database is a public repository for biological activities of small molecules and small interfering RNAs (siRNAs). It archives experimental descriptions of assays and biological test results. A PubChem BioAssay data entry includes an assay description, a summary and detailed test results. Each assay record is linked to the molecular target, whenever possible, and is cross-referenced to other NCBI database records.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biotechnology</li><li>Functional, regulatory and non-coding RNA</li><li>RNA</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Data retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>580</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/orchid" target="_blank">Orchid</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Novel management, annotation, and machine learning framework for analyzing cancer mutations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Nucleic acid structure analysis</li><li>Genetic variation</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>SNP annotation</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tcdb" target="_blank">Transport classification database (TCDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive IUBMB approved classification system for membrane transport proteins known as the Transporter Classification (TC) system. The TC system is analogous to the Enzyme Commission (EC) system for classification of enzymes, but incorporates phylogenetic information additionally.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Membrane and lipoproteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein features : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>277</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hubmed" target="_blank">HubMed</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This tool uses information from PubMed's database, provided by the NCBI through the EUtils web service, to produce a search interface focused on browsing, organising and gathering information from the biomedical literature.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Natural language processing</li><li>Biomedical science</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Text annotation</li><li>Literature search</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>40</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/partigenedb" target="_blank">PartiGeneDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of about 300 partial genomes from eukaryotic organisms that have been assembled from EST data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Sequence assembly</li><li>Genomics</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>EST assembly</li><li>Sequence assembly</li><li>Query and retrieval</li><li>Genome assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/base" target="_blank">BASE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive free web-based database solution for the massive amounts of data generated by microarray analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>40</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biojs-vis-pmccitation" target="_blank">biojs-vis-pmccitation</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Europe PMC citations viewer. It retrieves and displays the principal data (such as authors list, title, journal and source) either of a single citation or of the citations list that matches the query specified in the Europe PMC.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Data acquisition</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mndr" target="_blank">MNDR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mammal ncRNA-disease repository with increased coverage and annotation.<br>Mammalian ncRNA-Disease Repository.<br>MNDR v3.1: Mammal NcRNA-Disease Repository.<br>MNDR (Mammalian ncRNA-Disease Repository) is a database that curated associations between ncRNA and disease. Here, we have updated the MNDR to version 3.1 by integrating experimentally supported and predicted ncRNA-disease associations curated from manual literatures and other resources. MNDR v3.1 currently contains more than one million ncRNA-disease entries, including 6,301 miRNAs, 39,880 lncRNAs, 20,256 circRNAs, 10,894 piRNAs and 521 snoRNAs with over 1,600 diseases, and an increase of species coverage to 11 mammals.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li><li>Literature and language</li><li>Pathology</li><li>Model organisms</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Subcellular localisation prediction</li><li>Query and retrieval</li><li>miRNA target prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/massarray" target="_blank">MassArray</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package is designed for the import, quality control, analysis, and visualization of methylation data generated using Sequenom's platform. The tools herein contain a highly detailed amplicon prediction for optimal assay design. Also included are quality control measures of data, such as primer dimer and bisulfite conversion efficiency estimation. Methylation data are calculated using the same algorithms contained in the EpiTyper software package.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenetics</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Loading</li><li>Methylation analysis</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>64</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MITRE" target="_blank">MITRE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Inferring features from microbiota time-series data linked to host status | The Microbiome Interpretable Temporal Rule Engine | Input code and results from simulations supporting the MITRE paper | Supporting files for the Microbiome Interpretable Temporal Rule Engine manuscript | MITRE learns predictive models of patient outcomes from microbiome time-series data in the form of short lists of interpretable rules | This repository includes the configuration files used to set up simulations for the MITRE manuscript, and results from those simulations, with a few useful auxiliary files | MITRE will load data from Bokulich, N. A., et al., "Antibiotics, birth mode, and diet shape microbiome maturation during early life." Science Translational Medicine 8(343): 343ra82, which is packaged with MITRE</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Microbial ecology</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Aggregation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/gerberlab/mitre_paper_results" target="_blank">https://github.com/gerberlab/mitre_paper_results</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cppredictor" target="_blank">cpPredictor</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Template-based predictor of RNA secondary structure. It takes a template RNA secondary structure(s) and a query RNA sequence(s) and generates secondary structure of the query sequence(s) using the provided structure(s) as a template(s). Biological reliability of the generated structures is evaluated in form of z-scores.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Nucleic acid structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Structure visualisation</li><li>RNA secondary structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>rna secondary structure : dot-bracket format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>statistical estimate score : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="http://cppredictor.elixir-czech.cz/predict" target="_blank">http://cppredictor.elixir-czech.cz/predict</a></div></td>
      <td><div class='short click_expand'>Freeware</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bartweb" target="_blank">BARTweb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This web server is to perform Binding Analysis for Regulation of Transcription (BART) is a bioinformatics tool for predicting functional factors (including transcription factors and chromatin regulators) that bind at cis-regulatory regions to regulate gene expression in human or mouse, taking a query gene list or a ChIP-seq dataset as input.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>ChIP-seq</li><li>Epigenomics</li><li>Gene regulation</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Gene-set enrichment analysis</li><li>cis-regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ngsReports" target="_blank">ngsReports</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Bioconductor package for managing FastQC reports and other NGS related log files.<br>The shiny app that accompanies the ngsReports R package</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Database management</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SEAMIS" target="_blank">SEAMIS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Relative sea-level data from the SEAMIS database compared to ICE-5G model predictions of glacial isostatic adjustment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Experimental design and studies</li><li>Database management</li><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Odonate_Phenotypic_Database" target="_blank">Odonate Phenotypic Database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>New open data resource for comparative studies of an old insect order.<br>The Odonate Phenotypic DataBase is a resource for dragonfly and damselfly phenotypes. The database consists of a variety of morphological, behavioural, and bio-geographical information collected from various sources.<br>Complete Database Variable Definitions</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Genotype and phenotype</li><li>Taxonomy</li><li>Ecology</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Deposition</li><li>Phylogenetic tree editing</li><li>Phylogenetic tree generation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/BacTag" target="_blank">BacTag</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Pipeline for fast and accurate gene and allele typing in bacterial sequencing data based on database preprocessing.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Whole genome sequencing</li><li>Public health and epidemiology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Multilocus sequence typing</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://git.lumc.nl/l.khachatryan/BacTag/issues" target="_blank">https://git.lumc.nl/l.khachatryan/BacTag/issues</a></div></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cld" target="_blank">CLD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Multispecies design of CRISPR/Cas9 libraries.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Gene structure</li><li>Genetics</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>29</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/LinearFold" target="_blank">LinearFold</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Linear-time approximate RNA folding by 5'-to-3' dynamic programming and beam search | The new source code of the LinearFold, linear-time prediction for RNA secondary structures | This codebase replaces the now deprecated version: https://github.com/abentu0101/LinearFold. This version fixes many bugs and design problems in the old version | Add a sequence Paste or type your sequence here (length < 100,000): | GGUUAAGCGACUAAGCGUACACGGUGGAUGCCCUGGCAGUCAGAGGCGAUGAAGG | Choose model(s) LinearFold-C (using CONTRAfold v2.0 machine-learned model, Do et al 2006) | * Liang Huang, **He Zhang, **Dezhong Deng, Kai Zhao, Kaibo Liu, David Hendrix, and David H. Mathews (2019). LinearFold: Linear-Time Approximate RNA Folding by 5’-to-3’ Dynamic Programming and Beam Search. Bioinformatics, Volume 35, Issue 14, July 2019, Pages i295–i304. ISMB 2019</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein secondary structure</li><li>Gene transcripts</li><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>RNA secondary structure prediction</li><li>Parsing</li><li>Loop modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/LinearFold/LinearFold" target="_blank">https://github.com/LinearFold/LinearFold</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cipres" target="_blank">CIPRes</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Project aims to develop a computational infrastructure for systematics. Other goals of the project include providing a central resource enabling computational systematics and education and training initiatives. The website also contains a substantial list of links to related software.</div></td>
      <td><div class='topic short click_expand'><ul><li>Software engineering</li><li>Computational biology</li><li>Computer science</li><li>Phylogenetics</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree generation</li><li>Phylogenetic tree generation (parsimony methods)</li><li>Phylogenetic tree analysis</li><li>Tree-based sequence alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5,548</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fastr3d" target="_blank">FASTR3D</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>FASTR3D is a web based search tool that allows users to search the PDB database for structurally similar RNAs from input queries of PDB coordinates for an RNA molecule, an RNA secondary structure or RNA primary sequence.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li><li>Gene transcripts</li><li>Nucleic acid structure analysis</li><li>RNA</li><li>Structure prediction</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Structure visualisation</li><li>RNA secondary structure prediction</li><li>RNA secondary structure alignment</li><li>RNA secondary structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>15</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lipidomer" target="_blank">lipidomeR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Understanding the Lipidome at the Systems Level with lipidomeR.<br>Integrative Visualizations of the Lipidome.</div></td>
      <td><div class='topic short click_expand'><ul><li>Lipids</li><li>Metabolomics</li><li>Workflows</li><li>Medicine</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Regression analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://cran.r-project.org/web/packages/lipidomeR/" target="_blank">https://cran.r-project.org/web/packages/lipidomeR/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scowl" target="_blank">SCOWL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PDB interface interactions at atom, residue and domain level. The database contains protein interfaces and residue-residue interactions formed by structural units and interacting solvent.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein interaction data : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ecid" target="_blank">EcID</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Results of running an "in silico 2 hybrid system" for all the possible pairs among E. coli proteins.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data reference : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>275</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/libmolgrid" target="_blank">libmolgrid</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Graphics Processing Unit Accelerated Molecular Gridding for Deep Learning Applications.<br>libmolgrid is under active development, but should be suitable for use by early adopters.<br>libmolgrid is a C++ library with Python bindings that generates tensors from molecular structures that are optimized for machine learning workflows.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Machine learning</li><li>Computational chemistry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/gnina/libmolgrid" target="_blank">https://github.com/gnina/libmolgrid</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/siglasso" target="_blank">sigLASSO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Using sigLASSO to optimize cancer mutation signatures jointly with sampling likelihood.<br>Optimizing Cancer Mutation Signatures Jointly with Sampling Likelihood.<br>SigLASSO considers both sampling error (especially important when the mutation count is low) and signature fitting.<br>Moreover, it parameterizes the model empirically. Let the data speak for itself. Moreover, you will be able to feed prior knowledge of the signatures into the model in a soft thresholding way. No more picking up signature subsets by hand! SigLASSO achieves signature selection by using L1 regularization.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Genetic variation</li><li>Machine learning</li><li>Exome sequencing</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Standardisation and normalisation</li><li>Parsing</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/emgb" target="_blank">EMGB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>EMGB is a CWL-based workflow for metagenomics analyses including a web-based interface for filtering ind querying analysis results</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Sequence feature comparison</li><li>Visualisation</li><li>Sequence comparison</li><li>Phylogenetic tree comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid sequence record : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence set : fasta-like (text)</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://bibiserv.cebitec.uni-bielefeld.de/resources/emgb/" target="_blank">https://bibiserv.cebitec.uni-bielefeld.de/resources/emgb/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/botxminer" target="_blank">botXminer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BotXminer is a literature searching tool that allows you to search BotDB.  BotDB citations contain only those articles with either 'botulinum' or 'tetanus' anywhere in their text.  botXminer is an interface to this subset of complete medline xml files loaded into an Oracle XMLDB.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Literature search</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/3SRP" target="_blank">3SRP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Snakemake-based pipeline for 3' sequencing RNA profiling data analysis.  This 3’ Digital gene expression sequencing technique allows a precise and low-cost transcriptome profiling.<br>The main steps of the pipeline are:<br>- Samples demultiplexing transform the raw paired-end fastq files into a single-end fastq file for each sample.<br>- Alignment on refseq reference transcriptome is performed using bwa.<br>- Aligned reads are parsed and UMI are counted for each gene in each sample to create an expression matrix.<br>If secondary analysis has been asked (providing a comparisons file), the expression matrix is normalized and differentially expressed genes (DEG) are searched using deseq2.<br>- If DEG are found, annotation is performed using the database GO and KEGG.<br>- A report is provided listing the main quality controls performed and the results found.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Heat map generation</li><li>Clustering profile plotting</li><li>RNA-seq read count analysis</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>rna sequence : fastq-illumina</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Workflow</li></ul></div></td>
      <td><div class='citations short click_expand'>58</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/protonet" target="_blank">ProtoNet</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Automatic hierarchical classification of protein sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein family report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>34</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tRNAviz" target="_blank">tRNAviz</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web application for exploring and visualizing tRNA sequence features.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li><li>Imaging</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>tRNA gene prediction</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/UCSC-LoweLab/tRNAviz" target="_blank">https://github.com/UCSC-LoweLab/tRNAviz</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/brassica_genome" target="_blank">Brassica Genome</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This resource hosts Brassica genome databases. B. rapa genome browser: including comprehensive annotation of the repeat fraction of the genome using the genome viewer GBrowse2. B. oleracea pan genome browser: including SNPs and functional annotation of genes. BlastGbrowse: A portal for blasting a query sequence to B. rapa and B. oleracea pan genome and allow conveniently link hits to the above Genome Browsers.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,256</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/FawMine" target="_blank">FawMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated Omics data warehouse for FAW, Spodoptera frugiperda</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>155</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/eupathdb" target="_blank">Eukaryotic Pathongen Database Resources (EuPathDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrated database covering the eukaryotic pathogens of the genera Cryptosporidium, Giardia, Leishmania, Neospora, Plasmodium, Toxoplasma, Trichomonas and Trypanosoma. The database portal offers an entry point to all these resources, and the opportunity to leverage orthology for searches across genera.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Zoology</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Generation</li><li>Query and retrieval</li><li>Deposition</li><li>Database search</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>116</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/coil" target="_blank">coil</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An R package for cytochrome C oxidase I (COI) DNA barcode data cleaning, translation, and error evaluation.<br>Contextualization and Evaluation of COI-5P Barcode Data.<br>Designed for the cleaning, contextualization and assessment of cytochrome c oxidase I DNA barcode data (COI-5P, or the five prime portion of COI). It contains functions for placing COI-5P barcode sequences into a common reading frame, translating DNA sequences to amino acids and for assessing the likelihood that a given barcode sequence includes an insertion or deletion error. The error assessment relies on the comparison of input sequences against nucleotide and amino acid profile hidden Markov models (for details see Durbin et al. 1998, ISBN: 9780521629713) trained on a taxonomically diverse set of reference sequences. The functions are provided as a complete pipeline and are also available individually for efficient and targeted analysis of barcode data</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein expression</li><li>Plant biology</li><li>Probes and primers</li><li>Zoology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Community profiling</li><li>Sequence trimming</li><li>Protein super-secondary structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gaev" target="_blank">GAEV</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Gene Annotation Easy Viewer. It integrates the gene annotation data from the KEGG (Kyoto Encyclopedia of Genes and Genomes) Automatic Annotation Server and generates an easy-to-read table that summarizes the query gene name, the KO (KEGG Orthology) number, name of gene orthologs, functional definition of the ortholog, and the functional pathways that query gene has been mapped to.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional genomics</li><li>Function analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://zenodo.org/record/1251973#.W2wqly1kjfY" target="_blank">https://zenodo.org/record/1251973#.W2wqly1kjfY</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genemania" target="_blank">GeneMANIA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web interface for generating hypotheses about gene function, analyzing gene lists and prioritizing genes for functional assays. Users input a query list for extension and weighting with functionally similar genes. Six organisms are supported.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Protein interactions</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene prediction</li><li>Gene-set enrichment analysis</li><li>Weighted correlation network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2,068</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/webprc" target="_blank">webPRC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web server for searching for distant homologs or similar alignment in domain databases. Results may be provided as either multiple sequence alignments or aligned hidden Markov models. An alignment editor allows for user editing of output alignments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Molecular modelling</li><li>Sequence analysis</li><li>Database management</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Multiple sequence alignment</li><li>Sequence alignment</li><li>Sequence profile alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>15</div></td>
      <td><div class="link short click_expand"><a href="https://jib.tools/details.php?id=107" target="_blank">https://jib.tools/details.php?id=107</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/epimir" target="_blank">EpimiR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of curated mutual regulation between miRNAs and epigenetic modifications.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenetics</li><li>Functional, regulatory and non-coding RNA</li><li>Gene regulation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>32</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fmm" target="_blank">FMM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>From Metabolite to Metabolite (FMM) is a web server for metabolic pathway reconstruction from one metabolite to another metabolite in a different species based on the KEGG pathway and other integrated databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Endocrinology and metabolism</li><li>Database management</li><li>Molecular interactions, pathways and networks</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>61</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/DeepCirCode" target="_blank">DeepCirCode</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R package for deep learning of the back-splicing code for circular RNA formation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Gene transcripts</li><li>RNA splicing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Splice transcript prediction</li><li>Enrichment analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/BioDataLearning/DeepCirCode/issues" target="_blank">https://github.com/BioDataLearning/DeepCirCode/issues</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/eva" target="_blank">European Variation Archive (EVA)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The European Variation Archive (EVA) accepts submission of, and provides access to, all types of genetic variants from any species, observed in germline or somatic sources, ranging from SNVs to large structural variants. All of our data is open access via direct query through web and/or programmatic interfaces.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li><li>Genetic variation</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Aggregation</li><li>Deposition</li><li>Data retrieval</li><li>Sequence analysis</li><li>Sequencing error detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence variations : vcf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence : vcf</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/EBIvariation/" target="_blank">https://github.com/EBIvariation/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dextmp" target="_blank">DextMP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Extracts three categories of textual information for a protein: titles, abstracts from literature, and function description in UniProt.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Proteins</li><li>Machine learning</li><li>Data acquisition</li><li>Human biology</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fluxmap" target="_blank">FluxMap</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Easy to use tool for the advanced visualization of simulated or measured flux data in biological networks. Flux data import is achieved via a structured template basing on intuitive reaction equations. Flux data is mapped onto any network and visualized using edge thickness. Various visualization options and interaction possibilities enable comparison and visual analysis of complex experimental setups in an interactive way.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>26</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/fluxmap-1-13-visual-exploration-flux-distributions-biological-networks.html" target="_blank">http://www.mybiosoftware.com/fluxmap-1-13-visual-exploration-flux-distributions-biological-networks.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/msfilereader_python_bindings" target="_blank">MSFileReader_Python_bindings</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Python bindings for the (non-free) ThermoFisher Scientific MSFileReader.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : thermo raw</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/959_nematode_genomes" target="_blank">959 Nematode Genomes</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A community-curated semantic wiki to coordinate the sequencing efforts of individual labs to collectively sequence 959 genomes spanning the phylum Nematoda. The main goal of the wiki is to track sequencing projects that have been proposed, are in progress, or have been completed.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Zoology</li><li>Sequence analysis</li><li>Genomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome assembly</li><li>Query and retrieval</li><li>Sequence coordinate conversion</li><li>Genotyping</li><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>35</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/slivar" target="_blank">slivar</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Effective variant filtering and expected candidate variant yield in studies of rare human disease.<br>slivar: filter/annotate variants in VCF/BCF format with simple expressions.<br>If you use slivar, please cite the paper.<br>slivar is a set of command-line tools that enables rapid querying and filtering of VCF files. It facilitates operations on trios and groups and allows arbitrary expressions using simple javascript.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Exome sequencing</li><li>Rare diseases</li><li>DNA polymorphism</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Variant filtering</li><li>Variant calling</li><li>Variant prioritisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/k-map" target="_blank">K-Map</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Novel and user-friendly web-based program that systematically connects a set of query kinases to kinase inhibitors based on quantitative profiles of the kinase inhibitor activities. Users can find kinase inhibitors for a set of query kinases (obtained from high-throughput “omics” experiments) or to reveal new interactions between kinases and kinase inhibitors for rational drug combination studies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Drug discovery</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>19</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/k-map-kinase-connectivity-map.html" target="_blank">http://www.mybiosoftware.com/k-map-kinase-connectivity-map.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/affycompatible" target="_blank">AffyCompatible</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package provides an interface to Affymetrix chip annotation and sample attribute files. The package allows an easy way for users to download and manage local data bases of Affynmetrix NetAffx annotation files. The package also provides access to GeneChip Operating System (GCOS) and GeneChip Command Console (AGCC)-compatible sample annotation files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tableparser" target="_blank">TableParser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Extract sequences from tab-delimited assembly alignment files (e.g. SimAssembly and AceParser output). Parse out contigs, singletons, and assembled reads into separate files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequences</li><li>Assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/tableparser-extract-sequences-tab-delimited-assembly-alignment-files.html" target="_blank">http://www.mybiosoftware.com/tableparser-extract-sequences-tab-delimited-assembly-alignment-files.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bronco" target="_blank">BRONCO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Biomedical entity Relation ONcology COrpus for extracting gene-variant-disease-drug relations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Natural language processing</li><li>Drug development</li><li>Genetic variation</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>19</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/experimenthub" target="_blank">ExperimentHub</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package provides a client for the BioConductor corresponding web resource, which is a central location where curated data from experiments, publications or training courses can be accessed. Each resource has associated metadata, tags and date of modification. The client creates and manages a local cache of files retrieved enabling quick and reproducible access.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/wiff2dta" target="_blank">wiff2dta</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Converts ABI WIFF to .dta.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : wiff format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : dta</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>30</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/blastalign" target="_blank">BlastAlign</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Uses NCBI Blast to align nucleotide sequences that have large indels or are difficult to align globally. It selects the most representative sequence from the input sequences and then extracts the blast query anchored multiple alignment. Alternatively, the user can choose which sequence to use as an anchor. It also outputs a matrix representing regions of homology along the sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequences</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Multiple sequence alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>55</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/blastalign-1-2-blast-align-problematic-nucleotide-sequences.html" target="_blank">http://www.mybiosoftware.com/blastalign-1-2-blast-align-problematic-nucleotide-sequences.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/id-taxer" target="_blank">ID-TaxER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integration of ecological responses to a soil bacterial 16S rRNA gene database.<br>Abstract High-throughput sequencing 16S rRNA gene surveys have enabled new insights into the diversity of soil bacteria, and furthered understanding of the ecological drivers of abundances across landscapes. However, current analytical approaches are of limited use in formalising syntheses of the ecological attributes of taxa discovered, because derived taxonomic units are typically unique to individual studies and sequence identification databases only characterise taxonomy. To address this, we used sequences obtained from a large nationwide soil survey (GB Countryside Survey, henceforth CS) to create a comprehensive soil specific 16S reference database, with coupled ecological information derived from the survey metadata.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Taxonomy</li><li>Functional, regulatory and non-coding RNA</li><li>Ecology</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Community profiling</li><li>Query and retrieval</li><li>Regression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biogrid" target="_blank">The BioGRID Interaction Database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Public database that archives and disseminates genetic and protein interaction data from model organisms and humans. Complete coverage of the entire literature is maintained for budding yeast (Saccharomyces cerevisiae), fission yeast (Schizosaccharomyces pombe) and thale cress (Arabidopsis thaliana).</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network comparison</li><li>Pathway or network prediction</li><li>Query and retrieval</li><li>Protein interaction prediction</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>677</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pyfastx" target="_blank">pyfastx</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The pyfastx is a lightweight Python C extension that enables users to randomly access to sequences from plain and gzipped FASTA Q files.<br>All tools support for random access were used to perform benchmark.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>Transcription factors and regulatory sites</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Splitting</li><li>Reverse complement</li><li>Deposition</li><li>Data retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/lmdu/pyfastx" target="_blank">https://github.com/lmdu/pyfastx</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/datanator" target="_blank">Datanator</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Datanator is an integrated database of genomic and biochemical data designed to help investigators find data about specific molecules and reactions in specific organisms and specific environments for meta-analyses and mechanistic models. Datanator currently includes metabolite concentrations, RNA modifications and half-lives, protein abundances and modifications, and reaction kinetics integrated from several databases and numerous publications. The Datanator website and REST API provide tools for extracting clouds of data about specific molecules and reactions in specific organisms and specific environments, as well as data about similar molecules and reactions in taxonomically similar organisms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biological databases</li><li>Computational biology</li><li>Bioinformatics</li><li>Molecular interactions, pathways and networks</li><li>Systems biology</li><li>Molecular biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/KarrLab/datanator_frontend/issues" target="_blank">https://github.com/KarrLab/datanator_frontend/issues</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/few-shot_segmentation" target="_blank">few-shot segmentation</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>'Squeeze & excite' guided few-shot segmentation of volumetric images.<br>PyTorch implementation of 'Squeeze and Excite' Guided Few Shot Segmentation of Volumetric Scans.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Machine learning</li><li>Medical imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Image annotation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/abhi4ssj/few-shot-segmentation/issues" target="_blank">https://github.com/abhi4ssj/few-shot-segmentation/issues</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ewastools" target="_blank">ewastools</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This tool contains Infinium Human Methylation BeadChip pipeline that can be installed and used inside the Galaxy. The pipeline steps include: raw intensity data loading .idat preprocessing, optional normalisation of the data, quality control with additional sample check. It gives the user the opportunity to perform any of these preparation and data cleaning steps, including the next highly recommended genetic variation annotation step resulting in single nucleotide polymorphism identification and removal by simply running the tool. Finally, the dataset generated through all of these steps can be used to hunt find differentially-methylated positions DMP and regions DMR with respect to a phenotype co-variate.</div></td>
      <td><div class='topic short click_expand'><ul></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/easer" target="_blank">EASER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Python program (easer.py) based in PyCogent library for the homology-based retrieval of several sequence data types from Ensembl. A Perl program (enamer.pl) for the identification of sequence descriptions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular genetics</li><li>Data acquisition</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Workflow</li><li>Command-line tool</li><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/stanfordmicroarraydb" target="_blank">Stanford Microarray Database (SMD)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Stores raw and normalized data from microarray experiments, as well as their corresponding image files. In addition, it provides interfaces for data retrieval, analysis and visualization.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data architecture, analysis and design</li><li>Imaging</li><li>Gene expression</li><li>Microarray experiment</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein interaction analysis</li><li>Deposition</li><li>Data retrieval</li><li>Enrichment analysis</li><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>642</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cyto" target="_blank">Cyto</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Cyto is a docker-based application to run Anduril analysis pipelines on high dimensional cytometry data. Cyto aims to make available new computational cytometry tools to scientists with and without a bioinformatics background. Writing analysis pipelines using the workflow platform Anduril enables language-agnostic integration of new methods, Docker allows multi-OS compatibility, and Python libraries (flask and dash) allow user interaction.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cytometry</li><li>Proteomics</li><li>Protein expression</li><li>Imaging</li><li>Workflows</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/panelapp" target="_blank">PanelApp</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomics England PanelApp is a publically-available knowledgebase that allows virtual gene panels related to human disorders to be created, stored and queried. It includes a crowdsourcing tool that allows genes to be added or reviewed by experts throughout the worldwide scientific community, providing an opportunity for the standardisation of gene panels, and a consensus on which genes have sufficient evidence for disease association.</div></td>
      <td><div class='topic short click_expand'><ul><li>Personalised medicine</li><li>Medicine</li><li>Exome sequencing</li><li>Rare diseases</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Classification</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://panelapp.genomicsengland.co.uk/" target="_blank">https://panelapp.genomicsengland.co.uk/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/BovineMine" target="_blank">BovineMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated data warehouse for the Bovine Genome Database</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CoExpresso" target="_blank">CoExpresso</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CoExpresso is a tool to investigate the behavior of protein subunits in known complexes by comparing their<br>abundance profiles across up to 140 cell types available in ProteomicsDB. Thorough assessment of different<br>randomization methods and statistical scoring algorithms allows determining the significance of concurrent profiles<br>within a complex, therefore providing insights into the conservation of their composition across human cell types as<br>well as the identification of intrinsic structures in complex behavior to determine which proteins orchestrate complex<br>function.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li><li>Proteomics experiment</li><li>Proteomics</li><li>Protein interactions</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Expression analysis</li><li>Expression data visualisation</li><li>Statistical calculation</li><li>Query and retrieval</li><li>Expression correlation analysis</li><li>Heat map generation</li><li>Pathway or network visualisation</li><li>Clustering profile plotting</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene id : uniprotkb format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>plot : pdf</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="http://computproteomics.bmb.sdu.dk/" target="_blank">http://computproteomics.bmb.sdu.dk/</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/flowmapr" target="_blank">FLOWMAPR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>FLOWMAPR is an R package  for extensible analysis of single-cell datasets including FCS files generated from mass/flow cytometry or data.frames of other single-cell data processed in R.<br>It is a graph-based, force-directed layout algorithm for trajectory mapping in single-cell time course datasets.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Cytometry</li><li>Cell biology</li><li>Systems biology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li><li>Dimensionality reduction</li><li>Essential dynamics</li><li>Scaffolding</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/reindeer" target="_blank">REINDEER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Efficient indexing of k-mer presence and abundance in sequencing datasets.<br>REINDEER builds a data-structure that indexes k-mers and their abundances in a collection of datasets (raw RNA-seq or metagenomic reads for instance). Then, a sequence (FASTA) can be queried for its presence and abundance in each indexed dataset. While other tools (e.g. SBT, BIGSI) were also designed for large-scale k-mer presence/absence queries, retrieving abundances was so far unsupported (except for single datasets, e.g. using some k-mer counters like KMC, Jellyfish). REINDEER combines fast queries, small index size, and low memory footprint during indexing and queries. We showed it allows to index 2585 RNA-seq datasets (~4 billions k-mers) using less than 60GB of RAM and a final index size lower than 60GB on the disk. Then, a REINDEER index can either be queried on disk (experimental feature, low RAM usage) or be loaded in RAM for faster queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Metagenomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>k-mer counting</li><li>De-novo assembly</li><li>Quantification</li><li>Data retrieval</li><li>Genome indexing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_GeneSearch" target="_blank">IMGT GeneSearch</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/GeneSearch is the IMGT® tool allowing the search of a gene in IMGT/LocusView. Search is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunogenetics</li><li>Immunoinformatics</li><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>map : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>354</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/webflags" target="_blank">webFlaGs</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Predicting protein functional association by analysis of conservation of genomic context (Flanking Genes). discovering novel biology through the analysis of gene neighbourhood conservation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Cell biology</li><li>Gene structure</li><li>Model organisms</li><li>Sequence assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Phylogenetic tree visualisation</li><li>Phylogenetic tree annotation</li><li>Operon prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/GCA-VH-lab/FlaGs" target="_blank">https://github.com/GCA-VH-lab/FlaGs</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iRODS" target="_blank">iRODS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrated Rule-Oriented Data System (iRODS) is open source data management software for a cancer genome analysis workflow.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Data security</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Parsing</li><li>Service management</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MapStore" target="_blank">MapStore</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The opendata geoportal of the Lamma Consortium.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Database management</li><li>Data security</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phasing</li><li>Filtering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/elmmatcher" target="_blank">ELMMatcher</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mapping of ELM (Eucaryotic Linear Motifs) motifs to a query sequence.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>484</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/152" target="_blank">https://www.biocatalogue.org/services/152</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/exocarta" target="_blank">ExoCarta</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Manually curated database of exosomal proteins, RNA, and lipids. Catalogs information from both published and unpublished exosomal studies. Currently contains info on 11,261 protein entries, 2375 mRNA entries, and 764 miRNA entries obtained from 134 exosomal studies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Lipids</li><li>Gene transcripts</li><li>Functional, regulatory and non-coding RNA</li><li>Biophysics</li><li>Experimental design and studies</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Deposition</li><li>Enrichment analysis</li><li>Transcriptional regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>577</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/araport" target="_blank">Araport</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Resource for Arabidopsis thaliana genomics. Araport offers gene and protein reports with orthology, expression, interactions and the latest annotation, plus analysis tools, community apps, and web services</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Indexing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>222</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/Arabidopsis-Information-Portal?page=2" target="_blank">https://github.com/Arabidopsis-Information-Portal?page=2</a></div></td>
      <td><div class='short click_expand'>Not licensed</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/unigene" target="_blank">UniGene gene-oriented nucleotide sequence clusters</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Each entry is a set of transcript (expressed sequence tag) sequences that appear to come from the same transcription locus (gene or expressed pseudogene), together with information on protein similarities, gene expression, cDNA clone reagents, and genomic location.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Gene and protein families</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene transcript report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/htp-oligodesigner" target="_blank">HTP-OligoDesigner</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Created to provide the scientific community with a simple and intuitive online primer design tool for both laboratory-scale and high-throughput projects of sequence-independent gene cloning and site-directed mutagenesis and a Tm calculator for quick queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular genetics</li><li>PCR experiment</li><li>Probes and primers</li><li>Nucleic acids</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>PCR primer design</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/htp-oligodesigner-an-online-primer-design-tool.html" target="_blank">http://www.mybiosoftware.com/htp-oligodesigner-an-online-primer-design-tool.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PlaSMo" target="_blank">PlaSMo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PlaSMo stands for Plant Systems-biology Modelling and the mission of the project was ensuring the achievements of yesterday's Mathematical Modellers will be available for the Systems Biologists of tomorrow.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Database management</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Data retrieval</li><li>Loop modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://plasmo.ed.ac.uk/plasmo/models/model.shtml?accession=PLM_64&version=3" target="_blank">http://plasmo.ed.ac.uk/plasmo/models/model.shtml?accession=PLM_64&version=3</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lailaps-qsm" target="_blank">LAILAPS-QSM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A machine learning approach for optimizing data searching and filtering from life-science databases. It is integrated in the LAILAPS search engine.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Machine learning</li><li>Natural language processing</li><li>Data mining</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Query and retrieval</li><li>Optimisation and refinement</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Plug-in</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mmb_api" target="_blank">MMB-IRB Data Repository API</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>REST API to access MMB-IRB data repo (PDB, Uniprot, Protein Variants, NA Features)</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>het group name : plain text format (unformatted)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>small molecule structure : het group dictionary entry format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/epd" target="_blank">EPD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An annotated non-redundant collection of eukaryotic POL II promoters, for which the transcription start site has been determined experimentally. Access to promoter sequences is provided by pointers to positions in nucleotide sequence entries. The annotation part of an entry includes description of the initiation site mapping data, cross-references to other databases, and bibliographic references.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>56</div></td>
      <td><div class="link short click_expand"><a href="http://expasy.org/" target="_blank">http://expasy.org/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/color-data" target="_blank">Color Data</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Color Data is a user-friendly, open-access database with hereditary cancer and hereditary cardiovascular conditions datasets. Color Data is the largest public database of its kind, containing aggregated genetic and phenotypic information from 54,000 individuals who took a Color test.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Cardiology</li><li>Genotype and phenotype</li><li>Pathology</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/smgea" target="_blank">SMGEA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A New Ensemble Adversarial Attack Powered by Long-Term Gradient Memories.<br>Deep neural networks are vulnerable to adversarial attacks. More importantly, some adversarial examples crafted against an ensemble of source models transfer to other target models and, thus, pose a security threat to black-box applications (when attackers have no access to the target models). Current transfer-based ensemble attacks, however, only consider a limited number of source models to craft an adversarial example and, thus, obtain poor transferability. Besides, recent query-based black-box attacks, which require numerous queries to the target model, not only come under suspicion by the target model but also cause expensive query cost. In this article, we propose a novel transfer-based black-box attack, dubbed serial-minigroup-ensemble-attack (SMGEA)</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_PRIMER-DB" target="_blank">IMGT PRIMER-DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/PRIMER-DB is the IMGT® database for oligonucleotides (primers) of immunoglobulins (IG) and T cell receptors (TR).<br>Annotation is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunogenetics</li><li>Immunoinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid sequence : html</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid sequence : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>354</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hfa" target="_blank">HFA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Harmonic Feature Activation for Few-Shot Semantic Segmentation.<br>Few-shot semantic segmentation remains an open problem because limited support (training) images are insufficient to represent the diverse semantics within target categories.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/selectome" target="_blank">Selectome</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of positive selection based on a rigorous branch-site specific likelihood test.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>34</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lpcf" target="_blank">LPCF</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Allows the user to quickly find a high-quality community close to a queried protein in any network available from BioGRID or specified by the user.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Gene and protein families</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein identification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>47</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/lpcf-local-protein-community-finder.html" target="_blank">http://www.mybiosoftware.com/lpcf-local-protein-community-finder.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dampd" target="_blank">DAMPD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Manually curated database of known and putative antimicrobial peptides.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Gene transcripts</li><li>Data submission, annotation and curation</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Peptide database search</li><li>Query and retrieval</li><li>Database search</li><li>Blind peptide database search</li><li>Antimicrobial resistance prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>70</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MaizeSNPDB" target="_blank">MaizeSNPDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive database for efficient retrieve and analysis of SNPs among 1210 maize lines.<br>This R script chara-2-sparse.R is used to convert a traditional character genotype matrix into an integer sparse genotype matrix. This R script can be used in the system command line with R installed in the system or be used in the R enviroment.<br>test.snp.RData is the input data to this R script. test.snp.RData is an R data file contained the genotype data of 1210 maize lines across 1000 SNP sites in character matrix. The content of this R data can be viewed in the R enviroment as shown below.<br>MaizeSNPDB is deployed at http://150.109.59.144:3838/MaizeSNPDB/ for online use.<br>The genotypes of 1210 maize lines at 35,370,939 SNP sites are stored using Sparse Matrices in R.<br>MaizeSNPDB is idle until you activate it by accessing the URL.<br>After running the R script, a new R data file named as test.snp. Mat.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA polymorphism</li><li>Genotype and phenotype</li><li>Database management</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Deposition</li><li>Data retrieval</li><li>Genetic mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pmidcite" target="_blank">pmidcite</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Evaluating retrieval qualities of Google Scholar and PubMed.<br>Augment your PubMed literature search from the command-line by linking data downloaded from NIH's Open Citation Collection (NIH-OCC), which includes scientific influence, translation, and citation counts, with PubMed IDs (PMIDs), rather than clicking and clicking and clicking on Google Scholar Cited by N links.<br>1) Get citation counts, given PMIDs.<br>Command-line users can immediately augment their PubMed search results using the pmidcite scripts and library, which download citation data from the National Institutes of Health Open Citation Collection (NIH-OCC) database using NIH's "iCite" Application Programming Interface (API). We have found PubMed plus NIH-OCC citation data combined using pmidcite is a "Better" experience than using Google Scholar if your research is in the health sciences and you are amenable to consider working from the command line.<br>2) Sort citation counts, given PMIDs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Database management</li><li>Chemistry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Literature search</li><li>Data retrieval</li><li>Sorting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>LGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CATANA" target="_blank">CATANA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool for generating comprehensive annotations of alternative transcript events.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li><li>Gene transcripts</li><li>RNA splicing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Splice transcript prediction</li><li>RNA-Seq quantification</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/shiauck/CATANA/issues" target="_blank">https://github.com/shiauck/CATANA/issues</a></div></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/proteosushi" target="_blank">ProteoSushi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A software tool to biologically annotate and quantify modification-specific, peptide-centric proteomics datasets.<br>ProteoSushi transforms peptide-centric, PTM-enriched peptide data into condensed and annotated PTM site output that is easy to understand and analyze.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Protein modifications</li><li>Proteomics</li><li>Sequence analysis</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Quantification</li><li>Data retrieval</li><li>PTM localisation</li><li>Parsing</li><li>PTM site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_GeneInfo" target="_blank">IMGT GeneInfo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/GeneInfo is the IMGT® tool providing combination (V-J and V-V) information for the human and mouse T cell receptor (TR) loci. Analysis is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunogenetics</li><li>Immunoinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>keyword : html</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid sequence : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>243</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pfam" target="_blank">Pfam</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database which contains information about protein domains and families. For each entry a protein sequence alignment and a Hidden Markov Model is stored.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence composition, complexity and repeats</li><li>Proteins</li><li>Gene and protein families</li><li>Sequence analysis</li><li>Sequence sites, features and motifs</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein sequence : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>859</div></td>
      <td><div class="link short click_expand"><a href="http://pfam.sbc.su.se/" target="_blank">http://pfam.sbc.su.se/</a></div></td>
      <td><div class='short click_expand'>CC0-1.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/interminer" target="_blank">InterMineR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Databases based on the InterMine platform such as FlyMine, modMine (modENCODE), RatMine, YeastMine, HumanMine and TargetMine are integrated databases of genomic, expression and protein data for various organisms. Integrating data makes it possible to run sophisticated data mining queries that span domains of biological knowledge.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA splicing</li><li>Proteomics</li><li>Gene expression</li><li>Microarray experiment</li><li>GWAS study</li><li>Systems biology</li><li>Comparative genomics</li><li>Gene regulation</li><li>Functional genomics</li><li>DNA polymorphism</li><li>Function analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>152</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/InterMineR/" target="_blank">http://bioconductor.org/packages/InterMineR/</a></div></td>
      <td><div class='short click_expand'>LGPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pedican" target="_blank">Pedican</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An online gene resource for pediatric cancers with literature evidence.</div></td>
      <td><div class='topic short click_expand'><ul><li>Human biology</li><li>Oncology</li><li>DNA structural variation</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cdh" target="_blank">CDH</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Co-regulation Data Harvester (CDH) begins with a gene of interest and identifies co-regulated genes by accessing the Tetrahymena transcriptome database. It then identifies their closely related genes (orthologs) in other organisms by using reciprocal BLAST searches. Finally, it collates the annotations of those orthologs’ functions, which provides the user with information to help predict the cellular role of the initial query.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenomics</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ElsevierSoftwareX/SOFTX-D-17-00024/issues" target="_blank">https://github.com/ElsevierSoftwareX/SOFTX-D-17-00024/issues</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mmssimilarity" target="_blank">MMsSimilarity</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-oriented tool developed with the specific aim to calculate chemical similarity indexes between a query structure and a chemical database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Chemical similarity enrichment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/centomd" target="_blank">CentoMD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genetic mutation database for rare diseases that enables to search, sort and filter genetic variants from clinically diagnosed individuals worldwide and make symptoms-based queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Rare diseases</li><li>Genotype and phenotype</li><li>Genetic variation</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>SNP annotation</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>30</div></td>
      <td></td>
      <td><div class='short click_expand'>Proprietary</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genestoryteller" target="_blank">GeneStoryTeller</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mobile application that provides users with a synopsis regarding the information of human genes from various points of view. Synopsis is given as a little story, while, users may perform queries regarding a gene of interest. It is developed for Android and provides general, functional, clinical and network information for known and predicted human genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular genetics</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/genestoryteller-mobile-app-for-quick-and-comprehensive-information-retrieval-of-human-genes.html" target="_blank">http://www.mybiosoftware.com/genestoryteller-mobile-app-for-quick-and-comprehensive-information-retrieval-of-human-genes.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/presa2i" target="_blank">PRESa2i</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Incremental decision trees for prediction of Adenosine to Inosine RNA editing sites.<br>This repository contains datasets for the paper titled "Incremental Decision Trees for Prediction of Adenosine to Inosine RNA Editing Sites".<br>All Features.csv - Benchmark Dataset After Feature Extraction.<br>Enter the query RNA sequences here Predict Clear.<br>Selected Features.csv - Benchmark Dataset After Feature Selection.<br>Insert RNA string of any length.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Sequence sites, features and motifs</li><li>RNA-Seq</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Feature selection</li><li>Editing</li><li>RNA structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/swakkhar/RNA-Editing/" target="_blank">https://github.com/swakkhar/RNA-Editing/</a></div></td>
      <td><div class='short click_expand'>CC0-1.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/goslin" target="_blank">Goslin</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Goslin is the Grammar of succinct lipid nomenclature project. It defines multiple grammers compatible with ANTLRv4 for different sources of shorthand lipid nomenclature. This allows to generate parsers based on the defined grammars, which provide immediate feedback whether a processed lipid shorthand notation string is compliant with a particular grammar, or not.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Lipids</li><li>Metabolomics</li><li>Cheminformatics</li><li>Computational chemistry</li><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>molecule name : tsv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>molecule name : tsv</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Bioinformatics portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/lifs-tools/goslin" target="_blank">https://github.com/lifs-tools/goslin</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/swotein" target="_blank">SWOTein</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A structure-based approach to predict stability Strengths and Weaknesses of prOTEINs.<br>SWOTein method evaluates the contribution of each residue in a protein (or protein chain) to its overall stability. The predictions are based on the three dimensional structure of the protein. Residues that contribute very unfavorably to the folding free energy are defined as stability weaknesses and colored in a red scale, whereas residues that show a highly stabilizing contribution are called stability strengths and colored in blu.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein folding, stability and design</li><li>Physiology</li><li>Protein interactions</li><li>Structure prediction</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Fold recognition</li><li>Protein structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ArrowSAM" target="_blank">ArrowSAM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>In-Memory Genomics Data Processing through Apache Arrow Framework | Apache Arrow is a cross-language development platform for in-memory data. It specifies a standardized language-independent columnar memory format for flat and hierarchical data, organized for efficient analytic operations on modern hardware. It also provides computational libraries and zero-copy streaming messaging and interprocess communication. Languages currently supported include C, C++, C#, Go, Java, JavaScript, MATLAB, Python, R, Ruby, and Rust | A cross-language development platform for in-memory data | Join Mailing List Install (0.15.1 Release - 1 November 2019)</div></td>
      <td><div class='topic short click_expand'><ul><li>Whole genome sequencing</li><li>Exome sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/apache/arrow" target="_blank">https://github.com/apache/arrow</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mirgen" target="_blank">MirGeneDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A miRBase derived database for microRNA genes that have been validated and annotated. Currently microRNA genes are available for human, mouse, chicken and zebrafish and can be browsed, blasted and downloaded.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Sequence visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>rna secondary structure : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>245</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/VMD_Store" target="_blank">VMD Store</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>VMD Plugin to Browse, Discover, and Install VMD Extensions.<br>In this page, you can find all the possible alternatives to install the VMD Store extension. 1. Install VMD Store through VMD Tk Console This installation process can be used ...<br>Biomolecular SIMulations Research Group</div></td>
      <td><div class='topic short click_expand'><ul><li>Biomolecular simulation</li><li>Database management</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Aggregation</li><li>Molecular dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/locuszoom" target="_blank">LocusZoom</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Interactive and embeddable visualization of genetic association study results.<br>LocusZoom is a Javascript/d3 embeddable plugin for interactively visualizing statistical genetic data from customizable sources.<br>To see functional examples of plots generated with LocusZoom.js see statgen.github.io/locuszoom and statgen.github.io/locuszoom/#examples.<br>A Javascript/d3 embeddable plugin for interactively visualizing and navigating statistical genetic data from customizable sources.<br>See github.com/statgen/locuszoom/wiki for full documentation and API reference.<br>Shown here is a standard interactive LocusZoom plot, depicting GWAS association data with gene tracks loaded from a University of Michigan API.<br>The LocusZoom.js library provides several pre-defined layouts for entire plots and subdivisions of plots such as panels, data layers, tool tips, etc.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Sequencing</li><li>Mapping</li><li>GWAS study</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li><li>Anonymisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://statgen.github.io/locuszoom/" target="_blank">https://statgen.github.io/locuszoom/</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/alias" target="_blank">Alias</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A tool for converting identifiers in which multiple aliases are used to refer to sequences. Also available as a stand-alone tool.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Literature and language</li><li>Sequence assembly</li><li>Sequence analysis</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Multiple sequence alignment</li><li>Sequence assembly</li><li>Protein sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>19</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scirpy" target="_blank">Scirpy</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Scanpy extension for analyzing single-cell T-cell receptor sequencing data.<br>Scirpy: A Scanpy extension for analyzing single-cell immune-cell receptor sequencing data.<br>Scirpy is a scalable python-toolkit to analyse T cell receptor (TCR) or B cell receptor (BCR) repertoires from single-cell RNA sequencing (scRNA-seq) data. It seamlessly integrates with the popular scanpy library and provides various modules for data import, analysis and visualization.<br>Tutorials — scirpy documentation.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Infectious disease</li><li>Genotype and phenotype</li><li>Immunoproteins and antigens</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SingleCellNet" target="_blank">SingleCellNet</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>> HOMEPAGE MISSING! | A Computational Tool to Classify Single Cell RNA-Seq Data Across Platforms and Across Species | Single-cell RNA-seq has emerged as a powerful tool in diverse applications, from determining the cell-type composition of tissues to uncovering regulators of developmental programs. A near-universal step in the analysis of single-cell RNA-seq data is to hypothesize the identity of each cell. Often, this is achieved by searching for combinations of genes that have previously been implicated as being cell-type specific, an approach that is not quantitative and does not explicitly take advantage of other single-cell RNA-seq studies. Here, we describe our tool, SingleCellNet, which addresses these issues and enables the classification of query single-cell RNA-seq data in comparison to reference single-cell RNA-seq data. SingleCellNet compares favorably to other methods in sensitivity and specificity, and it is able to classify across platforms and species</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>RNA-Seq quantification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pepatac" target="_blank">PEPATAC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An optimized ATAC-seq pipeline with serial alignments.<br>PEPATAC is a pipeline designed to process ATAC-seq data. For more information see: http://pepatac.databio.org/.<br>PEPATAC is an ATAC-seq pipeline. It trims adapters, maps reads, calls peaks, and creates bigwig tracks, TSS enrichment files, and other outputs. It is optimized on unique features of ATAC-seq data to be fast and accurate and provides several unique analytical approaches.<br>An open structure and toolkit for organizing large-scale, sample-intensive biological research projects.<br>PEP, or Portable Encapsulated Projects, is a community effort to make sample metadata reusable. PEPs decouple metadata from analysis-</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Sequence assembly</li><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence trimming</li><li>Peak calling</li><li>Parsing</li><li>Sequence assembly validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://pep.databio.org" target="_blank">http://pep.databio.org</a></div></td>
      <td><div class='short click_expand'>BSD-2-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SoS" target="_blank">SoS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Script of Scripts (SoS) is a computational environment for the development and execution of scripts in multiple languages for daily computational research.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Environmental science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Deposition</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Workflow</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/vatlab/SoS/issues" target="_blank">https://github.com/vatlab/SoS/issues</a></div></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pariga" target="_blank">Pariga</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Performs BLAST searches on two sets of sequences selected by the user, storing the output in a python-serialized-objects database. Results can be interrogated by the user using logical operations, for example to retrieve cases where two queries match same targets, or when sequences from the two datasets are reciprocal best hits, or when a query matches a target in multiple regions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Local alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tassdb" target="_blank">Tandem splice site database (TassDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This tool stores extensive data about alternative splice events at GYNGYN donors and NAGNAG acceptors.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene structure</li><li>Nucleic acid sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid features : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>26</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mobidb" target="_blank">MobiDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of protein disorder and mobility annotations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structure analysis</li><li>Protein disordered structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>uniprot keyword : plain text format (unformatted)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence features : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>348</div></td>
      <td></td>
      <td><div class='short click_expand'>CC-BY-NC-1.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/covid-19-scenarios" target="_blank">COVID-19 Scenarios</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>COVID-19 Scenarios is an interactive tool to explore the spread and associated morbidity and mortality of SARS-CoV-2. This tool is based on the SIR model  that simulates a COVID19 outbreak. <br>The primary purpose of the tool is to explore the dynamics of COVID19 cases and the associated strain on the health care system in the near future.</div></td>
      <td><div class='topic short click_expand'><ul><li>Infectious disease</li><li>Public health and epidemiology</li><li>Mathematics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Editing</li><li>Parsing</li><li>Modelling and simulation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://github.com/neherlab/covid19_scenarios" target="_blank">http://github.com/neherlab/covid19_scenarios</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aceview" target="_blank">AceView genes database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A curated, comprehensive and non-redundant sequence representation of all public mRNA sequences (mRNAs from GenBank or RefSeq, and single pass cDNA sequences from dbEST and Trace).</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>234</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/seqretsetall" target="_blank">seqretsetall</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Read and write (return) many sets of sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cyanobase" target="_blank">CyanoBase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CyanoBase is the genome database for cyanobacteria, which are model organisms for photosynthesis. The database houses cyanobacteria species information, complete genome sequences, genome-scale experiment data, gene information, gene annotations and mutant information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Genomics</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene functional annotation</li><li>Genome annotation</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>112</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mptopoquerier" target="_blank">MPtopoQuerier</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Java applet developed for searching MPtopo, a curated database of membrane proteins with experimentally validated transmembrane (TM) segments. Retrieved sequences can be examined automatically by MPEx, a java-based hydropathy-plot tool.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Membrane and lipoproteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>141</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mptopoquerier-search-database-membrane-proteins.html" target="_blank">http://www.mybiosoftware.com/mptopoquerier-search-database-membrane-proteins.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/readqpcr" target="_blank">ReadqPCR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The package provides functions to read raw RT-qPCR data of different platforms.</div></td>
      <td><div class='topic short click_expand'><ul><li>PCR experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>84</div></td>
      <td><div class="link short click_expand"><a href="http://www.bioconductor.org/packages/release/bioc/html/ReadqPCR.html" target="_blank">http://www.bioconductor.org/packages/release/bioc/html/ReadqPCR.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mouseindeldb" target="_blank">MouseIndelDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrated database resource containing thousands of previously unreported mouse genomic indel (insertion and deletion) polymorphisms ranging from approximately 100 nt to 10 Kb in size. The database currently includes polymorphisms identified from our alignment of whole-genome shotgun sequence traces from four laboratory mouse strains mapped against the reference C57BL/6J genome using GMAP.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>DNA polymorphism</li><li>Genetic variation</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence tagged site (STS) mapping</li><li>Query and retrieval</li><li>Mapping</li><li>Genome feature comparison</li><li>Genotyping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scarches" target="_blank">scArches</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ScArches is a package to integrate newly produced single-cell datasets into integrated reference atlases. Our method can facilitate large collaborative projects with decentralized training and integration of multiple datasets by different groups. scArches is compatible with scanpy. and hosts efficient implementations of several conditional generative models for single-cell data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li><li>Proteomics</li><li>Surgery</li><li>Machine learning</li><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Standardisation and normalisation</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/theislab/scarches-reproducibility" target="_blank">https://github.com/theislab/scarches-reproducibility</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tcga-assembler_2" target="_blank">TCGA-Assembler 2</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Software Pipeline for Automatic Retrieval, Processing, and Integration of TCGA/CPTAC Data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Sequence assembly</li><li>Data acquisition</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>68</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/psamm" target="_blank">PSAMM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Open source software that is designed for the curation and analysis of metabolic models. It supports model version tracking, model annotation, data integration, data parsing and formatting, consistency checking, automatic gap filling, and model simulations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/proteomexchange" target="_blank">ProteomeXchange</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This consortium has been set up to provide a globally coordinated submission of mass spectrometry proteomics data to the main existing proteomics repositories, and to encourage optimal data dissemination.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Sequence analysis</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein sequence analysis</li><li>Data handling</li><li>Query and retrieval</li><li>Deposition</li><li>Spectral analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1,857</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PyRadiomics" target="_blank">PyRadiomics</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PyRadiomics is an ontology-guided radiomics analysis workflow (O-RAW).</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical imaging</li><li>Oncology</li><li>Genotype and phenotype</li><li>Workflows</li><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/giardiadb" target="_blank">Giardia genome resources (GiardiaDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomic-scale datasets for the eukaryotic pathogen Giardia.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>158</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/FosSahul" target="_blank">FosSahul</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database for the late quaternary fossil records of Sahul along with data processing scripts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Biodiversity</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/imgt_hla" target="_blank">IMGT database of human MHC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides a searchable repository of highly curated HLA sequences. The naming of these HLA genes and alleles and their quality control is the responsibility of the WHO Nomenclature Committee for Factors of the HLA System. Through the work of the HLA Informatics Group and in collaboration with the European Bioinformatics Institute, we are able to provide public access to this data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunoproteins and antigens</li><li>Proteins</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Query and retrieval</li><li>Deposition</li><li>Sequencing quality control</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>303</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bioinformatics_toolkit" target="_blank">Bioinformatics Toolkit</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This Toolkit is a collection of a wide range of tools and links for sequence analysis, function, and structure prediction. This resource offers convienent web interfaces for many freely available tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Structure analysis</li><li>Proteins</li><li>Structure prediction</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein structure analysis</li><li>Protein structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>258</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/contort" target="_blank">COnTORT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>COnTROT (COmprehensive Transcriptomic ORganizational Tool) is a program that will download and organize all expression data in GEO related to a search result, commonly an organism</div></td>
      <td><div class='topic short click_expand'><ul><li>ChIP-seq</li><li>Biotechnology</li><li>Model organisms</li><li>ChIP-on-chip</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Parsing</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://pypi.org/project/contort" target="_blank">https://pypi.org/project/contort</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/shinemas" target="_blank">SHiNeMaS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web tool dedicated to seed lots history, phenotyping and cultural practices.<br>In 2005, a collaboration started between INRA and the french farmers' seed network Réseau Semences Paysannes (RSP) on bread wheat. In this project, researchers and farmers' organizations needed to map the history and life cycle of the population-varieties using the network formalism. All this information needed to be centralized and stored in a database. Thus, we developed SHiNeMaS (Seeds History and Network Management System) a database with its web interface, dedicated to the management of the history of seed lots and related data like phenotyping, environment and cultural practices. SHiNeMaS is freely available and distributed under GNU Affero GPL Licence.<br>La forge Enseignement supérieur et Recherche.</div></td>
      <td><div class='topic short click_expand'><ul><li>Plant biology</li><li>Database management</li><li>Genotype and phenotype</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dbd" target="_blank">Transcription factor prediction database (DBD)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Predicted transcription factors in completely sequenced genomes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>204</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/3d-xguide" target="_blank">3D-XGuide</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>3D-XGuide is a software framework providing a foundation for rapid prototyping of new approaches in the field of XR navigation guidance. Actual software release offers graphical user interface with basic functionality including data import and handling, calculation of projection geometry and transformations between related coordinate systems, rigid 3D-3D registration, and template-matching-based tracking and motion compensation algorithms in 2D and 3D.</div></td>
      <td><div class='topic short click_expand'><ul><li>Echography</li><li>Anatomy</li><li>MRI</li><li>Tomography</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Image analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-2-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/EviMass" target="_blank">Evidence Based Microbial Associations Miner (EviMass)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Literature Evidence-Based Miner for Human Microbial Associations. The importance of understanding microbe-microbe as well as microbe-disease associations is one of the key thrust areas in human microbiome research</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomics</li><li>Natural language processing</li><li>Microbial ecology</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/enzyme" target="_blank">ENZYME - Enzyme nomenclature database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A repository of information relative to the nomenclature of enzymes. It is primarily based on the recommendations of the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology (IUBMB) and it describes each type of characterized enzyme for which an EC (Enzyme Commission) number has been provided.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence classification</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>671</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pharmmapper-drug" target="_blank">PharmMapper</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Potential drug target identification by reversed pharmacophore matching the query compound against an in-house pharmacophore model database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular modelling</li><li>Drug discovery</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Molecular docking</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>195</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/proteinprospector" target="_blank">ProteinProspector</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A suite of protein identification tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Database management</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein fragment weight comparison</li><li>Blind peptide database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>132</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/minmotif_miner" target="_blank">Minmotif Miner</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Analyzes protein queries for the presence of short contiguous peptide motifs that have a known function in at least one other protein (Minimotifs), its functions include posttranslational modification of the minimotifs (PTM), binding to a target protein or molecule, and protein trafficking. MnM 3.0 uses a rich semantic model and trained algorithm to reduce false positive predictions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein modifications</li><li>Protein sites, features and motifs</li><li>Protein structural motifs and surfaces</li><li>Protein interactions</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Post-translation modification site prediction</li><li>Sequence feature detection</li><li>Protein function prediction</li><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>209</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/umd" target="_blank">UMD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Universal Mutation Database (UMD) is a database for mutations. UMD-predictor is a tool that enables functional annotation of variants to find the relevant ones.</div></td>
      <td><div class='topic short click_expand'><ul><li>Public health and epidemiology</li><li>Genotype and phenotype</li><li>Genetic variation</li><li>Rare diseases</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Polymorphism detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>228</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/conoserver" target="_blank">Cone snail toxin database ConoServer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database specializing in the sequence and structures of conopeptides, which are peptides expressed by carnivorous marine cone snails.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>294</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/quartataweb" target="_blank">QuartataWeb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrated Chemical-Protein-Pathway Mapping for Polypharmacology and Chemogenomics.<br>Data on protein-drug and protein-chemical interactions are rapidly accumulating in databases such as DrugBank and STITCH.</div></td>
      <td><div class='topic short click_expand'><ul><li>Compound libraries and screening</li><li>Pharmacology</li><li>Molecular interactions, pathways and networks</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Expression profile pathway mapping</li><li>Query and retrieval</li><li>Chemical similarity enrichment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/wikidata" target="_blank">Wikidata</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Wikidata Python module integrating the MediaWiki API and the Wikidata SPARQL endpoint.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Phenomics</li><li>Model organisms</li><li>Transcription factors and regulatory sites</li><li>Molecular biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Editing</li><li>Data retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://tools.wmflabs.org/wikidata-todo/stats.php" target="_blank">https://tools.wmflabs.org/wikidata-todo/stats.php</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/snovault" target="_blank">SnoVault</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The tool supports metadata and file submission, a database used for metadata storage, web pages for displaying the metadata and a robust API for querying the metadata.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/seagrass_zmu_genome" target="_blank">Seagrass Zmu Genome</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This page provides access to Zostera muelleri genome assembly and annotation data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/matrixdb" target="_blank">MatrixDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>It is a database focused on interactions established by extracellular proteins and polysaccharides. It takes into account the multimeric nature of several extracellular protein families for the curation of interactions, and reports interactions with individual polypeptide chains or with multimers, considered as permanent complexes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network comparison</li><li>Pathway or network prediction</li><li>Query and retrieval</li><li>Pathway or network visualisation</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>102</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pathDIP" target="_blank">pathDIP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An extended pathway annotations and enrichment analysis resource for human, model organisms and domesticated species.<br>Search Genes Search miRNAs Search Pathways API Download Statistics Documentation Publications Team Contact.<br>pathDIP is an annotated database of signaling cascades in human and non-human organisms, comprising core pathways from major curated pathways databases, and pathways predicted based on orthology, and by using physical protein interactions. Data integration and predictions increase coverage of pathway annotations for human proteome to 92%. pathDIP annotates 122,131 unique proteins in 6,401 pathways in 17 organisms (including 18,454 human proteins), annotates 36,216 pathway orphans (including 5,366 human proteins), and provides multiple query, analysis and output options.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Molecular interactions, pathways and networks</li><li>Functional, regulatory and non-coding RNA</li><li>Zoology</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network comparison</li><li>Pathway or network prediction</li><li>Database search</li><li>Enrichment analysis</li><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nci_flicker" target="_blank">NCI Flicker</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides the means to compare images from different internet sources using a java-enabled web browser. It was originally designed for the comparison of 2D gels.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Imaging</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Protein comparison</li><li>Database comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>68</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/FORESEE" target="_blank">FORESEE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool for the systematic comparison of translational drug response modeling pipelines.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Pharmacology</li><li>Protein expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Scaffolding</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/JRC-COMBINE/FORESEE" target="_blank">https://github.com/JRC-COMBINE/FORESEE</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pybravo" target="_blank">pyBRAvo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>From a list of genes, pyBRAvo produces upstream regulation or signaling networks. pyBRAvo internally leverages the BioPAX ontolgy, the PathwayCommons knowledge graph, and the SPARQL semantic web query language. pyBRAvo can be used through either a Jupyter notebook, or a command line interface.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic pathway prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene symbol : csv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>pathway or network : sif</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/pyBRAvo/pyBRAvo" target="_blank">https://github.com/pyBRAvo/pyBRAvo</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/wit-input-output" target="_blank">WIT Input Output</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MATLAB toolbox with reader/writer and analysis tools for WITec Project/Data (*.wip/*.wid) file.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biomedical science</li><li>Biology</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rdrf" target="_blank">RDRF</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Rare Disease Registry Framework (RDRF) is an open source tool for the creation of web-based patient registries. It enables to create and modify registries without changes to the programming code. It has many features which include its ability to create multiple registries within the same framework, patients are defined once, and the ability for curators to create data elements dynamically well after the registry has been defined.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Pathology</li><li>Ontology and terminology</li><li>Rare diseases</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Deposition</li><li>Aggregation</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>20</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/2d-page" target="_blank">2D-PAGE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>2D-PAGE database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>25</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/riboseqr" target="_blank">riboSeqR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Plotting functions, frameshift detection and parsing of sequencing data from ribosome profiling experiments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>RNA-Seq analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>43</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mqf" target="_blank">MQF</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Quotient filters for efficient storage of k-mers with their counts and metadata.<br>Benchmarking MQF vs CQF vs Countmin.<br>Welcome to khmer: k-mer counting, filtering, and graph traversal FTW!.<br><br>.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Metagenomic sequencing</li><li>RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>k-mer counting</li><li>Splitting</li><li>Query and retrieval</li><li>Sequence trimming</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CandiMeth" target="_blank">CandiMeth</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Using existing tools in the Galaxy environment we generated the workflow CandiMeth (CANDIdate feature METHylation). This takes as input a differential methylation table from an R-based package and maps it to a reference genome build. A simple interface then allows the user to query the dataset with inputs as straightforward as lists of gene names, or as detailed as bed files containing specific chromosomal coordinates, for more sophisticated analyses.</div></td>
      <td><div class='topic short click_expand'><ul><li>Computational biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>text data : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>text data : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Workflow</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://bit.do/candimeth" target="_blank">http://bit.do/candimeth</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/collect_mgf" target="_blank">collect_mgf</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Collects MGF files and dd_results from an XMass setup_QDD.tcl experiment to a single MGF file.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Aggregation</li><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : mgf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : mgf</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/image_sxm" target="_blank">Image SXM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Image SXM is a version of the public domain image analysis software NIH Image that has been extended to handle the loading, display and analysis of scanning microscope images. Image SXM supports SAM, SCM, SEM, SFM, SLM, SNOM, SPM and STM images.</div></td>
      <td><div class='topic short click_expand'><ul><li>SEM</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/image-sxm-1-91-scanning-microscope-images-analysis.html" target="_blank">http://www.mybiosoftware.com/image-sxm-1-91-scanning-microscope-images-analysis.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ASRD" target="_blank">ASRD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An online database for exploring over 2,000 Arabidopsis small RNA libraries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Sequencing</li><li>Functional, regulatory and non-coding RNA</li><li>Plant biology</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/epiGeEC" target="_blank">epiGeEC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>EpiGenomic Efficient Correlator (epiGeEC) tool to enable genome-wide comparisons of very large numbers of public epigenomic datasets.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li><li>ChIP-seq</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://bitbucket.org/labjacquespe/epigeec/src/master/" target="_blank">https://bitbucket.org/labjacquespe/epigeec/src/master/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rouge" target="_blank">Rouge</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A sister database of HUGE protein database which has accumulated the results of comprehensive sequence analysis of human long cDNAs (KIAA cDNAs). The database has been created to publicize the information obtained from mouse homologues of the KIAA cDNAs (mKIAA cDNAs).</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genomegraphs" target="_blank">GenomeGraphs</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomic data analyses requires integrated visualization of known genomic information and new experimental data. This tool uses the biomaRt package to perform live annotation queries to Ensembl and translates this to e.g. gene/transcript structures in viewports of the grid graphics package. Genomic information is plotted together with your data. Plot different data types such as array CGH, gene expression, sequencing and other data, together in one plot using the same genome coordinate system.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>50</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/viennngs" target="_blank">ViennNGS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Build efficient pipelines for NGS data processing. Extracts and converts features from common NGS file formats, computation and evaluation of read mapping statistics, as well as normalization of RNA abundance. Moreover, It allows identification and characterization of splice junctions from RNA-seq data, parsing and condensing sequence motif data, automated construction of Assembly and Track Hubs for the UCSC genome browser, as well as wrapper routines for a set of commonly used NGS command line tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>RNA-seq</li><li>ChIP-seq</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Read mapping</li><li>Expression analysis</li><li>RNA-Seq quantification</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/mtw/Bio-ViennaNGS" target="_blank">https://github.com/mtw/Bio-ViennaNGS</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Databio" target="_blank">Datab.io</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Automated Gene Data Integration with Databio | Automated data transformation for bioinformatics | Automated data integration tools | A simple Gene ID mapping service for Bioinformatics. Upload a gene-oriented data set, we'll automatically detect the identifiers in use, and offer to convert them to match other providers in just a few clicks | A web portal that does automated identifier translation for bioinformatics</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Protein expression</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Parsing</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/joiningdata/databio/" target="_blank">https://github.com/joiningdata/databio/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/r3d-blast" target="_blank">R3D-BLAST</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BLAST-like search tool that allows users to quickly search against the PDB for RNA structures sharing similar substructures with a specified query RNA structure. Input is an RNA 3D structure in PDB format, output includes a graphical display showing structural superposition.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Nucleic acid structure analysis</li><li>Structure prediction</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Structure visualisation</li><li>Structured RNA prediction and optimisation</li><li>RNA structure prediction</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aspicdb" target="_blank">Alternative splicing prediction database (ASPICdb)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database designed to provide access to reliable annotations of the alternative splicing pattern of human genes, obtained by ASPic algorithm and Pintron algorithm, and to the functional annotation of predicted isoforms.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA splicing</li><li>Gene structure</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene transcript report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>29</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/snpdb" target="_blank">SNPdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of single nucleotide polymorphisms (SNPs) and multiple small-scale variations that include insertions/deletions, microsatellites, and non-polymorphic variants.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3,983</div></td>
      <td></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CytoGPS" target="_blank">CytoGPS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CytoGenetic Pattern Sleuth (CytoGPS) - web-enabled karyotype analysis tool for cytogenetics.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>DNA</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="http://cytogps.org/karyotype_parser" target="_blank">http://cytogps.org/karyotype_parser</a></div></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dredge" target="_blank">DrEdGE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DrEdGE (Differential Expression Gene Explorer) is a Web-based tool to visualize RNAseq data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>RNA-Seq</li><li>Bioinformatics</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rpsixml" target="_blank">RpsiXML</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Queries, data structure and interface to visualization of interaction datasets. This package inplements the PSI-MI 2.5 standard and supports up to now 8 databases. Further databases supporting PSI-MI 2.5 standard will be added continuously.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td><div class="link short click_expand"><a href="http://www.bioconductor.org" target="_blank">http://www.bioconductor.org</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/laminin_database" target="_blank">Laminin database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database focusing on the non-collagenous extracellular matrix protein family, the LMs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Gene and protein families</li><li>Gene expression</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression analysis</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PathwayMatcher" target="_blank">PathwayMatcher</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Proteoform-centric network construction enables fine-granularity multiomics pathway mapping | PathwayMatcher is a free, open-source software to search for pathways related to a list of genes, proteins or proteoforms in Reactome | PathwayMatcher was benchmarked against different reference datasets covering multiple types of omics data | We present next the Cypher language queries to find the numbers shown in the summary statistics. These can also be used to get the protein-pathway mapping used in PathwayMatcher | PathwayMatcher is a standalone command line tool to match human biomedical data to pathways | PathwayMatcher is a standalone command line tool to match human biomedical data to pathways | 7448 valid tools on Nov 10, 2019</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Molecular interactions, pathways and networks</li><li>Protein modifications</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Expression profile pathway mapping</li><li>Post-translation modification site prediction</li><li>Pathway or network comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://toolshed.g2.bx.psu.edu/view/galaxyp/reactome_pathwaymatcher" target="_blank">http://toolshed.g2.bx.psu.edu/view/galaxyp/reactome_pathwaymatcher</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/WPTherml" target="_blank">WPTherml</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Python Package for the Design of Materials for Harnessing Heat.<br>Pioneering the design of materials to harness heat.<br>WPTherml stands for William Paterson University's tool for Thermal Energy and Radiation management with Multi Layer nanostructures.<br>Project maintained by FoleyLab Hosted on GitHub Pages — Theme by mattgraham.<br>WPTherml stands for William Paterson University’s tool for Thermal Energy and Radiation management with Multi Layer nanostructures.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li><li>Mathematics</li><li>Chemistry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://foleylab.github.io/wptherml/" target="_blank">https://foleylab.github.io/wptherml/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bionda" target="_blank">BIONDA - Biomarker Database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The BIONDA biomarker database provides structured information on all biomarker candidates published in PubMed articles. There is no limitation to any kind of diseases. To this end, PubMed article abstracts and renowned databases such as UniProt and Human Disease Ontology are used as sources for BIONDA’s database entries. These are acquired automatically and updated regularly using text mining methods. BIONDA is available freely via a user-friendly web interface. As a specific characteristic, BIONDA’s database entries are rated by a scoring approach estimating biomarker reliability.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biological databases</li><li>Bioinformatics</li><li>Proteomics</li><li>Proteins</li><li>Biomarkers</li><li>Literature and language</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Relation extraction</li><li>Named-entity and concept recognition</li><li>Query and retrieval</li><li>Database search</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>name : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>text data : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://www.ruhr-uni-bochum.de/mpc/software/mature/index.html.en#BIONDA" target="_blank">https://www.ruhr-uni-bochum.de/mpc/software/mature/index.html.en#BIONDA</a></div></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/colombos" target="_blank">COLOMBOS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web based portal for exploring and analyzing comprehensive organism-specific cross-platform expression compendia of bacterial organisms. The compendia were carefully constructed by collecting, homogenizing and formally annotating publicly available microarray and RNA-seq experiments from Gene Expression Omnibus (GEO), and ArrayExpress.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Functional genomics</li><li>Model organisms</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Gene expression analysis</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>29</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lipidr" target="_blank">lipidr</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Software Tool for Data Mining and Analysis of Lipidomics Datasets.<br>Data Mining and Analysis of Lipidomics Datasets.<br>lipidr: Data Mining and Analysis of Lipidomics Datasets in R.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Lipids</li><li>Metabolomics</li><li>Data mining</li><li>Workflows</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Principal component visualisation</li><li>Box-Whisker plot plotting</li><li>Parsing</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ahmohamed/lipidr" target="_blank">https://github.com/ahmohamed/lipidr</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ergo" target="_blank">ERGO Genome Analysis and Discovery System</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides a systems-biology informatics toolkit centered on comparative genomics to capture, query and visualize sequenced genomes. Building upon the most comprehensive genomic database available anywhere integrated with the largest collection of microbial metabolic and non-metabolic pathways and using Igenbio's proprietary algorithms, it assigns functions to genes, integrates genes into pathways, and identifies previously unknown or mischaracterized genes, cryptic pathways and gene products.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Transcription factors and regulatory sites</li><li>Gene structure</li><li>Molecular interactions, pathways and networks</li><li>GWAS study</li><li>Comparative genomics</li><li>Metagenomics</li><li>Exome sequencing</li><li>Functional genomics</li><li>DNA polymorphism</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Pathway or network analysis</li><li>SNP detection</li><li>SNP annotation</li><li>Sequence annotation</li><li>Sequence alignment</li><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Workbench</li></ul></div></td>
      <td><div class='citations short click_expand'>181</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/embossdata" target="_blank">embossdata</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Find and retrieve EMBOSS db data files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Data loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sbml.jl" target="_blank">SBML.jl</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Systems Biology Markup Language model loader for Julia, suitable for constraint-based reconstruction and analysis (COBRA).</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>reaction data : sbml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dima" target="_blank">DIMA domain interaction map</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Functional and physical interactions among conserved protein-domains, including experimental data and computational predictions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein interaction data : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/deepimmuno" target="_blank">DeepImmuno</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DeepImmuno is a tool for deep learning-empowered prediction and generation of immunogenic peptides for T cell immunity.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Immunoproteins and antigens</li><li>Machine learning</li><li>Immunogenetics</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Peptide immunogenicity prediction</li><li>Epitope mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/frankligy/DeepImmuno" target="_blank">https://github.com/frankligy/DeepImmuno</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ftmsranalysis" target="_blank">ftmsRanalysis</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>FtmsRanalysis is an R package for exploratory data analysis and interactive visualization of FT-MS data. This R package provides functionality for data formatting, preprocessing, filtering, exploratory data analysis and visualization of fourier transform (FT) mass spectrometry (MS) data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Plant biology</li><li>Carbohydrates</li><li>Microbiology</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Filtering</li><li>Parsing</li><li>Peak detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://emsl-computing.github.io/ftmsRanalysis/" target="_blank">https://emsl-computing.github.io/ftmsRanalysis/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nannochloropsis" target="_blank">Nannocholopsis gaditana Database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This database provide several way to access to the genomic data, as a blast search page, a genome browser and a direct link to download the data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>120</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/monetfamily" target="_blank">MoNetFamily</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web server that can identify the modules, homologous modules (called module family), and MMI networks across multiple species for the query protein(s).</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/monetfamily-homologous-modules-and-module-module-interaction-networks-in-vertebrates.html" target="_blank">http://www.mybiosoftware.com/monetfamily-homologous-modules-and-module-module-interaction-networks-in-vertebrates.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/yabi" target="_blank">Yabi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>It is an open source software system which provides an analysis workflow environment that can create and reuse workflows as well as manage large amounts of both raw and processed data in a secure and flexible way across geographically distributed compute resources. It can be used via a web-based environment to drag-and-drop tools to create sophisticated workflows.</div></td>
      <td><div class='topic short click_expand'><ul><li>Software engineering</li><li>Data architecture, analysis and design</li><li>Bioinformatics</li><li>Workflows</li><li>Rare diseases</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Editing</li><li>Data retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>72</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nextprot" target="_blank">neXtProt</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>NeXtProt is an innovative knowledge platform dedicated to human proteins. This resource contains a wealth of high-quality data on all the human proteins that are produced by the 20'000 protein-coding genes found in the human genome. The content of neXtProt is continuously extended so as to provide many more carefully selected data sets and analysis tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein modifications</li><li>Proteomics</li><li>Proteins</li><li>Protein variants</li><li>Model organisms</li><li>Rare diseases</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Protein sequence analysis</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>161</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/calipho-sib/" target="_blank">https://github.com/calipho-sib/</a></div></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/michip" target="_blank">MiChip</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package takes the miRNA microarray .grp scanner output files and parses these out, providing summary and plotting functions to analyse hybridizations. A set of hybridizations is packaged into an ExpressionSet allowing it to be used by other BioConductor packages.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Data management</li><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/colt-cancer" target="_blank">COLT-Cancer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Collection of shRNA dropout signatures profiles, covering ~16000 human genes, and derived from more than 70 Pancreatic, Ovarian and Breast human cancer cell-lines using a microarray detection platform.</div></td>
      <td><div class='topic short click_expand'><ul><li>Human biology</li><li>Laboratory information management</li><li>Mapping</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Enrichment analysis</li><li>Genetic mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>37</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/avian-immunome-db" target="_blank">Avian Immunome DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Avian Immunome DB is a user-friendly interface for extracting genetic information about the avian immune system. The avian immune system is characterised by a cascade of complex biological processes underlaid by more than 1000 different genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li><li>Gene transcripts</li><li>Gene expression</li><li>Ontology and terminology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>DNA vaccine design</li><li>Query and retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://gitlab.com/rcmueller/immunomedb" target="_blank">https://gitlab.com/rcmueller/immunomedb</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/spiat" target="_blank">SPIAT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An R package for the Spatial Image Analysis of Cells in Tissues.<br>Spatial Image Analysis of Tissues.<br>R package for the analysis of OPAL multiplex immunohistochemistry images.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcriptomics</li><li>Genotype and phenotype</li><li>Imaging</li><li>Cell biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Query and retrieval</li><li>Visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Rbbt" target="_blank">Rbbt</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Rbbt stands for “Ruby Bioinformatics Toolkit”. It is a framework for software development in bioinformatics. It covers three aspects:.<br>Workflow wrapping the TF text-mining results from https://github.com/fnl plus other databases.<br>High Throughput Sequencing related functionalities.<br>Rbbt wrapper for the Variant Effect Predictor. Auto-downloads and installs the software.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Sequencing</li><li>Imaging</li><li>Personalised medicine</li><li>Workflows</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Enrichment analysis</li><li>Genotyping</li><li>Sorting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="http://mikisvaz.github.io/rbbt/" target="_blank">http://mikisvaz.github.io/rbbt/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/non-b_db" target="_blank">Non-B DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>It is a non-canonical (non-B) structure database database integrating annotations and analysis of non-B DNA-forming sequence motifs. The database provides alternative DNA structure predictions, including Z-DNA motifs, quadruplex-forming motifs, inverted repeats, mirror repeats and direct repeats and their associated subsets of cruciforms, triplex and slipped structures, respectively.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li><li>DNA</li><li>Structure prediction</li><li>DNA polymorphism</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>DNA structure prediction</li><li>Quadruplex formation site detection</li><li>Query and retrieval</li><li>Sequence feature detection</li><li>Repeat sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>52</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/TSDD" target="_blank">TSDD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Transcript Structure and Domain Display (TSDD) is a publicly available, web-based program that provides publication quality images of transcript structures and domains.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein folds and structural domains</li><li>Gene transcripts</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/upefinder" target="_blank">UPEFinder</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Bioinformatic Tool for the Study of Uncharacterized Proteins Based on Gene Expression Correlation and the PageRank Algorithm.<br>The Human Proteome Project (HPP) is leading the international effort to characterize the human proteome.<br>neXtProt Ensemble Id Gene Name GO Term GO Id MSIG Term Disease Chromosome.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Proteomics</li><li>Gene expression</li><li>Pathology</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein identification</li><li>Expression correlation analysis</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ubioinformat/UPEfinder" target="_blank">https://github.com/ubioinformat/UPEfinder</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/coordinateserver" target="_blank">CoordinateServer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A fast, web-based tool for returning a subset of mmCIF coordinate data for a PDB entry held in the PDB archive. The server is able to return the specific portions of the structure that are relevant, as specified in your query</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Freeware</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gelbank" target="_blank">GELBANK</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of 2D gel images of proteomes for species with completed genomes. The user can search by sequence description or fragment, or by gel characteristics. Links are made between the sequence and gel when available.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Proteomics</li><li>Imaging</li><li>Zoology</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein molecular weight calculation</li><li>Protein identification</li><li>Protein feature detection</li><li>Query and retrieval</li><li>Protein fragment weight comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>34</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genolist" target="_blank">GenoList comparative analysis of microbial genomes</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated environment for the analysis of microbial genomes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Comparative genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>50</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/RAPPAS" target="_blank">RAPPAS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>RAPPAS stands for Rapid Alignment-free Phylogenetic Placement via Ancestral Sequences. It uses an alignment-free approach for phylogenetic placement, thus removing the hurdle of query sequence alignment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomics</li><li>Taxonomy</li><li>Bioinformatics</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Tersect" target="_blank">Tersect</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A set theoretical utility for exploring sequence variant data | Command-line utility for conducting fast set theoretical operations and genetic distance estimation on biological sequence variant data | Tersect is a command-line utility for conducting fast set theoretical operations and genetic distance estimation on biological sequence variant data. The tool generates index files based on provided variant data (VCF files) which can then be used to rapidly execute flexible set theoretical queries and output the resulting lists of variants in selected regions</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Sequencing</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Data retrieval</li><li>Read binning</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/netview" target="_blank">Netview</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Free web tool that allow end users to explore the human and mouse gene regulatory networks. Users can query the system by providing a gene identifier and the boundaries of the subnetwork being explored. The system provides the list of interactions, the enriched Gene Ontology Terms together with a graphical representation of the subnetwork.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Systems biology</li><li>Gene regulation</li><li>Human biology</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene regulatory network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/netview-explore-the-human-and-mouse-gene-regulatory-networks.html" target="_blank">http://www.mybiosoftware.com/netview-explore-the-human-and-mouse-gene-regulatory-networks.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/globplot_ws_jabaws" target="_blank">GlobPlot WS (jabaws)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Allows the user to plot the tendency within the query protein for order/globularity and disorder.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>758</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/3714" target="_blank">https://www.biocatalogue.org/services/3714</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ncbi_genotyping" target="_blank">Genotyping - NCBI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Identifies the genotype (or subtype) of viral sequences by using a sliding window approach to BLAST analysis against reference sequences for different viral subtypes. Results are shown as a graphical output plotting the top-scoring genotype. An alignment tool is also available.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Genotyping experiment</li><li>Microarray experiment</li><li>Model organisms</li><li>Virology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Recombination detection</li><li>PCR primer design</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>162</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cytoml" target="_blank">CytoML</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package is designed to use GatingML2.0 as the standard format to exchange the gated data with other software platform.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/COLMARm" target="_blank">COLMARm</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Automated Tools for the Analysis of 1D-NMR and 2D-NMR Spectra | COLMARm 13C-1H HSQC, HSQC-TOCSY and TOCSY Query and Verification | Visit a step by step how-to with figures here | Bingol, K.; Li, DW.; Zhang, B.; Brüschweiler, R.; Comprehensive metabolite identification strategy using multiple two-dimensional NMR spectra of a complex mixture implemented in the COLMARm web server; Anal. Chem; 2016, 88, 12411-12418;</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>MRI</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Peak detection</li><li>Deisotoping</li><li>Natural product identification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="http://spin.ccic.osu.edu/index.php/colmarm/index2" target="_blank">http://spin.ccic.osu.edu/index.php/colmarm/index2</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ngsRelateV2" target="_blank">ngsRelateV2</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Fast and accurate relatedness estimation from high-throughput sequencing data in the presence of inbreeding.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Sequencing</li><li>Population genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Parsing</li><li>Protein threading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ANGSD/ngsRelate/issues" target="_blank">https://github.com/ANGSD/ngsRelate/issues</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/astd" target="_blank">ASTD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>AltSplice and AltExtron provide information on alternative intron/exons, alternative splice events, and isoform splice patterns. AEdb contains: AEdb-Sequence (sequence and properties of alternatively splice exons), AEdb-Function (data on functional aspects of alternative splicing), AEdb-motif (data and sequence of known splice regulatory motifs), and AEdb-minigene (a collection of known minigene constructs for alternative splice events).</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>76</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mpss" target="_blank">mpss</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>RNA and Genomic Informatics Resources for Plants.<br>About Next-Gen Sequence Databases.<br>The databases and the web pages (with customized web interfaces) were devloped to handle and display 'next-gen' sequencing data for several organisms and related varieties (mainly plants).<br>The database and the web pages that you see were produced by the Meyers lab at the Donald Danforth Plant Science Center.<br>if you use data from this page.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Functional, regulatory and non-coding RNA</li><li>Plant biology</li><li>Gene transcripts</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phasing</li><li>Query and retrieval</li><li>Expression analysis</li><li>Sequence tag mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/swiss-2dpage" target="_blank">SWISS-2DPAGE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Data on proteins identified on various 2-D PAGE and SDS-PAGE reference maps.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>experiment report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>88</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/a5" target="_blank">A5</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A5 is an integrative pipeline for genome assembly that automates sequence data cleaning, error correction, assembly, and quality control by chaining a number of programs together with additional custom algorithms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>De-novo assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>324</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/find" target="_blank">FIND</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Accurate genome-wide predictions of spatio-temporal gene expression during embryonic development | Fly tissue-developmental stage expression predictions | FIND (Fly in silico nanodissection) is a webserver for exploring the Drosophila embryonic development tissue-developmental stage-specific gene expression predictions. The predictions can be queried by genes (Genes Query or Genes Bulk Query) or by tissue-stages (Tissues query). The Similar Genes Query tool allows finding genes with similar spatiotemporal expression patterns. The Tissue Finder Query tool allows finding relevant tissues from ranked gene lists such as differential expression</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Functional, regulatory and non-coding RNA</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene prediction</li><li>Expression correlation analysis</li><li>Expression profile clustering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sitex" target="_blank">SitEx</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database that contains information on functional site amino acid positions in the exon structure of encoding gene. It contains BLAST and 3DExonScan programs. BLAST is used for searching sequence similarity between the query protein and polypeptides encoded by single exons stored in it. The 3DExonScan program is used for searching for structural similarity of the given protein with these polypeptides using superimpositions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein folds and structural domains</li><li>Protein sites, features and motifs</li><li>Gene transcripts</li><li>Protein binding sites</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Structural similarity search</li><li>Protein feature detection</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/yeast_gene_order_browser" target="_blank">Yeast gene order browser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool for visualising the syntenic context of any gene from several yeast genomes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>451</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/blackfan_anemia" target="_blank">Blackfan Anemia mutation database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Blackfan Anemia mutation database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>147</div></td>
      <td><div class="link short click_expand"><a href="http://www.dbagenes.unito.it/docs/LOVD_manual_2.0.pdf" target="_blank">http://www.dbagenes.unito.it/docs/LOVD_manual_2.0.pdf</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cancervar" target="_blank">CancerVar</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CancerVar is a web server for improved evidence-based clinical interpretation for cancer somatic mutations. CancerVar is a bioinformatics software tool for clinical interpretation of somatic variants.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Preclinical and clinical studies</li><li>Genetic variation</li><li>Gene transcripts</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant prioritisation</li><li>Variant calling</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/WGLab/CancerVar" target="_blank">https://github.com/WGLab/CancerVar</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cpla" target="_blank">Compendium of protein lysine acetylation (CPLA)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Resource of manually curated lysine acetylated substrates with their sites.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenetics</li><li>Proteins</li><li>Protein modifications</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein feature detection</li><li>Annotation</li><li>Protein post-translation modification site prediction</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>46</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gpsno" target="_blank">GPSno</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The genetic and pharmacogenomic landscape of snoRNAs in human cancer.<br>G enetic and p harmacogenomic landscape of sno RNAs in cancer.<br>GPSno systematically investigated the impact of genetic variants and pharmacogenomics landscape of snoRNAs in multiple cancer types by using thousands of data from The Cancer Genome Atlas (TCGA).</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Pharmacogenomics</li><li>Oncology</li><li>Genotype and phenotype</li><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>SNP detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/flowrepositoryr" target="_blank">FlowRepositoryR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package provides an interface to search and download data and annotations from FlowRepository (flowrepository.org). It uses the FlowRepository programming interface to communicate with a FlowRepository server.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>83</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/proteinatlas" target="_blank">The Human Protein Atlas</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database with millions of high-resolution images. Expression and localization of proteins in a large variety of normal human tissues, cancer cells and cell lines with the aid of immunohistochemistry (IHC) images and immunofluorescence (IF) confocal microscopy images.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein targeting and localisation</li><li>Oncology</li><li>Structural biology</li><li>Humans</li><li>Gene expression</li><li>Gene and protein families</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>4,306</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mgd" target="_blank">Mouse Genome Database (MGD)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Community model organism database for the laboratory mouse and the source for phenotype and functional annotations of mouse genes. It includes a complete catalog of mouse genes and genome features with integrated access to genetic, genomic and phenotypic information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Genetic variation</li><li>Pathology</li><li>Mapping</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Data retrieval</li><li>Genetic mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>645</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/prodom" target="_blank">ProDom protein domain database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive set of protein domain families automatically generated from the UniProt Knowledge Database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein family report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>289</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phosida" target="_blank">PHOSIDA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The database manages posttranslational modification sites of various species ranging from bacteria to human. It contains phosphorylated, N-glycosylated or acetylated sites. All sites are obtained from high-resolution mass spectrometric data. The database is comprised of three main components: the database environment, the prediction platform and the toolkit section.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteins</li><li>Protein modifications</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>de Novo sequencing</li><li>Protein feature detection</li><li>PTM localisation</li><li>Post-translation modification site prediction</li><li>Sequence feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>290</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genesnps" target="_blank">GeneSNPs</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Gene-centric map of the genome structure, coding sequences, and identified allelic variation in genes being targeted for a role in disease susceptibility by the NIEHS (National Institute of Evironmental Health Sciences). This database provides a graphical view of all available SNP data including allele frequencies and genotypes in select populations. This information is key in selecting the polymorphic sites needed to examine disease risk in human population studies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Human biology</li><li>DNA polymorphism</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence variations : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/diseasecard" target="_blank">Diseasecard</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Information retrieval tool for accessing and integrating genetic and medical information for health applications. Resorting to this integrated environment, clinicians are able to access and relate diseases data already available in the Internet, scattered along multiple databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical informatics</li><li>Molecular interactions, pathways and networks</li><li>Data integration and warehousing</li><li>Rare diseases</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network analysis</li><li>Data retrieval</li><li>Database search</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GENOMICUS" target="_blank">GENOMICUS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomicus is a genome browser that enables users to navigate in genomes in several dimensions: linearly along chromosome axes, transversaly across different species, and chronologicaly along evolutionary time.</div></td>
      <td><div class='topic short click_expand'><ul><li>Paleogenomics</li><li>Gene and protein families</li><li>Phylogenomics</li><li>Comparative genomics</li><li>Evolutionary biology</li><li>Genomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Phylogenetic tree visualisation</li><li>Query and retrieval</li><li>Sequence alignment</li><li>Dot plot plotting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>386</div></td>
      <td><div class="link short click_expand"><a href="https://www.genomicus.biologie.ens.fr/genomicus-plants" target="_blank">https://www.genomicus.biologie.ens.fr/genomicus-plants</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/yakusa" target="_blank">YAKUSA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Program devised to rapidly scan a structural database with a query protein structure. It searches for the longest common substructures, called SHSP for “structural high scoring pairs”, between a query structure and every structure in the structural database. It makes use of protein backbone internal coordinates (α angles) in order to describe protein structures as sequences of symbols. It uses a deterministic finite automaton for pattern matching.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular modelling</li><li>Proteins</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>60</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/yakusa-scan-structural-database-query-protein-structure.html" target="_blank">http://www.mybiosoftware.com/yakusa-scan-structural-database-query-protein-structure.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/repro" target="_blank">REPRO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The web application is able to recognise distant repeats in a single query sequence. The technique relies on a variation of the Smith-Waterman local alignment strategy to find non-overlapping top-scoring local alignments, followed by a graph-based iterative clustering procedure to delineate the repeat set(s) based on consistency of the pairwise top-alignments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence composition, complexity and repeats</li><li>Nucleic acid structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Repeat sequence detection</li><li>Local alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>57</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/repro-protein-repeats-analysis.html" target="_blank">http://www.mybiosoftware.com/repro-protein-repeats-analysis.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/uniparc" target="_blank">UniParc</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>UniParc is a comprehensive and non-redundant database that contains most of the publicly available protein sequences in the world. Proteins may exist in different source databases and in multiple copies in the same database. UniParc avoids such redundancy by storing each unique sequence only once and giving it a stable and unique identifier (UPI).</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence record : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>115</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sevenbridges" target="_blank">sevenbridges</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R client and utilities for Seven Bridges platform API, from Cancer Genomics Cloud to other Seven Bridges supported platforms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td><div class="link short click_expand"><a href="http://www.sevenbridges.com" target="_blank">http://www.sevenbridges.com</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cupp" target="_blank">CUPP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Conserved Unique Peptide Patterns (CUPP) is a approach for sequence analysis employing conserved peptide patterns for determination of similarities between proteins. CUPP performs unsupervised clustering of proteins for formation of protein groups and can be used for fast annotation of enzyme protein family, subfamily and EC function of carbohydrate-active enzymes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>DNA</li><li>Carbohydrates</li><li>Model organisms</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome annotation</li><li>Query and retrieval</li><li>Phylogenetic tree annotation</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rMSIKeyIon" target="_blank">rMSIKeyIon</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Ion Filtering R Package for Untargeted Analysis of Metabolomic LDI-MS Images.</div></td>
      <td><div class='topic short click_expand'><ul><li>Statistics and probability</li><li>Proteomics experiment</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enrichment analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/LlucSF/rMSIKeyIon/issues" target="_blank">https://github.com/LlucSF/rMSIKeyIon/issues</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/LupinMine" target="_blank">LupinMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>White and narrow-leaf lupin data from the Legume Information System</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>155</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mpblast" target="_blank">MPBLAST</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Software that increases the throughput of batch BLASTN searches by multiplexing (concatenating) query sequences and thereby reducing the number of actual database searches performed.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence alignment</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>19</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mpblast-multiplex-blast.html" target="_blank">http://www.mybiosoftware.com/mpblast-multiplex-blast.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/AMELIE" target="_blank">AMELIE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web application for automatic mendelian literature evaluation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Exome sequencing</li><li>Literature and language</li><li>Genotype and phenotype</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant prioritisation</li><li>Genotyping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/riken" target="_blank">RIKEN integrated database of mammals</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>It integrates not only RIKEN’s original databases, but also imported data from public databases, such as Ensembl, MGI and biomedical ontologies. The top-level ontology-based implementation of mammal-related data directly integrates the representative knowledge and individual data records in existing databases to ensure advanced cross-database searches and reduced unevenness of the data management operations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Ontology and terminology</li><li>Informatics</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Database search</li><li>Structure database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td><div class="link short click_expand"><a href="http://scinets.org/db/mammal" target="_blank">http://scinets.org/db/mammal</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sirnadb" target="_blank">siRNAdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This resource includes siSearch, AOSearch, and a siRNAdb which provides a platform for mining an siRNA database, and searching for non-specific matches to your siRNA (small interfering RNAs).</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li><li>Gene transcripts</li><li>RNA</li><li>Sequence analysis</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>siRNA duplex prediction</li><li>Database search</li><li>Sequence database search</li><li>Sequence database search (by property)</li><li>siRNA binding specificity prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>51</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rsbml" target="_blank">rsbml</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Links R to libsbml for SBML parsing, validating output, provides an S4 SBML DOM, converts SBML to R graph objects. Optionally links to the SBML ODE Solver Library (SOSLib) for simulating models.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td><div class="link short click_expand"><a href="http://www.sbml.org" target="_blank">http://www.sbml.org</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/worfdb" target="_blank">WORFDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Caenorhabditis elegans ORFeome cloning project.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>85</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/COMBIgor" target="_blank">COMBIgor</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Data-Analysis Package for Combinatorial Materials Science.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Environmental science</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>15</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/NREL/COMBIgor/" target="_blank">https://github.com/NREL/COMBIgor/</a></div></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ukbtools" target="_blank">ukbtools</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R package to manage and query UK Biobank data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pathology</li><li>Genotype and phenotype</li><li>Biobank</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Data retrieval</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/kenhanscombe/ukbtools" target="_blank">https://github.com/kenhanscombe/ukbtools</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/h-invdb" target="_blank">H-InvDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This tool provides curated annotations of human genes and transcripts that include gene structures, alternative splicing isoforms, non-coding functional RNAs, protein functions, functional domains, sub-cellular localizations, metabolic pathways, protein 3D structure, genetic polymorphisms (SNPs, indels and microsatellite repeats), relation with diseases, gene expression profiling, and molecular evolutionary features, protein-protein interactions and gene families/groups.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein targeting and localisation</li><li>Molecular interactions, pathways and networks</li><li>Genetic variation</li><li>Functional, regulatory and non-coding RNA</li><li>Gene and protein families</li><li>Pathology</li><li>Human biology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene transcript report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>34</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scfind" target="_blank">scfind</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Recently a very large collection of single-cell RNA-seq datasets has been generated and publicly released. For the collection to be useful, the information must be organized in a way that supports queries that are relevant to researchers. `scfind` builds an index from scRNA-seq datasets which organizes the information in a suitable and compact manner so that the datasets can be very efficiently searched for either cells or cell types in which a given list of genes is expressed.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-seq</li><li>Sequencing</li><li>Transcriptomics</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/hemberg-lab/scfind" target="_blank">https://github.com/hemberg-lab/scfind</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bktree" target="_blank">BkTree</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Utility capable of predicting all familites of RNA nucleotide interactions, including base-base, base-phosphate, base-ribose, and base-stacking interactions, from the input query sequence (along with predicted or known canonical Watson-Crick base pairs).</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Molecular interactions, pathways and networks</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Prediction and recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/bktree-prediction-of-rna-nucleotide-interactions-with-backbone-k-tree-model.html" target="_blank">http://www.mybiosoftware.com/bktree-prediction-of-rna-nucleotide-interactions-with-backbone-k-tree-model.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gwastools" target="_blank">GWASTools</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Classes for storing very large GWAS data sets and annotation, and functions for GWAS data cleaning and analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>GWAS study</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Analysis</li><li>Deposition</li><li>Annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>79</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/proticdb" target="_blank">PROTICdb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Proteomic database to store, track, query and compare proteome data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Spectral analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GEM-NET" target="_blank">GEM-NET</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Lessons in Multi-Institution Teamwork Using Collaboration Software | Dashboard to view/download files from segregated Google Shared Drives | Center for Genetically Encoded Materials (C-GEM) | GEM-NET is the home of the Center for Genetically Encoded Materials (C-GEM), an NSF CCI (Center for Chemical Innovation) | Center for Genetically Encoded Materials | C-GEM seeks to establish a fundamentally new way to program chemical matter and transform the way we produce materials and medicines</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Chemistry</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/gem-net" target="_blank">https://github.com/gem-net</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/KOFFI" target="_blank">KOFFI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>New binding kinetics database and its integration in an open-source binding analysis software.<br>Kinetics OF Featured Interactions.<br>KOFFI-DB (???) is an article based database for kinetic constants of biomolecular interactions. We focused on interactions between DNA, RNA, proteins and chemical compounds. All datapoints were manually annotated and reviewed based on open source articles available on Europe PMC</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Literature and language</li><li>Database management</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Database search</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/vistcr" target="_blank">VisTCR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An Interactive Software for T Cell Repertoire Sequencing Data Analysis.<br>VisTCR is an open source software that provides an interactive visualization of high-throughput TCR sequencing data, while also incorporating a friendly graphical user interface and a flexible workflow for data analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Immunoproteins and antigens</li><li>Sequencing</li><li>Probes and primers</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Clustering</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/FAIRifier" target="_blank">FAIRifier</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A general-purpose FAIRifier on the basis of the OpenRefine data cleaning and wrangling tool and the RDF plugin. This FAIRifier enables a post-hoc FAIRification workflow: load an existing dataset, perform data wrangling tasks, add FAIR attributes to the data, generate a linked data version of the data and, finally, push the result to an online FAIR data infrastructure to make it accessible and discoverable. Literal values in a dataset can be replaced by identifiers either manually or by embedded, customizable script expressions. The interoperability of the dataset can be improved by connecting these identifiers into a meaningful semantic graph-structure of ontological classes and properties using the integrated RDF model editor. A provenance trail automatically keeps track of each modification and additionally enables “undo” operations and repetition of operations on similar datasets. A FAIR data export function opens up a metadata editor to provide information about the dataset itself.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data identity and mapping</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/uniprotkb" target="_blank">UniProtKB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The central hub for the collection of functional information on proteins with accurate, consistent and rich annotation. It consists of: UniProtKB/Swiss-Prot (manually-annotated records and curator-evaluated computational analysis) and UniProtKB/TrEMBL (computationally analyzed records awaiting manual annotation).</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Proteomics</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein sequence analysis</li><li>Protein domain recognition</li><li>Gene functional annotation</li><li>Transmembrane protein analysis</li><li>Protein structure analysis</li><li>Protein function analysis</li><li>Protein secondary structure analysis</li><li>Post-translation modification site prediction</li><li>Protein quaternary structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3,851</div></td>
      <td><div class="link short click_expand"><a href="http://www.uniprot.org/downloads" target="_blank">http://www.uniprot.org/downloads</a></div></td>
      <td><div class='short click_expand'>CC-BY-ND-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hiannotator" target="_blank">hiAnnotator</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Set of functions to annotate a GRanges object with custom set of annotations. This package takes two GRanges objects (query & subject) with a common set of seqnames (i.e. chromosomes) and returns associated annotation per seqnames and rows from the query matching seqnames and rows from the subject (i.e. genes or cpg islands).</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence features : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence features : image format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/silva" target="_blank">SILVA rRNA database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>SILVA provides comprehensive, quality checked and regularly updated datasets of aligned small (16S/18S, SSU) and large subunit (23S/28S, LSU) ribosomal RNA (rRNA) sequences for all three domains of life (Bacteria, Archaea and Eukaryota).</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Taxonomy</li><li>Functional, regulatory and non-coding RNA</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>organism name : plain text format (unformatted)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment (nucleic acid) : fasta-aln</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>14,636</div></td>
      <td><div class="link short click_expand"><a href="https://www.arb-silva.de/download/archive/" target="_blank">https://www.arb-silva.de/download/archive/</a></div></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/RPGeNet" target="_blank">RPGeNet</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>RPGeNet v2.0 - Retinitis Pigmentosa Gene Network: interacting with the network of genes related to retinitis pigmentosa genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li><li>Microarray experiment</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Pathway or network prediction</li><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/chebi_ws" target="_blank">ChEBI WS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on ‘small’ chemical compounds. "Molecular entity" includes any constitutionally or isotopically distinct atom, molecule, ion, ion pair, radical, radical ion, complex, conformer, etc., identifiable as a separately distinguishable entity.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Cheminformatics</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>small molecule report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/2174" target="_blank">https://www.biocatalogue.org/services/2174</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cyp21" target="_blank">CYP21</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CYP21 mutation browser.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mzWiff" target="_blank">mzWiff</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MzWiff is a command-line program to convert Analyst native acquisition (.wiff) files to mzXML and mzML.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : wiff format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : mzxml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pep" target="_blank">PEP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Linking big biomedical datasets to modular analysis with Portable Encapsulated Projects.<br>Pipeline results reporting package.<br>Project metadata manager for PEPs in Python.<br>python interface reporting postgresql pipelines results reproducibility.<br>PEP, or Portable Encapsulated Projects, is a community effort to make sample metadata reusable. PEPs decouple metadata from analysis by providing:.<br>An open structure and toolkit for organizing large-scale, sample-intensive biological research projects.<br>A PEP is written using a familiar yaml + csv format that can be read by our R and Python packages, and well as by workflow frameworks. PEP is your metadata solution for sample-intensive research projects.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Sequencing</li><li>Bioinformatics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Standardisation and normalisation</li><li>Parsing</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://github.com/pepkit" target="_blank">http://github.com/pepkit</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/TZMD" target="_blank">TZMD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A high-resolution genomic composition-based method with the ability to distinguish similar bacterial organisms.<br>The "tranucleotide-derived Z-value Manhattan Distance” (TZMD) approach profiles genomic composition with high resolution, which can distinguish bacteria at even strain level</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogeny</li><li>Taxonomy</li><li>Metagenomics</li><li>Model organisms</li><li>DNA polymorphism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pannbuilder" target="_blank">PAnnBuilder</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Processing annotation data from public data repositories and building protein-centric annotation data packages.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Sequence annotation</li><li>Database comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data resource definition id : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database search results : sqlite format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td><div class="link short click_expand"><a href="http://www.biosino.org/PAnnBuilder" target="_blank">http://www.biosino.org/PAnnBuilder</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/vcfdbr" target="_blank">VCFdbR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A method for expressing biobank-scale Variant Call Format data in a SQLite database using R.<br>This processing pipeline converts a VCF into an SQLite representation, using R. This readme only covers the practical matters of executing the pipeline. Please see the wiki or FAQ for more detailed discussions and tutorials covering how to use a VCFdb once it is created.<br>VCFdbR was designed with Linux-based high performance computing clusters in mind, but it should work on any distro. It has not been tested on Windows or Mac operating systems, and changes to specifically support them are not planned.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Gene transcripts</li><li>Workflows</li><li>Exome sequencing</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Variant filtering</li><li>Formatting</li><li>Genotyping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GOT-10k" target="_blank">GOT-10k</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Large High-Diversity Benchmark for Generic Object Tracking in the Wild.<br>Generic Object Tracking Benchmark.<br>A large, high-diversity, one-shot database for generic object tracking in the wild</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Machine learning</li><li>Database management</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/inforesidue" target="_blank">inforesidue</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Return information on a given amino acid residue.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ROADMAP_epigenomics" target="_blank">ROADMAP epigenomics</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Data repository of NIH Roadmap Epigenomics Mapping Consortium containing genome-wide maps of histone modifications, chromatin accessibility, DNA methylation and mRNA expression across 100s of human cell types and tissues.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li><li>Human biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2,611</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rems" target="_blank">REMS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool that enables a researcher to submit and a Data Access Committee to process Data Access Applications to sensitive datasets.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data governance</li><li>Database management</li><li>Data security</li><li>Computer science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Service management</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>LGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/circatlas" target="_blank">circAtlas</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CircAtlas is an integrated resource of one million highly accurate circular RNAs from 1070 vertebrate transcriptomes. Existing circular RNA (circRNA) databases have become essential for transcriptomics. However, most are unsuitable for mining in-depth information for candidate circRNA prioritization. To address this, we integrate circular transcript collections to develop the circAtlas database based on 1070 RNA-seq samples collected from 19 normal tissues across six vertebrate species. This database contains 1,007,087 highly reliable circRNAs, of which over 81.3% have been assembled into full-length sequences. We profile their expression pattern, conservation, and functional annotation. We describe a novel multiple conservation score, co-expression, and regulatory networks for circRNA annotation and prioritization.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Functional, regulatory and non-coding RNA</li><li>Model organisms</li><li>Transcription factors and regulatory sites</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li><li>Query and retrieval</li><li>Expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>21</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nachrdb" target="_blank">NAChRDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Web Resource of Structure-Function Annotations to Unravel the Allostery of Nicotinic Acetylcholine Receptors.<br>Database of structural-functional annotations of nicotinic acetylcholine receptors.<br>NAChRDB is a web-accessible database of residue-level structural-functional annotations of nicotinic acetylcholine receptors.<br>NAChRDB is particularly useful for identifying gaps in knowledge and for guiding discovery through structural and molecular biology experiments, especially when exploring the allosteric mechanisms underlying neuromuscular nAChR function and pathology.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Literature and language</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/surf" target="_blank">SURF</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrative analysis of a compendium of RNA-seq and CLIP-seq datasets highlights complex governing of alternative transcriptional regulation by RNA-binding proteins.<br>The Statistical Utility for RBP Functions (SURF) is an integrative analysis framework to identify alternative splicing (AS), alternative transcription initiation (ATI), and alternative polyadenylation (APA) events regulated by individual RBPs and elucidate protein-RNA interactions governing these events.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>RNA immunoprecipitation</li><li>Gene transcripts</li><li>Gene expression</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>RNA-Seq analysis</li><li>RNA-Seq quantification</li><li>Parsing</li><li>Alternative splicing prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/keleslab/surf-paper" target="_blank">https://github.com/keleslab/surf-paper</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dataman" target="_blank">DATAMAN</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DATAMAN is a global database of nitrous oxide and ammonia emission factors for excreta deposited by livestock and land-applied manure. The overall aim of DATAMAN is to provide access to the most up-to-date knowledge on managing manure GHG emissions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Splitting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cytoscape_web" target="_blank">Cytoscape Web</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based network visualization tool-modeled after Cytoscape-which is open source, interactive, customizable and easily integrated into web sites.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Molecular interactions, pathways and networks</li><li>Imaging</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li><li>Pathway or network comparison</li><li>Pathway or network prediction</li><li>Pathway or network visualisation</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>446</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/teak" target="_blank">TEAK</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An open source GUI software consisting of two modules: Topology Enrichment Analysis frameworK  and Query Structure Enrichment Analysis. TEAK is used to find biologically activated KEGG subpathways given molecular profiling data whereas QSEA is used to perform hierarchical querying of the KEGG pathways.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>33</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/teak-20130207-topology-enrichment-analysis-framework.html" target="_blank">http://www.mybiosoftware.com/teak-20130207-topology-enrichment-analysis-framework.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CowpeaMine" target="_blank">CowpeaMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A mine containing both cowpea genetic and genomic data, courtesy UC-Riverside</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/reganalyst" target="_blank">RegAnalyst</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web server that integrates MoPP (Motif Prediction Program), MyPatternFinder and MycoRegDB (mycobacterial promoter and regulatory elements database).</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li><li>Molecular interactions, pathways and networks</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Promoter prediction</li><li>Transcriptional regulatory element prediction</li><li>Regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cabri" target="_blank">CABRI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Common Access to Biological Resources and Information - A collection of European Biological Resource Centre catalogues.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microbial collection</li><li>Microbiology</li><li>Cell culture collection</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>cell line report : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td><div class="link short click_expand"><a href="http://www.nettab.org/promano/" target="_blank">http://www.nettab.org/promano/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rxnat" target="_blank">Rxnat</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Rxnat: Query and retrieves neuroimaging sets from XNAT projects.<br>XNAT is an extensible open-source imaging informatics software platform dedicated to imaging-based research.<br><br>To use system environment variables we need to add them to the .Renviron file located in the user's home folder. Use Sys.getenv("HOME") to get the path (for unix/osx users the location can be easily accessed with ~, eg. vi ~/.Renviron). The Rxnat package will be able to automatically read / use a system environment variable provided the following format is used: XXXX_RXNAT_USER and XXXX_RXNAT_PASS. XXXX is provided as an argument when an XNAT connection is initiated.</div></td>
      <td><div class='topic short click_expand'><ul><li>Informatics</li><li>Machine learning</li><li>Medical imaging</li><li>MRI</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Data retrieval</li><li>Anonymisation</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://www.xnat.org" target="_blank">https://www.xnat.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/treehouse" target="_blank">treehouse</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A user-friendly application to obtain subtrees from large phylogenies | Phylogenies are rapidly becoming larger due to the rising number of available genomes. Beyond broad patterns, large phylogenies often become uninterpretable as tip labels are so small or excluded altogether. As a result, large phylogenies can be difficult to use for other researchers especially for groups interested in the relationships among a subset of taxa. Here, we introduce treehouse, a user friendly shiny app that allows researchers to obtain subtrees from large-scale phylogenies. Treehouse is populated with a handful of large-scale phylogenies available through treehouseDB. Additionally, treehouse features an additional function, userTree, which allows a user to upload and parse their own phylogeny | Our aim is to make phylogenies more accessible to all and facilitate researchers exploration in their own phylogenies</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenomics</li><li>Plant biology</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree reconstruction</li><li>Phylogenetic tree bootstrapping</li><li>Phylogenetic tree comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/straitrazor" target="_blank">STRaitRazor</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An enhanced user interface to facilitate interpretation of MPS data.<br>STRait Razor Online (SRO) serves as the user-interface (UI) for analyzing sequencing data with STRait Razor. Using the SRO, converts STRait Razor* sequence-based allele calls into genotype tables for and/or bar plots for downstream analysis. STRait Razor Online also allows the user to control the genotype calls using several thresholds (e.g., heterozygous balance, strand balance, etc.). Read strands are merged and reported as single strand (Default = Forward) in Genotype Tables.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>DNA polymorphism</li><li>Genotype and phenotype</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Variant calling</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ExpectationsManaged/STRaitRazorOnline" target="_blank">https://github.com/ExpectationsManaged/STRaitRazorOnline</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rmat" target="_blank">rMAT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package is an R version of the package MAT and contains functions to parse and merge Affymetrix BPMAP and CEL tiling array files (using C++ based Fusion SDK and BioConductor package affxparser), normalize tiling arrays using sequence specific models, detect enriched regions from ChIP-chip experiments.</div></td>
      <td><div class='topic short click_expand'><ul><li>ChIP-seq</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td><div class="link short click_expand"><a href="http://www.rglab.org" target="_blank">http://www.rglab.org</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scope-cheminformatics" target="_blank">SCOPE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A cheminformatics toolbox for investigating lipidomic changes in complex systems.<br>.<br>To run any of the r scripts provided, clone the repository and open the SCOPE_Lipid_Analysis.Rproj file in Rstudio. Run the scripts within the project, so that they can find the relevant paths on your machine. Example data files for dual Case vs. Control comparison are included for building biological-structure relationships.</div></td>
      <td><div class='topic short click_expand'><ul><li>Lipids</li><li>Genotype and phenotype</li><li>Toxicology</li><li>Cheminformatics</li><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Clustering</li><li>Genotyping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dipper" target="_blank">DIPPER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DIPPER is a spatiotemporal proteomics atlas of human intervertebral discs for exploring ageing and degeneration dynamics.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li><li>MRI</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Statistical calculation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://github.com/hkudclab/DIPPER" target="_blank">http://github.com/hkudclab/DIPPER</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/DEPOD" target="_blank">Human Dephosphorylation Database  (DEPOD)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DEPOD - the human DEPhOsphorylation Database, is a manually curated database collecting human phosphatases, their experimentally verified substrates and dephosphorylation site information, and pathways in which they are involved. It also provides links to popular kinase databases and protein-protein interaction databases for these phosphatases and substrates.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data submission, annotation and curation</li><li>Protein interactions</li><li>Database management</li><li>Sequence sites, features and motifs</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genome visualisation</li><li>Post-translation modification site prediction</li><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/interpro" target="_blank">InterPro</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Protein families, domains, regions, repeats and sites in which identifiable features found in known proteins can be applied to new protein sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Proteins</li><li>Sequence composition, complexity and repeats</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein signature : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>786</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SAMMI" target="_blank">SAMMI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Semi-Automated Tool for the Visualization of Metabolic Networks.<br>Semi-Automated Metabolic Map Illustrator.<br>SAMMI is a web-based tool for the visualization of metabolic networks and related data. Networks can be uploaded directly from KEGG annotated pathways or using constraint-based metabolic reconstructions</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Literature and language</li><li>Mapping</li><li>Endocrinology and metabolism</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Parsing</li><li>Map drawing</li><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/DrugR" target="_blank">DrugR+</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database for drug repurposing, combination therapy, and replacement therapy.</div></td>
      <td><div class='topic short click_expand'><ul><li>Drug development</li><li>Drug discovery</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/LBBSoft/DrugR-plus" target="_blank">https://github.com/LBBSoft/DrugR-plus</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/modbase" target="_blank">ModBase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Three-dimensional protein models calculated by comparative modelling.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein structure report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>169</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/censor" target="_blank">CENSOR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Protein and nucleotide sequence repeat identification and masking. Screens query sequences against a reference collection of repeats and "censors" (masks) homologous portions with masking symbols, as well as generating a report classifying all found repeats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence composition, complexity and repeats</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Repeat sequence detection</li><li>Sequence masking</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>74</div></td>
      <td><div class="link short click_expand"><a href="http://www.ebi.ac.uk/support/" target="_blank">http://www.ebi.ac.uk/support/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/QGIS" target="_blank">QGIS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>QGIS is a free and open source Geographic Information System.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Statistics and probability</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phasing</li><li>Molecular surface calculation</li><li>RNA-seq time series data analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Tripal_MapViewer" target="_blank">Tripal MapViewer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A tool for interactive visualization and comparison of genetic maps.<br>Tripal is an open-source, resource-efficient toolkit for construction of genomic, genetic and breeding databases. It facilitates development of biological websites by providing tools to integrate and display biological data using the generic database schema, Chado, together with Drupal, a popular website creation and content management system. Tripal MapViewer is a new interactive tool for visualizing genetic map data. Developed as a Tripal replacement for Comparative Map Viewer (CMap), it enables visualization of entire maps or linkage groups and features such as molecular markers, quantitative trait loci (QTLs) and heritable phenotypic markers. It also provides graphical comparison of maps sharing the same markers as well as dot plot and correspondence matrices</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Genotype and phenotype</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Genetic mapping</li><li>Dot plot plotting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/drlumi" target="_blank">drLumi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Quality control routines for multiplex immunoassay data, including several approaches for: treating the background noise of the assay, fitting the dose-response curves and estimating the limits of quantification.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data quality management</li><li>Data management</li><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/yeastract" target="_blank">YEASTRACT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The YEAst Search for Transcriptional Regulators And Consensus Tracking is a curated repository of regulatory associations between transcription factors (TF) and target genes in Saccharomyces cerevisiae. It includes DISCOVERER and allows the identification of documented or potential transcription regulators of a given gene and of documented or potential regulons for each transcription factor.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Gene expression</li><li>Gene regulation</li><li>Model organisms</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Transcription factor binding site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>141</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iprstats" target="_blank">IPRStats</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A tool for the visualization of Interproscan results. Interproscan results are parsed from the Interproscan XML or EBIXML file into an SQLite or MySQL database. The results for each signature database scan are read and displayed as pie-charts or bar charts as summary statistics.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/iprstats-0-4-2-visualization-interproscan-results.html" target="_blank">http://www.mybiosoftware.com/iprstats-0-4-2-visualization-interproscan-results.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CellProfiler" target="_blank">CellProfiler</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool for quantifying data from biological images, particularly in high-throughput experiments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Genotype and phenotype</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Quantification</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>517</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/CellProfiler/CellProfiler" target="_blank">https://github.com/CellProfiler/CellProfiler</a></div></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mutationaligner" target="_blank">MutationAligner</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This enables you to explore mutation hotspots identified in protein domains from more than 5000 patients across 22 cancer types.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA mutation</li><li>Oncology</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/geoboundaries" target="_blank">geoBoundaries</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A global database of political administrative boundaries.<br>geoBoundaries by the William & Mary geoLab.<br>Create dynamic maps, analyze and build location-aware and geospatial applications with your data using the power of PostGIS in the cloud.</div></td>
      <td><div class='topic short click_expand'><ul><li>Physics</li><li>Workflows</li><li>Quality affairs</li><li>Transcription factors and regulatory sites</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Data retrieval</li><li>Physical mapping</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td><div class="link short click_expand"><a href="http://geo.stp.gov.py" target="_blank">http://geo.stp.gov.py</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pybel" target="_blank">PyBEL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Parsing, validation, compilation, and data exchange of Biological Expression Language (BEL)</div></td>
      <td><div class='topic short click_expand'><ul><li>Data integration and warehousing</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pina" target="_blank">PINA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrated platform for protein interaction network construction, filtering, analysis, visualization and management. It integrates protein-protein interaction data from six public curated databases and builds a complete, non-redundant protein interaction dataset for six model organisms. Moreover, it provides a variety of built-in tools to filter and analyze the network for gaining insight into the network.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li><li>Model organisms</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network visualisation</li><li>Pathway or network prediction</li><li>Enrichment analysis</li><li>Pathway or network comparison</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>208</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PTRBC" target="_blank">PTRBC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PTRBC is a tool for predicting the required pre-surgery blood volume in surgical patients based on machine learning.<br>This clinical data analysis tool can be used to do data cleaning and predict the required transfusion volume of red blood cells in surgical patients based on machine learning methods. Meanwhile, this tool can process the similar classification or regression problems. In other words, a general prediction model can be established for the input data specified.</div></td>
      <td><div class='topic short click_expand'><ul><li>Surgery</li><li>Machine learning</li><li>Critical care medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Regression analysis</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sonoviewer" target="_blank">sonoviewer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-Based GPU-Accelerated Application for Multiplanar Reconstructions from Conventional 2D Ultrasound | ImagingAnatomy.com is an interactive atlas of normal imaging anatomy for the radiologist as well as a learning device for health professionals in general studying anatomy for any reason | Sonoviewer is an open access web application developed as an extension of the Mulrecon web-based viewer</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical imaging</li><li>Quality affairs</li><li>Anatomy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nutrichem" target="_blank">NutriChem</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database generated by text mining of 21 million MEDLINE abstracts, with information that links plant-based foods with their small molecule components and human disease phenotypes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Nutritional science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>disease id : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>pathway or network identifier : binary format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>49</div></td>
      <td><div class="link short click_expand"><a href="http://www.cbs.dtu.dk/services" target="_blank">http://www.cbs.dtu.dk/services</a></div></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sirw" target="_blank">SIRW</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>SIRW is a web interface to SIR (Simple Indexing and Retrieval System). It combines the ability to search protein/nucleotide databases with keywords and a sequence motif.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Proteins</li><li>Bioinformatics</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence feature detection</li><li>Query and retrieval</li><li>Data retrieval</li><li>Sequence motif recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rotl" target="_blank">rotl</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R package to interact with the Open Tree of Life data.<br>Interface to Open Tree of Life API.<br>Interface to the 'Open Tree of Life' API.<br>rotl is an R package to interact with the Open Tree of Life data APIs. It was initially developed as part of the NESCENT/OpenTree/Arbor hackathon.<br>rotl is an R package that can be used to query and download subsets of the synthetic Open Tree of Life directly in R. This repository contains a manuscript describing this package and how to get started with it.<br>An interface to the 'Open Tree of Life' API to retrieve phylogenetic trees, information about studies used to assemble the synthetic tree, and utilities to match taxonomic names to 'Open Tree identifiers'. The 'Open Tree of Life' aims at assembling a comprehensive phylogenetic tree for all named species.<br>Then explore how you can use rotl with other packages to combine your data with trees from the Open Tree of Life project by typing: vignette("data_mashups", package="rotl")</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>Biodiversity</li><li>Taxonomy</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree visualisation</li><li>Phylogenetic tree generation</li><li>Phylogenetic tree editing</li><li>Phylogenetic tree analysis (shape)</li><li>Phylogenetic tree comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ropensci/rotl" target="_blank">https://github.com/ropensci/rotl</a></div></td>
      <td><div class='short click_expand'>BSD-2-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mitozoa" target="_blank">MitoZoa</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A curated mitochondrial genome database of metazoans for comparative genomics studies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Evolutionary biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>38</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cmpi" target="_blank">CELLmicrocosmos 4 PathwayIntegration (CmPI)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CmPI (CELLmicrocosmos 4.2 PathwayIntegration) is a Java-based 3D software which can be used to map 3D pathways - such as metabolic networks, Protein-Protein interaction networks, etc. - onto a virtual cell model. For this purpose, a database is queried in the background using protein/gene identifiers as input. The user can chose between different localization information by using an interactive table or linked bar charts with different categories. Cell environments can be explored in 3D in different navigation modes. Also, stereoscopic rendering is supported.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene name : csv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://jib.tools/details.php?id=32" target="_blank">https://jib.tools/details.php?id=32</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/blastgraph" target="_blank">BlastGraph</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>New tool for computing intensive approximate pattern matching in a sequence graph or a de-Bruijn graph. Given an oriented graph and a set of  query sequences, it detects paths in the graph on which query sequences align at most at the given edit distance.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Computer science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/blastree-1-0-compute-intensive-approximate-pattern-matching-string-graph.html" target="_blank">http://www.mybiosoftware.com/blastree-1-0-compute-intensive-approximate-pattern-matching-string-graph.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/oryzabase" target="_blank">Oryzabase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrated rice genetic map and mutant database at the National Institute of Genetics in Japan.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Genomics</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>87</div></td>
      <td></td>
      <td><div class='short click_expand'>Not licensed</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/HumanMine" target="_blank">HumanMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>HumanMine integrates many types of data for Homo sapiens and Mus musculus</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>155</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SamPler" target="_blank">SamPler</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>> MEDIUM CONFIDENCE! | a novel method for selecting parameters for gene functional annotation routines | Novel strategy for selecting best parameters for gene functional annotation routines in merlin | readme instructions updated · 81b3a5a9</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Sequence assembly</li><li>Endocrinology and metabolism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Local alignment</li><li>Data retrieval</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cometa" target="_blank">CoMeta</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Assigns a query read (DNA fragment) from metagenomic sample into one of the groups (the most common the taxon).</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Metagenomics</li><li>Gene and protein families</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/blast_ebi" target="_blank">BLAST (EBI)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Find regions of sequence similarity and alignments between a query sequence and database sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence alignment</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>6,049</div></td>
      <td><div class="link short click_expand"><a href="http://www.ebi.ac.uk/support/" target="_blank">http://www.ebi.ac.uk/support/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ZebrafishMine" target="_blank">ZebrafishMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ZebrafishMine is powered by the InterMIne data warehouse system, and integrates biological data sets from multiple sources.\r<br>It currently includes updates of data from ZFIN, the zebrafish model organism database. There is also data from the Panther database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ipcoal" target="_blank">ipcoal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An interactive Python package for simulating and analyzing genealogies and sequences on a species tree or network.<br>Python package to interactively simulate genealogies and sequence data under the multispecies coalescent.<br>Full documentation here: https://ipcoal.readthedocs.io.<br>Welcome to ipcoal’s documentation! — ipcoal documentation.<br>Free document hosting provided by Read the Docs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenomics</li><li>DNA polymorphism</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree visualisation</li><li>Network analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mtctscan" target="_blank">mTCTScan</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mutation To Cancer Therapy Scan. Systematically analyze mutations affecting cancer drug sensitivity based on individual genomic profiles. It was developed by leveraging the latest knowledge on mutation-cancer drug sensitivity associations and the results from large-scale chemical screening using human cancer cell lines.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pharmacogenomics</li><li>Oncology</li><li>Genetic variation</li><li>Personalised medicine</li><li>Drug discovery</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SPROUTS" target="_blank">SPROUTS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>SPROUTS has two general functions.<br>The first is to provide existing mutation data given a protein specified by a PDB ID. This is Query mode.<br>The second is to generate new mutation data based on a new PDB ID or a user input. This is Submit mode.</div></td>
      <td><div class='topic short click_expand'><ul></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hogenom" target="_blank">HOGENOM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>It allows selection of sets of homologous genes among species, and visualisation of multiple alignments and phylogenetic trees. It is useful for comparative sequence analysis, phylogeny, molecular evolution studies and to get a view of what is known about a peculiar gene family.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Gene and protein families</li><li>Phylogenomics</li><li>Comparative genomics</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>phylogenetic tree : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>95</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/2031" target="_blank">https://www.biocatalogue.org/services/2031</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gydb" target="_blank">Gypsy Database (GyDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Gypsy Database of Mobile Genetic Elements is a database of viruses and transposable elements based on their phylogenetic classification. It includes long terminal repeats retroelements and relatives, Ty3/Gypsy, Retroviridae, Ty1/Copia and Bel/Pao LTR retroelements and the Caulimoviridae pararetroviruses of plants.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogeny</li><li>Transcription factors and regulatory sites</li><li>Virology</li><li>Mobile genetic elements</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Multiple sequence alignment</li><li>Query and retrieval</li><li>Deposition</li><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>177</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/textsearch" target="_blank">textsearch</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Search the textual description of sequence(s).</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genomereviews" target="_blank">GenomeReviews</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Annotated view of the genomic sequence of organisms with completely deciphered genomes. Currently, Genome Reviews contains the genomes of archaea, bacteria, bacteriophages and selected eukaryota.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>genome report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>22</div></td>
      <td><div class="link short click_expand"><a href="ftp://ftp.ebi.ac.uk/pub/databases/genome_reviews" target="_blank">ftp://ftp.ebi.ac.uk/pub/databases/genome_reviews</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/webscipio" target="_blank">WebScipio</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Scipio is a tool to determine the precise exon-intron gene structure given a protein sequence and a genome. It identifies splice sites and is able to cope with sequencing errors and genes spanning several contigs. The output contains information about discrepancies that may result from sequencing errors. Scipio has also successfully been used to find homologous genes in related species. WebScipio, allows to search for mutually exclusive spliced exons and tandemly arrayed gene duplicates.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Gene and protein families</li><li>Gene transcripts</li><li>Gene structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Gene prediction</li><li>Protein feature detection</li><li>Protein structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>129</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ComplexBrowser" target="_blank">ComplexBrowser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ComplexBrowser is a R based software for supervised analysis of changes in protein complex abundance and subunit co-expression in proteomic datasets. It uses information contained in CORUM and EBI Complex Portal databases to provide the user with visualizations helping with biological interpretation of obtained results. ComplexBrowser also incorporates a normalization and quality control steps.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li><li>Proteomics experiment</li><li>Proteomics</li><li>Protein interactions</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Scatter plot plotting</li><li>Pathway or network visualisation</li><li>Expression analysis</li><li>Expression data visualisation</li><li>ID mapping</li><li>Statistical calculation</li><li>Dendrogram visualisation</li><li>Standardisation and normalisation</li><li>Principal component visualisation</li><li>Query and retrieval</li><li>Regression analysis</li><li>Quantification</li><li>Expression correlation analysis</li><li>Heat map generation</li><li>Validation</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein interaction data : csv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>plot : pdf</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://computproteomics.bmb.sdu.dk/" target="_blank">http://computproteomics.bmb.sdu.dk/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pseudocap" target="_blank">PseudoCAP pseudomonas genome database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Pseudomonas genome database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>458</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/TogoGenome" target="_blank">TogoGenome</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genome database that is purely based on the Semantic Web technology, which enables the integration of heterogeneous data and flexible semantic searches.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Genotype and phenotype</li><li>Taxonomy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/togogenome/" target="_blank">https://github.com/togogenome/</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/falcon_server" target="_blank">Falcon Server</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A highly flexible open-source software for closed-loop neuroscience.<br>OBJECTIVE:Closed-loop experiments provide unique insights into brain dynamics and function. To facilitate a wide range of closed-loop experiments, we created an open-source software platform that enables high-performance real-time processing of streaming experimental data. APPROACH:We wrote Falcon, a C++ multi-threaded software in which the user can load and execute an arbitrary processing graph. Each node of a Falcon graph is mapped to a single thread and nodes communicate with each other through thread-safe buffers. The framework allows for easy implementation of new processing nodes and data types. Falcon was tested both on a 32-core and a 4-core workstation</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Neurobiology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>20</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/OdoBD" target="_blank">OdoBD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An online database for the dragonflies and damselflies of Bangladesh.<br>Welcome to the Odonata of Bangladesh, database of the dragonflies and damselflies of Bangladesh. Situated in the South-east Asia, Bangladesh is gifted with the rich network of water resources like ponds, lakes, marshes, rivers, streams and mangrove swamps. This diverse range of water bodies along with many tropical forest patches has generated suitable habitat for many Odonata species. Till date, nearly hundred species of Odonates have been recorded from Bangladesh.<br>The current project, supported by Rufford Small Grant, is designed to assemble and disseminate information of diversity, ecology and behavior of the Bangladeshi dragonflies and damselflies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Literature and language</li><li>Database management</li><li>Ecology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/macie" target="_blank">MACIE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mechanism, Annotation, and Classification in Enzymes: database of enzyme reaction mechanisms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Chemistry</li><li>Protein binding sites</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enzyme kinetics calculation</li><li>Annotation</li><li>Classification</li><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>58</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bipa" target="_blank">BIPA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Protein-nucleic acid interactions, with various features of protein-nuleic acid interfaces.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein interaction data : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>43</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ukbb-srmed" target="_blank">UKBB self-reported medications</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Categorising UK Biobank Self-Reported Medication Data using Text Matching.<br>UK Biobank self reported medications are represented in UK Biobank as a list of codes per participant captured at the time of the Baseline Assessment Interview.</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical informatics</li><li>Genotype and phenotype</li><li>Medicine</li><li>Mapping</li><li>Biobank</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>LGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mbwa_wrapper" target="_blank">mbwa_wrapper</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Fast light-weighted tool that aligns relatively short sequences (queries) to a sequence database (large), such as the human reference genome.</div></td>
      <td><div class='topic short click_expand'><ul><li>Human biology</li><li>Sequence analysis</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence alignment</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : fastq-illumina</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence trace : sam</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>20,216</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/mbwa_map/mbwa_wrapper/0.7.4" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/mbwa_map/mbwa_wrapper/0.7.4</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/croco-xlink" target="_blank">CroCo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A a user-centred tool to convert results from crosslinking mass spectrometry experiments | Convert data formats from chemical cross-linking mass spectrometry (XL-MS) | The CroCo cross-link converter — CroCo documentation | CroCo converts multiple data format from cross-linking mass spectrometry software tools to xTable format (in csv format) | The CroCo cross-link converter ¶ | CroCo converts multiple data format from cross-linking mass spectrometry software tools to xTable format (in csv format). It is distributed as graphical programme to be run from an executable and as a Python module to be integrated into workflows</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Mass spectrum visualisation</li><li>Parsing</li><li>Peptide identification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/cschmidtlab/croco" target="_blank">https://github.com/cschmidtlab/croco</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cuticledb" target="_blank">CuticleDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A relational database containing all structural proteins of Arthropod cuticle identified to date. Many come from direct sequencing of proteins isolated from cuticle and from sequences from cDNAs that share common features with these authentic cuticular proteins. It also includes proteins from the Drosophila melanogaster and the Anopheles gambiae genomes, that have been predicted to be cuticular proteins, based on a Pfam motif responsible for chitin binding in Arthropod cuticle.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>47</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/microbial_advanced_database_organisation_micado" target="_blank">Microbial advanced database organisation (Micado)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Microbial genome and functional analysis of Bacillus subtilis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data reference : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>27</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/massbank" target="_blank">MassBank</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MassBank is a public repository of mass spectra of small chemical compounds for life sciences (<3000 Da). The database contains spectra from EI‐MS, fast atom bombardment MS, electrospray ionization (ESI)‐MSn data of thousands of authentic standards, EI‐MS and other‐MS data of thousands of volatile natural and synthetic compounds, and ESI‐MS2 data of synthetic drugs contributed by research groups worldwide. ESI‐MS2 data were analyzed under nonstandardized, independent experimental conditions. MassBank users can access either all of the MassBank data or a subset of the data by specifying one or more experimental conditions. In a spectral search to retrieve mass spectra similar to a query mass spectrum, the similarity score is calculated by a weighted cosine correlation in which weighting exponents on peak intensity and the mass‐to‐charge ratio are optimized to the ESI‐MS2 data. MassBank is useful for the identification of chemical compounds and the publication of experimental data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,014</div></td>
      <td><div class="link short click_expand"><a href="https://massbank.eu/" target="_blank">https://massbank.eu/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iDNA6mA-PseKNC" target="_blank">iDNA6mA-PseKNC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Identification of DNA N6-methyladenosine sites by incorporating nucleotide physicochemical properties into PseKNC.</div></td>
      <td><div class='topic short click_expand'><ul><li>ChIP-seq</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Post-translation modification site prediction</li><li>PTM localisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>161</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bgee" target="_blank">Bgee</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database to retrieve and compare gene expression patterns between animal species.  Bgee first maps heterogeneous expression data (currently EST, Affymetrix, and in situ hybridization data) on anatomical and developmental ontologies.  Then, in order to perform automated cross species comparisons, homology relationships across anatomical ontologies, and comparison criteria between developmental ontologies, are designed.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional genomics</li><li>Comparative genomics</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>122</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/enzymeminer" target="_blank">EnzymeMiner</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>EnzymeMiner identifies putative members of enzyme families or subfamilies and facilitates the selection of promising targets for experimental characterization. Two key selection criteria are (i) the predicted solubility and (ii) the sequence identities visualised using an interactive sequence similarity network. The search query can be a sequence from the Swiss-Prot database or a custom sequence with a custom description of essential residues. The output is an interactive selection table containing annotated identified sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Proteomics</li><li>Proteins</li><li>Data mining</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence annotation</li><li>Sequence clustering</li><li>Protein solubility prediction</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein sequence : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>Proprietary</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/unitrap" target="_blank">UniTrap gene trap clones</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tools for the biologist enabling optimized use of gene trap clones.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/JRgui" target="_blank">JRgui</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Python Program of Joback and Reid Method | Python GUI program for a quick estimation of physicochemical properties of molecules | A brief description of JRgui program: | By using the modern object-oriented programming language Python (e.g. tkinter and pandas modules) and a chemoinformatics open source library (RDKit), the classic Joback and Reid group contribution method was revisited and written into a graphical user interface program JRgui</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rsffreader" target="_blank">rSFFreader</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Reads sequence, qualities and clip point values from sff files generated by Roche 454 and Life Sciences Ion Torrent sequencers into similar classes as are present for fastq files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/chebi" target="_blank">ChEBI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Dictionary of molecular entities focused on small chemical compounds. "Molecular entity" includes any constitutionally or isotopically distinct atom, molecule, ion, ion pair, radical, radical ion, complex, conformer, etc., identifiable as a separately distinguishable entity. The molecular entities in question are either products of nature or synthetic products used to intervene in the processes of living organisms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Chemical biology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>small molecule report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/minpath" target="_blank">MinPath</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MinPath (Minimal set of Pathways) is a parsimony approach for biological pathway reconstructions using protein family predictions, achieving a more conservative, yet more faithful, estimation of the biological pathways for a query dataset.</div></td>
      <td><div class='topic short click_expand'><ul></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GITAR" target="_blank">GITAR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>GITAR (Genome Interaction Tools and Resources) is a standardized, easy to use and flexible solution to manage Hi-C genomic interaction data, from processing to storage and visualization, composed of two modules: HiCtool and a processed data library.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Molecular interactions, pathways and networks</li><li>Immunoprecipitation experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enrichment analysis</li><li>Read binning</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Suite</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/Zhong-Lab-UCSD/HiCtool" target="_blank">https://github.com/Zhong-Lab-UCSD/HiCtool</a></div></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/taxallnomy" target="_blank">taxallnomy</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Closing gaps in the NCBI Taxonomy.<br>Taxallnomy is a taxonomic database that provides taxonomic lineages according to the ranks used on Linnean classification system (e.g. Kingdom, Phylum, Class etc.). Taxallnomy is based on the NCBI Taxonomy, thus you will find along the taxonomic lineage either taxa originally ranked in NCBI Taxonomy or some unique nodes created by the Taxallnomy algorithm, since some taxonomic ranks are missing in the original taxonomic lineage (for example, the Superclass rank is missing on the Homo sapiens lineage).<br>Taxallnomy is a taxonomic database based on NCBI Taxonomy that provides taxonomic lineages according to the ranks used on Linnean classification system (e.g. Kingdom, Phylum, Class etc.).<br>Taxallnomy is an extension of NCBI Taxonomy in which all taxa comprising it are organized in a balanced hierarchy, i.e all leaf (end) nodes are in the same hierarchical level.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Taxonomy</li><li>Cladistics</li><li>Metagenomics</li><li>Zoology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Taxonomic classification</li><li>Phylogenetic reconstruction</li><li>Phylogenetic tree annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://sourceforge.net/projects/taxallnomy" target="_blank">http://sourceforge.net/projects/taxallnomy</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Genus_biomolecules" target="_blank">Genus biomolecules</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of genus characteristics of proteins and RNA.<br>A database of genus characteristics.<br>The Genus database collects information about topological structure and complexity of proteins and RNA chains, which is captured by the genus of a given chain and its subchains. For each biomolecule this information is shown in the form of a genus trace plot, as well as a genus matrix diagram. We assemble such information for all protein and RNA structures deposited in the PDB. This database presents also various statistics and extensive information about biological function of analyzed biomolecules. Moreover, users can analyze their own structures</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Nucleic acid structure analysis</li><li>RNA</li><li>Sequence assembly</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Deposition</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/listinputs" target="_blank">listinputs</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Give a list of input files from different conditions/groups for DESeq analysis, DESeq can then charge these input files from the given list.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : tsv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>934</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/urgi_tools/listInputs/1.0.0" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/urgi_tools/listInputs/1.0.0</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/RepetDB" target="_blank">RepetDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Repeats intermine database</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cosore" target="_blank">COSORE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>COSORE is a community database for continuous soil respiration and other soil-atmosphere greenhouse gas flux data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Biological databases</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Formatting</li><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bpbond/cosore/issues" target="_blank">https://github.com/bpbond/cosore/issues</a></div></td>
      <td><div class='short click_expand'>CC-BY-4.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Genomicus-protists" target="_blank">Genomicus-protists</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomicus-protists is a genome browser that enables users to navigate in protists genomes in several dimensions: linearly along chromosome axes, transversaly across different species, and chronologicaly along evolutionary time.</div></td>
      <td><div class='topic short click_expand'><ul><li>Paleogenomics</li><li>Gene and protein families</li><li>Phylogenomics</li><li>Comparative genomics</li><li>Evolutionary biology</li><li>Genomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Genome visualisation</li><li>Phylogenetic tree visualisation</li><li>Dot plot plotting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>386</div></td>
      <td><div class="link short click_expand"><a href="https://www.genomicus.biologie.ens.fr/genomicus" target="_blank">https://www.genomicus.biologie.ens.fr/genomicus</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/BMEG" target="_blank">BMEG</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Exploring Integrative Analysis using the BioMedical Evidence Graph | A graph database for merging and analyzing connected data | Discover The BMEG is a graph database that collects data from over 20 different large scale resources and aggregates them into a single coherent resource. Sources include:</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li><li>Molecular interactions, pathways and networks</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Enrichment analysis</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bmeg/" target="_blank">https://github.com/bmeg/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gconvert" target="_blank">gConvert</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A tool of the g:Profiler web server. It converts mix of different types of gene, protein, microarray probe, standard name and other database IDs to a target database ID.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Data identity and mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Format detection</li><li>Query and retrieval</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>565</div></td>
      <td></td>
      <td><div class='short click_expand'>Freeware</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/organismdbi" target="_blank">OrganismDbi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The package enables a simple unified interface to several annotation packages each of which has its own schema by taking advantage of the fact that each of these packages implements a select methods.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Sequence annotation</li><li>Database search</li><li>Database comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein identifier : matrix format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database search results : matrix format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/meltdb" target="_blank">MeltDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Open source framework providing analysis methods for raw GC- and LC-MS metabolome datasets and offers methods to combine the respective results. Flexible tool pipelines allow both the import of pre-processed data as well as the integration of existing open source analysis packages such as XCMS, MassSpecWavelet or metaB. For the identification of metabolites based on mass spectra either the GMD database or user defined libraries in NIST are queried.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Chromatogram visualisation</li><li>Peak detection</li><li>Spectral analysis</li><li>Statistical calculation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : mass spectrometry data format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>74</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/wormbase" target="_blank">WormBase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Caenorhabditis elegans genome database. International consortium of biologists and computer scientists dedicated to providing the research community with accurate, current, accessible information concerning the genetics, genomics and biology of C. elegans and related nematodes. Founded in 2000, the Consortium is led by Paul Sternberg of CalTech, Paul Kersey of the EBI, Matt Berriman of the Wellcome Trust Sanger Institute, and Lincoln Stein of the Ontario Institute for Cancer Research.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>119</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/3738" target="_blank">https://www.biocatalogue.org/services/3738</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/simextargid" target="_blank">SimExTargId</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Comprehensive package for real-time LC-MS data acquisition and analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Structural biology</li><li>Data acquisition</li><li>Mass spectrometry</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li><li>Spectral analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/quosa" target="_blank">QUOSA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Complete end-to-end solution for literature search and document management in the life science fields.</div></td>
      <td><div class='topic short click_expand'><ul><li>Document management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Literature search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/quosa-8-07-249-manage-scientific-technical-documents.html" target="_blank">http://www.mybiosoftware.com/quosa-8-07-249-manage-scientific-technical-documents.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/clinotator" target="_blank">Clinotator</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Takes variants as input and queries NCBI eutilities to generate ClinVar Variation Report scoring metrics. The overall goal is to generate annotations of use for given batches of variants to inform clinical interpretation. The metrics include: Clinotator Raw Score, Average Clinical Assertion Age, Clinotator Predicted Significance and eclassification Recommendation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical informatics</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li><li>Sequence database search</li><li>Variant classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>report : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>score : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/treefam" target="_blank">TreeFam</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Search of TreeFam's HMM collection, allows query sequence to be aligned (MAFFT) and inserted into tree (RAxML) to significantly matching models.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Gene and protein families</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree generation</li><li>Sequence motif recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>326</div></td>
      <td><div class="link short click_expand"><a href="https://treefam.tenderapp.com/" target="_blank">https://treefam.tenderapp.com/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/BSA4Yeast" target="_blank">BSA4Yeast</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based quantitative trait locus linkage analysis and bulk segregant analysis of yeast sequencing data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Genotype and phenotype</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression QTL analysis</li><li>Parsing</li><li>Genetic mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://git-r3lab.uni.lu/zhi.zhang/bsa4yeast" target="_blank">https://git-r3lab.uni.lu/zhi.zhang/bsa4yeast</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ipd-imgt_hla" target="_blank">IMGT HLA database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database that provides a searchable repository of highly curated HLA sequences. The naming of these HLA genes and alleles and their quality control is the responsibility of the WHO Nomenclature Committee for Factors of the HLA System. Through the work of the HLA Informatics Group and in collaboration with the European Bioinformatics Institute, we are able to provide public access to this data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunoproteins and antigens</li><li>Proteins</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Query and retrieval</li><li>Deposition</li><li>Sequencing quality control</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>303</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/eximir" target="_blank">ExiMiR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package contains functions for reading raw data in ImaGene TXT format obtained from Exiqon miRCURY LNA arrays, annotating them with appropriate GAL files, and normalizing them using a spike-in probe-based method. Other platforms and data formats are also supported.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Data management</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/jphyloio" target="_blank">JPhyloIO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An open source Java library for reading and writing various tree and alignment formats. Application developers are able to implement format-independent data processing by including event based readers via the abstract strategy pattern. All readers are designed to deal with large amounts of data (alignments with many and/or very long sequences, large trees) without using a great amount of resources (CPU or RAM).</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li><li>Computational biology</li><li>Phylogenetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Format detection</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>alignment : phylip format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>phylogenetic tree : nexml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td><div class="link short click_expand"><a href="https://twitter.com/bioinfweb" target="_blank">https://twitter.com/bioinfweb</a></div></td>
      <td><div class='short click_expand'>LGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MIBiG_2.0" target="_blank">MIBiG 2.0</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A repository for biosynthetic gene clusters of known function.<br>REST API layer for the MIBiG database.<br>Repository to track changes in MIBiG curation data stored in JSON format.<br>Minimum Information about a Biosynthetic Gene cluster.<br>Refactor MIBiG (version 1.4: August 6th, 2018) JSON schema, both structurally and conceptually. Adopt the latest JSON schema (Draft V7), and update existing data to match the new schema.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li><li>Microbial ecology</li><li>Metagenomics</li><li>Drug discovery</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>62</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/mibig-secmet/mibig-api/" target="_blank">https://github.com/mibig-secmet/mibig-api/</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pubserver" target="_blank">PubServer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PubServer collects homologous sequences from NR database and retrieves and filters associated publications.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular modelling</li><li>Gene and protein families</li><li>Sequence analysis</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Protein feature detection</li><li>Protein modelling</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/singlecellexperiment" target="_blank">SingleCellExperiment</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Defines a S4 class for storing data from single-cell experiments. This includes specialized methods to store and retrieve spike-in information, dimensionality reduction coordinates and size factors for each cell, along with the usual metadata for genes and libraries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Clone library</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/SingleCellExperiment/" target="_blank">http://bioconductor.org/packages/SingleCellExperiment/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rdmp" target="_blank">RDMP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A novel, process driven, open-source tool for the management of longitudinal cohorts of clinical data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Laboratory information management</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ped" target="_blank">Pancreatic Expression Database (PED)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Repository for pancreatic-derived -omics data. It includes easurements derived from transcriptomics, proteomics, genomics and miRNA profiles from various pancreas-centred reports on a broad range of specimen and experimental types.</div></td>
      <td><div class='topic short click_expand'><ul><li>Omics</li><li>Functional, regulatory and non-coding RNA</li><li>Oncology</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression comparison</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene expression clustering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>39</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/iRNA-m2G" target="_blank">iRNA-m2G</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Identifying N2-methylguanosine Sites Based on Sequence-Derived Information.<br>iRNA-m2G: identifying N2-methylguanosine sites.<br>The benchmark datasets that uesed to train and test the proposed model are provided on this page.<br>Benchmark dataset S1 contains 143 m2G site containing sequences and 143 non-m2G site containing sequences from H. sapiens, M. musculus and S. cerevisiae, respectively.<br>Benchmark dataset S2 contains 143 non-m2G site containing sequences (same as dataset S1) and 1246 non-m2G site containing sequences from H. sapiens, M. musculus and S. cerevisiae, respectively</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Functional, regulatory and non-coding RNA</li><li>Model organisms</li><li>Transcriptomics</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td><div class="link short click_expand"><a href="http://lin-group.cn/server/iRNA-m2G/data.htm" target="_blank">http://lin-group.cn/server/iRNA-m2G/data.htm</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/intenz" target="_blank">IntEnz</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The recommendations of the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology (NC-IUBMB) on the nomenclature and classification of enzyme-catalysed reactions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/clineplotr" target="_blank">ClinePlotR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ClineHelpR is a R package that allows you to plot BGC (Bayesian Genomic Cline) output. After we ran BGC, we realized it wasn't easy to plot the BGC results, so we put together this package in the process of figuring it out.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Biodiversity</li><li>DNA polymorphism</li><li>Mapping</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Aggregation</li><li>Parsing</li><li>Genotyping</li><li>Scaffolding</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/microkit" target="_blank">MiCroKit</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of proteins that localize in midbody, centrosome and/or kinetochore. The database contains experimentally verified microkit proteins from the scientific literature that have unambiguous supportive evidence for subcellular localization under fluorescent microscope. Moreover, the orthologous information was provided for these microkit proteins, and could be a useful resource for further experimental identification.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Literature and language</li><li>Proteins</li><li>Animals</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein identification</li><li>Protein structural motif recognition</li><li>Protein feature detection</li><li>Protein modelling</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PMBD" target="_blank">PMBD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PMBD is a Comprehensive Plastics Microbial Biodegradation Database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Microbial ecology</li><li>Biotechnology</li><li>Natural language processing</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Literature search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Meffil" target="_blank">Meffil</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Algorithms for analyzing DNA methylation data generated using Infinium HumanMethylation450 or MethylationEPIC BeadChips<br>Epigenome-Wide Association Studies (EWAS)</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenetics</li><li>Epigenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>29</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/perishky/meffil/issues" target="_blank">https://github.com/perishky/meffil/issues</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/yeastgfp" target="_blank">Yeast GFP fusion localization database (yeastgfp)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Yeast GFP fusion localization database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Gene structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3,041</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Genomicus-metazoa" target="_blank">Genomicus-metazoa</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomicus-metazoa is a genome browser that enables users to navigate in metazoa genomes in several dimensions: linearly along chromosome axes, transversaly across different species, and chronologicaly along evolutionary time.</div></td>
      <td><div class='topic short click_expand'><ul><li>Paleogenomics</li><li>Gene and protein families</li><li>Phylogenomics</li><li>Comparative genomics</li><li>Evolutionary biology</li><li>Genomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Genome visualisation</li><li>Phylogenetic tree visualisation</li><li>Dot plot plotting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>386</div></td>
      <td><div class="link short click_expand"><a href="https://www.genomicus.biologie.ens.fr/genomicus" target="_blank">https://www.genomicus.biologie.ens.fr/genomicus</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ildgdb" target="_blank">ILDGDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A manually curated database of genomics, transcriptomics, proteomics and drug information for interstitial lung diseases.<br>ILDGDB is a manually curated database that provides comprehensive experimentally supported associations between genes and interstitial lung diseases (ILDs). The current version of ILDGDB incorporates 20 ILDs and over 600 genes obtained from 3000 articles in four species. The database consists of 5 sections: regulation of mRNA level (1082 entries, more than 450 genes), regulation of protein level (391 entries, more than 200 genes), variants (425 entries, more than 100 genes), drug information (35 entries, 29 genes) and knockout information (85 entries, 72 genes).</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Gene transcripts</li><li>Pathology</li><li>Sequence analysis</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hunflair" target="_blank">HunFlair</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An Easy-to-Use Tool for State-of-the-Art Biomedical Named Entity Recognition.<br>A very simple framework for state-of-the-art NLP. Developed by Humboldt University of Berlin and friends.<br>A powerful NLP library. Flair allows you to apply our state-of-the-art natural language processing (NLP) models to your text, such as named entity recognition (NER), part-of-speech tagging (PoS), special support for biomedical data, sense disambiguation and classification, with support for a rapidly growing number of languages.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Natural language processing</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Named-entity and concept recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/TPMCalculator" target="_blank">TPMCalculator</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Software to quantify mRNA abundance of genomic features.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-seq</li><li>Gene transcripts</li><li>RNA splicing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>RNA-Seq quantification</li><li>Parsing</li><li>Expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ncbi/TPMCalculator/issues" target="_blank">https://github.com/ncbi/TPMCalculator/issues</a></div></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/microarray_retriever" target="_blank">Microarray Retriever</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web tool that enables batch search and retrieval of microarray datasets matching user specified criteria from the data in GEO and ArrayExpress microarray repositories. Such batch download facilitates gene expression meta-analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Experimental design and studies</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression analysis</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/spurio" target="_blank">Spurio</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Identify spurious protein predictions in prokaryotes. It searches the query protein sequence against a prokaryotic nucleotide database using tblastn and identifies homologous sequences. Features of the matches are used to score the query sequences' likelihood to be a spurious protein prediction using a Gaussian process model.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Proteins</li><li>Biochemistry</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bridgedbr" target="_blank">BridgeDbR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Use BridgeDb functions and load identifier mapping databases in R.</div></td>
      <td><div class='topic short click_expand'><ul><li>Rare diseases</li><li>Data identity and mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>compound identifier : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>compound identifier : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bridgedb/BridgeDb" target="_blank">https://github.com/bridgedb/BridgeDb</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/abs" target="_blank">ABS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Annotated regulatory binding sites from orthologous promoters.</div></td>
      <td><div class='topic short click_expand'><ul><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid features : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>52</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/itsonedb" target="_blank">ITSoneDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive collection of the fungal ribosomal RNA Internal Transcribed Spacer 1 (ITS1) sequences aimed at supporting metagenomic surveys of fungal environmental communities.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>63</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/overcovid" target="_blank">OverCOVID</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Bioinformatics resources facilitate understanding and harnessing clinical research of SARS-CoV-2.<br>The COVID-19 pandemic, caused by the coronavirus SARS-CoV-2, has created an unprecedented threat to public health. The pandemic has been sweeping the globe, impacting more than 200 countries, with more outbreaks still lurking on the horizon. To tide over COVID-19, the web portal OverCOVID is provided to share bioinformatics resources and information that may contribute to research advances.<br>Various layers of interactions among the viruses and human hosts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Public health and epidemiology</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Bioinformatics portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://bis.zju.edu.cn/overcovid/phylogenies" target="_blank">http://bis.zju.edu.cn/overcovid/phylogenies</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/FoldamerDB" target="_blank">FoldamerDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The FoldamerDB is an open source database of foldamers. Foldamers are molecules which mimics the folding of biological molecules into well defined conformation</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>X-ray diffraction</li><li>Database management</li><li>Molecular biology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Visualisation</li><li>Phasing</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td><div class="link short click_expand"><a href="http://foldamerdb.ttk.hu/feedback.php" target="_blank">http://foldamerdb.ttk.hu/feedback.php</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bagel" target="_blank">BAGEL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>BActeriocin GEnome mining tooL (BAGEL) identifies putative bacteriocin ORFs (antimicrobial peptides)  based on a database containing information about known bacteriocins and adjacent genes involved in bacteriocin activity.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Gene regulation</li><li>Database management</li><li>Genomics</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence motif recognition</li><li>Gene functional annotation</li><li>Gene prediction</li><li>Sequence feature detection</li><li>Coding region prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>122</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nmpdr" target="_blank">National microbial pathogen</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Curated annotations in an environment for comparative analysis of genomes and biological subsystems, with an emphasis on the food-borne pathogens Campylobacter, Listeria, Staphylococcus, Streptococcus, and Vibrio as well as the STD pathogens Chlamydiaceae, Haemophilus, Mycoplasma, Neisseria, Treponema, and Ureaplasma.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Comparative genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>76</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/illuminaio" target="_blank">illuminaio</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tools for parsing Illumina's microarray output files, including IDAT.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Sequencing</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>28</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/HenrikBengtsson/illuminaio" target="_blank">https://github.com/HenrikBengtsson/illuminaio</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/orphanet" target="_blank">Orphanet</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Information on rare diseases and orphan drugs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pathology</li><li>Rare diseases</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>disease report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>182</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genetests" target="_blank">GeneTests</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A medical genetics information resource developed for physicians, other healthcare providers, and researchers, available at no cost to all interested persons.</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical informatics</li><li>Molecular interactions, pathways and networks</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li><li>Genetic mapping</li><li>Gene set testing</li><li>Mapping</li><li>Editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>46</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/TOFF" target="_blank">TOFF</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of traits of fish to promote advances in fish aquaculture.<br>toff-project – Database of Traits OF Fish.<br>TOFF (i.e. Traits OF Fish) is a curated database focusing on functional trait information for fish that aims at bringing together behavioral, morphological, phenological, and physiological traits coupled to environmental measurement context into a single open-source access repository. TOFF hosts data from published field and experimental studies. We ultimately aim to provide an inclusive and accessible data resource to facilitate advances in fish research</div></td>
      <td><div class='topic short click_expand'><ul><li>Physiology</li><li>Genotype and phenotype</li><li>Model organisms</li><li>Transcription factors and regulatory sites</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Information extraction</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dsp" target="_blank">DSP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Shape string predictor for protein sequences. It provides both predicted shape string and sequence-shape string profile for each query sequence allowing for comparison of protein structure or evolution in shape string space.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein secondary structure</li><li>Proteins</li><li>Protein structure analysis</li><li>Protein structural motifs and surfaces</li><li>Structure prediction</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein secondary structure prediction</li><li>Protein structure prediction</li><li>Protein secondary structure comparison</li><li>RNA secondary structure prediction</li><li>Protein secondary structure prediction (coils)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/unibind" target="_blank">UniBind</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A map of direct TF-DNA interactions across species.<br>UniBind is a comprehensive map of direct interactions between transcription factor (TFs) and DNA. High confidence TF binding site predictions were obtained from uniform processing of thousands of ChIP-seq data sets using the ChIP-eat software.<br>A RESTful API to access the latest version of UniBind database programmatically.<br>This API provides an easy-to-use REST web interface to query/retrieve data from the UniBind database. The API comes with a human browsable interface and also a programmatic interface which return the results in different formats, including json, jsonp, yaml, bed, and also api for a browsable interface.</div></td>
      <td><div class='topic short click_expand'><ul><li>ChIP-seq</li><li>Epigenetics</li><li>Gene transcripts</li><li>Mapping</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enrichment analysis</li><li>Data retrieval</li><li>Binding site prediction</li><li>cis-regulatory element prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://unibind.uio.no/enrichment/" target="_blank">https://unibind.uio.no/enrichment/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gwas-vcf" target="_blank">GWAS-VCF</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The variant call format provides efficient and robust storage of GWAS summary statistics.<br>VCF vs plain text GWAS storage formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>Computer science</li><li>Genotype and phenotype</li><li>Workflows</li><li>GWAS study</li><li>DNA polymorphism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant prioritisation</li><li>Variant calling</li><li>Collapsing methods</li><li>Query and retrieval</li><li>Genotyping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.ebi.ac.uk/gwas" target="_blank">http://www.ebi.ac.uk/gwas</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/germ-line_v_genes" target="_blank">germ-line V genes</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrative database of germ-line variable genes from the immunoglobulin loci of human and mouse. All variable gene sequences are extracted from the EMBL-Bank.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Human biology</li><li>Immunoproteins and antigens</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>128</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/germonline" target="_blank">Gametogenesis and reproductive tissue expression</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>High-throughput expression data relevant for germline development and fertility across species.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>37</div></td>
      <td><div class="link short click_expand"><a href="http://www.germonline.org/gol_4_userguide.pdf" target="_blank">http://www.germonline.org/gol_4_userguide.pdf</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tcmio" target="_blank">TCMIO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Comprehensive Database of Traditional Chinese Medicine on Immuno-Oncology.<br>Traditional Chinese medicine (TCM) has been widely used in China and show effective in long history clinical practice.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Molecular interactions, pathways and networks</li><li>Cheminformatics</li><li>Medicine</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cazy" target="_blank">Carbohydrate-active enzymes database (CAZy)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The database describes the families of structurally-related catalytic and carbohydrate-binding modules (or functional domains) of enzymes that degrade, modify, or create glycosidic bonds.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>2,881</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pfp-go" target="_blank">PFP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sequence similarity-based protein function prediction server designed to predict GO annotations for a query sequence beyond what can be found by conventional database search such as BLAST.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Sequence analysis</li><li>Function analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein function prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/pfp-esg-sequence-similarity-based-protein-function-prediction-server.html" target="_blank">http://www.mybiosoftware.com/pfp-esg-sequence-similarity-based-protein-function-prediction-server.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pepxmltab" target="_blank">pepXMLTab</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Parsing pepXML files based one XML package. The package tries to handle pepXML files generated from different softwares. The output will be a peptide-spectrum-matching tabular file. The package also provide function to filter the PSMs based on FDR.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,256</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/fqextract" target="_blank">fqextract</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Extract a subset of fastq reads.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Data acquisition</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : tsv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : fastq-illumina</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>937</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/odoppelt/fqtools/fqextract/1.0.1" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/odoppelt/fqtools/fqextract/1.0.1</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pdb-redo" target="_blank">PDB-REDO databank of optimised macromolecular structures</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Databank of optimised macromolecular structures. PDB-REDO entries are refined, rebuilt and validated with one consistent protocol using the equivalent entry in the Protein Data Bank and its experimental data. PDB-REDO entries typically have higher structural quality and a better fit to the experimental data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein structure : dssp</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>452</div></td>
      <td></td>
      <td><div class='short click_expand'>Freeware</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biocfilecache" target="_blank">BiocFileCache</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package creates a persistent on-disk cache of files that the user can add, update, and retrieve. It is useful for managing resources (such as custom Txdb objects) that are costly or difficult to create, web resources, and data files used across sessions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/BiocFileCache/" target="_blank">http://bioconductor.org/packages/BiocFileCache/</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ests_source" target="_blank">ESTs SOURCE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A unification tool which dynamically collects and compiles data from many scientific databases, and thereby attempts to encapsulate the genetics and molecular biology of genes from the genomes of Homo sapiens, Mus musculus, Rattus norvegicus into easy to navigate GeneReports.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene transcript report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>346</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/3dligandsite" target="_blank">3DLigandSite</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>3DLigandSite is a web server for the prediction of ligand binding sites. Ligands bound to similar protein structures as the query structure are used to predict the binding site.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structure analysis</li><li>Protein binding sites</li><li>Nucleic acid sites, features and motifs</li><li>Structure prediction</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein binding site prediction</li><li>Protein-ligand docking</li><li>Protein structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>394</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/plangexq" target="_blank">PlanGexQ</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PlanGexQ is a user-friendly interactive tool for the curation and annotation of planarian morphologies and gene expression patterns in a centralized database. PlanGexQ allows any user to define reference morphologies using mathematical graphs and a drag-and-drop interface. Gene expression pattern images can be uploaded to the program and then registered into the reference morphologies. Annotations with terms from the planarian anatomy ontology are autommatically created by analyzing the gene expression patterns and their textual descriptions. PlanGexQ represents a complete methodology for centralizing with formal and annotated descriptions the morphologies and gene expression patterns of planarian worms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Gene expression</li><li>Ontology and terminology</li><li>Mathematics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Ontology visualisation</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/oncoyeasti" target="_blank">Oncoyeasti</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Automatically identify the corresponding human homologs of S. cerevisiae and the status of these homologs genes in tumor samples from The Cancer Genome Atlas Database and cell line samples from the Cancer Cell Line Encyclopedia.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Homology-based gene prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/oncoadmn/oncoyeasti.org" target="_blank">https://github.com/oncoadmn/oncoyeasti.org</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/antigendb" target="_blank">AntigenDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Contains 500 antigens to pathogenic species curated from the literature and other immunological resources. In it, a database entry contains information regarding the sequence, structure, origin, etc. of an antigen with additional information such as B and T-cell epitopes, MHC binding, function, gene-expression and post translational modifications, where available. It also provides links to major internal and external databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Immunoproteins, genes and antigens</li><li>Infectious disease</li><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Virulence prediction</li><li>Peptide immunogenicity prediction</li><li>Epitope mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>45</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biobike" target="_blank">BioBIKE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based environment enabling biologists with little programming expertise to combine tools, data and knowledge in novel and complex ways. It is composed of three components: a biological knowledge base, a graphical programming interface and an extensible set of tools. Users may also invent their own tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Software engineering</li><li>Data visualisation</li><li>Systems biology</li><li>Biology</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Sequence motif discovery</li><li>Metabolic labeling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>30</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ohdsi" target="_blank">OHDSI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Feasibility and Evaluation of a Large-Scale External Validation Approach for Patient-Level Prediction in an International Data Network.<br>Observational Health Data Sciences and Informatics.<br>ExistingStrokeRiskExternalValidation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Medical informatics</li><li>Preclinical and clinical studies</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Regression analysis</li><li>Network analysis</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/OHDSI/" target="_blank">https://github.com/OHDSI/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/secisaln" target="_blank">SECISaln</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Predict a SECIS element in the query sequence, split it into its constituent parts and align these against a precompiled database of eukaryotic SECIS elements.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>32</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/psicquic" target="_blank">PSICQUIC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Project within the HUPO Proteomics Standard Initiative (HUPO-PSI) that standardises programmatic access to molecular interaction databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>249</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gxd" target="_blank">Gene Expression Database (GXD)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Community resource of mouse developmental expression information. It integrates different types of expression data at the transcript and protein level and captures expression information from many different mouse strains and mutants.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Data submission, annotation and curation</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Gene expression clustering</li><li>Gene expression analysis</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>65</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/taverna_compbio" target="_blank">taveRNA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>TaveRNA hosts three RNA web services: alteRNA, inteRNA and pRuNA. alteRNA is an alternative to many RNA folding methods. inteRNA predicts the joint secondary structure of two RNA sequences. pRuNA is an database pruning method which given a query RNA returns only a few ncRNAs as potential regulators.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li><li>Nucleic acid structure analysis</li><li>Gene transcripts</li><li>RNA</li><li>Structure prediction</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Structure visualisation</li><li>RNA secondary structure prediction</li><li>RNA secondary structure alignment</li><li>RNA secondary structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>102</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/arts_antibiotics" target="_blank">ARTS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Antibiotic Resistant Target Seeker. Specific and efficient genome mining for antibiotics with interesting and novel targets. Automate the screening of large amounts of sequence data and to focus on the most promising strains that produce antibiotics with new modes of action. It integrates target directed genome mining methods, antibiotic gene cluster predictions and ‘essential gene screening’ to provide an interactive page for rapid identification of known and putative targets in BGCs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Plant biology</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>60</div></td>
      <td><div class="link short click_expand"><a href="https://bitbucket.org/ziemertlab/arts" target="_blank">https://bitbucket.org/ziemertlab/arts</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GeoNode-Tools4MSP" target="_blank">GeoNode-Tools4MSP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Open source tools to support Integrated Coastal Management and Maritime Spatial Planning.<br>Geospatial tools to support Maritime Spatial Planning.<br>GeoNode is a web-based application and platform for developing geospatial information systems (GIS) and for deploying spatial data infrastructures (SDI).<br>Tools4MSP is a python-based Free and Open Source Software (FOSS) for geospatial analysis in support of Maritime Spatial Planning (MSP) and marine environmental management. Tools4MSP implements models for Cumulative Effects Assessment (CEA), Maritime Use Conflict (MUC) Analysis and Marine Ecosystem Services Threat (MES-Threat) analysis. The package can be used as stand-alone library or as integrated GeoNode Plugin providing additional functionalities to geospatial CMS and enhancing usability through a graphical user interface (GUI).<br>Gallery Quick Start Try the Demo Blog.<br>The webtools (GeoNode integration) analysis workflow follows a four step approach:</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Database management</li><li>Ecology</li><li>Environmental science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Aggregation</li><li>Service management</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/CNR-ISMAR/tools4msp" target="_blank">https://github.com/CNR-ISMAR/tools4msp</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genebase" target="_blank">GeneBase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>GeneBase 1.1 is a tool with a graphical interface useful for parsing, structuring and indexing data from the National Center for Biotechnology Information (NCBI) Gene data bank.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Data acquisition</li><li>Human genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>29</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/njplot" target="_blank">NJplot</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>NJplot is a tool for visualizing binary trees such as the phylogenetic trees output from the PHYLIP programs. Available for several platforms including Windows, MacOS, Linux and Solaris.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Phylogeny</li><li>Sequencing</li><li>Gene expression</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree generation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Phylogenetic tree analysis</li><li>Phylogenetic tree reconstruction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1,260</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/odb" target="_blank">ODB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Contains all known and conserved operons in completely sequenced genomes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene structure</li><li>Gene expression</li><li>Model organisms</li><li>Database management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Enrichment analysis</li><li>Operon prediction</li><li>Operon drawing</li><li>PCR primer design</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>21</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/repeatsdb" target="_blank">RepeatsDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of annotated tandem repeat protein structures.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence composition, complexity and repeats</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Structure classification</li><li>Sequence visualisation</li><li>Protein structure analysis</li><li>Query and retrieval</li><li>Structure visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence features : json</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/psicquic_view" target="_blank">PSICQUIC view</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Standardises programmatic access to molecular interaction databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/multimorbidity" target="_blank">Multimorbidity</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Multimorbidity is database of genetically interpretable comorbidities among common diseases in UK Biobank.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Molecular interactions, pathways and networks</li><li>Pathology</li><li>Endocrinology and metabolism</li><li>Biobank</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Query and retrieval</li><li>Regression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ggbase" target="_blank">GGBase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Infrastructure for genetics of gene expression package GGtools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mirbaseconverter" target="_blank">miRBaseConverter</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive tool for converting and retrieving the miRNA Name, Accession, Sequence, Version, History and Family information in different miRBase versions. It can process a huge number of miRNAs in a short time without other depends.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/taoshengxu/miRBaseConverter" target="_blank">https://github.com/taoshengxu/miRBaseConverter</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/kraken2" target="_blank">kraken2</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Kraken 2 is the newest version of Kraken, a taxonomic classification system using exact k-mer matches to achieve high accuracy and fast classification speeds. This classifier matches each k-mer within a query sequence to the lowest common ancestor (LCA) of all genomes containing the given k-mer. The k-mer assignments inform the classification algorithm.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomics</li><li>Taxonomy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>dna sequence : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>taxonomy : tsv</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/DerrickWood/kraken2" target="_blank">https://github.com/DerrickWood/kraken2</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/euhcvdb" target="_blank">European Hepatitis C virus database (euHCVdb)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>European Hepatitis C virus database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Virology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence record : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>114</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/yrgate" target="_blank">yrGATE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based gene-structure annotation tool for the identification and dissemination of eukaryotic genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td></td>
      <td><div class='short click_expand'>Not licensed</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/seqminer-variant" target="_blank">SEQMINER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An R package for annotating and querying files of sequence variants (e.g., VCF/BCF files) and summary association statistics (e.g., METAL/RAREMETAL files), and for integrating bioinformatics databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence analysis</li><li>Sequence annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>18</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/seqminer-efficiently-read-sequence-data-into-r.html" target="_blank">http://www.mybiosoftware.com/seqminer-efficiently-read-sequence-data-into-r.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ChickpeaMine" target="_blank">ChickpeaMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A mine with chickpea data (both desi and kabuli varieties) from the Legume Information Systems (LIS) tripal.chado database</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/VaDER" target="_blank">VaDER</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Deep learning for clustering of multivariate clinical patient trajectories with missing values.<br>Deep learning for clustering of multivariate short time series with potentially many missing values.<br>This repository contains code for a new method for clustering multivariate short time series with potentially many missing values (published here), a setting commonly encountered in the analysis of longitudinal clinical data, but generally still poorly addressed in the literature.<br>Supporting code for the manuscript "Deep learning for clustering of multivariate clinical patient trajectories with missing values".<br>Supporting code for the manuscript "Deep learning for clustering of multivariate clinical patient trajectories with missing values"</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical imaging</li><li>Geriatric medicine</li><li>Personalised medicine</li><li>Neurology</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Parsing</li><li>Imputation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/johanndejong/VaDER_supporting_code" target="_blank">https://github.com/johanndejong/VaDER_supporting_code</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/evag" target="_blank">European Virus Archive</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Non-profit organisation dedicated to the characterisation, conservation, production, distribution and characterisation of biological materials in the field of virology.</div></td>
      <td><div class='topic short click_expand'><ul><li>Virology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/proteomicsdb" target="_blank">ProteomicsDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Explorable, interactive human proteome database, including MS/MS data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li><li>Transcriptomics</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Differential protein expression analysis</li><li>Protein interaction analysis</li><li>Visualisation</li><li>Query and retrieval</li><li>Protein property comparison</li><li>Mass spectrum visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein name : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>plot : svg</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>1,100</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rna_helicase_database" target="_blank">RNA helicase database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrates information on RNA helicases. The database allows retrieval of comprehensive information on sequence, structure and on biochemical and cellular functions of all RNA helicases from the most widely used model organisms Escherichia coli, Saccharomyces cerevisiae, Caenorhabditis elegans, Drosophila melanogaster, mouse and human. The database is structured according to the most recent helicase classification into helicase superfamilies and families, emphasizes phyologenetic relations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>RNA</li><li>Nucleic acid structure analysis</li><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Data retrieval</li><li>Structure database search</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>43</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/EM-DAT" target="_blank">EM-DAT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A cross-sectional analysis of the emergency events database from a pediatric perspective.<br>The international disasters database.<br>Disaster group: Two main groups of disasters are distinguished in EM-DAT: natural disasters and technological disasters. This field is automatically linked to the disaster sub-group and the disaster type. There is a third group ‘Complex disasters’ which include some major famine situation for which the drought were not the main causal factor. See Table 1 for the Disasters Classification.<br>In 1988, the Centre for Research on the Epidemiology of Disasters (CRED) launched the Emergency Events Database (EM-DAT).</div></td>
      <td><div class='topic short click_expand'><ul><li>Public health and epidemiology</li><li>Ecology</li><li>Mapping</li><li>Database management</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Imputation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://www.emdat.be/guidelines" target="_blank">https://www.emdat.be/guidelines</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biojs-io-matrix" target="_blank">biojs-io-matrix</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Parses 2D matrices (e.g. PAM).</div></td>
      <td><div class='topic short click_expand'><ul><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>70</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/geoboost" target="_blank">GeoBoost</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Command-line software package developed to address sparse or incomplete metadata in GenBank sequence records that relate to the location of the infected host (LOIH) of viruses. Given a set of GenBank accession numbers corresponding to virus GenBank records, it extracts, integrates and normalizes geographic information reflecting the LOIH of the viruses using integrated information from GenBank metadata and related full-text publications.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data acquisition</li><li>Natural language processing</li><li>Virology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pannzer2" target="_blank">pannzer2</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Protein ANNotation with Z-scoRE. Fully automated  functional annotation of prokaryotic and eukaryotic proteins of unknown function. Designed to predict the functional description (DE) and GO classes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>text : fasta-like (text)</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>56</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CMVdb" target="_blank">CytoMegaloVirus DataBase (CMVdb)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Public Omics Database for Systematic and Comparable Information of CMV.<br>Cytomegalovirus is a double-stranded DNA virus in the same family as herpesvirus found every where,infect people of all ages.<br>Cytomegalovirus Treatment & Management Kauser Akhter, MD Assistant Professor, Department of Internal Medicine, Florida State University College of Medicine; Associate Program Director, Infectious Diseases Fellowship Program, Orlando Health; Updated: Jan 05, 2017</div></td>
      <td><div class='topic short click_expand'><ul><li>Infectious disease</li><li>Immunology</li><li>Pharmacology</li><li>Surgery</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>PCR primer design</li><li>Database search</li><li>Protein modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td><div class="link short click_expand"><a href="http://weislab.com/WeiDOCK/include/content/CMV/" target="_blank">http://weislab.com/WeiDOCK/include/content/CMV/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/seattlesnps" target="_blank">SeattleSNPs variation discovery resource</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database for identifying, genotyping, and modelling the associations between single nucleotide polymorphisms (SNPs) in candidate genes and pathways that underlie inflammatory responses in humans.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA polymorphism</li><li>Genotype and phenotype</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/spectre-cytometry" target="_blank">Spectre</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integration, exploration, and analysis of high-dimensional single-cell cytometry data using Spectre.<br>A computational toolkit in R for the integration, exploration, and analysis of high-dimensional single-cell cytometry data.<br><br>||| NAME (Spectre) SIMILAR TO (PUB. DIFFERENT) bio.tools/spectre-phylogeny (SPECTRE), bio.tools/spectre (SPECtre)</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Imaging</li><li>Cytometry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Dimensionality reduction</li><li>Essential dynamics</li><li>Principal component visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/peroxibase" target="_blank">PeroxiBase peroxidase family database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sequences from peroxidase superfamilies with annotation on putative functions and transcription regulation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>103</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ocean-barcode-atlas" target="_blank">Ocean-Barcode-Atlas</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web service to explore the biodiversity and biogeography of marine organisms.<br>- "Community ecological analysis" allows one to search for a taxon in a metabarcoding database in order to obtain.<br>ecological analyses (alpha and beta diversity) using graphical representations (NMDS, boxplot..).<br>- "Sequence based query" allows one to interrogate a metabarcoding database in order to obtain the abundance,.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogeny</li><li>Sequencing</li><li>Sample collections</li><li>Ecology</li><li>Metatranscriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>DNA barcoding</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://gitlab.osupytheas.fr/ocean_atlas/oba" target="_blank">https://gitlab.osupytheas.fr/ocean_atlas/oba</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/HowDe-SBT" target="_blank">HowDe-SBT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Improved representation of sequence Bloom trees | Sequence Bloom Tree, supporting determined/how split filters | Jellyfish (Version 2.2.0 or later)</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metabolite_autoplotter" target="_blank">Metabolite_AutoPlotter</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An application to process and visualise metabolite data in the web-browser.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li><li>Proteomics experiment</li><li>Mapping</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Natural product identification</li><li>Read pre-processing</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/crnt4sbml" target="_blank">CRNT4SBML</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CRNT4SBML is a Python package for the detection of bistability in biochemical reaction networks.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Mathematics</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway analysis</li><li>Parsing</li><li>Statistical calculation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/utrsite" target="_blank">UTRSite</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A collection of functional sequence patterns located in UTR sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular biology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>228</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sci.ai" target="_blank">sci.AI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sci.AI is a free platform that “understands” research papers and “explains” them to analytics and search algorithms.  <br>The platform takes all biomedical papers, trials, and projects and interlinks research findings in a semantic graph. Then, it mimics human reasoning to return discovered pathways and support research</div></td>
      <td><div class='topic short click_expand'><ul><li>Biomedical science</li><li>Biology</li><li>Biotechnology</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Document clustering</li><li>Named-entity and concept recognition</li><li>Text annotation</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/BeanMine" target="_blank">BeanMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A mine with common bean data from the Legume Info tripal.chado database</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/wigwams" target="_blank">Wigwams</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Biclustering approach for time series data, as it identifies genes exhibiting co-expression across subsets of available time course data sets.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene regulation</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>18</div></td>
      <td></td>
      <td><div class='short click_expand'>Not licensed</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/odg" target="_blank">ODG</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool for generating, querying, and analyzing multi-omics comparative databases to facilitate biological understanding.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data integration and warehousing</li><li>Comparative genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Annotation</li><li>Data handling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pdb-reader-matlab" target="_blank">Fast PDB Reader</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MATLAB tool for fast reading of PDB files.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Biology</li><li>Chemistry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein structure : pdb</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SDG" target="_blank">SDG</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Sequence Distance Graph framework for genome assembly and analysis | Sequence Distance Graph framework: graph + reads + mapping + analysis | The Sequence Distance Graph (SDG) is a framework to work with genome graphs and sequencing data. It provides a workspace built around a Sequence Distance Graph, datastores for paired, linked and long reads, read mappers, and k-mer counters. It can be used to perform different types of sequence analyses | Index of /opendata/data/sdg_datasets | Welcome to SDG’s documentation! — SDG documentation | Websites for you and your projects, hosted directly from your GitHub repository. Just edit, push, and your changes are live | Websites for you and your projects | Hosted directly from your GitHub repository. Just edit, push, and your changes are live | Ready to get started? Build your own site from scratch or generate one for your project</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Sequence assembly</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Scaffolding</li><li>Genome assembly</li><li>De-novo assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://opendata.earlham.ac.uk/opendata/data/sdg_datasets/" target="_blank">https://opendata.earlham.ac.uk/opendata/data/sdg_datasets/</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CIViCmine" target="_blank">CIViCmine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CIViCmine is a literature-mined database of clinically relevant cancer biomarkers from data from PubMed and Pubmed Central Open Access subset.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Machine learning</li><li>Biomarkers</li><li>Natural language processing</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text annotation</li><li>Text mining</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/jakelever/civicmine" target="_blank">https://github.com/jakelever/civicmine</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bugasbase" target="_blank">BugaSbase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A relational database containing microbial gene expression and comparative genomic hybridisation experimental data. The data is mostly derived from whole genome microarrays designed by the BµG@S group, however data from other microarrays is included and the database is being modified to support RNA-Seq analysis. Public access is available for datasets that have been published and login credentials can be obtained for data submission.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>RNA-Seq</li><li>Experimental design and studies</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Deposition</li><li>Gene expression analysis</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/FIT" target="_blank">FIT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Statistical modeling tool for transcriptome dynamics under fluctuating field conditions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Environmental science</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Quantification</li><li>Regression analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>MPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/binom" target="_blank">BiNoM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Cytoscape plugin, developed to facilitate the manipulation of biological networks represented in standard systems biology formats (SBML, SBGN, BioPAX) and to carry out studies on the network structure. It provides the user with a complete interface for the analysis of biological networks in Cytoscape environment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Data acquisition</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Plug-in</li></ul></div></td>
      <td><div class='citations short click_expand'>35</div></td>
      <td></td>
      <td><div class='short click_expand'>LGPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mzjava" target="_blank">MzJava</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Library for the analysis of mass spectrometry data from large scale proteomics and glycomics experiments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li><li>Protein sequence cleavage</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : mgf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>22</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/taxamat" target="_blank">Taxamat</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Automated biodiversity data management tool.<br> Taxamat is avialable either as a single executable file or as an installable package. In addition you can also download example test data together with a manual that contains a step-by-step user'guide.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microbial ecology</li><li>Biodiversity</li><li>Taxonomy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Parsing</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/catalysis-hub" target="_blank">Catalysis-hub.org</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web-platform for sharing data and software for computational catalysis research.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li><li>Database management</li><li>Chemistry</li><li>Physics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structure assignment</li><li>Molecular surface calculation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>36</div></td>
      <td><div class="link short click_expand"><a href="https://www.catalysis-hub.org/energies" target="_blank">https://www.catalysis-hub.org/energies</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/adenosiland" target="_blank">Adenosiland</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Adenosiland represents the first attempt of an integrated bioinformatics and chemoinformatics web-resource dedicated to adenosine receptors. This informatics platform provides a wide-ranging of structure based and ligand based query functions to facilitate the exploration of adenosine receptor structures from primary sequences to three-dimensional architectures. Here, we present an overview of Adenosiland platform describing the most valuable searching tools and their functionalities.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>26</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/orthodb" target="_blank">OrthoDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Catalog of eukaryotic orthologous protein-coding genes. OrthoDB explicitly delineates orthologs at each radiation along the species phylogeny. Available protein descriptors, together with Gene Ontology and InterPro attributes, serve to provide general descriptive annotations of the orthologous groups, and facilitate comprehensive database querying. Data sources include proteomes from arthropods, fungi, vertebrates and basal metazoans.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogeny</li><li>Function analysis</li><li>Gene and protein families</li><li>Evolutionary biology</li><li>Genomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene functional annotation</li><li>Protein function analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>203</div></td>
      <td><div class="link short click_expand"><a href="http://expasy.org" target="_blank">http://expasy.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lincs_canvas_browser" target="_blank">LINCS Canvas Browser (LCB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Interactive web server to query, browse and interrogate LINCS L1000 gene expression signatures.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pharmacology</li><li>Gene transcripts</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Enrichment analysis</li><li>Gene expression analysis</li><li>Gene-set enrichment analysis</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>160</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/drug-herb-interaction" target="_blank">Drug-herb-interaction</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A constructing a bilingual website with validated database for Herb and Western medicine interactions using Ginseng, Ginkgo and Dong Quai as examples.</div></td>
      <td><div class='topic short click_expand'><ul><li>Preclinical and clinical studies</li><li>Literature and language</li><li>Medicine</li><li>Database management</li><li>Drug metabolism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Database search</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rediportal" target="_blank">REDIportal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of A-to-I (deamination of adenosines to inosines) events that enables to search RNA editing sites by genomic region, gene name and other relevant features as the tissue of origin.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-seq</li><li>Sequencing</li><li>Gene transcripts</li><li>RNA</li><li>Rare diseases</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Structure editing</li><li>Genotyping</li><li>Editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>100</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Hayai-Annotation_Plants" target="_blank">Hayai-Annotation Plants</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R package for an Ultra-Fast and Comprehensive Gene Annotation in Plants.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Taxonomy</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/kdri-genomics/Hayai-Annotation-Plants/issues" target="_blank">https://github.com/kdri-genomics/Hayai-Annotation-Plants/issues</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/elminstancemapper" target="_blank">ELMInstanceMapper</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Map ELM predictions of a query sequence to known annotated ELM instances.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>484</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/153" target="_blank">https://www.biocatalogue.org/services/153</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/arion_4_omics" target="_blank">Arion 4 Omics</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>High performance, ‘end-to-end’ analysis pipeline for the classification of omics profiles. Incorporating highly parallel architecture and sophisticated database technologies to overcome inherent technology based bottlenecks, currently faced in the Life Science research path. Arion is a scalable platform providing rapid exploratory analysis via machine learning, visualization and statistical modules with topology planned for a future release.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Biomarkers</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Differential protein expression analysis</li><li>Query and retrieval</li><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://multicoredynamics.com/multicore-dynamics-media-content/" target="_blank">http://multicoredynamics.com/multicore-dynamics-media-content/</a></div></td>
      <td><div class='short click_expand'>Proprietary</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/predictprotein" target="_blank">PredictProtein</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PredictProtein is a protein sequence analysis and structure prediction tool.  Users provide a protein sequence, and PredictProtein reports similar sequences, PROSITE sequence motifs, and various types of structure prediction information.  You can also use META PredictProtein to submit a sequence to one web form and query several other servers at once, retrieving the results by e-mail.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Protein structural motifs and surfaces</li><li>Structure prediction</li><li>Sequence sites, features and motifs</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein secondary structure prediction (helices)</li><li>Protein fold recognition</li><li>Protein secondary structure prediction</li><li>Protein structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1,308</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genprobis" target="_blank">GenProBiS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Maps sequence variants to protein structures from the Protein Data Bank (PDB), and further to protein–protein, protein–nucleic acid, protein–compound, and protein–metal ion binding sites. Binding sites are defined by local structural comparisons of whole protein structures using the Protein Binding Sites (ProBiS) algorithm and transposition of ligands from the similar binding sites found to the query protein using the ProBiS-ligands approach</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein binding sites</li><li>Transcription factors and regulatory sites</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein binding site prediction</li><li>Post-translation modification site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein structure : pdb</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/RBP-Maps" target="_blank">RBP-Maps</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Enables robust generation of splicing regulatory maps and rapid query of regulatory patterns of RNA binding protein (RBP) of interest.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA immunoprecipitation</li><li>Gene transcripts</li><li>RNA splicing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Splice transcript prediction</li><li>Sequence merging</li><li>Splice site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/conet_app" target="_blank">CoNet app</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Inference of biological association networks using Cytoscape.<br>CoNet is a Cytoscape plugin that detects significant associations in presence/absence and abundance matrices.<br>Find relationships between repeatedly observed items.<br>CoNet is a Cytoscape plugin that detects significant non-random patterns of co-occurrence (copresence and mutual exclusion) in incidence and abundance data. It has been designed with (microbial) ecological data in mind, but can be applied in general to infer relationships between objects observed in different samples (for example between genes present or absent across organisms).<br>CoNet - Co-occurrence Network inference.<br>CoNet detects significant non-random patterns of co-occurrence (copresence and mutual exclusion) in incidence and abundance data.<br>12th January 2018: Thanks to Lisa Röttjers, there is now an R implementation of CoNet.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li><li>Microbial ecology</li><li>Metagenomics</li><li>Model organisms</li><li>Metatranscriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Relation extraction</li><li>Parsing</li><li>Expression correlation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>127</div></td>
      <td><div class="link short click_expand"><a href="http://sourceforge.net/projects/conet/" target="_blank">http://sourceforge.net/projects/conet/</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/AGONOTES" target="_blank">AGONOTES</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Robot Annotator for Argonaute Proteins.<br>Welcome to AgoNotes home page! AGONOTES is an online service tool to identify Argonaute protein in proteins of all domains of life and annotate the argonaute domains. Users can submit any protein with fasta formation into the input field, or save proteins as a file ended with .faa, .fasta, .fa or .txt and then upload this file</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Genetic engineering</li><li>Model organisms</li><li>Sequence analysis</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Editing</li><li>Genome annotation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/imagej_nih" target="_blank">ImageJ (NIH)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Public domain Java image processing program inspired by NIH Image for the Macintosh. It was designed with an open architecture that provides extensibility via Java plugins.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data architecture, analysis and design</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Conversion</li><li>Loading</li><li>Visualisation</li><li>Image annotation</li><li>Statistical calculation</li><li>Editing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>image : bmp</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>24,129</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lcmsworld" target="_blank">lcmsWorld</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>High-Performance 3D Visualization Software for Mass Spectrometry.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Metabolomics</li><li>Proteomics</li><li>Imaging</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li><li>Ion counting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrum : thermo raw</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/enzyme_portal" target="_blank">Enzyme Portal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web service which integrates publicly available information about enzymes, such as small-molecule chemistry, biochemical pathways and drug compounds.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pysradb" target="_blank">pysradb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Python package to query next-generation sequencing metadata and data from NCBI Sequence Read Archive.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Sequencing</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/WheatMine" target="_blank">WheatMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Wheat IWGSC RefSeq v1.0</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rbgl" target="_blank">RBGL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A fairly extensive and comprehensive interface to the graph algorithms contained in the BOOST library.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>57</div></td>
      <td><div class="link short click_expand"><a href="http://www.bioconductor.org" target="_blank">http://www.bioconductor.org</a></div></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/swepep" target="_blank">SwePep</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An endogenous peptide database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>101</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gwasrapidd" target="_blank">gwasrapidd</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Package to query, download and wrangle GWAS Catalog data.</div></td>
      <td><div class='topic short click_expand'><ul><li>GWAS study</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ramiromagno/gwasrapidd/issues" target="_blank">https://github.com/ramiromagno/gwasrapidd/issues</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/clingen-variant-curation-interface" target="_blank">ClinGen Variant Curation Interface</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ClinGen Variant Curation Interface is a variant classification platform for the application of evidence criteria from ACMG/AMP guidelines.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Personalised medicine</li><li>Data submission, annotation and curation</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Variant classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://www.ncbi.nlm.nih.gov/clinvar/" target="_blank">https://www.ncbi.nlm.nih.gov/clinvar/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/smpdb" target="_blank">SMPDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Interactive, visual database containing more than 350 small-molecule pathways found in humans. It is designed specifically to support pathway elucidation and pathway discovery in clinical metabolomics, transcriptomics, proteomics and systems biology. It provides exquisitely detailed, hyperlinked diagrams of human metabolic pathways, metabolic disease pathways, metabolite signaling pathways and drug-action pathways.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li><li>Molecular interactions, pathways and networks</li><li>Pathology</li><li>Endocrinology and metabolism</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Pathway or network analysis</li><li>Query and retrieval</li><li>Gene to pathway mapping</li><li>Protein quaternary structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>168</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/regBase" target="_blank">regBase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Whole genome base-wise aggregation and functional prediction for human non-coding regulatory variants | Base-wise aggregation and functional prediction for human non-coding regulatory variants | regBase is a comprehensively integrated non-coding regulatory prediction scores and composite prediction models from existing tools for base-wise annotation of human genome. As such, the regBase resource provides convenience to prioritize functional regulatory SNVs and assist the fine mapping of causal regulatory SNVs without queries from numerus sources</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li><li>Functional, regulatory and non-coding RNA</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Virulence prediction</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_LIGM-DB" target="_blank">IMGT LIGM-DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/LIGM-DB is the IMGT® database for nucleotide sequences with translation of immunoglobulins (IG) and T cell receptors (TR).<br>Annotation is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li><li>Immunoinformatics</li><li>Immunogenetics</li><li>Sequence analysis</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence annotation</li><li>Sequence database search</li><li>Sequence visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>ontology concept : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database search results : fasta</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>382</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/prex" target="_blank">PREX</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of peroxiredoxin (Prx or PRDX) protein sequences unambiguously classified into one of six distinct subfamilies. Subfamily members were identified using the Deacon Active Site Profiler (DASP) bioinformatics tool to focus in on functionally relevant sequence fragments surrounding key residues required for protein activity. Searches of this database can be conducted by protein annotation, accession number, PDB ID, organism name or protein sequence.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Gene and protein families</li><li>Proteins</li><li>Protein binding sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Protein feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>71</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SALTClass" target="_blank">SALTClass</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Classifying clinical short notes using background knowledge from unlabeled data.<br>Short and Long Text Classifier using clustering-based enrichment.<br>.. :Travis: image:: https://travis-ci.com/bagheria/saltclass.svg?token=fYbdQUbpnoucyHyb3fs2&branch=master .. _Travis: https://travis-ci.com/bagheria/saltclass.<br>saltclass (Short and Long Text Classifier) is a Python module for text classification built under the MIT license. The project was started in 2018 at the Department of Methodology & Statistics, Utrecht University</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical informatics</li><li>Cardiology</li><li>Preclinical and clinical studies</li><li>Machine learning</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Relation extraction</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bagheria/saltclass" target="_blank">https://github.com/bagheria/saltclass</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/libmgf" target="_blank">libmgf</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A flex/bison-based Mascot Generic Format (MGF) parser library.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Bioinformatics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Read pre-processing</li><li>Generation</li><li>Formatting</li><li>Format validation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : dta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : dta</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>64</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/simmap" target="_blank">SiMMap</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The SiMMap server statistically derives site-moiety maps which describe the relationship between moiety preferences and the physico-chemical properties of the binding site. Users input the interaction profiles of a query target protein and its docked compound.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li><li>Mapping</li><li>Drug discovery</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Hydrogen bond calculation</li><li>Protein-ligand docking</li><li>Molecular docking</li><li>Mapping</li><li>Protein model validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>30</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/stattag" target="_blank">StatTag</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Facilitating reproducible research through direct connection of data analysis with manuscript preparation.<br>StatTag is a free plug-in for conducting reproducible research and creating dynamic documents using Microsoft Word with the R, Stata and SAS statistical packages.</div></td>
      <td><div class='topic short click_expand'><ul><li>Statistics and probability</li><li>Informatics</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Editing</li><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://stattag.org" target="_blank">http://stattag.org</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/VegMarks" target="_blank">VegMarks</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Construction of a core collection of eggplant (Solanum melongena L.) based on genome-wide SNP and SSR genotypes.<br>a DNA marker database for vegetables</div></td>
      <td><div class='topic short click_expand'><ul><li>Plant biology</li><li>DNA polymorphism</li><li>Genotype and phenotype</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Genotyping</li><li>Nucleic acid feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mendelvar" target="_blank">MendelVar</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Gene prioritization at GWAS loci using phenotypic enrichment of Mendelian disease genes.<br>Initial set-up for standalone MendelVar usage.<br>A new tool that blends your everyday work apps into one. It's the all-in-one workspace for you and your team.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Molecular interactions, pathways and networks</li><li>Workflows</li><li>Pathology</li><li>GWAS study</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Enrichment analysis</li><li>Genetic mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://bitly.com/MendelVar" target="_blank">https://bitly.com/MendelVar</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/prosextract" target="_blank">prosextract</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Process the PROSITE motif database for use by patmatmotifs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/psi-mouse" target="_blank">PSI-MOUSE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Predicting Mouse Pseudouridine Sites From Sequence and Genome-Derived Features.<br>Pseudouridine (Ψ) is the first discovered and the most abundant RNA modification that has been widely study in the past decades.<br>Experimental-validated mouse Ψ Sites.<br>High-accuracy predictor for mouse Ψ site identification.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA splicing</li><li>Functional, regulatory and non-coding RNA</li><li>Gene transcripts</li><li>Transcriptomics</li><li>DNA binding sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence feature detection</li><li>PTM site prediction</li><li>Splice site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="http://psimouse.rnamd.com" target="_blank">http://psimouse.rnamd.com</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genevestigator" target="_blank">Genevestigator</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>High performance search engine for gene expression. It integrates thousands of manually curated, well described public microarray and RNAseq experiments and nicely visualizes gene expression across different biological contexts such as diseases, drugs, tissues, cancers, cell lines or genotypes. The high diversity of curated experiments allows the tool to project your genes or your data against a broad spectrum of reference profiles and datasets.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biomarkers</li><li>Genomics</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene functional annotation</li><li>Expression data visualisation</li><li>Query and retrieval</li><li>Gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cdfread" target="_blank">cdfread</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A simple reader of mass spectra in netCDF.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Chromatogram visualisation</li><li>Mass spectrum visualisation</li><li>Annotation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : netcdf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : netcdf</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/CRYSPLOT" target="_blank">CRYSPLOT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Visualization environment for plotting properties of crystalline solids as computed by means of the CRYSTAL code.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phasing</li><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hssp" target="_blank">HSSP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Homology-derived secondary structure of proteins database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein secondary structure</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein features : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>436</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cellcycledb" target="_blank">Cell cycle database (CellCycleDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A collection of genes and proteins involved in human and yeast cell cycle.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Gene and protein families</li><li>Cell biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>21</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cleanex" target="_blank">CleanEx database of gene expression profiles</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Public gene expression data via unique approved gene symbols and which represents heterogeneous expression data produced by different technologies in a way that facilitates joint analysis and cross-dataset comparisons.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene expression profile : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/operamate" target="_blank">OperaMate</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This flexible R package deals with the data generated by PerkinElmer's Opera High Content Screening System. Its functions include the data importing, normalization and quality control, hit detection and function analysis.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Function analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Data handling</li><li>Validation</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>matrix : scf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>image : image format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mgi" target="_blank">Mouse Genome Informatics (MGI)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrated genetic, genomic, and biological data for mouse.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data reference : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>247</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/resid" target="_blank">RESID database of protein modifications</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive collection of annotations and structures for protein modifications including amino-terminal, carboxyl-terminal and peptide chain cross-link post-translational modifications.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein structural motifs and surfaces</li><li>Protein modifications</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : xml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/expression_profiler" target="_blank">Expression Profiler</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web based platform for microarray data analysis developed at the EBI. This resource is integrated with the ArrayExpress database, a public repository for microarray data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data architecture, analysis and design</li><li>Gene expression</li><li>Microarray experiment</li><li>Data integration and warehousing</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression QTL analysis</li><li>Gene expression profiling</li><li>Gene expression profile clustering</li><li>Gene expression analysis</li><li>Gene expression profile comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>97</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/vectorinfo" target="_blank">VectorInfo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web resource for medically important Indian arthropod disease vectors.<br>VectorInfo is a freely accessible web resource, emphasised on medically important Indian arthropods funded by Indian Council of Medical Research (ICMR) and maintained by one of its premier institute, Vector Control Research Centre (VCRC). VectorInfo elucidates and curates medically important Indian arthropod's biological, omics technologies to adopt a holistic view of the molecules that make up an organism, aimed at the detection of genomics, transcriptomics, proteomics, enzymes & pathways and immune specific genes. The nitty-gritty of VectorInfo is aimed at scrutinizing all the possible information on Indian disease vectors in a single window for the scientific community. The database affords 53 medically important Indian arthropod's biological and omics information well-structured and provided with downloadable facilities</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Molecular interactions, pathways and networks</li><li>Pathology</li><li>Database management</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/regals" target="_blank">REGALS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A general method to deconvolve X-ray scattering data from evolving mixtures.<br>Software package for small angle X-ray scattering (SAXS) mixture deconvolution by REGularized Alternating Least Squares. It has been applied to datasets from chromatography-coupled SAXS, time-resolved SAXS, and equilibrium titrations. See our paper (Meisburger, Xu, & Ando, 2020) for details.<br>matlab/ - MATLAB implementation of the REGALS library.<br>The Python implementation was developed in Python 3. The REGALS library requires numpy and scipy. The demos use Jupyter notebooks with h5py for data import and matplotlib for plotting. The code has been tested with the following versions:.</div></td>
      <td><div class='topic short click_expand'><ul><li>X-ray diffraction</li><li>Proteomics experiment</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Deisotoping</li><li>Scatter plot plotting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ecogene" target="_blank">EcoGene</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Information about the E. coli K-12 genome and proteome sequences, including extensive gene bibliographies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>137</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/phyloskeleton" target="_blank">phyloSkeleton</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Taxon selection, data retrieval and marker identification for phylogenomics .</div></td>
      <td><div class='topic short click_expand'><ul><li>Evolutionary biology</li><li>Phylogenomics</li><li>Taxonomy</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gcanno" target="_blank">gCAnno</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A graph-based single cell type annotation method.<br>gCAnno-A tool for cell type annotation of single-cell transcriptome sequencing.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>RNA-Seq</li><li>Cell biology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Query and retrieval</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tripletres" target="_blank">TripletRes</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Deducing high-accuracy protein contact-maps from a triplet of coevolutionary matrices through deep residual convolutional networks.<br>A contact map prediction based on a triplet of coevolutionary features and deep residual neural networks.<br>TripletRes is a method for protein inter-residue contact prediction. For a query sequence, TripletRes starts with the collection of deep multiple sequence alignments (MSAs) through whole-genome and metagenome sequence databases. Next, three complimentary coevolutionay feature matrices (covariance martrix, precision matrix and the pseudolikelihood maximization) extracted from the MSAs are used to create contact-map models through deep residual convolutional neural network training.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein folding, stability and design</li><li>Machine learning</li><li>Metagenomics</li><li>Structure prediction</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein folding analysis</li><li>Contact map calculation</li><li>Fold recognition</li><li>Residue contact prediction</li><li>Ab initio structure prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/distild" target="_blank">DistiLD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Diseases and traits in linkage disequilibrium (LD) blocks. Data from HapMap Project is used to partition chromosomal regions which are associated with a disease into LD blocks. This allows researchers to answer the question of which SNPs associated with diseases are in LD with a gene of interest?</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>Pathology</li><li>Mapping</li><li>GWAS study</li><li>DNA polymorphism</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>SNP detection</li><li>Query and retrieval</li><li>Linkage disequilibrium calculation</li><li>Genetic mapping</li><li>Genotyping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>20</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mblastall" target="_blank">mblastall</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Search nucleotide database with nucleotide query sequence(s).</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence search results : tsv</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>62,099</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/mblastall/mblastall/2.2.26" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/mblastall/mblastall/2.2.26</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/IMGT_2Dstructure-DB" target="_blank">IMGT 2Dstructure-DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>IMGT/2Dstructure-DB is the IMGT® database for 2D structures (IMGT Colliers de Perles) of immunoglobulins (IG) or antibodies, T cell receptors (TR), major histocompatibility (MH) proteins and related proteins of the immune system (RPI).<br>Annotation is based on the IMGT-ONTOLOGY concepts.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunogenetics</li><li>Immunoinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence visualisation</li><li>Protein secondary structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>ontology concept : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>205</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/affxparser" target="_blank">affxparser</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Package that parses Affymetrix files using the Affymetrix' Fusion SDK. Both ASCII- and binary-based files are supported. Currently, there are methods for reading chip definition file (CDF) and a cell intensity file (CEL). These files can be read either in full or in part. For example, probe signals from a few probesets can be extracted very quickly from a set of CEL files into a convenient list structure.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/HenrikBengtsson/affxparser" target="_blank">https://github.com/HenrikBengtsson/affxparser</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ahmii" target="_blank">AHMII</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This tool offers a search engine for particular strains present in culture collections and databases in bacteria, fungi, yeasts and cell lines.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Microbial collection</li><li>Microbiology</li><li>Cell culture collection</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Query and retrieval</li><li>Deposition</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/spliceaid" target="_blank">SpliceAid</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Experimental RNA target motifs bound by splicing proteins in humans.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Gene structure</li><li>Sequence sites, features and motifs</li><li>Nucleic acid sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>66</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nembase4" target="_blank">NEMBASE4</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Resource for nematode transcriptome analysis, and a research tool for nematode biology, drug discovery and vaccine design.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Gene expression</li><li>Gene and protein families</li><li>Drug discovery</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>DNA vaccine design</li><li>Sequence tagged site (STS) mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>415</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/comptox_chemistry_dashboard" target="_blank">CompTox Chemistry Dashboard</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Community data resource for environmental chemistry.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cheminformatics</li><li>Compound libraries and screening</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>184</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pals-v" target="_blank">PALS-V</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Decomposing metabolite set activity levels with PALS.<br>Pathway analysis is an important task in understanding complex metabolomic data. Here we introduce PALS (Pathway Activity Level Scoring), a complete tool that performs database queries of pathways, decomposes activity levels in pathways via the PLAGE method, as well as presents the results in a user-friendly manner. The results are found to be more robust to noise and missing peaks compared to the alternatives (ORA, GSEA). This is particularly important for metabolomics peak data, where noise and missing peaks are prevalent.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li><li>Molecular interactions, pathways and networks</li><li>Plant biology</li><li>Endocrinology and metabolism</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic pathway prediction</li><li>Expression profile pathway mapping</li><li>Gene-set enrichment analysis</li><li>Natural product identification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/BPt" target="_blank">Brain Predictability toolbox</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Brain Predictability toolbox (BPt) is a Python based Machine Learning library designed for working with Neuroimaging data. This library is particularly suited towards working with large neuroimaging datasets, where a large number of subjects with potentially multimodal data are avaliable.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Machine learning</li><li>Medical imaging</li><li>Physiology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Regression analysis</li><li>Splitting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cgap" target="_blank">Cancer Genome Anatomy Project</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Goal is to determine the gene expression profiles of normal, precancer, and cancer cells; resources for human and mouse include ESTs, gene expression patterns, SNPs, cluster assemblies, cytogenetic information, and tools to query and analyze the data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li><li>Anatomy</li><li>Oncology</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression analysis</li><li>Gene expression QTL analysis</li><li>Gene expression clustering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Geneshot" target="_blank">Geneshot</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Search engine for ranking genes from arbitrary text queries | Submit biomedical terms to receive ranked lists of relevant genes | Submit any search terms to Geneshot to receive prioritized genes that are most relevant to the search terms. Geneshot finds publications that mention both the search terms and genes. It then prioritizes these genes using various methods: 1) list of genes from publications; 2) predicted genes using gene-gene similarity matrices derived from a variety of resources ( ARCHS4 : Enrichr : Tagger : AutoRIF : GeneRIF ) | Examples: Wound healing : Hair loss : Trichostatin A : Glioblastoma : Diabetes</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene prediction</li><li>Expression correlation analysis</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hamap_sparql" target="_blank">HAMAP_SPARQL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Portable annotation pipeline for genomes and proteomes.<br>HAMAP is a system for the classification and annotation of protein sequences. It consists of a collection of expert-curated protein family signatures and rules that specify annotations that apply to family members. HAMAP was originally developed by the Swiss-Prot group to help curators annotate UniProtKB/Swiss-Prot records and was subsequently made available to external users via the HAMAP-Scan web service. HAMAP signatures are also integrated into InterPro and the HAMAP rules into UniRule, the core part of the UniProt automatic annotation pipeline.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Gene expression</li><li>Workflows</li><li>Gene and protein families</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence classification</li><li>Genome annotation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/sib-swiss/HAMAP-SPARQL" target="_blank">https://github.com/sib-swiss/HAMAP-SPARQL</a></div></td>
      <td><div class='short click_expand'>CC0-1.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bmrb" target="_blank">BMRB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Biological Magnetic Resonance Data Bank (BioMagResBank or BMRB) is a repository for data from NMR Spectroscopy on proteins, peptides, nucleic acids, and other biomolecules.</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>Chemistry</li><li>Metabolomics</li><li>Biophysics</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Deposition</li><li>Data retrieval</li><li>Database search</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://bmrb.pdbj.org" target="_blank">https://bmrb.pdbj.org</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/synergyage" target="_blank">SynergyAge</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A curated database for synergistic and antagonistic interactions of longevity-associated genes.<br>Long- and short-lived genetic mutants.<br><br>SynergyAge database hosts high-quality, manually curated information about the synergistic and antagonistic lifespan effects of genetic interventions in model organisms, also allowing users to explore the longevity relationships between genes in a visual way.</div></td>
      <td><div class='topic short click_expand'><ul><li>Geriatric medicine</li><li>Genotype and phenotype</li><li>Genetic variation</li><li>Model organisms</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Network visualisation</li><li>Query and retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.synergyage.info/details/wildtype/" target="_blank">http://www.synergyage.info/details/wildtype/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/export_sbw" target="_blank">export_sbw</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Export history data for SynBioWatch projet files and user.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Data acquisition</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>934</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/odoppelt/export_sbw/export_sbw/1.0" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/odoppelt/export_sbw/export_sbw/1.0</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hgnc" target="_blank">Human gene nomenclature database (HGNC)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of unique and meaningful names for all human genes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Human biology</li><li>Ontology and terminology</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>352</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sigflow" target="_blank">Sigflow</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An automated and comprehensive pipeline for cancer genome mutational signature analysis.<br>Sigflow: Streamline Analysis Workflows for Mutational Signatures.<br>Sigflow provides useful mutational signature analysis workflows. It can auto-extract mutational signatures, fit mutation data to all/specified COSMIC reference signatures (SBS/DBS/INDEL) and run bootstrapping analysis for studying signature stability.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Genetic variation</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Information extraction</li><li>Variant calling</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>AFL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mascotdatfile" target="_blank">Mascotdatfile</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A library to fully parse and analyse MASCOT MS/MS search results.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteins</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Calculation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>40</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/mascotdatfile-3-4-6-java-api-msms-search-results-mascot.html" target="_blank">http://www.mybiosoftware.com/mascotdatfile-3-4-6-java-api-msms-search-results-mascot.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/idbases" target="_blank">Immunodeficiency mutation databases (IDbases)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Locus-specific databases for immunodeficiency-causing mutations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence variations : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>96</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ophid" target="_blank">ophid</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Focus on the Integrated Interactions Database (IID).<br>Integrated Interactions Database.<br>tissue specific PPI networks across species.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Pathway or network visualisation</li><li>Protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rawtools" target="_blank">RawTools</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A standalone tool for extracting data directly from raw files generated by Thermo Orbitrap family instruments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : thermo raw</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/kevinkovalchik/RawTools" target="_blank">https://github.com/kevinkovalchik/RawTools</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/spa" target="_blank">SPA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A global overview of single-cell type selectivity and pleiotropy in complex diseases and traits.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Cell biology</li><li>Genetic variation</li><li>Pathology</li><li>GWAS study</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Gene expression QTL analysis</li><li>Expression profile clustering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Hyppo-X" target="_blank">Hyppo-X</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Scalable Exploratory Framework for Analyzing Complex Phenomics Data.<br>A header only software library helps to visually discover the insights of high dimensional complex data set.<br>We can represent the category of points in a cluster using a pie chart. In our Hyppo-X framework, we have capability to define a category of a point using single or multiple columns of the dataset. Pie chart shows the percentage distribution of points among all categories. For instance, for the above sample dataset, we can define the locaion column named Loc as the category of all the points then we will get following graphs. Here, blue color indicates points those belong in location N and red color indicates points those belong in location K. This graph with pie chart node helps us to understand the behavior of points with respect to two locations.<br>This software library is built to delve the insights of a complex high dimensional dataset</div></td>
      <td><div class='topic short click_expand'><ul><li>Mathematics</li><li>Phenomics</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Clustering</li><li>Essential dynamics</li><li>Genetic mapping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aclame" target="_blank">ACLAME</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This database is dedicated to the collection, analysis and classification of sequenced mobile genetic elements (MGEs, in particular phages and plasmids). In addition to providing information on the MGEs content, classifications are available at various levels of organization.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Gene and protein families</li><li>Model organisms</li><li>Mobile genetic elements</li><li>Protein interaction experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence classification</li><li>Query and retrieval</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>171</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/OHNOLOGS_v2" target="_blank">OHNOLOGS v2</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A comprehensive resource for the genes retained from whole genome duplication in vertebrates.<br>The code repository for OHNOLOGS v2.0 sever and multi-genome synteny comparison.<br>A Repository of Genes Retained from Whole Genome Duplications in the Vertebrate Genomes.<br>OHNOLOGS version 2.0 is now live with more genomes and more genome duplications!.<br>Singh P.P. and Isambert H. (2019) OHNOLOGS v2: A comprehensive resource for the genes retained from whole genome duplication in vertebrates. Nucleic Acids Research.<br>Singh PP & Isambert H. (2019) OHNOLOGS v2: A comprehensive resource for the genes retained from whole genome duplication in vertebrates. Nucleic Acids Research. Full text.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Nucleic acids</li><li>Mapping</li><li>Sequence assembly</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence assembly</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/param-p-singh/Ohnologs-v2.0" target="_blank">https://github.com/param-p-singh/Ohnologs-v2.0</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/liputils" target="_blank">liputils</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Python module to manage individual fatty acid moieties from complex lipids.<br>A small Python package to manipulate complex lipids.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Lipids</li><li>Metabolomics</li><li>Endocrinology and metabolism</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Parsing</li><li>Essential dynamics</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://pypi.org/project/liputils/" target="_blank">https://pypi.org/project/liputils/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/oncoscore" target="_blank">OncoScore</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool to measure the association of genes to cancer based on citation frequencies in biomedical literature. The score is evaluated from PubMed literature by dynamically updatable web queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li><li>Literature and language</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Relation extraction</li><li>Database search</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/danro9685/OncoScore" target="_blank">https://github.com/danro9685/OncoScore</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/drugevar" target="_blank">DruGeVar</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database triangulating between drugs, genes and pharmacogenomics biomarkers in regulatory agencies-approved drugs</div></td>
      <td><div class='topic short click_expand'><ul><li>Pharmacology</li><li>Pharmacogenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Query and retrieval</li><li>Visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>16</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sapling" target="_blank">Sapling</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>SAPLING: Suffix Array Piecewise Linear INdex for Genomics.<br>A method for achieving faster suffix array queries compared to binary search.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Sequence analysis</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Read mapping</li><li>Database search</li><li>Genome indexing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/meccog" target="_blank">MecCog</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A knowledge representation framework for genetic disease mechanism.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Molecular interactions, pathways and networks</li><li>Machine learning</li><li>Pathology</li><li>Ontology and terminology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Sequence assembly</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mlibrary" target="_blank">MLibrary</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database search engine developed to assist the user in the detection and identification of small compounds, such as anabolic androgenic steroids, using the soft ionisation technique Matrix Assisted Laser Desorption/Ionisation (MALDI) coupled with mass spectrometry. It compares the results of the experimental data with theoretic values included in the database. It simplifies the interpretation of the mass spectrometry data and reduces the amount of time required to analyze the results manually.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein identification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/celavi" target="_blank">CeLaVi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CeLaVi (Cell Lineage Visualisation tool) is a web-based visualisation tool that allows to use data derived from multiple technologies to interrogate the cell lineage whilst visualising the spatial context, the identity and the properties of the cells.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Imaging</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ksugar/celavi-tools" target="_blank">https://github.com/ksugar/celavi-tools</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/LegumeMine" target="_blank">LegumeMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Multi-organism mine integrates data from legume species: string bean, soybean, peanut and barrel medic</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dvd" target="_blank">DVD</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Diagnostic Variant Database (DVD) is a nation-wide human genomic variant database launched by Dutch University Medical Centers. Its purpose is to share all genetic variants detected in the course of next-generation sequencing (NGS) applications between collaborators. It helps medical researchers to distinguish between functionally relevant and benign genetic variants. DVD also allows users to contact the researcher who originally identified a particular genetic mutation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Rare diseases</li><li>Data integration and warehousing</li><li>Genetic variation</li><li>Human genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Aggregation</li><li>Genetic variation analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/flowand" target="_blank">FlowAnd</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>FlowAnd is designed to analyze and integrate largescale, multi-color flow cytometry data. It implements methods for data importing, various transformations, several clustering algorithms for automatic clustering, visualization tools as well as straightforward statistical testing.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Statistical testing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/flowand-1-0-analyze-largescale-multi-color-flow-cytometry-data.html" target="_blank">http://www.mybiosoftware.com/flowand-1-0-analyze-largescale-multi-color-flow-cytometry-data.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dataanalysis2tpp" target="_blank">DataAnalysis2TPP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Converts MGF from Bruker DataAnalysis to TPP-friendly format for use with XPRESS and ASAPRatio.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : mgf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : mgf</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/covidmine" target="_blank">CovidMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CovidMine is an InterMine dedicated to COVID-19 / SARS-CoV-2 data which<br>integrates confirmed Covid-19 cases, deaths, Sars-CoV-2 reference genome<br>and nucleotide sequences from isolates deposited in Genbank</div></td>
      <td><div class='topic short click_expand'><ul><li>Data integration and warehousing</li><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>155</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org/" target="_blank">https://registry.intermine.org/</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/kusakidb" target="_blank">KusakiDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A novel approach for validation and completeness of protein orthologous groups.<br>A novel approach for validation and enrichment of protein orthologous groups.<br>KusakiDB is a database of protein orthologous groups (OGs) that:.</div></td>
      <td><div class='topic short click_expand'><ul><li>Plant biology</li><li>Proteins</li><li>Agricultural science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Homology-based gene prediction</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/aghelfi/kusakiDB" target="_blank">https://github.com/aghelfi/kusakiDB</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ncbi_gene" target="_blank">Database of genes from NCBI RefSeq genomes</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of genes from NCBI RefSeq genomes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Gene and protein families</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>237</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bisa" target="_blank">BiSA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Binding sites analyser is a database driven software and stores all of its data in a relational database management system (RDBMS) such as SQL Server in Windows or PostgreSQL on Linux.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA binding sites</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/bisa-genomic-regions-binding-sites-analysis.html" target="_blank">http://www.mybiosoftware.com/bisa-genomic-regions-binding-sites-analysis.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/amazonforest" target="_blank">AmazonForest</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A web tool to query data for a set of 5k variants that were predicted with high pathogenic probability</div></td>
      <td><div class='topic short click_expand'><ul><li>Exome sequencing</li><li>Genotype and phenotype</li><li>Statistics and probability</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Virtual PCR</li><li>Virulence prediction</li><li>Variant classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biotextquest" target="_blank">BioTextQuest+</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Implements an enhanced version of the TextQuest algorithm, providing a user friendly web-based interface. BTQ collects abstracts from Medline literature and OMIM databases matching a user query. Identification of relevant terms enables the representation of text records in a Vector Space Model and the calculation of pairwise document similarities. Employing suitable clustering algorithms, results are transformed into clusters of records along with their corresponding terms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Computational biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Document clustering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>12</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/peskaas" target="_blank">PeskAAS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A near-real-time, open-source monitoring and analytics system for small-scale fisheries.<br>PESKAAS: fisheries automated analytics system.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Database management</li><li>Workflows</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/shaunpwilkinson/PeskAAS" target="_blank">https://github.com/shaunpwilkinson/PeskAAS</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/crop-network" target="_blank">CroP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Coordinated Panel Visualization for Biological Networks Analysis | CroP – Coordinated Panels Visualization | The Computational Design and Visualization Lab focuses on multidisciplinary research bringing together Computer Science, Graphic Design and New Media Art | CroP is a visualization tool aimed at highlighting patterns of data in networks integrated with temporal data and analyzing their behaviors over time through dynamic visualization models in a coordinated multiple views layout</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Genomicus-fungi" target="_blank">Genomicus-fungi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomicus-fungi is a genome browser that enables users to navigate in fungi genomes in several dimensions: linearly along chromosome axes, transversaly across different species, and chronologicaly along evolutionary time.</div></td>
      <td><div class='topic short click_expand'><ul><li>Paleogenomics</li><li>Gene and protein families</li><li>Phylogenomics</li><li>Comparative genomics</li><li>Evolutionary biology</li><li>Genomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Genome visualisation</li><li>Phylogenetic tree visualisation</li><li>Dot plot plotting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>386</div></td>
      <td><div class="link short click_expand"><a href="https://www.genomicus.biologie.ens.fr/genomicus-plants" target="_blank">https://www.genomicus.biologie.ens.fr/genomicus-plants</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/peptideatlas" target="_blank">PeptideAtlas peptide database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A multi-organism compendium of peptides identified in a large set of tandem mass spectrometry proteomics experiments. Mass spectrometer output files are collected for human, mouse, yeast, and several other organisms, and searched using the latest search engines and protein sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>peptide identification : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>514</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/PeanutMine" target="_blank">PeanutMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A mine with peanut data from Legume Information Systems (LIS) PeanutBase</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/das-tmfilter" target="_blank">DAS-TMfilter</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DAS (Dense Alignment Surface) is based on low-stringency dot-plots of the query sequence against a set of library sequences - non-homologous membrane proteins - using a previously derived, special scoring matrix. The method provides a high precision hyrdophobicity profile for the query from which the location of the potential transmembrane segments can be obtained. The novelty of the DAS-TMfilter algorithm is a second prediction cycle to predict TM segments in the sequences of the TM-library.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein secondary structure</li><li>Proteomics</li><li>Protein modifications</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein subcellular localisation prediction</li><li>Prediction and recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>124</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/DatView" target="_blank">DatView</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A graphical user interface for visualizing and querying large data sets in serial femtosecond crystallography.<br>DatView displays plots of tabular data and syncs selections across plots. It was initially developed for serial crystallography data analysis with CrystFEL output files and includes a frame viewer for displaying 1D and 2D data from HDF5/CXI files.</div></td>
      <td><div class='topic short click_expand'><ul><li>X-ray diffraction</li><li>RNA immunoprecipitation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Visualisation</li><li>Aggregation</li><li>Splitting</li><li>Sorting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sesaw" target="_blank">SeSAW</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Identifying functionally or evolutionarily conserved motifs in protein structures by locating sequence and structural similarities, and quantifying these at the level of individual residues. Results can be visualized in 2D, as annotated alignments, or in 3D, as structural superpositions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Functional mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>10</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/sesaw-functional-mapping-sequence-derived-structural-alignment-weights.html" target="_blank">http://www.mybiosoftware.com/sesaw-functional-mapping-sequence-derived-structural-alignment-weights.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/infobase" target="_blank">infobase</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Return information on a given nucleotide base.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/neurotoxkb" target="_blank">NeurotoxKb</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Compilation, curation and exploration of a knowledgebase of environmental neurotoxicants specific to mammals.<br>Neurotoxicants are an important class of hazardous chemicals.</div></td>
      <td><div class='topic short click_expand'><ul><li>Toxicology</li><li>Neurology</li><li>Cheminformatics</li><li>Model organisms</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bat_detective" target="_blank">Bat Detective</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Passive acoustic sensing has emerged as a powerful tool for quantifying anthropogenic impacts on biodiversity, especially for echolocating bat species. This tool helps better assess bat population trends accurately and allow the detection and classification of bat calls in large collections of audio recordings.</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Machine learning</li><li>Ecology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>52</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ms-data-core-api" target="_blank">ms-data-core-api</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The ms-data-core-api is a free, open-source library for developing computational proteomics tools and pipelines. The Application Programming Interface, written in Java, enables rapid tool creation by providing a robust, pluggable programming interface and common data model. The data model is based on controlled vocabularies/ontologies and captures the whole range of data types included in common proteomics experimental workflows, going from spectra to peptide/protein identifications to quantitative results. The library contains readers for three of the most used Proteomics Standards Initiative standard file formats: mzML, mzIdentML, and mzTab. In addition to mzML, it also supports other common mass spectra data formats: dta, ms2, mgf, pkl, apl (text-based), mzXML and mzData (XML-based). Also, it can be used to read PRIDE XML, the original format used by the PRIDE database, one of the world-leading proteomics resources.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Software engineering</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : mzidentml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry data : mztab</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>26</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/ms-data-core-api-metadata-oriented-library-for-computational-proteomics.html" target="_blank">http://www.mybiosoftware.com/ms-data-core-api-metadata-oriented-library-for-computational-proteomics.html</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/evidence_finder" target="_blank">Evidence Finder</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Search tool under development in the UKPMC project. It locates sentences that present evidence in the text of research papers. Sentences from 1.5M papers in the UPMC corpus are indexed based on linguistic analysis and NER. Initial search results take the form of questions representing the most frequent types of relevant evidence in the index. On selecting a question, the user may review details of the documents containing the evidence, including the indexed sentences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data mining</li><li>Literature and language</li><li>Database management</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Text mining</li><li>Deposition</li><li>Enrichment analysis</li><li>Literature search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>63</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/threaderai" target="_blank">ThreaderAI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Template-based prediction of protein structure with deep learning.<br>We propose a new template-based modelling method called ThreaderAI to improve protein tertiary structure prediction. ThreaderAI formulates the task of aligning query sequence with template as the classical pixel classification problem in computer vision and naturally applies deep residual neural network in prediction. ThreaderAI first employs deep learning to predict residue-residue aligning probability matrix by integrating sequence profile, predicted sequential structural features, and predicted residue-residue contacts, and then builds template-query alignment by applying a dynamic programming algorithm on the probability matrix.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Structure prediction</li><li>Protein structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein secondary structure prediction</li><li>Protein modelling</li><li>Protein threading</li><li>Ab initio structure prediction</li><li>Network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dbnp" target="_blank">Nutritional Phenotype Database (dbnp)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An open source application suite to store biological studies. The central module (GSCF) stores the study design, which links out to modules that contain the measurements of specific measurement platforms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Nutritional science</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>56</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/skinsensdb" target="_blank">SkinSensDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Curated database for skin sensitization assays.</div></td>
      <td><div class='topic short click_expand'><ul><li>Physiology</li><li>Cheminformatics</li><li>Synthetic chemistry</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/prot2hg" target="_blank">Prot2HG</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Prot2HG is a database of protein domains mapped to the human genome.<br>Genetic variation occurring within conserved functional protein domains warrants special attention when examining DNA variation in the context of disease causation. Here we introduce a resource, freely available at www.prot2hg.com, that addresses the question of whether a particular variant falls onto an annotated protein domain and directly translates chromosomal coordinates onto protein residues. The tool can perform a multiple-site query in a simple way, and the whole dataset is available for download as well as incorporated into our own accessible pipeline. To create this resource, National Center for Biotechnology Information protein data were retrieved using the Entrez Programming Utilities. After processing all human protein domains, residue positions were reverse translated and mapped to the reference genome hg19 and stored in a MySQL database</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein expression</li><li>Mapping</li><li>Exome sequencing</li><li>Genomics</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant prioritisation</li><li>Variant calling</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hovergen" target="_blank">Homologous vertebrate genes database (HOVERGEN)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This database allows selection of sets of homologous genes among vertebrate species, and visualisation of multiple alignments and phylogenetic trees. It is particularly useful for comparative sequence analysis, phylogeny and molecular evolution studies or more generally, for an overall view of what is known about a peculiar gene family.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Gene and protein families</li><li>Phylogenomics</li><li>Comparative genomics</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>phylogenetic tree : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>121</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cascollect" target="_blank">CasCollect</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CasCollect is a pipeline for the detection of Cas genes and CRISPR arrays from unassembled raw illumina sequencing reads.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li><li>Gene structure</li><li>Workflows</li><li>Metagenomics</li><li>Sequence assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence merging</li><li>Sequence assembly</li><li>Sequence trimming</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Script</li><li>Workflow</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-2-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GenBankFlatFileReader" target="_blank">GenBank Flat File Reader</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Read GenBank-formatted flat file into structure</div></td>
      <td><div class='topic short click_expand'><ul><li>Biomedical science</li><li>Genomics</li><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metaQuantome" target="_blank">metaQuantome</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>MetaQuantome software suite analyzes the state of a microbiome by leveraging complex taxonomic and functional hierarchies to summarize peptide-level quantitative information. metaQuantome offers differential abundance analysis, principal components analysis, and clustered heat map visualizations, as well as exploratory analysis for a single sample or experimental condition.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li><li>Proteomics</li><li>Microbial ecology</li><li>Metagenomics</li><li>Metatranscriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Differential protein expression analysis</li><li>Indexing</li><li>Filtering</li><li>Visualisation</li><li>Principal component visualisation</li><li>Query and retrieval</li><li>Quantification</li><li>Heat map generation</li><li>Functional clustering</li><li>Statistical inference</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>ec number : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>taxonomy : image format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Suite</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sabio-rk" target="_blank">SABIO-RK: biochemical reactions and their kinetics</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Curated database that contains information about biochemical reactions, their kinetic rate equations with parameters and experimental conditions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biomedical science</li><li>Biochemistry</li><li>Enzymes</li><li>Molecular interactions, pathways and networks</li><li>Data mining</li><li>Systems biology</li><li>Biology</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>reaction data : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>140</div></td>
      <td><div class="link short click_expand"><a href="http://sabiork.h-its.org/" target="_blank">http://sabiork.h-its.org/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/swc-batch-check" target="_blank">SWC BATCH CHECK</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Module for SWC neuron morphology file validation and correction enabled for high throughput batch processing.<br>Batch validation of directory containing SWC files.<br>Ensures structures are correctly connected.<br>Fix structures with zero size diameter.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Neurite measurement</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/epivizrdata" target="_blank">epivizrData</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Serve data from BioConductor Objects through a WebSocket connection.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td><div class="link short click_expand"><a href="http://epiviz.github.io" target="_blank">http://epiviz.github.io</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mzXML2Search" target="_blank">mzXML2Search</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Converts mzXML and mzML files to SEQUEST dta, MASCOT mgf, and Micromass pkl files. Comes with the current version of TPP.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : mzxml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : dta</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gpcr-pendb" target="_blank">GPCR-PEnDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of protein sequences and derived features to facilitate prediction and classification of G protein-coupled receptors.<br>To analyze the sequence either upload a file with the sequences in FASTA format, or copy and paste the sequence(s) inside the text area below.</div></td>
      <td><div class='topic short click_expand'><ul><li>Physiology</li><li>Protein sites, features and motifs</li><li>Gene and protein families</li><li>Membrane and lipoproteins</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Query and retrieval</li><li>Sequence classification</li><li>Sequence feature detection</li><li>Transmembrane protein prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pga" target="_blank">PGA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package provides functions for construction of customized protein databases based on RNA-Seq data with/without genome guided, database searching, post-processing and report generation. This kind of customized protein database includes both the reference database (such as Refseq or ENSEMBL) and the novel peptide sequences form RNA-Seq data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data governance</li><li>Proteomics experiment</li><li>Proteomics</li><li>RNA-seq</li><li>Data integration and warehousing</li><li>Database management</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>RNA-Seq analysis</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence alignment (nucleic acid) : gtf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein sequence record : fasta</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>26</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/selectbesthit" target="_blank">selectbesthit</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Select the best hit for each query sequence in a Blast tabular file.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : tsv</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : tsv</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>948</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/afelten/microbiome_analyses/SelectBestHit/0.1%20beta" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/afelten/microbiome_analyses/SelectBestHit/0.1%20beta</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bsnet" target="_blank">BSNet</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Bi-Similarity Network (BSNet) consists of a single embedding module and a bi-similarity module of two similarity measures. After the support images and the query images pass through the convolution-based embedding module, the bi-similarity module learns feature maps according to two similarity measures of diverse characteristics.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Machine learning</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/eccdia" target="_blank">ECCDIA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Esophageal Cancer Clinical Data Interactive Analysis (ECCDIA) is an interactive web tool for the comprehensive analysis of clinical and survival data of esophageal cancer patients.</div></td>
      <td><div class='topic short click_expand'><ul><li>Surgery</li><li>Public health and epidemiology</li><li>Oncology</li><li>Statistics and probability</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Mapping</li><li>Incident curve plotting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/clinico-omics/ECCDIA" target="_blank">https://github.com/clinico-omics/ECCDIA</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/repairtoire" target="_blank">REPAIRtoire</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>REPAIRtoire is a database resource for systems biology of DNA damage and repair. The database collects and organizes the following types of information: (i) DNA damage linked to environmental mutagenic and cytotoxic agents, (ii) pathways comprising individual processes and enzymatic reactions involved in the removal of damage, (iii) proteins participating in DNA repair and (iv) diseases correlated with mutations in genes encoding DNA repair proteins.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li><li>Molecular interactions, pathways and networks</li><li>DNA</li><li>Pathology</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Query and retrieval</li><li>Protein modelling (mutation)</li><li>Protein modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>38</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ProtMiscuity" target="_blank">ProtMiscuity</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of promiscuous proteins.<br>Promiscuous behaviour in proteins and enzymes remains a challenging feature to understand the structure-function relationship. Here we present ProtMiscuity, a manually curated online database of proteins showing catalytic promiscuity. ProtMiscuity contains information about canonical and promiscuous activities comprising 88 different reactions in 57 proteins from 40 different organisms. It can be searched or browsed by protein names, organisms and descriptions of canonical and promiscuous reactions. Entries provide information on reaction substrates, products and kinetic parameters, mapping of active sites to sequence and structure and links to external resources with biological and functional annotations.</div></td>
      <td><div class='topic short click_expand'><ul><li>Enzymes</li><li>Protein binding sites</li><li>Literature and language</li><li>Database management</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Database search</li><li>Active site prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Genomicus-tunicates" target="_blank">Genomicus-tunicates</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomicus-tunicates is a genome browser that enables users to navigate in tunicates genomes in several dimensions: linearly along chromosome axes, transversaly across different species, and chronologicaly along evolutionary time. It uses ANISEED (https://www.aniseed.cnrs.fr) database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Paleogenomics</li><li>Gene and protein families</li><li>Phylogenomics</li><li>Comparative genomics</li><li>Evolutionary biology</li><li>Genomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Genome visualisation</li><li>Phylogenetic tree visualisation</li><li>Dot plot plotting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>45</div></td>
      <td><div class="link short click_expand"><a href="https://www.genomicus.biologie.ens.fr/genomicus" target="_blank">https://www.genomicus.biologie.ens.fr/genomicus</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nmrstarlib" target="_blank">nmrstarlib</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Ibrary for reading, writing, and accessing BMRB entries in NMR-STAR and JSONized NMR-STAR formats.</div></td>
      <td><div class='topic short click_expand'><ul><li>NMR</li><li>Structural biology</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dbd2bs" target="_blank">DBD2BS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Uses the structure of DNA binding domains (DBDs) to predict the corresponding binding sites on DNA. Queries can be specified by a PDB ID or the coordinates of a DNA-binding protein in PDB format.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>Nucleic acid structure analysis</li><li>DNA binding sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Nucleic acid feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/dbd2bs-connecting-dna-binding-protein-binding-sites.html" target="_blank">http://www.mybiosoftware.com/dbd2bs-connecting-dna-binding-protein-binding-sites.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dsm" target="_blank">DSM</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Software of content-based exploration and retrieval method for whole metagenome sequencing samples.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metagenomics</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>17</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/dsm-20140113-distributed-string-mining-framework.html" target="_blank">http://www.mybiosoftware.com/dsm-20140113-distributed-string-mining-framework.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sumrep" target="_blank">sumrep</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Sumrep is a package for summarizing, visualizing, and comparing immune receptor repertoires. It contains dozens of different summary statistics to be applied on various levels of assumptions, from raw query reads to inferred phylogenies, as well as easy and principled ways to plot and compare repertoires. It currently supports the IgH, IgL, IgK, TRA, and TRB loci.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li><li>Statistics and probability</li><li>Immunoproteins and antigens</li><li>Phylogeny</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Filtering</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>4</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/spidermir" target="_blank">SpidermiR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Aims: i) Facilitate the network open-access data retrieval from GeneMania data, ii) prepare the data with appropriate gene nomenclature, iii) integration of miRNA data in a specific network, iv) provide different standard analyses and v) allow the user to visualize the results. In more detail, the package provides multiple methods for query, prepare and download network data, and the integration with validated and predicted miRNA data, the use of standard analysis and visualization methods.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li><li>Gene regulation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network analysis</li><li>Gene regulatory network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>32</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/claudiacava/SpidermiR" target="_blank">https://github.com/claudiacava/SpidermiR</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/covid-19_knetminer" target="_blank">COVID-19 KnetMiner</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Visualises linked human biological data related to SARS-CoV-2 in the form of a knowledge graph, using a range of publicly available data. Includes gene, protein, gene ontology, reaction, pathway, homology, protein domain, phenotype/disease, and GWAS data and the interactions between them. KnetMiner scores and ranks genes to give the most relevant results relating to your query search.</div></td>
      <td><div class='topic short click_expand'><ul><li>Drug discovery</li><li>Data visualisation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/Rothamsted/covid19-kg/blob/master/RawDataEndPoints.md" target="_blank">https://github.com/Rothamsted/covid19-kg/blob/master/RawDataEndPoints.md</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/soyfn" target="_blank">SoyFN</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Knowledge database of Soybean Functional gene Networks and miRNA functional networks.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Microarray experiment</li><li>Functional, regulatory and non-coding RNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>miRNA expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>13</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ShuTu" target="_blank">ShuTu</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ShuTu is a software platform for semi-automated reconstruction of neuronal morphology. It is designed for neurons stained following patch-clamp recording and biocytin filling/staining.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>Transcription factors and regulatory sites</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Quantification</li><li>Parsing</li><li>Neurite measurement</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://www.janelia.org/shutu" target="_blank">https://www.janelia.org/shutu</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/jiffynet" target="_blank">JiffyNet</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Specifically designed to instantly construct genome-scale protein networks based on associalogs (functional associations transferred from a template network by orthology) for a query species with only protein sequences provided.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Network simulation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>24</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/jiffynet-jiffy-instant-gene-network-modeler-for-newly-sequenced-species.html" target="_blank">http://www.mybiosoftware.com/jiffynet-jiffy-instant-gene-network-modeler-for-newly-sequenced-species.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gpcr-ca" target="_blank">GPCR-CA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>2-layer predictor: the 1st layer prediction engine is for identifying a query protein sequence as GPCR on non-GPCR; if it is a GPCR protein, the process will be automatically continued with the 2nd-layer prediction engine to further identify its type among the following six main functional classes: (1) rhodopsin-like, (2) secretin-like, (3) metabotrophic/glutamate/pheromone, (4) fungal pheromone, (5) cAMP receptor, and (6) Frizzled/Smoothemed family</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Structure prediction</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Prediction and recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>189</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/gpcr-ca-predicting-gpcr-classification.html" target="_blank">http://www.mybiosoftware.com/gpcr-ca-predicting-gpcr-classification.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GenoSurf" target="_blank">GenoSurf</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Metadata driven semantic search system for integrated genomic datasets.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li><li>Ontology and terminology</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Database search</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="http://www.bioinformatics.deib.polimi.it/genosurf/dump/" target="_blank">http://www.bioinformatics.deib.polimi.it/genosurf/dump/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pdinfobuilder" target="_blank">pdInfoBuilder</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Builds platform design information packages. These consist of a SQLite database containing feature-level data such as x, y position on chip and featureSet ID. The database also incorporates featureSet-level annotation data. The products of this packages are used by the oligo pkg.</div></td>
      <td><div class='topic short click_expand'><ul><li>Microarray experiment</li><li>DNA polymorphism</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>raw microarray data : cel</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>processed microarray data : sqlite format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>725</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/regate" target="_blank">ReGaTE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Command line utility that automates the registration of the tools installed on any given Galaxy portal in ELIXIR bio.tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cryptodb" target="_blank">Cryptosporidium genome resources CryptoDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomic-scale dataset associated with the eukaryotic pathogens Cryptosporidium.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>105</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/experimenthubdata" target="_blank">ExperimentHubData</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Functions to add metadata to ExperimentHub db and resource files to AWS S3 buckets.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/xmlget" target="_blank">xmlget</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An Emboss tool that fetches XML entry from a file or a local or remote data resource, parses it and rewrites it in various formats</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Bioinformatics</li><li>Computer science</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Format detection</li><li>Data retrieval</li><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database metadata : xml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>database metadata : xml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/metabolic_pathway_prediction" target="_blank">metabolic pathway prediction</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A deep learning architecture for metabolic pathway prediction.<br>This code is an implementation of our graph-convolutional network (GCN) and random forest (RF) classifiers for predicting one or many metabolic pathway classes for a query compound (paper accepted at Bioinformatics, Oxford University Press).<br>Baranwal, Mayank, Abram Magner, Paolo Elvati, Jacob Saldinger, Angela Violi, and Alfred Hero. "A deep learning architecture for metabolic pathway prediction." Bioinformatics (2019).</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Machine learning</li><li>Endocrinology and metabolism</li><li>Medicinal chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Expression profile pathway mapping</li><li>Pathway or network prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>7</div></td>
      <td></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tigrfams" target="_blank">TIGRFAMS  protein family database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Protein families based on hidden Markov models (HMMs).</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene and protein families</li><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence profile : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/webgester_db" target="_blank">WebGeSTer DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of intrinsic transcription terminators identified in bacterial genome sequences and plasmids. Users can obtain both graphic and tabular results on putative terminators based on default or user-defined parameters. The results are arranged in different tiers to facilitate retrieval, as per the specific requirements. An interactive map has been incorporated to visualize the distribution of terminators across the whole genome.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Transcription factors and regulatory sites</li><li>Molecular interactions, pathways and networks</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Mapping</li><li>Query and retrieval</li><li>Data retrieval</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>61</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GET-IT" target="_blank">GET-IT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Exploiting observations and measurement data standard for distributed LTER-Italy freshwater sites. Water quality issues.<br>Geoinformation Enabling ToolkIT starterkit.<br>Geoinformation Enabling ToolkIT starterkit &REG;.<br>GET-IT allows you to easily share geospatial data on the web with simple actions.<br>Share your maps, measurements, and sensors.</div></td>
      <td><div class='topic short click_expand'><ul><li>Laboratory techniques</li><li>Database management</li><li>Ecology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/openphacts-vis-compoundinfo" target="_blank">openphacts-vis-compoundinfo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Displays the information available in the Open PHACTS Linked Data cache about a compound.</div></td>
      <td><div class='topic short click_expand'><ul><li>Drug discovery</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/diark" target="_blank">diArk - a resource for eukaryotic genome research</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database driven web application that is designed to store, organize, and present the most relevant information about completed genome projects and EST/cDNA data from eukaryotes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Model organisms</li><li>Sequence assembly</li><li>Database management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>EST assembly</li><li>Sequence assembly</li><li>Genome assembly</li><li>De-novo assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/huge" target="_blank">HUGE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of the human cDNA project at the Kazusa DNA Research Institute, covering various sets of large human proteins of hitherto unidentified functions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Human biology</li><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/eds" target="_blank">Electron density server at Uppsala university</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A service for evaluating the electron density (and, indirectly, some aspects of the model quality) of crystal structures deposited in the Protein Data Bank. In addition to electron density maps and tools to view these maps, statistics and plots are provided for assessing the goodness-of-fit between the deposited models (structures) and the experimental data represented by the electron density calculated using the models and their associated structure factor amplitudes.</div></td>
      <td><div class='topic short click_expand'><ul><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>304</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/HIV-DRLink" target="_blank">HIV-DRLink</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A tool for detecting linked HIV-1 drug resistance mutations in next generation sequencing data.<br>A script detect and calculate Linked HIV drug resistant mutations on single HIV genomes.<br>HIV-DRLink_github.pl is a script that detects and calculates Linked HIV drug resistant mutations on single HIV genomes.<br>The HIV Databases cover HIV sequences, HIV molecular immunology, HIV drug resistance mutations and nonhuman primate HIV SIV vaccine trials.<br>The HIV databases contain comprehensive data on HIV genetic sequences and immunological epitopes</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li><li>Sequencing</li><li>Genetic variation</li><li>Preclinical and clinical studies</li><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Polymorphism detection</li><li>Database search</li><li>Antimicrobial resistance prediction</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://hivdb.stanford.edu/page/webservice/" target="_blank">https://hivdb.stanford.edu/page/webservice/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gcsa2" target="_blank">gcsa2</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Haplotype-aware graph indexes | This is a reimplementation of the Generalized Compressed Suffix Array (GCSA), a BWT-based index for directed graphs | The old implementation indexed all paths in a directed acyclic graph, which had to be determinized before index construction. GCSA2 approximates the graph with a de Bruijn graph (with k = 256 or less), allowing it to index more complex graphs as well as graphs containing cycles. The order of the de Bruijn graph limits the length of the queries the index is able to answer correctly. Longer queries may result in false positives (but no false negatives)</div></td>
      <td><div class='topic short click_expand'><ul></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Haplotype mapping</li><li>Genome indexing</li><li>De-novo assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>8</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/vaxign-ml" target="_blank">Vaxign-ML</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Vaxign-ML is a supervised machine learning classification to predict protective antigens. This tool is a supervised machine learning reverse vaccinology model for improved prediction of bacterial protective antigens.<br>Vaxign-ML (Vaxign - Machine Learning) is a machine learning-based vaccine candidate prediction and analysis system based on the principle of reverse vaccinology.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Infectious disease</li><li>Immunoproteins and antigens</li><li>Preclinical and clinical studies</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Virulence prediction</li><li>Peptide immunogenicity prediction</li><li>Epitope mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/VIOLINet/Vaxign-ML-docker" target="_blank">https://github.com/VIOLINet/Vaxign-ML-docker</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rwrmtn" target="_blank">RWRMTN</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A tool for predicting disease-associated microRNAs based on a microRNA-target gene network.<br>The misregulation of microRNA (miRNA) has been shown to cause diseases. Recently, we have proposed a computational method based on a random walk framework on a miRNA-target gene network to predict disease-associated miRNAs. The prediction performance of our method is better than that of some existing state-of-the-art network- and machine learning-based methods since it exploits the mutual regulation between miRNAs and their target genes in the miRNA-target gene interaction networks.<br>To facilitate the use of this method, we have developed a Cytoscape app, named RWRMTN, to predict disease-associated miRNAs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Functional, regulatory and non-coding RNA</li><li>Workflows</li><li>Microarray experiment</li><li>Literature and language</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>miRNA expression analysis</li><li>Visualisation</li><li>Query and retrieval</li><li>miRNA target prediction</li><li>Network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/hauldhut/RWRMTN" target="_blank">https://github.com/hauldhut/RWRMTN</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/crossbar" target="_blank">CROssBAR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CROssBAR is a comprehensive Resource of Biomedical Relations with Deep Learning Applications and Knowledge Graph Representations. The purpose of the CROssBAR project is to address the limitations related to data diversity and connectivity in biological data resources, which hamper their real-world applications to biomedical problems. Within CROssBAR, we developed a comprehensive computational resource by linking various biomedical resources, generating relation predictions using machine/deep learning, and developing information rich knowledge graphs that incorporate available and predicted biomedical relationships with the aim of providing aid to biomedical researchers to further understand disease mechanisms and to discover/develop new drugs.</div></td>
      <td><div class='topic short click_expand'><ul><li>Infectious disease</li><li>Genotype and phenotype</li><li>Molecular interactions, pathways and networks</li><li>Machine learning</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Enrichment analysis</li><li>Protein interaction prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://crossbar.kansil.org/covid_main.php" target="_blank">https://crossbar.kansil.org/covid_main.php</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/revercsp" target="_blank">reverCSP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Time-Travelling in CSP Computations.<br>reverCSP is a tool that allows us to evaluate a CSP specification in both directions: forwards (traditional computation) and backwards (using our reversible computation framework). reverCSP uses internally csp_tracker to produce tracks of the computation so one can better follows what is being computed so far (or henceforth if going backwards).</div></td>
      <td><div class='topic short click_expand'><ul><li>Zoology</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>AGPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/bionode-ncbi" target="_blank">bionode-ncbi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>NCBI database tool to search, download and compare databases from NCBI.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/transmart" target="_blank">tranSMART</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Its knowledge management and high-content analysis platform is a flexible software framework featuring novel research capabilities. It enables analysis of integrated data for the purposes of hypothesis generation, hypothesis validation, and cohort discovery in translational research.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein sequence analysis</li><li>Visualisation</li><li>Gene expression analysis</li><li>Query and retrieval</li><li>Sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Workbench</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Ocean_Gene_Atlas" target="_blank">Ocean Gene Atlas</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Ocean Gene Atlas service provides data mining access to three complementary data objects: gene sequence catalogs (ENA), sample environmental context (PANGAEA), and gene abundances estimates in samples (computed by mapping sequence reads onto gene catalogs).<br>User queries are composed of either a sequence (nucleic or protein), or a hidden Markov model derived from a multiple sequence alignment. Homologs of the user query in the gene catalogs are identified using standard sequence similarity search tools (eg BLAST or HMMER), and their read based estimated abundance are displayed in interactive maps and plots. A phylogenetic tree is also inferred in order to situate the user query within its context of marine environmental homologs as well as known homologs from reference sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Marine biology</li><li>Metagenomics</li><li>Ecology</li><li>Metatranscriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Phylogenetic tree visualisation</li><li>Map drawing</li><li>Quantification</li><li>Sequence alignment</li><li>Phylogenetic tree reconstruction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene id : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>image : pdf</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>26</div></td>
      <td></td>
      <td><div class='short click_expand'>Not licensed</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MLML2R" target="_blank">MLML2R</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R package for maximum likelihood estimation of DNA methylation and hydroxymethylation proportions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenetics</li><li>Microarray experiment</li><li>Methylated DNA immunoprecipitation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/torquemada" target="_blank">ToRQuEMaDA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool for Retrieving Queried Eubacteria, Metadata and Dereplicating Assemblies.<br>TQMD is a tool which downloads, stores and produces lists of dereplicated prokaryotic genomes. It has been developed to counter the ever-growing number of prokaryotic genomes and their uneven taxonomic distribution. It is based on word-based alignment-free methods ( k -mers), an iterative single-linkage approach and a divide-and-conquer strategy to remain both efficient and scalable. We studied the performance of TQMD by verifying the influence of its parameters and heuristics on the clustering outcome. We further compared TQMD to two other dereplication tools (dRep and Assembly-Dereplicator). Our results showed that TQMD is optimized to dereplicate at high taxonomic levels (phylum class), whereas the other dereplication tools are optimized for lower taxonomic levels (species strain), making TQMD complementary to the existing dereplicating tools</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Sequence assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Sequence assembly</li><li>Data retrieval</li><li>k-mer counting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hypercldb" target="_blank">HyperCLDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The hypertextual version of the Cell Line Database (CLDB), which is not directly accessible, a collection of information on human and animal cell lines available from some of the main European cell banks and many Italian research laboratories</div></td>
      <td><div class='topic short click_expand'><ul><li>Data acquisition</li><li>Cell culture collection</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Formatting</li><li>Service invocation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>94</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/3CDB" target="_blank">3CDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Manually curated 3C (Chromosome conformation capture) database.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA packaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>62</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genepublisher" target="_blank">GenePublisher</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Automated data analysis from gene expression experiments on a number of different platforms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression analysis</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>raw microarray data : cel</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>28</div></td>
      <td><div class="link short click_expand"><a href="http://cbs.dtu.dk/services" target="_blank">http://cbs.dtu.dk/services</a></div></td>
      <td><div class='short click_expand'>Other</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Leaf" target="_blank">Leaf</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Open-source, model-agnostic, data-driven web application for cohort discovery and translational biomedical research.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biobank</li><li>Medical informatics</li><li>Medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Phylogenetic tree reconstruction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/uwrit/leaf/issues" target="_blank">https://github.com/uwrit/leaf/issues</a></div></td>
      <td><div class='short click_expand'>MPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SoyMine" target="_blank">SoyMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Soybean mine instance with genomic data from the Legume Information Systems (LIS) multi-species chado database and genetic data pulled from Soybase</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/guildify" target="_blank">GUILDify</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Uses network-topology based prioritization algorithms in GUILD to score relevance of gene products with respect to given keywords. First, BIANA knowledge base containing data integrated from publicly available major data repositories is queried for gene products associated with the keywords. These gene products are fed to a species-specific interaction network as seed proteins. Finally, a score of relevance for each gene product in the network is calculated by the prioritization algorithm.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>ontology concept name : uniprotkb format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>pathway or network : sif</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>95</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pubmeth" target="_blank">PubMeth reviewed methylation database in cancer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An annotated and reviewed database of methylation in cancer. It is based on automated textmining of literature and is afterwards manually curated and annotated.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>129</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/simtext" target="_blank">SimText</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A text mining framework for interactive analysis and visualization of similarities among biomedical entities.<br>For each search query, PMIDs or abstracts from PubMed are saved.<br>$ git clone https://github.com/dlal-group/simtext.<br>For all PMIDs in each row of a table the according abstracts are saved in additional columns.</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Machine learning</li><li>Natural language processing</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Text annotation</li><li>Text mining</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://simtext.shinyapps.io/genes" target="_blank">https://simtext.shinyapps.io/genes</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/whatsgnu" target="_blank">WhatsGNU</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A tool for identifying proteomic novelty.<br>WhatsGNU utilizes the natural variation in public databases to rank protein sequences based on the number of observed exact protein matches (the GNU score) in all known genomes of a certain species & can quickly create whole protein reports.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Sequence analysis</li><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Multilocus sequence typing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/zincpharmer" target="_blank">ZINCPharmer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ZINCPharmer is webserver for screening the purchasable subset of the ZINC database (updates occur monthly). ZINCPharmer can import LigandScout and MOE pharmacophore definitions, as well as identify pharmacophore features directly from a molecular structure.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Protein structural motifs and surfaces</li><li>Database management</li><li>Drug discovery</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Protein-ligand docking</li><li>Protein structural motif recognition</li><li>Structural similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>160</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/wge" target="_blank">WGE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A CRISPR database for genome engineering.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular genetics</li><li>Genomics</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>88</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/compareoverlappingsmallquery" target="_blank">compareoverlappingsmallquery</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provide the queries that overlap with a reference, when the query is small.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence feature comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : gtf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : gff3</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>937</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/urgi_tools/CompareOverlappingSmallQuery/1.0.0" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/urgi_tools/CompareOverlappingSmallQuery/1.0.0</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/annotationforge" target="_blank">AnnotationForge</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides code for generating Annotation packages and their databases. Packages produced are intended to be used with AnnotationDbi.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data integration and warehousing</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>ontology data : ontology format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>nucleic acid features : sqlite format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hic-hiker" target="_blank">HiC-Hiker</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A probabilistic model to determine contig orientation in chromosome-length scaffolds with Hi-C.<br>This will install the python package (and its cli command) hic_hiker.</div></td>
      <td><div class='topic short click_expand'><ul><li>DNA</li><li>Workflows</li><li>Mapping</li><li>Sequence assembly</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Parsing</li><li>Scaffolding</li><li>Genome assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/FASTXReader" target="_blank">FASTX::Reader</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A Perl module to parse FASTA/FASTQ files</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence : fasta</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/telatin/FASTQ-Parser" target="_blank">https://github.com/telatin/FASTQ-Parser</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/HmtVar" target="_blank">HmtVar</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A data and text mining pipeline to annotate human mitochondrial variants with functional and clinical information.<br>The main web resource to explore human mitochondrial variability data and their pathological correlation.<br>HmtVar is a manually-curated database offering variability and pathogenicity information about mtDNA variants.<br>Data are gathered from HmtVar's twin database HmtDB, and further integrated with pathogenicity predictions as well as additional information from several online resources focused on mtDNA, such as MITOMAP, 1000 Genomes Project, MutPred, SNPs&GO and many others.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Genotype and phenotype</li><li>Workflows</li><li>Pathology</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Virulence prediction</li><li>Text annotation</li><li>Query and retrieval</li><li>Text mining</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hs-blastn" target="_blank">HS-BLASTN</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Nucleotide-Nucleotide aligner that searches nucleotide queries against a large nucleotide database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence assembly</li><li>Sequence analysis</li><li>DNA</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence alignment</li><li>Sequence similarity search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>69</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/hs-blastn-accelerated-megablast-search-tool.html" target="_blank">http://www.mybiosoftware.com/hs-blastn-accelerated-megablast-search-tool.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/readw" target="_blank">ReAdW</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Converts Xcalibur native acquisition files to mzXML.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Formatting</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : thermo raw</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>mass spectrometry spectra : mzxml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://ms-utils.org" target="_blank">http://ms-utils.org</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/omero-lib" target="_blank">OMERO-lib</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Efficient data management infrastructure for the integration of imaging and omics data in life science research.<br>Abstract As technical developments in omics and biomedical imaging drive the increase in quality, modality, and throughput of data generation in life sciences, the need for information systems capable of integrative, long-term storage and management of these heterogeneous digital assets is also increasing. Here, we propose an approach based on principles of Service Oriented Architecture design, to allow the integrated management and analysis of multi-omics and biomedical imaging data. The proposed architecture introduces an interoperable image management system, the OMERO server, into the backend of qPortal, a FAIR-compliant web-based platform for omics data management.</div></td>
      <td><div class='topic short click_expand'><ul><li>Medical imaging</li><li>Data management</li><li>Plant biology</li><li>Light microscopy</li><li>Tomography</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li><li>Query and retrieval</li><li>Service management</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/wgcna" target="_blank">WGCNA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Functions necessary to perform Weighted Correlation Network Analysis on high-dimensional data. Includes functions for rudimentary data cleaning, construction of correlation networks, module identification, summarization, and relating of variables and modules to sample traits. Also includes a number of utility functions for data manipulation and visualization.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression correlation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>5,762</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/shifty" target="_blank">Shifty</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>With this tool you can predict 1H, 13C, and 15N chemical shifts for your favourite protein using only its amino acid sequence as input. The technique uses dynamic programming to detect sequence homologies between your query and sequences of hundreds of previously assigned protiens in the BioMagResBank.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>NMR</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein property calculation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>83</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/shifty-1-3-automated-protein-chemical-shift-prediction.html" target="_blank">http://www.mybiosoftware.com/shifty-1-3-automated-protein-chemical-shift-prediction.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/VRTK" target="_blank">VRTK</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Exploring Drugbank in Virtual Reality Chemical Space.<br>VRTK is a toolkit to rapidly build VR solutions.<br>VRTK is a collection of useful, reusable solutions to common problems found when building for virtual reality. VRTK aims to help productivity by speeding up the creation process from prototyping ideas to building complete solutions</div></td>
      <td><div class='topic short click_expand'><ul><li>Biochemistry</li><li>Cheminformatics</li><li>Mathematics</li><li>Computational chemistry</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/annotationhub" target="_blank">AnnotationHub</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package provides a client for the BioConductor corresponding web resource, which provides a central location where genomic files (e.g., VCF, bed, wig) and other resources from standard locations (e.g., UCSC, Ensembl) can be discovered. The resource includes metadata about each resource, e.g., a textual description, tags, and date of modification. The client creates and manages a local cache of files retrieved by the user, helping with quick and reproducible access.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Genomics</li><li>Informatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,245</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/aramemnon" target="_blank">ARAMEMNON database of plant membrane proteins</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Plant membrane proteins.</div></td>
      <td><div class='topic short click_expand'><ul><li>Membrane and lipoproteins</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein features : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>471</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cammiq" target="_blank">CAMMiQ</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CAMMiQ is a software tool for microbial identification and quantification. Specifically, it builds a compact index to manage a database of microbial genomes (in particular, bacterial/viral genomes in NCBI RefSeq database) and answers queries of the following form: given a set of high throughput sequencing (HTS) reads obtained from a mixture of genomes in the database, identify the corresponding genomes (in the database) and compute their relative abundance.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Metagenomics</li><li>Metatranscriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Quantification</li><li>k-mer counting</li><li>Genome indexing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pss-sql" target="_blank">PSS-SQL</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Allows searching a database in order to identify proteins having secondary structure similar to the structure specified by the user in a PSS-SQL query.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteins</li><li>Protein structure analysis</li><li>Structure analysis</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein sequence analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>31</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/pss-sql-protein-secondary-structure-similarity-searching-in-microsoft-sql-server.html" target="_blank">http://www.mybiosoftware.com/pss-sql-protein-secondary-structure-similarity-searching-in-microsoft-sql-server.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/intact" target="_blank">IntAct</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Protein interaction database and analysis system.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein interaction data : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>791</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mlst-db" target="_blank">MLST-DB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Chlamydia trachomatis multilocus sequence typing (MLST) database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>407</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scmontage" target="_blank">scMontage</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Fast and Robust Gene Expression Similarity Search for Massive Single-cell Data.<br>scMontage is a system for searching gene expression databases for similar cells to the query gene expression profile. Please enter your profile in the blank space or try sample queries.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Epigenetics</li><li>Cell biology</li><li>Gene expression</li><li>Microarray experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li><li>Expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cr-est" target="_blank">CR-EST</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Crop EST Database is a public available online resource providing access to sequence, classification, clustering, and annotation data of crop EST projects at the IPK. A view of these information give the summarized numbers about genomic data of species listed in the adjacent table.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence analysis</li><li>Plant biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Sequence annotation</li><li>Sequence clustering</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>27</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Impact_framework" target="_blank">Impact framework</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Python package for writing data analysis workflows to interpret microbial physiology.</div></td>
      <td><div class='topic short click_expand'><ul><li>Physiology</li><li>Bioengineering</li><li>Genotype and phenotype</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/biobb" target="_blank">BioExcel Building Blocks (biobb)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Biobb (BioExcel building blocks) are Python building blocks wrapping popular computational biomolecular tools.</div></td>
      <td><div class='topic short click_expand'><ul><li>Chemistry</li><li>Computational biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Modelling and simulation</li><li>Prediction and recognition</li><li>Query and retrieval</li><li>Calculation</li><li>Structure analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>structure : pdb</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/bioexcel/biobb" target="_blank">https://github.com/bioexcel/biobb</a></div></td>
      <td><div class='short click_expand'>Apache-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dropletutils" target="_blank">DropletUtils</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides a number of utility functions for handling single-cell (RNA-seq) data from droplet technologies such as 10X Genomics. This includes data loading, identification of cells from empty droplets, removal of barcode-swapped pseudo-cells, and downsampling of the count matrix.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-seq</li><li>Sequencing</li><li>Transcriptomics</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/DropletUtils/" target="_blank">http://bioconductor.org/packages/DropletUtils/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/isa_tools" target="_blank">ISA tools</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Framework and tools that help to manage an increasingly diverse set of life science, environmental and biomedical experiments that employing one or a combination of technologies.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data management</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Suite</li></ul></div></td>
      <td><div class='citations short click_expand'>235</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/cblaster" target="_blank">cblaster</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Cblaster is a tool for finding clusters of co-located homologous sequences in BLAST searches. Given a collection of protein sequences, cblaster can search sequence databases remotely (via NCBI BLAST API) or locally (via DIAMOND). Search results are parsed and filtered based on user thresholds for identity, coverage and e-value. The genomic coordinates of remaining hits are obtained from the NCBI's Identical Protein Group (IPG) database (or a local database in local searches).</div></td>
      <td><div class='topic short click_expand'><ul><li>Workflows</li><li>Endocrinology and metabolism</li><li>Drug metabolism</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Deposition</li><li>Database search</li><li>Scaffolding</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/New_Basis_Set_Exchange" target="_blank">New Basis Set Exchange</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An Open, Up-to-Date Resource for the Molecular Sciences Community.<br>A repository for quantum chemistry basis sets.<br>The new Basis Set Exchange (BSE) for Quantum Chemistry.<br>If you are looking for the Basis Set Exchange website (which can be used to browse and download this data in a more user-friendly way), visit https://www.basissetexchange.org.<br>This project is a library containing basis sets for use in quantum chemistry calculations. In addition, this library has functionality for manipulation of basis set data.<br>This library is used to form the backend of the new Basis Set Exchange website.<br>A New Basis Set Exchange: An Open, Up-to-date Resource for the Molecular Sciences Community Benjamin P. Pritchard, Doaa Altarawy, Brett Didier, Tara D. Gibson, and Theresa L. Windus J. Chem. Inf. Model. 2019, 59(11), 4814-4820 doi:10.1021/acs.jcim.9b00725.<br>A New Basis Set Exchange: An Open, Up-to-date Resource for the Molecular Sciences Community</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Database management</li><li>Chemistry</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Anonymisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>207</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/MolSSI-BSE/basis_set_exchange" target="_blank">https://github.com/MolSSI-BSE/basis_set_exchange</a></div></td>
      <td><div class='short click_expand'>BSD-3-Clause</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/treesapp" target="_blank">TreeSAPP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Tree-based Sensitive and Accurate Phylogenetic Profiler.<br>If you are working with a particularly complex reference package, from an orthologous group for example, or have extra phylogenetic information you'd like to include in your classifications, try annotating extra features with treesapp layer.</div></td>
      <td><div class='topic short click_expand'><ul><li>Phylogenetics</li><li>Molecular interactions, pathways and networks</li><li>Cladistics</li><li>Metagenomics</li><li>Sequence assembly</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Regression analysis</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ratspub" target="_blank">RatsPub</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A webservice aided by deep learning to mine PubMed for addiction-related genes.<br>RatsPub: Relationship with Addiction Through Searches of PubMed.<br>RatsPub searches PubMed to find sentences that contain the query terms (e.g., gene symbols) and a drug addiction related keyword. Over 300 of these keywords are organized into six categories:.<br>RatsPub is a tool to efficiently and comprehensively answer the question "What do we know about these genes and addiction?".<br>RatsPub answers this question by searching PubMed to find sentences containing the query terms (i.e., gene symbols) and over 300 drug addiction-related keywords that are organized into six categories. Data from NHGRI-EBI GWAS catalog are also included in the search. These gene-keyword relationships are presented as an interactive graph and a table.</div></td>
      <td><div class='topic short click_expand'><ul><li>Metabolomics</li><li>Machine learning</li><li>Ontology and terminology</li><li>GWAS study</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://rats.pub" target="_blank">http://rats.pub</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/singlecellvr" target="_blank">singlecellVR</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Interactive visualization of single-cell data in virtual reality.<br>Single cell visualization using Virtual Reality (VR).<br>It can be used with our preprocessed datasets found at the link above or by following the steps below to process your own dataset.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li><li>Cell biology</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Clustering</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/pinellolab/singlecellvr" target="_blank">https://github.com/pinellolab/singlecellvr</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ssif" target="_blank">SSIF</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Subsumption-based Sub-term Inference Framework to audit Gene Ontology.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioinformatics</li><li>Cell biology</li><li>Molecular interactions, pathways and networks</li><li>Ontology and terminology</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Relation extraction</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/oncogemini" target="_blank">OncoGEMINI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Software for Investigating Tumor Variants From Multiple Biopsies With Integrated Cancer Annotations.<br>OncoGEMINI is an adaptation of GEMINI intended for the improved identification of biologically and clincally relevant tumor variants from multi-sample and longitudinal tumor sequencing data. Using a GEMINI-compatible database (generated from an annotated VCF file), OncoGEMINI is able to filter tumor variants based on included genomic annotations and various allele frequency signatures.</div></td>
      <td><div class='topic short click_expand'><ul><li>Evolutionary biology</li><li>Genetic variation</li><li>Oncology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant prioritisation</li><li>Variant calling</li><li>Variant filtering</li><li>Query and retrieval</li><li>Genotyping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/nucleardb" target="_blank">NucleaRDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A molecular-class information system that collects, combines, validates and stores large amounts of data on Nuclear Hormone Receptors (NHRs). Contains data on sequences, ligand binding constants and mutations. In addition, many different types of computationally derived data are stored such as multiple sequence alignments and homology models.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Chemistry</li><li>Genetic variation</li><li>Molecular modelling</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein modelling</li><li>Multiple sequence alignment</li><li>Query and retrieval</li><li>Data retrieval</li><li>Validation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>15</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/protein_model_portal" target="_blank">Protein Model Portal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Developed to foster effective usage of molecular models in biomedical research by providing convenient and comprehensive access to structural information for a protein - both experimental structures and theoretical models. Protein Model Portal is a component of the Nature PSI Structural Biology Knowledgebase.<br>PMP has reached its end of life and no longer provides data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular modelling</li><li>Proteomics</li><li>Structure analysis</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>185</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pslid" target="_blank">PSLID</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PSLID provides open-access, structured, well-annotated databases of fluorescence microscope images depicting subcellular location proteins.</div></td>
      <td><div class='topic short click_expand'><ul><li>Databases</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Image analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/pslid-5-0-protein-subcellular-location-image-database.html" target="_blank">http://www.mybiosoftware.com/pslid-5-0-protein-subcellular-location-image-database.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/flimview" target="_blank">flimview</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Flimview is a software framework to handle, visualize and analyze Fluorescence-Lifetime Imaging Microscopy (FLIM) images.</div></td>
      <td><div class='topic short click_expand'><ul><li>Bioimaging</li><li>Medical imaging</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Read binning</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/printsextract" target="_blank">printsextract</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Extract data from PRINTS database for use by pscan.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequence sites, features and motifs</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>5,623</div></td>
      <td><div class="link short click_expand"><a href="http://emboss.open-bio.org/html/adm/ch01s01.html" target="_blank">http://emboss.open-bio.org/html/adm/ch01s01.html</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rimmport" target="_blank">RImmPort</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package simplifies access to ImmPort data for analysis in the R environment. It provides a standards-based interface to the ImmPort study data that is in a proprietary format.</div></td>
      <td><div class='topic short click_expand'><ul><li>Immunology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td><div class="link short click_expand"><a href="http://bioconductor.org/packages/RImmPort/" target="_blank">http://bioconductor.org/packages/RImmPort/</a></div></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dbest" target="_blank">dbEST database of EST sequences</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Division of GenBank that contains sequence data and other information on "single-pass" cDNA sequences, or "Expressed Sequence Tags", from a number of organisms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence record : genbank-html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,102</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/syfpeithi" target="_blank">SYFPEITHI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database comprising more than 7000 peptide sequences known to bind class I and class II MHC molecules. The entries are compiled from published reports only. The database can be used to predict the immunogenicity of given peptide sequences.</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>Proteins</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Sequence motif recognition</li><li>Protein feature detection</li><li>Peptide database search</li><li>Peptide immunogenicity prediction</li><li>Sequence feature detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,830</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/motoRneuron" target="_blank">motoRneuron</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An open-source R toolbox for time-domain motor unit analyses.<br>Analyzing Paired Neuron Discharge Times for Time-Domain Synchronization.<br>Using motoRneuron for motor unit synchronization.<br>The temporal relationship between motor neurons can offer explanations for neural strategies. We combined functions to reduce neuron action potential discharge data and analyze it for short-term, time-domain synchronization. Even more so, motoRneuron combines most available methods for the determining cross correlation histogram peaks and most available indices for calculating synchronization into simple functions. See Nordstrom, Fuglevand, and Enoka (1992) <doi:10.1113/jphysiol.1992.sp019244> for a more thorough introduction.<br>The purpose of this document is to get you started using the motoRneuron package for analyzing motor unit data, even if you've never used R before! So we will show you how to install the toolbox through CRAN, access the functions, help files, and sample motor unit data</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li><li>Statistics and probability</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Peak detection</li><li>Parsing</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://cran.r-project.org/web/packages/motoRneuron/index.html" target="_blank">https://cran.r-project.org/web/packages/motoRneuron/index.html</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/genatlas" target="_blank">GenAtlas human gene database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Information on gene mapping and genetic diseases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Human biology</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>51</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/load_multifastq" target="_blank">load_multifastq</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Load several FASTQ files from different conditions.</div></td>
      <td><div class='topic short click_expand'><ul><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Loading</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : fastq</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>934</div></td>
      <td><div class="link short click_expand"><a href="https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/urgi_tools/load_multiFASTQ/1.0.0" target="_blank">https://galaxy.pasteur.fr/tool_runner?tool_id=toolshed.pasteur.fr/repos/fmareuil/urgi_tools/load_multiFASTQ/1.0.0</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mmmdb" target="_blank">MMMDB</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mouse Multiple Tissue Metabolome Database: metabolomic database containing a collection of metabolites measured from multiple tissues from single mice. The datases are collected using a single instrument and not integrated from literatures, which is useful for capturing the holistic overview of large metabolomic pathway.</div></td>
      <td><div class='topic short click_expand'><ul><li>Small molecules</li><li>Molecular interactions, pathways and networks</li><li>Metabolomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data handling</li><li>Mass spectra calibration</li><li>Query and retrieval</li><li>Database comparison</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>33</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/efetch_pmc" target="_blank">eFetch Pmc</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Entrez Programming Utilities are tools that provide access to Entrez data outside of the regular web query interface and may be helpful for retrieving search results for future use in another environment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Literature and language</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Literature search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>277</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/2" target="_blank">https://www.biocatalogue.org/services/2</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/HiPPO" target="_blank">HiPPO</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Two Bioinformatics Tools to Support Analysis of Mass Cytometry Data.<br>High-dimensional mass cytometry (Cytometry by Time-Of-Flight; CyTOF) is a multiparametric single-cell approach that allows for more than 40 parameters to be evaluated simultaneously, opening the possibility to dissect cellular heterogeneity and elucidate functional interactions between different cell types. However, the complexity of these data makes analysis and interpretation daunting. We created High-throughput Population Profiler (HiPPO), a tool that reduces the complexity of the CyTOF data and allows homogeneous clusters of cells to be visualized in an intuitive manner. Each subpopulation is mapped to the Population Analysis Database (PANDA), an open-source, manually curated database containing protein expression profiles for selected markers of primary cells, allowing for cell type abundance in the analyzed samples to be monitored.</div></td>
      <td><div class='topic short click_expand'><ul><li>Cell biology</li><li>Immunoproteins and antigens</li><li>Protein expression</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>Visualisation</li><li>Expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/digsee" target="_blank">DigSee</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web server for understanding the association between diseases and genes. This cancer version searches Medline abstracts for evidence sentences describing the genes involved in the development of cancer through biological events.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene expression</li><li>Molecular interactions, pathways and networks</li><li>Oncology</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Named-entity and concept recognition</li><li>Query and retrieval</li><li>Gene expression analysis</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>37</div></td>
      <td></td>
      <td><div class='short click_expand'>Unlicense</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/scde_discovery" target="_blank">Stem Cell Discovery Engine (SCDE)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of curated cancer stem cell (CSC) experiments coupled to the Galaxy analytical framework.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Molecular interactions, pathways and networks</li><li>Medicine</li><li>Experimental design and studies</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Query and retrieval</li><li>Deposition</li><li>Comparison</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>18</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/XenMine" target="_blank">XenMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Search and retrieve integrated Xenopus tropicalis and Xenopus laevis data</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rmdisease" target="_blank">RMDisease</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A database of genetic variants that affect RNA modifications, with implications for epitranscriptome pathogenesis.<br>RMDisease a database of genetic variants that affect RNA modifications, with implications for epitranscriptome pathogenesis.<br>Deciphering the biological impacts of millions of single nucleotide variants remains a major challenge.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA splicing</li><li>Gene transcripts</li><li>Functional, regulatory and non-coding RNA</li><li>DNA polymorphism</li><li>Transcription factors and regulatory sites</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>SNP annotation</li><li>PTM site prediction</li><li>SNP detection</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ndexr" target="_blank">ndexr</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This package offers an interface to NDEx servers, e.g. the public server at http://ndexbio.org/. It can retrieve and save networks via the API. Networks are offered as RCX object and as igraph representation.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data acquisition</li><li>Data submission, annotation and curation</li><li>Biological networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/tarpan" target="_blank">TarPan</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>TarPan Viewer is a tool used to visually inspect targeted panel sequencing data. an easily adaptable targeted sequencing panel viewer for research and clinical use.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Sequencing</li><li>Genetic variation</li><li>Personalised medicine</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Variant calling</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ega" target="_blank">EGA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>European genome-phenome Archive (EGA) - service for permanent archiving and sharing of all types of personally identifiable genetic and phenotypic data resulting from biomedical research projects.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Data integration and warehousing</li><li>Laboratory information management</li><li>Rare diseases</li><li>Data security</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>experiment report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>138</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/alliance-genome" target="_blank">Alliance Genome Resources</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A modern data ecosystem for model organism databases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data governance</li><li>Literature and language</li><li>Ontology and terminology</li><li>Model organisms</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/alliance-genome/" target="_blank">https://github.com/alliance-genome/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/rnavirusdb" target="_blank">RNA virus genome database (RNAVirusDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>RNA virus genome database.</div></td>
      <td><div class='topic short click_expand'><ul><li>Virology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>organism report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ScType" target="_blank">ScType</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>ScType is a tool for fully-automated cell type identification from single-cell RNA-seq data. ScType provides a complete pipeline for single-cell RNA-seq data analysis (including data processing, normalization and clustering) and cell-type annotation.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-Seq</li><li>Proteomics</li><li>Cell biology</li><li>Database management</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Clustering</li><li>RNA-Seq analysis</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://sctype.fimm.fi/database.php" target="_blank">https://sctype.fimm.fi/database.php</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gdportal" target="_blank">GDportal</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Uniform Genomic Data Analysis in the NCI Genomic Data Commons.<br>The NCI's Genomic Data Commons (GDC) provides the cancer research community with a unified data repository that enables data sharing across cancer genomic studies in support of precision medicine.<br>The Next Generation Cancer Knowledge Network.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Epigenomics</li><li>Proteomics</li><li>Functional, regulatory and non-coding RNA</li><li>Personalised medicine</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Deposition</li><li>Query and retrieval</li><li>Genome indexing</li><li>Genotyping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://portal.gdc.cancer.gov/" target="_blank">https://portal.gdc.cancer.gov/</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/mutrank" target="_blank">MutRank</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An R shiny web-application for exploratory targeted mutual rank-based coexpression analyses integrated with user-provided supporting information.<br>Transcriptomic data can uncover complex biological processes in part through the improved understanding of gene coexpression patterns.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genotype and phenotype</li><li>Molecular interactions, pathways and networks</li><li>Plant biology</li><li>Endocrinology and metabolism</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic pathway prediction</li><li>Expression correlation analysis</li><li>Genotyping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Genomicus-Plants" target="_blank">Genomicus-plants</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomicus-Plants is a genome browser that enables users to navigate in plants genomes in several dimensions: linearly along chromosome axes, transversaly across different species, and chronologicaly along evolutionary time.</div></td>
      <td><div class='topic short click_expand'><ul><li>Paleogenomics</li><li>Plant biology</li><li>Gene and protein families</li><li>Comparative genomics</li><li>Evolutionary biology</li><li>Genomics</li><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Genome visualisation</li><li>Phylogenetic tree visualisation</li><li>Dot plot plotting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>394</div></td>
      <td><div class="link short click_expand"><a href="https://www.genomicus.biologie.ens.fr/genomicus" target="_blank">https://www.genomicus.biologie.ens.fr/genomicus</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/lunggens" target="_blank">LungGENS</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Query single-cell gene expression databases by entering a gene symbol or a list of genes or selecting a cell type of their interest. Gene query provides quantitative RNA expression of the gene of interest in each lung cell type. Cell type query returns associated selective gene signatures and genes encoding cell surface markers and transcription factors in interactive heatmap and tables.</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA-seq</li><li>Gene transcripts</li><li>Gene expression</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>60</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/molmeth" target="_blank">MolMeth</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A structured database that provides best practice-based protocols for molecular analyses.</div></td>
      <td><div class='topic short click_expand'><ul><li>Laboratory techniques</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/GrapeMine" target="_blank">GrapeMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An integrated database for grapevine data</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MapOptics" target="_blank">MapOptics</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Lightweight cross-platform tool that enables the user to visualise and interact with the alignment of Bionano optical mapping data and can be used for in depth exploration of hybrid scaffolding alignments.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Sequence assembly</li><li>Imaging</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Scaffolding</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Desktop application</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/FadyMohareb/mapoptics/issues" target="_blank">https://github.com/FadyMohareb/mapoptics/issues</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pymzml" target="_blank">pymzML</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Python module to parse mzML data based on cElementTree.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics experiment</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Spectral analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : mzml</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>data : mzml</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>70</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/pymzml/pymzML" target="_blank">https://github.com/pymzml/pymzML</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/topdb" target="_blank">Topology Data Bank of Transmembrane Proteins</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Transmembrane protein datasets containing experimentally derived topology information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Membrane and lipoproteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>protein features : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>63</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/toxodb" target="_blank">Toxoplasma genome resources (ToxoDB)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Genomic-scale datasets associated with the eukaryotic pathogens Toxoplasma.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>gene report : html</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>322</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/pri-cat" target="_blank">PRI-CAT</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>PRI-CAT is a web-based workflow tool for the management and analysis of plant ChIP-seq experiments, with focus on Arabidopsis. Secondary analysis can be performed with the aid of GALAXY.</div></td>
      <td><div class='topic short click_expand'><ul><li>ChIP-seq</li><li>Immunoprecipitation experiment</li><li>Plant biology</li><li>Transcription factors and regulatory sites</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>14</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/superfreq" target="_blank">SuperFreq</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Integrated mutation detection and clonal tracking in cancer.<br>SuperFreq is an R package that analyses cancer exomes.<br>Summarised data and code to reproduce figures for the TCGA analysis in the superFreq methods paper.<br>SuperFreq analyses and filters somatic SNVs and short indels, calls copy numbers and tracks clones over multiple samples from the same individual. It identifies the copy number alterations and point mutations in each clone, and highlights potentially causing mutations through variant annotation and COSMIC.</div></td>
      <td><div class='topic short click_expand'><ul><li>Oncology</li><li>Phylogeny</li><li>Genetic variation</li><li>Exome sequencing</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant prioritisation</li><li>Variant calling</li><li>Standardisation and normalisation</li><li>Genotyping</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/ChristofferFlensburg/SuperFreq" target="_blank">https://github.com/ChristofferFlensburg/SuperFreq</a></div></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/quincunx" target="_blank">quincunx</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>An R package to query, download and wrangle PGS Catalog data.<br>The goal of quincunx is to provide programmatic access to the Polygenic Score (PGS) Catalog, an open resource for polygenic scores and associated metadata describing their development and respective assessment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Filtering</li><li>Data retrieval</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ZincBind" target="_blank">ZincBind</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Database of zinc binding sites.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein binding sites</li><li>Transcription factors and regulatory sites</li><li>Database management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Transcription factor binding site prediction</li><li>Binding site prediction</li><li>Protein-nucleic acid interaction analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>6</div></td>
      <td><div class="link short click_expand"><a href="https://github.com/samirelanduk/ZincBindDB" target="_blank">https://github.com/samirelanduk/ZincBindDB</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/gorth" target="_blank">gOrth</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A tool of the g:Profiler web server. It maps a list of genes of interest to homologous genes in another related target organism.</div></td>
      <td><div class='topic short click_expand'><ul><li>Evolutionary biology</li><li>Data identity and mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Homology-based gene prediction</li><li>ID mapping</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : textual format</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>identifier : textual format</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>565</div></td>
      <td></td>
      <td><div class='short click_expand'>Freeware</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ncbi_taxonomy_database" target="_blank">NCBI Taxonomy Database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Curated classification and nomenclature for all of the organisms in the public sequence databases. This currently represents about 10% of the described species of life on the planet.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Genotype and phenotype</li><li>Taxonomy</li><li>Mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Genotyping</li><li>Data retrieval</li><li>Taxonomic classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>2,112</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/corbi" target="_blank">Corbi</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Biological network alignment and querying.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>9</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/hprd" target="_blank">The Human Protein Reference Database (HPRD)</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Centralized resource for information about human proteins, their interactions with other human proteins, and protein-disease relationships. The information contained in it is curated by experts, who manually extract it from published literature.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Proteins</li><li>Molecular interactions, pathways and networks</li><li>Protein interactions</li><li>Human biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein interaction analysis</li><li>Query and retrieval</li><li>Protein interaction prediction</li><li>Post-translation modification site prediction</li><li>Protein interaction network analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>1,209</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/castle-lethals" target="_blank">CASTLE</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>CASTLE (Computational Analysis of SynThetic LEthals) is a collection of synthetic lethal data for various pathogenic organisms, generated computationally. Here, you can find synthetic lethal genes and reactions for more than 70 organisms.</div></td>
      <td><div class='topic short click_expand'><ul><li>Model organisms</li><li>Endocrinology and metabolism</li><li>Drug metabolism</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Metabolic network modelling</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/allen_brain_atlas" target="_blank">Allen Brain Atlas</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The Allen Brain Atlas (ABA) project has developed a web-based application designed to aid the intersection of neuroscience and genomics. The ABA Application enables users to access an extensive database of high resolution in situ hybridization (ISH) images from over 10,000 genes, reference atlases in both the sagittal and coronal planes, gene expression masks capturing the intensity of gene expression or signal, and the ability to search for gene expression by anatomic region.</div></td>
      <td><div class='topic short click_expand'><ul><li>Gene transcripts</li><li>Imaging</li><li>Neurobiology</li><li>Gene expression</li><li>MRI</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Gene expression profiling</li><li>Gene expression comparison</li><li>Query and retrieval</li><li>Gene expression analysis</li><li>Differential gene expression analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>256</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ontoquestmain" target="_blank">OntoQuestMain</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Search terms starting for the query used for auto-complete service.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Data mining</li><li>Data management</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://www.biocatalogue.org/services/2715" target="_blank">https://www.biocatalogue.org/services/2715</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ensemblvep" target="_blank">BioConductor wrapper for Ensembl VEP</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Query the Ensembl Variant Effect Predictor via the perl API.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li><li>Bioinformatics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence variations : vcf</li></ul></div></td>
      <td><div class='formats short click_expand'><ul><li>sequence variations : vcf</li></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>1,440</div></td>
      <td></td>
      <td><div class='short click_expand'>Artistic-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/kg-covid-19" target="_blank">KG-COVID-19</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A framework to produce customized knowledge graphs for COVID-19 response.<br>Prebuilt versions of the KG-COVID-19 knowledge graph build from all available data are available in the following serialization formats:.<br>The KG-COVID-19 project is the first instantiation of such a KG Hub. Thus, KG-COVID-19 is a framework, that follows design patterns of the KG Hub, to download and transform COVID-19/SARS-COV-2 related datasets and emit a knowledge graph that can then be used for machine learning or others uses, to produce actionable knowledge.</div></td>
      <td><div class='topic short click_expand'><ul><li>Machine learning</li><li>Public health and epidemiology</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/ensembldb3" target="_blank">ensembldb3</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>This library provides capabilities for querying Ensembl MySQL databases and for "administering" them. It is compatible with a range of Ensembl releases.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/alphapart" target="_blank">AlphaPart</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>R implementation of the method for partitioning genetic trends. Partition/Decomposition of Breeding Values by Paths of Information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td><div class="link short click_expand"><a href="https://git.ecdf.ed.ac.uk/HighlanderLab_public/jobsteter_alphapart" target="_blank">https://git.ecdf.ed.ac.uk/HighlanderLab_public/jobsteter_alphapart</a></div></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/chanjo" target="_blank">Chanjo</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Chanjo is coverage analysis for clinical sequencing. It's implemented in Python with a command line interface that adheres to UNIX pipeline philosophy. Chanjo leverages Sambamba to annotate coverage and completeness for a general BED-file. The output can then easily to loaded into a SQL database that enables investigation of coverage across regions and samples. The database also works as an API to downstream tools like the Chanjo Coverage Report generator.</div></td>
      <td><div class='topic short click_expand'><ul><li>Exome sequencing</li><li>Workflows</li><li>Sequence assembly</li><li>Whole genome sequencing</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/vespucci" target="_blank">VESPUCCI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>COLOMBOS technology-based web portal for exploring and analyzing a comprehensive Vitis vinifera specific cross-platform expression compendium. This compendium was carefully constructed by collecting, homogenizing and formally annotating publicly available microarray and RNA-seq experiments from Gene Expression Omnibus (GEO), and ArrayExpress.</div></td>
      <td><div class='topic short click_expand'><ul><li>Systems biology</li><li>Plant biology</li><li>Functional genomics</li><li>Transcriptomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Gene expression analysis</li><li>Query and retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>29</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/miolite" target="_blank">MESBL GC-MS metabolite peak database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>The MESBL GC-MS metabolite peak database is a standardized library of more than 900 metabolite peaks from MS-reconstructed gas chromatograms integrating the in-house standard compound and peak library of the FORTH/ICE-HT Metabolic Engineering and Systems Biology Laboratory, Greece, appropriately filtered GOLM database peak information and Human Metabolome database information.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biochemistry</li><li>Metabolomics</li><li>Laboratory techniques</li><li>Systems biology</li><li>Cheminformatics</li><li>Data identity and mapping</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Peak detection</li><li>Query and retrieval</li><li>Formatting</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>5</div></td>
      <td><div class="link short click_expand"><a href="http://miolite2.iceht.forth.gr" target="_blank">http://miolite2.iceht.forth.gr</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/dna-features-viewer" target="_blank">DNA Features Viewer</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>DNA Features Viewer is a Python library to visualize DNA features.  Dna Features Viewer automatically produce simple and clear plots even for sequences with many overlapping features and long labels. The libray plays well with Matplotlib and Biopython, and the plots can be output to many different formats (PNG, JPEG, SVG, PDF), e.g. for report generation, article figures, or LIMS interfaces.</div></td>
      <td><div class='topic short click_expand'><ul><li>Mapping</li><li>Transcription factors and regulatory sites</li><li>Sequencing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Sequence annotation</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>MIT</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/domac" target="_blank">DOMAC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Accurate, protein domain prediction server that integrates homology modeling, domain parsing, and ab initio methods together.</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular modelling</li><li>Structure prediction</li><li>Proteins</li><li>Protein folds and structural domains</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein domain recognition</li><li>Protein feature detection</li><li>Protein modelling</li><li>Ab initio structure prediction</li><li>Protein fold recognition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>189</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/WormMine" target="_blank">WormMine</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Intermine data mining platform for C. elegans and related nematodes</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li><li>Data management</li><li>Genomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Visualisation</li><li>Query and retrieval</li><li>Data retrieval</li><li>Gene-set enrichment analysis</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li><li>Web service</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>150</div></td>
      <td><div class="link short click_expand"><a href="https://registry.intermine.org" target="_blank">https://registry.intermine.org</a></div></td>
      <td><div class='short click_expand'>LGPL-2.1</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/SCOC_database" target="_blank">SCOC database</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>SCOC database is a large, open, and global database with sediment community oxygen consumption rates.</div></td>
      <td><div class='topic short click_expand'><ul><li>Marine biology</li><li>Literature and language</li><li>Database management</li><li>Data submission, annotation and curation</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Deposition</li><li>Database search</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sapfinder" target="_blank">sapFinder</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Tool developed to automate (1) variation-associated database construction, (2) database searching, (3) post-processing, (4) HTML-based report generation in shotgun proteomics.</div></td>
      <td><div class='topic short click_expand'><ul><li>Proteomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li><li>Library</li></ul></div></td>
      <td><div class='citations short click_expand'>18</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-2.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/VIC" target="_blank">VIC</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A computational tool for assessing clinical impacts of somatic variants | A Computational Tool for Assessing Clinical Impacts of Somatic Variants Following the AMP-ASCO-CAP 2017 Guidelines | Variant Interpretation for Cancer (VIC) | Clinical Interpretations of Variants in Cancer | java -jar target/VIC-1.0-jar-with-dependencies.jar [-b ] [-d ] [-h] [-input_type ] [-i ] [-cancer_type ] [-convert2annovar ] [-l ] [-skip_annovar] [-o ] [-annotate_variation ] [-table_annovar ] [-s ] [-otherinfo ] [-db ]</div></td>
      <td><div class='topic short click_expand'><ul><li>Allergy, clinical immunology and immunotherapeutics</li><li>Oncology</li><li>Pathology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Variant calling</li><li>Parsing</li><li>Variant classification</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td><div class="link short click_expand"><a href="https://civicdb.org/releases" target="_blank">https://civicdb.org/releases</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/homa" target="_blank">HOMA</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-based interface that can create homology models of a protein with unknown structure (the target or query protein) based on a homologous protein with known structure (the template protein).</div></td>
      <td><div class='topic short click_expand'><ul><li>Molecular modelling</li><li>Sequence analysis</li><li>Proteins</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Protein modelling</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web service</li></ul></div></td>
      <td><div class='citations short click_expand'>32</div></td>
      <td><div class="link short click_expand"><a href="http://www.mybiosoftware.com/homa-create-homology-models-protein-unknown-structure.html" target="_blank">http://www.mybiosoftware.com/homa-create-homology-models-protein-unknown-structure.html</a></div></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/deepblue" target="_blank">DeepBlue Epigenomic Data Server</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Provides a central data access hub for large collections of epigenomic data. It organizes the data from different sources using controlled vocabularies and ontologies. The data is stored in our server, where the users can access the data programmatically or by our web interface. It contains a set of commands designed to perform operations on the epigenomic data. Users can execute all operations in a pipeline fashion in the server and transfer only meaningful data.</div></td>
      <td><div class='topic short click_expand'><ul><li>Epigenomics</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Aggregation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web API</li><li>Web application</li><li>Database portal</li></ul></div></td>
      <td><div class='citations short click_expand'>35</div></td>
      <td></td>
      <td><div class='short click_expand'>GPL-3.0</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/sdi" target="_blank">SDI</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Quantifying Gene Essentiality Based on the Context of Cellular Components.<br>SDI Database for the subcellular diversity index.<br>Database for the subcellular diversity index.</div></td>
      <td><div class='topic short click_expand'><ul><li>Protein targeting and localisation</li><li>Biological databases</li><li>Pharmacology</li><li>Gene expression</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Expression correlation analysis</li><li>Subcellular localisation prediction</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>1</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/neuroscinfframework" target="_blank">Neuroscience Information Framework</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>A dynamic inventory of Web-based neuroscience resources: data, materials, and tools accessible via any computer connected to the Internet. An initiative of the NIH Blueprint for Neuroscience Research, the Neuroscience Information Framework advances neuroscience research by enabling discovery and access to public research data and tools worldwide through an open source, networked environment.</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Neurobiology</li><li>Informatics</li><li>Molecular interactions, pathways and networks</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>Deposition</li><li>Data retrieval</li><li>Ontology visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>204</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/NLIMED" target="_blank">NLIMED</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Natural Language Interface for Model Entity Discovery in Biosimulation Model Repositories | This project is to convert user query to SPARQL in the Physiome Model Repository. The information looking by users usually is model entities | Natural Language Interface for Model Entity Discovery (NLIMED) is an interface to search model entities (i.e. flux of sodium across the basolateral plasma membrane, the concentration of potassium in the portion of tissue fluid) in the biosimulation models in repositories. The interface utilises the RDF inside biosimulation models and metadata from BioPortal. Currently, the interface can retrieve model entities from the Physiome Model Repository (PMR, https://models.physiomeproject.org) and the BioModels (https://www.ebi.ac.uk/biomodels/)</div></td>
      <td><div class='topic short click_expand'><ul><li>Ontology and terminology</li><li>Bioengineering</li><li>Natural language processing</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Named-entity and concept recognition</li><li>Data retrieval</li><li>Standardisation and normalisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>0</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/RNAm5CPred" target="_blank">RNAm5CPred</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Prediction of RNA 5-Methylcytosine Sites Based on Three Different Kinds of Nucleotide Composition.<br>RNAm5CPred: A webserver for RNA 5-methylcytosine sites prediction.<br>A webserver for RNA 5-methylcytosine sites prediction. RNAm5CPred was developed by using support vector machine based on three different kinds of nucleotide composition features including KNF (K-nucleotide frequencies), KSNPF (K-spaced nucleotide pair frequencies) and pseDNC (pseudo dinucleotide composition). The input for the server is the RNA sequence. More details can be found in the tutorial</div></td>
      <td><div class='topic short click_expand'><ul><li>RNA</li><li>Transcriptomics</li><li>RNA-Seq</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Query and retrieval</li><li>EST assembly</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>15</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/MEXCOwalk" target="_blank">MEXCOwalk</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Mutual Exclusion and Coverage Based Random Walk to Identify Cancer Modules | This is the original repository for the MEXCOwalk paper | The evaluation file can also be run with multiple models and compared to MEXCOwalk. Simply store the models in 'out/connected_components_isolarge/model_x' and add the model name to the evaluations section in the evaluation file as follows:</div></td>
      <td><div class='topic short click_expand'><ul><li>Genetic variation</li><li>Oncology</li><li>Protein interactions</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Enrichment analysis</li><li>Parsing</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Command-line tool</li></ul></div></td>
      <td><div class='citations short click_expand'>3</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/Zr-MOFs" target="_blank">Zr-MOFs</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Large-Scale Structural Refinement and Screening of Zirconium Metal-Organic Frameworks for H2S/CH4 Separation.<br>You can find more than 182 experimentally synthesized guest-free and disorder-free Zirconium-based metal-organic frameworks (Zr-MOFs), which contain no less than 10 coordinated Zr6O8 inner cores. These structures are optimized by a combination of periodic density functional theory (DFT) and molecular mechanics method, and are given in cif format</div></td>
      <td><div class='topic short click_expand'><ul><li>Database management</li><li>X-ray diffraction</li><li>Small molecules</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Data retrieval</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul></ul></div></td>
      <td><div class='citations short click_expand'>2</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
    <tr>
      <td><div class="name short click_expand"><a href="https://bio.tools/stemcellnet" target="_blank">StemCellNet</a></div></td>
      <td><div class='short click_expand'>0.333</div></td>
      <td><div class='desc short click_expand'>Web-server that allows users to interactively query, visualize and interrogate molecular networks in the context of stem cell biology. It includes a comprehensive set of physical and regulatory interactions for both human and mouse. Any retrieved network can be readily examined for the presence of genes linked to stemness.</div></td>
      <td><div class='topic short click_expand'><ul><li>Biomedical science</li><li>Computational biology</li></ul></div></td>
      <td><div class='operation short click_expand'><ul><li>Pathway or network visualisation</li></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='formats short click_expand'><ul></ul></div></td>
      <td><div class='type short click_expand'><ul><li>Web application</li></ul></div></td>
      <td><div class='citations short click_expand'>11</div></td>
      <td></td>
      <td><div class='short click_expand'>None</div></td>
    </tr>
  </tbody>
</table>
        </div>
    </div>
    

</body>

<script>
    $('#my-table').dataTable( {
      "order": [],
      } );

    var userSelection = document.getElementsByClassName('click_expand');

    for(var i = 0; i < userSelection.length; i++) {
      (function(index) {
        userSelection[index].addEventListener("click", function() {
          console.log("Clicked index: ");
          $(this).closest("tr").find('div').toggleClass("short");

        })
      })(i);
    }

    
    $('#my-table thead th').each(function() {
      var title = $('#my-table thead th').eq($(this).index()).text();
      $(this).css('text-align', 'left');
      });

    var r = $('#my-table thead th');
      r.find('input').each(function(){
      $(this).css('margin', 4);
      $(this).css('padding', 4);
    });

    // DataTable
    var table = $('#my-table').DataTable();

    // Apply the search
    table.columns().eq(0).each(function(colIdx) {
        $('input', table.column(colIdx).header()).on('keyup change', function() {
          table
                .column(colIdx)
                .search(this.value)
                .draw();
        });

        $('input', table.column(colIdx).header()).on('click', function(e) {
            e.stopPropagation();
        });
    });

    
</script>
