{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# bio.tools querying"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Overview"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Code"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "import json\n",
    "import requests\n",
    "import argparse\n",
    "import yaml\n",
    "import biotools_parse as bp\n",
    "import pandas as pd\n",
    "from pathlib import Path\n",
    "\n",
    "\n",
    "# The call to get all bio.tools entries through the api is: https://bio.tools/api/tool/?format=json.\n",
    "# Unfortunately, the response is limited to n tools. To get the whole set, the next pages must be retrieve.\n",
    "# use \"next\" in the response to get succesive entries.\n",
    " \n",
    "base_call = \"https://bio.tools/api/tool/?format=json\"\n",
    "\n",
    "\n",
    "def make_request(URL):\n",
    "    try:\n",
    "        response = requests.get(URL)\n",
    "    except:\n",
    "        print('Could not make the request')\n",
    "        return\n",
    "    else:\n",
    "        response = json.loads(response.text)\n",
    "        return(response)\n",
    "\n",
    "def build_url(next_page, filters):\n",
    "    call_template = \"https://bio.tools/api/tool/?{attributes}{next_page}&format=json\"\n",
    "    if next_page:\n",
    "        next_page = \"&%s\"%next_page\n",
    "    else:\n",
    "        next_page = \"\"\n",
    "    attributes = '&'.join(filters)    \n",
    "    url = call_template.format(next_page=next_page, attributes=attributes)\n",
    "    return(url)\n",
    "\n",
    "\n",
    "def get_all_pages(filters):\n",
    "    res = []\n",
    "    next_page = \"page=1\"\n",
    "    print('Starting making the requests...')\n",
    "    while next_page:\n",
    "        response = make_request(build_url(next_page, filters))\n",
    "        print(\"Requesting: \" + build_url(next_page, filters), end='\\r')\n",
    "        res = res + response[\"list\"]\n",
    "        next_page = response[\"next\"]\n",
    "        if next_page:\n",
    "            next_page = next_page[1:]\n",
    "    print('\\nRequests finished.')\n",
    "    return(res)\n",
    "\n",
    "def save_result(out_path, result):\n",
    "    with open(out_path, 'w') as out:\n",
    "        json.dump(result, out)\n",
    "    print('Result saved as ' + out_path )\n",
    "    \n",
    "\n",
    "colnames_general = ['name', 'description', 'type', 'topic', 'input', 'output']\n",
    "colnames_detailed = [ 'name', 'description', 'version', 'type', 'topic', 'links', 'publication', 'download', 'inst_instr', 'test', 'src', 'os', 'input', 'output', 'dependencies', 'documentation', 'license', 'termsUse', 'contribPolicy', 'authors', 'repository']\n",
    "def results_to_table(result, colnames):\n",
    "    tools = bp.biotoolsToolsGenerator(result).instances\n",
    "    # Load features into table\n",
    "    colnames_features = colnames\n",
    "    df_dict = {name : [] for name in colnames_features}\n",
    "\n",
    "    for tool in tools:\n",
    "        for field in colnames_features:\n",
    "            df_dict[field].append(tool.__dict__.get(field))\n",
    "\n",
    "    df_features = pd.DataFrame.from_dict(df_dict)\n",
    "    return(df_features)\n",
    "\n",
    "\n",
    "def parse_zooma_results(input_file):\n",
    "    with open(input_file, 'r') as inp:\n",
    "        zooma_terms_df = pd.read_csv(input_file)\n",
    "        # getting list of terms\n",
    "        terms = zooma_terms_df['iri']\n",
    "        terms = []\n",
    "        terms_label = {}\n",
    "        for index, row in zooma_terms_df.iterrows():\n",
    "            if type(row['iri']) == str:\n",
    "                term = row['iri'].split('http://edamontology.org/')[1]\n",
    "                terms.append(term)\n",
    "                terms_label[term] = row['label']\n",
    "    return(terms, terms_label)\n",
    "\n",
    "def build_filter(term):\n",
    "    filters_template =  {'topic': 'topicID=\"%s\"', 'format':'dataFormatID=\"%s\"', 'operation':'operationID=\"%s\"', 'data':'dataTypeID=\"%s\"'}\n",
    "    if 'topic' in term:\n",
    "        filters = filters_template['topic']%(term)\n",
    "    elif 'data' in term:\n",
    "        filters = filters_template['data']%(term)\n",
    "    elif 'operation' in term:\n",
    "        filters = filters_template['operation']%(term)\n",
    "    elif 'format' in term:\n",
    "        filters = filters_template['format']%(term)\n",
    "    return(filters)     \n",
    "\n",
    "def query_for_EDAM_terms(terms):\n",
    "    '''\n",
    "    Takes a list of EDAM terms and does a query for each\n",
    "    Return a dictionary of results of the form: {<term>: <dataframe of results>}\n",
    "    '''\n",
    "    results = dict()\n",
    "    results_detailed = dict()\n",
    "    # Iteratre through terms\n",
    "    for term in terms:\n",
    "        # Avoid duplicated queries\n",
    "        if term not in results.keys():\n",
    "            # Building the filters using the EDAM terms\n",
    "            filter_ = [build_filter(term)]\n",
    "            # Do the query\n",
    "            result = get_all_pages(filter_)\n",
    "            # Put result in table\n",
    "            result_df = results_to_table(result, colnames_general)\n",
    "            result_detailed_df = results_to_table(result, colnames_detailed)\n",
    "            # Put results table in dictionary with the results for the other terms\n",
    "            results[term] = result_df\n",
    "            results_detailed[term] = result_detailed_df\n",
    "    return(results, results_detailed)\n",
    "\n",
    "def count_tools_per_term(results):\n",
    "    count_tools_per_term = {term :len(results[term]) for term in results.keys()}\n",
    "    return(count_tools_per_term) \n",
    "\n",
    "def tools_per_term(results):\n",
    "    tools_per_term = {term:results[term]['name'] for term in results}  \n",
    "    return(tools_per_term)\n",
    "\n",
    "def merge_tools_lists(results):\n",
    "    tools_per_term_ = tools_per_term(results)\n",
    "    tools = set()\n",
    "    for term in tools_per_term_:\n",
    "        for tool in tools_per_term_[term]:\n",
    "            tools.add(tool)\n",
    "    return(tools)\n",
    "\n",
    "def save_results(results_general, terms_label, path):\n",
    "    template_output = path + '/tools_%s.tsv'\n",
    "    for term in results_general.keys():\n",
    "        file_name = template_output%(terms_label[term])\n",
    "        results_general[term].to_csv(file_name, index = False, sep='\\t')\n",
    "\n",
    "def save_lists_tools(tools_per_term,path):\n",
    "    template_output = path + '/tools_%s.txt'\n",
    "    for term in tools_per_term.keys():\n",
    "        with open(template_output%(term), 'w') as f:\n",
    "            for item in tools_per_term[term]:\n",
    "                f.write(\"%s\\n\" % item)\n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The attributes chosen to filter the query must be passed to the `get_all_pages` function in a list. Each attribute must be a string of the form `name=signalp`. This string will be put as a parameter in the query URL as it is. Please, follow the [bio.tools API reference](https://biotools.readthedocs.io/en/latest/api_reference.html) to construct this parameters successfully."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Simple Query Example 1:"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The following is an example of a query of tools annotted with the EDAM topic \"Sequence composition, complexity and repeats\" and with the word \"game\"  in their description. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?topic=\"Sequence composition, complexity and repeats\"&description=\"game\"&page=1&format=json\n",
      "Requests finished.\n"
     ]
    }
   ],
   "source": [
    "sample_filters = ['topic=\"Sequence composition, complexity and repeats\"', 'description=\"game\"']\n",
    "filters = sample_filters\n",
    "result = get_all_pages(filters)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'elixirCommunity': [], 'collectionID': ['EMBOSS'], 'accessibility': None, 'community': None, 'topic': [{'term': 'Sequence composition, complexity and repeats', 'uri': 'http://edamontology.org/topic_0157'}], 'owner': 'EMBOSS', 'cost': 'Free of charge', 'relation': [], 'download': [{'url': 'http://emboss.open-bio.org/html/adm/ch01s01.html', 'note': None, 'version': None, 'type': 'Source code'}, {'url': 'http://emboss.open-bio.org/html/adm/ch01s01.html', 'note': None, 'version': None, 'type': 'Binaries'}], 'validated': 1, 'publication': [{'doi': '10.1016/S0168-9525(00)02024-2', 'note': None, 'version': None, 'pmid': None, 'type': ['Primary'], 'pmcid': None, 'metadata': {'title': 'EMBOSS: The European Molecular Biology Open Software Suite', 'abstract': '', 'citationCount': 4828, 'authors': [{'name': 'Rice P.'}, {'name': 'Longden L.'}, {'name': 'Bleasby A.'}], 'date': '2000-06-01T00:00:00Z', 'journal': 'Trends in Genetics'}}, {'doi': '10.1017/CBO9781139151399', 'note': None, 'version': None, 'pmid': None, 'type': ['Other'], 'pmcid': None, 'metadata': None}, {'doi': '10.1017/CBO9781139151405', 'note': None, 'version': None, 'pmid': None, 'type': ['Other'], 'pmcid': None, 'metadata': None}], 'homepage_status': 0, 'credit': [{'typeRole': ['Contributor'], 'name': 'EMBOSS Contributors', 'rorid': None, 'url': 'http://emboss.open-bio.org/html/use/pr01s01.html', 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': None, 'email': None}, {'typeRole': ['Provider'], 'name': 'EMBL-EBI', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Institute', 'email': None}, {'typeRole': ['Contributor'], 'name': 'UK MRC', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Funding agency', 'email': None}, {'typeRole': ['Contributor'], 'name': 'Wellcome Trust', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Funding agency', 'email': None}, {'typeRole': ['Contributor'], 'name': 'UK BBSRC', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Funding agency', 'email': None}, {'typeRole': ['Provider'], 'name': 'EMBOSS', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Institute', 'email': None}, {'typeRole': ['Support'], 'name': 'EMBOSS', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': None, 'email': 'emboss@emboss.open-bio.org'}, {'typeRole': ['Primary contact'], 'name': None, 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Person', 'email': 'emboss@emboss.open-bio.org'}, {'typeRole': ['Primary contact'], 'name': None, 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Person', 'email': 'emboss-bug@emboss.open-bio.org'}], 'biotoolsCURIE': 'biotools:chaos', 'elixirPlatform': [], 'elixirNode': [], 'version': ['r6'], 'elixir_badge': 0, 'homepage': 'http://emboss.open-bio.org/rel/rel6/apps/chaos.html', 'function': [{'note': None, 'input': [{'data': {'term': 'Nucleic acid sequence record', 'uri': 'http://edamontology.org/data_2887'}, 'format': []}], 'operation': [{'term': 'Sequence composition calculation', 'uri': 'http://edamontology.org/operation_0236'}], 'cmd': None, 'output': [{'data': {'term': 'Sequence composition plot', 'uri': 'http://edamontology.org/data_2166'}, 'format': []}]}], 'lastUpdate': '2018-12-10T12:58:52Z', 'otherID': [], 'description': 'Draw a chaos game representation plot for a nucleotide sequence.', 'toolType': ['Command-line tool'], 'editPermission': {'type': 'private', 'authors': []}, 'language': ['C'], 'link': [{'url': 'http://emboss.open-bio.org/html/adm/ch01s01.html', 'note': None, 'type': ['Repository']}], 'biotoolsID': 'chaos', 'additionDate': '2015-11-08T14:40:19Z', 'name': 'chaos', 'license': 'GPL-3.0', 'documentation': [{'url': 'http://emboss.open-bio.org/html/dev/ch01s01.html', 'note': None, 'type': ['Terms of use']}, {'url': 'http://emboss.open-bio.org/html/use/pr02s04.html', 'note': None, 'type': ['Citation instructions']}, {'url': 'http://emboss.open-bio.org/rel/rel6/apps/chaos.html', 'note': None, 'type': ['General']}], 'maturity': 'Mature', 'confidence_flag': None, 'operatingSystem': ['Linux', 'Windows', 'Mac']}]\n"
     ]
    }
   ],
   "source": [
    "print(result)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To save the rusult, use the `save_result(out_path, result)` funtion. `out_path` is the path where the `result` will be saved."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Result saved as run_3.json\n"
     ]
    }
   ],
   "source": [
    "save_result('run_3.json', result)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_feaures = results_to_table(result)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###  Simple Query Example 2:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operation=\"Formatting\"&page=39&format=json\n",
      "Requests finished.\n",
      "Number of tools found: 387\n"
     ]
    }
   ],
   "source": [
    "sample_filters = ['operation=\"Formatting\"']\n",
    "filters = sample_filters\n",
    "result = get_all_pages(filters)\n",
    "\n",
    "print(\"Number of tools found: %d\"%len(result))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Case 1: ETL tools discovery\n",
    "\n",
    "We will query bio.tools using a set of EDAM terms that have being obtained after a mapping with ZOOMA.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Step 1: parsing zooma results"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "ZOOMA results must be in a csv format.The colunm names, in this order, must be: `keyword`,`iri`,`label` and `confidence`. An example of a valid: \n",
    "\n",
    "|keyword            |iri                                   |label                      |confidence|\n",
    "|-------------------|--------------------------------------|---------------------------|----------|\n",
    "|Ontology annotation|http://edamontology.org/operation_0226|Annotation                 |MEDIUM    |\n",
    "|Ontology annotation|                                      |                           |          |\n",
    "|semantic annotation|                                      |                           |          |\n",
    "|semantic annotation|http://edamontology.org/operation_3778|Text annotation            |MEDIUM    |\n",
    "|Text mining        |http://edamontology.org/topic_0218    |Natural language processing|GOOD      |\n",
    "\n",
    "Raw: \n",
    "\n",
    "```\n",
    "keyword,iri,label,confidence\n",
    "Ontology annotation,http://edamontology.org/operation_0226,Annotation,MEDIUM\n",
    "Ontology annotation,,,\n",
    "semantic annotation,,,\n",
    "semantic annotation,http://edamontology.org/operation_3778,Text annotation,MEDIUM\n",
    "Text mining,http://edamontology.org/topic_0218,Natural language processing,GOOD\n",
    "```\n",
    "\n",
    "The function `parse_zooma_results` parses this csv and returns a list of terms suitable for the querying."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "For a mapping of keyword related with ETL against EDAM, we got the ZOOMA results in the path `ETL_EDAM_curated.csv`."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['data_2337', 'data_0949', 'operation_3907', 'topic_3489', 'data_2337', 'topic_3572', 'operation_0336', 'operation_0335', 'operation_3434', 'operation_1812', 'data_0842', 'data_0006', 'topic_3366']\n",
      "{'data_2337': 'Resource metadata', 'data_0949': 'Workflow metadata', 'operation_3907': 'Information extraction', 'topic_3489': 'Database management', 'topic_3572': 'Data quality management', 'operation_0336': 'Format validation', 'operation_0335': 'Formatting', 'operation_3434': 'Conversion', 'operation_1812': 'Parsing', 'data_0842': 'Identifier', 'data_0006': 'Data', 'topic_3366': 'Data integration and warehousing'}\n"
     ]
    }
   ],
   "source": [
    "terms_file='ETL_EDAM_curated.csv'\n",
    "ETL_edam_terms, terms_label_ETL = parse_zooma_results(terms_file)\n",
    "print(ETL_edam_terms)\n",
    "print(terms_label_ETL)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Step 2: doing the query. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The function `query_for_EDAM_terms` does the quering for the previously obtained list of EDAM terms. It resturns two kind of outputs: general and detailed. Each is a dictionary of the form: `{<term>: <dataframe of results>}`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?dataTypeID=\"data_2337\"&page=2&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?dataTypeID=\"data_0949\"&page=1&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_3907\"&page=1&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?topicID=\"topic_3489\"&page=19&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?topicID=\"topic_3572\"&page=18&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_0336\"&page=2&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_0335\"&page=36&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_3434\"&page=3&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_1812\"&page=21&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?dataTypeID=\"data_0842\"&page=8&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?dataTypeID=\"data_0006\"&page=34&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?topicID=\"topic_3366\"&page=8&format=json\n",
      "Requests finished.\n"
     ]
    }
   ],
   "source": [
    "ETL_results_general, ETL_results_detailed  = query_for_EDAM_terms(ETL_edam_terms)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We save the result of each term in a separated file inside the `outputs/ETL` directory using the function `save_results`."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "path_output_ETL = \"outputs/ETL\"\n",
    "Path(path_output_ETL).mkdir(parents=True, exist_ok=True)\n",
    "save_results(ETL_results_general, terms_label_ETL, path_output_ETL)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Total number of terms"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "12"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(ETL_results_general.keys())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Tools per term"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'data_0006': 344,\n",
       " 'data_0842': 86,\n",
       " 'data_0949': 4,\n",
       " 'data_2337': 14,\n",
       " 'operation_0335': 364,\n",
       " 'operation_0336': 23,\n",
       " 'operation_1812': 210,\n",
       " 'operation_3434': 35,\n",
       " 'operation_3907': 6,\n",
       " 'topic_3366': 79,\n",
       " 'topic_3489': 187,\n",
       " 'topic_3572': 181}"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "count_tools_per_term(ETL_results_general)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Lists of tool names per term\n",
    "This is handy for validation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "tools_per_term_ETL = tools_per_term(ETL_results_general)\n",
    "tools_per_term_ETL = {terms_label_ETL[term]:tools_per_term_ETL[term] for term in tools_per_term_ETL.keys()}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We save the list of tools for each term in separated txt files."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "save_lists_tools(tools_per_term_ETL, path_output_ETL)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Merged tools lists\n",
    "All tools discovered merged in a single set"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'cytoscape web',\n",
       " 'sms',\n",
       " 'bionode-sra',\n",
       " 'fqduplicate',\n",
       " 'arrowsam',\n",
       " 'aops-svm',\n",
       " 'bam_to_sam',\n",
       " 'readw',\n",
       " 'tbl2knowngene',\n",
       " 'clean reads',\n",
       " 'noisy_oneclick',\n",
       " 'pdb2cle',\n",
       " 'gucal',\n",
       " 'mousebytes',\n",
       " 'getdifference',\n",
       " 'phred',\n",
       " 'hgv_gpass',\n",
       " 'metabosearch',\n",
       " 'fasta_to_phylip',\n",
       " 'gem-net',\n",
       " 'genenames.org',\n",
       " 'affxparser',\n",
       " 'snappy',\n",
       " 'marmoset',\n",
       " 'dataanalysis2tpp',\n",
       " 'asm2ace',\n",
       " 'diversityagents',\n",
       " 'phyd3',\n",
       " 'biobtree',\n",
       " 'biocworkflowtools',\n",
       " 'extended circadian harmonic oscillators (echo)',\n",
       " 'fconverter',\n",
       " 'jmzreader',\n",
       " 'hgv_pass',\n",
       " 'mesophotic.org',\n",
       " 'bioscripts.convert',\n",
       " 'cytomegalovirus database (cmvdb)',\n",
       " 'mdpbiome',\n",
       " 'stellar',\n",
       " 'prob2plinkbig',\n",
       " 'pride converter',\n",
       " 'imex',\n",
       " 'shutu',\n",
       " 'bioalcidae',\n",
       " 'mubd-decoymaker 2.0',\n",
       " 'the flux simulator',\n",
       " 'rtcga',\n",
       " 'svdetect_bam_preprocessing_pairs',\n",
       " 'readseq',\n",
       " 'biojs-vis-feature',\n",
       " 'circbase',\n",
       " 'filtercontrol',\n",
       " 'annotationtools',\n",
       " 'cytoconverter',\n",
       " 'clusterizebyslidingwindows',\n",
       " 'clc_assembler',\n",
       " 'selecthitscore',\n",
       " 'mkdom',\n",
       " 'myo-reg',\n",
       " 'genecards',\n",
       " 'masqot-gui',\n",
       " 'aging mouse brain',\n",
       " 'rare',\n",
       " 'gdtools_mask',\n",
       " 'rgestlibcomp',\n",
       " 'dig',\n",
       " 'bam2roi',\n",
       " 'compare_taxofile',\n",
       " 'davidquery',\n",
       " 'clc_unmapped_reads',\n",
       " 'clustering_resume',\n",
       " 'evidence finder',\n",
       " 'pchopper',\n",
       " 'customprodb',\n",
       " 'image sxm',\n",
       " 'sos',\n",
       " 'diana-mirpath',\n",
       " 'batchqc',\n",
       " 'mzstar',\n",
       " 'cellhts2',\n",
       " 's2o',\n",
       " 'gatk_depth_of_coverage',\n",
       " 'compareoverlappingsmallquery',\n",
       " 'sroogle',\n",
       " 'cafe variome',\n",
       " 'trimadaptor',\n",
       " 'imiss',\n",
       " 'sequanix',\n",
       " 'getnumber',\n",
       " 'pint',\n",
       " 'biojs-io-blast',\n",
       " 'indices_diversity',\n",
       " 'ophid',\n",
       " 'beaddatapackr',\n",
       " 'xde',\n",
       " 'out2summary',\n",
       " 'seal',\n",
       " 'void editor',\n",
       " 'proteomicsannotationhubdata',\n",
       " 'biomodels database converters',\n",
       " 'pyradiomics',\n",
       " 'bel2abm',\n",
       " 'sonoviewer',\n",
       " 'alias',\n",
       " 'mcrestimate',\n",
       " 'html4blast',\n",
       " 'eximir',\n",
       " 'encodexplorer',\n",
       " 'sdg',\n",
       " 'ngsreports',\n",
       " 'cosinor',\n",
       " 'uniprotr',\n",
       " 'prebs',\n",
       " 'blobtoolkit',\n",
       " 'machado',\n",
       " 'mircomp',\n",
       " 'sabre',\n",
       " 'plier',\n",
       " 'odonate phenotypic database',\n",
       " 'mascot2xml',\n",
       " 'linkprep',\n",
       " 'mmcif dictionary suite',\n",
       " 'pandoc scholar',\n",
       " 'rqc',\n",
       " 'mtbgt',\n",
       " 'svdetect_compare_step',\n",
       " 'international protein index (ipi)',\n",
       " 'mothur_krona',\n",
       " 'aneuvis',\n",
       " 'dr-thermo',\n",
       " 'myseq',\n",
       " 'qcmetrics',\n",
       " 'picard_samtofastq',\n",
       " 'msaccess',\n",
       " 'mosdepth',\n",
       " 'ordino',\n",
       " 'qvz',\n",
       " 'clc_extract_consensus',\n",
       " 'massarray',\n",
       " 'cyrest',\n",
       " 'recon',\n",
       " 'hms-ics',\n",
       " 'plato',\n",
       " 'fungenes',\n",
       " 'createcontrolsubset',\n",
       " 'mbcb',\n",
       " 'irods',\n",
       " 'printsextract',\n",
       " 'gemerr',\n",
       " 'judi',\n",
       " 'chipper',\n",
       " 'dsrc',\n",
       " 'bisque',\n",
       " 'cpfp',\n",
       " 'mirspring',\n",
       " 'biomaj',\n",
       " 'baalchip',\n",
       " 'presartools',\n",
       " 'martview',\n",
       " 'structure_threader',\n",
       " 'velvet',\n",
       " 'online chemical modeling environment',\n",
       " 'gpress',\n",
       " 'irefscape',\n",
       " 'wheatis',\n",
       " 'tophat2',\n",
       " 'bactag',\n",
       " 'rdrf',\n",
       " 'gdtools_subtract',\n",
       " 'fas2nex',\n",
       " 'genepattern',\n",
       " 'pcaviz',\n",
       " 'clima 2',\n",
       " 'tileqc',\n",
       " 'rmsikeyion',\n",
       " 'experimenthubdata',\n",
       " 'msgraph',\n",
       " 'scry',\n",
       " 'biojs-io-clustal',\n",
       " 'getintrons',\n",
       " 'hgv_linktogprofile',\n",
       " 'dvd',\n",
       " 'grok',\n",
       " 'rdp pyrosequencing pipeline',\n",
       " 'plotcoverage',\n",
       " 'sbml2latex',\n",
       " 'fastqz',\n",
       " 'vader',\n",
       " 'yarn',\n",
       " 'roi_report',\n",
       " 'deseq',\n",
       " 'arraytools',\n",
       " 'arrayxpath',\n",
       " 'vhcub',\n",
       " 't-psi-c',\n",
       " 'xia2',\n",
       " 'fqseqpair',\n",
       " 'bise bioimage informatics search engine',\n",
       " 'abi2xml',\n",
       " 'clc_sequence_info',\n",
       " 'abarray',\n",
       " 'lucy',\n",
       " 'escdb',\n",
       " 'tableparser',\n",
       " 'pymzml',\n",
       " 'peff',\n",
       " 'fqseqname',\n",
       " 'sratailor',\n",
       " 'writerestohtml',\n",
       " 'rentrez',\n",
       " 'compss',\n",
       " 'msdb',\n",
       " 't2d converter',\n",
       " 'dasir',\n",
       " 'gmap',\n",
       " 'noreturn',\n",
       " 'slinky',\n",
       " 'restrictfromsize',\n",
       " 'pyteomics',\n",
       " 'ccpdb',\n",
       " 'lea',\n",
       " 'geecc',\n",
       " 'volare',\n",
       " 'idbac',\n",
       " 'snm',\n",
       " 'gdtools_intersect',\n",
       " 'chilin',\n",
       " 'flowai',\n",
       " 'pcrclipreads',\n",
       " 'getdistance',\n",
       " 'mapperanalyzer',\n",
       " 'gatk_base_recalibrator',\n",
       " 'gdtools_apply',\n",
       " 'genetic and genomic information system (gnpis)',\n",
       " 'mesbl gc-ms metabolite peak database',\n",
       " 'flower',\n",
       " 'eegnet',\n",
       " 'melad',\n",
       " 'cleantranscriptfile',\n",
       " 'export',\n",
       " 'seamis',\n",
       " 'jmztab-m',\n",
       " 'cellfishing.jl',\n",
       " 'psmd',\n",
       " 'frogs_clusters_stat',\n",
       " 'pannbuilder',\n",
       " 'compareoverlappingsmallref',\n",
       " 'vmd store',\n",
       " 'alyx',\n",
       " 'kegdraw 0.1.14 beta',\n",
       " 'lcmix',\n",
       " 'apltomgfconverter',\n",
       " 'protmiscuity',\n",
       " 'wiff2dta',\n",
       " 'cov2html',\n",
       " 'skimmr',\n",
       " 'biomodels database',\n",
       " 'colorgff',\n",
       " 'tooldog',\n",
       " 'sickle',\n",
       " 'combigor',\n",
       " 'readqpcr',\n",
       " 'andes',\n",
       " 'affyqcreport',\n",
       " 'nanoj-squirrel',\n",
       " 'bioextract',\n",
       " 'agilepairedendreadscombiner',\n",
       " 'starank',\n",
       " 'bioproject',\n",
       " 'croco',\n",
       " 'epivizrdata',\n",
       " 'cellprofiler',\n",
       " 'idconvert',\n",
       " 'semdist',\n",
       " 'iconvert',\n",
       " 'collect_mgf',\n",
       " '.net bio',\n",
       " 'seqstrap',\n",
       " 'disgenet',\n",
       " 'catana',\n",
       " 'qualifier',\n",
       " 'imgt ligm-db',\n",
       " 'cytoml',\n",
       " 'mbv',\n",
       " 'gmap_iit_store',\n",
       " 'mapi',\n",
       " 'rgalaxy',\n",
       " 'qseq2fastq',\n",
       " 'reformat_snpeff_for_syntview',\n",
       " 'grs',\n",
       " 'gtz',\n",
       " 'seqcontrol',\n",
       " 'modifygenomiccoordinates',\n",
       " 'rems',\n",
       " 'imagej (nih)',\n",
       " 'doc2hpo',\n",
       " 'rre',\n",
       " 'ngs-qc generator',\n",
       " 'goby framework',\n",
       " 'gatk_variant_apply_recalibration',\n",
       " 'mres2x',\n",
       " 'cshl_fastx_artifacts_filter',\n",
       " 'expression profiler',\n",
       " 'cohorts',\n",
       " 'ptmphinder',\n",
       " 'beadarray',\n",
       " 'encyclopedia',\n",
       " 'metabolic',\n",
       " 'mdqc',\n",
       " 'nextbio gene-centric data',\n",
       " 'pappso',\n",
       " 'ddbj',\n",
       " 'geosubmission',\n",
       " 'bacdiver: a programmatic interface for bacdive',\n",
       " 'cornai',\n",
       " 'amr',\n",
       " 'goslin',\n",
       " 'cellbaser',\n",
       " 'mastervar2vcf4.1',\n",
       " 'abi2link',\n",
       " 'ebi patent sequence database',\n",
       " 'tomahaqcompanion',\n",
       " 'spades',\n",
       " 'covidmine',\n",
       " 'tximport',\n",
       " 'g-sqz',\n",
       " 'seqtypedetect',\n",
       " 'ciona robusta anatomy and development ontology',\n",
       " 'plaza monocots',\n",
       " 'taxonomy_summary',\n",
       " 'boiler',\n",
       " 'bio-formats',\n",
       " 'annovar_annotate_variation',\n",
       " 'modern research data portal',\n",
       " 'affygcqc',\n",
       " 'dat2mgf',\n",
       " 'phildb',\n",
       " 'tagdust',\n",
       " 'regate',\n",
       " 'compareoverlapping_parallel',\n",
       " 'u-ant',\n",
       " 'fgnet',\n",
       " 'arteria',\n",
       " 'nanostringqcpro',\n",
       " 'graph extract',\n",
       " 'koffi',\n",
       " 'ehrtemporalvariability',\n",
       " 'strictlyincludegff',\n",
       " 'ncbi resources',\n",
       " 'multperf',\n",
       " 'flowcal',\n",
       " 'lnkg2lmlnk',\n",
       " 'phyloligo',\n",
       " 'pipe-t',\n",
       " 'mergingdataclusterize',\n",
       " 'biojs-io-sam',\n",
       " 'cshl_fastx_quality_statistics',\n",
       " 'cytogps',\n",
       " 'fmm',\n",
       " 'drug-herb-interaction',\n",
       " 'manor',\n",
       " 'merox',\n",
       " 'libmgf',\n",
       " 'opossom',\n",
       " 'procon (proteomics conversion tool)',\n",
       " 'sepath',\n",
       " 'fastproject',\n",
       " 'mqc',\n",
       " 'chemfp',\n",
       " 'xvector',\n",
       " 'annovar_summarize',\n",
       " 'mlda',\n",
       " 'countnumber',\n",
       " 'gaggle',\n",
       " 'roi_pca',\n",
       " 'scaleus',\n",
       " 'pyronoise',\n",
       " 'bamhash',\n",
       " 'docknmine',\n",
       " 'seedgff',\n",
       " 'rawtools',\n",
       " 'mspire_mspire-sequest',\n",
       " 'qiime artifact extractor (qax)',\n",
       " 'gosim',\n",
       " 'jacks',\n",
       " 'diversity workbench',\n",
       " 'chipenrich',\n",
       " 'jaspextract',\n",
       " 'export_one_data',\n",
       " 'gatk_variant_annotator',\n",
       " 'genomehubs',\n",
       " 'stampy_mapper',\n",
       " 'hgv_lps',\n",
       " 'pisa-tree',\n",
       " 'gwas catalog',\n",
       " 'converttranscriptfile',\n",
       " 'variationget',\n",
       " 'hgv_beam',\n",
       " 'ngscheckmate',\n",
       " 'the hymenoptera genome database (hgd)',\n",
       " 'cbs das protein viewer',\n",
       " 'idmappingretrieval',\n",
       " 'dropclust',\n",
       " 'annotationhub',\n",
       " 'dose',\n",
       " 'marine metagenomics portal',\n",
       " 'cct',\n",
       " 'bridgedb api',\n",
       " 'methylaid',\n",
       " 'motorneuron',\n",
       " 'biosystems',\n",
       " 'seqassem 200807- molecular sequence assembly',\n",
       " 'kic',\n",
       " 'granvil',\n",
       " 'roi_plot_thumbnails',\n",
       " 'bsa4yeast',\n",
       " 'phen2gene',\n",
       " 'trimsequences',\n",
       " 'plotgenomecoverage',\n",
       " 'samtools_sort',\n",
       " 'xml2tsv',\n",
       " 'msquant',\n",
       " 'ipd project',\n",
       " 'converttranscriptfile_fastqtofasta',\n",
       " 'pybel',\n",
       " 'biolad-db',\n",
       " 'lava',\n",
       " 'multi experiment matrix',\n",
       " 'pklfilemerger',\n",
       " 'pcscs',\n",
       " 'metabase',\n",
       " 'cutadapt 1.12',\n",
       " 'ggost',\n",
       " 'samtools',\n",
       " 'emblmygff3',\n",
       " 'phplabware',\n",
       " 'hpo',\n",
       " 'xmlget',\n",
       " 'getdifferentialexpression',\n",
       " 'agilequalityfilter',\n",
       " 'bamstats',\n",
       " 'miamexpress',\n",
       " 'plasmidspades',\n",
       " 'rice genome hub',\n",
       " 'annotationfuncs',\n",
       " 'pindel',\n",
       " 'picardasmetrics',\n",
       " 'brisskit',\n",
       " 'mutanalyst',\n",
       " 'mass-up',\n",
       " 'lux score',\n",
       " 'breseq_bam2cov',\n",
       " 'sampledrift',\n",
       " 'getdistribution',\n",
       " 'frogs_biom_to_stdbiom',\n",
       " 'sdsw',\n",
       " 'bowtie',\n",
       " 'jvarkit ',\n",
       " 'multiphate',\n",
       " 'hgv_codingsnps',\n",
       " 'bcm search launcher sequence utilities',\n",
       " 'cipres',\n",
       " 'aclimatise',\n",
       " 'selectome',\n",
       " 'myvariant',\n",
       " 'mlibrary',\n",
       " 'sabund',\n",
       " 'celltag',\n",
       " 'jphyloio',\n",
       " 'adjustpkl',\n",
       " 'datasets2tools',\n",
       " 'pymsxml',\n",
       " 'biocyc knowledge library',\n",
       " 'oncowiki',\n",
       " 'oma',\n",
       " 'massypup',\n",
       " 'coeus',\n",
       " 'togostanza',\n",
       " 'reganalyst',\n",
       " 'bioqc',\n",
       " 'gdc',\n",
       " 'jtraml',\n",
       " 'gtdb-tk',\n",
       " 'prosextract',\n",
       " 'phenoseed',\n",
       " 'stampy_indexer',\n",
       " 'datadiscovery',\n",
       " 'ensembl apl',\n",
       " 'biobike',\n",
       " 'orcae',\n",
       " 'preprocesscore',\n",
       " 'chipwig',\n",
       " 'frogs_normalisation',\n",
       " 'proteinprospector',\n",
       " 'misfinder',\n",
       " 'mzwiff',\n",
       " 'groc-svs',\n",
       " 'rebaseextract',\n",
       " 'removeexonlines',\n",
       " 'restricttranscriptlist',\n",
       " 'matrix2png',\n",
       " 'clc_mapper_legacy',\n",
       " 'gatk_variant_recalibrator',\n",
       " 'gatk_validate_variants',\n",
       " 'damidseq_pipeline',\n",
       " 'harman',\n",
       " 'vcffilterjs',\n",
       " 'dna chromatogram explorer',\n",
       " 'changename',\n",
       " 'ngsutils',\n",
       " 'igvtools',\n",
       " 'vic',\n",
       " 'pslid',\n",
       " 'sartools_deseq2',\n",
       " 'ccdigest',\n",
       " 'extract_protein_names',\n",
       " 'tn-seq',\n",
       " 'sampler',\n",
       " 'bseqc',\n",
       " 'simextargid',\n",
       " 'picardinsertsize',\n",
       " 'tfanalysis',\n",
       " 'probamr',\n",
       " 'nmrml converter',\n",
       " 'baget',\n",
       " 'featureextract',\n",
       " 'rightfield',\n",
       " 'pobo',\n",
       " 'civicpy',\n",
       " 'genomernai',\n",
       " 'nels',\n",
       " 'wikipathways',\n",
       " 'tripal elasticsearch',\n",
       " 'lp-sda',\n",
       " 'interelementgff',\n",
       " 'biostrings',\n",
       " 'imgt mab-db',\n",
       " 'ms-isotope',\n",
       " 'qgis',\n",
       " 'pipelinedog',\n",
       " 'deformats',\n",
       " 'libr mocd',\n",
       " 'mlml2r',\n",
       " 'circos',\n",
       " 'chia-pet2',\n",
       " 'sam_to_bam',\n",
       " 'getwigdata',\n",
       " 'markduplicates (ip)',\n",
       " 'mzid',\n",
       " 'kddn',\n",
       " 'patternlab',\n",
       " 'metmatch',\n",
       " 'setuppromoter',\n",
       " 'open targets',\n",
       " 'jass',\n",
       " 'pileup_to_vcf',\n",
       " 'ucsc genome browser',\n",
       " 'mz2mgf',\n",
       " 'elixir elearning platform',\n",
       " 'nbrp resources',\n",
       " 'cshl_fastx_nucleotides_distribution',\n",
       " 'xtandem parser',\n",
       " 'soapnuke',\n",
       " 'genatlas',\n",
       " 'ensembl fugu genome browser',\n",
       " 'complex portal',\n",
       " 'sam2tsv',\n",
       " 'qcgwas',\n",
       " 'trimspace',\n",
       " 'scalce',\n",
       " 'bagel',\n",
       " 'gmap_build',\n",
       " 'otusfocus',\n",
       " 'fqcleaner',\n",
       " 'pubcrawler',\n",
       " 'luxr',\n",
       " 'gtool',\n",
       " 'vcfhead',\n",
       " 'gdtools_vcf2gd',\n",
       " 'msd file reader',\n",
       " 'haplotyper',\n",
       " 'uniprot.ws',\n",
       " 'cshl_fastx_collapser',\n",
       " 'meshr',\n",
       " 'bedtools_multiintersectbed',\n",
       " 'convert',\n",
       " 'pastmus',\n",
       " 'rcellminer',\n",
       " 'table',\n",
       " 'genosurf',\n",
       " 'biodb-loader',\n",
       " 'biomvcclass',\n",
       " 'art',\n",
       " 'zincpharmer',\n",
       " 'getletterdistribution1',\n",
       " 'reqon',\n",
       " 'x2h',\n",
       " 'multifit',\n",
       " 'uitagcreator',\n",
       " 'fit',\n",
       " 'selectbesthit',\n",
       " 'webspecmine',\n",
       " 'orfer',\n",
       " 'ehecregnet',\n",
       " 'evoker',\n",
       " 'hgv_sift',\n",
       " 'elimu-mdx',\n",
       " 'getreaddistribution',\n",
       " 'fusion',\n",
       " 'qq-snv',\n",
       " 'macsyfinder',\n",
       " 'prosight ptm',\n",
       " 'arraytv',\n",
       " 'gdhc',\n",
       " 'masswolf',\n",
       " 'ngsrelatev2',\n",
       " 'osdb api',\n",
       " 'gitar',\n",
       " 'changegfffeatures',\n",
       " 'fraggenescan',\n",
       " 'nwdisplay',\n",
       " 'atcoecis',\n",
       " 'bedgraphtobigwig',\n",
       " 'rdml',\n",
       " 'gatk_print_reads',\n",
       " 'paxtoolsr',\n",
       " 'seqassist',\n",
       " 'verifybamid',\n",
       " 'turbonorm',\n",
       " 'roi_table',\n",
       " 'rsat retrieve-ensembl-seq',\n",
       " 'picard_arrg',\n",
       " 'picard_replacesamheader',\n",
       " 'readdepth',\n",
       " 'lincs data portal 2.0',\n",
       " 'ctc',\n",
       " 'amelie',\n",
       " 'qmsat',\n",
       " 'msimageview',\n",
       " 'echmet',\n",
       " 'py-em',\n",
       " 'samjavascript',\n",
       " 'bedtools_mergebedgraph',\n",
       " 'bio2bel',\n",
       " 'frogs_clustering',\n",
       " 'bindingdb database of measured binding affinities',\n",
       " 'zlibbioc',\n",
       " 'cssp',\n",
       " 'evofreq',\n",
       " 'annotationforge',\n",
       " 'radar-base',\n",
       " 'rankoptimizer',\n",
       " 'pgxmine',\n",
       " 'sra',\n",
       " 'database of genes from ncbi refseq genomes',\n",
       " 'scan.upc',\n",
       " 'epigeec',\n",
       " 'medical relation extraction',\n",
       " 'crumbler',\n",
       " 'shortread',\n",
       " 'picard_fastqtosam',\n",
       " 'gblocks_oneclick',\n",
       " 'bedtools_coveragebed_counts',\n",
       " 'pri-cat',\n",
       " 'bel commons',\n",
       " 'ithanet',\n",
       " 'patentin',\n",
       " 'kobra',\n",
       " 'macs',\n",
       " 'hapzipper',\n",
       " 'pga',\n",
       " 'meigor',\n",
       " 'peakcalling_findpeaks',\n",
       " 'samtools_filter',\n",
       " 'cd-hit',\n",
       " 'aceparser',\n",
       " 'wft4galaxy',\n",
       " 'cuffdiff',\n",
       " 'lucegene',\n",
       " 'bedtools_bamtobed',\n",
       " 'chemmineob',\n",
       " 'scmap',\n",
       " 'gdtools_gd2vcf',\n",
       " 'hyppo-x',\n",
       " 'falcon server',\n",
       " 'rdgidb',\n",
       " 'roi_sort',\n",
       " 'meqa',\n",
       " 'sctype',\n",
       " 'lfqc',\n",
       " 'srna',\n",
       " 'catalysis-hub.org',\n",
       " 'keggparser',\n",
       " 'seecer',\n",
       " 'petkit',\n",
       " 'database manager',\n",
       " '1000genomes vcf2ped',\n",
       " 'taxonomy_krona_chart',\n",
       " 'alliance genome resources',\n",
       " 'lca',\n",
       " 'coveragegff',\n",
       " 't-bio.info',\n",
       " 'entrez',\n",
       " 'bridgedbr',\n",
       " 'berex',\n",
       " 'nohtml',\n",
       " 'nthseqset',\n",
       " 'aqua-mer',\n",
       " 'complete arabidopsis transcriptome database',\n",
       " 'countreadgcpercent',\n",
       " 'gatk_variant_combine',\n",
       " 'extractcentralregions',\n",
       " 'frogs_upload_tar',\n",
       " 'geonode-tools4msp',\n",
       " 'pepxml2excel',\n",
       " 'gdtools_annotate',\n",
       " 'pyrocleaner',\n",
       " 'gatk_variant_select',\n",
       " 'mview (ebi)',\n",
       " 'bedtools_intersectbed',\n",
       " 'tap',\n",
       " 'stattarget',\n",
       " 'sting',\n",
       " 'genecite',\n",
       " 'dbbqs',\n",
       " 'quantest',\n",
       " 'chemminer',\n",
       " 'gdtools_union',\n",
       " 'picardgcbiasmetrics',\n",
       " 'spotsegmentation',\n",
       " 'phylogenize',\n",
       " 'multi-q',\n",
       " 'pyrates',\n",
       " 'souporcell',\n",
       " 'cuffcompare',\n",
       " 'kpath',\n",
       " 'snpeff',\n",
       " 'restrictsequencelist',\n",
       " 'got-10k',\n",
       " 'vcf-server',\n",
       " 'impact framework',\n",
       " 'togotable',\n",
       " 'bam_to_sam_parallel',\n",
       " 'em-dat',\n",
       " 'getwigdistance',\n",
       " 'maizesnpdb',\n",
       " 'wikidata',\n",
       " 'gmap_snpindex',\n",
       " 'vegmarks',\n",
       " 'bam2fastq',\n",
       " 'illuminaio',\n",
       " 'genie-sys',\n",
       " 'pacsy',\n",
       " 'drlumi',\n",
       " 'mergeomics',\n",
       " 'icorn',\n",
       " 'conservation archive',\n",
       " 'hipathdb',\n",
       " 'saltclass',\n",
       " 'glycan array dashboard',\n",
       " 'envisionquery',\n",
       " 'roi_metrics',\n",
       " 'metaarray',\n",
       " 'snagee',\n",
       " 'cancerclass',\n",
       " 'stategra',\n",
       " 'plant datadiscovery',\n",
       " 'moat',\n",
       " 'getexons',\n",
       " 'tripal mapviewer',\n",
       " 'genscalpel',\n",
       " 'picardhsmetrics',\n",
       " 'meme suite',\n",
       " 'decomptumor2sig',\n",
       " 'clc_mapping_info',\n",
       " 'roi_feature_projection',\n",
       " 'frogs_remove_chimera',\n",
       " 'genbank2gff3',\n",
       " 'cshl_fastx_clipper',\n",
       " 'sva',\n",
       " 'spnconverter',\n",
       " 'msgfdb2pepxml',\n",
       " 'idmappinganalysis',\n",
       " 'oncocnv',\n",
       " 'faidare',\n",
       " 'rnassess',\n",
       " 'rgraph2js',\n",
       " 'coryneregnet',\n",
       " 'yaqcaffy',\n",
       " 'pina',\n",
       " 'translate',\n",
       " 'clc_cas_to_sam',\n",
       " 'pads arsenal',\n",
       " 'famfetch',\n",
       " 'getsizes',\n",
       " 'sights',\n",
       " 'gatk_realigner_target_creator',\n",
       " 'missequel',\n",
       " 'doubletdetection',\n",
       " 'libcsam',\n",
       " 'gatk_unified_genotyper',\n",
       " 'chipmunk_chiphorde',\n",
       " 'computecoverage',\n",
       " 'tvt viewer',\n",
       " 'gnomad',\n",
       " 'biomformat',\n",
       " 'skewr',\n",
       " 'get-it',\n",
       " 'convert2annovar',\n",
       " 'pyms',\n",
       " 'crysplot',\n",
       " 'shrec',\n",
       " 'genotypeeval',\n",
       " 'listinputs',\n",
       " 'listannotation',\n",
       " 'metaspades',\n",
       " 'collapsereads',\n",
       " 'zr-mofs',\n",
       " 'gostats',\n",
       " 'toff',\n",
       " 'biomoby',\n",
       " 'spikeli',\n",
       " 'tsdd',\n",
       " 'romop',\n",
       " 'fastool',\n",
       " 'fqconvert',\n",
       " 'codcopy',\n",
       " 'picard_reordersam',\n",
       " 'plasmo',\n",
       " 'platecore',\n",
       " 'gdtools_simulate-mutations',\n",
       " 'psea',\n",
       " 'alignparse',\n",
       " 'biojs-vis-hgv',\n",
       " 'selectbytag',\n",
       " 'phemto',\n",
       " 'webprc',\n",
       " 'urqt',\n",
       " 'indexcov',\n",
       " 'vcfutils',\n",
       " 'motivevalidator',\n",
       " 'frogs_filters',\n",
       " 'pdbusqlextractor',\n",
       " 'chlamdb',\n",
       " 'gatk_haplotype_caller',\n",
       " 'diversityexsiccatae',\n",
       " 'prepareannotation',\n",
       " 'meffil',\n",
       " 'centiserver',\n",
       " 'gsnap',\n",
       " 'frogs_demultiplex',\n",
       " 'inparanoid',\n",
       " 'qualtofastq',\n",
       " 'useq',\n",
       " 'guidance2',\n",
       " 'tfextract',\n",
       " 'jrgui',\n",
       " 'cutgextract',\n",
       " 'mzr',\n",
       " 'biojs-io-annots',\n",
       " 'gfakluge',\n",
       " 'p2s',\n",
       " 'mimic-cxr',\n",
       " 'biocfilecache',\n",
       " 'blast_best_score',\n",
       " 'fastaextract',\n",
       " 'pertinint',\n",
       " 'diversityprojects',\n",
       " 'makestatschipseq',\n",
       " 'rbiopaxparser',\n",
       " 'seda',\n",
       " 'michip',\n",
       " 'annotatepeaks',\n",
       " 'drfit',\n",
       " 'gcocoa',\n",
       " 'reformat_fastq',\n",
       " 'query tabular',\n",
       " 'sammi',\n",
       " 'nci flicker',\n",
       " 'raw2mzdb',\n",
       " 'ognm',\n",
       " 'affyilm',\n",
       " 'unfinnigan',\n",
       " 'foldamerdb',\n",
       " 'clc_find_variations',\n",
       " 'mitre',\n",
       " 'mergemaid',\n",
       " 'biojs-io-gff',\n",
       " 'raiss',\n",
       " 'keggtranslator',\n",
       " 'bionode-fasta',\n",
       " 'phenx toolkit',\n",
       " 'risa',\n",
       " 'pdbj',\n",
       " 'proovread',\n",
       " 'sff workbench',\n",
       " 'mexcowalk',\n",
       " 'sig2biopax',\n",
       " 'mac-inmv-ssr',\n",
       " 'ordo',\n",
       " 'cuneiform',\n",
       " 'cellity',\n",
       " 'benchmarkncvtools',\n",
       " 'dicom',\n",
       " 'blastgui',\n",
       " 'fp-lims',\n",
       " 'xml2raf 1.02 -create raf maps from pdb xml files',\n",
       " 'findpeaks 4.0 (vancouver short read package)',\n",
       " 'syfpeithi',\n",
       " 'piqmie',\n",
       " 'agilefileconverter',\n",
       " 'psvs',\n",
       " 'rdmp',\n",
       " 'the immune epitope database (iedb)',\n",
       " 'odobd',\n",
       " 'axima2analyze',\n",
       " 'domainsignatures',\n",
       " 'nextgen utility scripts',\n",
       " 'imgt 3dstructure-db',\n",
       " 'biojs-io-fasta',\n",
       " 'gdtools_not-evidence',\n",
       " 'cshl_fastx_reverse_complement',\n",
       " 'seq-annot',\n",
       " 'st-sim',\n",
       " 'notab',\n",
       " 'mirnameconverter',\n",
       " 'animalnexus',\n",
       " 'mappinghsp4taxo',\n",
       " 'hiv-drlink',\n",
       " 'ageguess',\n",
       " 'consensuspathdb',\n",
       " 'one to three',\n",
       " 'mapstore',\n",
       " 'empusa',\n",
       " 'mergetranscriptlists',\n",
       " 'circrna',\n",
       " 'metaproteomeanalyzer',\n",
       " 'linearfold',\n",
       " 'solid_qual_boxplot',\n",
       " 'mint',\n",
       " 'tcs',\n",
       " 'staden io_lib',\n",
       " 'diversitycollection',\n",
       " 'genbank_convertor',\n",
       " 'ispider central',\n",
       " 'qcewas',\n",
       " 'gblocks',\n",
       " 'lion-web',\n",
       " 'estprep',\n",
       " 'acdc',\n",
       " 'bigwig_to_bedgraph',\n",
       " 'escape excel',\n",
       " 'nnnorm',\n",
       " 'compareoverlapping',\n",
       " 'measurementerror.cor',\n",
       " 'biocstyle',\n",
       " 'swath2stats',\n",
       " 'arrayquality',\n",
       " 'evex',\n",
       " 'hmmcopy',\n",
       " 'plot',\n",
       " 'codon usage similarity index (cousin)',\n",
       " 'annotategenes',\n",
       " 'dgva',\n",
       " 'robustrankaggregation',\n",
       " 'pygmql',\n",
       " 'gwasinspector',\n",
       " 'laj_1',\n",
       " 'model',\n",
       " 'american type culture collection (atcc)',\n",
       " 'aaindexextract',\n",
       " 'brm',\n",
       " 'pergola-web',\n",
       " 'ega',\n",
       " 'scope',\n",
       " 'amon',\n",
       " 'chemdataextractor',\n",
       " 'amos',\n",
       " 'orcid api',\n",
       " 'msconvert',\n",
       " 'hdcytodata',\n",
       " 'organismdbi',\n",
       " 'dtsgui',\n",
       " 'biojs-io-graduates',\n",
       " 'kronaextract',\n",
       " 'lsd 3.0',\n",
       " 'findpromoter',\n",
       " 'siseq',\n",
       " 'formatomatic',\n",
       " 'sortgff',\n",
       " 'fasthmm',\n",
       " 'graphweb',\n",
       " 'fastqvalidator',\n",
       " 'maxreport',\n",
       " 'fetch-taxonomic-ranks',\n",
       " 'annotationdbi',\n",
       " 'transview',\n",
       " 'thermorawfileparser',\n",
       " 'acdtrace',\n",
       " 'askomics',\n",
       " ...}"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "all_ETL_tools = merge_tools_lists(ETL_results_general)\n",
    "all_ETL_tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1307"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(all_ETL_tools)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## ETL related annotations count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>tool</th>\n",
       "      <th>ETL annotations count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1296</th>\n",
       "      <td>objtables</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>772</th>\n",
       "      <td>meme suite</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>272</th>\n",
       "      <td>idconvert</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1001</th>\n",
       "      <td>extract_archive_merge</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1268</th>\n",
       "      <td>ms-data-core-api</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>bcm search launcher sequence utilities</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>459</th>\n",
       "      <td>hgv_codingsnps</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>458</th>\n",
       "      <td>multiphate</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>457</th>\n",
       "      <td>jvarkit</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1306</th>\n",
       "      <td>noisy</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1307 rows  2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                        tool  ETL annotations count\n",
       "1296                               objtables                      6\n",
       "772                               meme suite                      4\n",
       "272                                idconvert                      3\n",
       "1001                   extract_archive_merge                      3\n",
       "1268                        ms-data-core-api                      3\n",
       "...                                      ...                    ...\n",
       "460   bcm search launcher sequence utilities                      1\n",
       "459                           hgv_codingsnps                      1\n",
       "458                               multiphate                      1\n",
       "457                                 jvarkit                       1\n",
       "1306                                   noisy                      1\n",
       "\n",
       "[1307 rows x 2 columns]"
      ]
     },
     "execution_count": 86,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "matches_tools = {}\n",
    "for tool in list(all_ETL_tools):\n",
    "    matches = 0\n",
    "    for l in tools_per_term_ETL.values():\n",
    "        if tool in list(l):\n",
    "            matches += 1\n",
    "    matches_tools[tool] = matches\n",
    "\n",
    "pd.DataFrame(list(matches_tools.items()), columns= ['tool','ETL annotations count']).sort_values('ETL annotations count', ascending=False)\n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Case 2: Ontology tools discovery"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In this case, the file of zooma results is `ontology_annotation_EDAM_curated_csv.csv`."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Step 1: parsing zooma results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['operation_0226', 'operation_3778', 'topic_0089', 'operation_3280']\n",
      "{'operation_0226': 'Annotation', 'operation_3778': 'Text annotation', 'topic_0089': 'ontology and terminology', 'operation_3280': 'Named-entity and concept recognition'}\n"
     ]
    }
   ],
   "source": [
    "terms_file='ontology_annotation_EDAM_curated_csv.csv'\n",
    "ontology_edam_terms, terms_label_ontology = parse_zooma_results(terms_file)\n",
    "print(ontology_edam_terms)\n",
    "print(terms_label_ontology)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Step 2: doing the query. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_0226\"&page=14&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_3778\"&page=5&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?topicID=\"topic_0089\"&page=29&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_3280\"&page=6&format=json\n",
      "Requests finished.\n"
     ]
    }
   ],
   "source": [
    "ontology_results_general, ontology_results_detailed  = query_for_EDAM_terms(ontology_edam_terms)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [],
   "source": [
    "path_output_ontology = \"outputs/ontology\"\n",
    "Path(path_output_ontology).mkdir(parents=True, exist_ok=True)\n",
    "save_results(ontology_results_general, terms_label_ontology, path_output_ontology)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Total number of terms"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "4"
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(ontology_results_general.keys())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Tools per term"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'operation_0226': 145,\n",
       " 'operation_3280': 61,\n",
       " 'operation_3778': 47,\n",
       " 'topic_0089': 304}"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "count_tools_per_term(ontology_results_general)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Lists of tool names per term\n",
    "This is handy for validation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "tools_per_term_ontology = tools_per_term(ontology_results_general)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "save_lists_tools(tools_per_term_ontology, path_output_ontology)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Merged tools lists\n",
    "All tools discovered merged in a single set"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'a4preproc',\n",
       " 'aber-owl',\n",
       " 'acela',\n",
       " 'aceparser',\n",
       " 'ads',\n",
       " 'agrigo',\n",
       " 'ahtpdb',\n",
       " 'aiso',\n",
       " 'ajia',\n",
       " 'alliance genome resources',\n",
       " 'amigo 2',\n",
       " 'amtdb',\n",
       " 'anabel',\n",
       " 'aniseed',\n",
       " 'annot',\n",
       " 'annotationfilter',\n",
       " 'annotsv',\n",
       " 'apache taverna',\n",
       " 'arachnoserver',\n",
       " 'aristo',\n",
       " 'asa-ps',\n",
       " 'aspgd',\n",
       " 'association viewer',\n",
       " 'bagel2',\n",
       " 'becas',\n",
       " 'beere',\n",
       " 'bel commons',\n",
       " 'bel2abm',\n",
       " 'belief',\n",
       " 'bgeedb',\n",
       " 'bio.tools',\n",
       " 'bio2bel',\n",
       " 'biobert',\n",
       " 'biocaddie',\n",
       " 'biocatalogue',\n",
       " 'bioecs',\n",
       " 'bioexcel genome sequence quality control workflow',\n",
       " 'biogateway',\n",
       " 'biojs-vis-gprofiler',\n",
       " 'biolabeler',\n",
       " 'biolit',\n",
       " 'biomoby',\n",
       " 'biomodels database',\n",
       " 'bionda - biomarker database',\n",
       " 'bioner',\n",
       " 'bionet-mining',\n",
       " 'bioportal',\n",
       " 'bior',\n",
       " 'bittersweet',\n",
       " 'biwalklda',\n",
       " 'bo-lstm',\n",
       " 'brisskit',\n",
       " 'bubastis',\n",
       " 'bugbuilder',\n",
       " 'cagefightr',\n",
       " 'calchas',\n",
       " 'candida genome database cgd',\n",
       " 'card',\n",
       " 'cargo',\n",
       " 'carmo',\n",
       " 'causalbuilder',\n",
       " 'caw',\n",
       " 'cbo',\n",
       " 'cddapp',\n",
       " 'cdfread',\n",
       " 'cedar ondemand',\n",
       " 'cellbaser',\n",
       " 'ceu mass mediator',\n",
       " 'charprotdb',\n",
       " 'chebi ws',\n",
       " 'chemdistiller',\n",
       " 'chimera',\n",
       " 'chipinfo',\n",
       " 'ciona robusta anatomy and development ontology',\n",
       " 'circular gene ontology (cirgo)',\n",
       " 'civicmine',\n",
       " 'classyfire',\n",
       " 'clinical metadata ontology (cmdo)',\n",
       " 'clinotator',\n",
       " 'cocitestats',\n",
       " 'coeus',\n",
       " 'cogs',\n",
       " 'collabonet',\n",
       " 'collector',\n",
       " 'combigor',\n",
       " 'compendium of protein lysine acetylation (cpla)',\n",
       " 'copheism',\n",
       " 'cpgavas',\n",
       " 'craft',\n",
       " 'crispy',\n",
       " 'crowdgo',\n",
       " 'cui2vec',\n",
       " 'dasir',\n",
       " 'david bioinformatics resources',\n",
       " 'dbcan',\n",
       " 'dbcrid',\n",
       " 'dcgor',\n",
       " 'ddbj',\n",
       " 'deca',\n",
       " 'decres',\n",
       " 'deeppheno',\n",
       " 'deeppvp',\n",
       " 'deepred',\n",
       " 'defume',\n",
       " 'delayedarray',\n",
       " 'descseq',\n",
       " 'diana-lncbase',\n",
       " 'diana-microt v5.0',\n",
       " 'diana-mirgen',\n",
       " 'dictybase update 2011',\n",
       " 'dictyexpress',\n",
       " 'digchem',\n",
       " 'digsee',\n",
       " 'dingo',\n",
       " 'disgenet',\n",
       " 'disqover',\n",
       " 'diversity workbench',\n",
       " 'diversitycollection',\n",
       " 'diversityreferences',\n",
       " 'diversityscientificterms',\n",
       " 'diversitytaxonnames',\n",
       " 'dnatraffic',\n",
       " 'do',\n",
       " 'doc2hpo',\n",
       " 'dquest',\n",
       " 'drar-cpi',\n",
       " 'drawglycan-snfg',\n",
       " 'drosophila dnase i footprint database',\n",
       " 'easy-hla',\n",
       " 'ebi search',\n",
       " 'edamdef',\n",
       " 'edamhasinput',\n",
       " 'edamhasoutput',\n",
       " 'edamisformat',\n",
       " 'edamisid',\n",
       " 'edamname',\n",
       " 'elmmo',\n",
       " 'emma2 ws',\n",
       " 'enrichment map cytoscape plugin',\n",
       " 'enrichplot',\n",
       " 'est2assembly',\n",
       " 'estscan',\n",
       " 'etm-db',\n",
       " 'europhenome',\n",
       " 'eventepi',\n",
       " 'evntextrc',\n",
       " 'exoscore',\n",
       " 'extract',\n",
       " 'facta plus',\n",
       " 'faf-drugs',\n",
       " 'fair',\n",
       " 'fairdatapoint',\n",
       " 'famplex',\n",
       " 'fasto',\n",
       " 'feelnc',\n",
       " 'ffpred',\n",
       " 'fhir',\n",
       " 'flowrepositoryr',\n",
       " 'flyode',\n",
       " 'flytf',\n",
       " 'fontcell',\n",
       " 'funcexplorer',\n",
       " 'funmappone',\n",
       " 'fusionanalyser',\n",
       " 'g-sesame',\n",
       " 'g:profiler',\n",
       " 'galactic',\n",
       " 'gbol',\n",
       " 'gcocoa',\n",
       " 'gdsarray',\n",
       " 'gedex',\n",
       " 'gemtractor',\n",
       " 'gene ontology (go)',\n",
       " 'gene ontology browser',\n",
       " 'genease',\n",
       " 'generic go term finder',\n",
       " 'generic go term mapper',\n",
       " 'geneview',\n",
       " 'genex_ontology',\n",
       " 'genfam',\n",
       " 'genome trax',\n",
       " 'genome3d',\n",
       " 'genomicscores',\n",
       " 'genosurf',\n",
       " 'geseq',\n",
       " 'gfbio',\n",
       " 'ggost',\n",
       " 'gimli',\n",
       " 'globalfungi',\n",
       " 'go enrichment analysis tool',\n",
       " 'go-elite',\n",
       " 'go2msig',\n",
       " 'go2vec',\n",
       " 'goa',\n",
       " 'goa_gubbar',\n",
       " 'goblet',\n",
       " 'gocats',\n",
       " 'godef',\n",
       " 'goeast',\n",
       " 'gofuncr',\n",
       " 'gogene',\n",
       " 'gomap',\n",
       " 'gomapman',\n",
       " 'gominer ws',\n",
       " 'goname',\n",
       " 'gonet',\n",
       " 'gonuts',\n",
       " 'gopubmed',\n",
       " 'gorilla',\n",
       " 'gosemsim',\n",
       " 'gosim',\n",
       " 'goslin',\n",
       " 'gostag',\n",
       " 'gostats',\n",
       " 'gotools',\n",
       " 'gotrapper',\n",
       " 'gpannotate',\n",
       " 'gprofiler',\n",
       " 'gpsdb',\n",
       " 'grasp',\n",
       " 'gsan',\n",
       " 'gsso',\n",
       " 'gsvec',\n",
       " 'guide',\n",
       " 'gwastools',\n",
       " 'hgt-id',\n",
       " 'hgv_david',\n",
       " 'hgv_linktogprofile',\n",
       " 'higdb',\n",
       " 'hio',\n",
       " 'hipathia',\n",
       " 'histone coder',\n",
       " 'hmtvar',\n",
       " 'holovir',\n",
       " 'hpo',\n",
       " 'hpo2go',\n",
       " 'hubmed',\n",
       " 'human gene nomenclature database (hgnc)',\n",
       " 'hyper',\n",
       " 'iambee',\n",
       " 'icd-10',\n",
       " 'ictnet2',\n",
       " 'ideal',\n",
       " 'ietd',\n",
       " 'ikap_cells',\n",
       " 'imgt, the international immunogenetics information system',\n",
       " 'impres',\n",
       " 'inaturalist',\n",
       " 'ipd project',\n",
       " 'iscan',\n",
       " 'isotopident',\n",
       " 'itextmine',\n",
       " 'ivariantguide',\n",
       " 'j-express',\n",
       " 'jmztab-m',\n",
       " 'jprogo',\n",
       " 'karma',\n",
       " 'kbase',\n",
       " 'kccg patient archive',\n",
       " 'kegganim',\n",
       " 'keggprofile',\n",
       " 'kinetoplastid genome resources (tritrypdb)',\n",
       " 'koffi',\n",
       " 'l2l',\n",
       " 'lailaps-qsm',\n",
       " 'lion-web',\n",
       " 'lipid data analyzer (lda)',\n",
       " 'lipid mini-on',\n",
       " 'list ontologies',\n",
       " 'litinspector',\n",
       " 'litminer',\n",
       " 'lobstahs',\n",
       " 'locdb',\n",
       " 'loinc2hpo',\n",
       " 'lungmap',\n",
       " 'macie',\n",
       " 'magecomet',\n",
       " 'magi',\n",
       " 'mambo',\n",
       " 'mamot',\n",
       " 'markerhub',\n",
       " 'marky',\n",
       " 'matscholar',\n",
       " 'mbrole',\n",
       " 'medextractr',\n",
       " 'medical relation extraction',\n",
       " 'meme suite',\n",
       " 'meshsim',\n",
       " 'mesophotic.org',\n",
       " 'messa',\n",
       " 'metdisease',\n",
       " 'metfamily',\n",
       " 'mg-rast',\n",
       " 'microbe browser',\n",
       " 'microtss',\n",
       " 'mir-prefer',\n",
       " 'monarch initiative',\n",
       " 'morphdb',\n",
       " 'mouse phenome database',\n",
       " 'mousebook',\n",
       " 'mpagenomics',\n",
       " 'mtgo-sc',\n",
       " 'mucor',\n",
       " 'multiloc2',\n",
       " 'mutalyzer',\n",
       " 'mvgst',\n",
       " 'myminer',\n",
       " 'nadfinder',\n",
       " 'nbdc rdf portal',\n",
       " 'nbrp resources',\n",
       " 'nemine',\n",
       " 'neuropp',\n",
       " 'neuroscience information framework',\n",
       " 'ngly1',\n",
       " 'nlimed',\n",
       " 'nlprot',\n",
       " 'nlr-parser',\n",
       " 'nmrml converter',\n",
       " 'noa',\n",
       " 'npact',\n",
       " 'odae',\n",
       " 'odg',\n",
       " 'odmltables',\n",
       " 'ols dialog',\n",
       " 'omadb',\n",
       " 'ometa',\n",
       " 'omim explorer',\n",
       " 'onassis',\n",
       " 'onclass',\n",
       " 'onesense',\n",
       " 'ontobrowser',\n",
       " 'ontocat',\n",
       " 'ontocount',\n",
       " 'ontoget',\n",
       " 'ontogetcommon',\n",
       " 'ontogetdown',\n",
       " 'ontogetobsolete',\n",
       " 'ontogetroot',\n",
       " 'ontogetsibs',\n",
       " 'ontogetup',\n",
       " 'ontoisobsolete',\n",
       " 'ontokin',\n",
       " 'ontologizer',\n",
       " 'ontology lookup service',\n",
       " 'ontoproc',\n",
       " 'ontoquest ws',\n",
       " 'ontoquestmain',\n",
       " 'ontotext',\n",
       " 'openebench',\n",
       " 'openflu',\n",
       " 'openms',\n",
       " 'ordo',\n",
       " 'organism.dplyr',\n",
       " 'orphadata',\n",
       " 'pageman',\n",
       " 'pannotator',\n",
       " 'pannzer',\n",
       " 'pannzer2',\n",
       " 'parameciumdb',\n",
       " 'pavian',\n",
       " 'pbg-ld',\n",
       " 'pcosbase',\n",
       " 'pcpheno',\n",
       " 'pdbj',\n",
       " 'pdzpepint',\n",
       " 'peakanalyzer',\n",
       " 'peregrine',\n",
       " 'phen2gene',\n",
       " 'phenohm',\n",
       " 'phenopolis',\n",
       " 'phenoscape',\n",
       " 'phenotator',\n",
       " 'pint',\n",
       " 'plan2l',\n",
       " 'plant experimental assay ontology (peao)',\n",
       " 'plantcircnet',\n",
       " 'plato plugin for autocomplete on ontologies',\n",
       " 'poeas',\n",
       " 'pogos',\n",
       " 'pombase',\n",
       " 'pomelo ii',\n",
       " 'ppr-ssm',\n",
       " 'predictprotein open',\n",
       " 'progenetix',\n",
       " 'proloc-go',\n",
       " 'propheno',\n",
       " 'protege 5.0',\n",
       " 'protein naming utility',\n",
       " 'pubtator',\n",
       " 'pubtator central',\n",
       " 'pure',\n",
       " 'pyiomica',\n",
       " 'pyradiomics',\n",
       " 'python video annotator',\n",
       " 'qsbio',\n",
       " 'qtl tableminer++(qtm)',\n",
       " 'quickgo',\n",
       " 'quin',\n",
       " 'ramigo',\n",
       " 'rast',\n",
       " 'ratt',\n",
       " 'rdavidwebservice',\n",
       " 'rdfscape',\n",
       " 'rdrf',\n",
       " 'relna',\n",
       " 'riken integrated database of mammals',\n",
       " 'ril-contour',\n",
       " 'rnaindel',\n",
       " 'robot',\n",
       " 'rols',\n",
       " 'roma',\n",
       " 'romop',\n",
       " 'rulego',\n",
       " 'samsa2',\n",
       " 'sbpks2',\n",
       " 'scdo',\n",
       " 'sci.ai',\n",
       " 'seaweb',\n",
       " 'seclaf',\n",
       " 'secstrannotator',\n",
       " 'semdist',\n",
       " 'seqexpress',\n",
       " 'seqhound',\n",
       " 'sequence similarity maps (ssm)',\n",
       " 'sevenc',\n",
       " 'shinygo',\n",
       " 'simconcept',\n",
       " 'simdef',\n",
       " 'sinopedia',\n",
       " 'skimmr',\n",
       " 'smog',\n",
       " 'snomed ct',\n",
       " 'snpnotes',\n",
       " 'soba: sequence ontology bioinformatics analysis',\n",
       " 'sparql',\n",
       " 'spex2',\n",
       " 'starpdb',\n",
       " 'sts-nlsp',\n",
       " 'superfamily',\n",
       " 'synteny browser',\n",
       " 'tableannotator',\n",
       " 'thbp',\n",
       " 'the brenda tissue ontology (bto)',\n",
       " 'the comparative toxicogenomics database (ctd)',\n",
       " 'the protein ontology',\n",
       " 'tm-imc',\n",
       " 'togogenome',\n",
       " 'tooldog',\n",
       " 'topgo',\n",
       " 'trackfind',\n",
       " 'treeio',\n",
       " 'trifle',\n",
       " 'tromer',\n",
       " 'ts-goea',\n",
       " 'typeinfo',\n",
       " 'umlsmapper',\n",
       " 'uniconsig',\n",
       " 'variant',\n",
       " 'varimerge',\n",
       " 'vastdb',\n",
       " 'vio',\n",
       " 'viralzone',\n",
       " 'viseago',\n",
       " 'vist',\n",
       " 'webanno',\n",
       " 'webapollo',\n",
       " 'wego',\n",
       " 'wego2.0',\n",
       " 'workflow4metabolomics',\n",
       " 'wsd',\n",
       " 'wtfgenes',\n",
       " 'xenbase',\n",
       " 'yeast metaboliner',\n",
       " 'yogy',\n",
       " 'zooma'}"
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "all_ontology_tools = merge_tools_lists(ontology_results_general)\n",
    "all_ontology_tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "476"
      ]
     },
     "execution_count": 82,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(all_ontology_tools)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [],
   "source": [
    "## Ontology related annotations count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>tool</th>\n",
       "      <th>ontology annotations count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>258</th>\n",
       "      <td>acela</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>229</th>\n",
       "      <td>ppr-ssm</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>craft</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>237</th>\n",
       "      <td>belief</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>yeast metaboliner</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>brisskit</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>arachnoserver</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>ontobrowser</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>ggost</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>474</th>\n",
       "      <td>facta plus</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>475 rows  2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                  tool  ontology annotations count\n",
       "258              acela                           3\n",
       "229            ppr-ssm                           3\n",
       "211              craft                           3\n",
       "237             belief                           2\n",
       "112  yeast metaboliner                           2\n",
       "..                 ...                         ...\n",
       "166           brisskit                           1\n",
       "164      arachnoserver                           1\n",
       "163        ontobrowser                           1\n",
       "161              ggost                           1\n",
       "474         facta plus                           1\n",
       "\n",
       "[475 rows x 2 columns]"
      ]
     },
     "execution_count": 87,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "matches_tools = {}\n",
    "for tool in list(all_ontology_tools):\n",
    "    matches = 0\n",
    "    for l in tools_per_term_ontology.values():\n",
    "        if tool in list(l):\n",
    "            matches += 1\n",
    "    matches_tools[tool] = matches\n",
    "\n",
    "pd.DataFrame(list(matches_tools.items()), columns= ['tool','ontology annotations count']).sort_values('ontology annotations count', ascending=False)\n",
    "    "
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
