{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# bio.tools querying"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Overview"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Code"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "import json\n",
    "import requests\n",
    "import argparse\n",
    "import yaml\n",
    "import biotools_parse as bp\n",
    "import pandas as pd\n",
    "\n",
    "# The call to get all bio.tools entries through the api is: https://bio.tools/api/tool/?format=json.\n",
    "# Unfortunately, the response is limited to n tools. To get the whole set, the next pages must be retrieve.\n",
    "# use \"next\" in the response to get succesive entries.\n",
    " \n",
    "base_call = \"https://bio.tools/api/tool/?format=json\"\n",
    "\n",
    "\n",
    "def make_request(URL):\n",
    "    try:\n",
    "        response = requests.get(URL)\n",
    "    except:\n",
    "        print('Could not make the request')\n",
    "        return\n",
    "    else:\n",
    "        response = json.loads(response.text)\n",
    "        return(response)\n",
    "\n",
    "def build_url(next_page, filters):\n",
    "    call_template = \"https://bio.tools/api/tool/?{attributes}{next_page}&format=json\"\n",
    "    if next_page:\n",
    "        next_page = \"&%s\"%next_page\n",
    "    else:\n",
    "        next_page = \"\"\n",
    "    attributes = '&'.join(filters)    \n",
    "    url = call_template.format(next_page=next_page, attributes=attributes)\n",
    "    return(url)\n",
    "\n",
    "\n",
    "def get_all_pages(filters):\n",
    "    res = []\n",
    "    next_page = \"page=1\"\n",
    "    print('Starting making the requests...')\n",
    "    while next_page:\n",
    "        response = make_request(build_url(next_page, filters))\n",
    "        print(\"Requesting: \" + build_url(next_page, filters), end='\\r')\n",
    "        res = res + response[\"list\"]\n",
    "        next_page = response[\"next\"]\n",
    "        if next_page:\n",
    "            next_page = next_page[1:]\n",
    "    print('\\nRequests finished.')\n",
    "    return(res)\n",
    "\n",
    "def save_result(out_path, result):\n",
    "    with open(out_path, 'w') as out:\n",
    "        json.dump(result, out)\n",
    "    print('Result saved as ' + out_path )\n",
    "    \n",
    "\n",
    "colnames_general = ['name', 'description', 'type', 'topic', 'input', 'output']\n",
    "colnames_detailed = [ 'name', 'description', 'version', 'type', 'topic', 'links', 'publication', 'download', 'inst_instr', 'test', 'src', 'os', 'input', 'output', 'dependencies', 'documentation', 'license', 'termsUse', 'contribPolicy', 'authors', 'repository']\n",
    "def results_to_table(result, colnames):\n",
    "    tools = bp.biotoolsToolsGenerator(result).instances\n",
    "    # Load features into table\n",
    "    colnames_features = colnames\n",
    "    df_dict = {name : [] for name in colnames_features}\n",
    "\n",
    "    for tool in tools:\n",
    "        for field in colnames_features:\n",
    "            df_dict[field].append(tool.__dict__.get(field))\n",
    "\n",
    "    df_features = pd.DataFrame.from_dict(df_dict)\n",
    "    return(df_features)\n",
    "\n",
    "\n",
    "def parse_zooma_results(input_file):\n",
    "    with open(input_file, 'r') as inp:\n",
    "        zooma_terms_df = pd.read_csv(input_file)\n",
    "        terms = zooma_terms_df['iri']\n",
    "        terms = [term.split('http://edamontology.org/')[1] for term in terms if type(term) == str]\n",
    "    return(terms)\n",
    "\n",
    "def build_filter(term):\n",
    "    filters_template =  {'topic': 'topicID=\"%s\"', 'format':'dataFormatID=\"%s\"', 'operation':'operationID=\"%s\"', 'data':'dataTypeID=\"%s\"'}\n",
    "    if 'topic' in term:\n",
    "        filters = filters_template['topic']%(term)\n",
    "    elif 'data' in term:\n",
    "        filters = filters_template['data']%(term)\n",
    "    elif 'operation' in term:\n",
    "        filters = filters_template['operation']%(term)\n",
    "    elif 'format' in term:\n",
    "        filters = filters_template['format']%(term)\n",
    "    return(filters)     \n",
    "\n",
    "def query_for_EDAM_terms(terms):\n",
    "    '''\n",
    "    Takes a list of EDAM terms and does a query for each\n",
    "    Return a dictionary of results of the form: {<term>: <dataframe of results>}\n",
    "    '''\n",
    "    results = dict()\n",
    "    results_detailed = dict()\n",
    "    # Iteratre through terms\n",
    "    for term in terms:\n",
    "        # Avoid duplicated queries\n",
    "        if term not in results.keys():\n",
    "            # Building the filters using the EDAM terms\n",
    "            filter_ = [build_filter(term)]\n",
    "            # Do the query\n",
    "            result = get_all_pages(filter_)\n",
    "            # Put result in table\n",
    "            result_df = results_to_table(result, colnames_general)\n",
    "            result_detailed_df = results_to_table(result, colnames_detailed)\n",
    "            # Put results table in dictionary with the results for the other terms\n",
    "            results[term] = result_df\n",
    "            results_detailed[term] = result_detailed_df\n",
    "    return(results, results_detailed)\n",
    "\n",
    "def count_tools_per_term(results):\n",
    "    count_tools_per_term = {term :len(results[term]) for term in results.keys()}\n",
    "    return(count_tools_per_term) \n",
    "\n",
    "def tools_per_term(results):\n",
    "    tools_per_term = {term:results[term]['name'] for term in results}  \n",
    "    return(tools_per_term)\n",
    "\n",
    "def merge_tools_lists(results):\n",
    "    tools_per_term_ = tools_per_term(results)\n",
    "    tools = set()\n",
    "    for term in tools_per_term_:\n",
    "        for tool in tools_per_term_[term]:\n",
    "            tools.add(tool)\n",
    "    return(tools)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The attributes chosen to filter the query must be passed to the `get_all_pages` function in a list. Each attribute must be a string of the form `name=signalp`. This string will be put as a parameter in the query URL as it is. Please, follow the [bio.tools API reference](https://biotools.readthedocs.io/en/latest/api_reference.html) to construct this parameters successfully."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Simple Query Example 1:"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The following is an example of a query of tools annotted with the EDAM topic \"Sequence composition, complexity and repeats\" and with the word \"game\"  in their description. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?topic=\"Sequence composition, complexity and repeats\"&description=\"game\"&page=1&format=json\n",
      "Requests finished.\n"
     ]
    }
   ],
   "source": [
    "sample_filters = ['topic=\"Sequence composition, complexity and repeats\"', 'description=\"game\"']\n",
    "filters = sample_filters\n",
    "result = get_all_pages(filters)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'elixirCommunity': [], 'collectionID': ['EMBOSS'], 'accessibility': None, 'community': None, 'topic': [{'term': 'Sequence composition, complexity and repeats', 'uri': 'http://edamontology.org/topic_0157'}], 'owner': 'EMBOSS', 'cost': 'Free of charge', 'relation': [], 'download': [{'url': 'http://emboss.open-bio.org/html/adm/ch01s01.html', 'note': None, 'version': None, 'type': 'Source code'}, {'url': 'http://emboss.open-bio.org/html/adm/ch01s01.html', 'note': None, 'version': None, 'type': 'Binaries'}], 'validated': 1, 'publication': [{'doi': '10.1016/S0168-9525(00)02024-2', 'note': None, 'version': None, 'pmid': None, 'type': ['Primary'], 'pmcid': None, 'metadata': {'title': 'EMBOSS: The European Molecular Biology Open Software Suite', 'abstract': '', 'citationCount': 4828, 'authors': [{'name': 'Rice P.'}, {'name': 'Longden L.'}, {'name': 'Bleasby A.'}], 'date': '2000-06-01T00:00:00Z', 'journal': 'Trends in Genetics'}}, {'doi': '10.1017/CBO9781139151399', 'note': None, 'version': None, 'pmid': None, 'type': ['Other'], 'pmcid': None, 'metadata': None}, {'doi': '10.1017/CBO9781139151405', 'note': None, 'version': None, 'pmid': None, 'type': ['Other'], 'pmcid': None, 'metadata': None}], 'homepage_status': 0, 'credit': [{'typeRole': ['Contributor'], 'name': 'EMBOSS Contributors', 'rorid': None, 'url': 'http://emboss.open-bio.org/html/use/pr01s01.html', 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': None, 'email': None}, {'typeRole': ['Provider'], 'name': 'EMBL-EBI', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Institute', 'email': None}, {'typeRole': ['Contributor'], 'name': 'UK MRC', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Funding agency', 'email': None}, {'typeRole': ['Contributor'], 'name': 'Wellcome Trust', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Funding agency', 'email': None}, {'typeRole': ['Contributor'], 'name': 'UK BBSRC', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Funding agency', 'email': None}, {'typeRole': ['Provider'], 'name': 'EMBOSS', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Institute', 'email': None}, {'typeRole': ['Support'], 'name': 'EMBOSS', 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': None, 'email': 'emboss@emboss.open-bio.org'}, {'typeRole': ['Primary contact'], 'name': None, 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Person', 'email': 'emboss@emboss.open-bio.org'}, {'typeRole': ['Primary contact'], 'name': None, 'rorid': None, 'url': None, 'gridid': None, 'note': None, 'fundrefid': None, 'orcidid': None, 'typeEntity': 'Person', 'email': 'emboss-bug@emboss.open-bio.org'}], 'biotoolsCURIE': 'biotools:chaos', 'elixirPlatform': [], 'elixirNode': [], 'version': ['r6'], 'elixir_badge': 0, 'homepage': 'http://emboss.open-bio.org/rel/rel6/apps/chaos.html', 'function': [{'note': None, 'input': [{'data': {'term': 'Nucleic acid sequence record', 'uri': 'http://edamontology.org/data_2887'}, 'format': []}], 'operation': [{'term': 'Sequence composition calculation', 'uri': 'http://edamontology.org/operation_0236'}], 'cmd': None, 'output': [{'data': {'term': 'Sequence composition plot', 'uri': 'http://edamontology.org/data_2166'}, 'format': []}]}], 'lastUpdate': '2018-12-10T12:58:52Z', 'otherID': [], 'description': 'Draw a chaos game representation plot for a nucleotide sequence.', 'toolType': ['Command-line tool'], 'editPermission': {'type': 'private', 'authors': []}, 'language': ['C'], 'link': [{'url': 'http://emboss.open-bio.org/html/adm/ch01s01.html', 'note': None, 'type': ['Repository']}], 'biotoolsID': 'chaos', 'additionDate': '2015-11-08T14:40:19Z', 'name': 'chaos', 'license': 'GPL-3.0', 'documentation': [{'url': 'http://emboss.open-bio.org/html/dev/ch01s01.html', 'note': None, 'type': ['Terms of use']}, {'url': 'http://emboss.open-bio.org/html/use/pr02s04.html', 'note': None, 'type': ['Citation instructions']}, {'url': 'http://emboss.open-bio.org/rel/rel6/apps/chaos.html', 'note': None, 'type': ['General']}], 'maturity': 'Mature', 'confidence_flag': None, 'operatingSystem': ['Linux', 'Windows', 'Mac']}]\n"
     ]
    }
   ],
   "source": [
    "print(result)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To save the rusult, use the `save_result(out_path, result)` funtion. `out_path` is the path where the `result` will be saved."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Result saved as run_3.json\n"
     ]
    }
   ],
   "source": [
    "save_result('run_3.json', result)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_feaures = results_to_table(result)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###  Simple Query Example 2:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operation=\"Formatting\"&page=39&format=json\n",
      "Requests finished.\n",
      "Number of tools found: 387\n"
     ]
    }
   ],
   "source": [
    "sample_filters = ['operation=\"Formatting\"']\n",
    "filters = sample_filters\n",
    "result = get_all_pages(filters)\n",
    "\n",
    "print(\"Number of tools found: %d\"%len(result))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Case 1: ETL tools discovery\n",
    "\n",
    "We will query bio.tools using a set of EDAM terms that have being obtained after a mapping with ZOOMA.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Step 1: parsing zooma results"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "ZOOMA results must be in a csv format.The colunm names, in this order, must be: `keyword`,`iri`,`label` and `confidence`. An example of a valid: \n",
    "\n",
    "|keyword            |iri                                   |label                      |confidence|\n",
    "|-------------------|--------------------------------------|---------------------------|----------|\n",
    "|Ontology annotation|http://edamontology.org/operation_0226|Annotation                 |MEDIUM    |\n",
    "|Ontology annotation|                                      |                           |          |\n",
    "|semantic annotation|                                      |                           |          |\n",
    "|semantic annotation|http://edamontology.org/operation_3778|Text annotation            |MEDIUM    |\n",
    "|Text mining        |http://edamontology.org/topic_0218    |Natural language processing|GOOD      |\n",
    "\n",
    "Raw: \n",
    "\n",
    "```\n",
    "keyword,iri,label,confidence\n",
    "Ontology annotation,http://edamontology.org/operation_0226,Annotation,MEDIUM\n",
    "Ontology annotation,,,\n",
    "semantic annotation,,,\n",
    "semantic annotation,http://edamontology.org/operation_3778,Text annotation,MEDIUM\n",
    "Text mining,http://edamontology.org/topic_0218,Natural language processing,GOOD\n",
    "```\n",
    "\n",
    "The function `parse_zooma_results` parses this csv and returns a list of terms suitable for the querying."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "For a mapping of keyword related with ETL against EDAM, we got the ZOOMA results in the path `ETL_EDAM_curated.csv`."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['data_2337', 'data_0949', 'operation_3907', 'topic_3489', 'data_2337', 'topic_3572', 'operation_0336', 'operation_0335', 'operation_3434', 'operation_1812', 'data_0842', 'data_0006', 'topic_3366']\n"
     ]
    }
   ],
   "source": [
    "terms_file='ETL_EDAM_curated.csv'\n",
    "ETL_edam_terms = parse_zooma_results(terms_file)\n",
    "print(ETL_edam_terms)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Step 2: doing the query. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The function `query_for_EDAM_terms` does the quering for the previously obtained list of EDAM terms. It resturns two kind of outputs: general and detailed. Each is a dictionary of the form: `{<term>: <dataframe of results>}`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?dataTypeID=\"data_2337\"&page=2&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?dataTypeID=\"data_0949\"&page=1&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_3907\"&page=1&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?topicID=\"topic_3489\"&page=19&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?topicID=\"topic_3572\"&page=18&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_0336\"&page=2&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_0335\"&page=36&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_3434\"&page=3&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_1812\"&page=21&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?dataTypeID=\"data_0842\"&page=8&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?dataTypeID=\"data_0006\"&page=34&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?topicID=\"topic_3366\"&page=8&format=json\n",
      "Requests finished.\n"
     ]
    }
   ],
   "source": [
    "ETL_results_general, ETL_results_detailed  = query_for_EDAM_terms(ETL_edam_terms)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Total number of terms"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "12"
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(ETL_results_general.keys())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Tools per term"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'data_0006': 344,\n",
       " 'data_0842': 86,\n",
       " 'data_0949': 4,\n",
       " 'data_2337': 14,\n",
       " 'operation_0335': 364,\n",
       " 'operation_0336': 23,\n",
       " 'operation_1812': 210,\n",
       " 'operation_3434': 35,\n",
       " 'operation_3907': 6,\n",
       " 'topic_3366': 79,\n",
       " 'topic_3489': 187,\n",
       " 'topic_3572': 181}"
      ]
     },
     "execution_count": 67,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "count_tools_per_term(ETL_results_general)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Lists of tool names per term\n",
    "This is handy for validation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'data_0006': 0                       fconverter\n",
       " 1                            scope\n",
       " 2            gatk_haplotype_caller\n",
       " 3        elixir elearning platform\n",
       " 4                          phaseme\n",
       "                   ...             \n",
       " 339                       osdb api\n",
       " 340                   biojs-io-gff\n",
       " 341              biojs-vis-feature\n",
       " 342                  biojs-vis-hgv\n",
       " 343    imgt hla sequence alignment\n",
       " Name: name, Length: 344, dtype: object,\n",
       " 'data_0842': 0                                              disgenet\n",
       " 1                                              disgenet\n",
       " 2                                               faidare\n",
       " 3                                                pbg-ld\n",
       " 4                                                pbg-ld\n",
       "                             ...                        \n",
       " 81                                                 evex\n",
       " 82                                         plaza dicots\n",
       " 83           database of genes from ncbi refseq genomes\n",
       " 84    bindingdb database of measured binding affinities\n",
       " 85              american type culture collection (atcc)\n",
       " Name: name, Length: 86, dtype: object,\n",
       " 'data_0949': 0    bise bioimage informatics search engine\n",
       " 1    bise bioimage informatics search engine\n",
       " 2                                  cuneiform\n",
       " 3                                     compss\n",
       " Name: name, dtype: object,\n",
       " 'data_2337': 0                               bio.tools sum\n",
       " 1                                       phyd3\n",
       " 2     bise bioimage informatics search engine\n",
       " 3     bise bioimage informatics search engine\n",
       " 4                                      mageri\n",
       " 5                                    groc-svs\n",
       " 6                                      velvet\n",
       " 7                                   cellalign\n",
       " 8                                     abarray\n",
       " 9                                    prophecy\n",
       " 10                                    molmeth\n",
       " 11                                macsyfinder\n",
       " 12                                macsyfinder\n",
       " 13                                       barc\n",
       " Name: name, dtype: object,\n",
       " 'operation_0335': 0           pandoc scholar\n",
       " 1                 jmztab-m\n",
       " 2                    vhcub\n",
       " 3            trans-nanosim\n",
       " 4               mousebytes\n",
       "               ...         \n",
       " 359                   lfqc\n",
       " 360             visibioweb\n",
       " 361                 phemto\n",
       " 362               jvarkit \n",
       " 363    1000genomes vcf2ped\n",
       " Name: name, Length: 364, dtype: object,\n",
       " 'operation_0336': 0             jmztab-m\n",
       " 1            objtables\n",
       " 2            objtables\n",
       " 3              aniseed\n",
       " 4              aniseed\n",
       " 5          pgconverter\n",
       " 6     ms-data-core-api\n",
       " 7            otusfocus\n",
       " 8               libmgf\n",
       " 9               hmmer3\n",
       " 10              hmmer3\n",
       " 11         validatordb\n",
       " 12     motivevalidator\n",
       " 13     motivevalidator\n",
       " 14    psi-mi validator\n",
       " 15            typeinfo\n",
       " 16          rightfield\n",
       " 17         void editor\n",
       " 18            acdtrace\n",
       " 19            acdvalid\n",
       " 20                acdc\n",
       " 21         cancerclass\n",
       " 22            nrg-cing\n",
       " Name: name, dtype: object,\n",
       " 'operation_1812': 0            pgxmine\n",
       " 1           wptherml\n",
       " 2             pcaviz\n",
       " 3            pastmus\n",
       " 4             nanite\n",
       "            ...      \n",
       " 205           cytoml\n",
       " 206      jaspextract\n",
       " 207    printsextract\n",
       " 208    cytoscape web\n",
       " 209      cutgextract\n",
       " Name: name, Length: 210, dtype: object,\n",
       " 'operation_3434': 0                     thermorawfileparser\n",
       " 1     procon (proteomics conversion tool)\n",
       " 2                                     art\n",
       " 3                               objtables\n",
       " 4                               objtables\n",
       " 5                     diversitytaxonnames\n",
       " 6                     diversitytaxonnames\n",
       " 7                   diversitydescriptions\n",
       " 8                     diversity workbench\n",
       " 9                     diversitycollection\n",
       " 10                    diversitycollection\n",
       " 11                               raw2mzdb\n",
       " 12                                xml2tsv\n",
       " 13                  extract_protein_names\n",
       " 14                              translate\n",
       " 15               three to one letter code\n",
       " 16                           three to one\n",
       " 17                           one to three\n",
       " 18                                 m2lite\n",
       " 19                                 jtraml\n",
       " 20                                 jtraml\n",
       " 21                              idconvert\n",
       " 22                                 hmmer3\n",
       " 23                                 hmmer3\n",
       " 24                             t-bio.info\n",
       " 25                             meme suite\n",
       " 26                             meme suite\n",
       " 27                           imagej (nih)\n",
       " 28                        nmrml converter\n",
       " 29                                  andes\n",
       " 30                                3seq_2d\n",
       " 31                                convert\n",
       " 32                             sbml2latex\n",
       " 33                                 biomaj\n",
       " 34                                 biomaj\n",
       " Name: name, dtype: object,\n",
       " 'operation_3907': 0                       oncowiki\n",
       " 1            mubd-decoymaker 2.0\n",
       " 2                           toff\n",
       " 3    medical relation extraction\n",
       " 4                         lp-sda\n",
       " 5                        pyrates\n",
       " Name: name, dtype: object,\n",
       " 'topic_3366': 0                              disgenet\n",
       " 1                              disgenet\n",
       " 2                                  mint\n",
       " 3                                  amon\n",
       " 4                             pisa-tree\n",
       "                     ...                \n",
       " 74                             martview\n",
       " 75                        venn diagrams\n",
       " 76                robustrankaggregation\n",
       " 77    international protein index (ipi)\n",
       " 78                  expression profiler\n",
       " Name: name, Length: 79, dtype: object,\n",
       " 'topic_3489': 0                       fhir\n",
       " 1                    zr-mofs\n",
       " 2      drug-herb-interaction\n",
       " 3               pads arsenal\n",
       " 4                      ophid\n",
       "                ...          \n",
       " 182                 genecite\n",
       " 183                   fresco\n",
       " 184                  pri-cat\n",
       " 185                    alias\n",
       " 186      expression profiler\n",
       " Name: name, Length: 187, dtype: object,\n",
       " 'topic_3572': 0                                    gazevisual-lib\n",
       " 1                                               amr\n",
       " 2                                              amon\n",
       " 3                                              ordo\n",
       " 4                                              ordo\n",
       "                            ...                     \n",
       " 176    seqassem 200807- molecular sequence assembly\n",
       " 177                                       proovread\n",
       " 178                                             tcs\n",
       " 179                                     idna-methyl\n",
       " 180                                     verifybamid\n",
       " Name: name, Length: 181, dtype: object}"
      ]
     },
     "execution_count": 72,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tools_per_term(ETL_results_general)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Merged tools lists\n",
    "All tools discovered merged in a single set"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'restrictfromsize',\n",
       " 'goslin',\n",
       " '.net bio',\n",
       " 'uniprotr',\n",
       " 'evidence finder',\n",
       " 'presartools',\n",
       " 'imex',\n",
       " 'zlibbioc',\n",
       " 'genscalpel',\n",
       " 'prophecy',\n",
       " 'pgxmine',\n",
       " 'complex portal',\n",
       " 'gaggle',\n",
       " 'fish - profile search',\n",
       " 'laj_1',\n",
       " 'mtbgt',\n",
       " 'sbmlmod',\n",
       " 'prosextract',\n",
       " 'amelie',\n",
       " 'bcm search launcher sequence utilities',\n",
       " 'pyrates',\n",
       " 'removeexonlines',\n",
       " 'annotationforge',\n",
       " 'genatlas',\n",
       " 'ngsutils',\n",
       " 'nanoj-squirrel',\n",
       " 'rightfield',\n",
       " 'mowserv',\n",
       " 'xml2raf 1.02 -create raf maps from pdb xml files',\n",
       " 'bsa4yeast',\n",
       " 'nrg-cing',\n",
       " 'adjustpkl',\n",
       " 'mappinghsp4taxo',\n",
       " 'got-10k',\n",
       " 'nextgen utility scripts',\n",
       " 'aggregate_scores_in_intervals',\n",
       " 'genbank2gff3',\n",
       " 'aceparser',\n",
       " 'haplotyper',\n",
       " 'biojs-io-graduates',\n",
       " 'deseq',\n",
       " 'snpeff',\n",
       " 'imgt ligm-db',\n",
       " 'hgv_beam',\n",
       " 'rre',\n",
       " 'pride toolsuite',\n",
       " 'keggtranslator',\n",
       " 'foldamerdb',\n",
       " 'plotgenomecoverage',\n",
       " 'gdc',\n",
       " 'mimic-cxr',\n",
       " 'pfind studio',\n",
       " 'gdtools_simulate-mutations',\n",
       " 'getnumber',\n",
       " 'cregis-q',\n",
       " 'maketssdist',\n",
       " 'fastqvalidator',\n",
       " 'glycan array dashboard',\n",
       " 'dgva',\n",
       " 'kobra',\n",
       " 'meme_fimo',\n",
       " 'frogs_filters',\n",
       " 'graphweb',\n",
       " 'opossom',\n",
       " 'jvarkit ',\n",
       " 'atcoecis',\n",
       " 'domainsignatures',\n",
       " 'admixr',\n",
       " 'biomoby',\n",
       " 'vegmarks',\n",
       " 'pgconverter',\n",
       " 'bcf_view_mpileup',\n",
       " 'marky',\n",
       " 'markduplicates (ip)',\n",
       " 'affyilm',\n",
       " 'bedtools_coveragebed_counts',\n",
       " 'spades',\n",
       " 'compare_taxofile',\n",
       " 'em-dat',\n",
       " 'nels',\n",
       " 'bioscripts.convert',\n",
       " 'skimmr',\n",
       " 'pertinint',\n",
       " 'gblocks',\n",
       " 'brm',\n",
       " 'magpie',\n",
       " 'nmrml converter',\n",
       " 'gnomad',\n",
       " 'lincs data portal 2.0',\n",
       " 'genecite',\n",
       " 'drlumi',\n",
       " 'marine metagenomics portal',\n",
       " 'changetagname',\n",
       " 'askomics',\n",
       " 'venn diagrams',\n",
       " 'massarray',\n",
       " 'lion-web',\n",
       " 'dsrc',\n",
       " 'siseq',\n",
       " 'proteomicsannotationhubdata',\n",
       " 'mpileup_vcf',\n",
       " 'reformat_fastq',\n",
       " 'hgv_ldtools',\n",
       " 'tdimpute',\n",
       " 'foresee',\n",
       " 'backtranseq (ebi)',\n",
       " 'hiv-drlink',\n",
       " 'starank',\n",
       " 'bel commons',\n",
       " 'nci flicker',\n",
       " 'frogs_preprocess',\n",
       " 'metmatch',\n",
       " 'taxonomy_summary',\n",
       " 'chemminer',\n",
       " 'nthseqset',\n",
       " 'vader',\n",
       " 'pdb2cle',\n",
       " 'irods',\n",
       " 'sigprofilermatrixgenerator',\n",
       " 'gtdb-tk',\n",
       " 'pvca',\n",
       " 'sra',\n",
       " 'gdtools_vcf2gd',\n",
       " 'pisa-tree',\n",
       " 'bam2fastq',\n",
       " 'checker',\n",
       " 'clean_ngs',\n",
       " 'rmsikeyion',\n",
       " 'crumbler',\n",
       " 'rdmp',\n",
       " 'pubcrawler',\n",
       " 'getsizes',\n",
       " 'cutadapt 1.12',\n",
       " 'falcon server',\n",
       " 'cshl_fastx_clipper',\n",
       " 'psea',\n",
       " 'cellity',\n",
       " 'pcrclipreads',\n",
       " 'gdtools_not-evidence',\n",
       " 'tn-seq',\n",
       " 'seq-annot',\n",
       " 'togostanza',\n",
       " 'preprocesscore',\n",
       " 'trnaviz',\n",
       " 'rare',\n",
       " 'lea',\n",
       " 'breseq_bam2cov',\n",
       " 'mz2mgf',\n",
       " 'cshl_fastx_quality_statistics',\n",
       " 'genetic and genomic information system (gnpis)',\n",
       " 'gdtools_subtract',\n",
       " 'fp-lims',\n",
       " 'amon',\n",
       " 'spotsegmentation',\n",
       " 'raw2mzdb',\n",
       " 'blastgui',\n",
       " 'gem-net',\n",
       " 'inparanoid',\n",
       " 'taxoptimizer',\n",
       " 'plot',\n",
       " 'compareoverlapping_parallel',\n",
       " 'tooldog',\n",
       " 'diark - a resource for eukaryotic genome research',\n",
       " 'brisskit',\n",
       " 'mfpaq',\n",
       " 'multiplierz',\n",
       " 'logiqa',\n",
       " 'genepattern',\n",
       " 'xvector',\n",
       " 'libcsam',\n",
       " 'plant datadiscovery',\n",
       " 'biocyc knowledge library',\n",
       " 'merox',\n",
       " 'get-it',\n",
       " 'bagel',\n",
       " 'findpeaks 4.0 (vancouver short read package)',\n",
       " 'aphirm toolkit',\n",
       " 'qcgwas',\n",
       " 'msmstests',\n",
       " 'bioalcidae',\n",
       " 'codcopy',\n",
       " 'hms-ics',\n",
       " 'gconvert',\n",
       " 'entrez',\n",
       " 'export_one_data',\n",
       " 'deepcircode',\n",
       " 'mapi',\n",
       " 'mergeslidingwindowsclusters',\n",
       " 'annotatepeaks',\n",
       " 'diversity workbench',\n",
       " 'customprodb',\n",
       " 'scaleus',\n",
       " 'convert2annovar',\n",
       " 'chipenrich',\n",
       " 'genbankr',\n",
       " 'compss',\n",
       " 'pappso',\n",
       " 'phylogenize',\n",
       " 'qualtofastq',\n",
       " 'query tabular',\n",
       " 'baget',\n",
       " 'eegnet',\n",
       " 'collapsereads',\n",
       " 'stategra',\n",
       " 'hgv_pass',\n",
       " 'database manager',\n",
       " 'sdg',\n",
       " 'mecat',\n",
       " 'koffi',\n",
       " 'multiphate',\n",
       " 't-bio.info',\n",
       " 'alignparse',\n",
       " 'biojs-io-clustal',\n",
       " 'meme_meme',\n",
       " 'fqconvert',\n",
       " 'genbank_to_gene',\n",
       " 'deformats',\n",
       " 'pga',\n",
       " 'organismdbi',\n",
       " 'protsav',\n",
       " 'multperf',\n",
       " 'mzxml2search',\n",
       " 'lucegene',\n",
       " 'arraytv',\n",
       " 'fastqz',\n",
       " 'bwtool 1.0 gamma',\n",
       " 'vic',\n",
       " 'diversityexsiccatae',\n",
       " 'hybrid-shrec',\n",
       " 'flower',\n",
       " 'solexaqa',\n",
       " 'maxreport',\n",
       " 'urqt',\n",
       " 'nextbio gene-centric data',\n",
       " 'getflanking',\n",
       " 'trimsequences',\n",
       " 'grok',\n",
       " 'clc_cas_to_sam',\n",
       " 'msimageview',\n",
       " 'coeus',\n",
       " 'noreturn',\n",
       " 'gatk_reduce_reads',\n",
       " 'harman',\n",
       " 'breseq',\n",
       " 'gwas catalog',\n",
       " 'barc',\n",
       " 'diana-mirpath',\n",
       " 'bankit',\n",
       " 'sickle',\n",
       " 'annotationhub',\n",
       " 'measurementerror.cor',\n",
       " 'hapzipper',\n",
       " 'lfqc',\n",
       " 'extract_protein_names',\n",
       " 'mint',\n",
       " 'sci-hub',\n",
       " 'cuneiform',\n",
       " 'cytoml',\n",
       " 'model',\n",
       " 'melad',\n",
       " 'myo-reg',\n",
       " 'embossdata',\n",
       " 'mpileup',\n",
       " 'stampy_indexer',\n",
       " 'diversitydescriptions',\n",
       " 'pindel2vcf',\n",
       " 'luxr',\n",
       " 'bise bioimage informatics search engine',\n",
       " 'out2summary',\n",
       " 'collect_mgf',\n",
       " 'guidance2',\n",
       " 'nospace',\n",
       " 'bam_to_sam',\n",
       " 'biobtree',\n",
       " 'syfpeithi',\n",
       " 'codon usage database',\n",
       " 'chipwig',\n",
       " 'tap',\n",
       " 'x2h',\n",
       " 'jass',\n",
       " 'extractcentralregions',\n",
       " 'frogs_affiliations_stat',\n",
       " 'open targets',\n",
       " 'express',\n",
       " 'mageri',\n",
       " 'blobtoolkit',\n",
       " 'gblocks_oneclick',\n",
       " 'annotationfuncs',\n",
       " 'moirai',\n",
       " 'seqcontrol',\n",
       " 'getwigprofile',\n",
       " 'martview',\n",
       " 'rnassess',\n",
       " 'cshl_princeton_fastx_barcode_splitter',\n",
       " 'aa2nt',\n",
       " 'gatk_variant_apply_recalibration',\n",
       " 'rdrf',\n",
       " 'chlamdb',\n",
       " 'imgt domaindisplay',\n",
       " 'reformat_snpeff_for_syntview',\n",
       " 'indexcov',\n",
       " 'blast_best_score',\n",
       " 'prebs',\n",
       " 'circrna',\n",
       " 'linx',\n",
       " 'bio2bel',\n",
       " 't2d converter',\n",
       " 'peakcalling_findpeaks',\n",
       " 'zincbind',\n",
       " 'cshl_fastx_trimmer',\n",
       " '1000genomes vcf2ped',\n",
       " 'bacdive ws',\n",
       " 'ciona robusta anatomy and development ontology',\n",
       " 'gcocoa',\n",
       " 'converttranscriptfile_fastqtofasta',\n",
       " 'converttranscriptfile',\n",
       " 'ccpdb',\n",
       " 'load_multifastq',\n",
       " 'selectbytag',\n",
       " 'picardgcbiasmetrics',\n",
       " 'picard_samtofastq',\n",
       " 'sam_to_bam',\n",
       " 'swath2stats',\n",
       " 'beaddatapackr',\n",
       " 'stampy_mapper',\n",
       " 'metaqtl',\n",
       " 'phred',\n",
       " 'roi_sort',\n",
       " 'batchqc',\n",
       " 'metaspades',\n",
       " 'echo',\n",
       " 'pklfilemerger',\n",
       " 'lamma',\n",
       " 'boiler',\n",
       " 'fqseqname',\n",
       " 'redcap',\n",
       " 'myvariant',\n",
       " 'spikeli',\n",
       " 'tcs',\n",
       " 'docknmine',\n",
       " 'noisy_oneclick',\n",
       " 'escape excel',\n",
       " 'nohtml',\n",
       " 'bio-formats',\n",
       " 'crazyquant',\n",
       " 'epigeec',\n",
       " 'damidseq_pipeline',\n",
       " 'vcffilterjs',\n",
       " 'master2pgsnp',\n",
       " 'wiff2dta',\n",
       " 'mdqc',\n",
       " 'gatk_indel_realigner',\n",
       " 'gostats',\n",
       " 'fqcleaner',\n",
       " 'vcfhead',\n",
       " 'goby framework',\n",
       " 'mlibrary',\n",
       " 'affyqcreport',\n",
       " 'aging mouse brain',\n",
       " 'aaindexextract',\n",
       " 'pergola-web',\n",
       " 'jrgui',\n",
       " 'cd-hit',\n",
       " 'rcellminer',\n",
       " 'annotationdbi',\n",
       " 'bamhash',\n",
       " 'roi_filter',\n",
       " 'clc_unmapped_reads',\n",
       " 'missequel',\n",
       " 'international protein index (ipi)',\n",
       " 'skewr',\n",
       " 'coveragegff',\n",
       " 'aops-svm',\n",
       " 'stanford microarray database (smd)',\n",
       " 'oma',\n",
       " 'patternlab',\n",
       " 'visibioweb',\n",
       " 'pybel',\n",
       " 'arteria',\n",
       " 'selectome',\n",
       " 'taxa',\n",
       " 'selecthitscore',\n",
       " 'featureextract',\n",
       " 'gfakluge',\n",
       " 'picard_fastqtosam',\n",
       " 'xml2tsv',\n",
       " 'evoker',\n",
       " 'grep_count',\n",
       " 'chemmineob',\n",
       " 'orcom',\n",
       " 'g-sqz',\n",
       " 'datab.io',\n",
       " 'genomernai',\n",
       " 'arrayxpath',\n",
       " 'ngscheckmate',\n",
       " 'phplabware',\n",
       " 'gdhc',\n",
       " 'jacks',\n",
       " 'pedcut',\n",
       " 'staden io_lib',\n",
       " 'imgt genefrequency',\n",
       " 'jtraml',\n",
       " 'gatk_variant_eval',\n",
       " 'dr-thermo',\n",
       " 'proovread',\n",
       " 'getdifference',\n",
       " 'changegfffeatures',\n",
       " 'multifit',\n",
       " 'gemreads',\n",
       " 'rgraph2js',\n",
       " 'gmap_snpindex',\n",
       " 'codon usage similarity index (cousin)',\n",
       " 'extract_archive_merge',\n",
       " 'hipathdb',\n",
       " 'gmap_build',\n",
       " 'fusion',\n",
       " 'meshr',\n",
       " 'fraggenescan',\n",
       " 'bedtools_genomecoveragebed_histogram',\n",
       " 'pint',\n",
       " 'diversityagents',\n",
       " 'unfinnigan',\n",
       " 'cleantranscriptfile',\n",
       " 'openms',\n",
       " 'quip',\n",
       " 'phenoseed',\n",
       " 'mview (ebi)',\n",
       " 'm-iolite',\n",
       " 'qcmetrics',\n",
       " 'rentrez',\n",
       " 'biostrings',\n",
       " 'biojs-vis-hgv',\n",
       " 'ucsc genome browser',\n",
       " 'spnconverter',\n",
       " 'simextargid',\n",
       " 'marmoset',\n",
       " 'compareoverlapping',\n",
       " 'snagee',\n",
       " 'consensuspathdb',\n",
       " 'tfanalysis',\n",
       " 'cafe variome',\n",
       " 'sevenbridges',\n",
       " 'the flux simulator',\n",
       " 'catalysis-hub.org',\n",
       " 'rnaspades',\n",
       " 'sampledrift',\n",
       " 'readw',\n",
       " 'seqret (ebi)',\n",
       " 'mesbl gc-ms metabolite peak database',\n",
       " 'database of genes from ncbi refseq genomes',\n",
       " 'diversityprojects',\n",
       " 'getletterdistribution1',\n",
       " 'ngs_simulation',\n",
       " 'qvz',\n",
       " 'geecc',\n",
       " 'machado',\n",
       " 'datasets2tools',\n",
       " 'alliance genome resources',\n",
       " 'abi2xml',\n",
       " 'plotcoverage',\n",
       " 'genotypeeval',\n",
       " 'dasir',\n",
       " 'htseqcount',\n",
       " 'mousebytes',\n",
       " 'pdbusqlextractor',\n",
       " 'restricttranscriptlist',\n",
       " 'hyppo-x',\n",
       " 'msd file reader',\n",
       " 'crysplot',\n",
       " 'bowtie',\n",
       " 'grs',\n",
       " 'gs-lage',\n",
       " 'refseqget',\n",
       " 'masqot-gui',\n",
       " 'trimspace',\n",
       " 'graph extract',\n",
       " 'intermine',\n",
       " 'cshl_fastx_reverse_complement',\n",
       " 'ms-gf+',\n",
       " 'cshl_fastx_artifacts_filter',\n",
       " 'imiss',\n",
       " 'irefscape',\n",
       " 'phemto',\n",
       " 'gatk_realigner_target_creator',\n",
       " 'shortread',\n",
       " 'meigor',\n",
       " 'bissap',\n",
       " 'openswath',\n",
       " 'metabosearch',\n",
       " 'tvt viewer',\n",
       " 'tabsint',\n",
       " 'pulse',\n",
       " 'arrayqualitymetrics',\n",
       " 'ddbj',\n",
       " 'seda',\n",
       " 'igvtools',\n",
       " 'rbiopaxparser',\n",
       " 'ehecregnet',\n",
       " 'gdtools_union',\n",
       " 'scry',\n",
       " 'mubd-decoymaker 2.0',\n",
       " 'changename',\n",
       " 'cbs das protein viewer',\n",
       " 'genus biomolecules',\n",
       " 'cellhts2',\n",
       " 'lcmix',\n",
       " 'solid_qual_boxplot',\n",
       " 'rdgidb',\n",
       " 'scope',\n",
       " 'sting',\n",
       " 'gencompress',\n",
       " 'fgnet',\n",
       " 'bridgedbr',\n",
       " 'oncowiki',\n",
       " 'samjavascript',\n",
       " 'export',\n",
       " 'hgv_david',\n",
       " 'three to one',\n",
       " 'hypercldb',\n",
       " 'psmd',\n",
       " 'mbwa_wrapper',\n",
       " 'fasthmm',\n",
       " 'frogs_normalisation',\n",
       " 'macsyfinder',\n",
       " 'pindel',\n",
       " 'methylaid',\n",
       " 'patentin',\n",
       " 'idmappingretrieval',\n",
       " 'tileqc',\n",
       " 'fastool',\n",
       " 'pobo',\n",
       " 'ggost',\n",
       " 'saltclass',\n",
       " 'diversitycollection',\n",
       " 'breseq_bam2aln',\n",
       " 'cyrest',\n",
       " 'arrayquality',\n",
       " 'msmseda',\n",
       " 'idbac',\n",
       " 'fqextract',\n",
       " 'wptherml',\n",
       " 'seqassist',\n",
       " 'ppp',\n",
       " 'animalnexus',\n",
       " 'ordino',\n",
       " 'plaza monocots',\n",
       " 'mapperanalyzer',\n",
       " 'gatk_print_reads',\n",
       " 'the immune epitope database (iedb)',\n",
       " 'mqc',\n",
       " 'centiserver',\n",
       " 'gmap',\n",
       " 'mzwiff',\n",
       " 'setuppromoter',\n",
       " 'piqmie',\n",
       " 'bioplex 2.0',\n",
       " 'solid2fastq',\n",
       " 'ngs-qc generator',\n",
       " 'cellbaser',\n",
       " 'oncocnv',\n",
       " 'pchopper',\n",
       " 'gatk_depth_of_coverage',\n",
       " 'msgfdb2pepxml',\n",
       " 'genex_ontology',\n",
       " 'lux score',\n",
       " 'wft4galaxy',\n",
       " 'findpromoter',\n",
       " 'sva',\n",
       " 'countnumber_parallel',\n",
       " 'genosurf',\n",
       " 'mdpbiome',\n",
       " 'genomecrispr',\n",
       " 'pymzml',\n",
       " 'filtercontrol',\n",
       " 'depicter',\n",
       " 'jmztab-m',\n",
       " 'cytoconverter',\n",
       " 'samtools_sort',\n",
       " 'croco',\n",
       " 'samtools_flagstat',\n",
       " 'nnnorm',\n",
       " 'ophid',\n",
       " 'plottwist',\n",
       " 'svdetect_run_parallel_step',\n",
       " 'bceconsensus',\n",
       " 'rmscopii',\n",
       " 'turbonorm',\n",
       " 'mlml2r',\n",
       " 'ctc',\n",
       " 'fungenes',\n",
       " 'gatk_variant_recalibrator',\n",
       " 'flowai',\n",
       " 'mexcowalk',\n",
       " 'frogs_affiliation_otu',\n",
       " 'sceptre',\n",
       " 'dvd',\n",
       " 'rqc',\n",
       " 'shrec',\n",
       " 'dtsgui',\n",
       " 'mass-up',\n",
       " 'cshl_fastx_collapser',\n",
       " 'chordomics',\n",
       " 'pacsy',\n",
       " 'nthseq',\n",
       " 'american type culture collection (atcc)',\n",
       " 'clc_sequence_info',\n",
       " 'quosa',\n",
       " 'picard_arrg',\n",
       " 'interlab cell line collection catalogue (iclc)',\n",
       " 'bioproject',\n",
       " 'risa',\n",
       " 'uitagcreator',\n",
       " 'biojs-io-sam',\n",
       " 'pileup2cov',\n",
       " 'pcaviz',\n",
       " 'clc_extract_consensus',\n",
       " 'sos',\n",
       " 'annovar_summarize',\n",
       " 'gdtools_gd2gvf',\n",
       " 'mergetranscriptlists',\n",
       " 'gatk_variant_annotator',\n",
       " 'alyx',\n",
       " 'gdtools_intersect',\n",
       " 'cancerclass',\n",
       " 'jmzreader',\n",
       " 'chia-pet2',\n",
       " 'biojs-io-blast',\n",
       " 'chipmunk_chiphorde',\n",
       " 'pyteomics',\n",
       " 'vecscreen_plus_taxonomy',\n",
       " 'tripal mapviewer',\n",
       " 'nanostringqcpro',\n",
       " 'agilepairedendreadscombiner',\n",
       " 'micropita',\n",
       " 'p2s',\n",
       " 'biocworkflowtools',\n",
       " 'gatk_variant_select',\n",
       " 'samtools',\n",
       " 'affyio',\n",
       " 'bamstats',\n",
       " 'gatk_variant_filtration',\n",
       " 'msaccess',\n",
       " 'mothur_krona',\n",
       " 'sabund',\n",
       " 'zincpharmer',\n",
       " 'plier',\n",
       " 'dat2mgf',\n",
       " 'mosdepth',\n",
       " 'bio.tools sum',\n",
       " 'cipres',\n",
       " 'notab',\n",
       " 'combigor',\n",
       " 'tagdust',\n",
       " 'phenx toolkit',\n",
       " 'yarn',\n",
       " 'mascot parser',\n",
       " 'plaza dicots',\n",
       " 'romop',\n",
       " 'encyclopedia',\n",
       " 'chemdataextractor',\n",
       " 'scmap',\n",
       " 'mlda',\n",
       " 'rsffreader',\n",
       " 'moat',\n",
       " 'wikidata',\n",
       " 'thermorawfileparser',\n",
       " '3dbionotes',\n",
       " 'qcewas',\n",
       " 'genomehubs',\n",
       " 'axima2analyze',\n",
       " 'gro2mat',\n",
       " 'bedgraphtobigwig',\n",
       " 'amr',\n",
       " 'bel2abm',\n",
       " 'mzid',\n",
       " 'cytogps',\n",
       " 'lca',\n",
       " 'ithanet',\n",
       " 'pcscs',\n",
       " 'linearfold',\n",
       " 'arrowsam',\n",
       " 'roi_report',\n",
       " 's2o',\n",
       " 'gdtools_apply',\n",
       " 'imagej (nih)',\n",
       " 'cdfread',\n",
       " 'circbase',\n",
       " 'restrictsequencelist',\n",
       " 'plasmo',\n",
       " 'affxparser',\n",
       " 'metabolic',\n",
       " 'ipd project',\n",
       " 'regate',\n",
       " 'odonate phenotypic database',\n",
       " 'decomptumor2sig',\n",
       " 'mascot2xml',\n",
       " 'm2lite',\n",
       " 'cytoscape web',\n",
       " 'roi_pca',\n",
       " 'gmap_iit_store',\n",
       " 'metaarray',\n",
       " 'alias',\n",
       " 'sampler',\n",
       " 'bactag',\n",
       " 'qualimap',\n",
       " 'tbl2knowngene',\n",
       " 'noisy',\n",
       " 'kccg patient archive',\n",
       " 'compassxport',\n",
       " 'hgv_codingsnps',\n",
       " 'hgv_sift',\n",
       " 'objtables',\n",
       " 'sig2biopax',\n",
       " 'vcfr',\n",
       " 'civicpy',\n",
       " 'jphyloio',\n",
       " 'cshl_fastx_renamer',\n",
       " 'datadiscovery',\n",
       " 'idmappinganalysis',\n",
       " 'biojs-vis-feature',\n",
       " 'converter_genbank_to_fasta',\n",
       " 'pannbuilder',\n",
       " 'kegdraw 0.1.14 beta',\n",
       " 'variationget',\n",
       " 'st-sim',\n",
       " 'genbank_convertor',\n",
       " 'gtool',\n",
       " 'empusa',\n",
       " 'affygcqc',\n",
       " 'sbml2latex',\n",
       " 'anabel',\n",
       " 'civicmine',\n",
       " 'rsat retrieve-ensembl-seq',\n",
       " 'multi-q',\n",
       " 'fqduplicate',\n",
       " 'pdbj',\n",
       " 'biosystems',\n",
       " 'ordo',\n",
       " 'trowel',\n",
       " 'circos',\n",
       " 'frogs_demultiplex',\n",
       " 'listannotation',\n",
       " 'seqstrap',\n",
       " 'vcf-server',\n",
       " 'rdml',\n",
       " 'cct',\n",
       " 'svdetect_bam_preprocessing_pairs',\n",
       " 'mbcb',\n",
       " 'mres2x',\n",
       " 'genenames.org',\n",
       " 'arraytools',\n",
       " 'art',\n",
       " 'qq-snv',\n",
       " 'mapstore',\n",
       " 'human dephosphorylation database  (depod)',\n",
       " 'image sxm',\n",
       " 'tableparser',\n",
       " 'trimadaptor',\n",
       " 'pygmql',\n",
       " 'rsbml',\n",
       " 'keggrest',\n",
       " 'fconverter',\n",
       " 'yabi',\n",
       " 'abi2link',\n",
       " 'otusfocus',\n",
       " 'enmix',\n",
       " 'bsgenome',\n",
       " 'picardasmetrics',\n",
       " 'tfextract',\n",
       " 'pandoc scholar',\n",
       " 'diversitytaxonnames',\n",
       " 'printsextract',\n",
       " 'snpator',\n",
       " 'writerestohtml',\n",
       " 'interelementgff',\n",
       " 'countreadgcpercent',\n",
       " 'rgestlibcomp',\n",
       " 'tximport',\n",
       " 'pyradiomics',\n",
       " 'xmltext',\n",
       " 'additionally analogous',\n",
       " 'bcftools_view',\n",
       " 'tsdd',\n",
       " 'peff',\n",
       " 'imgt 2dstructure-db',\n",
       " 'gorth',\n",
       " 'ensembl apl',\n",
       " 'nxtrim',\n",
       " 'imgt 3dstructure-db',\n",
       " 'annotationhubdata',\n",
       " 'gatk_variant_combine',\n",
       " 'dna chromatogram explorer',\n",
       " 'impact framework',\n",
       " 'scalce',\n",
       " 'hgv_add_scores',\n",
       " 'pdinfobuilder',\n",
       " 'mspire_mspire-sequest',\n",
       " 'annotationtools',\n",
       " 'drug-herb-interaction',\n",
       " 'agilefileconverter',\n",
       " 'spiral genetics',\n",
       " 'sepath',\n",
       " 'bionode-sra',\n",
       " 'celltag',\n",
       " 'mzjava',\n",
       " 'metalims',\n",
       " 'gatk_haplotype_caller',\n",
       " 'maizesnpdb',\n",
       " 'rankoptimizer',\n",
       " 'getintrons',\n",
       " 'estprep',\n",
       " 'fastproject',\n",
       " 'biojs-io-annots',\n",
       " 'mitre',\n",
       " 'sartools_deseq2',\n",
       " 'covidmine',\n",
       " 'ngsreports',\n",
       " 'pbg-ld',\n",
       " 'ageguess',\n",
       " 'vmd store',\n",
       " 'fastqc_toolshed',\n",
       " 'cellfishing.jl',\n",
       " 'sabre',\n",
       " 'clc_mapper',\n",
       " 'bionimbus',\n",
       " 'orfer',\n",
       " 'emblmygff3',\n",
       " 'ms-isotope',\n",
       " 'wikipathways',\n",
       " 'cohorts',\n",
       " 'paxtoolsr',\n",
       " 'wellinverter',\n",
       " 'ehrtemporalvariability',\n",
       " 'evofreq',\n",
       " 'tophat2',\n",
       " 'odg',\n",
       " 'aqua-mer',\n",
       " 'html4blast',\n",
       " 'encodexplorer',\n",
       " 'diversityimageinspector',\n",
       " 'bridgedb api',\n",
       " 'staphopia',\n",
       " 'seal',\n",
       " 'sartools_edger',\n",
       " 'gdtools_validate',\n",
       " 'velvet',\n",
       " 'biomart',\n",
       " 'cov2html',\n",
       " 'createcontrolsubset',\n",
       " 'mutanalyst',\n",
       " 'hitec',\n",
       " 'platecore',\n",
       " 'metabase',\n",
       " 'void editor',\n",
       " 'affycompatible',\n",
       " 'biomodels database converters',\n",
       " 'shutu',\n",
       " 'crop',\n",
       " 'ognm',\n",
       " 'macs',\n",
       " 'granvil',\n",
       " 'selectbesthit',\n",
       " 'prcbench',\n",
       " 'reganalyst',\n",
       " 'picardinsertsize',\n",
       " 't-psi-c',\n",
       " 'miamexpress',\n",
       " 'proteinprospector',\n",
       " 'biolad-db',\n",
       " 'ngsrelatev2',\n",
       " 'almostsignificant',\n",
       " 'famfetch',\n",
       " 'mmcif dictionary suite',\n",
       " 'tomahaqcompanion',\n",
       " 'sam2tsv',\n",
       " 'rgpicfixmate',\n",
       " 'fasta canonical sequences',\n",
       " 'plato',\n",
       " 'useq',\n",
       " 'doubletdetection',\n",
       " 'mac-inmv-ssr',\n",
       " 'bioqc',\n",
       " 'hgv_lps',\n",
       " 'myseq',\n",
       " 'trans-nanosim',\n",
       " 'wheatis',\n",
       " 'gdtools_mask',\n",
       " 'clusterizebyslidingwindows',\n",
       " 'uniprot.ws',\n",
       " 'fas2nex',\n",
       " 'jaspextract',\n",
       " 'compareoverlappingsmallref',\n",
       " 'vrtk',\n",
       " 'frogs_biom_to_tsv',\n",
       " 'lsd 3.0',\n",
       " 'gdtools_annotate',\n",
       " 'clc_find_variations',\n",
       " 'bedtools_mergebedgraph',\n",
       " 'nwdisplay',\n",
       " 'mbv',\n",
       " 'judi',\n",
       " 'samtools_filter',\n",
       " 'psvs',\n",
       " 'gtz',\n",
       " 'diseasecard',\n",
       " 'qseq2fastq',\n",
       " 'seedgff',\n",
       " 'prosight ptm',\n",
       " 'mstmap',\n",
       " 'schicnorm',\n",
       " 'amos',\n",
       " 'webspecmine',\n",
       " 'mzbruker',\n",
       " 'scoc database',\n",
       " '3seq_2d',\n",
       " 'gatk_base_recalibrator',\n",
       " 'msfilereader_python_bindings',\n",
       " 'online chemical modeling environment',\n",
       " 'biomformat',\n",
       " 'fit',\n",
       " 'benchmarkncvtools',\n",
       " 'bindingdb database of measured binding affinities',\n",
       " 'plasmidspades',\n",
       " 'msdb',\n",
       " 'structure_threader',\n",
       " 'dta to mgf file converter',\n",
       " 'hmmcopy',\n",
       " 'xmlget',\n",
       " 'sirnadb',\n",
       " 'motivevalidator',\n",
       " 'mastervar2vcf4.1',\n",
       " 'conservation archive',\n",
       " 'acdc',\n",
       " 'cuffcompare',\n",
       " 'fqquality',\n",
       " 'ms-data-core-api',\n",
       " 'quantest',\n",
       " 'readqpcr',\n",
       " 'probamr',\n",
       " 'epivizrdata',\n",
       " 'masshunter file reader',\n",
       " 'kpath',\n",
       " 'protmiscuity',\n",
       " 'taxonomy_krona_chart',\n",
       " 'acdpretty',\n",
       " 'radar-base',\n",
       " 'sdsw',\n",
       " 'berex',\n",
       " 'illuminaio',\n",
       " 'biomaj',\n",
       " 'phenomecentral',\n",
       " 'bionode-fasta',\n",
       " 'coryneregnet',\n",
       " 'drfit',\n",
       " 'kronaextract',\n",
       " 'rawtools',\n",
       " 'lnkg2lmlnk',\n",
       " 'quality assessment',\n",
       " 'biocstyle',\n",
       " 'rgalaxy',\n",
       " 'mesophotic.org',\n",
       " 'chipper',\n",
       " 'pyrocleaner',\n",
       " 'pubmed',\n",
       " 'xia2',\n",
       " 'seqtypedetect',\n",
       " 'biocfilecache',\n",
       " 'clc_assembler',\n",
       " 'fastq_groomer_parallel',\n",
       " 'affyrnadegradation',\n",
       " 'indices_diversity',\n",
       " 'toff',\n",
       " 'idconvert',\n",
       " 'frogs_upload_tar',\n",
       " 'orcid api',\n",
       " 'the hymenoptera genome database (hgd)',\n",
       " 'bionode-ncbi',\n",
       " 'zr-mofs',\n",
       " 'recphyloxml',\n",
       " 'recon',\n",
       " 'ispider central',\n",
       " 'disqover',\n",
       " 'qmsat',\n",
       " 'psafe',\n",
       " 'modern research data portal',\n",
       " 'airlab',\n",
       " 'cytomegalovirus database (cmvdb)',\n",
       " 'acdtrace',\n",
       " 'genetests',\n",
       " 'doc2hpo',\n",
       " 'evex',\n",
       " 'opentrials api',\n",
       " 'disgenet',\n",
       " 'fastqc',\n",
       " 'geonode-tools4msp',\n",
       " 'stellar',\n",
       " 'msquant',\n",
       " 'asm2ace',\n",
       " 'cosinor',\n",
       " 'frogs_clusters_stat',\n",
       " ...}"
      ]
     },
     "execution_count": 78,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "all_ETL_tools = merge_tools_lists(ETL_results_general)\n",
    "all_ETL_tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1307"
      ]
     },
     "execution_count": 79,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(all_ETL_tools)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Case 2: Ontology tools discovery"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "In this case, the file of zooma results is `ontology_annotation_EDAM_curated_csv.csv`."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Step 1: parsing zooma results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['operation_0226', 'operation_3778', 'topic_0089', 'operation_3280']\n"
     ]
    }
   ],
   "source": [
    "terms_file='ontology_annotation_EDAM_curated_csv.csv'\n",
    "ontology_edam_terms = parse_zooma_results(terms_file)\n",
    "print(ontology_edam_terms)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Step 2: doing the query. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_0226\"&page=14&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_3778\"&page=5&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?topicID=\"topic_0089\"&page=29&format=json\n",
      "Requests finished.\n",
      "Starting making the requests...\n",
      "Requesting: https://bio.tools/api/tool/?operationID=\"operation_3280\"&page=6&format=json\n",
      "Requests finished.\n"
     ]
    }
   ],
   "source": [
    "ontology_results_general, ontology_results_detailed  = query_for_EDAM_terms(ontology_edam_terms)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Total number of terms"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "4"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(ontology_results_general.keys())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Tools per term"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'operation_0226': 145,\n",
       " 'operation_3280': 61,\n",
       " 'operation_3778': 48,\n",
       " 'topic_0089': 305}"
      ]
     },
     "execution_count": 61,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "count_tools_per_term(ontology_results_general)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Lists of tool names per term\n",
    "This is handy for validation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'operation_0226': 0                     treeio\n",
       " 1      workflow4metabolomics\n",
       " 2                globalfungi\n",
       " 3                      amtdb\n",
       " 4              causalbuilder\n",
       "                ...          \n",
       " 140               metdisease\n",
       " 141                     magi\n",
       " 142                    npact\n",
       " 143                     cogs\n",
       " 144                    marky\n",
       " Name: name, Length: 145, dtype: object, 'operation_3280': 0         dquest\n",
       " 1        ontokin\n",
       " 2         nlimed\n",
       " 3           vist\n",
       " 4       eventepi\n",
       "          ...    \n",
       " 56         acela\n",
       " 57          deca\n",
       " 58    facta plus\n",
       " 59    simconcept\n",
       " 60        bagel2\n",
       " Name: name, Length: 61, dtype: object, 'operation_3778': 0                                   webanno\n",
       " 1                        monarch initiative\n",
       " 2                            mesophotic.org\n",
       " 3               medical relation extraction\n",
       " 4                                  snpnotes\n",
       " 5                                 civicmine\n",
       " 6                                     koffi\n",
       " 7                                    hmtvar\n",
       " 8                                  propheno\n",
       " 9                                   ppr-ssm\n",
       " 10                               umlsmapper\n",
       " 11                                 genome3d\n",
       " 12                                   anabel\n",
       " 13                                      ads\n",
       " 14                         pubtator central\n",
       " 15                              ril-contour\n",
       " 16                                 disqover\n",
       " 17                                mutalyzer\n",
       " 18                                mutalyzer\n",
       " 19                                      hpo\n",
       " 20                                      hpo\n",
       " 21                                peregrine\n",
       " 22                            omim explorer\n",
       " 23                              lailaps-qsm\n",
       " 24                                  calchas\n",
       " 25                                    craft\n",
       " 26                                     rash\n",
       " 27                                  myminer\n",
       " 28                                      goa\n",
       " 29                    dictybase update 2011\n",
       " 30                                    aspgd\n",
       " 31              candida genome database cgd\n",
       " 32                                 geneview\n",
       " 33                                    cargo\n",
       " 34                        yeast metaboliner\n",
       " 35                                 typeinfo\n",
       " 36                                  pombase\n",
       " 37    drosophila dnase i footprint database\n",
       " 38                                 pubtator\n",
       " 39                                   hubmed\n",
       " 40                                   sci.ai\n",
       " 41                                    acela\n",
       " 42                                  extract\n",
       " 43                               charprotdb\n",
       " 44                                    ideal\n",
       " 45                                   biolit\n",
       " 46                                   gogene\n",
       " 47                                 litminer\n",
       " Name: name, dtype: object, 'topic_0089': 0                 fhir\n",
       " 1             pcosbase\n",
       " 2              ontokin\n",
       " 3              onclass\n",
       " 4           odmltables\n",
       "             ...       \n",
       " 300                cbo\n",
       " 301             biolit\n",
       " 302             jprogo\n",
       " 303    microbe browser\n",
       " 304           chipinfo\n",
       " Name: name, Length: 305, dtype: object}"
      ]
     },
     "execution_count": 80,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "tools_per_term(ontology_results_general)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Merged tools lists\n",
    "All tools discovered merged in a single set"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'a4preproc',\n",
       " 'aber-owl',\n",
       " 'acela',\n",
       " 'aceparser',\n",
       " 'ads',\n",
       " 'agrigo',\n",
       " 'ahtpdb',\n",
       " 'aiso',\n",
       " 'ajia',\n",
       " 'alliance genome resources',\n",
       " 'amigo 2',\n",
       " 'amtdb',\n",
       " 'anabel',\n",
       " 'aniseed',\n",
       " 'annot',\n",
       " 'annotationfilter',\n",
       " 'annotsv',\n",
       " 'apache taverna',\n",
       " 'arachnoserver',\n",
       " 'aristo',\n",
       " 'asa-ps',\n",
       " 'aspgd',\n",
       " 'association viewer',\n",
       " 'bagel2',\n",
       " 'becas',\n",
       " 'beere',\n",
       " 'bel commons',\n",
       " 'bel2abm',\n",
       " 'belief',\n",
       " 'bgeedb',\n",
       " 'bio.tools',\n",
       " 'bio2bel',\n",
       " 'biobert',\n",
       " 'biocaddie',\n",
       " 'biocatalogue',\n",
       " 'bioecs',\n",
       " 'bioexcel genome sequence quality control workflow',\n",
       " 'biogateway',\n",
       " 'biojs-vis-gprofiler',\n",
       " 'biolabeler',\n",
       " 'biolit',\n",
       " 'biomoby',\n",
       " 'biomodels database',\n",
       " 'bionda - biomarker database',\n",
       " 'bioner',\n",
       " 'bionet-mining',\n",
       " 'bioportal',\n",
       " 'bior',\n",
       " 'bittersweet',\n",
       " 'biwalklda',\n",
       " 'bo-lstm',\n",
       " 'brisskit',\n",
       " 'bubastis',\n",
       " 'bugbuilder',\n",
       " 'cagefightr',\n",
       " 'calchas',\n",
       " 'candida genome database cgd',\n",
       " 'card',\n",
       " 'cargo',\n",
       " 'carmo',\n",
       " 'causalbuilder',\n",
       " 'caw',\n",
       " 'cbo',\n",
       " 'cddapp',\n",
       " 'cdfread',\n",
       " 'cedar ondemand',\n",
       " 'cellbaser',\n",
       " 'ceu mass mediator',\n",
       " 'charprotdb',\n",
       " 'chebi ws',\n",
       " 'chemdistiller',\n",
       " 'chimera',\n",
       " 'chipinfo',\n",
       " 'ciona robusta anatomy and development ontology',\n",
       " 'circular gene ontology (cirgo)',\n",
       " 'civicmine',\n",
       " 'classyfire',\n",
       " 'clinical metadata ontology (cmdo)',\n",
       " 'clinotator',\n",
       " 'cocitestats',\n",
       " 'coeus',\n",
       " 'cogs',\n",
       " 'collabonet',\n",
       " 'collector',\n",
       " 'combigor',\n",
       " 'compendium of protein lysine acetylation (cpla)',\n",
       " 'copheism',\n",
       " 'cpgavas',\n",
       " 'craft',\n",
       " 'crispy',\n",
       " 'crowdgo',\n",
       " 'cui2vec',\n",
       " 'dasir',\n",
       " 'david bioinformatics resources',\n",
       " 'dbcan',\n",
       " 'dbcrid',\n",
       " 'dcgor',\n",
       " 'ddbj',\n",
       " 'deca',\n",
       " 'decres',\n",
       " 'deeppheno',\n",
       " 'deeppvp',\n",
       " 'deepred',\n",
       " 'defume',\n",
       " 'delayedarray',\n",
       " 'descseq',\n",
       " 'diana-lncbase',\n",
       " 'diana-microt v5.0',\n",
       " 'diana-mirgen',\n",
       " 'dictybase update 2011',\n",
       " 'dictyexpress',\n",
       " 'digchem',\n",
       " 'digsee',\n",
       " 'dingo',\n",
       " 'disgenet',\n",
       " 'disqover',\n",
       " 'diversity workbench',\n",
       " 'diversitycollection',\n",
       " 'diversityreferences',\n",
       " 'diversityscientificterms',\n",
       " 'diversitytaxonnames',\n",
       " 'dnatraffic',\n",
       " 'do',\n",
       " 'doc2hpo',\n",
       " 'dquest',\n",
       " 'drar-cpi',\n",
       " 'drawglycan-snfg',\n",
       " 'drosophila dnase i footprint database',\n",
       " 'easy-hla',\n",
       " 'ebi search',\n",
       " 'edamdef',\n",
       " 'edamhasinput',\n",
       " 'edamhasoutput',\n",
       " 'edamisformat',\n",
       " 'edamisid',\n",
       " 'edamname',\n",
       " 'elmmo',\n",
       " 'emma2 ws',\n",
       " 'enrichment map cytoscape plugin',\n",
       " 'enrichplot',\n",
       " 'est2assembly',\n",
       " 'estscan',\n",
       " 'etm-db',\n",
       " 'europhenome',\n",
       " 'eventepi',\n",
       " 'evntextrc',\n",
       " 'exoscore',\n",
       " 'extract',\n",
       " 'facta plus',\n",
       " 'faf-drugs',\n",
       " 'fair',\n",
       " 'fairdatapoint',\n",
       " 'famplex',\n",
       " 'fasto',\n",
       " 'feelnc',\n",
       " 'ffpred',\n",
       " 'fhir',\n",
       " 'flowrepositoryr',\n",
       " 'flyode',\n",
       " 'flytf',\n",
       " 'fontcell',\n",
       " 'funcexplorer',\n",
       " 'funmappone',\n",
       " 'fusionanalyser',\n",
       " 'g-sesame',\n",
       " 'g:profiler',\n",
       " 'galactic',\n",
       " 'gbol',\n",
       " 'gcocoa',\n",
       " 'gdsarray',\n",
       " 'gedex',\n",
       " 'gemtractor',\n",
       " 'gene ontology (go)',\n",
       " 'gene ontology browser',\n",
       " 'genease',\n",
       " 'generic go term finder',\n",
       " 'generic go term mapper',\n",
       " 'geneview',\n",
       " 'genex_ontology',\n",
       " 'genfam',\n",
       " 'genome trax',\n",
       " 'genome3d',\n",
       " 'genomicscores',\n",
       " 'genosurf',\n",
       " 'geseq',\n",
       " 'gfbio',\n",
       " 'ggost',\n",
       " 'gimli',\n",
       " 'globalfungi',\n",
       " 'go enrichment analysis tool',\n",
       " 'go-elite',\n",
       " 'go2msig',\n",
       " 'go2vec',\n",
       " 'goa',\n",
       " 'goa_gubbar',\n",
       " 'goblet',\n",
       " 'gocats',\n",
       " 'godef',\n",
       " 'goeast',\n",
       " 'gofuncr',\n",
       " 'gogene',\n",
       " 'gomap',\n",
       " 'gomapman',\n",
       " 'gominer ws',\n",
       " 'goname',\n",
       " 'gonet',\n",
       " 'gonuts',\n",
       " 'gopubmed',\n",
       " 'gorilla',\n",
       " 'gosemsim',\n",
       " 'gosim',\n",
       " 'goslin',\n",
       " 'gostag',\n",
       " 'gostats',\n",
       " 'gotools',\n",
       " 'gotrapper',\n",
       " 'gpannotate',\n",
       " 'gprofiler',\n",
       " 'gpsdb',\n",
       " 'grasp',\n",
       " 'gsan',\n",
       " 'gsso',\n",
       " 'gsvec',\n",
       " 'guide',\n",
       " 'gwastools',\n",
       " 'hgt-id',\n",
       " 'hgv_david',\n",
       " 'hgv_linktogprofile',\n",
       " 'higdb',\n",
       " 'hio',\n",
       " 'hipathia',\n",
       " 'histone coder',\n",
       " 'hmtvar',\n",
       " 'holovir',\n",
       " 'hpo',\n",
       " 'hpo2go',\n",
       " 'hubmed',\n",
       " 'human gene nomenclature database (hgnc)',\n",
       " 'hyper',\n",
       " 'iambee',\n",
       " 'icd-10',\n",
       " 'ictnet2',\n",
       " 'ideal',\n",
       " 'ietd',\n",
       " 'ikap_cells',\n",
       " 'imgt, the international immunogenetics information system',\n",
       " 'impres',\n",
       " 'inaturalist',\n",
       " 'ipd project',\n",
       " 'iscan',\n",
       " 'isotopident',\n",
       " 'itextmine',\n",
       " 'ivariantguide',\n",
       " 'j-express',\n",
       " 'jmztab-m',\n",
       " 'jprogo',\n",
       " 'karma',\n",
       " 'kbase',\n",
       " 'kccg patient archive',\n",
       " 'kegganim',\n",
       " 'keggprofile',\n",
       " 'kinetoplastid genome resources (tritrypdb)',\n",
       " 'koffi',\n",
       " 'l2l',\n",
       " 'lailaps-qsm',\n",
       " 'lion-web',\n",
       " 'lipid data analyzer (lda)',\n",
       " 'lipid mini-on',\n",
       " 'list ontologies',\n",
       " 'litinspector',\n",
       " 'litminer',\n",
       " 'lobstahs',\n",
       " 'locdb',\n",
       " 'loinc2hpo',\n",
       " 'lungmap',\n",
       " 'macie',\n",
       " 'magecomet',\n",
       " 'magi',\n",
       " 'mambo',\n",
       " 'mamot',\n",
       " 'markerhub',\n",
       " 'marky',\n",
       " 'matscholar',\n",
       " 'mbrole',\n",
       " 'medextractr',\n",
       " 'medical relation extraction',\n",
       " 'meme suite',\n",
       " 'meshsim',\n",
       " 'mesophotic.org',\n",
       " 'messa',\n",
       " 'metdisease',\n",
       " 'metfamily',\n",
       " 'mg-rast',\n",
       " 'microbe browser',\n",
       " 'microtss',\n",
       " 'mir-prefer',\n",
       " 'monarch initiative',\n",
       " 'morphdb',\n",
       " 'mouse phenome database',\n",
       " 'mousebook',\n",
       " 'mpagenomics',\n",
       " 'mtgo-sc',\n",
       " 'mucor',\n",
       " 'multiloc2',\n",
       " 'mutalyzer',\n",
       " 'mvgst',\n",
       " 'myminer',\n",
       " 'nadfinder',\n",
       " 'nbdc rdf portal',\n",
       " 'nbrp resources',\n",
       " 'nemine',\n",
       " 'neuropp',\n",
       " 'neuroscience information framework',\n",
       " 'ngly1',\n",
       " 'nlimed',\n",
       " 'nlprot',\n",
       " 'nlr-parser',\n",
       " 'nmrml converter',\n",
       " 'noa',\n",
       " 'npact',\n",
       " 'odae',\n",
       " 'odg',\n",
       " 'odmltables',\n",
       " 'ols dialog',\n",
       " 'omadb',\n",
       " 'ometa',\n",
       " 'omim explorer',\n",
       " 'onassis',\n",
       " 'onclass',\n",
       " 'onesense',\n",
       " 'ontobrowser',\n",
       " 'ontocat',\n",
       " 'ontocount',\n",
       " 'ontoget',\n",
       " 'ontogetcommon',\n",
       " 'ontogetdown',\n",
       " 'ontogetobsolete',\n",
       " 'ontogetroot',\n",
       " 'ontogetsibs',\n",
       " 'ontogetup',\n",
       " 'ontoisobsolete',\n",
       " 'ontokin',\n",
       " 'ontologizer',\n",
       " 'ontology lookup service',\n",
       " 'ontoproc',\n",
       " 'ontoquest ws',\n",
       " 'ontoquestmain',\n",
       " 'ontotext',\n",
       " 'openebench',\n",
       " 'openflu',\n",
       " 'openms',\n",
       " 'ordo',\n",
       " 'organism.dplyr',\n",
       " 'orphadata',\n",
       " 'pageman',\n",
       " 'pannotator',\n",
       " 'pannzer',\n",
       " 'pannzer2',\n",
       " 'parameciumdb',\n",
       " 'pavian',\n",
       " 'pbg-ld',\n",
       " 'pcosbase',\n",
       " 'pcpheno',\n",
       " 'pdbj',\n",
       " 'pdzpepint',\n",
       " 'peakanalyzer',\n",
       " 'peregrine',\n",
       " 'phen2gene',\n",
       " 'phenohm',\n",
       " 'phenopolis',\n",
       " 'phenoscape',\n",
       " 'phenotator',\n",
       " 'pint',\n",
       " 'plan2l',\n",
       " 'plant experimental assay ontology (peao)',\n",
       " 'plantcircnet',\n",
       " 'plato plugin for autocomplete on ontologies',\n",
       " 'poeas',\n",
       " 'pogos',\n",
       " 'pombase',\n",
       " 'pomelo ii',\n",
       " 'ppr-ssm',\n",
       " 'predictprotein open',\n",
       " 'progenetix',\n",
       " 'proloc-go',\n",
       " 'propheno',\n",
       " 'protege 5.0',\n",
       " 'protein naming utility',\n",
       " 'pubtator',\n",
       " 'pubtator central',\n",
       " 'pure',\n",
       " 'pyiomica',\n",
       " 'pyradiomics',\n",
       " 'python video annotator',\n",
       " 'qsbio',\n",
       " 'qtl tableminer++(qtm)',\n",
       " 'quickgo',\n",
       " 'quin',\n",
       " 'ramigo',\n",
       " 'rash',\n",
       " 'rast',\n",
       " 'ratt',\n",
       " 'rdavidwebservice',\n",
       " 'rdfscape',\n",
       " 'rdrf',\n",
       " 'relna',\n",
       " 'riken integrated database of mammals',\n",
       " 'ril-contour',\n",
       " 'rnaindel',\n",
       " 'robot',\n",
       " 'rols',\n",
       " 'roma',\n",
       " 'romop',\n",
       " 'rulego',\n",
       " 'samsa2',\n",
       " 'sbpks2',\n",
       " 'scdo',\n",
       " 'sci.ai',\n",
       " 'seaweb',\n",
       " 'seclaf',\n",
       " 'secstrannotator',\n",
       " 'semdist',\n",
       " 'seqexpress',\n",
       " 'seqhound',\n",
       " 'sequence similarity maps (ssm)',\n",
       " 'sevenc',\n",
       " 'shinygo',\n",
       " 'simconcept',\n",
       " 'simdef',\n",
       " 'sinopedia',\n",
       " 'skimmr',\n",
       " 'smog',\n",
       " 'snomed ct',\n",
       " 'snpnotes',\n",
       " 'soba: sequence ontology bioinformatics analysis',\n",
       " 'sparql',\n",
       " 'spex2',\n",
       " 'starpdb',\n",
       " 'sts-nlsp',\n",
       " 'superfamily',\n",
       " 'synteny browser',\n",
       " 'tableannotator',\n",
       " 'thbp',\n",
       " 'the brenda tissue ontology (bto)',\n",
       " 'the comparative toxicogenomics database (ctd)',\n",
       " 'the protein ontology',\n",
       " 'tm-imc',\n",
       " 'togogenome',\n",
       " 'tooldog',\n",
       " 'topgo',\n",
       " 'trackfind',\n",
       " 'treeio',\n",
       " 'trifle',\n",
       " 'tromer',\n",
       " 'ts-goea',\n",
       " 'typeinfo',\n",
       " 'umlsmapper',\n",
       " 'uniconsig',\n",
       " 'variant',\n",
       " 'varimerge',\n",
       " 'vastdb',\n",
       " 'vio',\n",
       " 'viralzone',\n",
       " 'viseago',\n",
       " 'vist',\n",
       " 'webanno',\n",
       " 'webapollo',\n",
       " 'wego',\n",
       " 'wego2.0',\n",
       " 'workflow4metabolomics',\n",
       " 'wsd',\n",
       " 'wtfgenes',\n",
       " 'xenbase',\n",
       " 'yeast metaboliner',\n",
       " 'yogy',\n",
       " 'zooma'}"
      ]
     },
     "execution_count": 81,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "all_ontology_tools = merge_tools_lists(ontology_results_general)\n",
    "all_ontology_tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "476"
      ]
     },
     "execution_count": 82,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(all_ontology_tools)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
